{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Sequence Alignments \n",
    "\n",
    "It is common to want to align sequences.  BioRuby can do this for you in simple cases:\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a?gc?\n",
      "ahgcr\n"
     ]
    }
   ],
   "source": [
    "require 'bio'\n",
    "\n",
    "seqs = [ 'atgca', 'aagca', 'acgca', 'acgcg' ]\n",
    "seqs = seqs.collect{ |x| Bio::Sequence::NA.new(x) }\n",
    "# creates alignment object\n",
    "a = Bio::Alignment.new(seqs)\n",
    "puts a.consensus \n",
    "puts a.consensus_iupac"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Try adding a few nucleotides to one of the sequences.  What happens in the alignment?  Add the same nucleotides to the other sequences, one-by-one... what happens in the alignment?   What do you learn from this?\n",
    "\n",
    "\n",
    "### more complicated alignments, e.g. with gaps, requires an alignment algorithm.\n",
    "\n",
    "A common way to do alignments is ClustalW.  This is installed on your virtual machine.  \n",
    "\n",
    "To access ClustalW from BioRuby's Alignment object, you do this:\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "BioRuby simple alignment\n",
      "??g???t?\n",
      "mhgsddt?\n",
      "\n",
      "ClustalW alignment\n",
      "???G???T?\n",
      "?wvgyrkt?\n"
     ]
    }
   ],
   "source": [
    "require 'bio'\n",
    "\n",
    "seqs = [ 'ctggtatt', 'aagcatt', 'acgcagtt', 'acgcgtt' ]\n",
    "seqs = seqs.collect{ |x| Bio::Sequence::NA.new(x) }\n",
    "# creates alignment object\n",
    "a = Bio::Alignment.new(seqs)\n",
    "puts \"BioRuby simple alignment\"\n",
    "puts a.consensus_string\n",
    "puts a.consensus_iupac\n",
    "puts\n",
    "puts \"ClustalW alignment\"\n",
    "factory = Bio::ClustalW.new\n",
    "a2 = a.do_align(factory)\n",
    "puts a2.consensus_string\n",
    "puts a2.consensus_iupac"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Sequence Searching\n",
    "\n",
    "We will look at sequence searches using FASTA and BLAST.  Both of these have been installed for you.\n",
    "\n",
    "See docs at http://bioruby.org/rdoc/Bio/Fasta and ....\n",
    "\n",
    "I have downloaded 160 Ebola virus sequences for you in the \"Databases\" folder.  You need to concatenate all of the single-sequence FASTA files into a single FASTA file, which is used as the database for a FASTA search:\n",
    "\n",
    "    $ cat *.fa  >  EBOLA.fa"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "#  <span style=\"color:red;\">NOTE:  BioRuby's FASTA report parser is broken</span>\n",
    "\n",
    "<span style=\"color:red;\">The code below WILL NOT WORK right now. However, you can look at the structure of the code to understand what it is supposed to do...</span>"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "require 'bio'\n",
    "require 'stringio'\n",
    "\n",
    "factory = Bio::Fasta.local('/home/osboxes/fasta-36.3.8d/bin/fasta36', '/home/osboxes/Sequences/EBOLA.fa') \n",
    "find = StringIO.new(\">myseq\\nAGCCCCGTTTTTTGCCACTGGTTATTTAATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\")\n",
    "\n",
    "ff = Bio::FlatFile.auto(find)   # 'find' is a streamable object\n",
    "\n",
    "# Iterates over each entry. the variable \"entry\" is a Bio::FastaFormat object:\n",
    "\n",
    "ff.each do |entry|\n",
    "   \n",
    "  # executes homology search. Returns Bio::Fasta::Report object.\n",
    "  report = factory.query(entry)\n",
    "\n",
    "  # Iterates over each hit\n",
    "  report.each do |hit|\n",
    "      print \"#{hit.query_id} : evalue #{hit.evalue}\\t#{hit.target_id} at \"\n",
    "      p hit.lap_at\n",
    "  end\n",
    "end"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# BLAST searches\n",
    "\n",
    "to install Blast for yourself:  \n",
    "    \n",
    "    $  sudo apt-get install ncbi-blast+\n",
    "    \n",
    "    $  sudo apt-get install ncbi-blast+-legacy   (this is required for BioRuby!)\n",
    "    \n",
    "### Custom BLAST databases\n",
    "    \n",
    "Blast requires:\n",
    "\n",
    "    A sequence to be used as the query\n",
    "\n",
    "    A BLAST-formatted database\n",
    "\n",
    "Common BLAST databases can be downloaded from NCBI:\n",
    "ftp://ftp.ncbi.nlm.nih.gov/blast/db/ \n",
    "\n",
    "We will create our own small database for this class using the EBOLA data.\n",
    "\n",
    "First, create a new folder \"Databases\" in your home directory, and **copy** EBOLA.fa into that location.\n",
    "\n",
    "Now, execute this command:\n",
    "\n",
    "    $  makeblastdb -in EBOLA.fa -dbtype 'nucl' -out EBOLA\n",
    "\n",
    "This creates some index files that are used by BLAST to speed-up the search.    \n",
    "    \n",
    "The search itself is very similar to the FASTA search above, but a little bit easier because the search query can be passed as a string:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "gnl|BL_ORD_ID|154 : evalue 9.63642e-21\t154 at [1, 64, 17514, 17583]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGTTATTTAATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|41 : evalue 9.63642e-21\t41 at [1, 64, 17514, 17583]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGTTATTTAATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|40 : evalue 9.63642e-21\t40 at [1, 64, 17514, 17583]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGTTATTTAATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|39 : evalue 9.63642e-21\t39 at [1, 64, 17514, 17583]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGTTATTTAATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|38 : evalue 9.63642e-21\t38 at [1, 64, 17514, 17583]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGTTATTTAATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|37 : evalue 9.63642e-21\t37 at [1, 64, 17514, 17583]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGTTATTTAATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|36 : evalue 9.63642e-21\t36 at [1, 64, 17514, 17583]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGTTATTTAATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|35 : evalue 9.63642e-21\t35 at [1, 64, 17514, 17583]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGTTATTTAATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|34 : evalue 9.63642e-21\t34 at [1, 64, 17514, 17583]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGTTATTTAATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|33 : evalue 9.63642e-21\t33 at [1, 64, 17514, 17583]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGTTATTTAATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|32 : evalue 9.63642e-21\t32 at [1, 64, 17514, 17583]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGTTATTTAATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|31 : evalue 9.63642e-21\t31 at [1, 64, 17514, 17583]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGTTATTTAATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|20 : evalue 9.63642e-21\t20 at [1, 64, 17501, 17570]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGTTATTTAATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|9 : evalue 9.63642e-21\t9 at [1, 64, 17514, 17583]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGTTATTTAATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|6 : evalue 9.63642e-21\t6 at [1, 64, 17515, 17584]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGTTATTTAATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|5 : evalue 9.63642e-21\t5 at [1, 64, 17514, 17583]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGTTATTTAATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|3 : evalue 9.63642e-21\t3 at [1, 64, 17515, 17584]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGTTATTTAATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|2 : evalue 9.63642e-21\t2 at [1, 64, 17514, 17583]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGTTATTTAATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|1 : evalue 9.63642e-21\t1 at [1, 64, 17514, 17583]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGTTATTTAATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|30 : evalue 2.34914e-18\t30 at [1, 64, 17514, 17583]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGTTATTTAATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAATGGTAT\n",
      "gnl|BL_ORD_ID|29 : evalue 2.34914e-18\t29 at [1, 64, 17514, 17583]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGTTATTTAATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAATGGTAT\n",
      "gnl|BL_ORD_ID|28 : evalue 2.34914e-18\t28 at [1, 64, 17514, 17583]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGTTATTTAATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAATGGTAT\n",
      "gnl|BL_ORD_ID|27 : evalue 2.34914e-18\t27 at [1, 64, 17514, 17583]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGTTATTTAATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAATGGTAT\n",
      "gnl|BL_ORD_ID|26 : evalue 2.34914e-18\t26 at [1, 64, 17514, 17583]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGTTATTTAATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAATGGTAT\n",
      "gnl|BL_ORD_ID|25 : evalue 2.34914e-18\t25 at [1, 64, 17514, 17583]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGTTATTTAATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAATGGTAT\n",
      "gnl|BL_ORD_ID|24 : evalue 2.34914e-18\t24 at [1, 64, 17514, 17583]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGTTATTTAATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAATGGTAT\n",
      "gnl|BL_ORD_ID|18 : evalue 2.34914e-18\t18 at [1, 64, 17466, 17535]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGTTATTTAATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAATGGTAT\n",
      "gnl|BL_ORD_ID|17 : evalue 2.34914e-18\t17 at [1, 64, 17433, 17502]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGTTATTTAATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAATGGTAT\n",
      "gnl|BL_ORD_ID|152 : evalue 5.72664e-16\t152 at [1, 64, 17393, 17462]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|151 : evalue 5.72664e-16\t151 at [1, 64, 17377, 17446]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|150 : evalue 5.72664e-16\t150 at [1, 64, 17484, 17553]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|149 : evalue 5.72664e-16\t149 at [1, 64, 17469, 17538]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|148 : evalue 5.72664e-16\t148 at [1, 64, 17497, 17566]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|147 : evalue 5.72664e-16\t147 at [1, 64, 17514, 17583]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|146 : evalue 5.72664e-16\t146 at [1, 64, 17479, 17548]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|145 : evalue 5.72664e-16\t145 at [1, 64, 17421, 17490]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|144 : evalue 5.72664e-16\t144 at [1, 64, 17512, 17581]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|143 : evalue 5.72664e-16\t143 at [1, 64, 17514, 17583]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|142 : evalue 5.72664e-16\t142 at [1, 64, 17470, 17539]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|141 : evalue 5.72664e-16\t141 at [1, 64, 17469, 17538]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|140 : evalue 5.72664e-16\t140 at [1, 64, 17501, 17570]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|139 : evalue 5.72664e-16\t139 at [1, 64, 17470, 17539]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|138 : evalue 5.72664e-16\t138 at [1, 64, 17514, 17583]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|137 : evalue 5.72664e-16\t137 at [1, 64, 17506, 17575]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|136 : evalue 5.72664e-16\t136 at [1, 64, 17501, 17570]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|135 : evalue 5.72664e-16\t135 at [1, 64, 17469, 17538]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|134 : evalue 5.72664e-16\t134 at [1, 64, 17490, 17559]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|133 : evalue 5.72664e-16\t133 at [1, 64, 17508, 17577]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|132 : evalue 5.72664e-16\t132 at [1, 64, 17506, 17575]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|131 : evalue 5.72664e-16\t131 at [1, 64, 17509, 17578]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|130 : evalue 5.72664e-16\t130 at [1, 64, 17511, 17580]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|129 : evalue 5.72664e-16\t129 at [1, 64, 17466, 17535]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|128 : evalue 5.72664e-16\t128 at [1, 64, 17483, 17552]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|127 : evalue 5.72664e-16\t127 at [1, 64, 17484, 17553]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|126 : evalue 5.72664e-16\t126 at [1, 64, 17506, 17575]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|125 : evalue 5.72664e-16\t125 at [1, 64, 17479, 17548]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|124 : evalue 5.72664e-16\t124 at [1, 64, 17406, 17475]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|123 : evalue 5.72664e-16\t123 at [1, 64, 17510, 17579]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|122 : evalue 5.72664e-16\t122 at [1, 64, 17479, 17548]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|121 : evalue 5.72664e-16\t121 at [1, 64, 17463, 17532]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|120 : evalue 5.72664e-16\t120 at [1, 64, 17463, 17532]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|119 : evalue 5.72664e-16\t119 at [1, 64, 17475, 17544]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|118 : evalue 5.72664e-16\t118 at [1, 64, 17479, 17548]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|117 : evalue 5.72664e-16\t117 at [1, 64, 17395, 17464]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|116 : evalue 5.72664e-16\t116 at [1, 64, 17434, 17503]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|115 : evalue 5.72664e-16\t115 at [1, 64, 17483, 17552]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|114 : evalue 5.72664e-16\t114 at [1, 64, 17503, 17572]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|113 : evalue 5.72664e-16\t113 at [1, 64, 17463, 17532]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|112 : evalue 5.72664e-16\t112 at [1, 64, 17470, 17539]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|111 : evalue 5.72664e-16\t111 at [1, 64, 17490, 17559]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|110 : evalue 5.72664e-16\t110 at [1, 64, 17463, 17532]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|109 : evalue 5.72664e-16\t109 at [1, 64, 17489, 17558]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|108 : evalue 5.72664e-16\t108 at [1, 64, 17483, 17552]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|107 : evalue 5.72664e-16\t107 at [1, 64, 17470, 17539]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|106 : evalue 5.72664e-16\t106 at [1, 64, 17506, 17575]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|105 : evalue 5.72664e-16\t105 at [1, 64, 17459, 17528]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|104 : evalue 5.72664e-16\t104 at [1, 64, 17514, 17583]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|103 : evalue 5.72664e-16\t103 at [1, 64, 17514, 17583]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|102 : evalue 5.72664e-16\t102 at [1, 64, 17483, 17552]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "gnl|BL_ORD_ID|101 : evalue 5.72664e-16\t101 at [1, 64, 17468, 17537]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|100 : evalue 5.72664e-16\t100 at [1, 64, 17469, 17538]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|99 : evalue 5.72664e-16\t99 at [1, 64, 17483, 17552]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|98 : evalue 5.72664e-16\t98 at [1, 64, 17466, 17535]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|97 : evalue 5.72664e-16\t97 at [1, 64, 17510, 17579]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|96 : evalue 5.72664e-16\t96 at [1, 64, 17495, 17564]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|95 : evalue 5.72664e-16\t95 at [1, 64, 17483, 17552]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|94 : evalue 5.72664e-16\t94 at [1, 64, 17399, 17468]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|93 : evalue 5.72664e-16\t93 at [1, 64, 17435, 17504]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|92 : evalue 5.72664e-16\t92 at [1, 64, 17459, 17528]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|91 : evalue 5.72664e-16\t91 at [1, 64, 17510, 17579]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|90 : evalue 5.72664e-16\t90 at [1, 64, 17513, 17582]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|89 : evalue 5.72664e-16\t89 at [1, 64, 17471, 17540]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|88 : evalue 5.72664e-16\t88 at [1, 64, 17503, 17572]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|87 : evalue 5.72664e-16\t87 at [1, 64, 17513, 17582]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|86 : evalue 5.72664e-16\t86 at [1, 64, 17499, 17568]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|85 : evalue 5.72664e-16\t85 at [1, 64, 17509, 17578]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|84 : evalue 5.72664e-16\t84 at [1, 64, 17514, 17583]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|83 : evalue 5.72664e-16\t83 at [1, 64, 17504, 17573]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|82 : evalue 5.72664e-16\t82 at [1, 64, 17463, 17532]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|81 : evalue 5.72664e-16\t81 at [1, 64, 17479, 17548]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|80 : evalue 5.72664e-16\t80 at [1, 64, 17471, 17540]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|79 : evalue 5.72664e-16\t79 at [1, 64, 17512, 17581]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|78 : evalue 5.72664e-16\t78 at [1, 64, 17470, 17539]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|77 : evalue 5.72664e-16\t77 at [1, 64, 17508, 17577]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|76 : evalue 5.72664e-16\t76 at [1, 64, 17470, 17539]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|75 : evalue 5.72664e-16\t75 at [1, 64, 17479, 17548]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|74 : evalue 5.72664e-16\t74 at [1, 64, 17480, 17549]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|73 : evalue 5.72664e-16\t73 at [1, 64, 17508, 17577]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|72 : evalue 5.72664e-16\t72 at [1, 64, 17470, 17539]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|71 : evalue 5.72664e-16\t71 at [1, 64, 17470, 17539]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|70 : evalue 5.72664e-16\t70 at [1, 64, 17466, 17535]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|69 : evalue 5.72664e-16\t69 at [1, 64, 17470, 17539]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|68 : evalue 5.72664e-16\t68 at [1, 64, 17318, 17387]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|67 : evalue 5.72664e-16\t67 at [1, 64, 17514, 17583]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|66 : evalue 5.72664e-16\t66 at [1, 64, 17481, 17550]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|65 : evalue 5.72664e-16\t65 at [1, 64, 17508, 17577]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|64 : evalue 5.72664e-16\t64 at [1, 64, 17484, 17553]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|63 : evalue 5.72664e-16\t63 at [1, 64, 17485, 17554]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|62 : evalue 5.72664e-16\t62 at [1, 64, 17512, 17581]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|61 : evalue 5.72664e-16\t61 at [1, 64, 17490, 17559]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|60 : evalue 5.72664e-16\t60 at [1, 64, 17506, 17575]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|59 : evalue 5.72664e-16\t59 at [1, 64, 17506, 17575]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|58 : evalue 5.72664e-16\t58 at [1, 64, 17485, 17554]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|57 : evalue 5.72664e-16\t57 at [1, 64, 17390, 17459]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|56 : evalue 5.72664e-16\t56 at [1, 64, 17484, 17553]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|55 : evalue 5.72664e-16\t55 at [1, 64, 17469, 17538]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|54 : evalue 5.72664e-16\t54 at [1, 64, 17469, 17538]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|53 : evalue 5.72664e-16\t53 at [1, 64, 17514, 17583]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|52 : evalue 5.72664e-16\t52 at [1, 64, 17514, 17583]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|51 : evalue 5.72664e-16\t51 at [1, 64, 17514, 17583]\n",
      "AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|23 : evalue 0.00769763\t23 at [32, 64, 17496, 17528]\n",
      "ATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "ATAACATCAAGTGCAAGGTCAAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|16 : evalue 0.00769763\t16 at [32, 64, 17496, 17528]\n",
      "ATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "ATAACATCAAGTGCAAGGTCAAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|10 : evalue 0.00769763\t10 at [32, 64, 17496, 17528]\n",
      "ATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "ATAACATCAAGTGCAAGGTCAAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|157 : evalue 0.120186\t157 at [49, 63, 17553, 17567]\n",
      "ATCTAGTGAGTGGTA\n",
      "ATCTAGTGAGTGGTA\n",
      "gnl|BL_ORD_ID|50 : evalue 0.120186\t50 at [26, 64, 17490, 17528]\n",
      "AAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "AAGCCTATAACATCAAGTGCAAAGTCAAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|12 : evalue 0.120186\t12 at [49, 63, 17553, 17567]\n",
      "ATCTAGTGAGTGGTA\n",
      "ATCTAGTGAGTGGTA\n",
      "gnl|BL_ORD_ID|156 : evalue 1.8765\t156 at [32, 64, 17496, 17528]\n",
      "ATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "ATAACATCAAGTGCAAAGTCAAGTGAGTGGTAT\n",
      "TTTTGCCACTGG\n",
      "TTTTGCCACTGG\n",
      "gnl|BL_ORD_ID|45 : evalue 1.8765\t45 at [32, 64, 17496, 17528]\n",
      "ATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "ATAACATCAAGTGCAAAGTCAAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|44 : evalue 1.8765\t44 at [32, 64, 17496, 17528]\n",
      "ATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "ATAACATCAAGTGCAAAGTCAAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|43 : evalue 1.8765\t43 at [32, 64, 17496, 17528]\n",
      "ATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "ATAACATCAAGTGCAAAGTCAAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|42 : evalue 1.8765\t42 at [32, 64, 17496, 17528]\n",
      "ATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "ATAACATCAAGTGCAAAGTCAAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|19 : evalue 1.8765\t19 at [32, 64, 17496, 17528]\n",
      "ATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "ATAACATCAAGTGCAAAGTCAAGTGAGTGGTAT\n",
      "gnl|BL_ORD_ID|7 : evalue 1.8765\t7 at [32, 64, 17496, 17528]\n",
      "ATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\n",
      "ATAACATCAAGTGCAAAGTCAAGTGAGTGGTAT\n",
      "TTTTGCCACTGG\n",
      "TTTTGCCACTGG\n",
      "gnl|BL_ORD_ID|159 : evalue 7.41476\t159 at [19, 8, 12730, 12741]\n",
      "AGTGGCAAAAAA\n",
      "AGTGGCAAAAAA\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "[#<Bio::Blast::Report::Hit:0x0000000003b8bcf0 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003b7dd58 @hsp={}, @num=1, @bit_score=93.6635, @score=47, @evalue=9.63642e-21, @query_from=1, @query_to=64, @hit_from=17514, @hit_to=17583, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=64, @positive=64, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGTTATTTAATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||||||||||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=1, @hit_id=\"gnl|BL_ORD_ID|154\", @len=18959, @definition=\"NC_002549v1\", @accession=\"154\">, #<Bio::Blast::Report::Hit:0x0000000003b70220 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003b51eb0 @hsp={}, @num=1, @bit_score=93.6635, @score=47, @evalue=9.63642e-21, @query_from=1, @query_to=64, @hit_from=17514, @hit_to=17583, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=64, @positive=64, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGTTATTTAATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||||||||||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=2, @hit_id=\"gnl|BL_ORD_ID|41\", @len=18959, @definition=\"KC242801v1\", @accession=\"41\">, #<Bio::Blast::Report::Hit:0x0000000003b45c28 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003b32f10 @hsp={}, @num=1, @bit_score=93.6635, @score=47, @evalue=9.63642e-21, @query_from=1, @query_to=64, @hit_from=17514, @hit_to=17583, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=64, @positive=64, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGTTATTTAATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||||||||||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=3, @hit_id=\"gnl|BL_ORD_ID|40\", @len=18958, @definition=\"KC242800v1\", @accession=\"40\">, #<Bio::Blast::Report::Hit:0x0000000003b263c8 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003b141c8 @hsp={}, @num=1, @bit_score=93.6635, @score=47, @evalue=9.63642e-21, @query_from=1, @query_to=64, @hit_from=17514, @hit_to=17583, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=64, @positive=64, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGTTATTTAATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||||||||||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=4, @hit_id=\"gnl|BL_ORD_ID|39\", @len=18959, @definition=\"KC242799v1\", @accession=\"39\">, #<Bio::Blast::Report::Hit:0x0000000003b07450 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003af9198 @hsp={}, @num=1, @bit_score=93.6635, @score=47, @evalue=9.63642e-21, @query_from=1, @query_to=64, @hit_from=17514, @hit_to=17583, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=64, @positive=64, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGTTATTTAATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||||||||||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=5, @hit_id=\"gnl|BL_ORD_ID|38\", @len=18959, @definition=\"KC242798v1\", @accession=\"38\">, #<Bio::Blast::Report::Hit:0x0000000003aec290 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003ad5c48 @hsp={}, @num=1, @bit_score=93.6635, @score=47, @evalue=9.63642e-21, @query_from=1, @query_to=64, @hit_from=17514, @hit_to=17583, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=64, @positive=64, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGTTATTTAATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||||||||||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=6, @hit_id=\"gnl|BL_ORD_ID|37\", @len=18958, @definition=\"KC242797v1\", @accession=\"37\">, #<Bio::Blast::Report::Hit:0x0000000003ac9538 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003abaec0 @hsp={}, @num=1, @bit_score=93.6635, @score=47, @evalue=9.63642e-21, @query_from=1, @query_to=64, @hit_from=17514, @hit_to=17583, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=64, @positive=64, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGTTATTTAATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||||||||||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=7, @hit_id=\"gnl|BL_ORD_ID|36\", @len=18959, @definition=\"KC242796v1\", @accession=\"36\">, #<Bio::Blast::Report::Hit:0x0000000003aae738 @hsps=[#<Bio::Blast::Report::Hsp:0x00000000033f67d8 @hsp={}, @num=1, @bit_score=93.6635, @score=47, @evalue=9.63642e-21, @query_from=1, @query_to=64, @hit_from=17514, @hit_to=17583, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=64, @positive=64, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGTTATTTAATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||||||||||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=8, @hit_id=\"gnl|BL_ORD_ID|35\", @len=18958, @definition=\"KC242795v1\", @accession=\"35\">, #<Bio::Blast::Report::Hit:0x000000000339d638 @hsps=[#<Bio::Blast::Report::Hsp:0x00000000030da658 @hsp={}, @num=1, @bit_score=93.6635, @score=47, @evalue=9.63642e-21, @query_from=1, @query_to=64, @hit_from=17514, @hit_to=17583, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=64, @positive=64, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGTTATTTAATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||||||||||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=9, @hit_id=\"gnl|BL_ORD_ID|34\", @len=18959, @definition=\"KC242794v1\", @accession=\"34\">, #<Bio::Blast::Report::Hit:0x0000000002797e60 @hsps=[#<Bio::Blast::Report::Hsp:0x000000000274ca28 @hsp={}, @num=1, @bit_score=93.6635, @score=47, @evalue=9.63642e-21, @query_from=1, @query_to=64, @hit_from=17514, @hit_to=17583, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=64, @positive=64, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGTTATTTAATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||||||||||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=10, @hit_id=\"gnl|BL_ORD_ID|33\", @len=18958, @definition=\"KC242793v1\", @accession=\"33\">, #<Bio::Blast::Report::Hit:0x00000000030fb650 @hsps=[#<Bio::Blast::Report::Hsp:0x00000000033a7d18 @hsp={}, @num=1, @bit_score=93.6635, @score=47, @evalue=9.63642e-21, @query_from=1, @query_to=64, @hit_from=17514, @hit_to=17583, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=64, @positive=64, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGTTATTTAATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||||||||||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=11, @hit_id=\"gnl|BL_ORD_ID|32\", @len=18959, @definition=\"KC242792v1\", @accession=\"32\">, #<Bio::Blast::Report::Hit:0x00000000033fd8f8 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003aac140 @hsp={}, @num=1, @bit_score=93.6635, @score=47, @evalue=9.63642e-21, @query_from=1, @query_to=64, @hit_from=17514, @hit_to=17583, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=64, @positive=64, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGTTATTTAATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||||||||||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=12, @hit_id=\"gnl|BL_ORD_ID|31\", @len=18959, @definition=\"KC242791v1\", @accession=\"31\">, #<Bio::Blast::Report::Hit:0x0000000003abf290 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003ac88e0 @hsp={}, @num=1, @bit_score=93.6635, @score=47, @evalue=9.63642e-21, @query_from=1, @query_to=64, @hit_from=17501, @hit_to=17570, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=64, @positive=64, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGTTATTTAATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||||||||||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=13, @hit_id=\"gnl|BL_ORD_ID|20\", @len=18944, @definition=\"JQ352763v1\", @accession=\"20\">, #<Bio::Blast::Report::Hit:0x0000000003adbee0 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003aeca10 @hsp={}, @num=1, @bit_score=93.6635, @score=47, @evalue=9.63642e-21, @query_from=1, @query_to=64, @hit_from=17514, @hit_to=17583, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=64, @positive=64, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGTTATTTAATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||||||||||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=14, @hit_id=\"gnl|BL_ORD_ID|9\", @len=18959, @definition=\"EU224440v2\", @accession=\"9\">, #<Bio::Blast::Report::Hit:0x0000000003aff840 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003b08c38 @hsp={}, @num=1, @bit_score=93.6635, @score=47, @evalue=9.63642e-21, @query_from=1, @query_to=64, @hit_from=17515, @hit_to=17584, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=64, @positive=64, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGTTATTTAATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||||||||||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=15, @hit_id=\"gnl|BL_ORD_ID|6\", @len=18961, @definition=\"AY354458v1\", @accession=\"6\">, #<Bio::Blast::Report::Hit:0x0000000003b14330 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003b28c90 @hsp={}, @num=1, @bit_score=93.6635, @score=47, @evalue=9.63642e-21, @query_from=1, @query_to=64, @hit_from=17514, @hit_to=17583, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=64, @positive=64, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGTTATTTAATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||||||||||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=16, @hit_id=\"gnl|BL_ORD_ID|5\", @len=18959, @definition=\"AY142960v1\", @accession=\"5\">, #<Bio::Blast::Report::Hit:0x0000000003b34b80 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003b498a0 @hsp={}, @num=1, @bit_score=93.6635, @score=47, @evalue=9.63642e-21, @query_from=1, @query_to=64, @hit_from=17515, @hit_to=17584, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=64, @positive=64, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGTTATTTAATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||||||||||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=17, @hit_id=\"gnl|BL_ORD_ID|3\", @len=18960, @definition=\"AF499101v1\", @accession=\"3\">, #<Bio::Blast::Report::Hit:0x0000000003b54890 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003b752e8 @hsp={}, @num=1, @bit_score=93.6635, @score=47, @evalue=9.63642e-21, @query_from=1, @query_to=64, @hit_from=17514, @hit_to=17583, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=64, @positive=64, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGTTATTTAATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||||||||||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=18, @hit_id=\"gnl|BL_ORD_ID|2\", @len=18959, @definition=\"AF272001v1\", @accession=\"2\">, #<Bio::Blast::Report::Hit:0x0000000003b80da0 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003b921e0 @hsp={}, @num=1, @bit_score=93.6635, @score=47, @evalue=9.63642e-21, @query_from=1, @query_to=64, @hit_from=17514, @hit_to=17583, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=64, @positive=64, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGTTATTTAATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||||||||||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=19, @hit_id=\"gnl|BL_ORD_ID|1\", @len=18959, @definition=\"AF086833v2\", @accession=\"1\">, #<Bio::Blast::Report::Hit:0x0000000003b9cd98 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003ba75e0 @hsp={}, @num=1, @bit_score=85.7341, @score=43, @evalue=2.34914e-18, @query_from=1, @query_to=64, @hit_from=17514, @hit_to=17583, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=63, @positive=63, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGTTATTTAATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAATGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||||||||||||||| ||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=20, @hit_id=\"gnl|BL_ORD_ID|30\", @len=18958, @definition=\"KC242790v1\", @accession=\"30\">, #<Bio::Blast::Report::Hit:0x0000000003baaa88 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003bad2b0 @hsp={}, @num=1, @bit_score=85.7341, @score=43, @evalue=2.34914e-18, @query_from=1, @query_to=64, @hit_from=17514, @hit_to=17583, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=63, @positive=63, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGTTATTTAATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAATGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||||||||||||||| ||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=21, @hit_id=\"gnl|BL_ORD_ID|29\", @len=18958, @definition=\"KC242789v1\", @accession=\"29\">, #<Bio::Blast::Report::Hit:0x0000000003bb4a10 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003bbf2d0 @hsp={}, @num=1, @bit_score=85.7341, @score=43, @evalue=2.34914e-18, @query_from=1, @query_to=64, @hit_from=17514, @hit_to=17583, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=63, @positive=63, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGTTATTTAATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAATGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||||||||||||||| ||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=22, @hit_id=\"gnl|BL_ORD_ID|28\", @len=18958, @definition=\"KC242788v1\", @accession=\"28\">, #<Bio::Blast::Report::Hit:0x0000000003bc2a48 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003bc9460 @hsp={}, @num=1, @bit_score=85.7341, @score=43, @evalue=2.34914e-18, @query_from=1, @query_to=64, @hit_from=17514, @hit_to=17583, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=63, @positive=63, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGTTATTTAATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAATGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||||||||||||||| ||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=23, @hit_id=\"gnl|BL_ORD_ID|27\", @len=18958, @definition=\"KC242787v1\", @accession=\"27\">, #<Bio::Blast::Report::Hit:0x0000000003bcca98 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003bd7600 @hsp={}, @num=1, @bit_score=85.7341, @score=43, @evalue=2.34914e-18, @query_from=1, @query_to=64, @hit_from=17514, @hit_to=17583, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=63, @positive=63, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGTTATTTAATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAATGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||||||||||||||| ||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=24, @hit_id=\"gnl|BL_ORD_ID|26\", @len=18958, @definition=\"KC242786v1\", @accession=\"26\">, #<Bio::Blast::Report::Hit:0x0000000003bdea68 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003be13a8 @hsp={}, @num=1, @bit_score=85.7341, @score=43, @evalue=2.34914e-18, @query_from=1, @query_to=64, @hit_from=17514, @hit_to=17583, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=63, @positive=63, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGTTATTTAATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAATGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||||||||||||||| ||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=25, @hit_id=\"gnl|BL_ORD_ID|25\", @len=18958, @definition=\"KC242785v1\", @accession=\"25\">, #<Bio::Blast::Report::Hit:0x0000000003be4af8 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003bf37b0 @hsp={}, @num=1, @bit_score=85.7341, @score=43, @evalue=2.34914e-18, @query_from=1, @query_to=64, @hit_from=17514, @hit_to=17583, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=63, @positive=63, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGTTATTTAATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAATGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||||||||||||||| ||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=26, @hit_id=\"gnl|BL_ORD_ID|24\", @len=18958, @definition=\"KC242784v1\", @accession=\"24\">, #<Bio::Blast::Report::Hit:0x0000000003bf6b40 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003bf96d8 @hsp={}, @num=1, @bit_score=85.7341, @score=43, @evalue=2.34914e-18, @query_from=1, @query_to=64, @hit_from=17466, @hit_to=17535, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=63, @positive=63, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGTTATTTAATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAATGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||||||||||||||| ||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=27, @hit_id=\"gnl|BL_ORD_ID|18\", @len=18807, @definition=\"HQ613403v1\", @accession=\"18\">, #<Bio::Blast::Report::Hit:0x0000000003bfca68 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003c0b6f8 @hsp={}, @num=1, @bit_score=85.7341, @score=43, @evalue=2.34914e-18, @query_from=1, @query_to=64, @hit_from=17433, @hit_to=17502, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=63, @positive=63, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGTTATTTAATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAATGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||||||||||||||| ||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=28, @hit_id=\"gnl|BL_ORD_ID|17\", @len=18774, @definition=\"HQ613402v1\", @accession=\"17\">, #<Bio::Blast::Report::Hit:0x0000000003c0eb78 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003c11878 @hsp={}, @num=1, @bit_score=77.8047, @score=39, @evalue=5.72664e-16, @query_from=1, @query_to=64, @hit_from=17393, @hit_to=17462, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=62, @positive=62, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||| || |||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=29, @hit_id=\"gnl|BL_ORD_ID|152\", @len=18613, @definition=\"KM233118v1\", @accession=\"152\">, #<Bio::Blast::Report::Hit:0x0000000003c18d08 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003c23758 @hsp={}, @num=1, @bit_score=77.8047, @score=39, @evalue=5.72664e-16, @query_from=1, @query_to=64, @hit_from=17377, @hit_to=17446, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=62, @positive=62, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||| || |||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=30, @hit_id=\"gnl|BL_ORD_ID|151\", @len=18685, @definition=\"KM233117v1\", @accession=\"151\">, #<Bio::Blast::Report::Hit:0x0000000003c26de0 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003c2d7d0 @hsp={}, @num=1, @bit_score=77.8047, @score=39, @evalue=5.72664e-16, @query_from=1, @query_to=64, @hit_from=17484, @hit_to=17553, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=62, @positive=62, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||| || |||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=31, @hit_id=\"gnl|BL_ORD_ID|150\", @len=18926, @definition=\"KM233116v1\", @accession=\"150\">, #<Bio::Blast::Report::Hit:0x0000000003c30c50 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003c3b8f8 @hsp={}, @num=1, @bit_score=77.8047, @score=39, @evalue=5.72664e-16, @query_from=1, @query_to=64, @hit_from=17469, @hit_to=17538, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=62, @positive=62, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||| || |||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=32, @hit_id=\"gnl|BL_ORD_ID|149\", @len=18835, @definition=\"KM233115v1\", @accession=\"149\">, #<Bio::Blast::Report::Hit:0x0000000003c42d88 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003c45948 @hsp={}, @num=1, @bit_score=77.8047, @score=39, @evalue=5.72664e-16, @query_from=1, @query_to=64, @hit_from=17497, @hit_to=17566, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=62, @positive=62, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||| || |||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=33, @hit_id=\"gnl|BL_ORD_ID|148\", @len=18939, @definition=\"KM233114v1\", @accession=\"148\">, #<Bio::Blast::Report::Hit:0x0000000003c48f80 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003c57850 @hsp={}, @num=1, @bit_score=77.8047, @score=39, @evalue=5.72664e-16, @query_from=1, @query_to=64, @hit_from=17514, @hit_to=17583, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=62, @positive=62, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||| || |||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=34, @hit_id=\"gnl|BL_ORD_ID|147\", @len=18958, @definition=\"KM233113v1\", @accession=\"147\">, #<Bio::Blast::Report::Hit:0x0000000003c5af78 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003c5d868 @hsp={}, @num=1, @bit_score=77.8047, @score=39, @evalue=5.72664e-16, @query_from=1, @query_to=64, @hit_from=17479, @hit_to=17548, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=62, @positive=62, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||| || |||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=35, @hit_id=\"gnl|BL_ORD_ID|146\", @len=18923, @definition=\"KM233112v1\", @accession=\"146\">, #<Bio::Blast::Report::Hit:0x0000000003c60fe0 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003c6f9a0 @hsp={}, @num=1, @bit_score=77.8047, @score=39, @evalue=5.72664e-16, @query_from=1, @query_to=64, @hit_from=17421, @hit_to=17490, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=62, @positive=62, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||| || |||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=36, @hit_id=\"gnl|BL_ORD_ID|145\", @len=18749, @definition=\"KM233111v1\", @accession=\"145\">, #<Bio::Blast::Report::Hit:0x0000000003c72ec0 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003c75800 @hsp={}, @num=1, @bit_score=77.8047, @score=39, @evalue=5.72664e-16, @query_from=1, @query_to=64, @hit_from=17512, @hit_to=17581, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=62, @positive=62, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||| || |||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=37, @hit_id=\"gnl|BL_ORD_ID|144\", @len=18956, @definition=\"KM233110v1\", @accession=\"144\">, #<Bio::Blast::Report::Hit:0x0000000003c7cc68 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003c87500 @hsp={}, @num=1, @bit_score=77.8047, @score=39, @evalue=5.72664e-16, @query_from=1, @query_to=64, @hit_from=17514, @hit_to=17583, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=62, @positive=62, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||| || |||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=38, @hit_id=\"gnl|BL_ORD_ID|143\", @len=18958, @definition=\"KM233109v1\", @accession=\"143\">, #<Bio::Blast::Report::Hit:0x0000000003c8ac78 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003c91690 @hsp={}, @num=1, @bit_score=77.8047, @score=39, @evalue=5.72664e-16, @query_from=1, @query_to=64, @hit_from=17470, @hit_to=17539, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=62, @positive=62, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||| || |||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=39, @hit_id=\"gnl|BL_ORD_ID|142\", @len=18915, @definition=\"KM233108v1\", @accession=\"142\">, #<Bio::Blast::Report::Hit:0x0000000003c94bb0 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003c9f858 @hsp={}, @num=1, @bit_score=77.8047, @score=39, @evalue=5.72664e-16, @query_from=1, @query_to=64, @hit_from=17469, @hit_to=17538, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=62, @positive=62, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||| || |||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=40, @hit_id=\"gnl|BL_ORD_ID|141\", @len=18876, @definition=\"KM233107v1\", @accession=\"141\">, #<Bio::Blast::Report::Hit:0x0000000003ca6900 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003ca94e8 @hsp={}, @num=1, @bit_score=77.8047, @score=39, @evalue=5.72664e-16, @query_from=1, @query_to=64, @hit_from=17501, @hit_to=17570, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=62, @positive=62, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||| || |||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=41, @hit_id=\"gnl|BL_ORD_ID|140\", @len=18945, @definition=\"KM233106v1\", @accession=\"140\">, #<Bio::Blast::Report::Hit:0x0000000003cac940 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003cbb328 @hsp={}, @num=1, @bit_score=77.8047, @score=39, @evalue=5.72664e-16, @query_from=1, @query_to=64, @hit_from=17470, @hit_to=17539, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=62, @positive=62, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||| || |||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=42, @hit_id=\"gnl|BL_ORD_ID|139\", @len=18914, @definition=\"KM233105v1\", @accession=\"139\">, #<Bio::Blast::Report::Hit:0x0000000003cbe898 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003cc1520 @hsp={}, @num=1, @bit_score=77.8047, @score=39, @evalue=5.72664e-16, @query_from=1, @query_to=64, @hit_from=17514, @hit_to=17583, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=62, @positive=62, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||| || |||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=43, @hit_id=\"gnl|BL_ORD_ID|138\", @len=18921, @definition=\"KM233104v1\", @accession=\"138\">, #<Bio::Blast::Report::Hit:0x0000000003cc45e0 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003cd3270 @hsp={}, @num=1, @bit_score=77.8047, @score=39, @evalue=5.72664e-16, @query_from=1, @query_to=64, @hit_from=17506, @hit_to=17575, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=62, @positive=62, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||| || |||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=44, @hit_id=\"gnl|BL_ORD_ID|137\", @len=18950, @definition=\"KM233103v1\", @accession=\"137\">, #<Bio::Blast::Report::Hit:0x0000000003cd65d8 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003cd9008 @hsp={}, @num=1, @bit_score=77.8047, @score=39, @evalue=5.72664e-16, @query_from=1, @query_to=64, @hit_from=17501, @hit_to=17570, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=62, @positive=62, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||| || |||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=45, @hit_id=\"gnl|BL_ORD_ID|136\", @len=18945, @definition=\"KM233102v1\", @accession=\"136\">, #<Bio::Blast::Report::Hit:0x0000000003ce0768 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003ceb1b8 @hsp={}, @num=1, @bit_score=77.8047, @score=39, @evalue=5.72664e-16, @query_from=1, @query_to=64, @hit_from=17469, @hit_to=17538, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=62, @positive=62, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||| || |||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=46, @hit_id=\"gnl|BL_ORD_ID|135\", @len=18876, @definition=\"KM233101v1\", @accession=\"135\">, #<Bio::Blast::Report::Hit:0x0000000003cee930 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003cf4f60 @hsp={}, @num=1, @bit_score=77.8047, @score=39, @evalue=5.72664e-16, @query_from=1, @query_to=64, @hit_from=17490, @hit_to=17559, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=62, @positive=62, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||| || |||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=47, @hit_id=\"gnl|BL_ORD_ID|134\", @len=18932, @definition=\"KM233100v1\", @accession=\"134\">, #<Bio::Blast::Report::Hit:0x0000000003cf82c8 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003d02598 @hsp={}, @num=1, @bit_score=77.8047, @score=39, @evalue=5.72664e-16, @query_from=1, @query_to=64, @hit_from=17508, @hit_to=17577, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=62, @positive=62, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||| || |||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=48, @hit_id=\"gnl|BL_ORD_ID|133\", @len=18953, @definition=\"KM233099v1\", @accession=\"133\">, #<Bio::Blast::Report::Hit:0x0000000003d09a28 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003d0c4d0 @hsp={}, @num=1, @bit_score=77.8047, @score=39, @evalue=5.72664e-16, @query_from=1, @query_to=64, @hit_from=17506, @hit_to=17575, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=62, @positive=62, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||| || |||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=49, @hit_id=\"gnl|BL_ORD_ID|132\", @len=18950, @definition=\"KM233098v1\", @accession=\"132\">, #<Bio::Blast::Report::Hit:0x0000000003d17bf0 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003d1e220 @hsp={}, @num=1, @bit_score=77.8047, @score=39, @evalue=5.72664e-16, @query_from=1, @query_to=64, @hit_from=17509, @hit_to=17578, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=62, @positive=62, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||| || |||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=50, @hit_id=\"gnl|BL_ORD_ID|131\", @len=18953, @definition=\"KM233097v1\", @accession=\"131\">, #<Bio::Blast::Report::Hit:0x0000000003d21858 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003d24148 @hsp={}, @num=1, @bit_score=77.8047, @score=39, @evalue=5.72664e-16, @query_from=1, @query_to=64, @hit_from=17511, @hit_to=17580, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=62, @positive=62, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||| || |||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=51, @hit_id=\"gnl|BL_ORD_ID|130\", @len=18953, @definition=\"KM233096v1\", @accession=\"130\">, #<Bio::Blast::Report::Hit:0x0000000003d335a8 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003d35da8 @hsp={}, @num=1, @bit_score=77.8047, @score=39, @evalue=5.72664e-16, @query_from=1, @query_to=64, @hit_from=17466, @hit_to=17535, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=62, @positive=62, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||| || |||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=52, @hit_id=\"gnl|BL_ORD_ID|129\", @len=18862, @definition=\"KM233095v1\", @accession=\"129\">, #<Bio::Blast::Report::Hit:0x0000000003d394d0 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003d47990 @hsp={}, @num=1, @bit_score=77.8047, @score=39, @evalue=5.72664e-16, @query_from=1, @query_to=64, @hit_from=17483, @hit_to=17552, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=62, @positive=62, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||| || |||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=53, @hit_id=\"gnl|BL_ORD_ID|128\", @len=18925, @definition=\"KM233094v1\", @accession=\"128\">, #<Bio::Blast::Report::Hit:0x0000000003d4afc8 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003d4db38 @hsp={}, @num=1, @bit_score=77.8047, @score=39, @evalue=5.72664e-16, @query_from=1, @query_to=64, @hit_from=17484, @hit_to=17553, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=62, @positive=62, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||| || |||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=54, @hit_id=\"gnl|BL_ORD_ID|127\", @len=18880, @definition=\"KM233093v1\", @accession=\"127\">, #<Bio::Blast::Report::Hit:0x0000000003d50d38 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003d5f8b0 @hsp={}, @num=1, @bit_score=77.8047, @score=39, @evalue=5.72664e-16, @query_from=1, @query_to=64, @hit_from=17506, @hit_to=17575, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=62, @positive=62, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||| || |||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=55, @hit_id=\"gnl|BL_ORD_ID|126\", @len=18948, @definition=\"KM233092v1\", @accession=\"126\">, #<Bio::Blast::Report::Hit:0x0000000003d62ec0 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003d65620 @hsp={}, @num=1, @bit_score=77.8047, @score=39, @evalue=5.72664e-16, @query_from=1, @query_to=64, @hit_from=17479, @hit_to=17548, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=62, @positive=62, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||| || |||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=56, @hit_id=\"gnl|BL_ORD_ID|125\", @len=18886, @definition=\"KM233091v1\", @accession=\"125\">, #<Bio::Blast::Report::Hit:0x0000000003d6c7e0 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003d77410 @hsp={}, @num=1, @bit_score=77.8047, @score=39, @evalue=5.72664e-16, @query_from=1, @query_to=64, @hit_from=17406, @hit_to=17475, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=62, @positive=62, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||| || |||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=57, @hit_id=\"gnl|BL_ORD_ID|124\", @len=18799, @definition=\"KM233090v1\", @accession=\"124\">, #<Bio::Blast::Report::Hit:0x0000000003d7a610 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003d81208 @hsp={}, @num=1, @bit_score=77.8047, @score=39, @evalue=5.72664e-16, @query_from=1, @query_to=64, @hit_from=17510, @hit_to=17579, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=62, @positive=62, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||| || |||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=58, @hit_id=\"gnl|BL_ORD_ID|123\", @len=18914, @definition=\"KM233089v1\", @accession=\"123\">, #<Bio::Blast::Report::Hit:0x0000000003d84818 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003d8f1c8 @hsp={}, @num=1, @bit_score=77.8047, @score=39, @evalue=5.72664e-16, @query_from=1, @query_to=64, @hit_from=17479, @hit_to=17548, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=62, @positive=62, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||| || |||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=59, @hit_id=\"gnl|BL_ORD_ID|122\", @len=18921, @definition=\"KM233088v1\", @accession=\"122\">, #<Bio::Blast::Report::Hit:0x0000000003d96298 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003d98d40 @hsp={}, @num=1, @bit_score=77.8047, @score=39, @evalue=5.72664e-16, @query_from=1, @query_to=64, @hit_from=17463, @hit_to=17532, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=62, @positive=62, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||| || |||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=60, @hit_id=\"gnl|BL_ORD_ID|121\", @len=18859, @definition=\"KM233087v1\", @accession=\"121\">, #<Bio::Blast::Report::Hit:0x0000000003da3ee8 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003daabd0 @hsp={}, @num=1, @bit_score=77.8047, @score=39, @evalue=5.72664e-16, @query_from=1, @query_to=64, @hit_from=17463, @hit_to=17532, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=62, @positive=62, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||| || |||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=61, @hit_id=\"gnl|BL_ORD_ID|120\", @len=18830, @definition=\"KM233086v1\", @accession=\"120\">, #<Bio::Blast::Report::Hit:0x0000000003dae0f0 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003db0ad0 @hsp={}, @num=1, @bit_score=77.8047, @score=39, @evalue=5.72664e-16, @query_from=1, @query_to=64, @hit_from=17475, @hit_to=17544, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=62, @positive=62, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||| || |||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=62, @hit_id=\"gnl|BL_ORD_ID|119\", @len=18882, @definition=\"KM233085v1\", @accession=\"119\">, #<Bio::Blast::Report::Hit:0x0000000003dbfe18 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003dc2370 @hsp={}, @num=1, @bit_score=77.8047, @score=39, @evalue=5.72664e-16, @query_from=1, @query_to=64, @hit_from=17479, @hit_to=17548, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=62, @positive=62, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||| || |||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=63, @hit_id=\"gnl|BL_ORD_ID|118\", @len=18921, @definition=\"KM233084v1\", @accession=\"118\">, #<Bio::Blast::Report::Hit:0x0000000003dc5ac0 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003dc84f0 @hsp={}, @num=1, @bit_score=77.8047, @score=39, @evalue=5.72664e-16, @query_from=1, @query_to=64, @hit_from=17395, @hit_to=17464, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=62, @positive=62, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||| || |||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=64, @hit_id=\"gnl|BL_ORD_ID|117\", @len=18773, @definition=\"KM233083v1\", @accession=\"117\">, #<Bio::Blast::Report::Hit:0x0000000003dd7ae0 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003dda678 @hsp={}, @num=1, @bit_score=77.8047, @score=39, @evalue=5.72664e-16, @query_from=1, @query_to=64, @hit_from=17434, @hit_to=17503, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=62, @positive=62, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||| || |||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=65, @hit_id=\"gnl|BL_ORD_ID|116\", @len=18812, @definition=\"KM233082v1\", @accession=\"116\">, #<Bio::Blast::Report::Hit:0x0000000003ddd738 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003debef0 @hsp={}, @num=1, @bit_score=77.8047, @score=39, @evalue=5.72664e-16, @query_from=1, @query_to=64, @hit_from=17483, @hit_to=17552, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=62, @positive=62, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||| || |||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=66, @hit_id=\"gnl|BL_ORD_ID|115\", @len=18925, @definition=\"KM233081v1\", @accession=\"115\">, #<Bio::Blast::Report::Hit:0x0000000003def398 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003df1dc8 @hsp={}, @num=1, @bit_score=77.8047, @score=39, @evalue=5.72664e-16, @query_from=1, @query_to=64, @hit_from=17503, @hit_to=17572, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=62, @positive=62, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||| || |||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=67, @hit_id=\"gnl|BL_ORD_ID|114\", @len=18945, @definition=\"KM233080v1\", @accession=\"114\">, #<Bio::Blast::Report::Hit:0x0000000003df9410 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003e03f78 @hsp={}, @num=1, @bit_score=77.8047, @score=39, @evalue=5.72664e-16, @query_from=1, @query_to=64, @hit_from=17463, @hit_to=17532, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=62, @positive=62, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||| || |||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=68, @hit_id=\"gnl|BL_ORD_ID|113\", @len=18870, @definition=\"KM233079v1\", @accession=\"113\">, #<Bio::Blast::Report::Hit:0x0000000003e073f8 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003e0d820 @hsp={}, @num=1, @bit_score=77.8047, @score=39, @evalue=5.72664e-16, @query_from=1, @query_to=64, @hit_from=17470, @hit_to=17539, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=62, @positive=62, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||| || |||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=69, @hit_id=\"gnl|BL_ORD_ID|112\", @len=18912, @definition=\"KM233078v1\", @accession=\"112\">, #<Bio::Blast::Report::Hit:0x0000000003e10cc8 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003e1f688 @hsp={}, @num=1, @bit_score=77.8047, @score=39, @evalue=5.72664e-16, @query_from=1, @query_to=64, @hit_from=17490, @hit_to=17559, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=62, @positive=62, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||| || |||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=70, @hit_id=\"gnl|BL_ORD_ID|111\", @len=18897, @definition=\"KM233077v1\", @accession=\"111\">, #<Bio::Blast::Report::Hit:0x0000000003e22ce8 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003e29890 @hsp={}, @num=1, @bit_score=77.8047, @score=39, @evalue=5.72664e-16, @query_from=1, @query_to=64, @hit_from=17463, @hit_to=17532, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=62, @positive=62, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||| || |||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=71, @hit_id=\"gnl|BL_ORD_ID|110\", @len=18811, @definition=\"KM233076v1\", @accession=\"110\">, #<Bio::Blast::Report::Hit:0x0000000003e2cce8 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003e3b1f8 @hsp={}, @num=1, @bit_score=77.8047, @score=39, @evalue=5.72664e-16, @query_from=1, @query_to=64, @hit_from=17489, @hit_to=17558, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=62, @positive=62, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||| || |||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=72, @hit_id=\"gnl|BL_ORD_ID|109\", @len=18931, @definition=\"KM233075v1\", @accession=\"109\">, #<Bio::Blast::Report::Hit:0x0000000003e3e678 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003e44fa0 @hsp={}, @num=1, @bit_score=77.8047, @score=39, @evalue=5.72664e-16, @query_from=1, @query_to=64, @hit_from=17483, @hit_to=17552, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=62, @positive=62, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||| || |||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=73, @hit_id=\"gnl|BL_ORD_ID|108\", @len=18890, @definition=\"KM233074v1\", @accession=\"108\">, #<Bio::Blast::Report::Hit:0x0000000003e486f0 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003e571a0 @hsp={}, @num=1, @bit_score=77.8047, @score=39, @evalue=5.72664e-16, @query_from=1, @query_to=64, @hit_from=17470, @hit_to=17539, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=62, @positive=62, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||| || |||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=74, @hit_id=\"gnl|BL_ORD_ID|107\", @len=18912, @definition=\"KM233073v1\", @accession=\"107\">, #<Bio::Blast::Report::Hit:0x0000000003e5a238 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003e60b38 @hsp={}, @num=1, @bit_score=77.8047, @score=39, @evalue=5.72664e-16, @query_from=1, @query_to=64, @hit_from=17506, @hit_to=17575, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=62, @positive=62, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||| || |||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=75, @hit_id=\"gnl|BL_ORD_ID|106\", @len=18949, @definition=\"KM233072v1\", @accession=\"106\">, #<Bio::Blast::Report::Hit:0x0000000003e6ff48 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003e72950 @hsp={}, @num=1, @bit_score=77.8047, @score=39, @evalue=5.72664e-16, @query_from=1, @query_to=64, @hit_from=17459, @hit_to=17528, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=62, @positive=62, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||| || |||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=76, @hit_id=\"gnl|BL_ORD_ID|105\", @len=18901, @definition=\"KM233071v1\", @accession=\"105\">, #<Bio::Blast::Report::Hit:0x0000000003e7a0b0 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003e7cb80 @hsp={}, @num=1, @bit_score=77.8047, @score=39, @evalue=5.72664e-16, @query_from=1, @query_to=64, @hit_from=17514, @hit_to=17583, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=62, @positive=62, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||| || |||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=77, @hit_id=\"gnl|BL_ORD_ID|104\", @len=18959, @definition=\"KM233070v1\", @accession=\"104\">, #<Bio::Blast::Report::Hit:0x0000000003e8be78 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003e8e4c0 @hsp={}, @num=1, @bit_score=77.8047, @score=39, @evalue=5.72664e-16, @query_from=1, @query_to=64, @hit_from=17514, @hit_to=17583, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=62, @positive=62, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||| || |||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=78, @hit_id=\"gnl|BL_ORD_ID|103\", @len=18958, @definition=\"KM233069v1\", @accession=\"103\">, #<Bio::Blast::Report::Hit:0x0000000003e95838 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003e98510 @hsp={}, @num=1, @bit_score=77.8047, @score=39, @evalue=5.72664e-16, @query_from=1, @query_to=64, @hit_from=17483, @hit_to=17552, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=62, @positive=62, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||| || |||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=79, @hit_id=\"gnl|BL_ORD_ID|102\", @len=18925, @definition=\"KM233068v1\", @accession=\"102\">, #<Bio::Blast::Report::Hit:0x0000000003ea7970 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003eaa440 @hsp={}, @num=1, @bit_score=77.8047, @score=39, @evalue=5.72664e-16, @query_from=1, @query_to=64, @hit_from=17468, @hit_to=17537, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=62, @positive=62, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||| || |||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=80, @hit_id=\"gnl|BL_ORD_ID|101\", @len=18910, @definition=\"KM233067v1\", @accession=\"101\">, #<Bio::Blast::Report::Hit:0x0000000003eb1a38 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003eb4080 @hsp={}, @num=1, @bit_score=77.8047, @score=39, @evalue=5.72664e-16, @query_from=1, @query_to=64, @hit_from=17469, @hit_to=17538, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=62, @positive=62, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||| || |||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=81, @hit_id=\"gnl|BL_ORD_ID|100\", @len=18876, @definition=\"KM233066v1\", @accession=\"100\">, #<Bio::Blast::Report::Hit:0x0000000003ec33a0 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003ec5da8 @hsp={}, @num=1, @bit_score=77.8047, @score=39, @evalue=5.72664e-16, @query_from=1, @query_to=64, @hit_from=17483, @hit_to=17552, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=62, @positive=62, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||| || |||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=82, @hit_id=\"gnl|BL_ORD_ID|99\", @len=18925, @definition=\"KM233065v1\", @accession=\"99\">, #<Bio::Blast::Report::Hit:0x0000000003ecd238 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003edbdb0 @hsp={}, @num=1, @bit_score=77.8047, @score=39, @evalue=5.72664e-16, @query_from=1, @query_to=64, @hit_from=17466, @hit_to=17535, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=62, @positive=62, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||| || |||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=83, @hit_id=\"gnl|BL_ORD_ID|98\", @len=18803, @definition=\"KM233064v1\", @accession=\"98\">, #<Bio::Blast::Report::Hit:0x0000000003edf2d0 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003ee5ba8 @hsp={}, @num=1, @bit_score=77.8047, @score=39, @evalue=5.72664e-16, @query_from=1, @query_to=64, @hit_from=17510, @hit_to=17579, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=62, @positive=62, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||| || |||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=84, @hit_id=\"gnl|BL_ORD_ID|97\", @len=18955, @definition=\"KM233063v1\", @accession=\"97\">, #<Bio::Blast::Report::Hit:0x0000000003ee8c68 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003ef7628 @hsp={}, @num=1, @bit_score=77.8047, @score=39, @evalue=5.72664e-16, @query_from=1, @query_to=64, @hit_from=17495, @hit_to=17564, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=62, @positive=62, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||| || |||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=85, @hit_id=\"gnl|BL_ORD_ID|96\", @len=18937, @definition=\"KM233062v1\", @accession=\"96\">, #<Bio::Blast::Report::Hit:0x0000000003efaaa8 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003f01650 @hsp={}, @num=1, @bit_score=77.8047, @score=39, @evalue=5.72664e-16, @query_from=1, @query_to=64, @hit_from=17483, @hit_to=17552, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=62, @positive=62, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||| || |||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=86, @hit_id=\"gnl|BL_ORD_ID|95\", @len=18890, @definition=\"KM233061v1\", @accession=\"95\">, #<Bio::Blast::Report::Hit:0x0000000003f04d78 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003f13288 @hsp={}, @num=1, @bit_score=77.8047, @score=39, @evalue=5.72664e-16, @query_from=1, @query_to=64, @hit_from=17399, @hit_to=17468, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=62, @positive=62, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||| || |||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=87, @hit_id=\"gnl|BL_ORD_ID|94\", @len=18711, @definition=\"KM233060v1\", @accession=\"94\">, #<Bio::Blast::Report::Hit:0x0000000003f164d8 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003f1cef0 @hsp={}, @num=1, @bit_score=77.8047, @score=39, @evalue=5.72664e-16, @query_from=1, @query_to=64, @hit_from=17435, @hit_to=17504, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=62, @positive=62, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||| || |||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=88, @hit_id=\"gnl|BL_ORD_ID|93\", @len=18791, @definition=\"KM233059v1\", @accession=\"93\">, #<Bio::Blast::Report::Hit:0x0000000003f20370 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003f2eee8 @hsp={}, @num=1, @bit_score=77.8047, @score=39, @evalue=5.72664e-16, @query_from=1, @query_to=64, @hit_from=17459, @hit_to=17528, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=62, @positive=62, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||| || |||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=89, @hit_id=\"gnl|BL_ORD_ID|92\", @len=18850, @definition=\"KM233058v1\", @accession=\"92\">, #<Bio::Blast::Report::Hit:0x0000000003f36508 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003f38ba0 @hsp={}, @num=1, @bit_score=77.8047, @score=39, @evalue=5.72664e-16, @query_from=1, @query_to=64, @hit_from=17510, @hit_to=17579, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=62, @positive=62, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||| || |||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=90, @hit_id=\"gnl|BL_ORD_ID|91\", @len=18954, @definition=\"KM233057v1\", @accession=\"91\">, #<Bio::Blast::Report::Hit:0x0000000003f3c0e8 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003f4aa30 @hsp={}, @num=1, @bit_score=77.8047, @score=39, @evalue=5.72664e-16, @query_from=1, @query_to=64, @hit_from=17513, @hit_to=17582, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=62, @positive=62, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||| || |||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=91, @hit_id=\"gnl|BL_ORD_ID|90\", @len=18957, @definition=\"KM233056v1\", @accession=\"90\">, #<Bio::Blast::Report::Hit:0x0000000003f51e98 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003f548a0 @hsp={}, @num=1, @bit_score=77.8047, @score=39, @evalue=5.72664e-16, @query_from=1, @query_to=64, @hit_from=17471, @hit_to=17540, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=62, @positive=62, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||| || |||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=92, @hit_id=\"gnl|BL_ORD_ID|89\", @len=18878, @definition=\"KM233055v1\", @accession=\"89\">, #<Bio::Blast::Report::Hit:0x0000000003f63da0 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003f66780 @hsp={}, @num=1, @bit_score=77.8047, @score=39, @evalue=5.72664e-16, @query_from=1, @query_to=64, @hit_from=17503, @hit_to=17572, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=62, @positive=62, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||| || |||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=93, @hit_id=\"gnl|BL_ORD_ID|88\", @len=18948, @definition=\"KM233054v1\", @accession=\"88\">, #<Bio::Blast::Report::Hit:0x0000000003f6db48 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003f70320 @hsp={}, @num=1, @bit_score=77.8047, @score=39, @evalue=5.72664e-16, @query_from=1, @query_to=64, @hit_from=17513, @hit_to=17582, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=62, @positive=62, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||| || |||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=94, @hit_id=\"gnl|BL_ORD_ID|87\", @len=18957, @definition=\"KM233053v1\", @accession=\"87\">, #<Bio::Blast::Report::Hit:0x0000000003f7f730 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003f82160 @hsp={}, @num=1, @bit_score=77.8047, @score=39, @evalue=5.72664e-16, @query_from=1, @query_to=64, @hit_from=17499, @hit_to=17568, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=62, @positive=62, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||| || |||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=95, @hit_id=\"gnl|BL_ORD_ID|86\", @len=18941, @definition=\"KM233052v1\", @accession=\"86\">, #<Bio::Blast::Report::Hit:0x0000000003f897a8 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003f8c098 @hsp={}, @num=1, @bit_score=77.8047, @score=39, @evalue=5.72664e-16, @query_from=1, @query_to=64, @hit_from=17509, @hit_to=17578, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=62, @positive=62, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||| || |||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=96, @hit_id=\"gnl|BL_ORD_ID|85\", @len=18951, @definition=\"KM233051v1\", @accession=\"85\">, #<Bio::Blast::Report::Hit:0x0000000003f9b4a8 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003fa1b78 @hsp={}, @num=1, @bit_score=77.8047, @score=39, @evalue=5.72664e-16, @query_from=1, @query_to=64, @hit_from=17514, @hit_to=17583, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=62, @positive=62, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||| || |||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=97, @hit_id=\"gnl|BL_ORD_ID|84\", @len=18956, @definition=\"KM233050v1\", @accession=\"84\">, #<Bio::Blast::Report::Hit:0x0000000003fa4ff8 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003fb39e0 @hsp={}, @num=1, @bit_score=77.8047, @score=39, @evalue=5.72664e-16, @query_from=1, @query_to=64, @hit_from=17504, @hit_to=17573, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=62, @positive=62, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||| || |||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=98, @hit_id=\"gnl|BL_ORD_ID|83\", @len=18946, @definition=\"KM233049v1\", @accession=\"83\">, #<Bio::Blast::Report::Hit:0x0000000003fb7040 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003fbdbe8 @hsp={}, @num=1, @bit_score=77.8047, @score=39, @evalue=5.72664e-16, @query_from=1, @query_to=64, @hit_from=17463, @hit_to=17532, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=62, @positive=62, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||| || |||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=99, @hit_id=\"gnl|BL_ORD_ID|82\", @len=18829, @definition=\"KM233048v1\", @accession=\"82\">, #<Bio::Blast::Report::Hit:0x0000000003fc1040 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003fcf6e0 @hsp={}, @num=1, @bit_score=77.8047, @score=39, @evalue=5.72664e-16, @query_from=1, @query_to=64, @hit_from=17479, @hit_to=17548, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=62, @positive=62, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||| || |||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=100, @hit_id=\"gnl|BL_ORD_ID|81\", @len=18921, @definition=\"KM233047v1\", @accession=\"81\">, #<Bio::Blast::Report::Hit:0x0000000003fd2b88 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003fd92d0 @hsp={}, @num=1, @bit_score=77.8047, @score=39, @evalue=5.72664e-16, @query_from=1, @query_to=64, @hit_from=17471, @hit_to=17540, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=62, @positive=62, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||| || |||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=101, @hit_id=\"gnl|BL_ORD_ID|80\", @len=18867, @definition=\"KM233046v1\", @accession=\"80\">, #<Bio::Blast::Report::Hit:0x0000000003fdca20 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003feb480 @hsp={}, @num=1, @bit_score=77.8047, @score=39, @evalue=5.72664e-16, @query_from=1, @query_to=64, @hit_from=17512, @hit_to=17581, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=62, @positive=62, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||| || |||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=102, @hit_id=\"gnl|BL_ORD_ID|79\", @len=18919, @definition=\"KM233045v1\", @accession=\"79\">, #<Bio::Blast::Report::Hit:0x0000000003fee8d8 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003ff50e8 @hsp={}, @num=1, @bit_score=77.8047, @score=39, @evalue=5.72664e-16, @query_from=1, @query_to=64, @hit_from=17470, @hit_to=17539, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=62, @positive=62, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||| || |||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=103, @hit_id=\"gnl|BL_ORD_ID|78\", @len=18914, @definition=\"KM233044v1\", @accession=\"78\">, #<Bio::Blast::Report::Hit:0x0000000003ff8478 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000004006e38 @hsp={}, @num=1, @bit_score=77.8047, @score=39, @evalue=5.72664e-16, @query_from=1, @query_to=64, @hit_from=17508, @hit_to=17577, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=62, @positive=62, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||| || |||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=104, @hit_id=\"gnl|BL_ORD_ID|77\", @len=18915, @definition=\"KM233043v1\", @accession=\"77\">, #<Bio::Blast::Report::Hit:0x000000000400e2c8 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000004010d70 @hsp={}, @num=1, @bit_score=77.8047, @score=39, @evalue=5.72664e-16, @query_from=1, @query_to=64, @hit_from=17470, @hit_to=17539, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=62, @positive=62, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||| || |||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=105, @hit_id=\"gnl|BL_ORD_ID|76\", @len=18912, @definition=\"KM233042v1\", @accession=\"76\">, #<Bio::Blast::Report::Hit:0x00000000040141f0 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000004022a98 @hsp={}, @num=1, @bit_score=77.8047, @score=39, @evalue=5.72664e-16, @query_from=1, @query_to=64, @hit_from=17479, @hit_to=17548, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=62, @positive=62, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||| || |||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=106, @hit_id=\"gnl|BL_ORD_ID|75\", @len=18886, @definition=\"KM233041v1\", @accession=\"75\">, #<Bio::Blast::Report::Hit:0x0000000004029d20 @hsps=[#<Bio::Blast::Report::Hsp:0x000000000402c9f8 @hsp={}, @num=1, @bit_score=77.8047, @score=39, @evalue=5.72664e-16, @query_from=1, @query_to=64, @hit_from=17480, @hit_to=17549, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=62, @positive=62, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||| || |||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=107, @hit_id=\"gnl|BL_ORD_ID|74\", @len=18922, @definition=\"KM233040v1\", @accession=\"74\">, #<Bio::Blast::Report::Hit:0x000000000403bb60 @hsps=[#<Bio::Blast::Report::Hsp:0x000000000403e630 @hsp={}, @num=1, @bit_score=77.8047, @score=39, @evalue=5.72664e-16, @query_from=1, @query_to=64, @hit_from=17508, @hit_to=17577, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=62, @positive=62, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||| || |||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=108, @hit_id=\"gnl|BL_ORD_ID|73\", @len=18953, @definition=\"KM233039v1\", @accession=\"73\">, #<Bio::Blast::Report::Hit:0x0000000004045a98 @hsps=[#<Bio::Blast::Report::Hsp:0x00000000040483b0 @hsp={}, @num=1, @bit_score=77.8047, @score=39, @evalue=5.72664e-16, @query_from=1, @query_to=64, @hit_from=17470, @hit_to=17539, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=62, @positive=62, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||| || |||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=109, @hit_id=\"gnl|BL_ORD_ID|72\", @len=18914, @definition=\"KM233038v1\", @accession=\"72\">, #<Bio::Blast::Report::Hit:0x0000000004057888 @hsps=[#<Bio::Blast::Report::Hsp:0x000000000405a290 @hsp={}, @num=1, @bit_score=77.8047, @score=39, @evalue=5.72664e-16, @query_from=1, @query_to=64, @hit_from=17470, @hit_to=17539, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=62, @positive=62, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||| || |||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=110, @hit_id=\"gnl|BL_ORD_ID|71\", @len=18915, @definition=\"KM233037v1\", @accession=\"71\">, #<Bio::Blast::Report::Hit:0x0000000004061450 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000004064038 @hsp={}, @num=1, @bit_score=77.8047, @score=39, @evalue=5.72664e-16, @query_from=1, @query_to=64, @hit_from=17466, @hit_to=17535, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=62, @positive=62, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||| || |||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=111, @hit_id=\"gnl|BL_ORD_ID|70\", @len=18873, @definition=\"KM233036v1\", @accession=\"70\">, #<Bio::Blast::Report::Hit:0x0000000004073358 @hsps=[#<Bio::Blast::Report::Hsp:0x000000000407a018 @hsp={}, @num=1, @bit_score=77.8047, @score=39, @evalue=5.72664e-16, @query_from=1, @query_to=64, @hit_from=17470, @hit_to=17539, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=62, @positive=62, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||| || |||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=112, @hit_id=\"gnl|BL_ORD_ID|69\", @len=18912, @definition=\"KM233035v1\", @accession=\"69\">, #<Bio::Blast::Report::Hit:0x000000000407d290 @hsps=[#<Bio::Blast::Report::Hsp:0x000000000408bb38 @hsp={}, @num=1, @bit_score=77.8047, @score=39, @evalue=5.72664e-16, @query_from=1, @query_to=64, @hit_from=17318, @hit_to=17387, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=62, @positive=62, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||| || |||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=113, @hit_id=\"gnl|BL_ORD_ID|68\", @len=18700, @definition=\"KM034563v1\", @accession=\"68\">, #<Bio::Blast::Report::Hit:0x00000000027dc970 @hsps=[#<Bio::Blast::Report::Hsp:0x000000000318fa80 @hsp={}, @num=1, @bit_score=77.8047, @score=39, @evalue=5.72664e-16, @query_from=1, @query_to=64, @hit_from=17514, @hit_to=17583, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=62, @positive=62, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||| || |||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=114, @hit_id=\"gnl|BL_ORD_ID|67\", @len=18957, @definition=\"KM034562v1\", @accession=\"67\">, #<Bio::Blast::Report::Hit:0x00000000033a4500 @hsps=[#<Bio::Blast::Report::Hsp:0x00000000034173c0 @hsp={}, @num=1, @bit_score=77.8047, @score=39, @evalue=5.72664e-16, @query_from=1, @query_to=64, @hit_from=17481, @hit_to=17550, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=62, @positive=62, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||| || |||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=115, @hit_id=\"gnl|BL_ORD_ID|66\", @len=18908, @definition=\"KM034561v1\", @accession=\"66\">, #<Bio::Blast::Report::Hit:0x0000000003ab1de8 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003ac2f80 @hsp={}, @num=1, @bit_score=77.8047, @score=39, @evalue=5.72664e-16, @query_from=1, @query_to=64, @hit_from=17508, @hit_to=17577, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=62, @positive=62, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||| || |||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=116, @hit_id=\"gnl|BL_ORD_ID|65\", @len=18952, @definition=\"KM034560v1\", @accession=\"65\">, #<Bio::Blast::Report::Hit:0x0000000003acde30 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003ae6ca0 @hsp={}, @num=1, @bit_score=77.8047, @score=39, @evalue=5.72664e-16, @query_from=1, @query_to=64, @hit_from=17484, @hit_to=17553, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=62, @positive=62, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||| || |||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=117, @hit_id=\"gnl|BL_ORD_ID|64\", @len=18926, @definition=\"KM034559v1\", @accession=\"64\">, #<Bio::Blast::Report::Hit:0x0000000003af17b8 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003b01988 @hsp={}, @num=1, @bit_score=77.8047, @score=39, @evalue=5.72664e-16, @query_from=1, @query_to=64, @hit_from=17485, @hit_to=17554, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=62, @positive=62, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||| || |||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=118, @hit_id=\"gnl|BL_ORD_ID|63\", @len=18915, @definition=\"KM034558v1\", @accession=\"63\">, #<Bio::Blast::Report::Hit:0x0000000003b0c810 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003b1cdf0 @hsp={}, @num=1, @bit_score=77.8047, @score=39, @evalue=5.72664e-16, @query_from=1, @query_to=64, @hit_from=17512, @hit_to=17581, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=62, @positive=62, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||| || |||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=119, @hit_id=\"gnl|BL_ORD_ID|62\", @len=18956, @definition=\"KM034557v1\", @accession=\"62\">, #<Bio::Blast::Report::Hit:0x0000000003b2c8e0 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003b413d0 @hsp={}, @num=1, @bit_score=77.8047, @score=39, @evalue=5.72664e-16, @query_from=1, @query_to=64, @hit_from=17490, @hit_to=17559, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=62, @positive=62, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||| || |||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=120, @hit_id=\"gnl|BL_ORD_ID|61\", @len=18932, @definition=\"KM034556v1\", @accession=\"61\">, #<Bio::Blast::Report::Hit:0x0000000003b4c0a0 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003b60c30 @hsp={}, @num=1, @bit_score=77.8047, @score=39, @evalue=5.72664e-16, @query_from=1, @query_to=64, @hit_from=17506, @hit_to=17575, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=62, @positive=62, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||| || |||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=121, @hit_id=\"gnl|BL_ORD_ID|60\", @len=18950, @definition=\"KM034555v1\", @accession=\"60\">, #<Bio::Blast::Report::Hit:0x0000000003b7f658 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003b8f4e0 @hsp={}, @num=1, @bit_score=77.8047, @score=39, @evalue=5.72664e-16, @query_from=1, @query_to=64, @hit_from=17506, @hit_to=17575, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=62, @positive=62, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||| || |||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=122, @hit_id=\"gnl|BL_ORD_ID|59\", @len=18948, @definition=\"KM034554v1\", @accession=\"59\">, #<Bio::Blast::Report::Hit:0x0000000003b9ee68 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003ba1398 @hsp={}, @num=1, @bit_score=77.8047, @score=39, @evalue=5.72664e-16, @query_from=1, @query_to=64, @hit_from=17485, @hit_to=17554, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=62, @positive=62, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||| || |||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=123, @hit_id=\"gnl|BL_ORD_ID|58\", @len=18927, @definition=\"KM034553v1\", @accession=\"58\">, #<Bio::Blast::Report::Hit:0x0000000003ba4840 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003baeed0 @hsp={}, @num=1, @bit_score=77.8047, @score=39, @evalue=5.72664e-16, @query_from=1, @query_to=64, @hit_from=17390, @hit_to=17459, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=62, @positive=62, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||| || |||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=124, @hit_id=\"gnl|BL_ORD_ID|57\", @len=18769, @definition=\"KM034552v1\", @accession=\"57\">, #<Bio::Blast::Report::Hit:0x0000000003bb6220 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003bb8868 @hsp={}, @num=1, @bit_score=77.8047, @score=39, @evalue=5.72664e-16, @query_from=1, @query_to=64, @hit_from=17484, @hit_to=17553, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=62, @positive=62, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||| || |||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=125, @hit_id=\"gnl|BL_ORD_ID|56\", @len=18926, @definition=\"KM034551v1\", @accession=\"56\">, #<Bio::Blast::Report::Hit:0x0000000003bc3df8 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003bca860 @hsp={}, @num=1, @bit_score=77.8047, @score=39, @evalue=5.72664e-16, @query_from=1, @query_to=64, @hit_from=17469, @hit_to=17538, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=62, @positive=62, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||| || |||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=126, @hit_id=\"gnl|BL_ORD_ID|55\", @len=18911, @definition=\"KM034550v1\", @accession=\"55\">, #<Bio::Blast::Report::Hit:0x0000000003bcd9e8 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003bd0378 @hsp={}, @num=1, @bit_score=77.8047, @score=39, @evalue=5.72664e-16, @query_from=1, @query_to=64, @hit_from=17469, @hit_to=17538, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=62, @positive=62, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||| || |||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=127, @hit_id=\"gnl|BL_ORD_ID|54\", @len=18835, @definition=\"KM034549v1\", @accession=\"54\">, #<Bio::Blast::Report::Hit:0x0000000003bdf558 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003be1cb8 @hsp={}, @num=1, @bit_score=77.8047, @score=39, @evalue=5.72664e-16, @query_from=1, @query_to=64, @hit_from=17514, @hit_to=17583, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=62, @positive=62, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||| || |||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=128, @hit_id=\"gnl|BL_ORD_ID|53\", @len=18959, @definition=\"KJ660348v2\", @accession=\"53\">, #<Bio::Blast::Report::Hit:0x0000000003be5188 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003bf3cb0 @hsp={}, @num=1, @bit_score=77.8047, @score=39, @evalue=5.72664e-16, @query_from=1, @query_to=64, @hit_from=17514, @hit_to=17583, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=62, @positive=62, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||| || |||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=129, @hit_id=\"gnl|BL_ORD_ID|52\", @len=18959, @definition=\"KJ660347v2\", @accession=\"52\">, #<Bio::Blast::Report::Hit:0x0000000003bf6d70 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003bf97c8 @hsp={}, @num=1, @bit_score=77.8047, @score=39, @evalue=5.72664e-16, @query_from=1, @query_to=64, @hit_from=17514, @hit_to=17583, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=62, @positive=62, @gaps=6, @align_len=70, @density=0, @qseq=\"AGCCCCGTTTTTTGCCACTGG------AATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AGCCCCGTTTTTTGCCACTGGGTATTTAATTAAGCCAATAACGTCAAGTGCCAGGTCTAGTGAGTGGTAT\", @midline=\"|||||||||||||||||||||      |||||||||||||||||||||||| || |||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=130, @hit_id=\"gnl|BL_ORD_ID|51\", @len=18959, @definition=\"KJ660346v2\", @accession=\"51\">, #<Bio::Blast::Report::Hit:0x0000000003bfc810 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003c0b388 @hsp={}, @num=1, @bit_score=34.1929, @score=17, @evalue=0.00769763, @query_from=32, @query_to=64, @hit_from=17496, @hit_to=17528, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=29, @positive=29, @gaps=0, @align_len=33, @density=0, @qseq=\"ATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"ATAACATCAAGTGCAAGGTCAAGTGAGTGGTAT\", @midline=\"||||| |||||||| || || ||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=131, @hit_id=\"gnl|BL_ORD_ID|23\", @len=18875, @definition=\"KC242783v2\", @accession=\"23\">, #<Bio::Blast::Report::Hit:0x0000000003c0e880 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003c11468 @hsp={}, @num=1, @bit_score=34.1929, @score=17, @evalue=0.00769763, @query_from=32, @query_to=64, @hit_from=17496, @hit_to=17528, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=29, @positive=29, @gaps=0, @align_len=33, @density=0, @qseq=\"ATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"ATAACATCAAGTGCAAGGTCAAGTGAGTGGTAT\", @midline=\"||||| |||||||| || || ||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=132, @hit_id=\"gnl|BL_ORD_ID|16\", @len=18875, @definition=\"FJ968794v1\", @accession=\"16\">, #<Bio::Blast::Report::Hit:0x0000000003c18970 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003c232a8 @hsp={}, @num=1, @bit_score=34.1929, @score=17, @evalue=0.00769763, @query_from=32, @query_to=64, @hit_from=17496, @hit_to=17528, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=29, @positive=29, @gaps=0, @align_len=33, @density=0, @qseq=\"ATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"ATAACATCAAGTGCAAGGTCAAGTGAGTGGTAT\", @midline=\"||||| |||||||| || || ||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=133, @hit_id=\"gnl|BL_ORD_ID|10\", @len=18875, @definition=\"EU338380v1\", @accession=\"10\">, #<Bio::Blast::Report::Hit:0x0000000003c269a8 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003c2d280 @hsp={}, @num=1, @bit_score=30.2282, @score=15, @evalue=0.120186, @query_from=49, @query_to=63, @hit_from=17553, @hit_to=17567, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=15, @positive=15, @gaps=0, @align_len=15, @density=0, @qseq=\"ATCTAGTGAGTGGTA\", @hseq=\"ATCTAGTGAGTGGTA\", @midline=\"|||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=134, @hit_id=\"gnl|BL_ORD_ID|157\", @len=18935, @definition=\"NC_014372v1\", @accession=\"157\">, #<Bio::Blast::Report::Hit:0x0000000003c30778 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003c3b308 @hsp={}, @num=1, @bit_score=30.2282, @score=15, @evalue=0.120186, @query_from=26, @query_to=64, @hit_from=17490, @hit_to=17528, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=33, @positive=33, @gaps=0, @align_len=39, @density=0, @qseq=\"AAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"AAGCCTATAACATCAAGTGCAAAGTCAAGTGAGTGGTAT\", @midline=\"||||| ||||| |||||||| |  || ||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=135, @hit_id=\"gnl|BL_ORD_ID|50\", @len=18875, @definition=\"KC589025v1\", @accession=\"50\">, #<Bio::Blast::Report::Hit:0x0000000003c42810 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003c451c8 @hsp={}, @num=1, @bit_score=30.2282, @score=15, @evalue=0.120186, @query_from=49, @query_to=63, @hit_from=17553, @hit_to=17567, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=15, @positive=15, @gaps=0, @align_len=15, @density=0, @qseq=\"ATCTAGTGAGTGGTA\", @hseq=\"ATCTAGTGAGTGGTA\", @midline=\"|||||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=136, @hit_id=\"gnl|BL_ORD_ID|12\", @len=18935, @definition=\"FJ217162v1\", @accession=\"12\">, #<Bio::Blast::Report::Hit:0x0000000003c48968 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003c547b8 @hsp={}, @num=1, @bit_score=26.2635, @score=13, @evalue=1.8765, @query_from=32, @query_to=64, @hit_from=17496, @hit_to=17528, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=28, @positive=28, @gaps=0, @align_len=33, @density=0, @qseq=\"ATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"ATAACATCAAGTGCAAAGTCAAGTGAGTGGTAT\", @midline=\"||||| |||||||| |  || ||||||||||||\">, #<Bio::Blast::Report::Hsp:0x0000000003c5fb90 @hsp={}, @num=2, @bit_score=24.2811, @score=12, @evalue=7.41476, @query_from=10, @query_to=21, @hit_from=16525, @hit_to=16536, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=12, @positive=12, @gaps=0, @align_len=12, @density=0, @qseq=\"TTTTGCCACTGG\", @hseq=\"TTTTGCCACTGG\", @midline=\"||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=137, @hit_id=\"gnl|BL_ORD_ID|156\", @len=18875, @definition=\"NC_006432v1\", @accession=\"156\">, #<Bio::Blast::Report::Hit:0x0000000003c63600 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003c69c58 @hsp={}, @num=1, @bit_score=26.2635, @score=13, @evalue=1.8765, @query_from=32, @query_to=64, @hit_from=17496, @hit_to=17528, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=28, @positive=28, @gaps=0, @align_len=33, @density=0, @qseq=\"ATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"ATAACATCAAGTGCAAAGTCAAGTGAGTGGTAT\", @midline=\"||||| |||||||| |  || ||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=138, @hit_id=\"gnl|BL_ORD_ID|45\", @len=18874, @definition=\"KC545392v1\", @accession=\"45\">, #<Bio::Blast::Report::Hit:0x0000000003c6d4c0 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003c77d80 @hsp={}, @num=1, @bit_score=26.2635, @score=13, @evalue=1.8765, @query_from=32, @query_to=64, @hit_from=17496, @hit_to=17528, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=28, @positive=28, @gaps=0, @align_len=33, @density=0, @qseq=\"ATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"ATAACATCAAGTGCAAAGTCAAGTGAGTGGTAT\", @midline=\"||||| |||||||| |  || ||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=139, @hit_id=\"gnl|BL_ORD_ID|44\", @len=18874, @definition=\"KC545391v1\", @accession=\"44\">, #<Bio::Blast::Report::Hit:0x0000000003c7f148 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003c81a60 @hsp={}, @num=1, @bit_score=26.2635, @score=13, @evalue=1.8765, @query_from=32, @query_to=64, @hit_from=17496, @hit_to=17528, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=28, @positive=28, @gaps=0, @align_len=33, @density=0, @qseq=\"ATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"ATAACATCAAGTGCAAAGTCAAGTGAGTGGTAT\", @midline=\"||||| |||||||| |  || ||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=140, @hit_id=\"gnl|BL_ORD_ID|43\", @len=18874, @definition=\"KC545390v1\", @accession=\"43\">, #<Bio::Blast::Report::Hit:0x0000000003c850e8 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003c93710 @hsp={}, @num=1, @bit_score=26.2635, @score=13, @evalue=1.8765, @query_from=32, @query_to=64, @hit_from=17496, @hit_to=17528, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=28, @positive=28, @gaps=0, @align_len=33, @density=0, @qseq=\"ATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"ATAACATCAAGTGCAAAGTCAAGTGAGTGGTAT\", @midline=\"||||| |||||||| |  || ||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=141, @hit_id=\"gnl|BL_ORD_ID|42\", @len=18874, @definition=\"KC545389v1\", @accession=\"42\">, #<Bio::Blast::Report::Hit:0x0000000003c97068 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003c99868 @hsp={}, @num=1, @bit_score=26.2635, @score=13, @evalue=1.8765, @query_from=32, @query_to=64, @hit_from=17496, @hit_to=17528, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=28, @positive=28, @gaps=0, @align_len=33, @density=0, @qseq=\"ATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"ATAACATCAAGTGCAAAGTCAAGTGAGTGGTAT\", @midline=\"||||| |||||||| |  || ||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=142, @hit_id=\"gnl|BL_ORD_ID|19\", @len=18875, @definition=\"JN638998v1\", @accession=\"19\">, #<Bio::Blast::Report::Hit:0x0000000003c9cef0 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003ca88b8 @hsp={}, @num=1, @bit_score=26.2635, @score=13, @evalue=1.8765, @query_from=32, @query_to=64, @hit_from=17496, @hit_to=17528, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=28, @positive=28, @gaps=0, @align_len=33, @density=0, @qseq=\"ATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\", @hseq=\"ATAACATCAAGTGCAAAGTCAAGTGAGTGGTAT\", @midline=\"||||| |||||||| |  || ||||||||||||\">, #<Bio::Blast::Report::Hsp:0x0000000003cac058 @hsp={}, @num=2, @bit_score=24.2811, @score=12, @evalue=7.41476, @query_from=10, @query_to=21, @hit_from=16525, @hit_to=16536, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=1, @identity=12, @positive=12, @gaps=0, @align_len=12, @density=0, @qseq=\"TTTTGCCACTGG\", @hseq=\"TTTTGCCACTGG\", @midline=\"||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=143, @hit_id=\"gnl|BL_ORD_ID|7\", @len=18875, @definition=\"AY729654v1\", @accession=\"7\">, #<Bio::Blast::Report::Hit:0x0000000003cbb760 @hsps=[#<Bio::Blast::Report::Hsp:0x0000000003cbdf10 @hsp={}, @num=1, @bit_score=24.2811, @score=12, @evalue=7.41476, @query_from=19, @query_to=8, @hit_from=12730, @hit_to=12741, @pattern_from=0, @pattern_to=0, @query_frame=1, @hit_frame=-1, @identity=12, @positive=12, @gaps=0, @align_len=12, @density=0, @qseq=\"AGTGGCAAAAAA\", @hseq=\"AGTGGCAAAAAA\", @midline=\"||||||||||||\">], @query_id=\"lcl|1_0\", @query_def=\"myseq\", @query_len=64, @num=144, @hit_id=\"gnl|BL_ORD_ID|159\", @len=19114, @definition=\"NC_024781v1\", @accession=\"159\">]"
      ]
     },
     "execution_count": 3,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "require 'bio'\n",
    "require 'stringio'\n",
    "\n",
    "# blast is installed globally, so you don't need to include the full path to blastn\n",
    "factory = Bio::Blast.local('blastn', './Databases/EBOLA') \n",
    "\n",
    "# executes homology search. Returns Bio::Blast::Report object.\n",
    "report = factory.query(\">myseq\\nAGCCCCGTTTTTTGCCACTGGAATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\") # this is a Bio::Blast::Report\n",
    "\n",
    "# Iterates over each hit\n",
    "report.each do |hit|\n",
    "  print \"#{hit.hit_id} : evalue #{hit.evalue}\\t#{hit.target_id} at \"\n",
    "  puts \"#{hit.lap_at}\"   # this tells you start and end of both the query and the hit sequences\n",
    "  hit.each do |hsp|\n",
    "    puts hsp.qseq  # this is the gapped Alignment as text of the query\n",
    "    puts hsp.hseq  # this is the gapped Alignment as text of the hit\n",
    "  end\n",
    "end"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "#  Put together things that we have learned\n",
    "\n",
    "## The Plan\n",
    "\n",
    "* For each gene on the Arabidopsis gene list, we need the FASTA formatted sequence\n",
    "* We can get this FASTA from DB fetch (we think!)\n",
    "* We then index this FASTA file using makeblastdb to create a BLAST database\n",
    "* We go for aperitivos to celebrate how smart we are :-)\n",
    "\n",
    "##### unfortunately, things are seldom as easy as you think they will be!\n",
    "\n",
    "\n",
    "**Your first task:  use dbfetch to retrieve the FASTA sequences for the <span style=\"color:red;\">first 3-4 Arabidopsis gene IDs</span> on the list we have been using (NOT all of them, yet!)**  \n",
    "\n",
    "* Go back in your course notes (or look-up using Google) the format for dbfetch URLs.  We need the ensemblgenomesgene database and fasta format.\n",
    "\n",
    "* In your script, put all of the IDs into a single URL (i.e. don't call DBFetch over and over again)\n",
    "\n",
    "* write the output to a file called \"ARA.fa\"\n",
    "\n",
    "TASK 7A and TASK 7B are in separate Lecutre files.  Open them now and complete the tasks.\n",
    "\n",
    "### be sure you understand Lesson 7 Task Solutions A and B before you continue!"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Accessing non-sequence data from NCBI - eUtils\n",
    "\n",
    "Before we begin, take note of the following announcement from NCBI:\n",
    "\n",
    "-----------\n",
    "<em>Coming in May 2018: API Keys\n",
    "\n",
    "On May 1, 2018, NCBI will begin enforcing the use of API keys that will offer enhanced levels of supported access to the E-utilities. After that date, any site (IP address) posting more than 3 requests per second to the E-utilities without an API key will receive an error message. By including an API key, a site can post up to 10 requests per second by default. Higher rates are available by request (vog.hin.mln.ibcn@seitilitue). Users can obtain an API key now from the Settings page of their NCBI account (to create an account, visit www.ncbi.nlm.nih.gov/account/). After creating the key, users should include it in each E-utility request by assigning it to the new api_key parameter.\n",
    "\n",
    "Example request including an API key:\n",
    "esummary.fcgi?db=pubmed&id=123456&api_key=ABCDE12345\n",
    "\n",
    "Example error message if rates are exceeded:\n",
    "{\"error\":\"API rate limit exceeded\",\"count\":\"11\"}\n",
    "\n",
    "Only one API key is allowed per NCBI account; however, a user may request a new key at any time. Such a request will invalidate any existing API key associated with that NCBI account.\n",
    "\n",
    "We encourage regular E-utility users to obtain an API key as soon as possible and begin the process of incorporating it into code. We also encourage users to monitor their request rates to determine if they will require rates higher than 10 per second. As stated above, we can potentially have higher rates negotiated prior to the beginning of enforcement on May 1, 2018.</em>\n",
    "\n",
    "----------------\n",
    "\n",
    "\n",
    "We may need to have all students register with NCBI; however, I will try to avoid this by providing examples that make only single requests.  \n",
    "\n",
    "\n",
    "# OVERVIEW \n",
    "\n",
    "**The Nine E-utilities in Brief**\n",
    "\n",
    "EUtilities consists of 9 \"REST\" style interfaces (i.e. parameterized URLs):\n",
    "\n",
    "\n",
    "* **EInfo** (database statistics)\n",
    "\n",
    "    http://eutils.ncbi.nlm.nih.gov/entrez/eutils/einfo.fcgi\n",
    "\n",
    "    Provides the number of records indexed in each field of a given database, the date of the last update of the database, and the available links from the database to other Entrez databases.\n",
    "\n",
    "* **ESearch** (text searches)\n",
    "\n",
    "    http://eutils.ncbi.nlm.nih.gov/entrez/eutils/esearch.fcgi\n",
    "\n",
    "    Responds to a text query with the list of matching UIDs in a given database (for later use in ESummary, EFetch or ELink), along with the term translations of the query.\n",
    "\n",
    "* **EPost** (UID uploads)\n",
    "\n",
    "    http://eutils.ncbi.nlm.nih.gov/entrez/eutils/epost.fcgi\n",
    "\n",
    "    Accepts a list of UIDs from a given database, stores the set on the History Server, and responds with a query key and web environment for the uploaded dataset.\n",
    "\n",
    "* **ESummary** (document summary downloads)\n",
    "\n",
    "    http://eutils.ncbi.nlm.nih.gov/entrez/eutils/esummary.fcgi\n",
    "\n",
    "    Responds to a list of UIDs from a given database with the corresponding document summaries.\n",
    "\n",
    "* **EFetch** (data record downloads)\n",
    "\n",
    "    http://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi\n",
    "\n",
    "    Responds to a list of UIDs in a given database with the corresponding data records in a specified format.\n",
    "\n",
    "* **ELink** (Entrez links)\n",
    "\n",
    "    http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi\n",
    "\n",
    "    Responds to a list of UIDs in a given database with either a list of related UIDs (and relevancy scores) in the same database or a list of linked UIDs in another Entrez database; checks for the existence of a specified link from a list of one or more UIDs; creates a hyperlink to the primary LinkOut provider for a specific UID and database, or lists LinkOut URLs and attributes for multiple UIDs.\n",
    "\n",
    "* **EGQuery** (global query)\n",
    "\n",
    "    http://eutils.ncbi.nlm.nih.gov/entrez/eutils/egquery.fcgi\n",
    "\n",
    "    Responds to a text query with the number of records matching the query in each Entrez database.\n",
    "\n",
    "* **ESpell** (spelling suggestions)\n",
    "\n",
    "    http://eutils.ncbi.nlm.nih.gov/entrez/eutils/espell.fcgi\n",
    "\n",
    "    Retrieves spelling suggestions for a text query in a given database.\n",
    "\n",
    "* **ECitMatch** (batch citation searching in PubMed)\n",
    "\n",
    "    http://eutils.ncbi.nlm.nih.gov/entrez/eutils/ecitmatch.cgi\n",
    "\n",
    "    Retrieves PubMed IDs (PMIDs) corresponding to a set of input citation strings.\n",
    "\n",
    "\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Getting Started with EUtils\n",
    "\n",
    "Before we look at the BioRuby Classes for EUtils, we will look at the EUtils REST interface.  The complete tutorial for EUtils is at https://www.ncbi.nlm.nih.gov/books/NBK25497/\n",
    "\n",
    "I will take you through a (small!) subset of these functionalities in class.  With that background, you should be able to understand the rest of the functionality yourself.\n",
    "\n",
    "To begin, click on the link for EInfo:\n",
    "\n",
    "\n",
    "http://eutils.ncbi.nlm.nih.gov/entrez/eutils/einfo.fcgi\n",
    "\n",
    "This lists all of the databases that you can query/access from NCBI.  Each of those databases uses a **different type of identifier.**  Those are listed [here](https://www.ncbi.nlm.nih.gov/books/NBK25497/table/chapter2.T._entrez_unique_identifiers_ui/?report=objectonly)\n",
    "\n",
    "To find out more about any of those databases (in particular, the fields that can be queried, and the database-to-database linkages that are available!) the pattern is like what we learned for dbFetch - a set of key/value pairs after a '?' (separated by ';' if there is more than one)\n",
    "\n",
    "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/einfo.fcgi?db=nuccore\n",
    "\n",
    "\n",
    "From the output of nuccore we can see, for example, that you can query the field \"Title\" (TITL) in the nuccore database:\n",
    "\n",
    "    <Field>\n",
    "        <Name>TITL</Name>\n",
    "        <FullName>Title</FullName>\n",
    "\n",
    "\n",
    "The important part of that is the field name:  **TITL** and we will now see how to use it...\n",
    "\n",
    "\n",
    "# Searching NCBI Databases\n",
    "\n",
    "\n",
    "NCBI databases use a query language called ENTREZ.  ENTREZ queries are structured as \"value\"[FIELD]\n",
    "\n",
    "To do queries on the, for example, the Nucleotide database ('nuccore'), you would use the ESearch utility.  If you were looking for nucleotide entries with titles including the phrase \"apoptosis inhibitor\", you might create a URL with the following structure:\n",
    "\n",
    "[http://eutils.ncbi.nlm.nih.gov/entrez/eutils/**esearch.fcgi?db=nuccore;term=\"apoptosis+inhibitor\"[TITL]**](http://eutils.ncbi.nlm.nih.gov/entrez/eutils/esearch.fcgi?db=nuccore;term=\"apoptosis+inhibitor\"[TITL])\n",
    "\n",
    "The output of that call is a list of identifiers (what kind of identifiers?  See the \"nuccore\" entry in the linked table above) for the first 20 records that have \"apoptosis inhibitor\" in their title.\n",
    "\n",
    "<code>\n",
    "\n",
    "    <eSearchResult>\n",
    "    <Count>2138</Count>   <----   The number of matches in the database\n",
    "    <RetMax>20</RetMax>   <----   The maximum number of records returned\n",
    "    <RetStart>0</RetStart>  <---- The \"offset\" - i.e. what record am I starting with?  (in this case the first)\n",
    "    <IdList>\n",
    "    <Id>219283154</Id><Id>219283151</Id><Id>219283149</Id><Id>197098447</Id><Id>148230698</Id><Id>148229867</Id><Id>57529973</Id><Id>1279832269</Id><Id>1279832267</Id><Id>1279831517</Id><Id>1279711068</Id><Id>1279105385</Id><Id>1279090872</Id><Id>1279061012</Id><Id>1279061010</Id><Id>1279048499</Id><Id>1279048498</Id><Id>1279048496</Id><Id>1279048494</Id><Id>816197788</Id>\n",
    "    </IdList>\n",
    "    <TranslationSet/>\n",
    "    <TranslationStack>\n",
    "    <TermSet>\n",
    "    <Term>\"apoptosis inhibitor\"[TITL]</Term>\n",
    "    <Field>TITL</Field>\n",
    "    <Count>2138</Count>\n",
    "    <Explode>N</Explode>\n",
    "    </TermSet>\n",
    "    <OP>GROUP</OP>\n",
    "    </TranslationStack>\n",
    "    <QueryTranslation>\"apoptosis inhibitor\"[TITL]</QueryTranslation>\n",
    "    </eSearchResult>\n",
    "\n",
    "</code>\n",
    "\n",
    "To get more results, you change/add parameters.  For example, to get the next 50 records AFTER these records, the URL would be:\n",
    "\n",
    "\n",
    "[http://eutils.ncbi.nlm.nih.gov/entrez/eutils/**esearch.fcgi?<span style=\"color:red;\">retstart=20;retmax=50</span>;db=nuccore;term=\"apoptosis+inhibitor\"[TITL]**](http://eutils.ncbi.nlm.nih.gov/entrez/eutils/esearch.fcgi?retstart=20;retmax=50;db=nuccore;term=\"apoptosis+inhibitor\"[TITL])\n",
    "\n",
    "Finally, to increase the complexity of the search, you use AND, OR, and NOT operators, with the same \"value\"[FIELD] structure.  So to limit the current query to only those from the genus \"Spodoptera\" (\"ORGN = Organism) (NOTE that we reset the retstart back to zero!!)\n",
    "\n",
    "[http://eutils.ncbi.nlm.nih.gov/entrez/eutils/**esearch.fcgi?retstart=0;retmax=50;db=nuccore;<span style=\"color:red;\">term=\"apoptosis+inhibitor\"[TITL]+AND+\"Spodoptera\"[ORGN]**</span>](http://eutils.ncbi.nlm.nih.gov/entrez/eutils/esearch.fcgi?retstart=0;retmax=50;db=nuccore;term=\"apoptosis+inhibitor\"[TITL]+AND+\"Spodoptera\"[ORGN])\n",
    "\n",
    "The result is a single DNA sequence:  gi:1274128812\n",
    "\n",
    "\n",
    "# Retrieving from search results\n",
    "\n",
    "To retrieve that sequence, we use the EFetch utility ([Documentation Here](https://www.ncbi.nlm.nih.gov/books/NBK25499/#chapter4.EFetch)).  The process is exactly the same - create URL parameters after <code>efetch.fcgi?</code>  The two parameters of greatest importance to us are \"db=\" (which database am I fetching from?) and \"id=\" (which records am I fetching?).  So our next URL looks like this:\n",
    "\n",
    "[http://eutils.ncbi.nlm.nih.gov/entrez/eutils/**efetch.fcgi?db=nuccore;id=1274128812**](http://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=nuccore;id=1274128812)\n",
    "\n",
    "Note:  if you click that, it will pop-up a window.  That is because the default data-type returned from the Nucleotide sequence database (nuccore) is called \"ASN.1\", and you probably don't have a utility that will read that format.  You **do** have a utility that will read GenBank format, though!  **BioRuby!!**   So is there a way to get that record in GenBank format?  Yes.  There are two additional parameters for EFetch:  retmode and rettype.  Each of the databases supports a subset of possible values for retmode and rettype.  You can look at the possibilities [HERE](https://www.ncbi.nlm.nih.gov/books/NBK25499/table/chapter4.T._valid_values_of__retmode_and/?report=objectonly).  For us to get a GenBank formatted file, we have to set retmode=gb and rettype=text:\n",
    "\n",
    "[http://eutils.ncbi.nlm.nih.gov/entrez/eutils/**efetch.fcgi?db=nuccore;id=1274128812;<span style=\"color:red;\">rettype=gb;retmode=text**</span>](http://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=nuccore;rettype=gb;retmode=text;id=1274128812)\n",
    "\n",
    "Of course, if you need more than one record, you can request them all at the same time by adding more GI numbers separated by commas:\n",
    "\n",
    "\n",
    "[http://eutils.ncbi.nlm.nih.gov/entrez/eutils/**efetch.fcgi?db=nuccore;rettype=gb;retmode=text;<span style=\"color:red;\">id=1274128812,1274128812,1274128812,1274128812**</span>](http://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=nuccore;rettype=gb;retmode=text;id=1274128812,1274128812,1274128812,1274128812)\n",
    "\n",
    "From this point, you know what to do!  This is exactly the same as what we learned with EFetch and using BioRuby to parse the output.\n",
    "\n",
    "**BE CAREFUL!!**   NCBI is very very strict about their limits on number of requests!  For an account with no \"license\", you can only call 3 times per second.  With a license, only 10 times per second!  **So do NOT do this:**\n",
    "\n",
    "   \n",
    "\n",
    "    ids.each do |id|\n",
    "     one_record = fetch(\"http://eutils.../efetch.fcgi?db=nuccore;rettype=gb;retmode=text;id=#{id}\")\n",
    "     process(one_record)\n",
    "    end\n",
    "\n",
    " \n",
    "\n",
    "**That is how you get your entire institute BANNED from using NCBI utilities!**  \n",
    "\n",
    "<span style=\"color:red;\">YOU HAVE BEEN WARNED!!  :-)</span>\n",
    "\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Prove that you understand\n",
    "\n",
    "* Find the PubMed publications written by an author named Mark D Wilkinson who studies \"Reproducibility of Results\" (find the [MESH keyword](https://meshb.nlm.nih.gov/search) and use it in the [MESH] field of the search query.  Retrieve the abstract for the first 3 of these, in text format.\n",
    "\n",
    "(You will need to search for how ENTREZ represents author names!)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "\n",
    "# Enough of this REST stuff, let's use BioRuby!\n",
    "\n",
    "The BioRuby interface is very \"thin\" - it simply makes it easier to construct the EUtilities REST URLs.\n",
    "\n",
    "http://bioruby.org/rdoc/Bio/NCBI/REST\n",
    "\n",
    "\n",
    "A typical call is like this:\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "1676439675,1388704174,33527965,33527963,33527961\n",
      "\n",
      "\n",
      "Records are\n",
      "\n",
      "\n",
      "LOCUS       NM_018156              16282 bp    mRNA    linear   PRI 21-AUG-2019\n",
      "DEFINITION  Homo sapiens vacuolar protein sorting 13 homolog D (VPS13D),\n",
      "            transcript variant 2, mRNA.\n",
      "ACCESSION   NM_018156\n",
      "VERSION     NM_018156.4\n",
      "KEYWORDS    RefSeq.\n",
      "SOURCE      Homo sapiens (human)\n",
      "  ORGANISM  Homo sapiens\n",
      "            Eukaryota; Metazoa; Chordata; Craniata; Vertebrata; Euteleostomi;\n",
      "            Mammalia; Eutheria; Euarchontoglires; Primates; Haplorrhini;\n",
      "            Catarrhini; Hominidae; Homo.\n",
      "REFERENCE   1  (bases 1 to 16282)\n",
      "  AUTHORS   Seong E, Insolera R, Dulovic M, Kamsteeg EJ, Trinh J, Bruggemann N,\n",
      "            Sandford E, Li S, Ozel AB, Li JZ, Jewett T, Kievit AJA, Munchau A,\n",
      "            Shakkottai V, Klein C, Collins CA, Lohmann K, van de Warrenburg BP\n",
      "            and Burmeister M.\n",
      "  TITLE     Mutations in VPS13D lead to a new recessive ataxia with spasticity\n",
      "            and mitochondrial defects\n",
      "  JOURNAL   Ann. Neurol. 83 (6), 1075-1088 (2018)\n",
      "   PUBMED   29604224\n",
      "  REMARK    GeneRIF: heterozygous mutations in VPS13D cause movement disorders\n",
      "            along the ataxia-spasticity spectrum\n",
      "REFERENCE   2  (bases 1 to 16282)\n",
      "  AUTHORS   Gauthier J, Meijer IA, Lessel D, Mencacci NE, Krainc D, Hempel M,\n",
      "            Tsiakas K, Prokisch H, Rossignol E, Helm MH, Rodan LH,\n",
      "            Karamchandani J, Carecchio M, Lubbe SJ, Telegrafi A, Henderson LB,\n",
      "            Lorenzo K, Wallace SE, Glass IA, Hamdan FF, Michaud JL, Rouleau GA\n",
      "            and Campeau PM.\n",
      "  TITLE     Recessive mutations in >VPS13D cause childhood onset movement\n",
      "            disorders\n",
      "  JOURNAL   Ann. Neurol. 83 (6), 1089-1095 (2018)\n",
      "   PUBMED   29518281\n",
      "  REMARK    GeneRIF: findings underline the importance of the VPS13 complex in\n",
      "            neurological diseases and a possible role in mitochondrial function\n",
      "REFERENCE   3  (bases 1 to 16282)\n",
      "  AUTHORS   Anding AL, Wang C, Chang TK, Sliter DA, Powers CM, Hofmann K, Youle\n",
      "            RJ and Baehrecke EH.\n",
      "  TITLE     Vps13D Encodes a Ubiquitin-Binding Protein that Is Required for the\n",
      "            Regulation of Mitochondrial Size and Clearance\n",
      "  JOURNAL   Curr. Biol. 28 (2), 287-295 (2018)\n",
      "   PUBMED   29307555\n",
      "REFERENCE   4  (bases 1 to 16282)\n",
      "  AUTHORS   Nakada TA, Boyd JH, Russell JA, Aguirre-Hernandez R, Wilkinson MD,\n",
      "            Thair SA, Nakada E, McConechy MK, Fjell CD and Walley KR.\n",
      "  TITLE     VPS13D Gene Variant Is Associated with Altered IL-6 Production and\n",
      "            Mortality in Septic Shock\n",
      "  JOURNAL   J Innate Immun 7 (5), 545-553 (2015)\n",
      "   PUBMED   25896417\n",
      "  REMARK    GeneRIF: genetic polymorphism is associated with il-6 production\n",
      "            and mortality in patients with septic shock\n",
      "REFERENCE   5  (bases 1 to 16282)\n",
      "  AUTHORS   Koriyama H, Nakagami H, Katsuya T, Sugimoto K, Yamashita H, Takami\n",
      "            Y, Maeda S, Kubo M, Takahashi A, Nakamura Y, Ogihara T, Rakugi H,\n",
      "            Kaneda Y and Morishita R.\n",
      "  TITLE     Identification of evidence suggestive of an association with\n",
      "            peripheral arterial disease at the OSBPL10 locus by genome-wide\n",
      "            investigation in the Japanese population\n",
      "  JOURNAL   J. Atheroscler. Thromb. 17 (10), 1054-1062 (2010)\n",
      "   PUBMED   20610895\n",
      "  REMARK    GeneRIF: Observational study and genome-wide association study of\n",
      "            gene-disease association. (HuGE Navigator)\n",
      "REFERENCE   6  (bases 1 to 16282)\n",
      "  AUTHORS   Gonzales PA, Pisitkun T, Hoffert JD, Tchapyjnikov D, Star RA, Kleta\n",
      "            R, Wang NS and Knepper MA.\n",
      "  TITLE     Large-scale proteomics and phosphoproteomics of urinary exosomes\n",
      "  JOURNAL   J. Am. Soc. Nephrol. 20 (2), 363-379 (2009)\n",
      "   PUBMED   19056867\n",
      "REFERENCE   7  (bases 1 to 16282)\n",
      "  AUTHORS   Velayos-Baeza A, Vettori A, Copley RR, Dobson-Stone C and Monaco\n",
      "            AP.\n",
      "  TITLE     Analysis of the human VPS13 gene family\n",
      "  JOURNAL   Genomics 84 (3), 536-549 (2004)\n",
      "   PUBMED   15498460\n",
      "REFERENCE   8  (bases 1 to 16282)\n",
      "  AUTHORS   Nakayama M, Kikuno R and Ohara O.\n",
      "  TITLE     Protein-protein interactions between large proteins: two-hybrid\n",
      "            screening using a functionally classified library composed of long\n",
      "            cDNAs\n",
      "  JOURNAL   Genome Res. 12 (11), 1773-1784 (2002)\n",
      "   PUBMED   12421765\n",
      "REFERENCE   9  (bases 1 to 16282)\n",
      "  AUTHORS   Meijer,I.A.\n",
      "  TITLE     VPS13D Movement Disorder\n",
      "  JOURNAL   (in) Adam MP, Ardinger HH, Pagon RA, Wallace SE, Bean LJH, Stephens\n",
      "            K and Amemiya A (Eds.);\n",
      "            GENEREVIEWS((R));\n",
      "            (1993)\n",
      "   PUBMED   30789691\n",
      "REFERENCE   10 (bases 1 to 16282)\n",
      "  AUTHORS   Bird,T.D.\n",
      "  TITLE     Hereditary Ataxia Overview\n",
      "  JOURNAL   (in) Adam MP, Ardinger HH, Pagon RA, Wallace SE, Bean LJH, Stephens\n",
      "            K and Amemiya A (Eds.);\n",
      "            GENEREVIEWS((R));\n",
      "            (1993)\n",
      "   PUBMED   20301317\n",
      "COMMENT     REVIEWED REFSEQ: This record has been curated by NCBI staff. The\n",
      "            reference sequence was derived from AL031276.1, HY121734.1,\n",
      "            AJ608775.1, BX682532.4 and BC051804.1.\n",
      "            On Jun 1, 2019 this sequence version replaced NM_018156.3.\n",
      "            \n",
      "            Summary: This gene encodes a protein belonging to the\n",
      "            vacuolar-protein-sorting-13 gene family. In yeast,\n",
      "            vacuolar-protein-sorting-13 proteins are involved in trafficking of\n",
      "            membrane proteins between the trans-Golgi network and the\n",
      "            prevacuolar compartment. While several transcript variants may\n",
      "            exist for this gene, the full-length natures of only two have been\n",
      "            described to date. These two represent the major variants of this\n",
      "            gene and encode distinct isoforms. [provided by RefSeq, Jul 2008].\n",
      "            \n",
      "            Transcript Variant: This variant (2) lacks an alternate in-frame\n",
      "            exon, compared to variant 1, resulting in a shorter protein\n",
      "            (isoform 2), compared to isoform 1.\n",
      "            \n",
      "            ##Evidence-Data-START##\n",
      "            Transcript exon combination :: AJ608775.1 [ECO:0000332]\n",
      "            RNAseq introns              :: single sample supports all introns\n",
      "                                           SAMEA1965299, SAMEA1968540\n",
      "                                           [ECO:0000348]\n",
      "            ##Evidence-Data-END##\n",
      "            COMPLETENESS: complete on the 3' end.\n",
      "PRIMARY     REFSEQ_SPAN         PRIMARY_IDENTIFIER PRIMARY_SPAN        COMP\n",
      "            1-9                 AL031276.1         81577-81585         c\n",
      "            10-37               HY121734.1         1-28\n",
      "            38-8513             AJ608775.1         1-8476\n",
      "            8514-8514           BX682532.4         21312-21312\n",
      "            8515-8749           AJ608775.1         8478-8712\n",
      "            8750-8750           BX682532.4         29838-29838\n",
      "            8751-14596          AJ608775.1         8714-14559\n",
      "            14597-16282         BC051804.1         4065-5750\n",
      "FEATURES             Location/Qualifiers\n",
      "     source          1..16282\n",
      "                     /organism=\"Homo sapiens\"\n",
      "                     /mol_type=\"mRNA\"\n",
      "                     /db_xref=\"taxon:9606\"\n",
      "                     /chromosome=\"1\"\n",
      "                     /map=\"1p36.22-p36.21\"\n",
      "     gene            1..16282\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /note=\"vacuolar protein sorting 13 homolog D\"\n",
      "                     /db_xref=\"GeneID:55187\"\n",
      "                     /db_xref=\"HGNC:HGNC:23595\"\n",
      "                     /db_xref=\"MIM:608877\"\n",
      "     exon            1..91\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            92..264\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     misc_feature    141..143\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /note=\"upstream in-frame stop codon\"\n",
      "     CDS             168..13259\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /note=\"isoform 2 is encoded by transcript variant 2;\n",
      "                     vacuolar protein sorting-associated protein 13D; vacuolar\n",
      "                     protein sorting 13D\"\n",
      "                     /codon_start=1\n",
      "                     /product=\"vacuolar protein sorting-associated protein 13D\n",
      "                     isoform 2\"\n",
      "                     /protein_id=\"NP_060626.2\"\n",
      "                     /db_xref=\"CCDS:CCDS30589.1\"\n",
      "                     /db_xref=\"GeneID:55187\"\n",
      "                     /db_xref=\"HGNC:HGNC:23595\"\n",
      "                     /db_xref=\"MIM:608877\"\n",
      "                     /translation=\"MLEGLVAWVLNTYLGKYVNNLNTDQLSVALLKGAVELENLPLKK\n",
      "                     DALKELELPFEVKAGFIGKVTLQIPFYRPHVDPWVISISSLHLIGAPEKIQDFNDEKE\n",
      "                     KLLERERKKALLQALEEKWKNDRQQKGESYWYSVTASVVTRIVENIELKIQDVHLRFE\n",
      "                     DGVTNPSHPFAFGICIKNVSMQNAVNEPVQKLMRKKQLDVAEFSIYWDVDCTLLGDLP\n",
      "                     QMELQEAMARSMESRSHHYVLEPVFASALLKRNCSKKPLRSRHSPRIDCDIQLETIPL\n",
      "                     KLSQLQYRQIMEFLKELERKERQVKFRRWKPKVAISKNCREWWYFALNANLYEIREQR\n",
      "                     KRCTWDFMLHRARDAVSYTDKYFNKLKGGLLSTDDKEEMCRIEEEQSFEELKILRELV\n",
      "                     HDRFHKQEELAESLREPQFDSPGACPGAPEPGGGSGMLQYLQSWFPGWGGWYGQQTPE\n",
      "                     GNVVEGLSAEQQEQWIPEEILGTEEFFDPTADASCMNTYTKRDHVFAKLNLQLQRGTV\n",
      "                     TLLHKEQGTPQMNESAFMQLEFSDVKLLAESLPRRNSSLLSVRLGGLFLRDLATEGTM\n",
      "                     FPLLVFPNPQKEVGRVSQSFGLQTTSADRSDHYPAADPDGPVFEMLYERNPAHSHFER\n",
      "                     RLNVSTRPLNIIYNPQAIKKVADFFYKGKVHTSGFGYQSELELRVAEAARRQYNKLKM\n",
      "                     QTKAEIRQTLDRLLVGDFIEESKRWTVRLDISAPQVIFPDDFKFKNPVLVVVDLGRML\n",
      "                     LTNTQDNSRRKSRDGSASEETQFSDDEYKTPLATPPNTPPPESSSSNGEKTPPFSGVE\n",
      "                     FSEEQLQAHLMSTKMYERYSLSFMDLQIMVGRVKDNWKHVQDIDVGPTHVVEKFNVHL\n",
      "                     QLERRLIYTSDPKYPGAVLSGNLPDLKIHINEDKISALKNCFALLTTPEMKTSDTQIK\n",
      "                     EKIFPQEEQRGSLQDSVMNLTQSIVLLEQHTREVLVESQLLLAEFKVNCMQLGVESNG\n",
      "                     RYISVLKVFGTNAHFVKRPYDAEVSLTVHGLLLVDTMQTYGADFDLLMASHKNLSFDI\n",
      "                     PTGSLRDSRAQSPVSGPNVAHLTDGATLNDRSATSVSLDKILTKEQESLIKLEYQFVS\n",
      "                     SECPSMNLDSTLQVISLQVNNLDIILNPETIVELIGFLQKSFPKEKDDLSPQPLMTDF\n",
      "                     ERSFREQGTYQSTYEQNTEVAVEIHRLNLLLLRTVGMANREKYGRKIATASIGGTKVN\n",
      "                     VSMGSTFDMNGSLGCLQLMDLTQDNVKNQYVVSIGNSVGYENIISDIGYFESVFVRME\n",
      "                     DAALTEALSFTFVERSKQECFLNLKMASLHYNHSAKFLKELTLSMDELEENFRGMLKS\n",
      "                     AATKVTTVLATKTAEYSEMVSLFETPRKTREPFILEENEIYGFDLASSHLDTVKLILN\n",
      "                     INIESPVVSIPRKPGSPELLVGHLGQIFIQNFVAGDDESRSDRLQVEIKDIKLYSLNC\n",
      "                     TQLAGREAVGSEGSRMFCPPSGSGSANSQEEAHFTRHDFFESLHRGQAFHILNNTTIQ\n",
      "                     FKLEKIPIERESELTFSLSPDDLGTSSIMKIEGKFVNPVQVVLAKHVYEQVLQTLDNL\n",
      "                     VYSEDLNKYPASATSSPCPDSPLPPLSTCGESSVERKENGLFSHSSLSNTSQKSLSVK\n",
      "                     EVKSFTQIQATFCISELQVQLSGDLTLGAQGLVSLKFQDFEVEFSKDHPQTLSIQIAL\n",
      "                     HSLLMEDLLEKNPDSKYKNLMVSRGAPKPSSLAQKEYLSQSCPSVSNVEYPDMPRSLP\n",
      "                     SHMEEAPNVFQLYQRPTSASRKKQKEVQDKDYPLTPPPSPTVDEPKILVGKSKFDDSL\n",
      "                     VHINIFLVDKKHPEFSSSYNRVNRSIDVDFNCLDVLITLQTWVVILDFFGIGSTADNH\n",
      "                     AMRLPPEGILHNVKLEPHASMESGLQDPVNTKLDLKVHSLSLVLNKTTSELAKANVSK\n",
      "                     LVAHLEMIEGDLALQGSIGSLSLSDLTCHGEFYRERFTTSGEEALIFQTFKYGRPDPL\n",
      "                     LRREHDIRVSLRMASVQYVHTQRFQAEVVAFIQHFTQLQDVLGRQRAAIEGQTVRDQA\n",
      "                     QRCSRVLLDIEAGAPVLLIPESSRSNNLIVANLGKLKVKNKFLFAGFPGTFSLQDKES\n",
      "                     VPSASPTGIPKHSLRKTTSTEEPRGTHSQGQFTMPLAGMSLGSLKSEFVPSTSTKQQG\n",
      "                     PQPTLSVGQESSSPEDHVCLLDCVVVDLQDMDIFAAERHPREYSKAPEDSSGDLIFPS\n",
      "                     YFVRQTGGSLLTEPCRLKLQVERNLDKEISHTVPDISIHGNLSSVHCSLDLYKYKLIR\n",
      "                     GLLENNLGEPIEEFMRPYDLQDPRIHTVLSGEVYTCMCFLIDMVNVSLELKDPKRKEG\n",
      "                     AGSLARFDFKKCKLLYESFSNQTKSINLVSHSMMAFDTRYAGQKTSPGMTNVFSCIFQ\n",
      "                     PAKNSSTTQGSIQIELHFRSTKDSSCFTVVLNNLRVFLIFDWLLLVHDFLHTPSDIKK\n",
      "                     QNHVTPSRHRNSSSESAIVPKTVKSGVVTKRSSLPVSNERHLEVKVNVTGTEFVVIED\n",
      "                     VSCFDTNAIILKGTTVLTYKPRFVDRPFSGSLFGIEVFSCRLGNEHDTALSIVDPVQI\n",
      "                     QMELVGNSSYQNSSGLMDAFNSEDFPPVLEIQLQALDIRLSYNDVQLFLAIAKSIPEQ\n",
      "                     ANAAVPDSVALESDSVGTYLPGASRVGEEIREGTRHTLDPVLELQLARLQELGFSMDD\n",
      "                     CRKALLACQGQLKKAASWLFKNAEPLKSLSLASTSRDSPGAVAAPLISGVEIKAESVC\n",
      "                     ICFIDDCMDCDVPLAELTFSRLNFLQRVRTSPEGYAHFTLSGDYYNRALSGWEPFIEP\n",
      "                     WPCSVSWQQQAASRLHPPRLKLEAKAKPRLDINITSVLIDQYVSTKESWMADYCKDDK\n",
      "                     DIESAKSEDWMGSSVDPPCFGQTEVKTPKRRQPFVPFALRNHTGCTLWFATLTTTPTR\n",
      "                     AALSHSGSPGVVPEGNGTFLDDTHNVSEWREVLTGEEIPFEFEARGKLRHRHTHDLRI\n",
      "                     HQLQVRVNGWEQVSPVSVDKVGTFFRYAAPDKNSSSSTIGSPSSRTNIIHPQVYFSSL\n",
      "                     PPVRVVFAVTMEGSARKVITVRSALIVRNRLETPMELRLDSPSAPDKPVVLPAIMPGD\n",
      "                     SFAVPLHLTSWRLQARPKGLGVFFCKAPIHWTNVVKTAEISSSKRECHSMDTEKSRFF\n",
      "                     RFCVAIKKENYPDYMPSNIFSDSAKQIFRQPGHTIYLLPTVVICNLLPCELDFYVKGM\n",
      "                     PINGTLKPGKEAALHTADTSQNIELGVSLENFPLCKELLIPPGTQNYMVRMRLYDVNR\n",
      "                     RQLNLTIRIVCRAEGSLKIFISAPYWLINKTGLPLIFRQDNAKTDAAGQFEEHELARS\n",
      "                     LSPLLFCYADKEQPNLCTMRIGRGIHPEGMPGWCQGFSLDGGSGVRALKVIQQGNRPG\n",
      "                     LIYNIGIDVKKGRGRYIDTCMVIFAPRYLLDNKSSHKLAFAQREFARGQGTANPEGYI\n",
      "                     STLPGSSVVFHWPRNDYDQLLCVRLMDVPNCIWSGGFEVNKNNSFHINMRDTLGKCFF\n",
      "                     LRVEITLRGATYRISFSDTDQLPPPFRIDNFSKVPVVFTQHGVAEPRLRTEVKPMTSL\n",
      "                     DYAWDEPTLPPFITLTVKGAGSSEINCNMNDFQDNRQLYYENFIYIAATYTFSGLQEG\n",
      "                     TGRPVASNKAITCAELVLDVSPKTQRVILKKKEPGKRSQLWRMTGTGMLAHEGSSVPH\n",
      "                     NPNKPSAARSTEGSAILDIAGLAAVTDNRYEPLMLRKPDRRRSTTQTWSFREGKLTCG\n",
      "                     LHGLVVQAKGGLSGLFDGAEVVLGPDTSMELLGPVPPEQQFINQKMRPGSGMLSIRVI\n",
      "                     PDGPTRALQITDFCHRKSSRSYEVDELPVTEQELQKLKNPDTEQELEVLVRLEGGIGL\n",
      "                     SLINKVPEELVFASLTGINVHYTQLATSHMLELSIQDVQVDNQLIGTTQPFMLYVTPL\n",
      "                     SNENEVIETGPAVQVNAVKFPSKSALTNIYKHLMITAQRFTVQIEEKLLLKLLSFFGY\n",
      "                     DQAESEVEKYDENLHEKTAEQGGTPIRYYFENLKISIPQIKLSVFTSNKLPLDLKALK\n",
      "                     STLGFPLIRFEDAVINLDPFTRVHPYETKEFIINDILKHFQEELLSQAARILGSVDFL\n",
      "                     GNPMGLLNDVSEGVTGLIKYGNVGGLIRNVTHGVSNSAAKFAGTLSDGLGKTMDNRHQ\n",
      "                     SEREYIRYHAATSGEHLVAGIHGLAHGIIGGLTSVITSTVEGVKTEGGVSGFISGLGK\n",
      "                     GLVGTVTKPVAGALDFASETAQAVRDTATLSGPRTQAQRVRKPRCCTGPQGLLPRYSE\n",
      "                     SQAEGQEQLFKLTDNIQDEFFIAVENIDSYCVLISSKAVYFLKSGDYVDREAIFLEVK\n",
      "                     YDDLYHCLVSKDHGKVYVQVTKKAVSTSSGVSIPGPSHQKPMVHVKSEVLAVKLSQEI\n",
      "                     NYAKSLYYEQQLMLRLSENREQLELDS\"\n",
      "     misc_feature    2154..2156\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"Phosphoserine. {ECO:0000244|PubMed:23186163,\n",
      "                     ECO:0000244|PubMed:24275569}; propagated from\n",
      "                     UniProtKB/Swiss-Prot (Q5THJ4.2); phosphorylation site\"\n",
      "     misc_feature    3267..3269\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"Phosphoserine. {ECO:0000244|PubMed:23186163};\n",
      "                     propagated from UniProtKB/Swiss-Prot (Q5THJ4.2);\n",
      "                     phosphorylation site\"\n",
      "     misc_feature    3279..3281\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"Phosphoserine. {ECO:0000244|PubMed:23186163};\n",
      "                     propagated from UniProtKB/Swiss-Prot (Q5THJ4.2);\n",
      "                     phosphorylation site\"\n",
      "     misc_feature    3291..3293\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"Phosphoserine. {ECO:0000244|PubMed:20068231,\n",
      "                     ECO:0000244|PubMed:23186163}; propagated from\n",
      "                     UniProtKB/Swiss-Prot (Q5THJ4.2); phosphorylation site\"\n",
      "     misc_feature    3579..3581\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"Phosphoserine. {ECO:0000244|PubMed:23186163,\n",
      "                     ECO:0000244|PubMed:24275569}; propagated from\n",
      "                     UniProtKB/Swiss-Prot (Q5THJ4.2); phosphorylation site\"\n",
      "     misc_feature    4188..4190\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"Phosphoserine. {ECO:0000244|PubMed:24275569};\n",
      "                     propagated from UniProtKB/Swiss-Prot (Q5THJ4.2);\n",
      "                     phosphorylation site\"\n",
      "     misc_feature    4959..4961\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"Phosphoserine. {ECO:0000244|PubMed:23186163};\n",
      "                     propagated from UniProtKB/Swiss-Prot (Q5THJ4.2);\n",
      "                     phosphorylation site\"\n",
      "     misc_feature    4974..4976\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"Phosphoserine. {ECO:0000244|PubMed:23186163};\n",
      "                     propagated from UniProtKB/Swiss-Prot (Q5THJ4.2);\n",
      "                     phosphorylation site\"\n",
      "     misc_feature    5262..5264\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"Phosphoserine. {ECO:0000244|PubMed:23186163};\n",
      "                     propagated from UniProtKB/Swiss-Prot (Q5THJ4.2);\n",
      "                     phosphorylation site\"\n",
      "     misc_feature    5448..5450\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"Phosphothreonine. {ECO:0000244|PubMed:18669648,\n",
      "                     ECO:0000244|PubMed:19690332, ECO:0000244|PubMed:23186163,\n",
      "                     ECO:0000244|PubMed:24275569}; propagated from\n",
      "                     UniProtKB/Swiss-Prot (Q5THJ4.2); phosphorylation site\"\n",
      "     misc_feature    5460..5462\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"Phosphoserine. {ECO:0000244|PubMed:24275569};\n",
      "                     propagated from UniProtKB/Swiss-Prot (Q5THJ4.2);\n",
      "                     phosphorylation site\"\n",
      "     misc_feature    7470..7472\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"Phosphoserine. {ECO:0000244|PubMed:19690332,\n",
      "                     ECO:0000244|PubMed:21406692}; propagated from\n",
      "                     UniProtKB/Swiss-Prot (Q5THJ4.2); phosphorylation site\"\n",
      "     misc_feature    8178..8180\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"Phosphoserine. {ECO:0000244|PubMed:23186163,\n",
      "                     ECO:0000244|PubMed:24275569}; propagated from\n",
      "                     UniProtKB/Swiss-Prot (Q5THJ4.2); phosphorylation site\"\n",
      "     misc_feature    9039..9041\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"Phosphoserine. {ECO:0000244|PubMed:23186163};\n",
      "                     propagated from UniProtKB/Swiss-Prot (Q5THJ4.2);\n",
      "                     phosphorylation site\"\n",
      "     misc_feature    10662..10664\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"N6-acetyllysine. {ECO:0000244|PubMed:19608861};\n",
      "                     propagated from UniProtKB/Swiss-Prot (Q5THJ4.2);\n",
      "                     acetylation site\"\n",
      "     exon            265..342\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            343..533\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            534..614\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            615..731\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            732..836\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            837..1007\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            1008..1108\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            1109..1277\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            1278..1379\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            1380..1581\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            1582..1761\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            1762..1892\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            1893..1968\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            1969..2139\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            2140..2270\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            2271..2403\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            2404..4617\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            4618..4769\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            4770..5801\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            5802..5892\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            5893..6019\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            6020..6200\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            6201..6383\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            6384..6606\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            6607..6817\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            6818..6989\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            6990..7102\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            7103..7315\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            7316..7581\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            7582..7715\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            7716..7871\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            7872..8082\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            8083..8157\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            8158..8364\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            8365..8454\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            8455..8595\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            8596..8718\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            8719..8824\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            8825..8977\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            8978..9113\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            9114..9161\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            9162..9312\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            9313..9523\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            9524..9771\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            9772..9963\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            9964..10090\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            10091..10233\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            10234..10364\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            10365..10540\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            10541..10664\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            10665..10900\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            10901..11009\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            11010..11173\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            11174..11282\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            11283..11462\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            11463..11576\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            11577..11726\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            11727..11876\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            11877..11973\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            11974..12122\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            12123..12257\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            12258..12425\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            12426..12558\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            12559..12754\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            12755..12886\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            12887..13127\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            13128..16282\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "ORIGIN      \n",
      "        1 ctgagcgccg cgggcctgcg ccattgagga gcggcgggga ggaaacgccg cgcagcgccg\n",
      "       61 ggctggggcg ggcggcccgg gacaccgaca gatttttctg tgaccatgaa agagagaaat\n",
      "      121 aaagaatgat ccatgatttc taaacacctt ttcctgagga tatagtcatg ttggaaggcc\n",
      "      181 ttgtagcctg ggttctcaat acctatttgg gaaaatatgt caataacctg aacactgacc\n",
      "      241 agctctcagt tgcacttctc aaaggtgctg ttgaattaga aaacttgcca ttaaagaaag\n",
      "      301 atgccttgaa agaattggaa ttaccatttg aagtcaaagc tggcttcatt gggaaagtaa\n",
      "      361 cccttcagat tcccttttat cgcccccatg tggacccttg ggtgatctcc atctccagcc\n",
      "      421 ttcacttaat tggagcccca gagaaaatac aggatttcaa tgatgaaaag gagaagctgt\n",
      "      481 tggaaaggga acgtaagaaa gcactacttc aagccctgga ggagaaatgg aagaatgacc\n",
      "      541 gccagcagaa aggggagtcc tattggtatt cagttaccgc ctccgtagtt acaaggattg\n",
      "      601 tggagaatat tgaattaaaa attcaagatg tccatttacg ctttgaagat ggtgtcacca\n",
      "      661 atccctccca tccttttgct tttggcatct gcattaagaa tgtgtccatg caaaatgctg\n",
      "      721 tgaatgagcc tgtacagaaa ctaatgcgga aaaagcaatt agacgtagca gaatttagca\n",
      "      781 tctattggga tgtcgattgc actttactgg gggatttgcc tcagatggag ttacaggagg\n",
      "      841 ccatggccag gagcatggag agtcgcagcc atcactacgt cctggagcct gtgtttgcat\n",
      "      901 ctgctctttt gaagagaaac tgctccaaga agcccctgcg gtctcggcac agtccccgta\n",
      "      961 ttgattgtga tattcagctg gagaccattc ccttgaaact ctctcagctg caataccggc\n",
      "     1021 aaatcatgga attcctcaag gagctggaac gaaaggagag gcaggtgaag ttccgaaggt\n",
      "     1081 ggaaacccaa ggtggcgata tctaagaact gccgagaatg gtggtatttt gctttgaatg\n",
      "     1141 ctaacttgta tgagatcaga gagcagagga aacgttgcac ctgggacttt atgttgcacc\n",
      "     1201 gcgctcgtga tgctgtatct tacactgaca aatatttcaa caagttaaaa ggaggcctgc\n",
      "     1261 tgtccacaga tgacaaggag gaaatgtgtc ggattgaaga ggaacagagc tttgaggaat\n",
      "     1321 tgaagatttt gcgtgaactg gttcatgatc gatttcacaa acaggaagaa ctagcagaga\n",
      "     1381 gtctgcggga gcctcagttt gattctccag gagcctgtcc gggagcccca gaacccggtg\n",
      "     1441 gaggcagtgg gatgctgcag tatctccagt cctggtttcc tggatggggt ggctggtacg\n",
      "     1501 ggcagcagac cccagaaggg aatgtggttg agggactgtc agcagagcaa caggagcagt\n",
      "     1561 ggattcctga agagatcctg ggcactgagg agttttttga ccccactgca gatgcctcgt\n",
      "     1621 gtatgaacac gtatacaaag cgagatcatg tctttgccaa actgaatttg cagttgcagc\n",
      "     1681 gaggtacagt gactctgtta cacaaggagc aaggaactcc tcaaatgaat gaaagtgctt\n",
      "     1741 tcatgcagct cgagttttca gatgtaaaac ttctagcaga gtctcttcct cgaagaaatt\n",
      "     1801 cctcgttgct ttcagtccgg ttgggtggac tgtttcttcg agacctggct acagaaggaa\n",
      "     1861 ctatgtttcc tcttctagtc ttccctaatc cacaaaaaga agttggcaga gtctcacaat\n",
      "     1921 cttttggtct acaaactaca tctgcagaca gaagtgatca ttacccagct gcagatccag\n",
      "     1981 atggccccgt ttttgagatg ctgtatgaga gaaatccggc gcacagccac tttgagaggc\n",
      "     2041 ggctcaatgt cagcacaagg cccttgaaca tcatatacaa tccgcaggcc attaaaaaag\n",
      "     2101 tagcagactt tttctacaag ggaaaggttc atacctcagg ttttggttat cagtctgaac\n",
      "     2161 ttgagctgag agtggctgaa gctgcccgaa gacaatataa caagctgaag atgcagacca\n",
      "     2221 aggcagaaat ccggcaaact cttgatcgtt tgctagtggg tgatttcatt gaggagagta\n",
      "     2281 aacgatggac cgtgcggctg gatatttctg cccctcaggt gatatttcct gatgatttca\n",
      "     2341 aattcaagaa tcctgtgtta gttgtcgtgg atctaggaag aatgcttttg acgaacaccc\n",
      "     2401 aagataactc caggaggaaa agtagggatg ggtcagcatc tgaagagacc cagtttagtg\n",
      "     2461 atgatgaata taagaccccc ctggccacac ctcctaacac cccacctccc gagtcaagca\n",
      "     2521 gcagcaacgg agagaaaaca cctccctttt ctggagttga gttcagtgaa gaacagcttc\n",
      "     2581 aagcacattt aatgagcaca aagatgtatg agaggtactc gctgtcattt atggacctcc\n",
      "     2641 agatcatggt tggacgagtg aaagacaatt ggaagcatgt ccaggatatt gacgtgggac\n",
      "     2701 caacacatgt ggtagagaag ttcaacgttc acctacagtt agagcgtcga ttgatttata\n",
      "     2761 cttcagatcc caaatatcca ggagccgtgc tctcaggcaa cttaccagac ttaaaaatcc\n",
      "     2821 acattaatga agataaaata tctgcactaa agaattgctt tgctctcctc accaccccag\n",
      "     2881 aaatgaaaac ttctgacact cagattaaag aaaagatttt tccccaggag gagcagcggg\n",
      "     2941 gaagtttgca agactccgta atgaatttaa cccagagcat tgtgttgttg gagcagcata\n",
      "     3001 cccgcgaggt tctggtggag tcgcagctcc tcctggcgga atttaaagtg aactgtatgc\n",
      "     3061 agcttggtgt tgagagcaat ggccggtaca tttctgtgct caaggtgttt ggtaccaatg\n",
      "     3121 ctcactttgt gaagaggcct tatgatgctg aagtctccct aactgttcat ggtttgctcc\n",
      "     3181 tggtggatac catgcagaca tatggtgctg attttgacct tttgatggct tcacataaga\n",
      "     3241 acttgagctt tgatattcca acgggaagcc ttcgggatag cagggcccag tctcctgtct\n",
      "     3301 ctggaccgaa tgtggcccac ttaactgatg gagctacact gaacgaccga tcagctacta\n",
      "     3361 gtgtttcact tgacaaaatt cttaccaaag agcaagagtc ccttattaag ttggaatatc\n",
      "     3421 agtttgtgag ttcagagtgc ccatcgatga atttagacag tactcttcag gtgatttccc\n",
      "     3481 tacaggtgaa taatttagat attatcctca atccagagac gattgtggag ctaattggtt\n",
      "     3541 ttcttcaaaa atcctttccc aaggaaaaag atgatttaag tcctcaacct ttaatgactg\n",
      "     3601 attttgaaag aagcttcaga gaacaaggaa cttaccagtc tacatatgaa caaaacactg\n",
      "     3661 aggttgcagt ggaaatccat aggctgaact tactgcttct tcggacagtg ggcatggcaa\n",
      "     3721 atagagagaa atatggcaga aaaattgcaa ctgcaagtat aggtggcacc aaagttaatg\n",
      "     3781 tctcaatggg tagcacgttt gacatgaatg gttctcttgg ctgtttacag cttatggatt\n",
      "     3841 tgacacaaga taacgttaaa aaccagtatg ttgtcagcat tgggaattct gtaggctatg\n",
      "     3901 aaaatatcat cagtgatatt ggctactttg aatctgtgtt tgtcagaatg gaagatgcag\n",
      "     3961 ccctcactga agctttgagt ttcacgtttg ttgagagatc taaacaggag tgttttctca\n",
      "     4021 acctgaagat ggcttcttta cattataacc actctgctaa gtttttgaag gagttgacgt\n",
      "     4081 tatccatgga tgaactggaa gaaaattttc gaggtatgct gaaaagcgca gccaccaaag\n",
      "     4141 tcaccacagt actagctacc aagactgccg agtatagcga gatggtatcg ctctttgaaa\n",
      "     4201 ctccaaggaa gactcgggaa ccctttatct tagaggaaaa tgaaatatat gggtttgacc\n",
      "     4261 tagcttcgtc tcatttggac actgtaaagc taatcttgaa cataaacatt gaatcaccag\n",
      "     4321 ttgtttctat ccctcggaag ccggggagtc ctgagttgtt ggtgggacac ttgggacaga\n",
      "     4381 tattcatcca gaattttgtg gcgggagatg atgaatccag aagtgaccgt ctgcaggtgg\n",
      "     4441 aaatcaagga cattaaactg tattctttga attgcaccca gttggcaggt agagaagctg\n",
      "     4501 ttgggtctga aggaagccgg atgttttgcc caccttccgg gtctggcagt gccaacagtc\n",
      "     4561 aggaggaagc tcatttcaca cgacatgatt tctttgaatc tttgcataga ggtcaagctt\n",
      "     4621 ttcacatcct gaacaacacc accattcagt ttaaactgga gaagatccct atagagagag\n",
      "     4681 aatctgaatt gactttttct cttagcccag atgacctggg aacttctagc atcatgaaga\n",
      "     4741 ttgaaggaaa atttgtcaat ccagttcagg tggtgttagc aaagcatgta tatgagcagg\n",
      "     4801 ttttacaaac cctggacaat ctcgtgtaca gtgaagatct gaataagtat ccagccagtg\n",
      "     4861 ctacctcctc cccttgccct gattctcctc tgcctcccct cagtacctgt ggagaatctt\n",
      "     4921 ctgttgaaag gaaggagaat ggattgttca gccactccag cctttctaac acctctcaga\n",
      "     4981 agtcattgtc agtgaaggaa gtcaaatcct ttactcagat tcaagccacc ttttgtatat\n",
      "     5041 cagagcttca ggttcagcta agtggagatc tgactttggg ggcccaaggt cttgtgagct\n",
      "     5101 taaagtttca ggactttgag gtggaattca gtaaagacca tccccagact ttatctattc\n",
      "     5161 agattgccct gcattctctg ctgatggagg acttattgga gaagaatcca gattctaaat\n",
      "     5221 ataagaacct gatggtgtct cgaggagccc ctaagccatc tagtttagca caaaaagaat\n",
      "     5281 acctttctca gtcttgcccc tcagtgtcca atgtggaata tcctgatatg cctcggtctc\n",
      "     5341 tcccttccca catggaagaa gctcctaatg tcttccagtt gtatcaaagg cccacctctg\n",
      "     5401 cgtcccggaa aaagcaaaag gaagtccaag acaaggacta tcccttgacc ccacctcctt\n",
      "     5461 ctccaacagt ggatgagccc aagatacttg ttggaaagag taaatttgat gattccttag\n",
      "     5521 tccacatcaa catattcttg gtagataaga aacatccaga attctcttcc agttacaatc\n",
      "     5581 gagttaaccg gagcattgat gttgatttta attgcttgga tgtgctgatc acactgcaaa\n",
      "     5641 cctgggttgt gatattagac ttttttggaa tcggctccac tgcagacaac cacgcaatga\n",
      "     5701 ggctgcctcc tgagggcatt ctgcacaacg tgaagttgga gccacatgcc tccatggagt\n",
      "     5761 ctggacttca ggatccagtg aacaccaaac tggatctcaa ggttcattca ctttctctag\n",
      "     5821 tgctgaataa gaccaccagt gagcttgcca aagcaaatgt gtccaaatta gtagcacacc\n",
      "     5881 tggaaatgat tgagggagac ctggccttac agggcagcat tgggagtctg tctctaagtg\n",
      "     5941 acctcacatg ccatggagag ttctacagag aacggttcac taccagtggt gaagaagcac\n",
      "     6001 tcatcttcca gacttttaaa tacggacggc ctgaccctct gctccggaga gaacacgaca\n",
      "     6061 ttcgcgtgag cctccggatg gcctctgtgc agtatgtgca tactcagcgt ttccaggcag\n",
      "     6121 aggtggtggc cttcattcag catttcactc agctgcagga tgtcttaggg cgccagcgag\n",
      "     6181 ctgctattga ggggcagacg gtgagagatc aagcccagcg ctgttcacgg gttctcctgg\n",
      "     6241 atattgaggc tggtgctccc gttctcttga tcccagaaag ttccagatca aataatctga\n",
      "     6301 ttgtagcaaa tttggggaag ttgaaagtca aaaataagtt tctgtttgct ggttttcctg\n",
      "     6361 gcaccttttc cctacaagat aaggaatctg tgccttcagc ttccccaacg ggtattccca\n",
      "     6421 aacacagtct gaggaaaacg acaagcacgg aggagcccag gggaacccat tcccaggggc\n",
      "     6481 agttcacgat gcctcttgct ggaatgagcc taggaagcct gaagagtgag tttgtgccca\n",
      "     6541 gtacctccac caagcagcaa gggccgcaac ccacactgtc tgttggccaa gagtccagta\n",
      "     6601 gtccagaaga ccatgtctgc ctgctggatt gcgttgtcgt ggatctccag gacatggaca\n",
      "     6661 tctttgctgc agagagacat ccgagagaat actcgaaggc accagaggat agtagtggag\n",
      "     6721 atctgatctt cccttcctat tttgtgcgac agacaggagg aagcctctta accgagcctt\n",
      "     6781 gtaggctgaa attgcaggtg gaaaggaatt tggacaaaga aataagtcat actgtgccag\n",
      "     6841 acatatctat ccatggcaat ctctcctcag tccactgctc tctggatctg tataaataca\n",
      "     6901 agctgatccg cggcttatta gagaacaacc tgggagaacc catagaggaa tttatgcggc\n",
      "     6961 cttatgattt acaagatcca agaattcata ctgtcctgag tggagaagtg tacacctgta\n",
      "     7021 tgtgcttcct cattgatatg gtgaatgtaa gtctggagct taaagatcca aaaagaaaag\n",
      "     7081 aaggtgctgg gtccctagcc agatttgact tcaagaaatg caaactgctc tatgaaagtt\n",
      "     7141 tttccaacca aaccaagtcc attaacttgg tttcccattc catgatggct tttgacaccc\n",
      "     7201 gttatgctgg gcagaagacc agccctggca tgacgaatgt gttcagctgt atctttcagc\n",
      "     7261 ccgctaagaa cagcagcacc acccaagggt ccattcagat tgaactacat ttcagatcta\n",
      "     7321 ccaaggattc ctcctgcttt acagtagttc tcaacaatct ccgtgtgttt ctcatatttg\n",
      "     7381 actggctact gttagtccat gattttctcc acactcccag tgatattaag aaacaaaatc\n",
      "     7441 atgttactcc ttctcgccac cgtaactcta gcagcgaatc tgctatagtt cccaaaactg\n",
      "     7501 tgaagagtgg agtagttacc aagcggtctt cccttcctgt gtccaatgaa aggcacctgg\n",
      "     7561 aggtcaaggt caatgtaaca ggtacggagt ttgtggtcat tgaagatgtg tcctgcttcg\n",
      "     7621 acaccaatgc cattattctg aaaggcacca cagtgctcac ctataagccc cggtttgttg\n",
      "     7681 atcgcccctt ttcaggaagt ttgtttggca ttgaggtgtt ttcatgccga ctagggaatg\n",
      "     7741 agcatgatac agctctttca attgtggatc ccgtacaaat tcaaatggag ttggtgggga\n",
      "     7801 attcttctta tcaaaatagt tcaggattga tggatgcatt caatagtgaa gatttcccac\n",
      "     7861 ctgtcctgga gattcagtta caagccctgg atatcagact ctcctataat gatgttcagc\n",
      "     7921 tgtttcttgc cattgcaaaa tccatcccag agcaagctaa tgctgcagtg ccagactcag\n",
      "     7981 tggccctgga gtcagactcc gttggcactt accttccagg tgcatctcgc gttggagagg\n",
      "     8041 aaatcagaga agggacaaga cacaccttag atcctgtctt ggagttacag ctggctaggc\n",
      "     8101 tgcaggagct gggattcagc atggatgatt gtcgcaaagc tcttttggcg tgtcaaggcc\n",
      "     8161 aattgaaaaa ggcagcaagt tggttgttta agaatgcgga acctctgaag tctctttcct\n",
      "     8221 tggcctccac cagccgagat agcccagggg ctgtggcagc gccattgatc tctggcgtgg\n",
      "     8281 agatcaaagc tgagagtgtg tgcatctgtt tcatcgatga ctgcatggat tgtgatgttc\n",
      "     8341 ctctcgctga actcaccttt tcccgtctga attttcttca gcgtgtaaga actagccctg\n",
      "     8401 aaggctatgc ccacttcacc ctttctggag attattataa ccgtgctctt tcaggctggg\n",
      "     8461 agccatttat tgagccttgg ccatgctctg tatcctggca acagcaggca gctagtcgtc\n",
      "     8521 tccatcctcc tcgactgaag ctagaagcca aggccaaacc tcgtttggat atcaatatca\n",
      "     8581 cttctgtgct aattgaccag tatgtaagta ccaaggaatc gtggatggca gattactgta\n",
      "     8641 aagatgacaa ggacatagag tcagctaaat cagaagactg gatgggctct tcggtggatc\n",
      "     8701 ctccatgttt tggacaaaca gaggtgaaaa cccccaagcg ccggcagcca tttgtcccct\n",
      "     8761 ttgctctgag gaaccacacg gggtgcactt tgtggtttgc caccctgacc accacaccca\n",
      "     8821 ccagagctgc actctctcac agtgggagtc caggggtagt tccagaaggg aacggaacat\n",
      "     8881 ttctcgatga tactcacaat gttagtgaat ggcgagaagt ccttacaggt gaagagattc\n",
      "     8941 cctttgaatt tgaagcaaga ggaaagttaa gacacagaca cacccatgac ctccggattc\n",
      "     9001 atcaactgca agtgagagta aatggctggg agcaagtgag cccagtgtct gtggacaaag\n",
      "     9061 tcgggacctt ttttcgatat gcagcaccag ataaaaattc atcttcctct acgattggca\n",
      "     9121 gcccaagcag cagaacaaat attatacatc cccaggttta tttctcttca ctcccaccag\n",
      "     9181 tgcgggtggt ctttgcagtg actatggaag gcagtgcacg gaaagtcatc actgtccggt\n",
      "     9241 cagccctcat tgtgaggaac agacttgaga caccaatgga actaagactg gatagcccat\n",
      "     9301 cagctccaga caagccagtg gtgcttcctg ctatcatgcc aggggattcg tttgctgtgc\n",
      "     9361 ctttacacct cacttcttgg cggctacagg cccggcccaa aggattgggt gtatttttct\n",
      "     9421 gtaaggctcc cattcattgg accaatgtag tgaagactgc agaaattagt agcagtaaac\n",
      "     9481 gagagtgcca ctctatggac acagaaaaaa gccgattttt caggttttgt gtggctataa\n",
      "     9541 agaaagagaa ttatccagat tatatgccct caaacatatt ttctgacagt gcaaaacaga\n",
      "     9601 ttttcagaca gcctgggcat accatatatc tcctgccaac tgtggtaatc tgcaacttgc\n",
      "     9661 taccctgtga acttgatttt tatgttaaag gaatgccaat taatgggacg ctgaaacctg\n",
      "     9721 gcaaggaggc agctctccat acagctgata catcccagaa cattgagctg ggggtatcac\n",
      "     9781 tggagaattt ccccctctgt aaagaattgc tcattccacc tggaacccaa aactatatgg\n",
      "     9841 tgagaatgcg actctatgac gtcaaccgtc ggcagctgaa cctcaccatc cggattgtgt\n",
      "     9901 gtcgagcaga aggatcctta aagatcttca tttctgctcc atattggctg attaacaaaa\n",
      "     9961 cagggttgcc actgatcttc agacaggaca atgccaagac agatgctgca ggccagtttg\n",
      "    10021 aggagcatga gctggcccgt agcctgagtc ctctcttatt ctgctatgct gacaaagagc\n",
      "    10081 agccaaacct ctgcacgatg agaatcggaa gggggattca tccagaaggc atgccgggct\n",
      "    10141 ggtgtcaggg cttctccctg gatggtggta gtggtgtccg agctttgaaa gtcatccagc\n",
      "    10201 aaggaaaccg cccagggctg atctataaca ttggtattga tgtcaagaaa ggccgaggtc\n",
      "    10261 gatacattga tacctgcatg gtcatctttg ccccccgtta cctgttagat aataaatcat\n",
      "    10321 ctcacaagct tgcatttgca cagagggaat ttgccagggg acagggaaca gccaatcccg\n",
      "    10381 aaggttacat ttccaccctt cctggttcca gtgtggtgtt ccactggcct cggaatgact\n",
      "    10441 atgatcagct attgtgtgtc agactgatgg acgttcccaa ttgtatttgg tctggaggct\n",
      "    10501 ttgaagtcaa caagaataat tccttccata tcaacatgag ggataccttg ggaaaatgct\n",
      "    10561 tcttcctacg agtggaaatt actctccgag gagctacgta taggatctca tttagtgaca\n",
      "    10621 cagatcagtt acctcctcct ttccgaattg acaacttttc taaggtcccg gttgtcttta\n",
      "    10681 ctcagcatgg cgtagctgaa cccaggctcc ggactgaagt gaagcccatg acttcattgg\n",
      "    10741 attatgcctg ggacgaaccc accttgccac cttttatcac tctgactgtt aaaggggcag\n",
      "    10801 ggtcctctga gatcaactgc aacatgaatg atttccagga taatcggcag ctttattatg\n",
      "    10861 aaaatttcat ttacattgct gctacatata cattctctgg cttgcaggag ggaacaggca\n",
      "    10921 ggcctgtggc ttccaacaag gccattacct gtgcggagct cgttttggat gtctcaccca\n",
      "    10981 agacacaaag agtcatttta aaaaagaagg aaccaggaaa gcgttctcag ctgtggagga\n",
      "    11041 tgacaggaac aggaatgctg gcccatgagg gctcctcagt tcctcacaat cccaataagc\n",
      "    11101 cctcagccgc ccgctccacc gaggggtctg ccatcttaga tattgctggt ctcgctgcag\n",
      "    11161 tgactgacaa cagatacgag ccactgatgc tgagaaagcc tgaccgcagg cgaagcacaa\n",
      "    11221 ctcagacgtg gagtttccga gaaggaaaac tgacctgtgg gttacatggg ttggtcgtcc\n",
      "    11281 aggccaaagg aggactttct ggtttgtttg atggagctga agttgttctt ggtcctgaca\n",
      "    11341 cttccatgga gcttttgggg ccagttccac ctgaacaaca atttattaat caaaaaatga\n",
      "    11401 gacctggttc tggaatgtta tccatcagag tcatcccaga tggaccaact agagcactcc\n",
      "    11461 agataacaga tttctgccac cggaaaagca gccgttcata tgaagtggat gaacttcctg\n",
      "    11521 tcaccgaaca agagctgcag aaattaaaga atccagatac agagcaggaa ttggaagtgc\n",
      "    11581 ttgtgaggtt agaaggtgga attgggttgt ccttaattaa taaagtccca gaagaactgg\n",
      "    11641 tctttgcaag tcttacagga atcaatgtgc actatacaca gctggcaacc agtcacatgc\n",
      "    11701 ttgaactcag catacaggat gtacaggtgg acaatcagct cattggtacc acgcagccct\n",
      "    11761 tcatgctcta tgtgactccc ctgagcaatg agaatgaggt catcgagacc ggcccagctg\n",
      "    11821 tgcaagtcaa cgcagtgaag ttccccagta agagtgcact gaccaacatc tacaagcatc\n",
      "    11881 tgatgatcac agctcagaga ttcacagtgc aaattgagga gaaactgctc ctcaagctgc\n",
      "    11941 taagtttctt tggctacgat caagcagaat cagaggtgga aaaatatgat gaaaacctcc\n",
      "    12001 atgaaaagac agctgagcaa ggtggaacac caattcgata ctactttgaa aatctcaaaa\n",
      "    12061 tcagcattcc tcagatcaag ctaagtgtgt tcacctccaa caagctccca ttggatctta\n",
      "    12121 aggccctaaa aagcaccttg gggtttcctt tgatacggtt tgaagacgct gtgattaatc\n",
      "    12181 tagatccatt cactcgggta catccctatg agaccaagga gttcatcatc aatgatatcc\n",
      "    12241 tcaaacattt ccaggaggaa ctcctcagcc aggcagctcg aatcctggga tcagtggatt\n",
      "    12301 ttcttggcaa tcctatgggg cttttgaatg atgtttctga aggggttact ggactgataa\n",
      "    12361 aatatggaaa tgtcgggggc ctcatcagaa atgttacaca cggagtatca aactctgctg\n",
      "    12421 ccaagtttgc tggaacatta tcagatggct tagggaagac gatggacaat cggcatcagt\n",
      "    12481 cagagcggga gtacatcagg taccatgcag ccacaagtgg tgaacacctt gtagccggca\n",
      "    12541 tccatggcct ggctcatggt atcattggtg gactgaccag tgttataact tcgacagtgg\n",
      "    12601 aaggtgtgaa aacagaaggg ggtgtcagcg gtttcatatc tggccttgga aaagggcttg\n",
      "    12661 ttggcactgt aaccaagcca gtggcaggcg ccctggattt tgcatcagaa acagcccagg\n",
      "    12721 cggtgagaga cacagccaca ctcagcggcc ccaggactca agcacagagg gttcggaaac\n",
      "    12781 cgcgttgctg cacggggccc caggggctgc ttccccgata ttctgagagc caggcggaag\n",
      "    12841 gacaggagca gctcttcaaa ctcacagaca acatacagga cgaattcttc atcgctgtgg\n",
      "    12901 agaacattga cagctactgc gtgctcatct cctccaaagc tgtttacttc ctgaaaagtg\n",
      "    12961 gagactacgt ggatcgagaa gccattttcc tagaagtcaa atacgatgac ctctaccact\n",
      "    13021 gccttgtctc caaagaccat gggaaggtgt atgtgcaggt gaccaagaaa gccgtgagca\n",
      "    13081 cgagcagtgg agtgtccatc cccggcccct cccaccagaa gcccatggtc catgtgaaat\n",
      "    13141 ctgaggtcct tgctgtcaag ttgtcacaag aaataaacta cgcaaagagc ctctactatg\n",
      "    13201 aacagcagct tatgttaaga ctcagcgaaa accgagagca gctggagctg gactcctgaa\n",
      "    13261 gccccgctgc tgagatgggc gctcccgaca cagcgcagac ccaccaggag gaaagaggcc\n",
      "    13321 cagctctcag ctgacgatgg aggcagaacc ggagtcgggt ttggggaagt tgtcaaggaa\n",
      "    13381 tgagggaaag taaatcctca tgaggaaaag tacaaatgga aatcgtatta atttgtgagg\n",
      "    13441 cagggagtta ttttagatta tgggaaataa tttttaaagg tattggttaa ataacgttta\n",
      "    13501 aaaacatgta ctgagatgaa tctaattttt agattgccct gtattttgtt aacatgtata\n",
      "    13561 tatgtacaac agtgtgtttg taaatatata ggaacgtttc tgaacagggt ctgtgctatg\n",
      "    13621 tgtaaaggtt tgttaactgt aaagtaatat aaagttatat tggatcttct attgcactaa\n",
      "    13681 ttctagatgt ctaattcagg atactgtcta tagaaaggca ttcttaaaag ttaaagaatg\n",
      "    13741 ttacgtctta gttttggaga ctaaagtatt cccagtaaag tgggttgagg tgagggctgt\n",
      "    13801 ggtcctgaaa gggacgcctt tgacatcgtg gctgtccagt tgggctgtga gctgtggcac\n",
      "    13861 ccaggactgg cgctggccct tcagaaggat ctaggagagg ggcttgggag cccactttta\n",
      "    13921 atttctcacc cccattttac aaagagtgct tagattctta caaattatga tgtaagttat\n",
      "    13981 ccatttggct ttttcctaac tagtcttacc aaacttaggg ggaaacctgt gctccattac\n",
      "    14041 cacatgggtg caagtcagca ttgtaagttt tctcaggtta ttattattag agaggttgga\n",
      "    14101 aacattggta aactctgttg attgagaagg aaaaaaaaag tcccattgaa ctgttgcaac\n",
      "    14161 aaatcagaaa tccacataaa agtgctctcc tgcctgggca gcaacaacca agaacaaagc\n",
      "    14221 cccgggactg ttttcttttt aataaagcca caggcaggca tcgtagctcc acagcccgag\n",
      "    14281 gggacacagg atggaaaccc caggatgaga agggagcagg gagagttcca gaaaggggga\n",
      "    14341 tgaaatagga gtattaaaaa gctgcgttgg taagtttttc atggaaccaa gatttgacaa\n",
      "    14401 aggcatctct tatccttggt tttaaattcc tgctgggagc aaggcctggt atgagcgccc\n",
      "    14461 tgggtcttgt ttttggtgtt tcgcttttct gtaaggatta agcagatagg gagaagggaa\n",
      "    14521 aaggggcctc actttagaat gaatgagtca ccttgtgatt tttaaatttt tattttaata\n",
      "    14581 aagctaatca atttctacaa ccttgtcaca tgtagctgag tctgggatga ctcagtggat\n",
      "    14641 cagtggatgc gtggaaggtt ttggtgttta taactcatga cccaaatcct tccaagacac\n",
      "    14701 aaatgaagct aatcttcttc ccctcccttt cctctcccac ttcccctctg tgtctgtgtc\n",
      "    14761 tgtgtgtgtg tgtgtgtgtg tgtgtgtgtg cgcgcgcgcg cgtgcatgca gagaggaagg\n",
      "    14821 aagggagctt tagcacaagt agctacatag tgttcaagga tcagctttct tgaactccga\n",
      "    14881 ggcttggttg cctttgctga actccaagtt agtggattgc agaatggaaa cttggctttt\n",
      "    14941 gcggcactgg gtgagtttta gtttgtgtgt gtcttgctgg ggggtggtga tgattgtctc\n",
      "    15001 agcactcacg cactgcacaa gatggcagca ggatacagca ctgcacaaga tggcagctcc\n",
      "    15061 tctgcagctt cctcctcagc ctccctcctt gcacccccac aggtttggct tgtggttttt\n",
      "    15121 gtcatcagta acctactgcc tgagatcatg atctcttaaa agatgagact ctcggaaggg\n",
      "    15181 ttgattgtat gcgtcagtga gccttctatc accttctgga acaaagtcac ttgaaatctc\n",
      "    15241 ttgatgagat taaggagttt agtgttacta agaaaatctg ctttgggccg cagcagtgct\n",
      "    15301 gggtgttctc agacctgact gaggaagtta gctgcgggct gccctgtggg ctggtgcttc\n",
      "    15361 aggaggaatc cagagaagtg ttcagatgcc ccccttgggc tcctttctaa ttttaatcag\n",
      "    15421 ctctttaaat agctgcccat ctcctgtgat tgcacaacca agcactttga catttgcacc\n",
      "    15481 ttaggagagg cagatgttaa aatggaatcc aaagaccacc tagggcgggg ctgggtggga\n",
      "    15541 gatgggaggg ccaactgcga gctgctccac ttctcagctc tcccctgccc tgcagccctg\n",
      "    15601 ggccagacaa ggccagaagg tttcaggggc atttgacatc ccctcctggt tctcaccagg\n",
      "    15661 aaaacatcca aagctttgga ggaaacaggc cctgcccctg gctccttaaa tgccccgtct\n",
      "    15721 ctttgtaaac tgatattcag ccagcaatgc ctaagacttt gttaagatca tttctactgc\n",
      "    15781 ttttctttct gcttcaaaca cacagttcgt ctctgaggaa agtaaaataa atggaataag\n",
      "    15841 agtaaattgg gtaaggagat atccaaagct acccagtccc ttgacccagc acagttggcc\n",
      "    15901 gacccgtgtc actccctggc tgtcgctgct tctctgtgct cactgaaggg tgagccaggc\n",
      "    15961 cagtgcttcc ccagcccctg ggcctggtca ctacacagtg gaaaacagac aagcggcccc\n",
      "    16021 ttccccaaat cccaagagtg tcttgctgct tggtgggtgc tcatcgcaat gttctgaagg\n",
      "    16081 ctccagggcc actttgtttg taagtatgat ctgggcctca aaataccata gtagctgctt\n",
      "    16141 gataaaattc taaaaatatc tggttctcta ttatgtaaac actattacag tcaccagtgt\n",
      "    16201 gtgaagactc ttgagtctgg ttctcatatc agagtcatca tttttcttcc tgtggaataa\n",
      "    16261 aatgccttgt ggacttccca aa\n",
      "//\n",
      "\n",
      "LOCUS       NM_015378              16357 bp    mRNA    linear   PRI 21-AUG-2019\n",
      "DEFINITION  Homo sapiens vacuolar protein sorting 13 homolog D (VPS13D),\n",
      "            transcript variant 1, mRNA.\n",
      "ACCESSION   NM_015378 XM_044546\n",
      "VERSION     NM_015378.4\n",
      "KEYWORDS    RefSeq; MANE Select.\n",
      "SOURCE      Homo sapiens (human)\n",
      "  ORGANISM  Homo sapiens\n",
      "            Eukaryota; Metazoa; Chordata; Craniata; Vertebrata; Euteleostomi;\n",
      "            Mammalia; Eutheria; Euarchontoglires; Primates; Haplorrhini;\n",
      "            Catarrhini; Hominidae; Homo.\n",
      "REFERENCE   1  (bases 1 to 16357)\n",
      "  AUTHORS   Seong E, Insolera R, Dulovic M, Kamsteeg EJ, Trinh J, Bruggemann N,\n",
      "            Sandford E, Li S, Ozel AB, Li JZ, Jewett T, Kievit AJA, Munchau A,\n",
      "            Shakkottai V, Klein C, Collins CA, Lohmann K, van de Warrenburg BP\n",
      "            and Burmeister M.\n",
      "  TITLE     Mutations in VPS13D lead to a new recessive ataxia with spasticity\n",
      "            and mitochondrial defects\n",
      "  JOURNAL   Ann. Neurol. 83 (6), 1075-1088 (2018)\n",
      "   PUBMED   29604224\n",
      "  REMARK    GeneRIF: heterozygous mutations in VPS13D cause movement disorders\n",
      "            along the ataxia-spasticity spectrum\n",
      "REFERENCE   2  (bases 1 to 16357)\n",
      "  AUTHORS   Gauthier J, Meijer IA, Lessel D, Mencacci NE, Krainc D, Hempel M,\n",
      "            Tsiakas K, Prokisch H, Rossignol E, Helm MH, Rodan LH,\n",
      "            Karamchandani J, Carecchio M, Lubbe SJ, Telegrafi A, Henderson LB,\n",
      "            Lorenzo K, Wallace SE, Glass IA, Hamdan FF, Michaud JL, Rouleau GA\n",
      "            and Campeau PM.\n",
      "  TITLE     Recessive mutations in >VPS13D cause childhood onset movement\n",
      "            disorders\n",
      "  JOURNAL   Ann. Neurol. 83 (6), 1089-1095 (2018)\n",
      "   PUBMED   29518281\n",
      "  REMARK    GeneRIF: findings underline the importance of the VPS13 complex in\n",
      "            neurological diseases and a possible role in mitochondrial function\n",
      "REFERENCE   3  (bases 1 to 16357)\n",
      "  AUTHORS   Anding AL, Wang C, Chang TK, Sliter DA, Powers CM, Hofmann K, Youle\n",
      "            RJ and Baehrecke EH.\n",
      "  TITLE     Vps13D Encodes a Ubiquitin-Binding Protein that Is Required for the\n",
      "            Regulation of Mitochondrial Size and Clearance\n",
      "  JOURNAL   Curr. Biol. 28 (2), 287-295 (2018)\n",
      "   PUBMED   29307555\n",
      "REFERENCE   4  (bases 1 to 16357)\n",
      "  AUTHORS   Nakada TA, Boyd JH, Russell JA, Aguirre-Hernandez R, Wilkinson MD,\n",
      "            Thair SA, Nakada E, McConechy MK, Fjell CD and Walley KR.\n",
      "  TITLE     VPS13D Gene Variant Is Associated with Altered IL-6 Production and\n",
      "            Mortality in Septic Shock\n",
      "  JOURNAL   J Innate Immun 7 (5), 545-553 (2015)\n",
      "   PUBMED   25896417\n",
      "  REMARK    GeneRIF: genetic polymorphism is associated with il-6 production\n",
      "            and mortality in patients with septic shock\n",
      "REFERENCE   5  (bases 1 to 16357)\n",
      "  AUTHORS   Koriyama H, Nakagami H, Katsuya T, Sugimoto K, Yamashita H, Takami\n",
      "            Y, Maeda S, Kubo M, Takahashi A, Nakamura Y, Ogihara T, Rakugi H,\n",
      "            Kaneda Y and Morishita R.\n",
      "  TITLE     Identification of evidence suggestive of an association with\n",
      "            peripheral arterial disease at the OSBPL10 locus by genome-wide\n",
      "            investigation in the Japanese population\n",
      "  JOURNAL   J. Atheroscler. Thromb. 17 (10), 1054-1062 (2010)\n",
      "   PUBMED   20610895\n",
      "  REMARK    GeneRIF: Observational study and genome-wide association study of\n",
      "            gene-disease association. (HuGE Navigator)\n",
      "REFERENCE   6  (bases 1 to 16357)\n",
      "  AUTHORS   Gonzales PA, Pisitkun T, Hoffert JD, Tchapyjnikov D, Star RA, Kleta\n",
      "            R, Wang NS and Knepper MA.\n",
      "  TITLE     Large-scale proteomics and phosphoproteomics of urinary exosomes\n",
      "  JOURNAL   J. Am. Soc. Nephrol. 20 (2), 363-379 (2009)\n",
      "   PUBMED   19056867\n",
      "REFERENCE   7  (bases 1 to 16357)\n",
      "  AUTHORS   Velayos-Baeza A, Vettori A, Copley RR, Dobson-Stone C and Monaco\n",
      "            AP.\n",
      "  TITLE     Analysis of the human VPS13 gene family\n",
      "  JOURNAL   Genomics 84 (3), 536-549 (2004)\n",
      "   PUBMED   15498460\n",
      "REFERENCE   8  (bases 1 to 16357)\n",
      "  AUTHORS   Nakayama M, Kikuno R and Ohara O.\n",
      "  TITLE     Protein-protein interactions between large proteins: two-hybrid\n",
      "            screening using a functionally classified library composed of long\n",
      "            cDNAs\n",
      "  JOURNAL   Genome Res. 12 (11), 1773-1784 (2002)\n",
      "   PUBMED   12421765\n",
      "REFERENCE   9  (bases 1 to 16357)\n",
      "  AUTHORS   Meijer,I.A.\n",
      "  TITLE     VPS13D Movement Disorder\n",
      "  JOURNAL   (in) Adam MP, Ardinger HH, Pagon RA, Wallace SE, Bean LJH, Stephens\n",
      "            K and Amemiya A (Eds.);\n",
      "            GENEREVIEWS((R));\n",
      "            (1993)\n",
      "   PUBMED   30789691\n",
      "REFERENCE   10 (bases 1 to 16357)\n",
      "  AUTHORS   Bird,T.D.\n",
      "  TITLE     Hereditary Ataxia Overview\n",
      "  JOURNAL   (in) Adam MP, Ardinger HH, Pagon RA, Wallace SE, Bean LJH, Stephens\n",
      "            K and Amemiya A (Eds.);\n",
      "            GENEREVIEWS((R));\n",
      "            (1993)\n",
      "   PUBMED   20301317\n",
      "COMMENT     REVIEWED REFSEQ: This record has been curated by NCBI staff. The\n",
      "            reference sequence was derived from AL031276.1, HY121734.1,\n",
      "            AJ608774.1, BX682532.4 and BC051804.1.\n",
      "            This sequence is a reference standard in the RefSeqGene project.\n",
      "            On May 11, 2018 this sequence version replaced NM_015378.3.\n",
      "            \n",
      "            Summary: This gene encodes a protein belonging to the\n",
      "            vacuolar-protein-sorting-13 gene family. In yeast,\n",
      "            vacuolar-protein-sorting-13 proteins are involved in trafficking of\n",
      "            membrane proteins between the trans-Golgi network and the\n",
      "            prevacuolar compartment. While several transcript variants may\n",
      "            exist for this gene, the full-length natures of only two have been\n",
      "            described to date. These two represent the major variants of this\n",
      "            gene and encode distinct isoforms. [provided by RefSeq, Jul 2008].\n",
      "            \n",
      "            Transcript Variant: This variant (1) represents the longer\n",
      "            transcript and encodes the longer isoform (1).\n",
      "            \n",
      "            ##Evidence-Data-START##\n",
      "            Transcript exon combination :: AJ608774.1 [ECO:0000332]\n",
      "            RNAseq introns              :: single sample supports all introns\n",
      "                                           SAMEA1968968, SAMEA1970526\n",
      "                                           [ECO:0000348]\n",
      "            ##Evidence-Data-END##\n",
      "            \n",
      "            ##RefSeq-Attributes-START##\n",
      "            MANE Ensembl match     :: ENST00000620676.6/ ENSP00000478104.1\n",
      "            RefSeq Select criteria :: based on conservation, longest protein\n",
      "            ##RefSeq-Attributes-END##\n",
      "            COMPLETENESS: complete on the 3' end.\n",
      "PRIMARY     REFSEQ_SPAN         PRIMARY_IDENTIFIER PRIMARY_SPAN        COMP\n",
      "            1-9                 AL031276.1         81577-81585         c\n",
      "            10-37               HY121734.1         1-28\n",
      "            38-8513             AJ608774.1         1-8476\n",
      "            8514-8514           BX682532.4         21312-21312\n",
      "            8515-8824           AJ608774.1         8478-8787\n",
      "            8825-8825           BX682532.4         29838-29838\n",
      "            8826-14671          AJ608774.1         8789-14634\n",
      "            14672-16357         BC051804.1         4065-5750\n",
      "FEATURES             Location/Qualifiers\n",
      "     source          1..16357\n",
      "                     /organism=\"Homo sapiens\"\n",
      "                     /mol_type=\"mRNA\"\n",
      "                     /db_xref=\"taxon:9606\"\n",
      "                     /chromosome=\"1\"\n",
      "                     /map=\"1p36.22-p36.21\"\n",
      "     gene            1..16357\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /note=\"vacuolar protein sorting 13 homolog D\"\n",
      "                     /db_xref=\"GeneID:55187\"\n",
      "                     /db_xref=\"HGNC:HGNC:23595\"\n",
      "                     /db_xref=\"MIM:608877\"\n",
      "     exon            1..91\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            92..264\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     misc_feature    141..143\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /note=\"upstream in-frame stop codon\"\n",
      "     CDS             168..13334\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /note=\"isoform 1 is encoded by transcript variant 1;\n",
      "                     vacuolar protein sorting-associated protein 13D; vacuolar\n",
      "                     protein sorting 13D\"\n",
      "                     /codon_start=1\n",
      "                     /product=\"vacuolar protein sorting-associated protein 13D\n",
      "                     isoform 1\"\n",
      "                     /protein_id=\"NP_056193.2\"\n",
      "                     /db_xref=\"CCDS:CCDS30588.1\"\n",
      "                     /db_xref=\"GeneID:55187\"\n",
      "                     /db_xref=\"HGNC:HGNC:23595\"\n",
      "                     /db_xref=\"MIM:608877\"\n",
      "                     /translation=\"MLEGLVAWVLNTYLGKYVNNLNTDQLSVALLKGAVELENLPLKK\n",
      "                     DALKELELPFEVKAGFIGKVTLQIPFYRPHVDPWVISISSLHLIGAPEKIQDFNDEKE\n",
      "                     KLLERERKKALLQALEEKWKNDRQQKGESYWYSVTASVVTRIVENIELKIQDVHLRFE\n",
      "                     DGVTNPSHPFAFGICIKNVSMQNAVNEPVQKLMRKKQLDVAEFSIYWDVDCTLLGDLP\n",
      "                     QMELQEAMARSMESRSHHYVLEPVFASALLKRNCSKKPLRSRHSPRIDCDIQLETIPL\n",
      "                     KLSQLQYRQIMEFLKELERKERQVKFRRWKPKVAISKNCREWWYFALNANLYEIREQR\n",
      "                     KRCTWDFMLHRARDAVSYTDKYFNKLKGGLLSTDDKEEMCRIEEEQSFEELKILRELV\n",
      "                     HDRFHKQEELAESLREPQFDSPGACPGAPEPGGGSGMLQYLQSWFPGWGGWYGQQTPE\n",
      "                     GNVVEGLSAEQQEQWIPEEILGTEEFFDPTADASCMNTYTKRDHVFAKLNLQLQRGTV\n",
      "                     TLLHKEQGTPQMNESAFMQLEFSDVKLLAESLPRRNSSLLSVRLGGLFLRDLATEGTM\n",
      "                     FPLLVFPNPQKEVGRVSQSFGLQTTSADRSDHYPAADPDGPVFEMLYERNPAHSHFER\n",
      "                     RLNVSTRPLNIIYNPQAIKKVADFFYKGKVHTSGFGYQSELELRVAEAARRQYNKLKM\n",
      "                     QTKAEIRQTLDRLLVGDFIEESKRWTVRLDISAPQVIFPDDFKFKNPVLVVVDLGRML\n",
      "                     LTNTQDNSRRKSRDGSASEETQFSDDEYKTPLATPPNTPPPESSSSNGEKTPPFSGVE\n",
      "                     FSEEQLQAHLMSTKMYERYSLSFMDLQIMVGRVKDNWKHVQDIDVGPTHVVEKFNVHL\n",
      "                     QLERRLIYTSDPKYPGAVLSGNLPDLKIHINEDKISALKNCFALLTTPEMKTSDTQIK\n",
      "                     EKIFPQEEQRGSLQDSVMNLTQSIVLLEQHTREVLVESQLLLAEFKVNCMQLGVESNG\n",
      "                     RYISVLKVFGTNAHFVKRPYDAEVSLTVHGLLLVDTMQTYGADFDLLMASHKNLSFDI\n",
      "                     PTGSLRDSRAQSPVSGPNVAHLTDGATLNDRSATSVSLDKILTKEQESLIKLEYQFVS\n",
      "                     SECPSMNLDSTLQVISLQVNNLDIILNPETIVELIGFLQKSFPKEKDDLSPQPLMTDF\n",
      "                     ERSFREQGTYQSTYEQNTEVAVEIHRLNLLLLRTVGMANREKYGRKIATASIGGTKVN\n",
      "                     VSMGSTFDMNGSLGCLQLMDLTQDNVKNQYVVSIGNSVGYENIISDIGYFESVFVRME\n",
      "                     DAALTEALSFTFVERSKQECFLNLKMASLHYNHSAKFLKELTLSMDELEENFRGMLKS\n",
      "                     AATKVTTVLATKTAEYSEMVSLFETPRKTREPFILEENEIYGFDLASSHLDTVKLILN\n",
      "                     INIESPVVSIPRKPGSPELLVGHLGQIFIQNFVAGDDESRSDRLQVEIKDIKLYSLNC\n",
      "                     TQLAGREAVGSEGSRMFCPPSGSGSANSQEEAHFTRHDFFESLHRGQAFHILNNTTIQ\n",
      "                     FKLEKIPIERESELTFSLSPDDLGTSSIMKIEGKFVNPVQVVLAKHVYEQVLQTLDNL\n",
      "                     VYSEDLNKYPASATSSPCPDSPLPPLSTCGESSVERKENGLFSHSSLSNTSQKSLSVK\n",
      "                     EVKSFTQIQATFCISELQVQLSGDLTLGAQGLVSLKFQDFEVEFSKDHPQTLSIQIAL\n",
      "                     HSLLMEDLLEKNPDSKYKNLMVSRGAPKPSSLAQKEYLSQSCPSVSNVEYPDMPRSLP\n",
      "                     SHMEEAPNVFQLYQRPTSASRKKQKEVQDKDYPLTPPPSPTVDEPKILVGKSKFDDSL\n",
      "                     VHINIFLVDKKHPEFSSSYNRVNRSIDVDFNCLDVLITLQTWVVILDFFGIGSTADNH\n",
      "                     AMRLPPEGILHNVKLEPHASMESGLQDPVNTKLDLKVHSLSLVLNKTTSELAKANVSK\n",
      "                     LVAHLEMIEGDLALQGSIGSLSLSDLTCHGEFYRERFTTSGEEALIFQTFKYGRPDPL\n",
      "                     LRREHDIRVSLRMASVQYVHTQRFQAEVVAFIQHFTQLQDVLGRQRAAIEGQTVRDQA\n",
      "                     QRCSRVLLDIEAGAPVLLIPESSRSNNLIVANLGKLKVKNKFLFAGFPGTFSLQDKES\n",
      "                     VPSASPTGIPKHSLRKTTSTEEPRGTHSQGQFTMPLAGMSLGSLKSEFVPSTSTKQQG\n",
      "                     PQPTLSVGQESSSPEDHVCLLDCVVVDLQDMDIFAAERHPREYSKAPEDSSGDLIFPS\n",
      "                     YFVRQTGGSLLTEPCRLKLQVERNLDKEISHTVPDISIHGNLSSVHCSLDLYKYKLIR\n",
      "                     GLLENNLGEPIEEFMRPYDLQDPRIHTVLSGEVYTCMCFLIDMVNVSLELKDPKRKEG\n",
      "                     AGSLARFDFKKCKLLYESFSNQTKSINLVSHSMMAFDTRYAGQKTSPGMTNVFSCIFQ\n",
      "                     PAKNSSTTQGSIQIELHFRSTKDSSCFTVVLNNLRVFLIFDWLLLVHDFLHTPSDIKK\n",
      "                     QNHVTPSRHRNSSSESAIVPKTVKSGVVTKRSSLPVSNERHLEVKVNVTGTEFVVIED\n",
      "                     VSCFDTNAIILKGTTVLTYKPRFVDRPFSGSLFGIEVFSCRLGNEHDTALSIVDPVQI\n",
      "                     QMELVGNSSYQNSSGLMDAFNSEDFPPVLEIQLQALDIRLSYNDVQLFLAIAKSIPEQ\n",
      "                     ANAAVPDSVALESDSVGTYLPGASRVGEEIREGTRHTLDPVLELQLARLQELGFSMDD\n",
      "                     CRKALLACQGQLKKAASWLFKNAEPLKSLSLASTSRDSPGAVAAPLISGVEIKAESVC\n",
      "                     ICFIDDCMDCDVPLAELTFSRLNFLQRVRTSPEGYAHFTLSGDYYNRALSGWEPFIEP\n",
      "                     WPCSVSWQQQAASRLHPPRLKLEAKAKPRLDINITSVLIDQYVSTKESWMADYCKDDK\n",
      "                     DIESAKSEDWMGSSVDPPCFGQSLPLVYLRTRSTASLTNLEHQIYARAEVKTPKRRQP\n",
      "                     FVPFALRNHTGCTLWFATLTTTPTRAALSHSGSPGVVPEGNGTFLDDTHNVSEWREVL\n",
      "                     TGEEIPFEFEARGKLRHRHTHDLRIHQLQVRVNGWEQVSPVSVDKVGTFFRYAAPDKN\n",
      "                     SSSSTIGSPSSRTNIIHPQVYFSSLPPVRVVFAVTMEGSARKVITVRSALIVRNRLET\n",
      "                     PMELRLDSPSAPDKPVVLPAIMPGDSFAVPLHLTSWRLQARPKGLGVFFCKAPIHWTN\n",
      "                     VVKTAEISSSKRECHSMDTEKSRFFRFCVAIKKENYPDYMPSNIFSDSAKQIFRQPGH\n",
      "                     TIYLLPTVVICNLLPCELDFYVKGMPINGTLKPGKEAALHTADTSQNIELGVSLENFP\n",
      "                     LCKELLIPPGTQNYMVRMRLYDVNRRQLNLTIRIVCRAEGSLKIFISAPYWLINKTGL\n",
      "                     PLIFRQDNAKTDAAGQFEEHELARSLSPLLFCYADKEQPNLCTMRIGRGIHPEGMPGW\n",
      "                     CQGFSLDGGSGVRALKVIQQGNRPGLIYNIGIDVKKGRGRYIDTCMVIFAPRYLLDNK\n",
      "                     SSHKLAFAQREFARGQGTANPEGYISTLPGSSVVFHWPRNDYDQLLCVRLMDVPNCIW\n",
      "                     SGGFEVNKNNSFHINMRDTLGKCFFLRVEITLRGATYRISFSDTDQLPPPFRIDNFSK\n",
      "                     VPVVFTQHGVAEPRLRTEVKPMTSLDYAWDEPTLPPFITLTVKGAGSSEINCNMNDFQ\n",
      "                     DNRQLYYENFIYIAATYTFSGLQEGTGRPVASNKAITCAELVLDVSPKTQRVILKKKE\n",
      "                     PGKRSQLWRMTGTGMLAHEGSSVPHNPNKPSAARSTEGSAILDIAGLAAVTDNRYEPL\n",
      "                     MLRKPDRRRSTTQTWSFREGKLTCGLHGLVVQAKGGLSGLFDGAEVVLGPDTSMELLG\n",
      "                     PVPPEQQFINQKMRPGSGMLSIRVIPDGPTRALQITDFCHRKSSRSYEVDELPVTEQE\n",
      "                     LQKLKNPDTEQELEVLVRLEGGIGLSLINKVPEELVFASLTGINVHYTQLATSHMLEL\n",
      "                     SIQDVQVDNQLIGTTQPFMLYVTPLSNENEVIETGPAVQVNAVKFPSKSALTNIYKHL\n",
      "                     MITAQRFTVQIEEKLLLKLLSFFGYDQAESEVEKYDENLHEKTAEQGGTPIRYYFENL\n",
      "                     KISIPQIKLSVFTSNKLPLDLKALKSTLGFPLIRFEDAVINLDPFTRVHPYETKEFII\n",
      "                     NDILKHFQEELLSQAARILGSVDFLGNPMGLLNDVSEGVTGLIKYGNVGGLIRNVTHG\n",
      "                     VSNSAAKFAGTLSDGLGKTMDNRHQSEREYIRYHAATSGEHLVAGIHGLAHGIIGGLT\n",
      "                     SVITSTVEGVKTEGGVSGFISGLGKGLVGTVTKPVAGALDFASETAQAVRDTATLSGP\n",
      "                     RTQAQRVRKPRCCTGPQGLLPRYSESQAEGQEQLFKLTDNIQDEFFIAVENIDSYCVL\n",
      "                     ISSKAVYFLKSGDYVDREAIFLEVKYDDLYHCLVSKDHGKVYVQVTKKAVSTSSGVSI\n",
      "                     PGPSHQKPMVHVKSEVLAVKLSQEINYAKSLYYEQQLMLRLSENREQLELDS\"\n",
      "     misc_feature    2154..2156\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"Phosphoserine. {ECO:0000244|PubMed:23186163,\n",
      "                     ECO:0000244|PubMed:24275569}; propagated from\n",
      "                     UniProtKB/Swiss-Prot (Q5THJ4.2); phosphorylation site\"\n",
      "     misc_feature    3267..3269\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"Phosphoserine. {ECO:0000244|PubMed:23186163};\n",
      "                     propagated from UniProtKB/Swiss-Prot (Q5THJ4.2);\n",
      "                     phosphorylation site\"\n",
      "     misc_feature    3279..3281\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"Phosphoserine. {ECO:0000244|PubMed:23186163};\n",
      "                     propagated from UniProtKB/Swiss-Prot (Q5THJ4.2);\n",
      "                     phosphorylation site\"\n",
      "     misc_feature    3291..3293\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"Phosphoserine. {ECO:0000244|PubMed:20068231,\n",
      "                     ECO:0000244|PubMed:23186163}; propagated from\n",
      "                     UniProtKB/Swiss-Prot (Q5THJ4.2); phosphorylation site\"\n",
      "     misc_feature    3579..3581\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"Phosphoserine. {ECO:0000244|PubMed:23186163,\n",
      "                     ECO:0000244|PubMed:24275569}; propagated from\n",
      "                     UniProtKB/Swiss-Prot (Q5THJ4.2); phosphorylation site\"\n",
      "     misc_feature    4188..4190\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"Phosphoserine. {ECO:0000244|PubMed:24275569};\n",
      "                     propagated from UniProtKB/Swiss-Prot (Q5THJ4.2);\n",
      "                     phosphorylation site\"\n",
      "     misc_feature    4959..4961\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"Phosphoserine. {ECO:0000244|PubMed:23186163};\n",
      "                     propagated from UniProtKB/Swiss-Prot (Q5THJ4.2);\n",
      "                     phosphorylation site\"\n",
      "     misc_feature    4974..4976\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"Phosphoserine. {ECO:0000244|PubMed:23186163};\n",
      "                     propagated from UniProtKB/Swiss-Prot (Q5THJ4.2);\n",
      "                     phosphorylation site\"\n",
      "     misc_feature    5262..5264\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"Phosphoserine. {ECO:0000244|PubMed:23186163};\n",
      "                     propagated from UniProtKB/Swiss-Prot (Q5THJ4.2);\n",
      "                     phosphorylation site\"\n",
      "     misc_feature    5448..5450\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"Phosphothreonine. {ECO:0000244|PubMed:18669648,\n",
      "                     ECO:0000244|PubMed:19690332, ECO:0000244|PubMed:23186163,\n",
      "                     ECO:0000244|PubMed:24275569}; propagated from\n",
      "                     UniProtKB/Swiss-Prot (Q5THJ4.2); phosphorylation site\"\n",
      "     misc_feature    5460..5462\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"Phosphoserine. {ECO:0000244|PubMed:24275569};\n",
      "                     propagated from UniProtKB/Swiss-Prot (Q5THJ4.2);\n",
      "                     phosphorylation site\"\n",
      "     misc_feature    7470..7472\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"Phosphoserine. {ECO:0000244|PubMed:19690332,\n",
      "                     ECO:0000244|PubMed:21406692}; propagated from\n",
      "                     UniProtKB/Swiss-Prot (Q5THJ4.2); phosphorylation site\"\n",
      "     misc_feature    8178..8180\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"Phosphoserine. {ECO:0000244|PubMed:23186163,\n",
      "                     ECO:0000244|PubMed:24275569}; propagated from\n",
      "                     UniProtKB/Swiss-Prot (Q5THJ4.2); phosphorylation site\"\n",
      "     misc_feature    8748..8750\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"Phosphoserine. {ECO:0000244|PubMed:23186163};\n",
      "                     propagated from UniProtKB/Swiss-Prot (Q5THJ4.2);\n",
      "                     phosphorylation site\"\n",
      "     misc_feature    8757..8759\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"Phosphoserine. {ECO:0000244|PubMed:23186163};\n",
      "                     propagated from UniProtKB/Swiss-Prot (Q5THJ4.2);\n",
      "                     phosphorylation site\"\n",
      "     misc_feature    9114..9116\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"Phosphoserine. {ECO:0000244|PubMed:23186163};\n",
      "                     propagated from UniProtKB/Swiss-Prot (Q5THJ4.2);\n",
      "                     phosphorylation site\"\n",
      "     misc_feature    10737..10739\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"N6-acetyllysine. {ECO:0000244|PubMed:19608861};\n",
      "                     propagated from UniProtKB/Swiss-Prot (Q5THJ4.2);\n",
      "                     acetylation site\"\n",
      "     exon            265..342\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            343..533\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            534..614\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            615..731\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            732..836\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            837..1007\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            1008..1108\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            1109..1277\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            1278..1379\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            1380..1581\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            1582..1761\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            1762..1892\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            1893..1968\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            1969..2139\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            2140..2270\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            2271..2403\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            2404..4617\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            4618..4769\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            4770..5801\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            5802..5892\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            5893..6019\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            6020..6200\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            6201..6383\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            6384..6606\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            6607..6817\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            6818..6989\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            6990..7102\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            7103..7315\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            7316..7581\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            7582..7715\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            7716..7871\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            7872..8082\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            8083..8157\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            8158..8364\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            8365..8454\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            8455..8595\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            8596..8718\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            8719..8793\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            8794..8899\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            8900..9052\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            9053..9188\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            9189..9236\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            9237..9387\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            9388..9598\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            9599..9846\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            9847..10038\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            10039..10165\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            10166..10308\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            10309..10439\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            10440..10615\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            10616..10739\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            10740..10975\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            10976..11084\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            11085..11248\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            11249..11357\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            11358..11537\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            11538..11651\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            11652..11801\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            11802..11951\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            11952..12048\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            12049..12197\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            12198..12332\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            12333..12500\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            12501..12633\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            12634..12829\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            12830..12961\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            12962..13202\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            13203..16357\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /gene_synonym=\"SCAR4\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "ORIGIN      \n",
      "        1 ctgagcgccg cgggcctgcg ccattgagga gcggcgggga ggaaacgccg cgcagcgccg\n",
      "       61 ggctggggcg ggcggcccgg gacaccgaca gatttttctg tgaccatgaa agagagaaat\n",
      "      121 aaagaatgat ccatgatttc taaacacctt ttcctgagga tatagtcatg ttggaaggcc\n",
      "      181 ttgtagcctg ggttctcaat acctatttgg gaaaatatgt caataacctg aacactgacc\n",
      "      241 agctctcagt tgcacttctc aaaggtgctg ttgaattaga aaacttgcca ttaaagaaag\n",
      "      301 atgccttgaa agaattggaa ttaccatttg aagtcaaagc tggcttcatt gggaaagtaa\n",
      "      361 cccttcagat tcccttttat cgcccccatg tggacccttg ggtgatctcc atctccagcc\n",
      "      421 ttcacttaat tggagcccca gagaaaatac aggatttcaa tgatgaaaag gagaagctgt\n",
      "      481 tggaaaggga acgtaagaaa gcactacttc aagccctgga ggagaaatgg aagaatgacc\n",
      "      541 gccagcagaa aggggagtcc tattggtatt cagttaccgc ctccgtagtt acaaggattg\n",
      "      601 tggagaatat tgaattaaaa attcaagatg tccatttacg ctttgaagat ggtgtcacca\n",
      "      661 atccctccca tccttttgct tttggcatct gcattaagaa tgtgtccatg caaaatgctg\n",
      "      721 tgaatgagcc tgtacagaaa ctaatgcgga aaaagcaatt agacgtagca gaatttagca\n",
      "      781 tctattggga tgtcgattgc actttactgg gggatttgcc tcagatggag ttacaggagg\n",
      "      841 ccatggccag gagcatggag agtcgcagcc atcactacgt cctggagcct gtgtttgcat\n",
      "      901 ctgctctttt gaagagaaac tgctccaaga agcccctgcg gtctcggcac agtccccgta\n",
      "      961 ttgattgtga tattcagctg gagaccattc ccttgaaact ctctcagctg caataccggc\n",
      "     1021 aaatcatgga attcctcaag gagctggaac gaaaggagag gcaggtgaag ttccgaaggt\n",
      "     1081 ggaaacccaa ggtggcgata tctaagaact gccgagaatg gtggtatttt gctttgaatg\n",
      "     1141 ctaacttgta tgagatcaga gagcagagga aacgttgcac ctgggacttt atgttgcacc\n",
      "     1201 gcgctcgtga tgctgtatct tacactgaca aatatttcaa caagttaaaa ggaggcctgc\n",
      "     1261 tgtccacaga tgacaaggag gaaatgtgtc ggattgaaga ggaacagagc tttgaggaat\n",
      "     1321 tgaagatttt gcgtgaactg gttcatgatc gatttcacaa acaggaagaa ctagcagaga\n",
      "     1381 gtctgcggga gcctcagttt gattctccag gagcctgtcc gggagcccca gaacccggtg\n",
      "     1441 gaggcagtgg gatgctgcag tatctccagt cctggtttcc tggatggggt ggctggtacg\n",
      "     1501 ggcagcagac cccagaaggg aatgtggttg agggactgtc agcagagcaa caggagcagt\n",
      "     1561 ggattcctga agagatcctg ggcactgagg agttttttga ccccactgca gatgcctcgt\n",
      "     1621 gtatgaacac gtatacaaag cgagatcatg tctttgccaa actgaatttg cagttgcagc\n",
      "     1681 gaggtacagt gactctgtta cacaaggagc aaggaactcc tcaaatgaat gaaagtgctt\n",
      "     1741 tcatgcagct cgagttttca gatgtaaaac ttctagcaga gtctcttcct cgaagaaatt\n",
      "     1801 cctcgttgct ttcagtccgg ttgggtggac tgtttcttcg agacctggct acagaaggaa\n",
      "     1861 ctatgtttcc tcttctagtc ttccctaatc cacaaaaaga agttggcaga gtctcacaat\n",
      "     1921 cttttggtct acaaactaca tctgcagaca gaagtgatca ttacccagct gcagatccag\n",
      "     1981 atggccccgt ttttgagatg ctgtatgaga gaaatccggc gcacagccac tttgagaggc\n",
      "     2041 ggctcaatgt cagcacaagg cccttgaaca tcatatacaa tccgcaggcc attaaaaaag\n",
      "     2101 tagcagactt tttctacaag ggaaaggttc atacctcagg ttttggttat cagtctgaac\n",
      "     2161 ttgagctgag agtggctgaa gctgcccgaa gacaatataa caagctgaag atgcagacca\n",
      "     2221 aggcagaaat ccggcaaact cttgatcgtt tgctagtggg tgatttcatt gaggagagta\n",
      "     2281 aacgatggac cgtgcggctg gatatttctg cccctcaggt gatatttcct gatgatttca\n",
      "     2341 aattcaagaa tcctgtgtta gttgtcgtgg atctaggaag aatgcttttg acgaacaccc\n",
      "     2401 aagataactc caggaggaaa agtagggatg ggtcagcatc tgaagagacc cagtttagtg\n",
      "     2461 atgatgaata taagaccccc ctggccacac ctcctaacac cccacctccc gagtcaagca\n",
      "     2521 gcagcaacgg agagaaaaca cctccctttt ctggagttga gttcagtgaa gaacagcttc\n",
      "     2581 aagcacattt aatgagcaca aagatgtatg agaggtactc gctgtcattt atggacctcc\n",
      "     2641 agatcatggt tggacgagtg aaagacaatt ggaagcatgt ccaggatatt gacgtgggac\n",
      "     2701 caacacatgt ggtagagaag ttcaacgttc acctacagtt agagcgtcga ttgatttata\n",
      "     2761 cttcagatcc caaatatcca ggagccgtgc tctcaggcaa cttaccagac ttaaaaatcc\n",
      "     2821 acattaatga agataaaata tctgcactaa agaattgctt tgctctcctc accaccccag\n",
      "     2881 aaatgaaaac ttctgacact cagattaaag aaaagatttt tccccaggag gagcagcggg\n",
      "     2941 gaagtttgca agactccgta atgaatttaa cccagagcat tgtgttgttg gagcagcata\n",
      "     3001 cccgcgaggt tctggtggag tcgcagctcc tcctggcgga atttaaagtg aactgtatgc\n",
      "     3061 agcttggtgt tgagagcaat ggccggtaca tttctgtgct caaggtgttt ggtaccaatg\n",
      "     3121 ctcactttgt gaagaggcct tatgatgctg aagtctccct aactgttcat ggtttgctcc\n",
      "     3181 tggtggatac catgcagaca tatggtgctg attttgacct tttgatggct tcacataaga\n",
      "     3241 acttgagctt tgatattcca acgggaagcc ttcgggatag cagggcccag tctcctgtct\n",
      "     3301 ctggaccgaa tgtggcccac ttaactgatg gagctacact gaacgaccga tcagctacta\n",
      "     3361 gtgtttcact tgacaaaatt cttaccaaag agcaagagtc ccttattaag ttggaatatc\n",
      "     3421 agtttgtgag ttcagagtgc ccatcgatga atttagacag tactcttcag gtgatttccc\n",
      "     3481 tacaggtgaa taatttagat attatcctca atccagagac gattgtggag ctaattggtt\n",
      "     3541 ttcttcaaaa atcctttccc aaggaaaaag atgatttaag tcctcaacct ttaatgactg\n",
      "     3601 attttgaaag aagcttcaga gaacaaggaa cttaccagtc tacatatgaa caaaacactg\n",
      "     3661 aggttgcagt ggaaatccat aggctgaact tactgcttct tcggacagtg ggcatggcaa\n",
      "     3721 atagagagaa atatggcaga aaaattgcaa ctgcaagtat aggtggcacc aaagttaatg\n",
      "     3781 tctcaatggg tagcacgttt gacatgaatg gttctcttgg ctgtttacag cttatggatt\n",
      "     3841 tgacacaaga taacgttaaa aaccagtatg ttgtcagcat tgggaattct gtaggctatg\n",
      "     3901 aaaatatcat cagtgatatt ggctactttg aatctgtgtt tgtcagaatg gaagatgcag\n",
      "     3961 ccctcactga agctttgagt ttcacgtttg ttgagagatc taaacaggag tgttttctca\n",
      "     4021 acctgaagat ggcttcttta cattataacc actctgctaa gtttttgaag gagttgacgt\n",
      "     4081 tatccatgga tgaactggaa gaaaattttc gaggtatgct gaaaagcgca gccaccaaag\n",
      "     4141 tcaccacagt actagctacc aagactgccg agtatagcga gatggtatcg ctctttgaaa\n",
      "     4201 ctccaaggaa gactcgggaa ccctttatct tagaggaaaa tgaaatatat gggtttgacc\n",
      "     4261 tagcttcgtc tcatttggac actgtaaagc taatcttgaa cataaacatt gaatcaccag\n",
      "     4321 ttgtttctat ccctcggaag ccggggagtc ctgagttgtt ggtgggacac ttgggacaga\n",
      "     4381 tattcatcca gaattttgtg gcgggagatg atgaatccag aagtgaccgt ctgcaggtgg\n",
      "     4441 aaatcaagga cattaaactg tattctttga attgcaccca gttggcaggt agagaagctg\n",
      "     4501 ttgggtctga aggaagccgg atgttttgcc caccttccgg gtctggcagt gccaacagtc\n",
      "     4561 aggaggaagc tcatttcaca cgacatgatt tctttgaatc tttgcataga ggtcaagctt\n",
      "     4621 ttcacatcct gaacaacacc accattcagt ttaaactgga gaagatccct atagagagag\n",
      "     4681 aatctgaatt gactttttct cttagcccag atgacctggg aacttctagc atcatgaaga\n",
      "     4741 ttgaaggaaa atttgtcaat ccagttcagg tggtgttagc aaagcatgta tatgagcagg\n",
      "     4801 ttttacaaac cctggacaat ctcgtgtaca gtgaagatct gaataagtat ccagccagtg\n",
      "     4861 ctacctcctc cccttgccct gattctcctc tgcctcccct cagtacctgt ggagaatctt\n",
      "     4921 ctgttgaaag gaaggagaat ggattgttca gccactccag cctttctaac acctctcaga\n",
      "     4981 agtcattgtc agtgaaggaa gtcaaatcct ttactcagat tcaagccacc ttttgtatat\n",
      "     5041 cagagcttca ggttcagcta agtggagatc tgactttggg ggcccaaggt cttgtgagct\n",
      "     5101 taaagtttca ggactttgag gtggaattca gtaaagacca tccccagact ttatctattc\n",
      "     5161 agattgccct gcattctctg ctgatggagg acttattgga gaagaatcca gattctaaat\n",
      "     5221 ataagaacct gatggtgtct cgaggagccc ctaagccatc tagtttagca caaaaagaat\n",
      "     5281 acctttctca gtcttgcccc tcagtgtcca atgtggaata tcctgatatg cctcggtctc\n",
      "     5341 tcccttccca catggaagaa gctcctaatg tcttccagtt gtatcaaagg cccacctctg\n",
      "     5401 cgtcccggaa aaagcaaaag gaagtccaag acaaggacta tcccttgacc ccacctcctt\n",
      "     5461 ctccaacagt ggatgagccc aagatacttg ttggaaagag taaatttgat gattccttag\n",
      "     5521 tccacatcaa catattcttg gtagataaga aacatccaga attctcttcc agttacaatc\n",
      "     5581 gagttaaccg gagcattgat gttgatttta attgcttgga tgtgctgatc acactgcaaa\n",
      "     5641 cctgggttgt gatattagac ttttttggaa tcggctccac tgcagacaac cacgcaatga\n",
      "     5701 ggctgcctcc tgagggcatt ctgcacaacg tgaagttgga gccacatgcc tccatggagt\n",
      "     5761 ctggacttca ggatccagtg aacaccaaac tggatctcaa ggttcattca ctttctctag\n",
      "     5821 tgctgaataa gaccaccagt gagcttgcca aagcaaatgt gtccaaatta gtagcacacc\n",
      "     5881 tggaaatgat tgagggagac ctggccttac agggcagcat tgggagtctg tctctaagtg\n",
      "     5941 acctcacatg ccatggagag ttctacagag aacggttcac taccagtggt gaagaagcac\n",
      "     6001 tcatcttcca gacttttaaa tacggacggc ctgaccctct gctccggaga gaacacgaca\n",
      "     6061 ttcgcgtgag cctccggatg gcctctgtgc agtatgtgca tactcagcgt ttccaggcag\n",
      "     6121 aggtggtggc cttcattcag catttcactc agctgcagga tgtcttaggg cgccagcgag\n",
      "     6181 ctgctattga ggggcagacg gtgagagatc aagcccagcg ctgttcacgg gttctcctgg\n",
      "     6241 atattgaggc tggtgctccc gttctcttga tcccagaaag ttccagatca aataatctga\n",
      "     6301 ttgtagcaaa tttggggaag ttgaaagtca aaaataagtt tctgtttgct ggttttcctg\n",
      "     6361 gcaccttttc cctacaagat aaggaatctg tgccttcagc ttccccaacg ggtattccca\n",
      "     6421 aacacagtct gaggaaaacg acaagcacgg aggagcccag gggaacccat tcccaggggc\n",
      "     6481 agttcacgat gcctcttgct ggaatgagcc taggaagcct gaagagtgag tttgtgccca\n",
      "     6541 gtacctccac caagcagcaa gggccgcaac ccacactgtc tgttggccaa gagtccagta\n",
      "     6601 gtccagaaga ccatgtctgc ctgctggatt gcgttgtcgt ggatctccag gacatggaca\n",
      "     6661 tctttgctgc agagagacat ccgagagaat actcgaaggc accagaggat agtagtggag\n",
      "     6721 atctgatctt cccttcctat tttgtgcgac agacaggagg aagcctctta accgagcctt\n",
      "     6781 gtaggctgaa attgcaggtg gaaaggaatt tggacaaaga aataagtcat actgtgccag\n",
      "     6841 acatatctat ccatggcaat ctctcctcag tccactgctc tctggatctg tataaataca\n",
      "     6901 agctgatccg cggcttatta gagaacaacc tgggagaacc catagaggaa tttatgcggc\n",
      "     6961 cttatgattt acaagatcca agaattcata ctgtcctgag tggagaagtg tacacctgta\n",
      "     7021 tgtgcttcct cattgatatg gtgaatgtaa gtctggagct taaagatcca aaaagaaaag\n",
      "     7081 aaggtgctgg gtccctagcc agatttgact tcaagaaatg caaactgctc tatgaaagtt\n",
      "     7141 tttccaacca aaccaagtcc attaacttgg tttcccattc catgatggct tttgacaccc\n",
      "     7201 gttatgctgg gcagaagacc agccctggca tgacgaatgt gttcagctgt atctttcagc\n",
      "     7261 ccgctaagaa cagcagcacc acccaagggt ccattcagat tgaactacat ttcagatcta\n",
      "     7321 ccaaggattc ctcctgcttt acagtagttc tcaacaatct ccgtgtgttt ctcatatttg\n",
      "     7381 actggctact gttagtccat gattttctcc acactcccag tgatattaag aaacaaaatc\n",
      "     7441 atgttactcc ttctcgccac cgtaactcta gcagcgaatc tgctatagtt cccaaaactg\n",
      "     7501 tgaagagtgg agtagttacc aagcggtctt cccttcctgt gtccaatgaa aggcacctgg\n",
      "     7561 aggtcaaggt caatgtaaca ggtacggagt ttgtggtcat tgaagatgtg tcctgcttcg\n",
      "     7621 acaccaatgc cattattctg aaaggcacca cagtgctcac ctataagccc cggtttgttg\n",
      "     7681 atcgcccctt ttcaggaagt ttgtttggca ttgaggtgtt ttcatgccga ctagggaatg\n",
      "     7741 agcatgatac agctctttca attgtggatc ccgtacaaat tcaaatggag ttggtgggga\n",
      "     7801 attcttctta tcaaaatagt tcaggattga tggatgcatt caatagtgaa gatttcccac\n",
      "     7861 ctgtcctgga gattcagtta caagccctgg atatcagact ctcctataat gatgttcagc\n",
      "     7921 tgtttcttgc cattgcaaaa tccatcccag agcaagctaa tgctgcagtg ccagactcag\n",
      "     7981 tggccctgga gtcagactcc gttggcactt accttccagg tgcatctcgc gttggagagg\n",
      "     8041 aaatcagaga agggacaaga cacaccttag atcctgtctt ggagttacag ctggctaggc\n",
      "     8101 tgcaggagct gggattcagc atggatgatt gtcgcaaagc tcttttggcg tgtcaaggcc\n",
      "     8161 aattgaaaaa ggcagcaagt tggttgttta agaatgcgga acctctgaag tctctttcct\n",
      "     8221 tggcctccac cagccgagat agcccagggg ctgtggcagc gccattgatc tctggcgtgg\n",
      "     8281 agatcaaagc tgagagtgtg tgcatctgtt tcatcgatga ctgcatggat tgtgatgttc\n",
      "     8341 ctctcgctga actcaccttt tcccgtctga attttcttca gcgtgtaaga actagccctg\n",
      "     8401 aaggctatgc ccacttcacc ctttctggag attattataa ccgtgctctt tcaggctggg\n",
      "     8461 agccatttat tgagccttgg ccatgctctg tatcctggca acagcaggca gctagtcgtc\n",
      "     8521 tccatcctcc tcgactgaag ctagaagcca aggccaaacc tcgtttggat atcaatatca\n",
      "     8581 cttctgtgct aattgaccag tatgtaagta ccaaggaatc gtggatggca gattactgta\n",
      "     8641 aagatgacaa ggacatagag tcagctaaat cagaagactg gatgggctct tcggtggatc\n",
      "     8701 ctccatgttt tggacaaagc ctcccccttg tctaccttag aactaggagt acagccagtc\n",
      "     8761 tgactaacct agagcaccag atctatgcta gagcagaggt gaaaaccccc aagcgccggc\n",
      "     8821 agccatttgt cccctttgct ctgaggaacc acacggggtg cactttgtgg tttgccaccc\n",
      "     8881 tgaccaccac acccaccaga gctgcactct ctcacagtgg gagtccaggg gtagttccag\n",
      "     8941 aagggaacgg aacatttctc gatgatactc acaatgttag tgaatggcga gaagtcctta\n",
      "     9001 caggtgaaga gattcccttt gaatttgaag caagaggaaa gttaagacac agacacaccc\n",
      "     9061 atgacctccg gattcatcaa ctgcaagtga gagtaaatgg ctgggagcaa gtgagcccag\n",
      "     9121 tgtctgtgga caaagtcggg accttttttc gatatgcagc accagataaa aattcatctt\n",
      "     9181 cctctacgat tggcagccca agcagcagaa caaatattat acatccccag gtttatttct\n",
      "     9241 cttcactccc accagtgcgg gtggtctttg cagtgactat ggaaggcagt gcacggaaag\n",
      "     9301 tcatcactgt ccggtcagcc ctcattgtga ggaacagact tgagacacca atggaactaa\n",
      "     9361 gactggatag cccatcagct ccagacaagc cagtggtgct tcctgctatc atgccagggg\n",
      "     9421 attcgtttgc tgtgccttta cacctcactt cttggcggct acaggcccgg cccaaaggat\n",
      "     9481 tgggtgtatt tttctgtaag gctcccattc attggaccaa tgtagtgaag actgcagaaa\n",
      "     9541 ttagtagcag taaacgagag tgccactcta tggacacaga aaaaagccga tttttcaggt\n",
      "     9601 tttgtgtggc tataaagaaa gagaattatc cagattatat gccctcaaac atattttctg\n",
      "     9661 acagtgcaaa acagattttc agacagcctg ggcataccat atatctcctg ccaactgtgg\n",
      "     9721 taatctgcaa cttgctaccc tgtgaacttg atttttatgt taaaggaatg ccaattaatg\n",
      "     9781 ggacgctgaa acctggcaag gaggcagctc tccatacagc tgatacatcc cagaacattg\n",
      "     9841 agctgggggt atcactggag aatttccccc tctgtaaaga attgctcatt ccacctggaa\n",
      "     9901 cccaaaacta tatggtgaga atgcgactct atgacgtcaa ccgtcggcag ctgaacctca\n",
      "     9961 ccatccggat tgtgtgtcga gcagaaggat ccttaaagat cttcatttct gctccatatt\n",
      "    10021 ggctgattaa caaaacaggg ttgccactga tcttcagaca ggacaatgcc aagacagatg\n",
      "    10081 ctgcaggcca gtttgaggag catgagctgg cccgtagcct gagtcctctc ttattctgct\n",
      "    10141 atgctgacaa agagcagcca aacctctgca cgatgagaat cggaaggggg attcatccag\n",
      "    10201 aaggcatgcc gggctggtgt cagggcttct ccctggatgg tggtagtggt gtccgagctt\n",
      "    10261 tgaaagtcat ccagcaagga aaccgcccag ggctgatcta taacattggt attgatgtca\n",
      "    10321 agaaaggccg aggtcgatac attgatacct gcatggtcat ctttgccccc cgttacctgt\n",
      "    10381 tagataataa atcatctcac aagcttgcat ttgcacagag ggaatttgcc aggggacagg\n",
      "    10441 gaacagccaa tcccgaaggt tacatttcca cccttcctgg ttccagtgtg gtgttccact\n",
      "    10501 ggcctcggaa tgactatgat cagctattgt gtgtcagact gatggacgtt cccaattgta\n",
      "    10561 tttggtctgg aggctttgaa gtcaacaaga ataattcctt ccatatcaac atgagggata\n",
      "    10621 ccttgggaaa atgcttcttc ctacgagtgg aaattactct ccgaggagct acgtatagga\n",
      "    10681 tctcatttag tgacacagat cagttacctc ctcctttccg aattgacaac ttttctaagg\n",
      "    10741 tcccggttgt ctttactcag catggcgtag ctgaacccag gctccggact gaagtgaagc\n",
      "    10801 ccatgacttc attggattat gcctgggacg aacccacctt gccacctttt atcactctga\n",
      "    10861 ctgttaaagg ggcagggtcc tctgagatca actgcaacat gaatgatttc caggataatc\n",
      "    10921 ggcagcttta ttatgaaaat ttcatttaca ttgctgctac atatacattc tctggcttgc\n",
      "    10981 aggagggaac aggcaggcct gtggcttcca acaaggccat tacctgtgcg gagctcgttt\n",
      "    11041 tggatgtctc acccaagaca caaagagtca ttttaaaaaa gaaggaacca ggaaagcgtt\n",
      "    11101 ctcagctgtg gaggatgaca ggaacaggaa tgctggccca tgagggctcc tcagttcctc\n",
      "    11161 acaatcccaa taagccctca gccgcccgct ccaccgaggg gtctgccatc ttagatattg\n",
      "    11221 ctggtctcgc tgcagtgact gacaacagat acgagccact gatgctgaga aagcctgacc\n",
      "    11281 gcaggcgaag cacaactcag acgtggagtt tccgagaagg aaaactgacc tgtgggttac\n",
      "    11341 atgggttggt cgtccaggcc aaaggaggac tttctggttt gtttgatgga gctgaagttg\n",
      "    11401 ttcttggtcc tgacacttcc atggagcttt tggggccagt tccacctgaa caacaattta\n",
      "    11461 ttaatcaaaa aatgagacct ggttctggaa tgttatccat cagagtcatc ccagatggac\n",
      "    11521 caactagagc actccagata acagatttct gccaccggaa aagcagccgt tcatatgaag\n",
      "    11581 tggatgaact tcctgtcacc gaacaagagc tgcagaaatt aaagaatcca gatacagagc\n",
      "    11641 aggaattgga agtgcttgtg aggttagaag gtggaattgg gttgtcctta attaataaag\n",
      "    11701 tcccagaaga actggtcttt gcaagtctta caggaatcaa tgtgcactat acacagctgg\n",
      "    11761 caaccagtca catgcttgaa ctcagcatac aggatgtaca ggtggacaat cagctcattg\n",
      "    11821 gtaccacgca gcccttcatg ctctatgtga ctcccctgag caatgagaat gaggtcatcg\n",
      "    11881 agaccggccc agctgtgcaa gtcaacgcag tgaagttccc cagtaagagt gcactgacca\n",
      "    11941 acatctacaa gcatctgatg atcacagctc agagattcac agtgcaaatt gaggagaaac\n",
      "    12001 tgctcctcaa gctgctaagt ttctttggct acgatcaagc agaatcagag gtggaaaaat\n",
      "    12061 atgatgaaaa cctccatgaa aagacagctg agcaaggtgg aacaccaatt cgatactact\n",
      "    12121 ttgaaaatct caaaatcagc attcctcaga tcaagctaag tgtgttcacc tccaacaagc\n",
      "    12181 tcccattgga tcttaaggcc ctaaaaagca ccttggggtt tcctttgata cggtttgaag\n",
      "    12241 acgctgtgat taatctagat ccattcactc gggtacatcc ctatgagacc aaggagttca\n",
      "    12301 tcatcaatga tatcctcaaa catttccagg aggaactcct cagccaggca gctcgaatcc\n",
      "    12361 tgggatcagt ggattttctt ggcaatccta tggggctttt gaatgatgtt tctgaagggg\n",
      "    12421 ttactggact gataaaatat ggaaatgtcg ggggcctcat cagaaatgtt acacacggag\n",
      "    12481 tatcaaactc tgctgccaag tttgctggaa cattatcaga tggcttaggg aagacgatgg\n",
      "    12541 acaatcggca tcagtcagag cgggagtaca tcaggtacca tgcagccaca agtggtgaac\n",
      "    12601 accttgtagc cggcatccat ggcctggctc atggtatcat tggtggactg accagtgtta\n",
      "    12661 taacttcgac agtggaaggt gtgaaaacag aagggggtgt cagcggtttc atatctggcc\n",
      "    12721 ttggaaaagg gcttgttggc actgtaacca agccagtggc aggcgccctg gattttgcat\n",
      "    12781 cagaaacagc ccaggcggtg agagacacag ccacactcag cggccccagg actcaagcac\n",
      "    12841 agagggttcg gaaaccgcgt tgctgcacgg ggccccaggg gctgcttccc cgatattctg\n",
      "    12901 agagccaggc ggaaggacag gagcagctct tcaaactcac agacaacata caggacgaat\n",
      "    12961 tcttcatcgc tgtggagaac attgacagct actgcgtgct catctcctcc aaagctgttt\n",
      "    13021 acttcctgaa aagtggagac tacgtggatc gagaagccat tttcctagaa gtcaaatacg\n",
      "    13081 atgacctcta ccactgcctt gtctccaaag accatgggaa ggtgtatgtg caggtgacca\n",
      "    13141 agaaagccgt gagcacgagc agtggagtgt ccatccccgg cccctcccac cagaagccca\n",
      "    13201 tggtccatgt gaaatctgag gtccttgctg tcaagttgtc acaagaaata aactacgcaa\n",
      "    13261 agagcctcta ctatgaacag cagcttatgt taagactcag cgaaaaccga gagcagctgg\n",
      "    13321 agctggactc ctgaagcccc gctgctgaga tgggcgctcc cgacacagcg cagacccacc\n",
      "    13381 aggaggaaag aggcccagct ctcagctgac gatggaggca gaaccggagt cgggtttggg\n",
      "    13441 gaagttgtca aggaatgagg gaaagtaaat cctcatgagg aaaagtacaa atggaaatcg\n",
      "    13501 tattaatttg tgaggcaggg agttatttta gattatggga aataattttt aaaggtattg\n",
      "    13561 gttaaataac gtttaaaaac atgtactgag atgaatctaa tttttagatt gccctgtatt\n",
      "    13621 ttgttaacat gtatatatgt acaacagtgt gtttgtaaat atataggaac gtttctgaac\n",
      "    13681 agggtctgtg ctatgtgtaa aggtttgtta actgtaaagt aatataaagt tatattggat\n",
      "    13741 cttctattgc actaattcta gatgtctaat tcaggatact gtctatagaa aggcattctt\n",
      "    13801 aaaagttaaa gaatgttacg tcttagtttt ggagactaaa gtattcccag taaagtgggt\n",
      "    13861 tgaggtgagg gctgtggtcc tgaaagggac gcctttgaca tcgtggctgt ccagttgggc\n",
      "    13921 tgtgagctgt ggcacccagg actggcgctg gcccttcaga aggatctagg agaggggctt\n",
      "    13981 gggagcccac ttttaatttc tcacccccat tttacaaaga gtgcttagat tcttacaaat\n",
      "    14041 tatgatgtaa gttatccatt tggctttttc ctaactagtc ttaccaaact tagggggaaa\n",
      "    14101 cctgtgctcc attaccacat gggtgcaagt cagcattgta agttttctca ggttattatt\n",
      "    14161 attagagagg ttggaaacat tggtaaactc tgttgattga gaaggaaaaa aaaagtccca\n",
      "    14221 ttgaactgtt gcaacaaatc agaaatccac ataaaagtgc tctcctgcct gggcagcaac\n",
      "    14281 aaccaagaac aaagccccgg gactgttttc tttttaataa agccacaggc aggcatcgta\n",
      "    14341 gctccacagc ccgaggggac acaggatgga aaccccagga tgagaaggga gcagggagag\n",
      "    14401 ttccagaaag ggggatgaaa taggagtatt aaaaagctgc gttggtaagt ttttcatgga\n",
      "    14461 accaagattt gacaaaggca tctcttatcc ttggttttaa attcctgctg ggagcaaggc\n",
      "    14521 ctggtatgag cgccctgggt cttgtttttg gtgtttcgct tttctgtaag gattaagcag\n",
      "    14581 atagggagaa gggaaaaggg gcctcacttt agaatgaatg agtcaccttg tgatttttaa\n",
      "    14641 atttttattt taataaagct aatcaatttc tacaaccttg tcacatgtag ctgagtctgg\n",
      "    14701 gatgactcag tggatcagtg gatgcgtgga aggttttggt gtttataact catgacccaa\n",
      "    14761 atccttccaa gacacaaatg aagctaatct tcttcccctc cctttcctct cccacttccc\n",
      "    14821 ctctgtgtct gtgtctgtgt gtgtgtgtgt gtgtgtgtgt gtgtgcgcgc gcgcgcgtgc\n",
      "    14881 atgcagagag gaaggaaggg agctttagca caagtagcta catagtgttc aaggatcagc\n",
      "    14941 tttcttgaac tccgaggctt ggttgccttt gctgaactcc aagttagtgg attgcagaat\n",
      "    15001 ggaaacttgg cttttgcggc actgggtgag ttttagtttg tgtgtgtctt gctggggggt\n",
      "    15061 ggtgatgatt gtctcagcac tcacgcactg cacaagatgg cagcaggata cagcactgca\n",
      "    15121 caagatggca gctcctctgc agcttcctcc tcagcctccc tccttgcacc cccacaggtt\n",
      "    15181 tggcttgtgg tttttgtcat cagtaaccta ctgcctgaga tcatgatctc ttaaaagatg\n",
      "    15241 agactctcgg aagggttgat tgtatgcgtc agtgagcctt ctatcacctt ctggaacaaa\n",
      "    15301 gtcacttgaa atctcttgat gagattaagg agtttagtgt tactaagaaa atctgctttg\n",
      "    15361 ggccgcagca gtgctgggtg ttctcagacc tgactgagga agttagctgc gggctgccct\n",
      "    15421 gtgggctggt gcttcaggag gaatccagag aagtgttcag atgcccccct tgggctcctt\n",
      "    15481 tctaatttta atcagctctt taaatagctg cccatctcct gtgattgcac aaccaagcac\n",
      "    15541 tttgacattt gcaccttagg agaggcagat gttaaaatgg aatccaaaga ccacctaggg\n",
      "    15601 cggggctggg tgggagatgg gagggccaac tgcgagctgc tccacttctc agctctcccc\n",
      "    15661 tgccctgcag ccctgggcca gacaaggcca gaaggtttca ggggcatttg acatcccctc\n",
      "    15721 ctggttctca ccaggaaaac atccaaagct ttggaggaaa caggccctgc ccctggctcc\n",
      "    15781 ttaaatgccc cgtctctttg taaactgata ttcagccagc aatgcctaag actttgttaa\n",
      "    15841 gatcatttct actgcttttc tttctgcttc aaacacacag ttcgtctctg aggaaagtaa\n",
      "    15901 aataaatgga ataagagtaa attgggtaag gagatatcca aagctaccca gtcccttgac\n",
      "    15961 ccagcacagt tggccgaccc gtgtcactcc ctggctgtcg ctgcttctct gtgctcactg\n",
      "    16021 aagggtgagc caggccagtg cttccccagc ccctgggcct ggtcactaca cagtggaaaa\n",
      "    16081 cagacaagcg gccccttccc caaatcccaa gagtgtcttg ctgcttggtg ggtgctcatc\n",
      "    16141 gcaatgttct gaaggctcca gggccacttt gtttgtaagt atgatctggg cctcaaaata\n",
      "    16201 ccatagtagc tgcttgataa aattctaaaa atatctggtt ctctattatg taaacactat\n",
      "    16261 tacagtcacc agtgtgtgaa gactcttgag tctggttctc atatcagagt catcattttt\n",
      "    16321 cttcctgtgg aataaaatgc cttgtggact tcccaaa\n",
      "//\n",
      "\n",
      "LOCUS       CF259228                 374 bp    mRNA    linear   EST 04-FEB-2011\n",
      "DEFINITION  j425r.ab1 Juvenile-specific Acetabularia acetabulum cDNA, mRNA\n",
      "            sequence.\n",
      "ACCESSION   CF259228\n",
      "VERSION     CF259228.1\n",
      "DBLINK      BioSample: SAMN00173644\n",
      "KEYWORDS    EST.\n",
      "SOURCE      Acetabularia acetabulum (mermaid's wine glass)\n",
      "  ORGANISM  Acetabularia acetabulum\n",
      "            Eukaryota; Viridiplantae; Chlorophyta; Ulvophyceae; Dasycladales;\n",
      "            Polyphysaceae; Acetabularia.\n",
      "REFERENCE   1  (bases 1 to 374)\n",
      "  AUTHORS   Henry,I.M., Wilkinson,M.D., Hernandez,J.M., Schwarz-Sommer,Z.,\n",
      "            Grotewold,E. and Mandoli,D.F.\n",
      "  TITLE     Comparison of ESTs from juvenile and adult phases of the giant\n",
      "            unicellular green alga Acetabularia acetabulum\n",
      "  JOURNAL   BMC Plant Biol. 4 (1), 3 (2004)\n",
      "   PUBMED   15070428\n",
      "COMMENT     Contact: Henry, IH and Mandoli, DF\n",
      "            Department of Biology\n",
      "            University of Washington\n",
      "            Box 355325, 1521 NE Pacific Street, Seattle, WA, 98195-5325\n",
      "            Tel: 206 543 8917\n",
      "            Fax: 206 685 1728\n",
      "            Email: mandoli@u.washington.edu\n",
      "            possibly in cluster containing EST(s) : A397 / J481 / J525 / J537 /\n",
      "            J553 / j028-f / j042 / j066 / j088-f / j126-f / j185-f / j233 /\n",
      "            j408r.ab1 / j425r.ab1\n",
      "            Insert Length: 374   Std Error: 0.00\n",
      "            POLYA=Yes.\n",
      "FEATURES             Location/Qualifiers\n",
      "     source          1..374\n",
      "                     /organism=\"Acetabularia acetabulum\"\n",
      "                     /mol_type=\"mRNA\"\n",
      "                     /strain=\"Aa0005 (=Ladenburg 17)\"\n",
      "                     /db_xref=\"taxon:35845\"\n",
      "                     /tissue_type=\"whole unicells\"\n",
      "                     /clone_lib=\"SAMN00173644 Juvenile-specific\"\n",
      "                     /dev_stage=\"juvenile\"\n",
      "                     /note=\"To reveal differences in gene expression between\n",
      "                     adult and juvenile phases of Acetabularia acetabulum, we\n",
      "                     constructed two subtracted EST libraries using suppressive\n",
      "                     subtractive hybridization. One library is putatively\n",
      "                     enriched in adult-specific transcripts and the other is\n",
      "                     putatively enriched in juvenile-specific transcripts. The\n",
      "                     two mRNA samples used to create these libraries were\n",
      "                     isolated from Acetabularia acetabulum whole late adult and\n",
      "                     early reproductive unicells and from whole Acetabularia\n",
      "                     acetabulum juvenile unicells. This EST was obtained from\n",
      "                     the juvenile-specific library.\"\n",
      "ORIGIN      \n",
      "        1 cattttttac acattcttgc cttgaaagtt aagttgccaa caatttgatt taacattgct\n",
      "       61 tcgtttttct tcgaagcgtt aaaaacaaat tagcaaaaca cgcaatcgct gtgtttgcca\n",
      "      121 cttagtcttc ttcgtccctc ggtatctgat aagattcggt gcaggaatat tgacctgctt\n",
      "      181 gccatcgact acgccgttcg gcctcgcctt aggtcctgac tgacccctcg cggacgaacc\n",
      "      241 ttgcgaagga attcttaggt tttagggcat tggattctta ccaatgtttt cgttactcaa\n",
      "      301 gcctacattc tcacttccgc cttgtccacc aaagctcacg ctttagcttc acccataggc\n",
      "      361 ggaacacctg cccg\n",
      "//\n",
      "\n",
      "LOCUS       CF259227                 542 bp    mRNA    linear   EST 04-FEB-2011\n",
      "DEFINITION  j409cons.ab1 Juvenile-specific Acetabularia acetabulum cDNA, mRNA\n",
      "            sequence.\n",
      "ACCESSION   CF259227\n",
      "VERSION     CF259227.1\n",
      "DBLINK      BioSample: SAMN00173644\n",
      "KEYWORDS    EST.\n",
      "SOURCE      Acetabularia acetabulum (mermaid's wine glass)\n",
      "  ORGANISM  Acetabularia acetabulum\n",
      "            Eukaryota; Viridiplantae; Chlorophyta; Ulvophyceae; Dasycladales;\n",
      "            Polyphysaceae; Acetabularia.\n",
      "REFERENCE   1  (bases 1 to 542)\n",
      "  AUTHORS   Henry,I.M., Wilkinson,M.D., Hernandez,J.M., Schwarz-Sommer,Z.,\n",
      "            Grotewold,E. and Mandoli,D.F.\n",
      "  TITLE     Comparison of ESTs from juvenile and adult phases of the giant\n",
      "            unicellular green alga Acetabularia acetabulum\n",
      "  JOURNAL   BMC Plant Biol. 4 (1), 3 (2004)\n",
      "   PUBMED   15070428\n",
      "COMMENT     Contact: Henry, IH and Mandoli, DF\n",
      "            Department of Biology\n",
      "            University of Washington\n",
      "            Box 355325, 1521 NE Pacific Street, Seattle, WA, 98195-5325\n",
      "            Tel: 206 543 8917\n",
      "            Fax: 206 685 1728\n",
      "            Email: mandoli@u.washington.edu\n",
      "            possibly in cluster containing EST(s) : J501 / J502 / J565 / j021-f\n",
      "            / j227-f / j387r.ab1 / j409cons.ab1\n",
      "            Insert Length: 542   Std Error: 0.00\n",
      "            POLYA=Yes.\n",
      "FEATURES             Location/Qualifiers\n",
      "     source          1..542\n",
      "                     /organism=\"Acetabularia acetabulum\"\n",
      "                     /mol_type=\"mRNA\"\n",
      "                     /strain=\"Aa0005 (=Ladenburg 17)\"\n",
      "                     /db_xref=\"taxon:35845\"\n",
      "                     /tissue_type=\"whole unicells\"\n",
      "                     /clone_lib=\"SAMN00173644 Juvenile-specific\"\n",
      "                     /dev_stage=\"juvenile\"\n",
      "                     /note=\"To reveal differences in gene expression between\n",
      "                     adult and juvenile phases of Acetabularia acetabulum, we\n",
      "                     constructed two subtracted EST libraries using suppressive\n",
      "                     subtractive hybridization. One library is putatively\n",
      "                     enriched in adult-specific transcripts and the other is\n",
      "                     putatively enriched in juvenile-specific transcripts. The\n",
      "                     two mRNA samples used to create these libraries were\n",
      "                     isolated from Acetabularia acetabulum whole late adult and\n",
      "                     early reproductive unicells and from whole Acetabularia\n",
      "                     acetabulum juvenile unicells. This EST was obtained from\n",
      "                     the juvenile-specific library.\"\n",
      "ORIGIN      \n",
      "        1 acaccgctgc tgccgcagaa gcttttgcgg ctgccagtgc tgttgcacaa ggaatgggag\n",
      "       61 gagcaagtgc aaatgccttt gctgaggcaa ttgcccaagc atcatcttct ggaggaaaat\n",
      "      121 gcgacgctgt cgcaatagct attgcagaag ctgttgcggt cgcagaaggc gatggcgcag\n",
      "      181 ctgttgccca agctgctgct gctgcataag ctcaagctga gtgcttaaaa aaacgttcct\n",
      "      241 tgtcctttga tgaaatgcct tggaattgac aaatcagtgt gtgaaggaaa ccactctttg\n",
      "      301 ggatagaaat ataccgataa atttggtaag gaattctact acgcaggtgt ttgcgaaatg\n",
      "      361 gaacatagcg tttgggagtc catgaccttt ggcaatgggt aaggtattgt tattgntgaa\n",
      "      421 atattttgtg aatnatgtaa attaatagna ttnttttatt nttttttata nggngtttna\n",
      "      481 aatctgttta tgcatttgta gtaagtaagt aatgtaattg aaattcttaa atttggtgcn\n",
      "      541 cc\n",
      "//\n",
      "\n",
      "LOCUS       CF259226                 260 bp    mRNA    linear   EST 04-FEB-2011\n",
      "DEFINITION  a022-f Adult-specific Acetabularia acetabulum cDNA, mRNA sequence.\n",
      "ACCESSION   CF259226\n",
      "VERSION     CF259226.1\n",
      "DBLINK      BioSample: SAMN00173643\n",
      "KEYWORDS    EST.\n",
      "SOURCE      Acetabularia acetabulum (mermaid's wine glass)\n",
      "  ORGANISM  Acetabularia acetabulum\n",
      "            Eukaryota; Viridiplantae; Chlorophyta; Ulvophyceae; Dasycladales;\n",
      "            Polyphysaceae; Acetabularia.\n",
      "REFERENCE   1  (bases 1 to 260)\n",
      "  AUTHORS   Henry,I.M., Wilkinson,M.D., Hernandez,J.M., Schwarz-Sommer,Z.,\n",
      "            Grotewold,E. and Mandoli,D.F.\n",
      "  TITLE     Comparison of ESTs from juvenile and adult phases of the giant\n",
      "            unicellular green alga Acetabularia acetabulum\n",
      "  JOURNAL   BMC Plant Biol. 4 (1), 3 (2004)\n",
      "   PUBMED   15070428\n",
      "COMMENT     Contact: Henry, IH and Mandoli, DF\n",
      "            Department of Biology\n",
      "            University of Washington\n",
      "            Box 355325, 1521 NE Pacific Street, Seattle, WA, 98195-5325\n",
      "            Tel: 206 543 8917\n",
      "            Fax: 206 685 1728\n",
      "            Email: mandoli@u.washington.edu\n",
      "            Insert Length: 260   Std Error: 0.00\n",
      "            POLYA=Yes.\n",
      "FEATURES             Location/Qualifiers\n",
      "     source          1..260\n",
      "                     /organism=\"Acetabularia acetabulum\"\n",
      "                     /mol_type=\"mRNA\"\n",
      "                     /strain=\"Aa0005 (=Ladenburg 17)\"\n",
      "                     /db_xref=\"taxon:35845\"\n",
      "                     /tissue_type=\"whole unicells\"\n",
      "                     /clone_lib=\"SAMN00173643 Adult-specific\"\n",
      "                     /dev_stage=\"late adult and early reproductive\"\n",
      "                     /note=\"To reveal differences in gene expression between\n",
      "                     adult and juvenile phases of Acetabularia acetabulum, we\n",
      "                     constructed two subtracted EST libraries using suppressive\n",
      "                     subtractive hybridization. One library is putatively\n",
      "                     enriched in adult-specific transcripts and the other is\n",
      "                     putatively enriched in juvenile-specific transcripts. The\n",
      "                     two mRNA samples used to create these libraries were\n",
      "                     isolated from Acetabularia acetabulum whole late adult and\n",
      "                     early reproductive unicells and from whole Acetabularia\n",
      "                     acetabulum juvenile unicells. This EST was obtained from\n",
      "                     the adult-specific library.\"\n",
      "ORIGIN      \n",
      "        1 actaacccat ccttgaaaat gcgaatttca taatacaaat tgtataattg ttaattgtag\n",
      "       61 tgacgtgcaa atttgatgaa ttgtggttta cagttaccta atgtagatgc attgattaag\n",
      "      121 aagtttgatc cattgataaa gagatgccga taatggtctc gttttgttat aatatgttta\n",
      "      181 cagtcctaat aaacaatata gttatccaag gacccttacc agtttaccat tactcaaatc\n",
      "      241 aatctaagtt tatgaataca\n",
      "//\n"
     ]
    }
   ],
   "source": [
    "require 'bio'\n",
    "\n",
    "ncbi = Bio::NCBI::REST.new\n",
    "max = 5\n",
    "ids = ncbi.esearch('\"Wilkinson MD\"[AUTH]', {\"db\"=>\"nucleotide\", \"rettype\"=>\"gb\", \"retmax\" => max})\n",
    "idlist = ids.join(\",\")\n",
    "puts idlist\n",
    "puts \"\\n\\nRecords are\\n\\n\"\n",
    "records = ncbi.efetch(idlist, {\"db\"=>\"nucleotide\", \"rettype\"=>\"gb\", \"retmode\"=>\"text\", \"retmax\"=> max})\n",
    "puts records"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Better use of PubMed\n",
    "\n",
    "Most reference managers (EndNote, Mendeley, and even LaTex) are able to work with files in BibTex format.  Let's do one last BioRuby call to PubMed to retrieve articles in that format.\n",
    "\n",
    "Our task is to get the abstracts for the nucleotide sequence files that were discovered in the search above.  This is called a \"Link\".  Earlier in this Lesson, we looked at the output from the EInfo tool - it told you what FIELDs you could query for each database, but also what LINKS existed between one database and another.  Let's look at that again - the EInfo about the nuccore database:\n",
    "\n",
    "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/einfo.fcgi?db=nuccore\n",
    "\n",
    "Near the end of that page you will find:\n",
    "\n",
    "    <Link>\n",
    "        <Name>nuccore_pubmed</Name>\n",
    "        <Menu>PubMed Links</Menu>\n",
    "        <Description>PubMed articles cited by Nucleotide sequence record</Description>\n",
    "        <DbTo>pubmed</DbTo>\n",
    "    </Link>\n",
    "\n",
    "This describes that there is a link between nuccore (the sequence database) and pubmed (the publications database).  The tool that we need to find the links between the list of IDs for Database A and the list of IDs for Database B is the ELink tool\n",
    "\n",
    "http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi\n",
    "\n",
    "It works like all other EUtils; however, there is no BioRuby wrapper for this one, so we must do the hard work ourselves.  To see some example code in Perl go to:  https://www.ncbi.nlm.nih.gov/books/NBK25498/#chapter3.Application_4_Finding_unique_se.  We will now do the same thing in Ruby.\n",
    "\n",
    "\n",
    "### ELink parameters are:  \n",
    "* dbfrom\n",
    "* db  (i.e. which db are you linking TO)\n",
    "* id = ...list of ids...\n",
    "\n",
    "So, like all other EUtils, we can construct a URL that calls the ELink service:\n",
    "\n",
    "http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=nuccore;db=pubmed;id=732693005,732693505,1129728243,1129728241,1129728239\n",
    "\n",
    "If you click on that, you will see the XML document that contains the PubMed IDs that correspond to the nucleotide sequence IDs we sent to the **elink** tool.  They are separated into \"Link Sets\", corresponding to different subsections of pubmed (e.g. **nuccore_pubmed_refseq**).  \n",
    "\n",
    "HOWEVER:  If you read a little bit further in the documentation (for all EUtilities!), the instructions say \"use POST when you have many IDs (>200)\" \n",
    "\n",
    ">**Required Parameter  Used only when input is from a UID list**\n",
    "\n",
    ">**id**\n",
    "\n",
    ">UID list. Either a single UID or a comma-delimited list of UIDs may be provided. All of the UIDs must be from the database specified by dbfrom. There is no set maximum for the number of UIDs that can be passed to ELink, but **if more than about 200 UIDs are to be provided, the request should be made using the HTTP POST method.**\n",
    "\n",
    "\n",
    "What does \"HTTP POST\" mean?   We will discuss it in detail in a later lecture, but for now, you only need to understand that it is another way of sending information to a Web server.  Using HTTP GET, you put the information at the end of the URL (<code>scriptname?a=b;c=d;...</code>).   With HTTP POST you send it in the \"body\" of a message.  \n",
    "\n",
    "It is important to learn how to create an HTTP POST request, so even though we _can_ use the URL above (because we only have 5 ID numbers), we should look at and understand the other solution.  An example of an HTTP POST is below, where we are sending a request to the ELink service of NCBI:\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "1676439675,1388704174,33527965,33527963,33527961\n",
      "<?xml version=\"1.0\" encoding=\"UTF-8\" ?>\n",
      "<!DOCTYPE eLinkResult PUBLIC \"-//NLM//DTD elink 20101123//EN\" \"https://eutils.ncbi.nlm.nih.gov/eutils/dtd/20101123/elink.dtd\">\n",
      "<eLinkResult>\n",
      "\n",
      "  <LinkSet>\n",
      "    <DbFrom>nuccore</DbFrom>\n",
      "    <IdList>\n",
      "      <Id>1676439675</Id>\n",
      "      <Id>1388704174</Id>\n",
      "      <Id>33527965</Id>\n",
      "      <Id>33527963</Id>\n",
      "      <Id>33527961</Id>\n",
      "    </IdList>\n",
      "    <LinkSetDb>\n",
      "      <DbTo>pubmed</DbTo>\n",
      "      <LinkName>nuccore_pubmed</LinkName>\n",
      "      \n",
      "        <Link>\n",
      "\t\t\t\t<Id>30789691</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>29604224</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>29518281</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>29307555</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>25896417</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>20610895</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>20301317</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>19056867</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>15498460</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>15070428</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>12421765</Id>\n",
      "\t\t\t</Link>\n",
      "      \n",
      "    </LinkSetDb>\n",
      "    \n",
      "    <LinkSetDb>\n",
      "      <DbTo>pubmed</DbTo>\n",
      "      <LinkName>nuccore_pubmed_refseq</LinkName>\n",
      "      \n",
      "        <Link>\n",
      "\t\t\t\t<Id>30789691</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>30258100</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>29604224</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>29568061</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>29518281</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>29507755</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>29307555</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>26344197</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>25896417</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>21873635</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>21182205</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>20610895</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>20379614</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>20301317</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>19056867</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>16710414</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>15498460</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>15489334</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>14702039</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>12477932</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>12421765</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>9455484</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>9110174</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>8619474</Id>\n",
      "\t\t\t</Link>\n",
      "      \n",
      "    </LinkSetDb>\n",
      "    <LinkSetDb>\n",
      "      <DbTo>pubmed</DbTo>\n",
      "      <LinkName>nuccore_pubmed_weighted</LinkName>\n",
      "      \n",
      "        <Link>\n",
      "\t\t\t\t<Id>30789691</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>30258100</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>29604224</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>29568061</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>29518281</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>29507755</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>29307555</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>26344197</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>25896417</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>21873635</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>21182205</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>20610895</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>20379614</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>20301317</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>19056867</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>16710414</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>15498460</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>15489334</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>14702039</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>14681893</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>12477932</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>12421765</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>11960835</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>9455484</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>9110174</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>8619474</Id>\n",
      "\t\t\t</Link>\n",
      "      \n",
      "    </LinkSetDb>\n",
      "  </LinkSet>\n",
      "</eLinkResult>\n",
      "\n"
     ]
    }
   ],
   "source": [
    "require 'net/http'\n",
    "require 'bio'\n",
    "\n",
    "ncbi = Bio::NCBI::REST.new\n",
    "max = 5\n",
    "ids = ncbi.esearch('\"Wilkinson MD\"[AUTH]', {\"db\"=>\"nucleotide\", \"rettype\"=>\"gb\", \"retmax\" => max})\n",
    "idlist = ids.join(\",\")\n",
    "puts idlist\n",
    "\n",
    "# prepare for HTTP POST - this is very similar to what is in the \"fetch\" routine from your other scripts.\n",
    "\n",
    "elink = URI(\"https://eutils.ncbi.nlm.nih.gov:443/entrez/eutils/elink.fcgi\")\n",
    "request = Net::HTTP::Post.new(elink)\n",
    "https = Net::HTTP.new(elink.host,elink.port)\n",
    "https.use_ssl = true  # tell the HTTP connection to use ecryption (SSL = Secure Socket Layer)\n",
    "\n",
    "# here is where we create the content of the body\n",
    "\n",
    "request.set_form_data(\n",
    "  'email' => 'markw@illuminae.com',\n",
    "  'dbfrom' => 'nuccore', \n",
    "  'db' => 'pubmed',\n",
    "  'retmode' => 'xml',\n",
    "  'id' => idlist)\n",
    "\n",
    "output = https.request(request)   # make the request (we really should check if it is successful)\n",
    "puts output.body"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "\n",
    "#  Parsing XML\n",
    "\n",
    "In an earlier lesson we learned how to extract information from JSON.  In the example above the output of the EUtils service is in XML format, so we now need to learn how to extract information from XML.\n",
    "\n",
    "The most popular Ruby library for XML parsing is called 'nokogiri' (this translates to English as \"saw\" and Spanish as \"sierra\" como \"sierra circular\").  So that's what we will use.  The XML output from EUtils is quite simple (we will see more complex examples later in the course!) so let's use it as our learning example\n",
    "\n",
    "\n",
    "    <eLinkResult>\n",
    "        <LinkSet>\n",
    "            <LinkSetDb>\n",
    "                  <DbTo>pubmed</DbTo>\n",
    "                  <LinkName>nuccore_pubmed</LinkName>\n",
    "\n",
    "                  <Link>\n",
    "                        <Id>25896417</Id>\n",
    "                  </Link>\n",
    "                  <Link>\n",
    "                        <Id>20610895</Id>\n",
    "                  </Link>\n",
    "                  <Link>\n",
    "                        <Id>19056867</Id>\n",
    "                  </Link>\n",
    "                  <Link>\n",
    "                        <Id>15498460</Id>\n",
    "                  </Link>\n",
    "                  <Link>\n",
    "                        <Id>12421765</Id>\n",
    "                  </Link>\n",
    "\n",
    "            </LinkSetDb>\n",
    "        </LinkSet>\n",
    "     <eLinkResut>\n",
    "\n",
    "This is the relevant section we want to extract IDs from.  XML documents are \"hierarchical\", sub-sections are enclosed in parent sections in the pattern:\n",
    "        \n",
    "        <A> <B> data </B>  </A>\n",
    "\n",
    "Given this structure, it is normal to talk about the \"path\" to the data.  In this case, the data is A-->B-->data.  \n",
    "\n",
    "In the output from ELink, the path is:  LinkSetDb-->Link-->Id-->data\n",
    "\n",
    "These \"paths\" are used in Ruby to define the piece of data we want to extract.  For example:\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PubMed = 25896417\n",
      "PubMed = 20610895\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<Link><Link>"
      ],
      "text/plain": [
       "[#<Nokogiri::XML::Element:0x1ea765c name=\"Link\" children=[#<Nokogiri::XML::Text:0x1eac3a0 \"\\n                      \">, #<Nokogiri::XML::Element:0x1ea7454 name=\"Id\" children=[#<Nokogiri::XML::Text:0x1eadf5c \"25896417\">]>, #<Nokogiri::XML::Text:0x1eadd18 \"\\n                \">]>, #<Nokogiri::XML::Element:0x1ea69b4 name=\"Link\" children=[#<Nokogiri::XML::Text:0x1ead994 \"\\n                      \">, #<Nokogiri::XML::Element:0x1ea6860 name=\"Id\" children=[#<Nokogiri::XML::Text:0x1ead714 \"20610895\">]>, #<Nokogiri::XML::Text:0x1ead570 \"\\n                \">]>]"
      ]
     },
     "execution_count": 6,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "require 'nokogiri'\n",
    "\n",
    "xml = <<EOF\n",
    "    <eLinkResult>\n",
    "        <LinkSet>\n",
    "\n",
    "            <LinkSetDb>\n",
    "                <Link>\n",
    "                      <Id>25896417</Id>\n",
    "                </Link>\n",
    "                <Link>\n",
    "                      <Id>20610895</Id>\n",
    "                </Link>\n",
    "            </LinkSetDb>\n",
    "        </LinkSet>\n",
    "    <eLinkResut>\n",
    "EOF\n",
    "\n",
    "doc = Nokogiri::XML(xml)\n",
    "doc.xpath('//LinkSet/LinkSetDb/Link').each do |node|    # the \"path\" to the data, starts with '//'\n",
    "  puts \"PubMed = \" + node.at_xpath('Id').content\n",
    "end"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "#  Putting it all together!\n",
    "\n",
    "Finally, we will create a script that:\n",
    "\n",
    "* retrieves the nucleotide database ids\n",
    "* retrieves the cross-reference links to PubMed\n",
    "* extracts the PubMed IDs\n",
    "* retrieves the PubMed Abstracts for those IDs\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PUBMED IDs are: [\"30789691\", \"29604224\", \"29518281\", \"29307555\", \"25896417\", \"20610895\", \"20301317\", \"19056867\", \"15498460\", \"15070428\", \"12421765\", \"30789691\", \"30258100\", \"29604224\", \"29568061\", \"29518281\", \"29507755\", \"29307555\", \"26344197\", \"25896417\", \"21873635\", \"21182205\", \"20610895\", \"20379614\", \"20301317\", \"19056867\", \"16710414\", \"15498460\", \"15489334\", \"14702039\", \"12477932\", \"12421765\", \"9455484\", \"9110174\", \"8619474\", \"30789691\", \"30258100\", \"29604224\", \"29568061\", \"29518281\", \"29507755\", \"29307555\", \"26344197\", \"25896417\", \"21873635\", \"21182205\", \"20610895\", \"20379614\", \"20301317\", \"19056867\", \"16710414\", \"15498460\", \"15489334\", \"14702039\", \"14681893\", \"12477932\", \"12421765\", \"11960835\", \"9455484\", \"9110174\", \"8619474\"]\n",
      "1. VPS13D Movement Disorder.\n",
      "\n",
      "Meijer IA(1).\n",
      "In: Adam MP, Ardinger HH, Pagon RA, Wallace SE, Bean LJH, Stephens K, Amemiya A, \n",
      "editors. GeneReviews [Internet]. Seattle (WA): University of Washington,\n",
      "Seattle; 1993-2019.\n",
      "2019 Feb 21.\n",
      "\n",
      "Author information: \n",
      "(1)CHU Sainte-Justine, Universit de Montral, Montral, Qubec, Canada\n",
      "\n",
      "CLINICAL CHARACTERISTICS: VPS13D movement disorder is a hyperkinetic movement\n",
      "disorder (dystonia, chorea, and/or ataxia) of variable age of onset that can be\n",
      "associated with developmental delay. Onset ranges from birth to adulthood.\n",
      "Individuals can present in childhood with motor delays and gait instability.\n",
      "Cognitive impairment ranging from mild intellectual disability to developmental\n",
      "delay has been reported, and several individuals have normal cognitive function. \n",
      "Individuals have also presented as young adults with gait difficulties caused by \n",
      "spastic ataxia or ataxia. In addition to gait ataxia, affected individuals had\n",
      "limb ataxia, dysarthria, and eye movement abnormalities (macro-saccadic\n",
      "oscillations, nystagmus, and saccadic pursuit). Additional features reported in\n",
      "some individuals include peripheral neuropathy and/or seizures. The disorder\n",
      "progresses to spastic ataxia or generalized dystonia, which can lead to loss of\n",
      "independent ambulation.\n",
      "DIAGNOSIS: The diagnosis of VPS13D movement disorder is established in a proband \n",
      "by identification of biallelic pathogenic variants in VPS13D on molecular genetic\n",
      "testing.\n",
      "MANAGEMENT: Treatment of manifestations: Standard treatment for seizures;\n",
      "spasticity treatments include baclofen, tizanidine, benzodiazepines, dantrolene\n",
      "sodium, gabapentin, botulinum toxin injections, and intrathecal baclofen;\n",
      "treatment options for dystonia include trihexyphenidyl, botulinum toxin\n",
      "injections, benzodiazepines, baclofen, and levodopa. A multidisciplinary team\n",
      "including occupational and physical therapists and a physiatrist is important;\n",
      "supportive developmental therapies should be provided as needed. Surveillance:\n",
      "Monitor for urinary urgency, contractures, and seizures; evaluate fall risk and\n",
      "gait stability; monitor developmental progress and educational needs.\n",
      "GENETIC COUNSELING: VPS13D movement disorder is inherited in an autosomal\n",
      "recessive manner. At conception, each sib of an affected individual has a 25%\n",
      "chance of being affected, a 50% chance of being an asymptomatic carrier, and a\n",
      "25% chance of being unaffected and not a carrier. Carrier testing for at-risk\n",
      "relatives and prenatal testing for pregnancies at increased risk are possible if \n",
      "the VPS13D pathogenic variants in the family are known.\n",
      "\n",
      "PMID: 30789691 \n",
      "\n",
      "\n",
      "2. Ann Neurol. 2018 Jun;83(6):1075-1088. doi: 10.1002/ana.25220. Epub 2018 Jun 30.\n",
      "\n",
      "Mutations in VPS13D lead to a new recessive ataxia with spasticity and\n",
      "mitochondrial defects.\n",
      "\n",
      "Seong E(1), Insolera R(2), Dulovic M(3), Kamsteeg EJ(4), Trinh J(3), Brggemann\n",
      "N(5), Sandford E(1), Li S(1), Ozel AB(6), Li JZ(6)(7), Jewett T(8), Kievit\n",
      "AJA(9), Mnchau A(3), Shakkottai V(10), Klein C(3), Collins CA(2), Lohmann K(3), \n",
      "van de Warrenburg BP(11), Burmeister M(1)(6)(7)(12).\n",
      "\n",
      "Author information: \n",
      "(1)Molecular and Behavioral Neuroscience Institute, University of Michigan, Ann\n",
      "Arbor, MI.\n",
      "(2)Department of Molecular, Cellular, and Developmental Biology, University of\n",
      "Michigan, Ann Arbor, MI.\n",
      "(3)Institute of Neurogenetics, University of Lbeck, Lbeck, Germany.\n",
      "(4)Department of Human Genetics, Radboud University Medical Centre, Nijmegen, the\n",
      "Netherlands.\n",
      "(5)Department of Neurology, University of Lbeck, Lbeck, Germany.\n",
      "(6)Department of Human Genetics, University of Michigan, Ann Arbor, MI.\n",
      "(7)Department of Computational Medicine and Bioinformatics, University of\n",
      "Michigan, Ann Arbor, MI.\n",
      "(8)Department of Pediatrics, Section on Medical Genetics, Wake Forest School of\n",
      "Medicine, Winston-Salem, NC.\n",
      "(9)Department of Clinical Genetics, Erasmus MC, Rotterdam, the Netherlands.\n",
      "(10)Departments of Neurology and Molecular and Integrative Physiology, University\n",
      "of Michigan, Ann Arbor, MI.\n",
      "(11)Department of Neurology, Donders Institute for Brain, Cognition, and\n",
      "Behavior, Radboud University Medical Centre, Nijmegen, the Netherlands.\n",
      "(12)Department of Psychiatry, University of Michigan, Ann Arbor, MI.\n",
      "\n",
      "Comment in\n",
      "    Ann Neurol. 2018 Jun;83(6):1072-1074.\n",
      "\n",
      "OBJECTIVE: To identify novel causes of recessive ataxias, including\n",
      "spinocerebellar ataxia with saccadic intrusions, spastic ataxias, and spastic\n",
      "paraplegia.\n",
      "METHODS: In an international collaboration, we independently performed exome\n",
      "sequencing in 7 families with recessive ataxia and/or spastic paraplegia. To\n",
      "evaluate the role of VPS13D mutations, we evaluated a Drosophila knockout model\n",
      "and investigated mitochondrial function in patient-derived fibroblast cultures.\n",
      "RESULTS: Exome sequencing identified compound heterozygous mutations in VPS13D on\n",
      "chromosome 1p36 in all 7 families. This included a large family with 5 affected\n",
      "siblings with spinocerebellar ataxia with saccadic intrusions (SCASI), or\n",
      "spinocerebellar ataxia, recessive, type 4 (SCAR4). Linkage to chromosome 1p36 was\n",
      "found in this family with a logarithm of odds score of 3.1. The phenotypic\n",
      "spectrum in our 12 patients was broad. Although most presented with ataxia,\n",
      "additional or predominant spasticity was present in 5 patients. Disease onset\n",
      "ranged from infancy to 39 years, and symptoms were slowly progressive and\n",
      "included loss of independent ambulation in 5. All but 2 patients carried a\n",
      "loss-of-function (nonsense or splice site) mutation on one and a missense\n",
      "mutation on the other allele. Knockdown or removal of Vps13D in Drosophila\n",
      "neurons led to changes in mitochondrial morphology and impairment in\n",
      "mitochondrial distribution along axons. Patient fibroblasts showed altered\n",
      "morphology and functionality including reduced energy production.\n",
      "INTERPRETATION: Our study demonstrates that compound heterozygous mutations in\n",
      "VPS13D cause movement disorders along the ataxia-spasticity spectrum, making\n",
      "VPS13D the fourth VPS13 paralog involved in neurological disorders. Ann Neurol\n",
      "2018.\n",
      "\n",
      " 2018 American Neurological Association.\n",
      "\n",
      "DOI: 10.1002/ana.25220 \n",
      "PMCID: PMC6105379\n",
      "PMID: 29604224  [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "3. Ann Neurol. 2018 Jun;83(6):1089-1095. doi: 10.1002/ana.25204. Epub 2018 Apr 10.\n",
      "\n",
      "Recessive mutations in VPS13D cause childhood onset movement disorders.\n",
      "\n",
      "Gauthier J(1), Meijer IA(2), Lessel D(3), Mencacci NE(4), Krainc D(4), Hempel\n",
      "M(3), Tsiakas K(5), Prokisch H(6)(7), Rossignol E(2), Helm MH(8), Rodan LH(9),\n",
      "Karamchandani J(10), Carecchio M(11), Lubbe SJ(4), Telegrafi A(12), Henderson\n",
      "LB(12), Lorenzo K(13), Wallace SE(14), Glass IA(14), Hamdan FF(1), Michaud\n",
      "JL(15), Rouleau GA(16), Campeau PM(15).\n",
      "\n",
      "Author information: \n",
      "(1)Molecular Diagnostic Laboratory and Division of Medical Genetics, Department\n",
      "of Pediatrics, Saint Justine University Hospital Center, Montreal, Canada.\n",
      "(2)Department of Neuroscience, University of Montreal, Montreal, Canada.\n",
      "(3)Institute of Human Genetics, University Medical Center Hamburg-Eppendorf,\n",
      "Hamburg, Germany.\n",
      "(4)Department of Neurology, Northwestern University, Feinberg School of Medicine,\n",
      "Chicago, IL.\n",
      "(5)Department of Pediatrics, University Medical Center Hamburg-Eppendorf,\n",
      "Hamburg, Germany.\n",
      "(6)Institute of Human Genetics, Helmholtz Center Munich, Neuherberg, Germany.\n",
      "(7)Institute of Human Genetics, Technical University Munich, Munich, Germany.\n",
      "(8)Maine Medical Partners Pediatric Specialty Care, Portland, ME.\n",
      "(9)Department of Neurology, Boston Children's Hospital, Boston, MA.\n",
      "(10)Department of Pathology, McGill University, Montreal Neurological Institute, \n",
      "Montreal, Canada.\n",
      "(11)Molecular Neurogenetics Unit, Institute for Research and Health Care (IRCCS) \n",
      "Foundation Carlo Besta Neurological Institute, Milan, Italy.\n",
      "(12)GeneDx, Gaithersburg, MD.\n",
      "(13)Kadlec Clinic Genetic Counseling, Richland, WA.\n",
      "(14)Division of Genetic Medicine, Department of Pediatrics, Seattle Children's\n",
      "Hospital and University of Washington, Seattle, WA.\n",
      "(15)Department of Pediatrics, Saint Justine University Hospital Center and\n",
      "University of Montreal, Montreal, Canada.\n",
      "(16)Montreal Neurological Institute, Department of Neurology and Neurosurgery,\n",
      "McGill University, Montreal, Canada.\n",
      "\n",
      "Comment in\n",
      "    Ann Neurol. 2018 Jun;83(6):1072-1074.\n",
      "\n",
      "VPS13 protein family members VPS13A through VPS13C have been associated with\n",
      "various recessive movement disorders. We describe the first disease association\n",
      "of rare recessive VPS13D variants including frameshift, missense, and partial\n",
      "duplication mutations with a novel complex, hyperkinetic neurological disorder.\n",
      "The clinical features include developmental delay, a childhood onset movement\n",
      "disorder (chorea, dystonia, or tremor), and progressive spastic ataxia or\n",
      "paraparesis. Characteristic brain magnetic resonance imaging shows basal ganglia \n",
      "or diffuse white matter T2 hyperintensities as seen in Leigh syndrome and\n",
      "choreoacanthocytosis. Muscle biopsy in 1 case showed mitochondrial aggregates and\n",
      "lipidosis, suggesting mitochondrial dysfunction. These findings underline the\n",
      "importance of the VPS13 complex in neurological diseases and a possible role in\n",
      "mitochondrial function. Ann Neurol 2018;83:1089-1095.\n",
      "\n",
      " 2018 American Neurological Association.\n",
      "\n",
      "DOI: 10.1002/ana.25204 \n",
      "PMID: 29518281  [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "4. Curr Biol. 2018 Jan 22;28(2):287-295.e6. doi: 10.1016/j.cub.2017.11.064. Epub\n",
      "2018 Jan 4.\n",
      "\n",
      "Vps13D Encodes a Ubiquitin-Binding Protein that Is Required for the Regulation of\n",
      "Mitochondrial Size and Clearance.\n",
      "\n",
      "Anding AL(1), Wang C(2), Chang TK(1), Sliter DA(2), Powers CM(1), Hofmann K(3),\n",
      "Youle RJ(2), Baehrecke EH(4).\n",
      "\n",
      "Author information: \n",
      "(1)Department of Molecular, Cell and Cancer Biology, University of Massachusetts \n",
      "Medical School, Worcester, MA 01605, USA.\n",
      "(2)Biochemistry Section, Surgical Neurology Branch, National Institute of\n",
      "Neurological Disorders and Stroke, NIH, Bethesda, MD 20892, USA.\n",
      "(3)Institute for Genetics, University of Cologne, 50674 Cologne, Germany.\n",
      "(4)Department of Molecular, Cell and Cancer Biology, University of Massachusetts \n",
      "Medical School, Worcester, MA 01605, USA. Electronic address:\n",
      "eric.baehrecke@umassmed.edu.\n",
      "\n",
      "Comment in\n",
      "    Curr Biol. 2018 Jan 22;28(2):R66-R68.\n",
      "\n",
      "The clearance of mitochondria by autophagy, mitophagy, is important for cell and \n",
      "organism health [1], and known to be regulated by ubiquitin. During Drosophila\n",
      "intestine development, cells undergo a dramatic reduction in cell size and\n",
      "clearance of mitochondria that depends on autophagy, the E1 ubiquitin-activating \n",
      "enzyme Uba1, and ubiquitin [2]. Here we screen a collection of putative\n",
      "ubiquitin-binding domain-encoding genes for cell size reduction and autophagy\n",
      "phenotypes. We identify the endosomal sorting complex required for transport\n",
      "(ESCRT) components TSG101 and Vps36, as well as the novel gene Vps13D. Vps13D is \n",
      "an essential gene that is necessary for autophagy, mitochondrial size, and\n",
      "mitochondrial clearance in Drosophila. Interestingly, a similar mitochondrial\n",
      "phenotype is observed in VPS13D mutant human cells. The ubiquitin-associated\n",
      "(UBA) domain of Vps13D binds K63 ubiquitin chains, and mutants lacking the UBA\n",
      "domain have defects in mitochondrial size and clearance and exhibit\n",
      "semi-lethality, highlighting the importance of Vps13D ubiquitin binding in both\n",
      "mitochondrial health and development. VPS13D mutant cells possess phosphorylated \n",
      "DRP1 and mitochondrial fission factor (MFF) as well as DRP1 association with\n",
      "mitochondria, suggesting that VPS13D functions downstream of these known\n",
      "regulators of mitochondrial fission. In addition, the large Vps13D mitochondrial \n",
      "and cell size phenotypes are suppressed by decreased mitochondrial fusion gene\n",
      "function. Thus, these results provide a previously unknown link between\n",
      "ubiquitin, mitochondrial size regulation, and autophagy.\n",
      "\n",
      "Copyright  2017 Elsevier Ltd. All rights reserved.\n",
      "\n",
      "DOI: 10.1016/j.cub.2017.11.064 \n",
      "PMCID: PMC5787036\n",
      "PMID: 29307555  [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "5. J Innate Immun. 2015;7(5):545-53. doi: 10.1159/000381265. Epub 2015 Apr 17.\n",
      "\n",
      "VPS13D Gene Variant Is Associated with Altered IL-6 Production and Mortality in\n",
      "Septic Shock.\n",
      "\n",
      "Nakada TA(1), Boyd JH, Russell JA, Aguirre-Hernndez R, Wilkinson MD, Thair SA,\n",
      "Nakada E, McConechy MK, Fjell CD, Walley KR.\n",
      "\n",
      "Author information: \n",
      "(1)Critical Care Research Laboratories, Centre for Heart Lung Innovation, St.\n",
      "Paul's Hospital, University of British Columbia, Vancouver, B.C., Canada.\n",
      "\n",
      "BACKGROUND: Genetic variations contribute to septic shock mortality. To discover \n",
      "a novel locus, we performed in vitro genome-wide association studies (GWAS) and\n",
      "further tested the result in a cohort of septic shock patients.\n",
      "METHODS: Two in vitro GWAS using a quantitative trait locus analysis of\n",
      "stimulated IL-6 production in lymphoblastoid cells from 60 individuals of\n",
      "European ancestry were performed. VPS13D rs6685273 was genotyped in European\n",
      "ancestry patients (n = 498). The VPS13D gene was silenced in vitro.\n",
      "RESULTS: Two GWAS using lymphoblastoid cells identified the locus of VPS13D\n",
      "rs6685273 that was significant in the same direction in both GWAS. The VPS13D\n",
      "rs6685273 C allele was associated with increased IL-6 production. Patients with\n",
      "septic shock who had the VPS13D rs6685273 CC genotype had an increased 28-day\n",
      "mortality (p = 0.023) and more organ failure (p < 0.05) compared to the CT/TT\n",
      "genotypes. VPS13D in vitro gene silencing in the HeLa cell line increased IL-6\n",
      "production. Furthermore, the rs6685273 genotype was associated with differential \n",
      "VPS13D splice variant expression.\n",
      "CONCLUSIONS: The VPS13D rs6685273 C allele was associated with increased IL-6\n",
      "production in vitro. The patients with the VPS13D rs6685273 CC genotype had\n",
      "increased 28-day mortality and increased organ failure. VPS13D appears to\n",
      "regulate IL-6 production.\n",
      "\n",
      " 2015 S. Karger AG, Basel.\n",
      "\n",
      "DOI: 10.1159/000381265 \n",
      "PMID: 25896417  [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "6. J Atheroscler Thromb. 2010 Oct 27;17(10):1054-62. Epub 2010 Jul 2.\n",
      "\n",
      "Identification of evidence suggestive of an association with peripheral arterial \n",
      "disease at the OSBPL10 locus by genome-wide investigation in the Japanese\n",
      "population.\n",
      "\n",
      "Koriyama H(1), Nakagami H, Katsuya T, Sugimoto K, Yamashita H, Takami Y, Maeda S,\n",
      "Kubo M, Takahashi A, Nakamura Y, Ogihara T, Rakugi H, Kaneda Y, Morishita R.\n",
      "\n",
      "Author information: \n",
      "(1)Department of Geriatric Medicine and Nephrology, Osaka University Graduate\n",
      "School of Medicine, 2-2 Yamada-oka, Suita, Japan.\n",
      "\n",
      "AIM: Peripheral arterial disease (PAD) is a common cause of cardiovascular\n",
      "morbidity and an independent predictor of cardiovascular mortality. However,\n",
      "little is known about the genetic basis of PAD. To elucidate this, we performed a\n",
      "two-staged genome-wide association study in Japanese individuals.\n",
      "METHODS: We initially tested 222,285 single-nucleotide polymorphisms (SNPs).\n",
      "After the first screening in a panel of 195 PAD cases and 1,358 controls, 2,696\n",
      "SNPs (1.2%) were further genotyped in the second screening using another panel of\n",
      "699 PAD cases and 1540 controls. In both screenings, controls were subjects\n",
      "affected with some diseases other than PAD.\n",
      "RESULTS: When analyzed in the combined panel, the strongest signal of PAD\n",
      "association was observed at rs1902341 in the intron of OSBPL10 (p=4.7E-7 for\n",
      "trend test; OR=1.31, 95% CI 1.18-1.46). Also, PAD was modestly associated at\n",
      "several other loci such as rs2554503 in CSMD1 (p=5.7E-5; OR=1.32, 95% CI\n",
      "1.15-1.51) or rs235243 in VSP13D (p=0.04; OR=1.18, 95% CI 1.01-1.37).\n",
      "CONCLUSION: Our genome-wide exploration identified suggestive evidence of PAD\n",
      "association at the OSBPL10 locus. Because the association has not reached a\n",
      "genome-wide significant level, further replication study is warranted for\n",
      "verification in the Japanese population.\n",
      "\n",
      "DOI: 10.5551/jat.4291 \n",
      "PMID: 20610895  [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "7. Hereditary Ataxia Overview.\n",
      "\n",
      "Bird TD(1).\n",
      "In: Adam MP, Ardinger HH, Pagon RA, Wallace SE, Bean LJH, Stephens K, Amemiya A, \n",
      "editors. GeneReviews [Internet]. Seattle (WA): University of Washington,\n",
      "Seattle; 1993-2019.\n",
      "1998 Oct 28.\n",
      "\n",
      "Author information: \n",
      "(1)Seattle VA Medical Center; Departments of Neurology and Medicine, University\n",
      "of Washington, Seattle, Washington\n",
      "\n",
      "CLINICAL CHARACTERISTICS: The hereditary ataxias are a group of genetic disorders\n",
      "characterized by slowly progressive incoordination of gait and often associated\n",
      "with poor coordination of hands, speech, and eye movements. Frequently, atrophy\n",
      "of the cerebellum occurs. In this GeneReview the hereditary ataxias are\n",
      "categorized by mode of inheritance and gene (or chromosome locus) in which\n",
      "pathogenic variants occur.\n",
      "DIAGNOSIS/TESTING: Inherited (genetic) forms of ataxia must be distinguished from\n",
      "the many acquired (non-genetic) causes of ataxia. The genetic forms of ataxia are\n",
      "diagnosed by family history, physical examination, neuroimaging, and molecular\n",
      "genetic testing.\n",
      "GENETIC COUNSELING: The hereditary ataxias can be inherited in an autosomal\n",
      "dominant, autosomal recessive, or X-linked manner. Genetic counseling and risk\n",
      "assessment depend on determination of the specific cause of an inherited ataxia\n",
      "in an individual.\n",
      "MANAGEMENT: Treatment of manifestations: Canes, walkers, and wheelchairs for gait\n",
      "ataxia; use of special devices to assist with handwriting, buttoning, and use of \n",
      "eating utensils; speech therapy and/or computer-based devices for those with\n",
      "dysarthria and severe speech deficits. Prevention of primary manifestations: With\n",
      "the exception of vitamin E therapy for ataxia with vitamin E deficiency (AVED),\n",
      "no specific treatments exist for hereditary ataxia.\n",
      "\n",
      "PMID: 20301317 \n",
      "\n",
      "\n",
      "8. J Am Soc Nephrol. 2009 Feb;20(2):363-79. doi: 10.1681/ASN.2008040406. Epub 2008\n",
      "Dec 3.\n",
      "\n",
      "Large-scale proteomics and phosphoproteomics of urinary exosomes.\n",
      "\n",
      "Gonzales PA(1), Pisitkun T, Hoffert JD, Tchapyjnikov D, Star RA, Kleta R, Wang\n",
      "NS, Knepper MA.\n",
      "\n",
      "Author information: \n",
      "(1)Laboratory of Kidney and Electrolyte Metabolism, National Heart, Lung, and\n",
      "Blood Institute, National Institutes of Health, Bethesda, MD 20892, USA.\n",
      "\n",
      "Normal human urine contains large numbers of exosomes, which are 40- to 100-nm\n",
      "vesicles that originate as the internal vesicles in multivesicular bodies from\n",
      "every renal epithelial cell type facing the urinary space. Here, we used LC-MS/MS\n",
      "to profile the proteome of human urinary exosomes. Overall, the analysis\n",
      "identified 1132 proteins unambiguously, including 177 that are represented on the\n",
      "Online Mendelian Inheritance in Man database of disease-related genes, suggesting\n",
      "that exosome analysis is a potential approach to discover urinary biomarkers. We \n",
      "extended the proteomic analysis to phosphoproteomic profiling using neutral loss \n",
      "scanning, and this yielded multiple novel phosphorylation sites, including\n",
      "serine-811 in the thiazide-sensitive Na-Cl co-transporter, NCC. To demonstrate\n",
      "the potential use of exosome analysis to identify a genetic renal disease, we\n",
      "carried out immunoblotting of exosomes from urine samples of patients with a\n",
      "clinical diagnosis of Bartter syndrome type I, showing an absence of the\n",
      "sodium-potassium-chloride co-transporter 2, NKCC2. The proteomic data are\n",
      "publicly accessible at http://dir.nhlbi.nih.gov/papers/lkem/exosome/.\n",
      "\n",
      "DOI: 10.1681/ASN.2008040406 \n",
      "PMCID: PMC2637050\n",
      "PMID: 19056867  [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "9. Genomics. 2004 Sep;84(3):536-49.\n",
      "\n",
      "Analysis of the human VPS13 gene family.\n",
      "\n",
      "Velayos-Baeza A(1), Vettori A, Copley RR, Dobson-Stone C, Monaco AP.\n",
      "\n",
      "Author information: \n",
      "(1)Wellcome Trust Centre for Human Genetics, University of Oxford, Headington,\n",
      "OX3 7BN Oxford, UK.\n",
      "\n",
      "The gene mutated in chorea-acanthocytosis (CHAC; approved gene symbol VPS13A)\n",
      "encodes chorein, a protein similar to yeast Vps13p. We detected several similar\n",
      "putative human proteins by BLAST analysis of chorein. We characterized the\n",
      "structure of three new genes encoding these CHAC-similar proteins, located on\n",
      "chromosomes 1p36, 8q22, and 15q21. The most similar gene in yeast to all four\n",
      "human genes is Vps13, and therefore the human genes were named VPS13A (CHAC,\n",
      "9q21), VPS13B (8q22), VPS13C (15q21), and VPS13D (1p36). VPS13B has recently been\n",
      "reported as COH1, altered in Cohen syndrome. For each gene, we describe several\n",
      "alternative splicing variants; at least two transcripts per gene are major forms.\n",
      "The expression pattern of these genes is ubiquitous, with some tissue-specific\n",
      "differences between several transcript variants. Protein sequence comparisons\n",
      "suggest that intramolecular duplications have played an important role in the\n",
      "evolution of this gene family.\n",
      "\n",
      "DOI: 10.1016/j.ygeno.2004.04.012 \n",
      "PMID: 15498460  [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "10. BMC Plant Biol. 2004 Mar 12;4:3.\n",
      "\n",
      "Comparison of ESTs from juvenile and adult phases of the giant unicellular green \n",
      "alga Acetabularia acetabulum.\n",
      "\n",
      "Henry IM(1), Wilkinson MD, Hernandez JM, Schwarz-Sommer Z, Grotewold E, Mandoli\n",
      "DF.\n",
      "\n",
      "Author information: \n",
      "(1)Department of Biology, University of Washington, Box 355325, 1521 NE Pacific\n",
      "Street, Seattle, WA 98195-5325, USA. henryi@u.washington.edu\n",
      "\n",
      "BACKGROUND: Acetabularia acetabulum is a giant unicellular green alga whose size \n",
      "and complex life cycle make it an attractive model for understanding\n",
      "morphogenesis and subcellular compartmentalization. The life cycle of this marine\n",
      "unicell is composed of several developmental phases. Juvenile and adult phases\n",
      "are temporally sequential but physiologically and morphologically distinct. To\n",
      "identify genes specific to juvenile and adult phases, we created two subtracted\n",
      "cDNA libraries, one adult-specific and one juvenile-specific, and analyzed 941\n",
      "randomly chosen ESTs from them.\n",
      "RESULTS: Clustering analysis suggests virtually no overlap between the two\n",
      "libraries. Preliminary expression data also suggests that we were successful at\n",
      "isolating transcripts differentially expressed between the two developmental\n",
      "phases and that many transcripts are specific to one phase or the other.\n",
      "Comparison of our EST sequences against publicly available sequence databases\n",
      "indicates that ESTs from the adult and the juvenile libraries partition into\n",
      "different functional classes. Three conserved sequence elements were common to\n",
      "several of the ESTs and were also found within the genomic sequence of the\n",
      "carbonic anhydrase1 gene from A. acetabulum. To date, these conserved elements\n",
      "are specific to A. acetabulum.\n",
      "CONCLUSIONS: Our data provide strong evidence that adult and juvenile phases in\n",
      "A. acetabulum vary significantly in gene expression. We discuss their possible\n",
      "roles in cell growth and morphogenesis as well as in phase change. We also\n",
      "discuss the potential role of the conserved elements found within the EST\n",
      "sequences in post-transcriptional regulation, particularly mRNA localization\n",
      "and/or stability.\n",
      "\n",
      "DOI: 10.1186/1471-2229-4-3 \n",
      "PMCID: PMC385229\n",
      "PMID: 15070428  [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "11. Genome Res. 2002 Nov;12(11):1773-84.\n",
      "\n",
      "Protein-protein interactions between large proteins: two-hybrid screening using a\n",
      "functionally classified library composed of long cDNAs.\n",
      "\n",
      "Nakayama M(1), Kikuno R, Ohara O.\n",
      "\n",
      "Author information: \n",
      "(1)Department of Human Gene Research, Kazusa DNA Research Institute, Kisarazu,\n",
      "Chiba 292-0818, Japan. nmanabu@kazusa.or.jp\n",
      "\n",
      "Large proteins have multiple domains that are potentially capable of binding many\n",
      "kinds of partners. It is conceivable, therefore, that such proteins could\n",
      "function as an intricate framework of assembly protein complexes. To\n",
      "comprehensively study protein-protein interactions between large KIAA proteins,\n",
      "we have constructed a library composed of 1087 KIAA cDNA clones based on prior\n",
      "functional classifications done in silico. We were guided by two principles that \n",
      "raise the success rate for detecting interactions per tested combination: we\n",
      "avoided testing low-probability combinations, and reduced the number of potential\n",
      "false negatives that arise from the fact that large proteins cannot reliably be\n",
      "expressed in yeast. The latter was addressed by constructing a cDNA library\n",
      "comprised of random fragments encoding large proteins. Cytoplasmic domains of\n",
      "KIAA transmembrane proteins (>1000 amino acids) were used as bait for yeast\n",
      "two-hybrid screening. Our analyses reveal that several KIAA proteins bearing a\n",
      "transmembrane region have the capability of binding to other KIAA proteins\n",
      "containing domains (e.g., PDZ, SH3, rhoGEF, and spectrin) known to be localized\n",
      "to highly specialized submembranous sites, indicating that they participate in\n",
      "cellular junction formation, receptor or channel clustering, and intracellular\n",
      "signaling events. Our representative library should be a very useful resource for\n",
      "detecting previously unidentified interactions because it complements\n",
      "conventional expression libraries, which seldom contain large cDNAs.\n",
      "\n",
      "DOI: 10.1101/gr.406902 \n",
      "PMCID: PMC187542\n",
      "PMID: 12421765  [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "12. VPS13D Movement Disorder.\n",
      "\n",
      "Meijer IA(1).\n",
      "In: Adam MP, Ardinger HH, Pagon RA, Wallace SE, Bean LJH, Stephens K, Amemiya A, \n",
      "editors. GeneReviews [Internet]. Seattle (WA): University of Washington,\n",
      "Seattle; 1993-2019.\n",
      "2019 Feb 21.\n",
      "\n",
      "Author information: \n",
      "(1)CHU Sainte-Justine, Universit de Montral, Montral, Qubec, Canada\n",
      "\n",
      "CLINICAL CHARACTERISTICS: VPS13D movement disorder is a hyperkinetic movement\n",
      "disorder (dystonia, chorea, and/or ataxia) of variable age of onset that can be\n",
      "associated with developmental delay. Onset ranges from birth to adulthood.\n",
      "Individuals can present in childhood with motor delays and gait instability.\n",
      "Cognitive impairment ranging from mild intellectual disability to developmental\n",
      "delay has been reported, and several individuals have normal cognitive function. \n",
      "Individuals have also presented as young adults with gait difficulties caused by \n",
      "spastic ataxia or ataxia. In addition to gait ataxia, affected individuals had\n",
      "limb ataxia, dysarthria, and eye movement abnormalities (macro-saccadic\n",
      "oscillations, nystagmus, and saccadic pursuit). Additional features reported in\n",
      "some individuals include peripheral neuropathy and/or seizures. The disorder\n",
      "progresses to spastic ataxia or generalized dystonia, which can lead to loss of\n",
      "independent ambulation.\n",
      "DIAGNOSIS: The diagnosis of VPS13D movement disorder is established in a proband \n",
      "by identification of biallelic pathogenic variants in VPS13D on molecular genetic\n",
      "testing.\n",
      "MANAGEMENT: Treatment of manifestations: Standard treatment for seizures;\n",
      "spasticity treatments include baclofen, tizanidine, benzodiazepines, dantrolene\n",
      "sodium, gabapentin, botulinum toxin injections, and intrathecal baclofen;\n",
      "treatment options for dystonia include trihexyphenidyl, botulinum toxin\n",
      "injections, benzodiazepines, baclofen, and levodopa. A multidisciplinary team\n",
      "including occupational and physical therapists and a physiatrist is important;\n",
      "supportive developmental therapies should be provided as needed. Surveillance:\n",
      "Monitor for urinary urgency, contractures, and seizures; evaluate fall risk and\n",
      "gait stability; monitor developmental progress and educational needs.\n",
      "GENETIC COUNSELING: VPS13D movement disorder is inherited in an autosomal\n",
      "recessive manner. At conception, each sib of an affected individual has a 25%\n",
      "chance of being affected, a 50% chance of being an asymptomatic carrier, and a\n",
      "25% chance of being unaffected and not a carrier. Carrier testing for at-risk\n",
      "relatives and prenatal testing for pregnancies at increased risk are possible if \n",
      "the VPS13D pathogenic variants in the family are known.\n",
      "\n",
      "PMID: 30789691 \n",
      "\n",
      "\n",
      "13. Sci Rep. 2018 Sep 26;8(1):14375. doi: 10.1038/s41598-018-32685-8.\n",
      "\n",
      "USP45 and Spindly are part of the same complex implicated in cell migration.\n",
      "\n",
      "Conte C(1)(2), Griffis ER(1)(3), Hickson I(4)(5), Perez-Oliva AB(6).\n",
      "\n",
      "Author information: \n",
      "(1)MRC Protein Phosphorylation and Ubiquitylation Unit, College of Life Sciences,\n",
      "University of Dundee, Dundee, UK.\n",
      "(2)Cell and Developmental Biology, Center of Genomic Regulation (CRG), Barcelona,\n",
      "Spain.\n",
      "(3)Nikon Instruments Incorporated, Melville, NY, 11747, USA.\n",
      "(4)Janssen Research & Development, LLC, Spring House, PA, USA.\n",
      "(5)Northern Institute for Cancer Research, Newcastle University, Medical School, \n",
      "Newcastle upon Tyne, NE2 4HH, UK.\n",
      "(6)MRC Protein Phosphorylation and Ubiquitylation Unit, College of Life Sciences,\n",
      "University of Dundee, Dundee, UK. anabpo@um.es.\n",
      "\n",
      "Ubiquitylation is a protein modification implicated in several cellular\n",
      "processes. This process is reversible by the action of deubiquinating enzymes\n",
      "(DUBs). USP45 is a ubiquitin specific protease about which little is known, aside\n",
      "from roles in DNA damage repair and differentiation of the vertebrate retina.\n",
      "Here, by using mass spectrometry we have identified Spindly as a new target of\n",
      "USP45. Our data show that Spindly and USP45 are part of the same complex and that\n",
      "their interaction specifically depends on the catalytic activity of USP45. In\n",
      "addition, we describe the type of ubiquitin chains associated with the complex\n",
      "that can be cleaved by USP45, with a preferential activity on K48 ubiquitin chain\n",
      "type and potentially K6. Here, we also show that Spindly is mono-ubiquitylated\n",
      "and this can be specifically removed by USP45 in its active form but not by the\n",
      "catalytic inactive form. Lastly, we identified a new role for USP45 in cell\n",
      "migration, similar to that which was recently described for Spindly.\n",
      "\n",
      "DOI: 10.1038/s41598-018-32685-8 \n",
      "PMCID: PMC6158257\n",
      "PMID: 30258100 \n",
      "\n",
      "\n",
      "14. Ann Neurol. 2018 Jun;83(6):1075-1088. doi: 10.1002/ana.25220. Epub 2018 Jun 30.\n",
      "\n",
      "Mutations in VPS13D lead to a new recessive ataxia with spasticity and\n",
      "mitochondrial defects.\n",
      "\n",
      "Seong E(1), Insolera R(2), Dulovic M(3), Kamsteeg EJ(4), Trinh J(3), Brggemann\n",
      "N(5), Sandford E(1), Li S(1), Ozel AB(6), Li JZ(6)(7), Jewett T(8), Kievit\n",
      "AJA(9), Mnchau A(3), Shakkottai V(10), Klein C(3), Collins CA(2), Lohmann K(3), \n",
      "van de Warrenburg BP(11), Burmeister M(1)(6)(7)(12).\n",
      "\n",
      "Author information: \n",
      "(1)Molecular and Behavioral Neuroscience Institute, University of Michigan, Ann\n",
      "Arbor, MI.\n",
      "(2)Department of Molecular, Cellular, and Developmental Biology, University of\n",
      "Michigan, Ann Arbor, MI.\n",
      "(3)Institute of Neurogenetics, University of Lbeck, Lbeck, Germany.\n",
      "(4)Department of Human Genetics, Radboud University Medical Centre, Nijmegen, the\n",
      "Netherlands.\n",
      "(5)Department of Neurology, University of Lbeck, Lbeck, Germany.\n",
      "(6)Department of Human Genetics, University of Michigan, Ann Arbor, MI.\n",
      "(7)Department of Computational Medicine and Bioinformatics, University of\n",
      "Michigan, Ann Arbor, MI.\n",
      "(8)Department of Pediatrics, Section on Medical Genetics, Wake Forest School of\n",
      "Medicine, Winston-Salem, NC.\n",
      "(9)Department of Clinical Genetics, Erasmus MC, Rotterdam, the Netherlands.\n",
      "(10)Departments of Neurology and Molecular and Integrative Physiology, University\n",
      "of Michigan, Ann Arbor, MI.\n",
      "(11)Department of Neurology, Donders Institute for Brain, Cognition, and\n",
      "Behavior, Radboud University Medical Centre, Nijmegen, the Netherlands.\n",
      "(12)Department of Psychiatry, University of Michigan, Ann Arbor, MI.\n",
      "\n",
      "Comment in\n",
      "    Ann Neurol. 2018 Jun;83(6):1072-1074.\n",
      "\n",
      "OBJECTIVE: To identify novel causes of recessive ataxias, including\n",
      "spinocerebellar ataxia with saccadic intrusions, spastic ataxias, and spastic\n",
      "paraplegia.\n",
      "METHODS: In an international collaboration, we independently performed exome\n",
      "sequencing in 7 families with recessive ataxia and/or spastic paraplegia. To\n",
      "evaluate the role of VPS13D mutations, we evaluated a Drosophila knockout model\n",
      "and investigated mitochondrial function in patient-derived fibroblast cultures.\n",
      "RESULTS: Exome sequencing identified compound heterozygous mutations in VPS13D on\n",
      "chromosome 1p36 in all 7 families. This included a large family with 5 affected\n",
      "siblings with spinocerebellar ataxia with saccadic intrusions (SCASI), or\n",
      "spinocerebellar ataxia, recessive, type 4 (SCAR4). Linkage to chromosome 1p36 was\n",
      "found in this family with a logarithm of odds score of 3.1. The phenotypic\n",
      "spectrum in our 12 patients was broad. Although most presented with ataxia,\n",
      "additional or predominant spasticity was present in 5 patients. Disease onset\n",
      "ranged from infancy to 39 years, and symptoms were slowly progressive and\n",
      "included loss of independent ambulation in 5. All but 2 patients carried a\n",
      "loss-of-function (nonsense or splice site) mutation on one and a missense\n",
      "mutation on the other allele. Knockdown or removal of Vps13D in Drosophila\n",
      "neurons led to changes in mitochondrial morphology and impairment in\n",
      "mitochondrial distribution along axons. Patient fibroblasts showed altered\n",
      "morphology and functionality including reduced energy production.\n",
      "INTERPRETATION: Our study demonstrates that compound heterozygous mutations in\n",
      "VPS13D cause movement disorders along the ataxia-spasticity spectrum, making\n",
      "VPS13D the fourth VPS13 paralog involved in neurological disorders. Ann Neurol\n",
      "2018.\n",
      "\n",
      " 2018 American Neurological Association.\n",
      "\n",
      "DOI: 10.1002/ana.25220 \n",
      "PMCID: PMC6105379\n",
      "PMID: 29604224  [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "15. Nat Commun. 2018 Mar 22;9(1):1188. doi: 10.1038/s41467-018-03523-2.\n",
      "\n",
      "An AP-MS- and BioID-compatible MAC-tag enables comprehensive mapping of protein\n",
      "interactions and subcellular localizations.\n",
      "\n",
      "Liu X(1)(2), Salokas K(1)(2), Tamene F(1)(2)(3), Jiu Y(1)(2), Weldatsadik\n",
      "RG(1)(2)(3), hman T(1)(2)(3), Varjosalo M(4)(5)(6).\n",
      "\n",
      "Author information: \n",
      "(1)Institute of Biotechnology, University of Helsinki, Helsinki, 00014, Finland.\n",
      "(2)Helsinki Institute of Life Science, University of Helsinki, Helsinki, 00014,\n",
      "Finland.\n",
      "(3)Proteomics Unit, University of Helsinki, Helsinki, 00014, Finland.\n",
      "(4)Institute of Biotechnology, University of Helsinki, Helsinki, 00014, Finland. \n",
      "markku.varjosalo@helsinki.fi.\n",
      "(5)Helsinki Institute of Life Science, University of Helsinki, Helsinki, 00014,\n",
      "Finland. markku.varjosalo@helsinki.fi.\n",
      "(6)Proteomics Unit, University of Helsinki, Helsinki, 00014, Finland.\n",
      "markku.varjosalo@helsinki.fi.\n",
      "\n",
      "Protein-protein interactions govern almost all cellular functions. These complex \n",
      "networks of stable and transient associations can be mapped by affinity\n",
      "purification mass spectrometry (AP-MS) and complementary proximity-based labeling\n",
      "methods such as BioID. To exploit the advantages of both strategies, we here\n",
      "design and optimize an integrated approach combining AP-MS and BioID in a single \n",
      "construct, which we term MAC-tag. We systematically apply the MAC-tag approach to\n",
      "18 subcellular and 3 sub-organelle localization markers, generating a molecular\n",
      "context database, which can be used to define a protein's molecular location. In \n",
      "addition, we show that combining the AP-MS and BioID results makes it possible to\n",
      "obtain interaction distances within a protein complex. Taken together, our\n",
      "integrated strategy enables the comprehensive mapping of the physical and\n",
      "functional interactions of proteins, defining their molecular context and\n",
      "improving our understanding of the cellular interactome.\n",
      "\n",
      "DOI: 10.1038/s41467-018-03523-2 \n",
      "PMCID: PMC5864832\n",
      "PMID: 29568061  [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "16. Ann Neurol. 2018 Jun;83(6):1089-1095. doi: 10.1002/ana.25204. Epub 2018 Apr 10.\n",
      "\n",
      "Recessive mutations in VPS13D cause childhood onset movement disorders.\n",
      "\n",
      "Gauthier J(1), Meijer IA(2), Lessel D(3), Mencacci NE(4), Krainc D(4), Hempel\n",
      "M(3), Tsiakas K(5), Prokisch H(6)(7), Rossignol E(2), Helm MH(8), Rodan LH(9),\n",
      "Karamchandani J(10), Carecchio M(11), Lubbe SJ(4), Telegrafi A(12), Henderson\n",
      "LB(12), Lorenzo K(13), Wallace SE(14), Glass IA(14), Hamdan FF(1), Michaud\n",
      "JL(15), Rouleau GA(16), Campeau PM(15).\n",
      "\n",
      "Author information: \n",
      "(1)Molecular Diagnostic Laboratory and Division of Medical Genetics, Department\n",
      "of Pediatrics, Saint Justine University Hospital Center, Montreal, Canada.\n",
      "(2)Department of Neuroscience, University of Montreal, Montreal, Canada.\n",
      "(3)Institute of Human Genetics, University Medical Center Hamburg-Eppendorf,\n",
      "Hamburg, Germany.\n",
      "(4)Department of Neurology, Northwestern University, Feinberg School of Medicine,\n",
      "Chicago, IL.\n",
      "(5)Department of Pediatrics, University Medical Center Hamburg-Eppendorf,\n",
      "Hamburg, Germany.\n",
      "(6)Institute of Human Genetics, Helmholtz Center Munich, Neuherberg, Germany.\n",
      "(7)Institute of Human Genetics, Technical University Munich, Munich, Germany.\n",
      "(8)Maine Medical Partners Pediatric Specialty Care, Portland, ME.\n",
      "(9)Department of Neurology, Boston Children's Hospital, Boston, MA.\n",
      "(10)Department of Pathology, McGill University, Montreal Neurological Institute, \n",
      "Montreal, Canada.\n",
      "(11)Molecular Neurogenetics Unit, Institute for Research and Health Care (IRCCS) \n",
      "Foundation Carlo Besta Neurological Institute, Milan, Italy.\n",
      "(12)GeneDx, Gaithersburg, MD.\n",
      "(13)Kadlec Clinic Genetic Counseling, Richland, WA.\n",
      "(14)Division of Genetic Medicine, Department of Pediatrics, Seattle Children's\n",
      "Hospital and University of Washington, Seattle, WA.\n",
      "(15)Department of Pediatrics, Saint Justine University Hospital Center and\n",
      "University of Montreal, Montreal, Canada.\n",
      "(16)Montreal Neurological Institute, Department of Neurology and Neurosurgery,\n",
      "McGill University, Montreal, Canada.\n",
      "\n",
      "Comment in\n",
      "    Ann Neurol. 2018 Jun;83(6):1072-1074.\n",
      "\n",
      "VPS13 protein family members VPS13A through VPS13C have been associated with\n",
      "various recessive movement disorders. We describe the first disease association\n",
      "of rare recessive VPS13D variants including frameshift, missense, and partial\n",
      "duplication mutations with a novel complex, hyperkinetic neurological disorder.\n",
      "The clinical features include developmental delay, a childhood onset movement\n",
      "disorder (chorea, dystonia, or tremor), and progressive spastic ataxia or\n",
      "paraparesis. Characteristic brain magnetic resonance imaging shows basal ganglia \n",
      "or diffuse white matter T2 hyperintensities as seen in Leigh syndrome and\n",
      "choreoacanthocytosis. Muscle biopsy in 1 case showed mitochondrial aggregates and\n",
      "lipidosis, suggesting mitochondrial dysfunction. These findings underline the\n",
      "importance of the VPS13 complex in neurological diseases and a possible role in\n",
      "mitochondrial function. Ann Neurol 2018;83:1089-1095.\n",
      "\n",
      " 2018 American Neurological Association.\n",
      "\n",
      "DOI: 10.1002/ana.25204 \n",
      "PMID: 29518281  [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "17. Cell Discov. 2018 Feb 27;4:10. doi: 10.1038/s41421-018-0019-0. eCollection 2018.\n",
      "\n",
      "VIRMA mediates preferential m6A mRNA methylation in 3'UTR and near stop codon and\n",
      "associates with alternative polyadenylation.\n",
      "\n",
      "Yue Y(1), Liu J(2), Cui X(2), Cao J(1), Luo G(2), Zhang Z(1), Cheng T(1), Gao\n",
      "M(1), Shu X(1), Ma H(2), Wang F(3), Wang X(3), Shen B(4), Wang Y(3), Feng X(5),\n",
      "He C(2), Liu J(1)(5).\n",
      "\n",
      "Author information: \n",
      "(1)1MOE Key Laboratory of Macromolecular Synthesis and Functionalization,\n",
      "Department of Polymer Science and Engineering, Zhejiang University, Hangzhou,\n",
      "Zhejiang 310027 China.\n",
      "(2)2Department of Chemistry, Department of Biochemistry and Molecular Biology,\n",
      "Institute for Biophysical Dynamics, Howard Hughes Medical Institute, The\n",
      "University of Chicago, Chicago, IL 60637 USA.\n",
      "(3)3College of Animal Sciences, Key Laboratory of Molecular Nutrition, Ministry\n",
      "of Education, Zhejiang University, Hangzhou, Zhejiang 310058 China.\n",
      "(4)4State Key Laboratory of Reproductive Medicine, Department of Histology and\n",
      "Embryology, Nanjing Medical University, Nanjing, Jiangsu 210029 China.\n",
      "(5)5Life Sciences Institute, Zhejiang University, Hangzhou, Zhejiang 310058\n",
      "China.\n",
      "\n",
      "N6-methyladenosine (m6A) is enriched in 3'untranslated region (3'UTR) and near\n",
      "stop codon of mature polyadenylated mRNAs in mammalian systems and has regulatory\n",
      "roles in eukaryotic mRNA transcriptome switch. Significantly, the mechanism for\n",
      "this modification preference remains unknown, however. Herein we report a\n",
      "characterization of the full m6A methyltransferase complex in HeLa cells\n",
      "identifying METTL3/METTL14/WTAP/VIRMA/HAKAI/ZC3H13 as the key components, and we \n",
      "show that VIRMA mediates preferential mRNA methylation in 3'UTR and near stop\n",
      "codon. Biochemical studies reveal that VIRMA recruits the catalytic core\n",
      "components METTL3/METTL14/WTAP to guide region-selective methylations. Around 60%\n",
      "of VIRMA mRNA immunoprecipitation targets manifest strong m6A enrichment in\n",
      "3'UTR. Depletions of VIRMA and METTL3 induce 3'UTR lengthening of several hundred\n",
      "mRNAs with over 50% targets in common. VIRMA associates with polyadenylation\n",
      "cleavage factors CPSF5 and CPSF6 in an RNA-dependent manner. Depletion of CPSF5\n",
      "leads to significant shortening of 3'UTR of over 2800 mRNAs, 84% of which are\n",
      "modified with m6A and have increased m6A peak density in 3'UTR and near stop\n",
      "codon after CPSF5 knockdown. Together, our studies provide insights into m6A\n",
      "deposition specificity in 3'UTR and its correlation with alternative\n",
      "polyadenylation.\n",
      "\n",
      "DOI: 10.1038/s41421-018-0019-0 \n",
      "PMCID: PMC5826926\n",
      "PMID: 29507755 \n",
      "\n",
      "Conflict of interest statement: C.H. is a scientific founder of the Accent\n",
      "Therapeutics and a member of the Scientific Advisory Committee. The remaining\n",
      "authors declare that they have no conflict of interest.\n",
      "\n",
      "\n",
      "18. Curr Biol. 2018 Jan 22;28(2):287-295.e6. doi: 10.1016/j.cub.2017.11.064. Epub\n",
      "2018 Jan 4.\n",
      "\n",
      "Vps13D Encodes a Ubiquitin-Binding Protein that Is Required for the Regulation of\n",
      "Mitochondrial Size and Clearance.\n",
      "\n",
      "Anding AL(1), Wang C(2), Chang TK(1), Sliter DA(2), Powers CM(1), Hofmann K(3),\n",
      "Youle RJ(2), Baehrecke EH(4).\n",
      "\n",
      "Author information: \n",
      "(1)Department of Molecular, Cell and Cancer Biology, University of Massachusetts \n",
      "Medical School, Worcester, MA 01605, USA.\n",
      "(2)Biochemistry Section, Surgical Neurology Branch, National Institute of\n",
      "Neurological Disorders and Stroke, NIH, Bethesda, MD 20892, USA.\n",
      "(3)Institute for Genetics, University of Cologne, 50674 Cologne, Germany.\n",
      "(4)Department of Molecular, Cell and Cancer Biology, University of Massachusetts \n",
      "Medical School, Worcester, MA 01605, USA. Electronic address:\n",
      "eric.baehrecke@umassmed.edu.\n",
      "\n",
      "Comment in\n",
      "    Curr Biol. 2018 Jan 22;28(2):R66-R68.\n",
      "\n",
      "The clearance of mitochondria by autophagy, mitophagy, is important for cell and \n",
      "organism health [1], and known to be regulated by ubiquitin. During Drosophila\n",
      "intestine development, cells undergo a dramatic reduction in cell size and\n",
      "clearance of mitochondria that depends on autophagy, the E1 ubiquitin-activating \n",
      "enzyme Uba1, and ubiquitin [2]. Here we screen a collection of putative\n",
      "ubiquitin-binding domain-encoding genes for cell size reduction and autophagy\n",
      "phenotypes. We identify the endosomal sorting complex required for transport\n",
      "(ESCRT) components TSG101 and Vps36, as well as the novel gene Vps13D. Vps13D is \n",
      "an essential gene that is necessary for autophagy, mitochondrial size, and\n",
      "mitochondrial clearance in Drosophila. Interestingly, a similar mitochondrial\n",
      "phenotype is observed in VPS13D mutant human cells. The ubiquitin-associated\n",
      "(UBA) domain of Vps13D binds K63 ubiquitin chains, and mutants lacking the UBA\n",
      "domain have defects in mitochondrial size and clearance and exhibit\n",
      "semi-lethality, highlighting the importance of Vps13D ubiquitin binding in both\n",
      "mitochondrial health and development. VPS13D mutant cells possess phosphorylated \n",
      "DRP1 and mitochondrial fission factor (MFF) as well as DRP1 association with\n",
      "mitochondria, suggesting that VPS13D functions downstream of these known\n",
      "regulators of mitochondrial fission. In addition, the large Vps13D mitochondrial \n",
      "and cell size phenotypes are suppressed by decreased mitochondrial fusion gene\n",
      "function. Thus, these results provide a previously unknown link between\n",
      "ubiquitin, mitochondrial size regulation, and autophagy.\n",
      "\n",
      "Copyright  2017 Elsevier Ltd. All rights reserved.\n",
      "\n",
      "DOI: 10.1016/j.cub.2017.11.064 \n",
      "PMCID: PMC5787036\n",
      "PMID: 29307555  [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "19. Nature. 2015 Sep 17;525(7569):339-44. doi: 10.1038/nature14877. Epub 2015 Sep 7.\n",
      "\n",
      "Panorama of ancient metazoan macromolecular complexes.\n",
      "\n",
      "Wan C(1)(2), Borgeson B(2), Phanse S(1), Tu F(2), Drew K(2), Clark G(3), Xiong\n",
      "X(4)(5), Kagan O(1), Kwan J(1)(4), Bezginov A(3), Chessman K(4)(5), Pal S(5),\n",
      "Cromar G(4)(5), Papoulas O(2), Ni Z(1), Boutz DR(2), Stoilova S(1), Havugimana\n",
      "PC(1), Guo X(1), Malty RH(6), Sarov M(7), Greenblatt J(1)(4), Babu M(6), Derry\n",
      "WB(4)(5), Tillier ER(3), Wallingford JB(2)(8), Parkinson J(4)(5), Marcotte\n",
      "EM(2)(8), Emili A(1)(4).\n",
      "\n",
      "Author information: \n",
      "(1)Donnelly Centre for Cellular and Biomolecular Research, University of Toronto,\n",
      "Toronto, Ontario M5S 3E1, Canada.\n",
      "(2)Center for Systems and Synthetic Biology, Institute for Cellular and Molecular\n",
      "Biology, University of Texas at Austin, Austin, Texas 78712, USA.\n",
      "(3)Department of Medical Biophysics, Toronto, Ontario M5G 1L7, Canada.\n",
      "(4)Department of Molecular Genetics, University of Toronto, Toronto, Ontario M5S \n",
      "1A8, Canada.\n",
      "(5)Hospital for Sick Children, Toronto, Ontario M5G 1X8, Canada.\n",
      "(6)Department of Biochemistry, University of Regina, Regina, Saskatchewan S4S\n",
      "0A2, Canada.\n",
      "(7)Max Planck Institute of Molecular Cell Biology and Genetics, 01307 Dresden,\n",
      "Germany.\n",
      "(8)Department of Molecular Biosciences, University of Texas at Austin, Austin,\n",
      "Texas 78712, USA.\n",
      "\n",
      "Comment in\n",
      "    Trends Biochem Sci. 2016 Jan;41(1):4-6.\n",
      "    Nat Methods. 2015 Nov;12(11):1011.\n",
      "\n",
      "Macromolecular complexes are essential to conserved biological processes, but\n",
      "their prevalence across animals is unclear. By combining extensive biochemical\n",
      "fractionation with quantitative mass spectrometry, here we directly examined the \n",
      "composition of soluble multiprotein complexes among diverse metazoan models.\n",
      "Using an integrative approach, we generated a draft conservation map consisting\n",
      "of more than one million putative high-confidence co-complex interactions for\n",
      "species with fully sequenced genomes that encompasses functional modules present \n",
      "broadly across all extant animals. Clustering reveals a spectrum of conservation,\n",
      "ranging from ancient eukaryotic assemblies that have probably served cellular\n",
      "housekeeping roles for at least one billion years, ancestral complexes that have \n",
      "accrued contemporary components, and rarer metazoan innovations linked to\n",
      "multicellularity. We validated these projections by independent co-fractionation \n",
      "experiments in evolutionarily distant species, affinity purification and\n",
      "functional analyses. The comprehensiveness, centrality and modularity of these\n",
      "reconstructed interactomes reflect their fundamental mechanistic importance and\n",
      "adaptive value to animal cell systems.\n",
      "\n",
      "DOI: 10.1038/nature14877 \n",
      "PMCID: PMC5036527\n",
      "PMID: 26344197  [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "20. J Innate Immun. 2015;7(5):545-53. doi: 10.1159/000381265. Epub 2015 Apr 17.\n",
      "\n",
      "VPS13D Gene Variant Is Associated with Altered IL-6 Production and Mortality in\n",
      "Septic Shock.\n",
      "\n",
      "Nakada TA(1), Boyd JH, Russell JA, Aguirre-Hernndez R, Wilkinson MD, Thair SA,\n",
      "Nakada E, McConechy MK, Fjell CD, Walley KR.\n",
      "\n",
      "Author information: \n",
      "(1)Critical Care Research Laboratories, Centre for Heart Lung Innovation, St.\n",
      "Paul's Hospital, University of British Columbia, Vancouver, B.C., Canada.\n",
      "\n",
      "BACKGROUND: Genetic variations contribute to septic shock mortality. To discover \n",
      "a novel locus, we performed in vitro genome-wide association studies (GWAS) and\n",
      "further tested the result in a cohort of septic shock patients.\n",
      "METHODS: Two in vitro GWAS using a quantitative trait locus analysis of\n",
      "stimulated IL-6 production in lymphoblastoid cells from 60 individuals of\n",
      "European ancestry were performed. VPS13D rs6685273 was genotyped in European\n",
      "ancestry patients (n = 498). The VPS13D gene was silenced in vitro.\n",
      "RESULTS: Two GWAS using lymphoblastoid cells identified the locus of VPS13D\n",
      "rs6685273 that was significant in the same direction in both GWAS. The VPS13D\n",
      "rs6685273 C allele was associated with increased IL-6 production. Patients with\n",
      "septic shock who had the VPS13D rs6685273 CC genotype had an increased 28-day\n",
      "mortality (p = 0.023) and more organ failure (p < 0.05) compared to the CT/TT\n",
      "genotypes. VPS13D in vitro gene silencing in the HeLa cell line increased IL-6\n",
      "production. Furthermore, the rs6685273 genotype was associated with differential \n",
      "VPS13D splice variant expression.\n",
      "CONCLUSIONS: The VPS13D rs6685273 C allele was associated with increased IL-6\n",
      "production in vitro. The patients with the VPS13D rs6685273 CC genotype had\n",
      "increased 28-day mortality and increased organ failure. VPS13D appears to\n",
      "regulate IL-6 production.\n",
      "\n",
      " 2015 S. Karger AG, Basel.\n",
      "\n",
      "DOI: 10.1159/000381265 \n",
      "PMID: 25896417  [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "21. Brief Bioinform. 2011 Sep;12(5):449-62. doi: 10.1093/bib/bbr042. Epub 2011 Aug\n",
      "27.\n",
      "\n",
      "Phylogenetic-based propagation of functional annotations within the Gene Ontology\n",
      "consortium.\n",
      "\n",
      "Gaudet P(1), Livstone MS, Lewis SE, Thomas PD.\n",
      "\n",
      "Author information: \n",
      "(1)Swiss Institute for Bioinformatics, CMU, 1 Rue Michel Servet, 1211 Geneva 4,\n",
      "Switzerland. pascale.gaudet@isb-sib.ch\n",
      "\n",
      "The goal of the Gene Ontology (GO) project is to provide a uniform way to\n",
      "describe the functions of gene products from organisms across all kingdoms of\n",
      "life and thereby enable analysis of genomic data. Protein annotations are either \n",
      "based on experiments or predicted from protein sequences. Since most sequences\n",
      "have not been experimentally characterized, most available annotations need to be\n",
      "based on predictions. To make as accurate inferences as possible, the GO\n",
      "Consortium's Reference Genome Project is using an explicit evolutionary framework\n",
      "to infer annotations of proteins from a broad set of genomes from experimental\n",
      "annotations in a semi-automated manner. Most components in the pipeline, such as \n",
      "selection of sequences, building multiple sequence alignments and phylogenetic\n",
      "trees, retrieving experimental annotations and depositing inferred annotations,\n",
      "are fully automated. However, the most crucial step in our pipeline relies on\n",
      "software-assisted curation by an expert biologist. This curation tool,\n",
      "Phylogenetic Annotation and INference Tool (PAINT) helps curators to infer\n",
      "annotations among members of a protein family. PAINT allows curators to make\n",
      "precise assertions as to when functions were gained and lost during evolution and\n",
      "record the evidence (e.g. experimentally supported GO annotations and\n",
      "phylogenetic information including orthology) for those assertions. In this\n",
      "article, we describe how we use PAINT to infer protein function in a phylogenetic\n",
      "context with emphasis on its strengths, limitations and guidelines. We also\n",
      "discuss specific examples showing how PAINT annotations compare with those\n",
      "generated by other highly used homology-based methods.\n",
      "\n",
      "DOI: 10.1093/bib/bbr042 \n",
      "PMCID: PMC3178059\n",
      "PMID: 21873635  [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "22. Proteomics. 2011 Jan;11(1):172-9. doi: 10.1002/pmic.201000217. Epub 2010 Dec 1.\n",
      "\n",
      "Identification of proteins associated with ligand-activated estrogen receptor \n",
      "in human breast cancer cell nuclei by tandem affinity purification and nano\n",
      "LC-MS/MS.\n",
      "\n",
      "Tarallo R(1), Bamundo A, Nassa G, Nola E, Paris O, Ambrosino C, Facchiano A,\n",
      "Baumann M, Nyman TA, Weisz A.\n",
      "\n",
      "Author information: \n",
      "(1)Department of General Pathology, Second University of Naples, Napoli, Italy.\n",
      "\n",
      "Estrogen receptor  (ER-) is a key mediator of estrogen actions in breast cancer\n",
      "(BC) cells. Understanding the effects of ligand-activated ER- in target cells\n",
      "requires identification of the molecular partners acting in concert with this\n",
      "nuclear receptor to transduce the hormonal signal. We applied tandem affinity\n",
      "purification (TAP), glycerol gradient centrifugation and MS analysis to isolate\n",
      "and identify proteins interacting with ligand-activated ER- in MCF-7 cell\n",
      "nuclei. This led to the identification of 264 ER-associated proteins, whose\n",
      "functions highlight the hinge role of ER- in the coordination of multiple\n",
      "hormone-regulated nuclear processes in BC cells.\n",
      "\n",
      "Copyright  2011 WILEY-VCH Verlag GmbH & Co. KGaA, Weinheim.\n",
      "\n",
      "DOI: 10.1002/pmic.201000217 \n",
      "PMID: 21182205  [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "23. J Atheroscler Thromb. 2010 Oct 27;17(10):1054-62. Epub 2010 Jul 2.\n",
      "\n",
      "Identification of evidence suggestive of an association with peripheral arterial \n",
      "disease at the OSBPL10 locus by genome-wide investigation in the Japanese\n",
      "population.\n",
      "\n",
      "Koriyama H(1), Nakagami H, Katsuya T, Sugimoto K, Yamashita H, Takami Y, Maeda S,\n",
      "Kubo M, Takahashi A, Nakamura Y, Ogihara T, Rakugi H, Kaneda Y, Morishita R.\n",
      "\n",
      "Author information: \n",
      "(1)Department of Geriatric Medicine and Nephrology, Osaka University Graduate\n",
      "School of Medicine, 2-2 Yamada-oka, Suita, Japan.\n",
      "\n",
      "AIM: Peripheral arterial disease (PAD) is a common cause of cardiovascular\n",
      "morbidity and an independent predictor of cardiovascular mortality. However,\n",
      "little is known about the genetic basis of PAD. To elucidate this, we performed a\n",
      "two-staged genome-wide association study in Japanese individuals.\n",
      "METHODS: We initially tested 222,285 single-nucleotide polymorphisms (SNPs).\n",
      "After the first screening in a panel of 195 PAD cases and 1,358 controls, 2,696\n",
      "SNPs (1.2%) were further genotyped in the second screening using another panel of\n",
      "699 PAD cases and 1540 controls. In both screenings, controls were subjects\n",
      "affected with some diseases other than PAD.\n",
      "RESULTS: When analyzed in the combined panel, the strongest signal of PAD\n",
      "association was observed at rs1902341 in the intron of OSBPL10 (p=4.7E-7 for\n",
      "trend test; OR=1.31, 95% CI 1.18-1.46). Also, PAD was modestly associated at\n",
      "several other loci such as rs2554503 in CSMD1 (p=5.7E-5; OR=1.32, 95% CI\n",
      "1.15-1.51) or rs235243 in VSP13D (p=0.04; OR=1.18, 95% CI 1.01-1.37).\n",
      "CONCLUSION: Our genome-wide exploration identified suggestive evidence of PAD\n",
      "association at the OSBPL10 locus. Because the association has not reached a\n",
      "genome-wide significant level, further replication study is warranted for\n",
      "verification in the Japanese population.\n",
      "\n",
      "DOI: 10.5551/jat.4291 \n",
      "PMID: 20610895  [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "24. Mol Med. 2010 Jul-Aug;16(7-8):247-53. doi: 10.2119/molmed.2009.00159. Epub 2010\n",
      "Mar 17.\n",
      "\n",
      "Personalized smoking cessation: interactions between nicotine dose, dependence\n",
      "and quit-success genotype score.\n",
      "\n",
      "Rose JE(1), Behm FM, Drgon T, Johnson C, Uhl GR.\n",
      "\n",
      "Author information: \n",
      "(1)Department of Psychiatry and Center for Nicotine and Smoking Cessation\n",
      "Research, Duke University, Durham, North Carolina 27705, USA.\n",
      "rose0003@mc.duke.edu\n",
      "\n",
      "Erratum in\n",
      "    Mol Med. 2012;18(1):729.\n",
      "\n",
      "Improving and targeting nicotine replacement therapy (NRT) are cost-effective\n",
      "strategies for reducing adverse health consequences for smokers. Treatment\n",
      "studies document the efficacy of precessation NRT and support important roles for\n",
      "level of nicotine dependence and precessation smoking reduction in successful\n",
      "quitting. However, prior work has not identified the optimal precessation dose or\n",
      "means for personalizing NRT. Genome-wide association has identified groups of\n",
      "genomic markers associated with successful quitting, allowing us to develop a\n",
      "v1.0 \"quit-success\" genotype score. We now report influences of v1.0 quit-success\n",
      "genotype score, level of dependence and precessation smoking reduction in a\n",
      "smoking cessation trial that examined effects of 21 versus 42 mg/24 h\n",
      "precessation NRT. Four hundred seventy-nine smokers were randomized to 21 or 42\n",
      "mg NRT, initiated 2 wks prior to target quit dates. We monitored self-reported\n",
      "abstinence and end-expired air carbon monoxide (CO). Genotyping used Affymetrix\n",
      "arrays (Santa Clara, CA, USA). The primary outcome was 10-wk continuous smoking\n",
      "abstinence. NRT dose, level of nicotine dependence and genotype scores displayed \n",
      "significant interactive effects on successful quitting. Successful abstinence\n",
      "also was predicted by CO reductions during precessation NRT. These results\n",
      "document ways in which smoking cessation strategies can be personalized based on \n",
      "levels of nicotine dependence, genotype scores and CO monitoring. These\n",
      "assessments, taken together, can help match most smokers with optimal NRT doses\n",
      "and help rapidly identify some who may be better treated using other methods.\n",
      "\n",
      "DOI: 10.2119/molmed.2009.00159 \n",
      "PMCID: PMC2896464\n",
      "PMID: 20379614  [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "25. Hereditary Ataxia Overview.\n",
      "\n",
      "Bird TD(1).\n",
      "In: Adam MP, Ardinger HH, Pagon RA, Wallace SE, Bean LJH, Stephens K, Amemiya A, \n",
      "editors. GeneReviews [Internet]. Seattle (WA): University of Washington,\n",
      "Seattle; 1993-2019.\n",
      "1998 Oct 28.\n",
      "\n",
      "Author information: \n",
      "(1)Seattle VA Medical Center; Departments of Neurology and Medicine, University\n",
      "of Washington, Seattle, Washington\n",
      "\n",
      "CLINICAL CHARACTERISTICS: The hereditary ataxias are a group of genetic disorders\n",
      "characterized by slowly progressive incoordination of gait and often associated\n",
      "with poor coordination of hands, speech, and eye movements. Frequently, atrophy\n",
      "of the cerebellum occurs. In this GeneReview the hereditary ataxias are\n",
      "categorized by mode of inheritance and gene (or chromosome locus) in which\n",
      "pathogenic variants occur.\n",
      "DIAGNOSIS/TESTING: Inherited (genetic) forms of ataxia must be distinguished from\n",
      "the many acquired (non-genetic) causes of ataxia. The genetic forms of ataxia are\n",
      "diagnosed by family history, physical examination, neuroimaging, and molecular\n",
      "genetic testing.\n",
      "GENETIC COUNSELING: The hereditary ataxias can be inherited in an autosomal\n",
      "dominant, autosomal recessive, or X-linked manner. Genetic counseling and risk\n",
      "assessment depend on determination of the specific cause of an inherited ataxia\n",
      "in an individual.\n",
      "MANAGEMENT: Treatment of manifestations: Canes, walkers, and wheelchairs for gait\n",
      "ataxia; use of special devices to assist with handwriting, buttoning, and use of \n",
      "eating utensils; speech therapy and/or computer-based devices for those with\n",
      "dysarthria and severe speech deficits. Prevention of primary manifestations: With\n",
      "the exception of vitamin E therapy for ataxia with vitamin E deficiency (AVED),\n",
      "no specific treatments exist for hereditary ataxia.\n",
      "\n",
      "PMID: 20301317 \n",
      "\n",
      "\n",
      "26. J Am Soc Nephrol. 2009 Feb;20(2):363-79. doi: 10.1681/ASN.2008040406. Epub 2008\n",
      "Dec 3.\n",
      "\n",
      "Large-scale proteomics and phosphoproteomics of urinary exosomes.\n",
      "\n",
      "Gonzales PA(1), Pisitkun T, Hoffert JD, Tchapyjnikov D, Star RA, Kleta R, Wang\n",
      "NS, Knepper MA.\n",
      "\n",
      "Author information: \n",
      "(1)Laboratory of Kidney and Electrolyte Metabolism, National Heart, Lung, and\n",
      "Blood Institute, National Institutes of Health, Bethesda, MD 20892, USA.\n",
      "\n",
      "Normal human urine contains large numbers of exosomes, which are 40- to 100-nm\n",
      "vesicles that originate as the internal vesicles in multivesicular bodies from\n",
      "every renal epithelial cell type facing the urinary space. Here, we used LC-MS/MS\n",
      "to profile the proteome of human urinary exosomes. Overall, the analysis\n",
      "identified 1132 proteins unambiguously, including 177 that are represented on the\n",
      "Online Mendelian Inheritance in Man database of disease-related genes, suggesting\n",
      "that exosome analysis is a potential approach to discover urinary biomarkers. We \n",
      "extended the proteomic analysis to phosphoproteomic profiling using neutral loss \n",
      "scanning, and this yielded multiple novel phosphorylation sites, including\n",
      "serine-811 in the thiazide-sensitive Na-Cl co-transporter, NCC. To demonstrate\n",
      "the potential use of exosome analysis to identify a genetic renal disease, we\n",
      "carried out immunoblotting of exosomes from urine samples of patients with a\n",
      "clinical diagnosis of Bartter syndrome type I, showing an absence of the\n",
      "sodium-potassium-chloride co-transporter 2, NKCC2. The proteomic data are\n",
      "publicly accessible at http://dir.nhlbi.nih.gov/papers/lkem/exosome/.\n",
      "\n",
      "DOI: 10.1681/ASN.2008040406 \n",
      "PMCID: PMC2637050\n",
      "PMID: 19056867  [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "27. Nature. 2006 May 18;441(7091):315-21.\n",
      "\n",
      "The DNA sequence and biological annotation of human chromosome 1.\n",
      "\n",
      "Gregory SG(1), Barlow KF, McLay KE, Kaul R, Swarbreck D, Dunham A, Scott CE, Howe\n",
      "KL, Woodfine K, Spencer CC, Jones MC, Gillson C, Searle S, Zhou Y, Kokocinski F, \n",
      "McDonald L, Evans R, Phillips K, Atkinson A, Cooper R, Jones C, Hall RE, Andrews \n",
      "TD, Lloyd C, Ainscough R, Almeida JP, Ambrose KD, Anderson F, Andrew RW, Ashwell \n",
      "RI, Aubin K, Babbage AK, Bagguley CL, Bailey J, Beasley H, Bethel G, Bird CP,\n",
      "Bray-Allen S, Brown JY, Brown AJ, Buckley D, Burton J, Bye J, Carder C, Chapman\n",
      "JC, Clark SY, Clarke G, Clee C, Cobley V, Collier RE, Corby N, Coville GJ, Davies\n",
      "J, Deadman R, Dunn M, Earthrowl M, Ellington AG, Errington H, Frankish A,\n",
      "Frankland J, French L, Garner P, Garnett J, Gay L, Ghori MR, Gibson R, Gilby LM, \n",
      "Gillett W, Glithero RJ, Grafham DV, Griffiths C, Griffiths-Jones S, Grocock R,\n",
      "Hammond S, Harrison ES, Hart E, Haugen E, Heath PD, Holmes S, Holt K, Howden PJ, \n",
      "Hunt AR, Hunt SE, Hunter G, Isherwood J, James R, Johnson C, Johnson D, Joy A,\n",
      "Kay M, Kershaw JK, Kibukawa M, Kimberley AM, King A, Knights AJ, Lad H, Laird G, \n",
      "Lawlor S, Leongamornlert DA, Lloyd DM, Loveland J, Lovell J, Lush MJ, Lyne R,\n",
      "Martin S, Mashreghi-Mohammadi M, Matthews L, Matthews NS, McLaren S, Milne S,\n",
      "Mistry S, Moore MJ, Nickerson T, O'Dell CN, Oliver K, Palmeiri A, Palmer SA,\n",
      "Parker A, Patel D, Pearce AV, Peck AI, Pelan S, Phelps K, Phillimore BJ, Plumb R,\n",
      "Rajan J, Raymond C, Rouse G, Saenphimmachak C, Sehra HK, Sheridan E, Shownkeen R,\n",
      "Sims S, Skuce CD, Smith M, Steward C, Subramanian S, Sycamore N, Tracey A,\n",
      "Tromans A, Van Helmond Z, Wall M, Wallis JM, White S, Whitehead SL, Wilkinson JE,\n",
      "Willey DL, Williams H, Wilming L, Wray PW, Wu Z, Coulson A, Vaudin M, Sulston JE,\n",
      "Durbin R, Hubbard T, Wooster R, Dunham I, Carter NP, McVean G, Ross MT, Harrow J,\n",
      "Olson MV, Beck S, Rogers J, Bentley DR, Banerjee R, Bryant SP, Burford DC,\n",
      "Burrill WD, Clegg SM, Dhami P, Dovey O, Faulkner LM, Gribble SM, Langford CF,\n",
      "Pandian RD, Porter KM, Prigmore E.\n",
      "\n",
      "Author information: \n",
      "(1)The Wellcome Trust Sanger Institute, The Wellcome Trust Genome Campus,\n",
      "Hinxton, Cambridgeshire CB10 1SA, UK. sgregory@chg.duhs.duke.edu\n",
      "\n",
      "Erratum in\n",
      "    Nature. 2006 Oct 26;443(7114):1013. Banerjee, R [added]; Bryant, SP [added];\n",
      "Burford, DC [added]; Burrill, WDH [added]; Clegg, SM [added]; Dhami, P [added];\n",
      "Dovey, O [added]; Faulkner, LM [added]; Gribble, SM [added]; Langford, CF\n",
      "[added]; Pandian, RD [added]; Porter, KM [added]; Prigmore, E.\n",
      "\n",
      "The reference sequence for each human chromosome provides the framework for\n",
      "understanding genome function, variation and evolution. Here we report the\n",
      "finished sequence and biological annotation of human chromosome 1. Chromosome 1\n",
      "is gene-dense, with 3,141 genes and 991 pseudogenes, and many coding sequences\n",
      "overlap. Rearrangements and mutations of chromosome 1 are prevalent in cancer and\n",
      "many other diseases. Patterns of sequence variation reveal signals of recent\n",
      "selection in specific genes that may contribute to human fitness, and also in\n",
      "regions where no function is evident. Fine-scale recombination occurs in hotspots\n",
      "of varying intensity along the sequence, and is enriched near genes. These and\n",
      "other studies of human biology and disease encoded within chromosome 1 are made\n",
      "possible with the highly accurate annotated sequence, as part of the completed\n",
      "set of chromosome sequences that comprise the reference human genome.\n",
      "\n",
      "DOI: 10.1038/nature04727 \n",
      "PMID: 16710414  [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "28. Genomics. 2004 Sep;84(3):536-49.\n",
      "\n",
      "Analysis of the human VPS13 gene family.\n",
      "\n",
      "Velayos-Baeza A(1), Vettori A, Copley RR, Dobson-Stone C, Monaco AP.\n",
      "\n",
      "Author information: \n",
      "(1)Wellcome Trust Centre for Human Genetics, University of Oxford, Headington,\n",
      "OX3 7BN Oxford, UK.\n",
      "\n",
      "The gene mutated in chorea-acanthocytosis (CHAC; approved gene symbol VPS13A)\n",
      "encodes chorein, a protein similar to yeast Vps13p. We detected several similar\n",
      "putative human proteins by BLAST analysis of chorein. We characterized the\n",
      "structure of three new genes encoding these CHAC-similar proteins, located on\n",
      "chromosomes 1p36, 8q22, and 15q21. The most similar gene in yeast to all four\n",
      "human genes is Vps13, and therefore the human genes were named VPS13A (CHAC,\n",
      "9q21), VPS13B (8q22), VPS13C (15q21), and VPS13D (1p36). VPS13B has recently been\n",
      "reported as COH1, altered in Cohen syndrome. For each gene, we describe several\n",
      "alternative splicing variants; at least two transcripts per gene are major forms.\n",
      "The expression pattern of these genes is ubiquitous, with some tissue-specific\n",
      "differences between several transcript variants. Protein sequence comparisons\n",
      "suggest that intramolecular duplications have played an important role in the\n",
      "evolution of this gene family.\n",
      "\n",
      "DOI: 10.1016/j.ygeno.2004.04.012 \n",
      "PMID: 15498460  [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "29. Genome Res. 2004 Oct;14(10B):2121-7.\n",
      "\n",
      "The status, quality, and expansion of the NIH full-length cDNA project: the\n",
      "Mammalian Gene Collection (MGC).\n",
      "\n",
      "Gerhard DS, Wagner L, Feingold EA, Shenmen CM, Grouse LH, Schuler G, Klein SL,\n",
      "Old S, Rasooly R, Good P, Guyer M, Peck AM, Derge JG, Lipman D, Collins FS, Jang \n",
      "W, Sherry S, Feolo M, Misquitta L, Lee E, Rotmistrovsky K, Greenhut SF, Schaefer \n",
      "CF, Buetow K, Bonner TI, Haussler D, Kent J, Kiekhaus M, Furey T, Brent M, Prange\n",
      "C, Schreiber K, Shapiro N, Bhat NK, Hopkins RF, Hsie F, Driscoll T, Soares MB,\n",
      "Casavant TL, Scheetz TE, Brown-stein MJ, Usdin TB, Toshiyuki S, Carninci P, Piao \n",
      "Y, Dudekula DB, Ko MS, Kawakami K, Suzuki Y, Sugano S, Gruber CE, Smith MR,\n",
      "Simmons B, Moore T, Waterman R, Johnson SL, Ruan Y, Wei CL, Mathavan S, Gunaratne\n",
      "PH, Wu J, Garcia AM, Hulyk SW, Fuh E, Yuan Y, Sneed A, Kowis C, Hodgson A, Muzny \n",
      "DM, McPherson J, Gibbs RA, Fahey J, Helton E, Ketteman M, Madan A, Rodrigues S,\n",
      "Sanchez A, Whiting M, Madari A, Young AC, Wetherby KD, Granite SJ, Kwong PN,\n",
      "Brinkley CP, Pearson RL, Bouffard GG, Blakesly RW, Green ED, Dickson MC,\n",
      "Rodriguez AC, Grimwood J, Schmutz J, Myers RM, Butterfield YS, Griffith M,\n",
      "Griffith OL, Krzywinski MI, Liao N, Morin R, Palmquist D, Petrescu AS, Skalska U,\n",
      "Smailus DE, Stott JM, Schnerch A, Schein JE, Jones SJ, Holt RA, Baross A, Marra\n",
      "MA, Clifton S, Makowski KA, Bosak S, Malek J; MGC Project Team.\n",
      "\n",
      "Erratum in\n",
      "    Genome Res. 2006 Jun;16(6):804. Morrin, Ryan [corrected to Morin, Ryan].\n",
      "\n",
      "The National Institutes of Health's Mammalian Gene Collection (MGC) project was\n",
      "designed to generate and sequence a publicly accessible cDNA resource containing \n",
      "a complete open reading frame (ORF) for every human and mouse gene. The project\n",
      "initially used a random strategy to select clones from a large number of cDNA\n",
      "libraries from diverse tissues. Candidate clones were chosen based on 5'-EST\n",
      "sequences, and then fully sequenced to high accuracy and analyzed by algorithms\n",
      "developed for this project. Currently, more than 11,000 human and 10,000 mouse\n",
      "genes are represented in MGC by at least one clone with a full ORF. The random\n",
      "selection approach is now reaching a saturation point, and a transition to\n",
      "protocols targeted at the missing transcripts is now required to complete the\n",
      "mouse and human collections. Comparison of the sequence of the MGC clones to\n",
      "reference genome sequences reveals that most cDNA clones are of very high\n",
      "sequence quality, although it is likely that some cDNAs may carry missense\n",
      "variants as a consequence of experimental artifact, such as PCR, cloning, or\n",
      "reverse transcriptase errors. Recently, a rat cDNA component was added to the\n",
      "project, and ongoing frog (Xenopus) and zebrafish (Danio) cDNA projects were\n",
      "expanded to take advantage of the high-throughput MGC pipeline.\n",
      "\n",
      "DOI: 10.1101/gr.2596504 \n",
      "PMCID: PMC528928\n",
      "PMID: 15489334  [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "30. Nat Genet. 2004 Jan;36(1):40-5. Epub 2003 Dec 21.\n",
      "\n",
      "Complete sequencing and characterization of 21,243 full-length human cDNAs.\n",
      "\n",
      "Ota T(1), Suzuki Y, Nishikawa T, Otsuki T, Sugiyama T, Irie R, Wakamatsu A,\n",
      "Hayashi K, Sato H, Nagai K, Kimura K, Makita H, Sekine M, Obayashi M, Nishi T,\n",
      "Shibahara T, Tanaka T, Ishii S, Yamamoto J, Saito K, Kawai Y, Isono Y, Nakamura\n",
      "Y, Nagahari K, Murakami K, Yasuda T, Iwayanagi T, Wagatsuma M, Shiratori A, Sudo \n",
      "H, Hosoiri T, Kaku Y, Kodaira H, Kondo H, Sugawara M, Takahashi M, Kanda K, Yokoi\n",
      "T, Furuya T, Kikkawa E, Omura Y, Abe K, Kamihara K, Katsuta N, Sato K, Tanikawa\n",
      "M, Yamazaki M, Ninomiya K, Ishibashi T, Yamashita H, Murakawa K, Fujimori K,\n",
      "Tanai H, Kimata M, Watanabe M, Hiraoka S, Chiba Y, Ishida S, Ono Y, Takiguchi S, \n",
      "Watanabe S, Yosida M, Hotuta T, Kusano J, Kanehori K, Takahashi-Fujii A, Hara H, \n",
      "Tanase TO, Nomura Y, Togiya S, Komai F, Hara R, Takeuchi K, Arita M, Imose N,\n",
      "Musashino K, Yuuki H, Oshima A, Sasaki N, Aotsuka S, Yoshikawa Y, Matsunawa H,\n",
      "Ichihara T, Shiohata N, Sano S, Moriya S, Momiyama H, Satoh N, Takami S,\n",
      "Terashima Y, Suzuki O, Nakagawa S, Senoh A, Mizoguchi H, Goto Y, Shimizu F,\n",
      "Wakebe H, Hishigaki H, Watanabe T, Sugiyama A, Takemoto M, Kawakami B, Yamazaki\n",
      "M, Watanabe K, Kumagai A, Itakura S, Fukuzumi Y, Fujimori Y, Komiyama M, Tashiro \n",
      "H, Tanigami A, Fujiwara T, Ono T, Yamada K, Fujii Y, Ozaki K, Hirao M, Ohmori Y, \n",
      "Kawabata A, Hikiji T, Kobatake N, Inagaki H, Ikema Y, Okamoto S, Okitani R,\n",
      "Kawakami T, Noguchi S, Itoh T, Shigeta K, Senba T, Matsumura K, Nakajima Y,\n",
      "Mizuno T, Morinaga M, Sasaki M, Togashi T, Oyama M, Hata H, Watanabe M, Komatsu\n",
      "T, Mizushima-Sugano J, Satoh T, Shirai Y, Takahashi Y, Nakagawa K, Okumura K,\n",
      "Nagase T, Nomura N, Kikuchi H, Masuho Y, Yamashita R, Nakai K, Yada T, Nakamura\n",
      "Y, Ohara O, Isogai T, Sugano S.\n",
      "\n",
      "Author information: \n",
      "(1)Helix Research Institute, 1532-3 Yana, Kisarazu, Chiba 292-0812, Japan.\n",
      "\n",
      "As a base for human transcriptome and functional genomics, we created the\n",
      "\"full-length long Japan\" (FLJ) collection of sequenced human cDNAs. We determined\n",
      "the entire sequence of 21,243 selected clones and found that 14,490 cDNAs (10,897\n",
      "clusters) were unique to the FLJ collection. About half of them (5,416) seemed to\n",
      "be protein-coding. Of those, 1,999 clusters had not been predicted by\n",
      "computational methods. The distribution of GC content of nonpredicted cDNAs had a\n",
      "peak at approximately 58% compared with a peak at approximately 42%for predicted \n",
      "cDNAs. Thus, there seems to be a slight bias against GC-rich transcripts in\n",
      "current gene prediction procedures. The rest of the cDNAs unique to the FLJ\n",
      "collection (5,481) contained no obvious open reading frames (ORFs) and thus are\n",
      "candidate noncoding RNAs. About one-fourth of them (1,378) showed a clear pattern\n",
      "of splicing. The distribution of GC content of noncoding cDNAs was narrow and had\n",
      "a peak at approximately 42%, relatively low compared with that of protein-coding \n",
      "cDNAs.\n",
      "\n",
      "DOI: 10.1038/ng1285 \n",
      "PMID: 14702039  [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "31. Proc Natl Acad Sci U S A. 2002 Dec 24;99(26):16899-903. Epub 2002 Dec 11.\n",
      "\n",
      "Generation and initial analysis of more than 15,000 full-length human and mouse\n",
      "cDNA sequences.\n",
      "\n",
      "Strausberg RL(1), Feingold EA, Grouse LH, Derge JG, Klausner RD, Collins FS,\n",
      "Wagner L, Shenmen CM, Schuler GD, Altschul SF, Zeeberg B, Buetow KH, Schaefer CF,\n",
      "Bhat NK, Hopkins RF, Jordan H, Moore T, Max SI, Wang J, Hsieh F, Diatchenko L,\n",
      "Marusina K, Farmer AA, Rubin GM, Hong L, Stapleton M, Soares MB, Bonaldo MF,\n",
      "Casavant TL, Scheetz TE, Brownstein MJ, Usdin TB, Toshiyuki S, Carninci P, Prange\n",
      "C, Raha SS, Loquellano NA, Peters GJ, Abramson RD, Mullahy SJ, Bosak SA, McEwan\n",
      "PJ, McKernan KJ, Malek JA, Gunaratne PH, Richards S, Worley KC, Hale S, Garcia\n",
      "AM, Gay LJ, Hulyk SW, Villalon DK, Muzny DM, Sodergren EJ, Lu X, Gibbs RA, Fahey \n",
      "J, Helton E, Ketteman M, Madan A, Rodrigues S, Sanchez A, Whiting M, Madan A,\n",
      "Young AC, Shevchenko Y, Bouffard GG, Blakesley RW, Touchman JW, Green ED, Dickson\n",
      "MC, Rodriguez AC, Grimwood J, Schmutz J, Myers RM, Butterfield YS, Krzywinski MI,\n",
      "Skalska U, Smailus DE, Schnerch A, Schein JE, Jones SJ, Marra MA; Mammalian Gene \n",
      "Collection Program Team.\n",
      "\n",
      "Author information: \n",
      "(1)National Cancer Institute, Bethessda, MD 20892-2580, USA. rls@nih.gov\n",
      "\n",
      "The National Institutes of Health Mammalian Gene Collection (MGC) Program is a\n",
      "multiinstitutional effort to identify and sequence a cDNA clone containing a\n",
      "complete ORF for each human and mouse gene. ESTs were generated from libraries\n",
      "enriched for full-length cDNAs and analyzed to identify candidate full-ORF\n",
      "clones, which then were sequenced to high accuracy. The MGC has currently\n",
      "sequenced and verified the full ORF for a nonredundant set of >9,000 human and\n",
      ">6,000 mouse genes. Candidate full-ORF clones for an additional 7,800 human and\n",
      "3,500 mouse genes also have been identified. All MGC sequences and clones are\n",
      "available without restriction through public databases and clone distribution\n",
      "networks (see http:mgc.nci.nih.gov).\n",
      "\n",
      "DOI: 10.1073/pnas.242603899 \n",
      "PMCID: PMC139241\n",
      "PMID: 12477932  [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "32. Genome Res. 2002 Nov;12(11):1773-84.\n",
      "\n",
      "Protein-protein interactions between large proteins: two-hybrid screening using a\n",
      "functionally classified library composed of long cDNAs.\n",
      "\n",
      "Nakayama M(1), Kikuno R, Ohara O.\n",
      "\n",
      "Author information: \n",
      "(1)Department of Human Gene Research, Kazusa DNA Research Institute, Kisarazu,\n",
      "Chiba 292-0818, Japan. nmanabu@kazusa.or.jp\n",
      "\n",
      "Large proteins have multiple domains that are potentially capable of binding many\n",
      "kinds of partners. It is conceivable, therefore, that such proteins could\n",
      "function as an intricate framework of assembly protein complexes. To\n",
      "comprehensively study protein-protein interactions between large KIAA proteins,\n",
      "we have constructed a library composed of 1087 KIAA cDNA clones based on prior\n",
      "functional classifications done in silico. We were guided by two principles that \n",
      "raise the success rate for detecting interactions per tested combination: we\n",
      "avoided testing low-probability combinations, and reduced the number of potential\n",
      "false negatives that arise from the fact that large proteins cannot reliably be\n",
      "expressed in yeast. The latter was addressed by constructing a cDNA library\n",
      "comprised of random fragments encoding large proteins. Cytoplasmic domains of\n",
      "KIAA transmembrane proteins (>1000 amino acids) were used as bait for yeast\n",
      "two-hybrid screening. Our analyses reveal that several KIAA proteins bearing a\n",
      "transmembrane region have the capability of binding to other KIAA proteins\n",
      "containing domains (e.g., PDZ, SH3, rhoGEF, and spectrin) known to be localized\n",
      "to highly specialized submembranous sites, indicating that they participate in\n",
      "cellular junction formation, receptor or channel clustering, and intracellular\n",
      "signaling events. Our representative library should be a very useful resource for\n",
      "detecting previously unidentified interactions because it complements\n",
      "conventional expression libraries, which seldom contain large cDNAs.\n",
      "\n",
      "DOI: 10.1101/gr.406902 \n",
      "PMCID: PMC187542\n",
      "PMID: 12421765  [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "33. DNA Res. 1997 Oct 31;4(5):345-9.\n",
      "\n",
      "Characterization of cDNA clones in size-fractionated cDNA libraries from human\n",
      "brain.\n",
      "\n",
      "Seki N(1), Ohira M, Nagase T, Ishikawa K, Miyajima N, Nakajima D, Nomura N, Ohara\n",
      "O.\n",
      "\n",
      "Author information: \n",
      "(1)Kazusa DNA Research Institute, Chiba, Japan. nseki@kazusa.or.jp\n",
      "\n",
      "To evaluate the size-fractionated cDNA libraries of human brain previously\n",
      "constructed (O. O'hara et al. DNA Research, 4, 53-59, 1997), the occurrence of\n",
      "chimeric clones and the content of clones with coding potentiality were analyzed \n",
      "using the randomly sampled clones with insert sizes of 5 to 7 kb. When the\n",
      "chromosomal location of 30 clones was determined by the radiation-hybrid mapping \n",
      "method, the map positions assigned from the 3'- and 5'-end sequences separately\n",
      "were coincident for 29 clones, suggesting that the occurrence of chimeric clones \n",
      "is at most 1/30. Using 91 clones mapped to chromosome 1, the content of clones\n",
      "that have the potentiality coding for proteins larger than 100 amino acid\n",
      "residues was estimated to be approximately 50% (46 out of 91 clones) on the basis\n",
      "of nucleotide sequence analysis and coding potentiality assay in vitro. No\n",
      "significant open reading frames were detected in the remaining clones. Although\n",
      "the clones coding for short peptides may not have been included in the above\n",
      "estimation, the libraries constructed from the whole brain mRNA fraction appear\n",
      "to contain a considerable amount of clones corresponding to the 5'-truncated\n",
      "transcripts in an unprocessed form and/or those with long 3'-untranslated\n",
      "regions.\n",
      "\n",
      "DOI: 10.1093/dnares/4.5.345 \n",
      "PMID: 9455484  [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "34. Genome Res. 1997 Apr;7(4):353-8.\n",
      "\n",
      "Large-scale concatenation cDNA sequencing.\n",
      "\n",
      "Yu W, Andersson B, Worley KC, Muzny DM, Ding Y, Liu W, Ricafrente JY, Wentland\n",
      "MA, Lennon G, Gibbs RA.\n",
      "\n",
      "A total of 100 kb of DNA derived from 69 individual human brain cDNA clones of\n",
      "0.7-2.0 kb were sequenced by concatenated cDNA sequencing (CCS), whereby multiple\n",
      "individual DNA fragments are sequenced simultaneously in a single shotgun\n",
      "library. The method yielded accurate sequences and a similar efficiency compared \n",
      "with other shotgun libraries constructed from single DNA fragments (> 20 kb).\n",
      "Computer analyses were carried out on 65 cDNA clone sequences and their\n",
      "corresponding end sequences to examine both nucleic acid and amino acid sequence \n",
      "similarities in the databases. Thirty-seven clones revealed no DNA database\n",
      "matches, 12 clones generated exact matches (> or = 98% identity), and 16 clones\n",
      "generated nonexact matches (57%-97% identity) to either known human or other\n",
      "species genes. Of those 28 matched clones, 8 had corresponding end sequences that\n",
      "failed to identify similarities. In a protein similarity search, 27 clone\n",
      "sequences displayed significant matches, whereas only 20 of the end sequences had\n",
      "matches to known protein sequences. Our data indicate that full-length cDNA\n",
      "insert sequences provide significantly more nucleic acid and protein sequence\n",
      "similarity matches than expressed sequence tags (ESTs) for database searching.\n",
      "\n",
      "DOI: 10.1101/gr.7.4.353 \n",
      "PMCID: PMC139146\n",
      "PMID: 9110174  [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "35. Anal Biochem. 1996 Apr 5;236(1):107-13.\n",
      "\n",
      "A \"double adaptor\" method for improved shotgun library construction.\n",
      "\n",
      "Andersson B(1), Wentland MA, Ricafrente JY, Liu W, Gibbs RA.\n",
      "\n",
      "Author information: \n",
      "(1)Department of Molecular and Human Genetics, Baylor College of Medicine, One\n",
      "Baylor Plaza, Houston, Texas, 77030, USA.\n",
      "\n",
      "The efficiency of shotgun DNA sequencing depends to a great extent on the quality\n",
      "of the random-subclone libraries used. We here describe a novel \"double adaptor\" \n",
      "strategy for efficient construction of high-quality shotgun libraries. In this\n",
      "method, randomly sheared and end-repaired fragments are ligated to\n",
      "oligonucleotide adaptors creating 12-base overhangs. Nonphosphorylated\n",
      "oligonucleotides are used, which prevents formation of adaptor dimers and ensures\n",
      "efficient ligation of insert to adaptor. The vector is prepared from a modified\n",
      "M13 vector, by KpnI/PstI digestion followed by ligation to oligonucleotides with \n",
      "ends complementary to the overhangs created in the digest. These adaptors create \n",
      "5'-overhangs complementary to those on the inserts. Following annealing of insert\n",
      "to vector, the DNA is directly used for transformation without a ligation step.\n",
      "This protocol is robust and shows three- to fivefold higher yield of clones\n",
      "compared to previous protocols. No chimeric clones can be detected and the\n",
      "background of clones without an insert is <1%. The procedure is rapid and shows\n",
      "potential for automation.\n",
      "\n",
      "DOI: 10.1006/abio.1996.0138 \n",
      "PMID: 8619474  [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "36. VPS13D Movement Disorder.\n",
      "\n",
      "Meijer IA(1).\n",
      "In: Adam MP, Ardinger HH, Pagon RA, Wallace SE, Bean LJH, Stephens K, Amemiya A, \n",
      "editors. GeneReviews [Internet]. Seattle (WA): University of Washington,\n",
      "Seattle; 1993-2019.\n",
      "2019 Feb 21.\n",
      "\n",
      "Author information: \n",
      "(1)CHU Sainte-Justine, Universit de Montral, Montral, Qubec, Canada\n",
      "\n",
      "CLINICAL CHARACTERISTICS: VPS13D movement disorder is a hyperkinetic movement\n",
      "disorder (dystonia, chorea, and/or ataxia) of variable age of onset that can be\n",
      "associated with developmental delay. Onset ranges from birth to adulthood.\n",
      "Individuals can present in childhood with motor delays and gait instability.\n",
      "Cognitive impairment ranging from mild intellectual disability to developmental\n",
      "delay has been reported, and several individuals have normal cognitive function. \n",
      "Individuals have also presented as young adults with gait difficulties caused by \n",
      "spastic ataxia or ataxia. In addition to gait ataxia, affected individuals had\n",
      "limb ataxia, dysarthria, and eye movement abnormalities (macro-saccadic\n",
      "oscillations, nystagmus, and saccadic pursuit). Additional features reported in\n",
      "some individuals include peripheral neuropathy and/or seizures. The disorder\n",
      "progresses to spastic ataxia or generalized dystonia, which can lead to loss of\n",
      "independent ambulation.\n",
      "DIAGNOSIS: The diagnosis of VPS13D movement disorder is established in a proband \n",
      "by identification of biallelic pathogenic variants in VPS13D on molecular genetic\n",
      "testing.\n",
      "MANAGEMENT: Treatment of manifestations: Standard treatment for seizures;\n",
      "spasticity treatments include baclofen, tizanidine, benzodiazepines, dantrolene\n",
      "sodium, gabapentin, botulinum toxin injections, and intrathecal baclofen;\n",
      "treatment options for dystonia include trihexyphenidyl, botulinum toxin\n",
      "injections, benzodiazepines, baclofen, and levodopa. A multidisciplinary team\n",
      "including occupational and physical therapists and a physiatrist is important;\n",
      "supportive developmental therapies should be provided as needed. Surveillance:\n",
      "Monitor for urinary urgency, contractures, and seizures; evaluate fall risk and\n",
      "gait stability; monitor developmental progress and educational needs.\n",
      "GENETIC COUNSELING: VPS13D movement disorder is inherited in an autosomal\n",
      "recessive manner. At conception, each sib of an affected individual has a 25%\n",
      "chance of being affected, a 50% chance of being an asymptomatic carrier, and a\n",
      "25% chance of being unaffected and not a carrier. Carrier testing for at-risk\n",
      "relatives and prenatal testing for pregnancies at increased risk are possible if \n",
      "the VPS13D pathogenic variants in the family are known.\n",
      "\n",
      "PMID: 30789691 \n",
      "\n",
      "\n",
      "37. Sci Rep. 2018 Sep 26;8(1):14375. doi: 10.1038/s41598-018-32685-8.\n",
      "\n",
      "USP45 and Spindly are part of the same complex implicated in cell migration.\n",
      "\n",
      "Conte C(1)(2), Griffis ER(1)(3), Hickson I(4)(5), Perez-Oliva AB(6).\n",
      "\n",
      "Author information: \n",
      "(1)MRC Protein Phosphorylation and Ubiquitylation Unit, College of Life Sciences,\n",
      "University of Dundee, Dundee, UK.\n",
      "(2)Cell and Developmental Biology, Center of Genomic Regulation (CRG), Barcelona,\n",
      "Spain.\n",
      "(3)Nikon Instruments Incorporated, Melville, NY, 11747, USA.\n",
      "(4)Janssen Research & Development, LLC, Spring House, PA, USA.\n",
      "(5)Northern Institute for Cancer Research, Newcastle University, Medical School, \n",
      "Newcastle upon Tyne, NE2 4HH, UK.\n",
      "(6)MRC Protein Phosphorylation and Ubiquitylation Unit, College of Life Sciences,\n",
      "University of Dundee, Dundee, UK. anabpo@um.es.\n",
      "\n",
      "Ubiquitylation is a protein modification implicated in several cellular\n",
      "processes. This process is reversible by the action of deubiquinating enzymes\n",
      "(DUBs). USP45 is a ubiquitin specific protease about which little is known, aside\n",
      "from roles in DNA damage repair and differentiation of the vertebrate retina.\n",
      "Here, by using mass spectrometry we have identified Spindly as a new target of\n",
      "USP45. Our data show that Spindly and USP45 are part of the same complex and that\n",
      "their interaction specifically depends on the catalytic activity of USP45. In\n",
      "addition, we describe the type of ubiquitin chains associated with the complex\n",
      "that can be cleaved by USP45, with a preferential activity on K48 ubiquitin chain\n",
      "type and potentially K6. Here, we also show that Spindly is mono-ubiquitylated\n",
      "and this can be specifically removed by USP45 in its active form but not by the\n",
      "catalytic inactive form. Lastly, we identified a new role for USP45 in cell\n",
      "migration, similar to that which was recently described for Spindly.\n",
      "\n",
      "DOI: 10.1038/s41598-018-32685-8 \n",
      "PMCID: PMC6158257\n",
      "PMID: 30258100 \n",
      "\n",
      "\n",
      "38. Ann Neurol. 2018 Jun;83(6):1075-1088. doi: 10.1002/ana.25220. Epub 2018 Jun 30.\n",
      "\n",
      "Mutations in VPS13D lead to a new recessive ataxia with spasticity and\n",
      "mitochondrial defects.\n",
      "\n",
      "Seong E(1), Insolera R(2), Dulovic M(3), Kamsteeg EJ(4), Trinh J(3), Brggemann\n",
      "N(5), Sandford E(1), Li S(1), Ozel AB(6), Li JZ(6)(7), Jewett T(8), Kievit\n",
      "AJA(9), Mnchau A(3), Shakkottai V(10), Klein C(3), Collins CA(2), Lohmann K(3), \n",
      "van de Warrenburg BP(11), Burmeister M(1)(6)(7)(12).\n",
      "\n",
      "Author information: \n",
      "(1)Molecular and Behavioral Neuroscience Institute, University of Michigan, Ann\n",
      "Arbor, MI.\n",
      "(2)Department of Molecular, Cellular, and Developmental Biology, University of\n",
      "Michigan, Ann Arbor, MI.\n",
      "(3)Institute of Neurogenetics, University of Lbeck, Lbeck, Germany.\n",
      "(4)Department of Human Genetics, Radboud University Medical Centre, Nijmegen, the\n",
      "Netherlands.\n",
      "(5)Department of Neurology, University of Lbeck, Lbeck, Germany.\n",
      "(6)Department of Human Genetics, University of Michigan, Ann Arbor, MI.\n",
      "(7)Department of Computational Medicine and Bioinformatics, University of\n",
      "Michigan, Ann Arbor, MI.\n",
      "(8)Department of Pediatrics, Section on Medical Genetics, Wake Forest School of\n",
      "Medicine, Winston-Salem, NC.\n",
      "(9)Department of Clinical Genetics, Erasmus MC, Rotterdam, the Netherlands.\n",
      "(10)Departments of Neurology and Molecular and Integrative Physiology, University\n",
      "of Michigan, Ann Arbor, MI.\n",
      "(11)Department of Neurology, Donders Institute for Brain, Cognition, and\n",
      "Behavior, Radboud University Medical Centre, Nijmegen, the Netherlands.\n",
      "(12)Department of Psychiatry, University of Michigan, Ann Arbor, MI.\n",
      "\n",
      "Comment in\n",
      "    Ann Neurol. 2018 Jun;83(6):1072-1074.\n",
      "\n",
      "OBJECTIVE: To identify novel causes of recessive ataxias, including\n",
      "spinocerebellar ataxia with saccadic intrusions, spastic ataxias, and spastic\n",
      "paraplegia.\n",
      "METHODS: In an international collaboration, we independently performed exome\n",
      "sequencing in 7 families with recessive ataxia and/or spastic paraplegia. To\n",
      "evaluate the role of VPS13D mutations, we evaluated a Drosophila knockout model\n",
      "and investigated mitochondrial function in patient-derived fibroblast cultures.\n",
      "RESULTS: Exome sequencing identified compound heterozygous mutations in VPS13D on\n",
      "chromosome 1p36 in all 7 families. This included a large family with 5 affected\n",
      "siblings with spinocerebellar ataxia with saccadic intrusions (SCASI), or\n",
      "spinocerebellar ataxia, recessive, type 4 (SCAR4). Linkage to chromosome 1p36 was\n",
      "found in this family with a logarithm of odds score of 3.1. The phenotypic\n",
      "spectrum in our 12 patients was broad. Although most presented with ataxia,\n",
      "additional or predominant spasticity was present in 5 patients. Disease onset\n",
      "ranged from infancy to 39 years, and symptoms were slowly progressive and\n",
      "included loss of independent ambulation in 5. All but 2 patients carried a\n",
      "loss-of-function (nonsense or splice site) mutation on one and a missense\n",
      "mutation on the other allele. Knockdown or removal of Vps13D in Drosophila\n",
      "neurons led to changes in mitochondrial morphology and impairment in\n",
      "mitochondrial distribution along axons. Patient fibroblasts showed altered\n",
      "morphology and functionality including reduced energy production.\n",
      "INTERPRETATION: Our study demonstrates that compound heterozygous mutations in\n",
      "VPS13D cause movement disorders along the ataxia-spasticity spectrum, making\n",
      "VPS13D the fourth VPS13 paralog involved in neurological disorders. Ann Neurol\n",
      "2018.\n",
      "\n",
      " 2018 American Neurological Association.\n",
      "\n",
      "DOI: 10.1002/ana.25220 \n",
      "PMCID: PMC6105379\n",
      "PMID: 29604224  [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "39. Nat Commun. 2018 Mar 22;9(1):1188. doi: 10.1038/s41467-018-03523-2.\n",
      "\n",
      "An AP-MS- and BioID-compatible MAC-tag enables comprehensive mapping of protein\n",
      "interactions and subcellular localizations.\n",
      "\n",
      "Liu X(1)(2), Salokas K(1)(2), Tamene F(1)(2)(3), Jiu Y(1)(2), Weldatsadik\n",
      "RG(1)(2)(3), hman T(1)(2)(3), Varjosalo M(4)(5)(6).\n",
      "\n",
      "Author information: \n",
      "(1)Institute of Biotechnology, University of Helsinki, Helsinki, 00014, Finland.\n",
      "(2)Helsinki Institute of Life Science, University of Helsinki, Helsinki, 00014,\n",
      "Finland.\n",
      "(3)Proteomics Unit, University of Helsinki, Helsinki, 00014, Finland.\n",
      "(4)Institute of Biotechnology, University of Helsinki, Helsinki, 00014, Finland. \n",
      "markku.varjosalo@helsinki.fi.\n",
      "(5)Helsinki Institute of Life Science, University of Helsinki, Helsinki, 00014,\n",
      "Finland. markku.varjosalo@helsinki.fi.\n",
      "(6)Proteomics Unit, University of Helsinki, Helsinki, 00014, Finland.\n",
      "markku.varjosalo@helsinki.fi.\n",
      "\n",
      "Protein-protein interactions govern almost all cellular functions. These complex \n",
      "networks of stable and transient associations can be mapped by affinity\n",
      "purification mass spectrometry (AP-MS) and complementary proximity-based labeling\n",
      "methods such as BioID. To exploit the advantages of both strategies, we here\n",
      "design and optimize an integrated approach combining AP-MS and BioID in a single \n",
      "construct, which we term MAC-tag. We systematically apply the MAC-tag approach to\n",
      "18 subcellular and 3 sub-organelle localization markers, generating a molecular\n",
      "context database, which can be used to define a protein's molecular location. In \n",
      "addition, we show that combining the AP-MS and BioID results makes it possible to\n",
      "obtain interaction distances within a protein complex. Taken together, our\n",
      "integrated strategy enables the comprehensive mapping of the physical and\n",
      "functional interactions of proteins, defining their molecular context and\n",
      "improving our understanding of the cellular interactome.\n",
      "\n",
      "DOI: 10.1038/s41467-018-03523-2 \n",
      "PMCID: PMC5864832\n",
      "PMID: 29568061  [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "40. Ann Neurol. 2018 Jun;83(6):1089-1095. doi: 10.1002/ana.25204. Epub 2018 Apr 10.\n",
      "\n",
      "Recessive mutations in VPS13D cause childhood onset movement disorders.\n",
      "\n",
      "Gauthier J(1), Meijer IA(2), Lessel D(3), Mencacci NE(4), Krainc D(4), Hempel\n",
      "M(3), Tsiakas K(5), Prokisch H(6)(7), Rossignol E(2), Helm MH(8), Rodan LH(9),\n",
      "Karamchandani J(10), Carecchio M(11), Lubbe SJ(4), Telegrafi A(12), Henderson\n",
      "LB(12), Lorenzo K(13), Wallace SE(14), Glass IA(14), Hamdan FF(1), Michaud\n",
      "JL(15), Rouleau GA(16), Campeau PM(15).\n",
      "\n",
      "Author information: \n",
      "(1)Molecular Diagnostic Laboratory and Division of Medical Genetics, Department\n",
      "of Pediatrics, Saint Justine University Hospital Center, Montreal, Canada.\n",
      "(2)Department of Neuroscience, University of Montreal, Montreal, Canada.\n",
      "(3)Institute of Human Genetics, University Medical Center Hamburg-Eppendorf,\n",
      "Hamburg, Germany.\n",
      "(4)Department of Neurology, Northwestern University, Feinberg School of Medicine,\n",
      "Chicago, IL.\n",
      "(5)Department of Pediatrics, University Medical Center Hamburg-Eppendorf,\n",
      "Hamburg, Germany.\n",
      "(6)Institute of Human Genetics, Helmholtz Center Munich, Neuherberg, Germany.\n",
      "(7)Institute of Human Genetics, Technical University Munich, Munich, Germany.\n",
      "(8)Maine Medical Partners Pediatric Specialty Care, Portland, ME.\n",
      "(9)Department of Neurology, Boston Children's Hospital, Boston, MA.\n",
      "(10)Department of Pathology, McGill University, Montreal Neurological Institute, \n",
      "Montreal, Canada.\n",
      "(11)Molecular Neurogenetics Unit, Institute for Research and Health Care (IRCCS) \n",
      "Foundation Carlo Besta Neurological Institute, Milan, Italy.\n",
      "(12)GeneDx, Gaithersburg, MD.\n",
      "(13)Kadlec Clinic Genetic Counseling, Richland, WA.\n",
      "(14)Division of Genetic Medicine, Department of Pediatrics, Seattle Children's\n",
      "Hospital and University of Washington, Seattle, WA.\n",
      "(15)Department of Pediatrics, Saint Justine University Hospital Center and\n",
      "University of Montreal, Montreal, Canada.\n",
      "(16)Montreal Neurological Institute, Department of Neurology and Neurosurgery,\n",
      "McGill University, Montreal, Canada.\n",
      "\n",
      "Comment in\n",
      "    Ann Neurol. 2018 Jun;83(6):1072-1074.\n",
      "\n",
      "VPS13 protein family members VPS13A through VPS13C have been associated with\n",
      "various recessive movement disorders. We describe the first disease association\n",
      "of rare recessive VPS13D variants including frameshift, missense, and partial\n",
      "duplication mutations with a novel complex, hyperkinetic neurological disorder.\n",
      "The clinical features include developmental delay, a childhood onset movement\n",
      "disorder (chorea, dystonia, or tremor), and progressive spastic ataxia or\n",
      "paraparesis. Characteristic brain magnetic resonance imaging shows basal ganglia \n",
      "or diffuse white matter T2 hyperintensities as seen in Leigh syndrome and\n",
      "choreoacanthocytosis. Muscle biopsy in 1 case showed mitochondrial aggregates and\n",
      "lipidosis, suggesting mitochondrial dysfunction. These findings underline the\n",
      "importance of the VPS13 complex in neurological diseases and a possible role in\n",
      "mitochondrial function. Ann Neurol 2018;83:1089-1095.\n",
      "\n",
      " 2018 American Neurological Association.\n",
      "\n",
      "DOI: 10.1002/ana.25204 \n",
      "PMID: 29518281  [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "41. Cell Discov. 2018 Feb 27;4:10. doi: 10.1038/s41421-018-0019-0. eCollection 2018.\n",
      "\n",
      "VIRMA mediates preferential m6A mRNA methylation in 3'UTR and near stop codon and\n",
      "associates with alternative polyadenylation.\n",
      "\n",
      "Yue Y(1), Liu J(2), Cui X(2), Cao J(1), Luo G(2), Zhang Z(1), Cheng T(1), Gao\n",
      "M(1), Shu X(1), Ma H(2), Wang F(3), Wang X(3), Shen B(4), Wang Y(3), Feng X(5),\n",
      "He C(2), Liu J(1)(5).\n",
      "\n",
      "Author information: \n",
      "(1)1MOE Key Laboratory of Macromolecular Synthesis and Functionalization,\n",
      "Department of Polymer Science and Engineering, Zhejiang University, Hangzhou,\n",
      "Zhejiang 310027 China.\n",
      "(2)2Department of Chemistry, Department of Biochemistry and Molecular Biology,\n",
      "Institute for Biophysical Dynamics, Howard Hughes Medical Institute, The\n",
      "University of Chicago, Chicago, IL 60637 USA.\n",
      "(3)3College of Animal Sciences, Key Laboratory of Molecular Nutrition, Ministry\n",
      "of Education, Zhejiang University, Hangzhou, Zhejiang 310058 China.\n",
      "(4)4State Key Laboratory of Reproductive Medicine, Department of Histology and\n",
      "Embryology, Nanjing Medical University, Nanjing, Jiangsu 210029 China.\n",
      "(5)5Life Sciences Institute, Zhejiang University, Hangzhou, Zhejiang 310058\n",
      "China.\n",
      "\n",
      "N6-methyladenosine (m6A) is enriched in 3'untranslated region (3'UTR) and near\n",
      "stop codon of mature polyadenylated mRNAs in mammalian systems and has regulatory\n",
      "roles in eukaryotic mRNA transcriptome switch. Significantly, the mechanism for\n",
      "this modification preference remains unknown, however. Herein we report a\n",
      "characterization of the full m6A methyltransferase complex in HeLa cells\n",
      "identifying METTL3/METTL14/WTAP/VIRMA/HAKAI/ZC3H13 as the key components, and we \n",
      "show that VIRMA mediates preferential mRNA methylation in 3'UTR and near stop\n",
      "codon. Biochemical studies reveal that VIRMA recruits the catalytic core\n",
      "components METTL3/METTL14/WTAP to guide region-selective methylations. Around 60%\n",
      "of VIRMA mRNA immunoprecipitation targets manifest strong m6A enrichment in\n",
      "3'UTR. Depletions of VIRMA and METTL3 induce 3'UTR lengthening of several hundred\n",
      "mRNAs with over 50% targets in common. VIRMA associates with polyadenylation\n",
      "cleavage factors CPSF5 and CPSF6 in an RNA-dependent manner. Depletion of CPSF5\n",
      "leads to significant shortening of 3'UTR of over 2800 mRNAs, 84% of which are\n",
      "modified with m6A and have increased m6A peak density in 3'UTR and near stop\n",
      "codon after CPSF5 knockdown. Together, our studies provide insights into m6A\n",
      "deposition specificity in 3'UTR and its correlation with alternative\n",
      "polyadenylation.\n",
      "\n",
      "DOI: 10.1038/s41421-018-0019-0 \n",
      "PMCID: PMC5826926\n",
      "PMID: 29507755 \n",
      "\n",
      "Conflict of interest statement: C.H. is a scientific founder of the Accent\n",
      "Therapeutics and a member of the Scientific Advisory Committee. The remaining\n",
      "authors declare that they have no conflict of interest.\n",
      "\n",
      "\n",
      "42. Curr Biol. 2018 Jan 22;28(2):287-295.e6. doi: 10.1016/j.cub.2017.11.064. Epub\n",
      "2018 Jan 4.\n",
      "\n",
      "Vps13D Encodes a Ubiquitin-Binding Protein that Is Required for the Regulation of\n",
      "Mitochondrial Size and Clearance.\n",
      "\n",
      "Anding AL(1), Wang C(2), Chang TK(1), Sliter DA(2), Powers CM(1), Hofmann K(3),\n",
      "Youle RJ(2), Baehrecke EH(4).\n",
      "\n",
      "Author information: \n",
      "(1)Department of Molecular, Cell and Cancer Biology, University of Massachusetts \n",
      "Medical School, Worcester, MA 01605, USA.\n",
      "(2)Biochemistry Section, Surgical Neurology Branch, National Institute of\n",
      "Neurological Disorders and Stroke, NIH, Bethesda, MD 20892, USA.\n",
      "(3)Institute for Genetics, University of Cologne, 50674 Cologne, Germany.\n",
      "(4)Department of Molecular, Cell and Cancer Biology, University of Massachusetts \n",
      "Medical School, Worcester, MA 01605, USA. Electronic address:\n",
      "eric.baehrecke@umassmed.edu.\n",
      "\n",
      "Comment in\n",
      "    Curr Biol. 2018 Jan 22;28(2):R66-R68.\n",
      "\n",
      "The clearance of mitochondria by autophagy, mitophagy, is important for cell and \n",
      "organism health [1], and known to be regulated by ubiquitin. During Drosophila\n",
      "intestine development, cells undergo a dramatic reduction in cell size and\n",
      "clearance of mitochondria that depends on autophagy, the E1 ubiquitin-activating \n",
      "enzyme Uba1, and ubiquitin [2]. Here we screen a collection of putative\n",
      "ubiquitin-binding domain-encoding genes for cell size reduction and autophagy\n",
      "phenotypes. We identify the endosomal sorting complex required for transport\n",
      "(ESCRT) components TSG101 and Vps36, as well as the novel gene Vps13D. Vps13D is \n",
      "an essential gene that is necessary for autophagy, mitochondrial size, and\n",
      "mitochondrial clearance in Drosophila. Interestingly, a similar mitochondrial\n",
      "phenotype is observed in VPS13D mutant human cells. The ubiquitin-associated\n",
      "(UBA) domain of Vps13D binds K63 ubiquitin chains, and mutants lacking the UBA\n",
      "domain have defects in mitochondrial size and clearance and exhibit\n",
      "semi-lethality, highlighting the importance of Vps13D ubiquitin binding in both\n",
      "mitochondrial health and development. VPS13D mutant cells possess phosphorylated \n",
      "DRP1 and mitochondrial fission factor (MFF) as well as DRP1 association with\n",
      "mitochondria, suggesting that VPS13D functions downstream of these known\n",
      "regulators of mitochondrial fission. In addition, the large Vps13D mitochondrial \n",
      "and cell size phenotypes are suppressed by decreased mitochondrial fusion gene\n",
      "function. Thus, these results provide a previously unknown link between\n",
      "ubiquitin, mitochondrial size regulation, and autophagy.\n",
      "\n",
      "Copyright  2017 Elsevier Ltd. All rights reserved.\n",
      "\n",
      "DOI: 10.1016/j.cub.2017.11.064 \n",
      "PMCID: PMC5787036\n",
      "PMID: 29307555  [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "43. Nature. 2015 Sep 17;525(7569):339-44. doi: 10.1038/nature14877. Epub 2015 Sep 7.\n",
      "\n",
      "Panorama of ancient metazoan macromolecular complexes.\n",
      "\n",
      "Wan C(1)(2), Borgeson B(2), Phanse S(1), Tu F(2), Drew K(2), Clark G(3), Xiong\n",
      "X(4)(5), Kagan O(1), Kwan J(1)(4), Bezginov A(3), Chessman K(4)(5), Pal S(5),\n",
      "Cromar G(4)(5), Papoulas O(2), Ni Z(1), Boutz DR(2), Stoilova S(1), Havugimana\n",
      "PC(1), Guo X(1), Malty RH(6), Sarov M(7), Greenblatt J(1)(4), Babu M(6), Derry\n",
      "WB(4)(5), Tillier ER(3), Wallingford JB(2)(8), Parkinson J(4)(5), Marcotte\n",
      "EM(2)(8), Emili A(1)(4).\n",
      "\n",
      "Author information: \n",
      "(1)Donnelly Centre for Cellular and Biomolecular Research, University of Toronto,\n",
      "Toronto, Ontario M5S 3E1, Canada.\n",
      "(2)Center for Systems and Synthetic Biology, Institute for Cellular and Molecular\n",
      "Biology, University of Texas at Austin, Austin, Texas 78712, USA.\n",
      "(3)Department of Medical Biophysics, Toronto, Ontario M5G 1L7, Canada.\n",
      "(4)Department of Molecular Genetics, University of Toronto, Toronto, Ontario M5S \n",
      "1A8, Canada.\n",
      "(5)Hospital for Sick Children, Toronto, Ontario M5G 1X8, Canada.\n",
      "(6)Department of Biochemistry, University of Regina, Regina, Saskatchewan S4S\n",
      "0A2, Canada.\n",
      "(7)Max Planck Institute of Molecular Cell Biology and Genetics, 01307 Dresden,\n",
      "Germany.\n",
      "(8)Department of Molecular Biosciences, University of Texas at Austin, Austin,\n",
      "Texas 78712, USA.\n",
      "\n",
      "Comment in\n",
      "    Trends Biochem Sci. 2016 Jan;41(1):4-6.\n",
      "    Nat Methods. 2015 Nov;12(11):1011.\n",
      "\n",
      "Macromolecular complexes are essential to conserved biological processes, but\n",
      "their prevalence across animals is unclear. By combining extensive biochemical\n",
      "fractionation with quantitative mass spectrometry, here we directly examined the \n",
      "composition of soluble multiprotein complexes among diverse metazoan models.\n",
      "Using an integrative approach, we generated a draft conservation map consisting\n",
      "of more than one million putative high-confidence co-complex interactions for\n",
      "species with fully sequenced genomes that encompasses functional modules present \n",
      "broadly across all extant animals. Clustering reveals a spectrum of conservation,\n",
      "ranging from ancient eukaryotic assemblies that have probably served cellular\n",
      "housekeeping roles for at least one billion years, ancestral complexes that have \n",
      "accrued contemporary components, and rarer metazoan innovations linked to\n",
      "multicellularity. We validated these projections by independent co-fractionation \n",
      "experiments in evolutionarily distant species, affinity purification and\n",
      "functional analyses. The comprehensiveness, centrality and modularity of these\n",
      "reconstructed interactomes reflect their fundamental mechanistic importance and\n",
      "adaptive value to animal cell systems.\n",
      "\n",
      "DOI: 10.1038/nature14877 \n",
      "PMCID: PMC5036527\n",
      "PMID: 26344197  [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "44. J Innate Immun. 2015;7(5):545-53. doi: 10.1159/000381265. Epub 2015 Apr 17.\n",
      "\n",
      "VPS13D Gene Variant Is Associated with Altered IL-6 Production and Mortality in\n",
      "Septic Shock.\n",
      "\n",
      "Nakada TA(1), Boyd JH, Russell JA, Aguirre-Hernndez R, Wilkinson MD, Thair SA,\n",
      "Nakada E, McConechy MK, Fjell CD, Walley KR.\n",
      "\n",
      "Author information: \n",
      "(1)Critical Care Research Laboratories, Centre for Heart Lung Innovation, St.\n",
      "Paul's Hospital, University of British Columbia, Vancouver, B.C., Canada.\n",
      "\n",
      "BACKGROUND: Genetic variations contribute to septic shock mortality. To discover \n",
      "a novel locus, we performed in vitro genome-wide association studies (GWAS) and\n",
      "further tested the result in a cohort of septic shock patients.\n",
      "METHODS: Two in vitro GWAS using a quantitative trait locus analysis of\n",
      "stimulated IL-6 production in lymphoblastoid cells from 60 individuals of\n",
      "European ancestry were performed. VPS13D rs6685273 was genotyped in European\n",
      "ancestry patients (n = 498). The VPS13D gene was silenced in vitro.\n",
      "RESULTS: Two GWAS using lymphoblastoid cells identified the locus of VPS13D\n",
      "rs6685273 that was significant in the same direction in both GWAS. The VPS13D\n",
      "rs6685273 C allele was associated with increased IL-6 production. Patients with\n",
      "septic shock who had the VPS13D rs6685273 CC genotype had an increased 28-day\n",
      "mortality (p = 0.023) and more organ failure (p < 0.05) compared to the CT/TT\n",
      "genotypes. VPS13D in vitro gene silencing in the HeLa cell line increased IL-6\n",
      "production. Furthermore, the rs6685273 genotype was associated with differential \n",
      "VPS13D splice variant expression.\n",
      "CONCLUSIONS: The VPS13D rs6685273 C allele was associated with increased IL-6\n",
      "production in vitro. The patients with the VPS13D rs6685273 CC genotype had\n",
      "increased 28-day mortality and increased organ failure. VPS13D appears to\n",
      "regulate IL-6 production.\n",
      "\n",
      " 2015 S. Karger AG, Basel.\n",
      "\n",
      "DOI: 10.1159/000381265 \n",
      "PMID: 25896417  [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "45. Brief Bioinform. 2011 Sep;12(5):449-62. doi: 10.1093/bib/bbr042. Epub 2011 Aug\n",
      "27.\n",
      "\n",
      "Phylogenetic-based propagation of functional annotations within the Gene Ontology\n",
      "consortium.\n",
      "\n",
      "Gaudet P(1), Livstone MS, Lewis SE, Thomas PD.\n",
      "\n",
      "Author information: \n",
      "(1)Swiss Institute for Bioinformatics, CMU, 1 Rue Michel Servet, 1211 Geneva 4,\n",
      "Switzerland. pascale.gaudet@isb-sib.ch\n",
      "\n",
      "The goal of the Gene Ontology (GO) project is to provide a uniform way to\n",
      "describe the functions of gene products from organisms across all kingdoms of\n",
      "life and thereby enable analysis of genomic data. Protein annotations are either \n",
      "based on experiments or predicted from protein sequences. Since most sequences\n",
      "have not been experimentally characterized, most available annotations need to be\n",
      "based on predictions. To make as accurate inferences as possible, the GO\n",
      "Consortium's Reference Genome Project is using an explicit evolutionary framework\n",
      "to infer annotations of proteins from a broad set of genomes from experimental\n",
      "annotations in a semi-automated manner. Most components in the pipeline, such as \n",
      "selection of sequences, building multiple sequence alignments and phylogenetic\n",
      "trees, retrieving experimental annotations and depositing inferred annotations,\n",
      "are fully automated. However, the most crucial step in our pipeline relies on\n",
      "software-assisted curation by an expert biologist. This curation tool,\n",
      "Phylogenetic Annotation and INference Tool (PAINT) helps curators to infer\n",
      "annotations among members of a protein family. PAINT allows curators to make\n",
      "precise assertions as to when functions were gained and lost during evolution and\n",
      "record the evidence (e.g. experimentally supported GO annotations and\n",
      "phylogenetic information including orthology) for those assertions. In this\n",
      "article, we describe how we use PAINT to infer protein function in a phylogenetic\n",
      "context with emphasis on its strengths, limitations and guidelines. We also\n",
      "discuss specific examples showing how PAINT annotations compare with those\n",
      "generated by other highly used homology-based methods.\n",
      "\n",
      "DOI: 10.1093/bib/bbr042 \n",
      "PMCID: PMC3178059\n",
      "PMID: 21873635  [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "46. Proteomics. 2011 Jan;11(1):172-9. doi: 10.1002/pmic.201000217. Epub 2010 Dec 1.\n",
      "\n",
      "Identification of proteins associated with ligand-activated estrogen receptor \n",
      "in human breast cancer cell nuclei by tandem affinity purification and nano\n",
      "LC-MS/MS.\n",
      "\n",
      "Tarallo R(1), Bamundo A, Nassa G, Nola E, Paris O, Ambrosino C, Facchiano A,\n",
      "Baumann M, Nyman TA, Weisz A.\n",
      "\n",
      "Author information: \n",
      "(1)Department of General Pathology, Second University of Naples, Napoli, Italy.\n",
      "\n",
      "Estrogen receptor  (ER-) is a key mediator of estrogen actions in breast cancer\n",
      "(BC) cells. Understanding the effects of ligand-activated ER- in target cells\n",
      "requires identification of the molecular partners acting in concert with this\n",
      "nuclear receptor to transduce the hormonal signal. We applied tandem affinity\n",
      "purification (TAP), glycerol gradient centrifugation and MS analysis to isolate\n",
      "and identify proteins interacting with ligand-activated ER- in MCF-7 cell\n",
      "nuclei. This led to the identification of 264 ER-associated proteins, whose\n",
      "functions highlight the hinge role of ER- in the coordination of multiple\n",
      "hormone-regulated nuclear processes in BC cells.\n",
      "\n",
      "Copyright  2011 WILEY-VCH Verlag GmbH & Co. KGaA, Weinheim.\n",
      "\n",
      "DOI: 10.1002/pmic.201000217 \n",
      "PMID: 21182205  [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "47. J Atheroscler Thromb. 2010 Oct 27;17(10):1054-62. Epub 2010 Jul 2.\n",
      "\n",
      "Identification of evidence suggestive of an association with peripheral arterial \n",
      "disease at the OSBPL10 locus by genome-wide investigation in the Japanese\n",
      "population.\n",
      "\n",
      "Koriyama H(1), Nakagami H, Katsuya T, Sugimoto K, Yamashita H, Takami Y, Maeda S,\n",
      "Kubo M, Takahashi A, Nakamura Y, Ogihara T, Rakugi H, Kaneda Y, Morishita R.\n",
      "\n",
      "Author information: \n",
      "(1)Department of Geriatric Medicine and Nephrology, Osaka University Graduate\n",
      "School of Medicine, 2-2 Yamada-oka, Suita, Japan.\n",
      "\n",
      "AIM: Peripheral arterial disease (PAD) is a common cause of cardiovascular\n",
      "morbidity and an independent predictor of cardiovascular mortality. However,\n",
      "little is known about the genetic basis of PAD. To elucidate this, we performed a\n",
      "two-staged genome-wide association study in Japanese individuals.\n",
      "METHODS: We initially tested 222,285 single-nucleotide polymorphisms (SNPs).\n",
      "After the first screening in a panel of 195 PAD cases and 1,358 controls, 2,696\n",
      "SNPs (1.2%) were further genotyped in the second screening using another panel of\n",
      "699 PAD cases and 1540 controls. In both screenings, controls were subjects\n",
      "affected with some diseases other than PAD.\n",
      "RESULTS: When analyzed in the combined panel, the strongest signal of PAD\n",
      "association was observed at rs1902341 in the intron of OSBPL10 (p=4.7E-7 for\n",
      "trend test; OR=1.31, 95% CI 1.18-1.46). Also, PAD was modestly associated at\n",
      "several other loci such as rs2554503 in CSMD1 (p=5.7E-5; OR=1.32, 95% CI\n",
      "1.15-1.51) or rs235243 in VSP13D (p=0.04; OR=1.18, 95% CI 1.01-1.37).\n",
      "CONCLUSION: Our genome-wide exploration identified suggestive evidence of PAD\n",
      "association at the OSBPL10 locus. Because the association has not reached a\n",
      "genome-wide significant level, further replication study is warranted for\n",
      "verification in the Japanese population.\n",
      "\n",
      "DOI: 10.5551/jat.4291 \n",
      "PMID: 20610895  [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "48. Mol Med. 2010 Jul-Aug;16(7-8):247-53. doi: 10.2119/molmed.2009.00159. Epub 2010\n",
      "Mar 17.\n",
      "\n",
      "Personalized smoking cessation: interactions between nicotine dose, dependence\n",
      "and quit-success genotype score.\n",
      "\n",
      "Rose JE(1), Behm FM, Drgon T, Johnson C, Uhl GR.\n",
      "\n",
      "Author information: \n",
      "(1)Department of Psychiatry and Center for Nicotine and Smoking Cessation\n",
      "Research, Duke University, Durham, North Carolina 27705, USA.\n",
      "rose0003@mc.duke.edu\n",
      "\n",
      "Erratum in\n",
      "    Mol Med. 2012;18(1):729.\n",
      "\n",
      "Improving and targeting nicotine replacement therapy (NRT) are cost-effective\n",
      "strategies for reducing adverse health consequences for smokers. Treatment\n",
      "studies document the efficacy of precessation NRT and support important roles for\n",
      "level of nicotine dependence and precessation smoking reduction in successful\n",
      "quitting. However, prior work has not identified the optimal precessation dose or\n",
      "means for personalizing NRT. Genome-wide association has identified groups of\n",
      "genomic markers associated with successful quitting, allowing us to develop a\n",
      "v1.0 \"quit-success\" genotype score. We now report influences of v1.0 quit-success\n",
      "genotype score, level of dependence and precessation smoking reduction in a\n",
      "smoking cessation trial that examined effects of 21 versus 42 mg/24 h\n",
      "precessation NRT. Four hundred seventy-nine smokers were randomized to 21 or 42\n",
      "mg NRT, initiated 2 wks prior to target quit dates. We monitored self-reported\n",
      "abstinence and end-expired air carbon monoxide (CO). Genotyping used Affymetrix\n",
      "arrays (Santa Clara, CA, USA). The primary outcome was 10-wk continuous smoking\n",
      "abstinence. NRT dose, level of nicotine dependence and genotype scores displayed \n",
      "significant interactive effects on successful quitting. Successful abstinence\n",
      "also was predicted by CO reductions during precessation NRT. These results\n",
      "document ways in which smoking cessation strategies can be personalized based on \n",
      "levels of nicotine dependence, genotype scores and CO monitoring. These\n",
      "assessments, taken together, can help match most smokers with optimal NRT doses\n",
      "and help rapidly identify some who may be better treated using other methods.\n",
      "\n",
      "DOI: 10.2119/molmed.2009.00159 \n",
      "PMCID: PMC2896464\n",
      "PMID: 20379614  [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "49. Hereditary Ataxia Overview.\n",
      "\n",
      "Bird TD(1).\n",
      "In: Adam MP, Ardinger HH, Pagon RA, Wallace SE, Bean LJH, Stephens K, Amemiya A, \n",
      "editors. GeneReviews [Internet]. Seattle (WA): University of Washington,\n",
      "Seattle; 1993-2019.\n",
      "1998 Oct 28.\n",
      "\n",
      "Author information: \n",
      "(1)Seattle VA Medical Center; Departments of Neurology and Medicine, University\n",
      "of Washington, Seattle, Washington\n",
      "\n",
      "CLINICAL CHARACTERISTICS: The hereditary ataxias are a group of genetic disorders\n",
      "characterized by slowly progressive incoordination of gait and often associated\n",
      "with poor coordination of hands, speech, and eye movements. Frequently, atrophy\n",
      "of the cerebellum occurs. In this GeneReview the hereditary ataxias are\n",
      "categorized by mode of inheritance and gene (or chromosome locus) in which\n",
      "pathogenic variants occur.\n",
      "DIAGNOSIS/TESTING: Inherited (genetic) forms of ataxia must be distinguished from\n",
      "the many acquired (non-genetic) causes of ataxia. The genetic forms of ataxia are\n",
      "diagnosed by family history, physical examination, neuroimaging, and molecular\n",
      "genetic testing.\n",
      "GENETIC COUNSELING: The hereditary ataxias can be inherited in an autosomal\n",
      "dominant, autosomal recessive, or X-linked manner. Genetic counseling and risk\n",
      "assessment depend on determination of the specific cause of an inherited ataxia\n",
      "in an individual.\n",
      "MANAGEMENT: Treatment of manifestations: Canes, walkers, and wheelchairs for gait\n",
      "ataxia; use of special devices to assist with handwriting, buttoning, and use of \n",
      "eating utensils; speech therapy and/or computer-based devices for those with\n",
      "dysarthria and severe speech deficits. Prevention of primary manifestations: With\n",
      "the exception of vitamin E therapy for ataxia with vitamin E deficiency (AVED),\n",
      "no specific treatments exist for hereditary ataxia.\n",
      "\n",
      "PMID: 20301317 \n",
      "\n",
      "\n",
      "50. J Am Soc Nephrol. 2009 Feb;20(2):363-79. doi: 10.1681/ASN.2008040406. Epub 2008\n",
      "Dec 3.\n",
      "\n",
      "Large-scale proteomics and phosphoproteomics of urinary exosomes.\n",
      "\n",
      "Gonzales PA(1), Pisitkun T, Hoffert JD, Tchapyjnikov D, Star RA, Kleta R, Wang\n",
      "NS, Knepper MA.\n",
      "\n",
      "Author information: \n",
      "(1)Laboratory of Kidney and Electrolyte Metabolism, National Heart, Lung, and\n",
      "Blood Institute, National Institutes of Health, Bethesda, MD 20892, USA.\n",
      "\n",
      "Normal human urine contains large numbers of exosomes, which are 40- to 100-nm\n",
      "vesicles that originate as the internal vesicles in multivesicular bodies from\n",
      "every renal epithelial cell type facing the urinary space. Here, we used LC-MS/MS\n",
      "to profile the proteome of human urinary exosomes. Overall, the analysis\n",
      "identified 1132 proteins unambiguously, including 177 that are represented on the\n",
      "Online Mendelian Inheritance in Man database of disease-related genes, suggesting\n",
      "that exosome analysis is a potential approach to discover urinary biomarkers. We \n",
      "extended the proteomic analysis to phosphoproteomic profiling using neutral loss \n",
      "scanning, and this yielded multiple novel phosphorylation sites, including\n",
      "serine-811 in the thiazide-sensitive Na-Cl co-transporter, NCC. To demonstrate\n",
      "the potential use of exosome analysis to identify a genetic renal disease, we\n",
      "carried out immunoblotting of exosomes from urine samples of patients with a\n",
      "clinical diagnosis of Bartter syndrome type I, showing an absence of the\n",
      "sodium-potassium-chloride co-transporter 2, NKCC2. The proteomic data are\n",
      "publicly accessible at http://dir.nhlbi.nih.gov/papers/lkem/exosome/.\n",
      "\n",
      "DOI: 10.1681/ASN.2008040406 \n",
      "PMCID: PMC2637050\n",
      "PMID: 19056867  [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "51. Nature. 2006 May 18;441(7091):315-21.\n",
      "\n",
      "The DNA sequence and biological annotation of human chromosome 1.\n",
      "\n",
      "Gregory SG(1), Barlow KF, McLay KE, Kaul R, Swarbreck D, Dunham A, Scott CE, Howe\n",
      "KL, Woodfine K, Spencer CC, Jones MC, Gillson C, Searle S, Zhou Y, Kokocinski F, \n",
      "McDonald L, Evans R, Phillips K, Atkinson A, Cooper R, Jones C, Hall RE, Andrews \n",
      "TD, Lloyd C, Ainscough R, Almeida JP, Ambrose KD, Anderson F, Andrew RW, Ashwell \n",
      "RI, Aubin K, Babbage AK, Bagguley CL, Bailey J, Beasley H, Bethel G, Bird CP,\n",
      "Bray-Allen S, Brown JY, Brown AJ, Buckley D, Burton J, Bye J, Carder C, Chapman\n",
      "JC, Clark SY, Clarke G, Clee C, Cobley V, Collier RE, Corby N, Coville GJ, Davies\n",
      "J, Deadman R, Dunn M, Earthrowl M, Ellington AG, Errington H, Frankish A,\n",
      "Frankland J, French L, Garner P, Garnett J, Gay L, Ghori MR, Gibson R, Gilby LM, \n",
      "Gillett W, Glithero RJ, Grafham DV, Griffiths C, Griffiths-Jones S, Grocock R,\n",
      "Hammond S, Harrison ES, Hart E, Haugen E, Heath PD, Holmes S, Holt K, Howden PJ, \n",
      "Hunt AR, Hunt SE, Hunter G, Isherwood J, James R, Johnson C, Johnson D, Joy A,\n",
      "Kay M, Kershaw JK, Kibukawa M, Kimberley AM, King A, Knights AJ, Lad H, Laird G, \n",
      "Lawlor S, Leongamornlert DA, Lloyd DM, Loveland J, Lovell J, Lush MJ, Lyne R,\n",
      "Martin S, Mashreghi-Mohammadi M, Matthews L, Matthews NS, McLaren S, Milne S,\n",
      "Mistry S, Moore MJ, Nickerson T, O'Dell CN, Oliver K, Palmeiri A, Palmer SA,\n",
      "Parker A, Patel D, Pearce AV, Peck AI, Pelan S, Phelps K, Phillimore BJ, Plumb R,\n",
      "Rajan J, Raymond C, Rouse G, Saenphimmachak C, Sehra HK, Sheridan E, Shownkeen R,\n",
      "Sims S, Skuce CD, Smith M, Steward C, Subramanian S, Sycamore N, Tracey A,\n",
      "Tromans A, Van Helmond Z, Wall M, Wallis JM, White S, Whitehead SL, Wilkinson JE,\n",
      "Willey DL, Williams H, Wilming L, Wray PW, Wu Z, Coulson A, Vaudin M, Sulston JE,\n",
      "Durbin R, Hubbard T, Wooster R, Dunham I, Carter NP, McVean G, Ross MT, Harrow J,\n",
      "Olson MV, Beck S, Rogers J, Bentley DR, Banerjee R, Bryant SP, Burford DC,\n",
      "Burrill WD, Clegg SM, Dhami P, Dovey O, Faulkner LM, Gribble SM, Langford CF,\n",
      "Pandian RD, Porter KM, Prigmore E.\n",
      "\n",
      "Author information: \n",
      "(1)The Wellcome Trust Sanger Institute, The Wellcome Trust Genome Campus,\n",
      "Hinxton, Cambridgeshire CB10 1SA, UK. sgregory@chg.duhs.duke.edu\n",
      "\n",
      "Erratum in\n",
      "    Nature. 2006 Oct 26;443(7114):1013. Banerjee, R [added]; Bryant, SP [added];\n",
      "Burford, DC [added]; Burrill, WDH [added]; Clegg, SM [added]; Dhami, P [added];\n",
      "Dovey, O [added]; Faulkner, LM [added]; Gribble, SM [added]; Langford, CF\n",
      "[added]; Pandian, RD [added]; Porter, KM [added]; Prigmore, E.\n",
      "\n",
      "The reference sequence for each human chromosome provides the framework for\n",
      "understanding genome function, variation and evolution. Here we report the\n",
      "finished sequence and biological annotation of human chromosome 1. Chromosome 1\n",
      "is gene-dense, with 3,141 genes and 991 pseudogenes, and many coding sequences\n",
      "overlap. Rearrangements and mutations of chromosome 1 are prevalent in cancer and\n",
      "many other diseases. Patterns of sequence variation reveal signals of recent\n",
      "selection in specific genes that may contribute to human fitness, and also in\n",
      "regions where no function is evident. Fine-scale recombination occurs in hotspots\n",
      "of varying intensity along the sequence, and is enriched near genes. These and\n",
      "other studies of human biology and disease encoded within chromosome 1 are made\n",
      "possible with the highly accurate annotated sequence, as part of the completed\n",
      "set of chromosome sequences that comprise the reference human genome.\n",
      "\n",
      "DOI: 10.1038/nature04727 \n",
      "PMID: 16710414  [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "52. Genomics. 2004 Sep;84(3):536-49.\n",
      "\n",
      "Analysis of the human VPS13 gene family.\n",
      "\n",
      "Velayos-Baeza A(1), Vettori A, Copley RR, Dobson-Stone C, Monaco AP.\n",
      "\n",
      "Author information: \n",
      "(1)Wellcome Trust Centre for Human Genetics, University of Oxford, Headington,\n",
      "OX3 7BN Oxford, UK.\n",
      "\n",
      "The gene mutated in chorea-acanthocytosis (CHAC; approved gene symbol VPS13A)\n",
      "encodes chorein, a protein similar to yeast Vps13p. We detected several similar\n",
      "putative human proteins by BLAST analysis of chorein. We characterized the\n",
      "structure of three new genes encoding these CHAC-similar proteins, located on\n",
      "chromosomes 1p36, 8q22, and 15q21. The most similar gene in yeast to all four\n",
      "human genes is Vps13, and therefore the human genes were named VPS13A (CHAC,\n",
      "9q21), VPS13B (8q22), VPS13C (15q21), and VPS13D (1p36). VPS13B has recently been\n",
      "reported as COH1, altered in Cohen syndrome. For each gene, we describe several\n",
      "alternative splicing variants; at least two transcripts per gene are major forms.\n",
      "The expression pattern of these genes is ubiquitous, with some tissue-specific\n",
      "differences between several transcript variants. Protein sequence comparisons\n",
      "suggest that intramolecular duplications have played an important role in the\n",
      "evolution of this gene family.\n",
      "\n",
      "DOI: 10.1016/j.ygeno.2004.04.012 \n",
      "PMID: 15498460  [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "53. Genome Res. 2004 Oct;14(10B):2121-7.\n",
      "\n",
      "The status, quality, and expansion of the NIH full-length cDNA project: the\n",
      "Mammalian Gene Collection (MGC).\n",
      "\n",
      "Gerhard DS, Wagner L, Feingold EA, Shenmen CM, Grouse LH, Schuler G, Klein SL,\n",
      "Old S, Rasooly R, Good P, Guyer M, Peck AM, Derge JG, Lipman D, Collins FS, Jang \n",
      "W, Sherry S, Feolo M, Misquitta L, Lee E, Rotmistrovsky K, Greenhut SF, Schaefer \n",
      "CF, Buetow K, Bonner TI, Haussler D, Kent J, Kiekhaus M, Furey T, Brent M, Prange\n",
      "C, Schreiber K, Shapiro N, Bhat NK, Hopkins RF, Hsie F, Driscoll T, Soares MB,\n",
      "Casavant TL, Scheetz TE, Brown-stein MJ, Usdin TB, Toshiyuki S, Carninci P, Piao \n",
      "Y, Dudekula DB, Ko MS, Kawakami K, Suzuki Y, Sugano S, Gruber CE, Smith MR,\n",
      "Simmons B, Moore T, Waterman R, Johnson SL, Ruan Y, Wei CL, Mathavan S, Gunaratne\n",
      "PH, Wu J, Garcia AM, Hulyk SW, Fuh E, Yuan Y, Sneed A, Kowis C, Hodgson A, Muzny \n",
      "DM, McPherson J, Gibbs RA, Fahey J, Helton E, Ketteman M, Madan A, Rodrigues S,\n",
      "Sanchez A, Whiting M, Madari A, Young AC, Wetherby KD, Granite SJ, Kwong PN,\n",
      "Brinkley CP, Pearson RL, Bouffard GG, Blakesly RW, Green ED, Dickson MC,\n",
      "Rodriguez AC, Grimwood J, Schmutz J, Myers RM, Butterfield YS, Griffith M,\n",
      "Griffith OL, Krzywinski MI, Liao N, Morin R, Palmquist D, Petrescu AS, Skalska U,\n",
      "Smailus DE, Stott JM, Schnerch A, Schein JE, Jones SJ, Holt RA, Baross A, Marra\n",
      "MA, Clifton S, Makowski KA, Bosak S, Malek J; MGC Project Team.\n",
      "\n",
      "Erratum in\n",
      "    Genome Res. 2006 Jun;16(6):804. Morrin, Ryan [corrected to Morin, Ryan].\n",
      "\n",
      "The National Institutes of Health's Mammalian Gene Collection (MGC) project was\n",
      "designed to generate and sequence a publicly accessible cDNA resource containing \n",
      "a complete open reading frame (ORF) for every human and mouse gene. The project\n",
      "initially used a random strategy to select clones from a large number of cDNA\n",
      "libraries from diverse tissues. Candidate clones were chosen based on 5'-EST\n",
      "sequences, and then fully sequenced to high accuracy and analyzed by algorithms\n",
      "developed for this project. Currently, more than 11,000 human and 10,000 mouse\n",
      "genes are represented in MGC by at least one clone with a full ORF. The random\n",
      "selection approach is now reaching a saturation point, and a transition to\n",
      "protocols targeted at the missing transcripts is now required to complete the\n",
      "mouse and human collections. Comparison of the sequence of the MGC clones to\n",
      "reference genome sequences reveals that most cDNA clones are of very high\n",
      "sequence quality, although it is likely that some cDNAs may carry missense\n",
      "variants as a consequence of experimental artifact, such as PCR, cloning, or\n",
      "reverse transcriptase errors. Recently, a rat cDNA component was added to the\n",
      "project, and ongoing frog (Xenopus) and zebrafish (Danio) cDNA projects were\n",
      "expanded to take advantage of the high-throughput MGC pipeline.\n",
      "\n",
      "DOI: 10.1101/gr.2596504 \n",
      "PMCID: PMC528928\n",
      "PMID: 15489334  [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "54. Nat Genet. 2004 Jan;36(1):40-5. Epub 2003 Dec 21.\n",
      "\n",
      "Complete sequencing and characterization of 21,243 full-length human cDNAs.\n",
      "\n",
      "Ota T(1), Suzuki Y, Nishikawa T, Otsuki T, Sugiyama T, Irie R, Wakamatsu A,\n",
      "Hayashi K, Sato H, Nagai K, Kimura K, Makita H, Sekine M, Obayashi M, Nishi T,\n",
      "Shibahara T, Tanaka T, Ishii S, Yamamoto J, Saito K, Kawai Y, Isono Y, Nakamura\n",
      "Y, Nagahari K, Murakami K, Yasuda T, Iwayanagi T, Wagatsuma M, Shiratori A, Sudo \n",
      "H, Hosoiri T, Kaku Y, Kodaira H, Kondo H, Sugawara M, Takahashi M, Kanda K, Yokoi\n",
      "T, Furuya T, Kikkawa E, Omura Y, Abe K, Kamihara K, Katsuta N, Sato K, Tanikawa\n",
      "M, Yamazaki M, Ninomiya K, Ishibashi T, Yamashita H, Murakawa K, Fujimori K,\n",
      "Tanai H, Kimata M, Watanabe M, Hiraoka S, Chiba Y, Ishida S, Ono Y, Takiguchi S, \n",
      "Watanabe S, Yosida M, Hotuta T, Kusano J, Kanehori K, Takahashi-Fujii A, Hara H, \n",
      "Tanase TO, Nomura Y, Togiya S, Komai F, Hara R, Takeuchi K, Arita M, Imose N,\n",
      "Musashino K, Yuuki H, Oshima A, Sasaki N, Aotsuka S, Yoshikawa Y, Matsunawa H,\n",
      "Ichihara T, Shiohata N, Sano S, Moriya S, Momiyama H, Satoh N, Takami S,\n",
      "Terashima Y, Suzuki O, Nakagawa S, Senoh A, Mizoguchi H, Goto Y, Shimizu F,\n",
      "Wakebe H, Hishigaki H, Watanabe T, Sugiyama A, Takemoto M, Kawakami B, Yamazaki\n",
      "M, Watanabe K, Kumagai A, Itakura S, Fukuzumi Y, Fujimori Y, Komiyama M, Tashiro \n",
      "H, Tanigami A, Fujiwara T, Ono T, Yamada K, Fujii Y, Ozaki K, Hirao M, Ohmori Y, \n",
      "Kawabata A, Hikiji T, Kobatake N, Inagaki H, Ikema Y, Okamoto S, Okitani R,\n",
      "Kawakami T, Noguchi S, Itoh T, Shigeta K, Senba T, Matsumura K, Nakajima Y,\n",
      "Mizuno T, Morinaga M, Sasaki M, Togashi T, Oyama M, Hata H, Watanabe M, Komatsu\n",
      "T, Mizushima-Sugano J, Satoh T, Shirai Y, Takahashi Y, Nakagawa K, Okumura K,\n",
      "Nagase T, Nomura N, Kikuchi H, Masuho Y, Yamashita R, Nakai K, Yada T, Nakamura\n",
      "Y, Ohara O, Isogai T, Sugano S.\n",
      "\n",
      "Author information: \n",
      "(1)Helix Research Institute, 1532-3 Yana, Kisarazu, Chiba 292-0812, Japan.\n",
      "\n",
      "As a base for human transcriptome and functional genomics, we created the\n",
      "\"full-length long Japan\" (FLJ) collection of sequenced human cDNAs. We determined\n",
      "the entire sequence of 21,243 selected clones and found that 14,490 cDNAs (10,897\n",
      "clusters) were unique to the FLJ collection. About half of them (5,416) seemed to\n",
      "be protein-coding. Of those, 1,999 clusters had not been predicted by\n",
      "computational methods. The distribution of GC content of nonpredicted cDNAs had a\n",
      "peak at approximately 58% compared with a peak at approximately 42%for predicted \n",
      "cDNAs. Thus, there seems to be a slight bias against GC-rich transcripts in\n",
      "current gene prediction procedures. The rest of the cDNAs unique to the FLJ\n",
      "collection (5,481) contained no obvious open reading frames (ORFs) and thus are\n",
      "candidate noncoding RNAs. About one-fourth of them (1,378) showed a clear pattern\n",
      "of splicing. The distribution of GC content of noncoding cDNAs was narrow and had\n",
      "a peak at approximately 42%, relatively low compared with that of protein-coding \n",
      "cDNAs.\n",
      "\n",
      "DOI: 10.1038/ng1285 \n",
      "PMID: 14702039  [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "55. Ann Neurol. 2003 Dec;54(6):824-8.\n",
      "\n",
      "Pathogenesis of clinical signs in recessive ataxia with saccadic intrusions.\n",
      "\n",
      "Swartz BE(1), Li S, Bespalova I, Burmeister M, Dulaney E, Robinson FR, Leigh RJ.\n",
      "\n",
      "Author information: \n",
      "(1)Department of Neurology, Veterans Affairs Medical Center, and University\n",
      "Hospitals, Case Western Reserve University, 11100 Euclid Avenue, Cleveland, OH\n",
      "44106-5040, USA.\n",
      "\n",
      "We describe a family of Slovenian descent with progressive ataxia, corticospinal \n",
      "signs, axonal sensorimotor neuropathy, and disruption of visual fixation by\n",
      "saccadic intrusions. Chromosome mapping indicated a mutation on 1p36, and this\n",
      "recessive disorder has been designated spinocerebellar ataxia with saccadic\n",
      "intrusions. Affected patients showed overshooting horizontal saccades,\n",
      "macrosaccadic oscillations, and increased velocity of larger saccades; other eye \n",
      "movements were normal. Slowed conduction in axons that are selectively vulnerable\n",
      "to the molecular defect could explain both the sensorimotor neuropathy and the\n",
      "saccadic disorder, which would be caused by delayed feedback control because of\n",
      "slow conduction in cerebellar parallel fibers.\n",
      "\n",
      "DOI: 10.1002/ana.10758 \n",
      "PMID: 14681893  [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "56. Proc Natl Acad Sci U S A. 2002 Dec 24;99(26):16899-903. Epub 2002 Dec 11.\n",
      "\n",
      "Generation and initial analysis of more than 15,000 full-length human and mouse\n",
      "cDNA sequences.\n",
      "\n",
      "Strausberg RL(1), Feingold EA, Grouse LH, Derge JG, Klausner RD, Collins FS,\n",
      "Wagner L, Shenmen CM, Schuler GD, Altschul SF, Zeeberg B, Buetow KH, Schaefer CF,\n",
      "Bhat NK, Hopkins RF, Jordan H, Moore T, Max SI, Wang J, Hsieh F, Diatchenko L,\n",
      "Marusina K, Farmer AA, Rubin GM, Hong L, Stapleton M, Soares MB, Bonaldo MF,\n",
      "Casavant TL, Scheetz TE, Brownstein MJ, Usdin TB, Toshiyuki S, Carninci P, Prange\n",
      "C, Raha SS, Loquellano NA, Peters GJ, Abramson RD, Mullahy SJ, Bosak SA, McEwan\n",
      "PJ, McKernan KJ, Malek JA, Gunaratne PH, Richards S, Worley KC, Hale S, Garcia\n",
      "AM, Gay LJ, Hulyk SW, Villalon DK, Muzny DM, Sodergren EJ, Lu X, Gibbs RA, Fahey \n",
      "J, Helton E, Ketteman M, Madan A, Rodrigues S, Sanchez A, Whiting M, Madan A,\n",
      "Young AC, Shevchenko Y, Bouffard GG, Blakesley RW, Touchman JW, Green ED, Dickson\n",
      "MC, Rodriguez AC, Grimwood J, Schmutz J, Myers RM, Butterfield YS, Krzywinski MI,\n",
      "Skalska U, Smailus DE, Schnerch A, Schein JE, Jones SJ, Marra MA; Mammalian Gene \n",
      "Collection Program Team.\n",
      "\n",
      "Author information: \n",
      "(1)National Cancer Institute, Bethessda, MD 20892-2580, USA. rls@nih.gov\n",
      "\n",
      "The National Institutes of Health Mammalian Gene Collection (MGC) Program is a\n",
      "multiinstitutional effort to identify and sequence a cDNA clone containing a\n",
      "complete ORF for each human and mouse gene. ESTs were generated from libraries\n",
      "enriched for full-length cDNAs and analyzed to identify candidate full-ORF\n",
      "clones, which then were sequenced to high accuracy. The MGC has currently\n",
      "sequenced and verified the full ORF for a nonredundant set of >9,000 human and\n",
      ">6,000 mouse genes. Candidate full-ORF clones for an additional 7,800 human and\n",
      "3,500 mouse genes also have been identified. All MGC sequences and clones are\n",
      "available without restriction through public databases and clone distribution\n",
      "networks (see http:mgc.nci.nih.gov).\n",
      "\n",
      "DOI: 10.1073/pnas.242603899 \n",
      "PMCID: PMC139241\n",
      "PMID: 12477932  [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "57. Genome Res. 2002 Nov;12(11):1773-84.\n",
      "\n",
      "Protein-protein interactions between large proteins: two-hybrid screening using a\n",
      "functionally classified library composed of long cDNAs.\n",
      "\n",
      "Nakayama M(1), Kikuno R, Ohara O.\n",
      "\n",
      "Author information: \n",
      "(1)Department of Human Gene Research, Kazusa DNA Research Institute, Kisarazu,\n",
      "Chiba 292-0818, Japan. nmanabu@kazusa.or.jp\n",
      "\n",
      "Large proteins have multiple domains that are potentially capable of binding many\n",
      "kinds of partners. It is conceivable, therefore, that such proteins could\n",
      "function as an intricate framework of assembly protein complexes. To\n",
      "comprehensively study protein-protein interactions between large KIAA proteins,\n",
      "we have constructed a library composed of 1087 KIAA cDNA clones based on prior\n",
      "functional classifications done in silico. We were guided by two principles that \n",
      "raise the success rate for detecting interactions per tested combination: we\n",
      "avoided testing low-probability combinations, and reduced the number of potential\n",
      "false negatives that arise from the fact that large proteins cannot reliably be\n",
      "expressed in yeast. The latter was addressed by constructing a cDNA library\n",
      "comprised of random fragments encoding large proteins. Cytoplasmic domains of\n",
      "KIAA transmembrane proteins (>1000 amino acids) were used as bait for yeast\n",
      "two-hybrid screening. Our analyses reveal that several KIAA proteins bearing a\n",
      "transmembrane region have the capability of binding to other KIAA proteins\n",
      "containing domains (e.g., PDZ, SH3, rhoGEF, and spectrin) known to be localized\n",
      "to highly specialized submembranous sites, indicating that they participate in\n",
      "cellular junction formation, receptor or channel clustering, and intracellular\n",
      "signaling events. Our representative library should be a very useful resource for\n",
      "detecting previously unidentified interactions because it complements\n",
      "conventional expression libraries, which seldom contain large cDNAs.\n",
      "\n",
      "DOI: 10.1101/gr.406902 \n",
      "PMCID: PMC187542\n",
      "PMID: 12421765  [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "58. Ann N Y Acad Sci. 2002 Apr;956:441-4.\n",
      "\n",
      "A form of inherited cerebellar ataxia with saccadic intrusions, increased\n",
      "saccadic speed, sensory neuropathy, and myoclonus.\n",
      "\n",
      "Swartz BE(1), Burmeister M, Somers JT, Rottach KG, Bespalova IN, Leigh RJ.\n",
      "\n",
      "Author information: \n",
      "(1)Veterans' Affairs Medical Center and Case Western University, Cleveland, Ohio \n",
      "44106, USA. Barbara.Swartz@uhhs.com\n",
      "\n",
      "DOI: 10.1111/j.1749-6632.2002.tb02850.x \n",
      "PMID: 11960835  [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "59. DNA Res. 1997 Oct 31;4(5):345-9.\n",
      "\n",
      "Characterization of cDNA clones in size-fractionated cDNA libraries from human\n",
      "brain.\n",
      "\n",
      "Seki N(1), Ohira M, Nagase T, Ishikawa K, Miyajima N, Nakajima D, Nomura N, Ohara\n",
      "O.\n",
      "\n",
      "Author information: \n",
      "(1)Kazusa DNA Research Institute, Chiba, Japan. nseki@kazusa.or.jp\n",
      "\n",
      "To evaluate the size-fractionated cDNA libraries of human brain previously\n",
      "constructed (O. O'hara et al. DNA Research, 4, 53-59, 1997), the occurrence of\n",
      "chimeric clones and the content of clones with coding potentiality were analyzed \n",
      "using the randomly sampled clones with insert sizes of 5 to 7 kb. When the\n",
      "chromosomal location of 30 clones was determined by the radiation-hybrid mapping \n",
      "method, the map positions assigned from the 3'- and 5'-end sequences separately\n",
      "were coincident for 29 clones, suggesting that the occurrence of chimeric clones \n",
      "is at most 1/30. Using 91 clones mapped to chromosome 1, the content of clones\n",
      "that have the potentiality coding for proteins larger than 100 amino acid\n",
      "residues was estimated to be approximately 50% (46 out of 91 clones) on the basis\n",
      "of nucleotide sequence analysis and coding potentiality assay in vitro. No\n",
      "significant open reading frames were detected in the remaining clones. Although\n",
      "the clones coding for short peptides may not have been included in the above\n",
      "estimation, the libraries constructed from the whole brain mRNA fraction appear\n",
      "to contain a considerable amount of clones corresponding to the 5'-truncated\n",
      "transcripts in an unprocessed form and/or those with long 3'-untranslated\n",
      "regions.\n",
      "\n",
      "DOI: 10.1093/dnares/4.5.345 \n",
      "PMID: 9455484  [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "60. Genome Res. 1997 Apr;7(4):353-8.\n",
      "\n",
      "Large-scale concatenation cDNA sequencing.\n",
      "\n",
      "Yu W, Andersson B, Worley KC, Muzny DM, Ding Y, Liu W, Ricafrente JY, Wentland\n",
      "MA, Lennon G, Gibbs RA.\n",
      "\n",
      "A total of 100 kb of DNA derived from 69 individual human brain cDNA clones of\n",
      "0.7-2.0 kb were sequenced by concatenated cDNA sequencing (CCS), whereby multiple\n",
      "individual DNA fragments are sequenced simultaneously in a single shotgun\n",
      "library. The method yielded accurate sequences and a similar efficiency compared \n",
      "with other shotgun libraries constructed from single DNA fragments (> 20 kb).\n",
      "Computer analyses were carried out on 65 cDNA clone sequences and their\n",
      "corresponding end sequences to examine both nucleic acid and amino acid sequence \n",
      "similarities in the databases. Thirty-seven clones revealed no DNA database\n",
      "matches, 12 clones generated exact matches (> or = 98% identity), and 16 clones\n",
      "generated nonexact matches (57%-97% identity) to either known human or other\n",
      "species genes. Of those 28 matched clones, 8 had corresponding end sequences that\n",
      "failed to identify similarities. In a protein similarity search, 27 clone\n",
      "sequences displayed significant matches, whereas only 20 of the end sequences had\n",
      "matches to known protein sequences. Our data indicate that full-length cDNA\n",
      "insert sequences provide significantly more nucleic acid and protein sequence\n",
      "similarity matches than expressed sequence tags (ESTs) for database searching.\n",
      "\n",
      "DOI: 10.1101/gr.7.4.353 \n",
      "PMCID: PMC139146\n",
      "PMID: 9110174  [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "61. Anal Biochem. 1996 Apr 5;236(1):107-13.\n",
      "\n",
      "A \"double adaptor\" method for improved shotgun library construction.\n",
      "\n",
      "Andersson B(1), Wentland MA, Ricafrente JY, Liu W, Gibbs RA.\n",
      "\n",
      "Author information: \n",
      "(1)Department of Molecular and Human Genetics, Baylor College of Medicine, One\n",
      "Baylor Plaza, Houston, Texas, 77030, USA.\n",
      "\n",
      "The efficiency of shotgun DNA sequencing depends to a great extent on the quality\n",
      "of the random-subclone libraries used. We here describe a novel \"double adaptor\" \n",
      "strategy for efficient construction of high-quality shotgun libraries. In this\n",
      "method, randomly sheared and end-repaired fragments are ligated to\n",
      "oligonucleotide adaptors creating 12-base overhangs. Nonphosphorylated\n",
      "oligonucleotides are used, which prevents formation of adaptor dimers and ensures\n",
      "efficient ligation of insert to adaptor. The vector is prepared from a modified\n",
      "M13 vector, by KpnI/PstI digestion followed by ligation to oligonucleotides with \n",
      "ends complementary to the overhangs created in the digest. These adaptors create \n",
      "5'-overhangs complementary to those on the inserts. Following annealing of insert\n",
      "to vector, the DNA is directly used for transformation without a ligation step.\n",
      "This protocol is robust and shows three- to fivefold higher yield of clones\n",
      "compared to previous protocols. No chimeric clones can be detected and the\n",
      "background of clones without an insert is <1%. The procedure is rapid and shows\n",
      "potential for automation.\n",
      "\n",
      "DOI: 10.1006/abio.1996.0138 \n",
      "PMID: 8619474  [Indexed for MEDLINE]\n"
     ]
    }
   ],
   "source": [
    "require 'net/http'\n",
    "require 'bio'\n",
    "require 'nokogiri'\n",
    "\n",
    "ncbi = Bio::NCBI::REST.new\n",
    "max = 5\n",
    "ids = ncbi.esearch('\"Wilkinson MD\"[AUTH]', {\"db\"=>\"nucleotide\", \"rettype\"=>\"gb\", \"retmax\" => max})\n",
    "idlist = ids.join(\",\")\n",
    "\n",
    "elink = URI(\"https://eutils.ncbi.nlm.nih.gov:443/entrez/eutils/elink.fcgi\")\n",
    "request = Net::HTTP::Post.new(elink)\n",
    "https = Net::HTTP.new(elink.host,elink.port)\n",
    "https.use_ssl = true  # tell the HTTP connection to use ecryption (SSL = Secure Socket Layer)\n",
    "\n",
    "request.set_form_data(\n",
    "  'email' => 'markw@illuminae.com',\n",
    "  'dbfrom' => 'nuccore', \n",
    "  'db' => 'pubmed',\n",
    "  'retmode' => 'xml',\n",
    "  'id' => idlist)\n",
    "\n",
    "output = https.request(request)   # make the request (we really should check if it is successful)\n",
    "linkset = output.body\n",
    "\n",
    "doc = Nokogiri::XML(linkset)\n",
    "pubmedids = []\n",
    "doc.xpath('//LinkSet/LinkSetDb/Link').each do |node|    # the \"path\" (\"xpath\")to the data, starts with '//'\n",
    "   pubmedids << node.at_xpath('Id').content.to_s\n",
    "end\n",
    "puts \"PUBMED IDs are: #{pubmedids}\"\n",
    "\n",
    "pubmedrecords = ncbi.efetch(pubmedids, {\"db\"=>\"pubmed\", \"rettype\"=>\"abstract\", \"retmode\"=>\"text\"})\n",
    "puts pubmedrecords.force_encoding(\"utf-8\")  #This is a useful trick for text that has international characters!"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "\n",
    "# Bio::PubMed , Bio::MEDLINE and bibtex format\n",
    "\n",
    "The output above is great for a human, but not very useful for a machine.  Most reference managers can accept new records in bibtex format.  Can we get bibtex format for these records?  Sure!\n",
    "\n",
    "Bio::PubMed is a shortcut to the ESearch and EFetch of utilities for the pubmed database.  It is used like this:\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "@article{PMID:25896417,\n",
      "  author       = {Nakada, T. A. and Boyd, J. H. and Russell, J. A. and Aguirre-Hernandez, R. and Wilkinson, M. D. and Thair, S. A. and Nakada, E. and McConechy, M. K. and Fjell, C. D. and Walley, K. R.},\n",
      "  title        = {VPS13D Gene Variant Is Associated with Altered IL-6 Production and Mortality in Septic Shock.},\n",
      "  journal      = {J Innate Immun},\n",
      "  year         = {2015},\n",
      "  volume       = {7},\n",
      "  number       = {5},\n",
      "  pages        = {545--553},\n",
      "  url          = {http://www.ncbi.nlm.nih.gov/pubmed/25896417},\n",
      "}\n",
      "\n",
      "@article{PMID:20610895,\n",
      "  author       = {Koriyama, H. and Nakagami, H. and Katsuya, T. and Sugimoto, K. and Yamashita, H. and Takami, Y. and Maeda, S. and Kubo, M. and Takahashi, A. and Nakamura, Y. and Ogihara, T. and Rakugi, H. and Kaneda, Y. and Morishita, R.},\n",
      "  title        = {Identification of evidence suggestive of an association with peripheral arterial disease at the OSBPL10 locus by genome-wide investigation in the Japanese population.},\n",
      "  journal      = {J Atheroscler Thromb},\n",
      "  year         = {2010},\n",
      "  volume       = {17},\n",
      "  number       = {10},\n",
      "  pages        = {1054--1062},\n",
      "  url          = {http://www.ncbi.nlm.nih.gov/pubmed/20610895},\n",
      "}\n",
      "\n",
      "@article{PMID:19056867,\n",
      "  author       = {Gonzales, P. A. and Pisitkun, T. and Hoffert, J. D. and Tchapyjnikov, D. and Star, R. A. and Kleta, R. and Wang, N. S. and Knepper, M. A.},\n",
      "  title        = {Large-scale proteomics and phosphoproteomics of urinary exosomes.},\n",
      "  journal      = {J Am Soc Nephrol},\n",
      "  year         = {2009},\n",
      "  volume       = {20},\n",
      "  number       = {2},\n",
      "  pages        = {363--379},\n",
      "  url          = {http://www.ncbi.nlm.nih.gov/pubmed/19056867},\n",
      "}\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "[\"PMID- 25896417\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20160620\\nLR  - 20150828\\nIS  - 1662-8128 (Electronic)\\nIS  - 1662-811X (Linking)\\nVI  - 7\\nIP  - 5\\nDP  - 2015\\nTI  - VPS13D Gene Variant Is Associated with Altered IL-6 Production and Mortality in\\n      Septic Shock.\\nPG  - 545-53\\nLID - 10.1159/000381265 [doi]\\nAB  - BACKGROUND: Genetic variations contribute to septic shock mortality. To discover \\n      a novel locus, we performed in vitro genome-wide association studies (GWAS) and\\n      further tested the result in a cohort of septic shock patients. METHODS: Two in\\n      vitro GWAS using a quantitative trait locus analysis of stimulated IL-6\\n      production in lymphoblastoid cells from 60 individuals of European ancestry were \\n      performed. VPS13D rs6685273 was genotyped in European ancestry patients (n =\\n      498). The VPS13D gene was silenced in vitro. RESULTS: Two GWAS using\\n      lymphoblastoid cells identified the locus of VPS13D rs6685273 that was\\n      significant in the same direction in both GWAS. The VPS13D rs6685273 C allele was\\n      associated with increased IL-6 production. Patients with septic shock who had the\\n      VPS13D rs6685273 CC genotype had an increased 28-day mortality (p = 0.023) and\\n      more organ failure (p < 0.05) compared to the CT/TT genotypes. VPS13D in vitro\\n      gene silencing in the HeLa cell line increased IL-6 production. Furthermore, the \\n      rs6685273 genotype was associated with differential VPS13D splice variant\\n      expression. CONCLUSIONS: The VPS13D rs6685273 C allele was associated with\\n      increased IL-6 production in vitro. The patients with the VPS13D rs6685273 CC\\n      genotype had increased 28-day mortality and increased organ failure. VPS13D\\n      appears to regulate IL-6 production.\\nCI  - (c) 2015 S. Karger AG, Basel.\\nFAU - Nakada, Taka-aki\\nAU  - Nakada TA\\nAD  - Critical Care Research Laboratories, Centre for Heart Lung Innovation, St. Paul's\\n      Hospital, University of British Columbia, Vancouver, B.C., Canada.\\nFAU - Boyd, John H\\nAU  - Boyd JH\\nFAU - Russell, James A\\nAU  - Russell JA\\nFAU - Aguirre-Hernandez, Rosalia\\nAU  - Aguirre-Hernandez R\\nFAU - Wilkinson, Mark D\\nAU  - Wilkinson MD\\nFAU - Thair, Simone A\\nAU  - Thair SA\\nFAU - Nakada, Emiri\\nAU  - Nakada E\\nFAU - McConechy, Melissa K\\nAU  - McConechy MK\\nFAU - Fjell, Christopher D\\nAU  - Fjell CD\\nFAU - Walley, Keith R\\nAU  - Walley KR\\nLA  - eng\\nGR  - Canadian Institutes of Health Research/Canada\\nPT  - Journal Article\\nPT  - Multicenter Study\\nPT  - Randomized Controlled Trial\\nPT  - Research Support, Non-U.S. Gov't\\nDEP - 20150417\\nPL  - Switzerland\\nTA  - J Innate Immun\\nJT  - Journal of innate immunity\\nJID - 101469471\\nRN  - 0 (Interleukin-6)\\nRN  - 0 (Proteins)\\nRN  - 0 (VPS13D protein, human)\\nSB  - IM\\nMH  - Aged\\nMH  - Cohort Studies\\nMH  - European Continental Ancestry Group/genetics\\nMH  - Female\\nMH  - Gene Frequency\\nMH  - Genetic Predisposition to Disease\\nMH  - Genome-Wide Association Study\\nMH  - Genotype\\nMH  - HeLa Cells\\nMH  - Humans\\nMH  - Interleukin-6/genetics/*metabolism\\nMH  - Lymphocyte Activation/genetics\\nMH  - Lymphocytes/*physiology\\nMH  - Male\\nMH  - Middle Aged\\nMH  - Polymorphism, Single Nucleotide\\nMH  - Proteins/*genetics\\nMH  - Shock, Septic/*genetics/immunology/*mortality\\nMH  - Spain\\nMH  - Survival Analysis\\nEDAT- 2015/04/22 06:00\\nMHDA- 2016/06/21 06:00\\nCRDT- 2015/04/22 06:00\\nPHST- 2014/12/17 00:00 [received]\\nPHST- 2015/02/26 00:00 [accepted]\\nPHST- 2015/04/22 06:00 [entrez]\\nPHST- 2015/04/22 06:00 [pubmed]\\nPHST- 2016/06/21 06:00 [medline]\\nAID - 000381265 [pii]\\nAID - 10.1159/000381265 [doi]\\nPST - ppublish\\nSO  - J Innate Immun. 2015;7(5):545-53. doi: 10.1159/000381265. Epub 2015 Apr 17.\", \"PMID- 20610895\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20110308\\nLR  - 20190608\\nIS  - 1880-3873 (Electronic)\\nIS  - 1340-3478 (Linking)\\nVI  - 17\\nIP  - 10\\nDP  - 2010 Oct 27\\nTI  - Identification of evidence suggestive of an association with peripheral arterial \\n      disease at the OSBPL10 locus by genome-wide investigation in the Japanese\\n      population.\\nPG  - 1054-62\\nAB  - AIM: Peripheral arterial disease (PAD) is a common cause of cardiovascular\\n      morbidity and an independent predictor of cardiovascular mortality. However,\\n      little is known about the genetic basis of PAD. To elucidate this, we performed a\\n      two-staged genome-wide association study in Japanese individuals. METHODS: We\\n      initially tested 222,285 single-nucleotide polymorphisms (SNPs). After the first \\n      screening in a panel of 195 PAD cases and 1,358 controls, 2,696 SNPs (1.2%) were \\n      further genotyped in the second screening using another panel of 699 PAD cases\\n      and 1540 controls. In both screenings, controls were subjects affected with some \\n      diseases other than PAD. RESULTS: When analyzed in the combined panel, the\\n      strongest signal of PAD association was observed at rs1902341 in the intron of\\n      OSBPL10 (p=4.7E-7 for trend test; OR=1.31, 95% CI 1.18-1.46). Also, PAD was\\n      modestly associated at several other loci such as rs2554503 in CSMD1 (p=5.7E-5;\\n      OR=1.32, 95% CI 1.15-1.51) or rs235243 in VSP13D (p=0.04; OR=1.18, 95% CI\\n      1.01-1.37). CONCLUSION: Our genome-wide exploration identified suggestive\\n      evidence of PAD association at the OSBPL10 locus. Because the association has not\\n      reached a genome-wide significant level, further replication study is warranted\\n      for verification in the Japanese population.\\nFAU - Koriyama, Hiroshi\\nAU  - Koriyama H\\nAD  - Department of Geriatric Medicine and Nephrology, Osaka University Graduate School\\n      of Medicine, 2-2 Yamada-oka, Suita, Japan.\\nFAU - Nakagami, Hironori\\nAU  - Nakagami H\\nFAU - Katsuya, Tomohiro\\nAU  - Katsuya T\\nFAU - Sugimoto, Ken\\nAU  - Sugimoto K\\nFAU - Yamashita, Hidetoshi\\nAU  - Yamashita H\\nFAU - Takami, Yoichi\\nAU  - Takami Y\\nFAU - Maeda, Shiro\\nAU  - Maeda S\\nFAU - Kubo, Michiaki\\nAU  - Kubo M\\nFAU - Takahashi, Atsushi\\nAU  - Takahashi A\\nFAU - Nakamura, Yusuke\\nAU  - Nakamura Y\\nFAU - Ogihara, Toshio\\nAU  - Ogihara T\\nFAU - Rakugi, Hiromi\\nAU  - Rakugi H\\nFAU - Kaneda, Yasufumi\\nAU  - Kaneda Y\\nFAU - Morishita, Ryuichi\\nAU  - Morishita R\\nLA  - eng\\nPT  - Comparative Study\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov't\\nDEP - 20100702\\nPL  - Japan\\nTA  - J Atheroscler Thromb\\nJT  - Journal of atherosclerosis and thrombosis\\nJID - 9506298\\nRN  - 0 (Receptors, Steroid)\\nRN  - 0 (oxysterol binding protein)\\nSB  - IM\\nMH  - Aged\\nMH  - Asian Continental Ancestry Group/*genetics\\nMH  - Case-Control Studies\\nMH  - Female\\nMH  - Genetic Predisposition to Disease\\nMH  - Genome, Human\\nMH  - *Genome-Wide Association Study\\nMH  - Genotype\\nMH  - Humans\\nMH  - Japan/epidemiology\\nMH  - Linkage Disequilibrium\\nMH  - Male\\nMH  - Middle Aged\\nMH  - Peripheral Vascular Diseases/epidemiology/*genetics\\nMH  - Polymorphism, Single Nucleotide/*genetics\\nMH  - Receptors, Steroid/*genetics\\nEDAT- 2010/07/09 06:00\\nMHDA- 2011/03/09 06:00\\nCRDT- 2010/07/09 06:00\\nPHST- 2010/07/09 06:00 [entrez]\\nPHST- 2010/07/09 06:00 [pubmed]\\nPHST- 2011/03/09 06:00 [medline]\\nAID - JST.JSTAGE/jat/4291 [pii]\\nAID - 10.5551/jat.4291 [doi]\\nPST - ppublish\\nSO  - J Atheroscler Thromb. 2010 Oct 27;17(10):1054-62. doi: 10.5551/jat.4291. Epub\\n      2010 Jul 2.\", \"PMID- 19056867\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20090226\\nLR  - 20191003\\nIS  - 1533-3450 (Electronic)\\nIS  - 1046-6673 (Linking)\\nVI  - 20\\nIP  - 2\\nDP  - 2009 Feb\\nTI  - Large-scale proteomics and phosphoproteomics of urinary exosomes.\\nPG  - 363-79\\nLID - 10.1681/ASN.2008040406 [doi]\\nAB  - Normal human urine contains large numbers of exosomes, which are 40- to 100-nm\\n      vesicles that originate as the internal vesicles in multivesicular bodies from\\n      every renal epithelial cell type facing the urinary space. Here, we used LC-MS/MS\\n      to profile the proteome of human urinary exosomes. Overall, the analysis\\n      identified 1132 proteins unambiguously, including 177 that are represented on the\\n      Online Mendelian Inheritance in Man database of disease-related genes, suggesting\\n      that exosome analysis is a potential approach to discover urinary biomarkers. We \\n      extended the proteomic analysis to phosphoproteomic profiling using neutral loss \\n      scanning, and this yielded multiple novel phosphorylation sites, including\\n      serine-811 in the thiazide-sensitive Na-Cl co-transporter, NCC. To demonstrate\\n      the potential use of exosome analysis to identify a genetic renal disease, we\\n      carried out immunoblotting of exosomes from urine samples of patients with a\\n      clinical diagnosis of Bartter syndrome type I, showing an absence of the\\n      sodium-potassium-chloride co-transporter 2, NKCC2. The proteomic data are\\n      publicly accessible at http://dir.nhlbi.nih.gov/papers/lkem/exosome/.\\nFAU - Gonzales, Patricia A\\nAU  - Gonzales PA\\nAD  - Laboratory of Kidney and Electrolyte Metabolism, National Heart, Lung, and Blood \\n      Institute, National Institutes of Health, Bethesda, MD 20892, USA.\\nFAU - Pisitkun, Trairak\\nAU  - Pisitkun T\\nFAU - Hoffert, Jason D\\nAU  - Hoffert JD\\nFAU - Tchapyjnikov, Dmitry\\nAU  - Tchapyjnikov D\\nFAU - Star, Robert A\\nAU  - Star RA\\nFAU - Kleta, Robert\\nAU  - Kleta R\\nFAU - Wang, Nam Sun\\nAU  - Wang NS\\nFAU - Knepper, Mark A\\nAU  - Knepper MA\\nLA  - eng\\nGR  - Z01 HL001285-21/Intramural NIH HHS/United States\\nGR  - Z99 HL999999/Intramural NIH HHS/United States\\nPT  - Journal Article\\nDEP - 20081203\\nPL  - United States\\nTA  - J Am Soc Nephrol\\nJT  - Journal of the American Society of Nephrology : JASN\\nJID - 9013836\\nRN  - 0 (Phosphoproteins)\\nRN  - 0 (Proteome)\\nRN  - 0 (Thiazides)\\nRN  - EC 3.6.1.- (Adenosine Triphosphatases)\\nSB  - IM\\nMH  - Adenosine Triphosphatases/chemistry\\nMH  - Adult\\nMH  - Bartter Syndrome/urine\\nMH  - Chromatography, Liquid/methods\\nMH  - Exosomes/metabolism\\nMH  - Female\\nMH  - Humans\\nMH  - Male\\nMH  - Mass Spectrometry/methods\\nMH  - Phosphoproteins/chemistry\\nMH  - Phosphorylation\\nMH  - Proteome\\nMH  - Proteomics/*methods\\nMH  - Thiazides/chemistry\\nMH  - *Urine\\nPMC - PMC2637050\\nEDAT- 2008/12/06 09:00\\nMHDA- 2009/02/27 09:00\\nCRDT- 2008/12/06 09:00\\nPHST- 2008/12/06 09:00 [pubmed]\\nPHST- 2009/02/27 09:00 [medline]\\nPHST- 2008/12/06 09:00 [entrez]\\nAID - ASN.2008040406 [pii]\\nAID - 10.1681/ASN.2008040406 [doi]\\nPST - ppublish\\nSO  - J Am Soc Nephrol. 2009 Feb;20(2):363-79. doi: 10.1681/ASN.2008040406. Epub 2008\\n      Dec 3.\"]"
      ]
     },
     "execution_count": 8,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "pubmedids = [\"25896417\", \"20610895\", \"19056867\"]\n",
    "\n",
    "Bio::PubMed.efetch(pubmedids).each do |pubmedrecord|\n",
    "  # pubmedrecord is a string of the record, in pubmed format\n",
    "  # we can use that to create a new Bio::MEDLINE object\n",
    "  medline = Bio::MEDLINE.new(pubmedrecord)\n",
    "  reference = medline.reference  # reference is a Bio::Reference object\n",
    "  puts reference.bibtex   # the .bibtex method creates the bibtex string of the Bio::Reference\n",
    "end"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "<pre>\n",
    "  \n",
    "\n",
    "</pre>\n",
    "  \n",
    "# The Final Code\n",
    "\n",
    "So finally we put the BibTex code at the end of our script, and...\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "@article{PMID:30789691,\n",
      "  author       = {Meijer, I. A.},\n",
      "  title        = {VPS13D Movement Disorder},\n",
      "  journal      = {},\n",
      "  year         = {1993},\n",
      "  volume       = {},\n",
      "  url          = {http://www.ncbi.nlm.nih.gov/pubmed/30789691},\n",
      "}\n",
      "\n",
      "@article{PMID:29604224,\n",
      "  author       = {Seong, E. and Insolera, R. and Dulovic, M. and Kamsteeg, E. J. and Trinh, J. and Bruggemann, N. and Sandford, E. and Li, S. and Ozel, A. B. and Li, J. Z. and Jewett, T. and Kievit, A. J. A. and Munchau, A. and Shakkottai, V. and Klein, C. and Collins, C. A. and Lohmann, K. and van de Warrenburg, B. P. and Burmeister, M.},\n",
      "  title        = {Mutations in VPS13D lead to a new recessive ataxia with spasticity and mitochondrial defects.},\n",
      "  journal      = {Ann Neurol},\n",
      "  year         = {2018},\n",
      "  volume       = {83},\n",
      "  number       = {6},\n",
      "  pages        = {1075--1088},\n",
      "  url          = {http://www.ncbi.nlm.nih.gov/pubmed/29604224},\n",
      "}\n",
      "\n",
      "@article{PMID:29518281,\n",
      "  author       = {Gauthier, J. and Meijer, I. A. and Lessel, D. and Mencacci, N. E. and Krainc, D. and Hempel, M. and Tsiakas, K. and Prokisch, H. and Rossignol, E. and Helm, M. H. and Rodan, L. H. and Karamchandani, J. and Carecchio, M. and Lubbe, S. J. and Telegrafi, A. and Henderson, L. B. and Lorenzo, K. and Wallace, S. E. and Glass, I. A. and Hamdan, F. F. and Michaud, J. L. and Rouleau, G. A. and Campeau, P. M.},\n",
      "  title        = {Recessive mutations in VPS13D cause childhood onset movement disorders.},\n",
      "  journal      = {Ann Neurol},\n",
      "  year         = {2018},\n",
      "  volume       = {83},\n",
      "  number       = {6},\n",
      "  pages        = {1089--1095},\n",
      "  url          = {http://www.ncbi.nlm.nih.gov/pubmed/29518281},\n",
      "}\n",
      "\n",
      "@article{PMID:29307555,\n",
      "  author       = {Anding, A. L. and Wang, C. and Chang, T. K. and Sliter, D. A. and Powers, C. M. and Hofmann, K. and Youle, R. J. and Baehrecke, E. H.},\n",
      "  title        = {Vps13D Encodes a Ubiquitin-Binding Protein that Is Required for the Regulation of Mitochondrial Size and Clearance.},\n",
      "  journal      = {Curr Biol},\n",
      "  year         = {2018},\n",
      "  volume       = {28},\n",
      "  number       = {2},\n",
      "  pages        = {287--295.e6},\n",
      "  url          = {http://www.ncbi.nlm.nih.gov/pubmed/29307555},\n",
      "}\n",
      "\n",
      "@article{PMID:25896417,\n",
      "  author       = {Nakada, T. A. and Boyd, J. H. and Russell, J. A. and Aguirre-Hernandez, R. and Wilkinson, M. D. and Thair, S. A. and Nakada, E. and McConechy, M. K. and Fjell, C. D. and Walley, K. R.},\n",
      "  title        = {VPS13D Gene Variant Is Associated with Altered IL-6 Production and Mortality in Septic Shock.},\n",
      "  journal      = {J Innate Immun},\n",
      "  year         = {2015},\n",
      "  volume       = {7},\n",
      "  number       = {5},\n",
      "  pages        = {545--553},\n",
      "  url          = {http://www.ncbi.nlm.nih.gov/pubmed/25896417},\n",
      "}\n",
      "\n",
      "@article{PMID:20610895,\n",
      "  author       = {Koriyama, H. and Nakagami, H. and Katsuya, T. and Sugimoto, K. and Yamashita, H. and Takami, Y. and Maeda, S. and Kubo, M. and Takahashi, A. and Nakamura, Y. and Ogihara, T. and Rakugi, H. and Kaneda, Y. and Morishita, R.},\n",
      "  title        = {Identification of evidence suggestive of an association with peripheral arterial disease at the OSBPL10 locus by genome-wide investigation in the Japanese population.},\n",
      "  journal      = {J Atheroscler Thromb},\n",
      "  year         = {2010},\n",
      "  volume       = {17},\n",
      "  number       = {10},\n",
      "  pages        = {1054--1062},\n",
      "  url          = {http://www.ncbi.nlm.nih.gov/pubmed/20610895},\n",
      "}\n",
      "\n",
      "@article{PMID:20301317,\n",
      "  author       = {Bird, T. D.},\n",
      "  title        = {Hereditary Ataxia Overview},\n",
      "  journal      = {},\n",
      "  year         = {1993},\n",
      "  volume       = {},\n",
      "  url          = {http://www.ncbi.nlm.nih.gov/pubmed/20301317},\n",
      "}\n",
      "\n",
      "@article{PMID:19056867,\n",
      "  author       = {Gonzales, P. A. and Pisitkun, T. and Hoffert, J. D. and Tchapyjnikov, D. and Star, R. A. and Kleta, R. and Wang, N. S. and Knepper, M. A.},\n",
      "  title        = {Large-scale proteomics and phosphoproteomics of urinary exosomes.},\n",
      "  journal      = {J Am Soc Nephrol},\n",
      "  year         = {2009},\n",
      "  volume       = {20},\n",
      "  number       = {2},\n",
      "  pages        = {363--379},\n",
      "  url          = {http://www.ncbi.nlm.nih.gov/pubmed/19056867},\n",
      "}\n",
      "\n",
      "@article{PMID:15498460,\n",
      "  author       = {Velayos-Baeza, A. and Vettori, A. and Copley, R. R. and Dobson-Stone, C. and Monaco, A. P.},\n",
      "  title        = {Analysis of the human VPS13 gene family.},\n",
      "  journal      = {Genomics},\n",
      "  year         = {2004},\n",
      "  volume       = {84},\n",
      "  number       = {3},\n",
      "  pages        = {536--549},\n",
      "  url          = {http://www.ncbi.nlm.nih.gov/pubmed/15498460},\n",
      "}\n",
      "\n",
      "@article{PMID:15070428,\n",
      "  author       = {Henry, I. M. and Wilkinson, M. D. and Hernandez, J. M. and Schwarz-Sommer, Z. and Grotewold, E. and Mandoli, D. F.},\n",
      "  title        = {Comparison of ESTs from juvenile and adult phases of the giant unicellular green alga Acetabularia acetabulum.},\n",
      "  journal      = {BMC Plant Biol},\n",
      "  year         = {2004},\n",
      "  volume       = {4},\n",
      "  pages        = {3},\n",
      "  url          = {http://www.ncbi.nlm.nih.gov/pubmed/15070428},\n",
      "}\n",
      "\n",
      "@article{PMID:12421765,\n",
      "  author       = {Nakayama, M. and Kikuno, R. and Ohara, O.},\n",
      "  title        = {Protein-protein interactions between large proteins: two-hybrid screening using a functionally classified library composed of long cDNAs.},\n",
      "  journal      = {Genome Res},\n",
      "  year         = {2002},\n",
      "  volume       = {12},\n",
      "  number       = {11},\n",
      "  pages        = {1773--1784},\n",
      "  url          = {http://www.ncbi.nlm.nih.gov/pubmed/12421765},\n",
      "}\n",
      "\n",
      "@article{PMID:30789691,\n",
      "  author       = {Meijer, I. A.},\n",
      "  title        = {VPS13D Movement Disorder},\n",
      "  journal      = {},\n",
      "  year         = {1993},\n",
      "  volume       = {},\n",
      "  url          = {http://www.ncbi.nlm.nih.gov/pubmed/30789691},\n",
      "}\n",
      "\n",
      "@article{PMID:30258100,\n",
      "  author       = {Conte, C. and Griffis, E. R. and Hickson, I. and Perez-Oliva, A. B.},\n",
      "  title        = {USP45 and Spindly are part of the same complex implicated in cell migration.},\n",
      "  journal      = {Sci Rep},\n",
      "  year         = {2018},\n",
      "  volume       = {8},\n",
      "  number       = {1},\n",
      "  pages        = {14375},\n",
      "  url          = {http://www.ncbi.nlm.nih.gov/pubmed/30258100},\n",
      "}\n",
      "\n",
      "@article{PMID:29604224,\n",
      "  author       = {Seong, E. and Insolera, R. and Dulovic, M. and Kamsteeg, E. J. and Trinh, J. and Bruggemann, N. and Sandford, E. and Li, S. and Ozel, A. B. and Li, J. Z. and Jewett, T. and Kievit, A. J. A. and Munchau, A. and Shakkottai, V. and Klein, C. and Collins, C. A. and Lohmann, K. and van de Warrenburg, B. P. and Burmeister, M.},\n",
      "  title        = {Mutations in VPS13D lead to a new recessive ataxia with spasticity and mitochondrial defects.},\n",
      "  journal      = {Ann Neurol},\n",
      "  year         = {2018},\n",
      "  volume       = {83},\n",
      "  number       = {6},\n",
      "  pages        = {1075--1088},\n",
      "  url          = {http://www.ncbi.nlm.nih.gov/pubmed/29604224},\n",
      "}\n",
      "\n",
      "@article{PMID:29568061,\n",
      "  author       = {Liu, X. and Salokas, K. and Tamene, F. and Jiu, Y. and Weldatsadik, R. G. and Ohman, T. and Varjosalo, M.},\n",
      "  title        = {An AP-MS- and BioID-compatible MAC-tag enables comprehensive mapping of protein interactions and subcellular localizations.},\n",
      "  journal      = {Nat Commun},\n",
      "  year         = {2018},\n",
      "  volume       = {9},\n",
      "  number       = {1},\n",
      "  pages        = {1188},\n",
      "  url          = {http://www.ncbi.nlm.nih.gov/pubmed/29568061},\n",
      "}\n",
      "\n",
      "@article{PMID:29518281,\n",
      "  author       = {Gauthier, J. and Meijer, I. A. and Lessel, D. and Mencacci, N. E. and Krainc, D. and Hempel, M. and Tsiakas, K. and Prokisch, H. and Rossignol, E. and Helm, M. H. and Rodan, L. H. and Karamchandani, J. and Carecchio, M. and Lubbe, S. J. and Telegrafi, A. and Henderson, L. B. and Lorenzo, K. and Wallace, S. E. and Glass, I. A. and Hamdan, F. F. and Michaud, J. L. and Rouleau, G. A. and Campeau, P. M.},\n",
      "  title        = {Recessive mutations in VPS13D cause childhood onset movement disorders.},\n",
      "  journal      = {Ann Neurol},\n",
      "  year         = {2018},\n",
      "  volume       = {83},\n",
      "  number       = {6},\n",
      "  pages        = {1089--1095},\n",
      "  url          = {http://www.ncbi.nlm.nih.gov/pubmed/29518281},\n",
      "}\n",
      "\n",
      "@article{PMID:29507755,\n",
      "  author       = {Yue, Y. and Liu, J. and Cui, X. and Cao, J. and Luo, G. and Zhang, Z. and Cheng, T. and Gao, M. and Shu, X. and Ma, H. and Wang, F. and Wang, X. and Shen, B. and Wang, Y. and Feng, X. and He, C. and Liu, J.},\n",
      "  title        = {VIRMA mediates preferential m(6)A mRNA methylation in 3'UTR and near stop codon and associates with alternative polyadenylation.},\n",
      "  journal      = {Cell Discov},\n",
      "  year         = {2018},\n",
      "  volume       = {4},\n",
      "  pages        = {10},\n",
      "  url          = {http://www.ncbi.nlm.nih.gov/pubmed/29507755},\n",
      "}\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "@article{PMID:29307555,\n",
      "  author       = {Anding, A. L. and Wang, C. and Chang, T. K. and Sliter, D. A. and Powers, C. M. and Hofmann, K. and Youle, R. J. and Baehrecke, E. H.},\n",
      "  title        = {Vps13D Encodes a Ubiquitin-Binding Protein that Is Required for the Regulation of Mitochondrial Size and Clearance.},\n",
      "  journal      = {Curr Biol},\n",
      "  year         = {2018},\n",
      "  volume       = {28},\n",
      "  number       = {2},\n",
      "  pages        = {287--295.e6},\n",
      "  url          = {http://www.ncbi.nlm.nih.gov/pubmed/29307555},\n",
      "}\n",
      "\n",
      "@article{PMID:26344197,\n",
      "  author       = {Wan, C. and Borgeson, B. and Phanse, S. and Tu, F. and Drew, K. and Clark, G. and Xiong, X. and Kagan, O. and Kwan, J. and Bezginov, A. and Chessman, K. and Pal, S. and Cromar, G. and Papoulas, O. and Ni, Z. and Boutz, D. R. and Stoilova, S. and Havugimana, P. C. and Guo, X. and Malty, R. H. and Sarov, M. and Greenblatt, J. and Babu, M. and Derry, W. B. and Tillier, E. R. and Wallingford, J. B. and Parkinson, J. and Marcotte, E. M. and Emili, A.},\n",
      "  title        = {Panorama of ancient metazoan macromolecular complexes.},\n",
      "  journal      = {Nature},\n",
      "  year         = {2015},\n",
      "  volume       = {525},\n",
      "  number       = {7569},\n",
      "  pages        = {339--344},\n",
      "  url          = {http://www.ncbi.nlm.nih.gov/pubmed/26344197},\n",
      "}\n",
      "\n",
      "@article{PMID:25896417,\n",
      "  author       = {Nakada, T. A. and Boyd, J. H. and Russell, J. A. and Aguirre-Hernandez, R. and Wilkinson, M. D. and Thair, S. A. and Nakada, E. and McConechy, M. K. and Fjell, C. D. and Walley, K. R.},\n",
      "  title        = {VPS13D Gene Variant Is Associated with Altered IL-6 Production and Mortality in Septic Shock.},\n",
      "  journal      = {J Innate Immun},\n",
      "  year         = {2015},\n",
      "  volume       = {7},\n",
      "  number       = {5},\n",
      "  pages        = {545--553},\n",
      "  url          = {http://www.ncbi.nlm.nih.gov/pubmed/25896417},\n",
      "}\n",
      "\n",
      "@article{PMID:21873635,\n",
      "  author       = {Gaudet, P. and Livstone, M. S. and Lewis, S. E. and Thomas, P. D.},\n",
      "  title        = {Phylogenetic-based propagation of functional annotations within the Gene Ontology consortium.},\n",
      "  journal      = {Brief Bioinform},\n",
      "  year         = {2011},\n",
      "  volume       = {12},\n",
      "  number       = {5},\n",
      "  pages        = {449--462},\n",
      "  url          = {http://www.ncbi.nlm.nih.gov/pubmed/21873635},\n",
      "}\n",
      "\n",
      "@article{PMID:21182205,\n",
      "  author       = {Tarallo, R. and Bamundo, A. and Nassa, G. and Nola, E. and Paris, O. and Ambrosino, C. and Facchiano, A. and Baumann, M. and Nyman, T. A. and Weisz, A.},\n",
      "  title        = {Identification of proteins associated with ligand-activated estrogen receptor alpha in human breast cancer cell nuclei by tandem affinity purification and nano LC-MS/MS.},\n",
      "  journal      = {Proteomics},\n",
      "  year         = {2011},\n",
      "  volume       = {11},\n",
      "  number       = {1},\n",
      "  pages        = {172--179},\n",
      "  url          = {http://www.ncbi.nlm.nih.gov/pubmed/21182205},\n",
      "}\n",
      "\n",
      "@article{PMID:20610895,\n",
      "  author       = {Koriyama, H. and Nakagami, H. and Katsuya, T. and Sugimoto, K. and Yamashita, H. and Takami, Y. and Maeda, S. and Kubo, M. and Takahashi, A. and Nakamura, Y. and Ogihara, T. and Rakugi, H. and Kaneda, Y. and Morishita, R.},\n",
      "  title        = {Identification of evidence suggestive of an association with peripheral arterial disease at the OSBPL10 locus by genome-wide investigation in the Japanese population.},\n",
      "  journal      = {J Atheroscler Thromb},\n",
      "  year         = {2010},\n",
      "  volume       = {17},\n",
      "  number       = {10},\n",
      "  pages        = {1054--1062},\n",
      "  url          = {http://www.ncbi.nlm.nih.gov/pubmed/20610895},\n",
      "}\n",
      "\n",
      "@article{PMID:20379614,\n",
      "  author       = {Rose, J. E. and Behm, F. M. and Drgon, T. and Johnson, C. and Uhl, G. R.},\n",
      "  title        = {Personalized smoking cessation: interactions between nicotine dose, dependence and quit-success genotype score.},\n",
      "  journal      = {Mol Med},\n",
      "  year         = {2010},\n",
      "  volume       = {16},\n",
      "  number       = {7-8},\n",
      "  pages        = {247--253},\n",
      "  url          = {http://www.ncbi.nlm.nih.gov/pubmed/20379614},\n",
      "}\n",
      "\n",
      "@article{PMID:20301317,\n",
      "  author       = {Bird, T. D.},\n",
      "  title        = {Hereditary Ataxia Overview},\n",
      "  journal      = {},\n",
      "  year         = {1993},\n",
      "  volume       = {},\n",
      "  url          = {http://www.ncbi.nlm.nih.gov/pubmed/20301317},\n",
      "}\n",
      "\n",
      "@article{PMID:19056867,\n",
      "  author       = {Gonzales, P. A. and Pisitkun, T. and Hoffert, J. D. and Tchapyjnikov, D. and Star, R. A. and Kleta, R. and Wang, N. S. and Knepper, M. A.},\n",
      "  title        = {Large-scale proteomics and phosphoproteomics of urinary exosomes.},\n",
      "  journal      = {J Am Soc Nephrol},\n",
      "  year         = {2009},\n",
      "  volume       = {20},\n",
      "  number       = {2},\n",
      "  pages        = {363--379},\n",
      "  url          = {http://www.ncbi.nlm.nih.gov/pubmed/19056867},\n",
      "}\n",
      "\n",
      "@article{PMID:16710414,\n",
      "  author       = {Gregory, S. G. and Barlow, K. F. and McLay, K. E. and Kaul, R. and Swarbreck, D. and Dunham, A. and Scott, C. E. and Howe, K. L. and Woodfine, K. and Spencer, C. C. and Jones, M. C. and Gillson, C. and Searle, S. and Zhou, Y. and Kokocinski, F. and McDonald, L. and Evans, R. and Phillips, K. and Atkinson, A. and Cooper, R. and Jones, C. and Hall, R. E. and Andrews, T. D. and Lloyd, C. and Ainscough, R. and Almeida, J. P. and Ambrose, K. D. and Anderson, F. and Andrew, R. W. and Ashwell, R. I. and Aubin, K. and Babbage, A. K. and Bagguley, C. L. and Bailey, J. and Beasley, H. and Bethel, G. and Bird, C. P. and Bray-Allen, S. and Brown, J. Y. and Brown, A. J. and Buckley, D. and Burton, J. and Bye, J. and Carder, C. and Chapman, J. C. and Clark, S. Y. and Clarke, G. and Clee, C. and Cobley, V. and Collier, R. E. and Corby, N. and Coville, G. J. and Davies, J. and Deadman, R. and Dunn, M. and Earthrowl, M. and Ellington, A. G. and Errington, H. and Frankish, A. and Frankland, J. and French, L. and Garner, P. and Garnett, J. and Gay, L. and Ghori, M. R. and Gibson, R. and Gilby, L. M. and Gillett, W. and Glithero, R. J. and Grafham, D. V. and Griffiths, C. and Griffiths-Jones, S. and Grocock, R. and Hammond, S. and Harrison, E. S. and Hart, E. and Haugen, E. and Heath, P. D. and Holmes, S. and Holt, K. and Howden, P. J. and Hunt, A. R. and Hunt, S. E. and Hunter, G. and Isherwood, J. and James, R. and Johnson, C. and Johnson, D. and Joy, A. and Kay, M. and Kershaw, J. K. and Kibukawa, M. and Kimberley, A. M. and King, A. and Knights, A. J. and Lad, H. and Laird, G. and Lawlor, S. and Leongamornlert, D. A. and Lloyd, D. M. and Loveland, J. and Lovell, J. and Lush, M. J. and Lyne, R. and Martin, S. and Mashreghi-Mohammadi, M. and Matthews, L. and Matthews, N. S. and McLaren, S. and Milne, S. and Mistry, S. and Moore, M. J. and Nickerson, T. and O'Dell, C. N. and Oliver, K. and Palmeiri, A. and Palmer, S. A. and Parker, A. and Patel, D. and Pearce, A. V. and Peck, A. I. and Pelan, S. and Phelps, K. and Phillimore, B. J. and Plumb, R. and Rajan, J. and Raymond, C. and Rouse, G. and Saenphimmachak, C. and Sehra, H. K. and Sheridan, E. and Shownkeen, R. and Sims, S. and Skuce, C. D. and Smith, M. and Steward, C. and Subramanian, S. and Sycamore, N. and Tracey, A. and Tromans, A. and Van Helmond, Z. and Wall, M. and Wallis, J. M. and White, S. and Whitehead, S. L. and Wilkinson, J. E. and Willey, D. L. and Williams, H. and Wilming, L. and Wray, P. W. and Wu, Z. and Coulson, A. and Vaudin, M. and Sulston, J. E. and Durbin, R. and Hubbard, T. and Wooster, R. and Dunham, I. and Carter, N. P. and McVean, G. and Ross, M. T. and Harrow, J. and Olson, M. V. and Beck, S. and Rogers, J. and Bentley, D. R. and Banerjee, R. and Bryant, S. P. and Burford, D. C. and Burrill, W. D. and Clegg, S. M. and Dhami, P. and Dovey, O. and Faulkner, L. M. and Gribble, S. M. and Langford, C. F. and Pandian, R. D. and Porter, K. M. and Prigmore, E.},\n",
      "  title        = {The DNA sequence and biological annotation of human chromosome 1.},\n",
      "  journal      = {Nature},\n",
      "  year         = {2006},\n",
      "  volume       = {441},\n",
      "  number       = {7091},\n",
      "  pages        = {315--321},\n",
      "  url          = {http://www.ncbi.nlm.nih.gov/pubmed/16710414},\n",
      "}\n",
      "\n",
      "@article{PMID:15498460,\n",
      "  author       = {Velayos-Baeza, A. and Vettori, A. and Copley, R. R. and Dobson-Stone, C. and Monaco, A. P.},\n",
      "  title        = {Analysis of the human VPS13 gene family.},\n",
      "  journal      = {Genomics},\n",
      "  year         = {2004},\n",
      "  volume       = {84},\n",
      "  number       = {3},\n",
      "  pages        = {536--549},\n",
      "  url          = {http://www.ncbi.nlm.nih.gov/pubmed/15498460},\n",
      "}\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "@article{PMID:15489334,\n",
      "  author       = {Gerhard, D. S. and Wagner, L. and Feingold, E. A. and Shenmen, C. M. and Grouse, L. H. and Schuler, G. and Klein, S. L. and Old, S. and Rasooly, R. and Good, P. and Guyer, M. and Peck, A. M. and Derge, J. G. and Lipman, D. and Collins, F. S. and Jang, W. and Sherry, S. and Feolo, M. and Misquitta, L. and Lee, E. and Rotmistrovsky, K. and Greenhut, S. F. and Schaefer, C. F. and Buetow, K. and Bonner, T. I. and Haussler, D. and Kent, J. and Kiekhaus, M. and Furey, T. and Brent, M. and Prange, C. and Schreiber, K. and Shapiro, N. and Bhat, N. K. and Hopkins, R. F. and Hsie, F. and Driscoll, T. and Soares, M. B. and Casavant, T. L. and Scheetz, T. E. and Brown-stein, M. J. and Usdin, T. B. and Toshiyuki, S. and Carninci, P. and Piao, Y. and Dudekula, D. B. and Ko, M. S. and Kawakami, K. and Suzuki, Y. and Sugano, S. and Gruber, C. E. and Smith, M. R. and Simmons, B. and Moore, T. and Waterman, R. and Johnson, S. L. and Ruan, Y. and Wei, C. L. and Mathavan, S. and Gunaratne, P. H. and Wu, J. and Garcia, A. M. and Hulyk, S. W. and Fuh, E. and Yuan, Y. and Sneed, A. and Kowis, C. and Hodgson, A. and Muzny, D. M. and McPherson, J. and Gibbs, R. A. and Fahey, J. and Helton, E. and Ketteman, M. and Madan, A. and Rodrigues, S. and Sanchez, A. and Whiting, M. and Madari, A. and Young, A. C. and Wetherby, K. D. and Granite, S. J. and Kwong, P. N. and Brinkley, C. P. and Pearson, R. L. and Bouffard, G. G. and Blakesly, R. W. and Green, E. D. and Dickson, M. C. and Rodriguez, A. C. and Grimwood, J. and Schmutz, J. and Myers, R. M. and Butterfield, Y. S. and Griffith, M. and Griffith, O. L. and Krzywinski, M. I. and Liao, N. and Morin, R. and Palmquist, D. and Petrescu, A. S. and Skalska, U. and Smailus, D. E. and Stott, J. M. and Schnerch, A. and Schein, J. E. and Jones, S. J. and Holt, R. A. and Baross, A. and Marra, M. A. and Clifton, S. and Makowski, K. A. and Bosak, S. and Malek, J.},\n",
      "  title        = {The status, quality, and expansion of the NIH full-length cDNA project: the Mammalian Gene Collection (MGC).},\n",
      "  journal      = {Genome Res},\n",
      "  year         = {2004},\n",
      "  volume       = {14},\n",
      "  number       = {10B},\n",
      "  pages        = {2121--2127},\n",
      "  url          = {http://www.ncbi.nlm.nih.gov/pubmed/15489334},\n",
      "}\n",
      "\n",
      "@article{PMID:14702039,\n",
      "  author       = {Ota, T. and Suzuki, Y. and Nishikawa, T. and Otsuki, T. and Sugiyama, T. and Irie, R. and Wakamatsu, A. and Hayashi, K. and Sato, H. and Nagai, K. and Kimura, K. and Makita, H. and Sekine, M. and Obayashi, M. and Nishi, T. and Shibahara, T. and Tanaka, T. and Ishii, S. and Yamamoto, J. and Saito, K. and Kawai, Y. and Isono, Y. and Nakamura, Y. and Nagahari, K. and Murakami, K. and Yasuda, T. and Iwayanagi, T. and Wagatsuma, M. and Shiratori, A. and Sudo, H. and Hosoiri, T. and Kaku, Y. and Kodaira, H. and Kondo, H. and Sugawara, M. and Takahashi, M. and Kanda, K. and Yokoi, T. and Furuya, T. and Kikkawa, E. and Omura, Y. and Abe, K. and Kamihara, K. and Katsuta, N. and Sato, K. and Tanikawa, M. and Yamazaki, M. and Ninomiya, K. and Ishibashi, T. and Yamashita, H. and Murakawa, K. and Fujimori, K. and Tanai, H. and Kimata, M. and Watanabe, M. and Hiraoka, S. and Chiba, Y. and Ishida, S. and Ono, Y. and Takiguchi, S. and Watanabe, S. and Yosida, M. and Hotuta, T. and Kusano, J. and Kanehori, K. and Takahashi-Fujii, A. and Hara, H. and Tanase, T. O. and Nomura, Y. and Togiya, S. and Komai, F. and Hara, R. and Takeuchi, K. and Arita, M. and Imose, N. and Musashino, K. and Yuuki, H. and Oshima, A. and Sasaki, N. and Aotsuka, S. and Yoshikawa, Y. and Matsunawa, H. and Ichihara, T. and Shiohata, N. and Sano, S. and Moriya, S. and Momiyama, H. and Satoh, N. and Takami, S. and Terashima, Y. and Suzuki, O. and Nakagawa, S. and Senoh, A. and Mizoguchi, H. and Goto, Y. and Shimizu, F. and Wakebe, H. and Hishigaki, H. and Watanabe, T. and Sugiyama, A. and Takemoto, M. and Kawakami, B. and Yamazaki, M. and Watanabe, K. and Kumagai, A. and Itakura, S. and Fukuzumi, Y. and Fujimori, Y. and Komiyama, M. and Tashiro, H. and Tanigami, A. and Fujiwara, T. and Ono, T. and Yamada, K. and Fujii, Y. and Ozaki, K. and Hirao, M. and Ohmori, Y. and Kawabata, A. and Hikiji, T. and Kobatake, N. and Inagaki, H. and Ikema, Y. and Okamoto, S. and Okitani, R. and Kawakami, T. and Noguchi, S. and Itoh, T. and Shigeta, K. and Senba, T. and Matsumura, K. and Nakajima, Y. and Mizuno, T. and Morinaga, M. and Sasaki, M. and Togashi, T. and Oyama, M. and Hata, H. and Watanabe, M. and Komatsu, T. and Mizushima-Sugano, J. and Satoh, T. and Shirai, Y. and Takahashi, Y. and Nakagawa, K. and Okumura, K. and Nagase, T. and Nomura, N. and Kikuchi, H. and Masuho, Y. and Yamashita, R. and Nakai, K. and Yada, T. and Nakamura, Y. and Ohara, O. and Isogai, T. and Sugano, S.},\n",
      "  title        = {Complete sequencing and characterization of 21,243 full-length human cDNAs.},\n",
      "  journal      = {Nat Genet},\n",
      "  year         = {2004},\n",
      "  volume       = {36},\n",
      "  number       = {1},\n",
      "  pages        = {40--45},\n",
      "  url          = {http://www.ncbi.nlm.nih.gov/pubmed/14702039},\n",
      "}\n",
      "\n",
      "@article{PMID:12477932,\n",
      "  author       = {Strausberg, R. L. and Feingold, E. A. and Grouse, L. H. and Derge, J. G. and Klausner, R. D. and Collins, F. S. and Wagner, L. and Shenmen, C. M. and Schuler, G. D. and Altschul, S. F. and Zeeberg, B. and Buetow, K. H. and Schaefer, C. F. and Bhat, N. K. and Hopkins, R. F. and Jordan, H. and Moore, T. and Max, S. I. and Wang, J. and Hsieh, F. and Diatchenko, L. and Marusina, K. and Farmer, A. A. and Rubin, G. M. and Hong, L. and Stapleton, M. and Soares, M. B. and Bonaldo, M. F. and Casavant, T. L. and Scheetz, T. E. and Brownstein, M. J. and Usdin, T. B. and Toshiyuki, S. and Carninci, P. and Prange, C. and Raha, S. S. and Loquellano, N. A. and Peters, G. J. and Abramson, R. D. and Mullahy, S. J. and Bosak, S. A. and McEwan, P. J. and McKernan, K. J. and Malek, J. A. and Gunaratne, P. H. and Richards, S. and Worley, K. C. and Hale, S. and Garcia, A. M. and Gay, L. J. and Hulyk, S. W. and Villalon, D. K. and Muzny, D. M. and Sodergren, E. J. and Lu, X. and Gibbs, R. A. and Fahey, J. and Helton, E. and Ketteman, M. and Madan, A. and Rodrigues, S. and Sanchez, A. and Whiting, M. and Madan, A. and Young, A. C. and Shevchenko, Y. and Bouffard, G. G. and Blakesley, R. W. and Touchman, J. W. and Green, E. D. and Dickson, M. C. and Rodriguez, A. C. and Grimwood, J. and Schmutz, J. and Myers, R. M. and Butterfield, Y. S. and Krzywinski, M. I. and Skalska, U. and Smailus, D. E. and Schnerch, A. and Schein, J. E. and Jones, S. J. and Marra, M. A.},\n",
      "  title        = {Generation and initial analysis of more than 15,000 full-length human and mouse cDNA sequences.},\n",
      "  journal      = {Proc Natl Acad Sci U S A},\n",
      "  year         = {2002},\n",
      "  volume       = {99},\n",
      "  number       = {26},\n",
      "  pages        = {16899--16903},\n",
      "  url          = {http://www.ncbi.nlm.nih.gov/pubmed/12477932},\n",
      "}\n",
      "\n",
      "@article{PMID:12421765,\n",
      "  author       = {Nakayama, M. and Kikuno, R. and Ohara, O.},\n",
      "  title        = {Protein-protein interactions between large proteins: two-hybrid screening using a functionally classified library composed of long cDNAs.},\n",
      "  journal      = {Genome Res},\n",
      "  year         = {2002},\n",
      "  volume       = {12},\n",
      "  number       = {11},\n",
      "  pages        = {1773--1784},\n",
      "  url          = {http://www.ncbi.nlm.nih.gov/pubmed/12421765},\n",
      "}\n",
      "\n",
      "@article{PMID:9455484,\n",
      "  author       = {Seki, N. and Ohira, M. and Nagase, T. and Ishikawa, K. and Miyajima, N. and Nakajima, D. and Nomura, N. and Ohara, O.},\n",
      "  title        = {Characterization of cDNA clones in size-fractionated cDNA libraries from human brain.},\n",
      "  journal      = {DNA Res},\n",
      "  year         = {1997},\n",
      "  volume       = {4},\n",
      "  number       = {5},\n",
      "  pages        = {345--349},\n",
      "  url          = {http://www.ncbi.nlm.nih.gov/pubmed/9455484},\n",
      "}\n",
      "\n",
      "@article{PMID:9110174,\n",
      "  author       = {Yu, W. and Andersson, B. and Worley, K. C. and Muzny, D. M. and Ding, Y. and Liu, W. and Ricafrente, J. Y. and Wentland, M. A. and Lennon, G. and Gibbs, R. A.},\n",
      "  title        = {Large-scale concatenation cDNA sequencing.},\n",
      "  journal      = {Genome Res},\n",
      "  year         = {1997},\n",
      "  volume       = {7},\n",
      "  number       = {4},\n",
      "  pages        = {353--358},\n",
      "  url          = {http://www.ncbi.nlm.nih.gov/pubmed/9110174},\n",
      "}\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "@article{PMID:8619474,\n",
      "  author       = {Andersson, B. and Wentland, M. A. and Ricafrente, J. Y. and Liu, W. and Gibbs, R. A.},\n",
      "  title        = {A \"double adaptor\" method for improved shotgun library construction.},\n",
      "  journal      = {Anal Biochem},\n",
      "  year         = {1996},\n",
      "  volume       = {236},\n",
      "  number       = {1},\n",
      "  pages        = {107--113},\n",
      "  url          = {http://www.ncbi.nlm.nih.gov/pubmed/8619474},\n",
      "}\n",
      "\n",
      "@article{PMID:30789691,\n",
      "  author       = {Meijer, I. A.},\n",
      "  title        = {VPS13D Movement Disorder},\n",
      "  journal      = {},\n",
      "  year         = {1993},\n",
      "  volume       = {},\n",
      "  url          = {http://www.ncbi.nlm.nih.gov/pubmed/30789691},\n",
      "}\n",
      "\n",
      "@article{PMID:30258100,\n",
      "  author       = {Conte, C. and Griffis, E. R. and Hickson, I. and Perez-Oliva, A. B.},\n",
      "  title        = {USP45 and Spindly are part of the same complex implicated in cell migration.},\n",
      "  journal      = {Sci Rep},\n",
      "  year         = {2018},\n",
      "  volume       = {8},\n",
      "  number       = {1},\n",
      "  pages        = {14375},\n",
      "  url          = {http://www.ncbi.nlm.nih.gov/pubmed/30258100},\n",
      "}\n",
      "\n",
      "@article{PMID:29604224,\n",
      "  author       = {Seong, E. and Insolera, R. and Dulovic, M. and Kamsteeg, E. J. and Trinh, J. and Bruggemann, N. and Sandford, E. and Li, S. and Ozel, A. B. and Li, J. Z. and Jewett, T. and Kievit, A. J. A. and Munchau, A. and Shakkottai, V. and Klein, C. and Collins, C. A. and Lohmann, K. and van de Warrenburg, B. P. and Burmeister, M.},\n",
      "  title        = {Mutations in VPS13D lead to a new recessive ataxia with spasticity and mitochondrial defects.},\n",
      "  journal      = {Ann Neurol},\n",
      "  year         = {2018},\n",
      "  volume       = {83},\n",
      "  number       = {6},\n",
      "  pages        = {1075--1088},\n",
      "  url          = {http://www.ncbi.nlm.nih.gov/pubmed/29604224},\n",
      "}\n",
      "\n",
      "@article{PMID:29568061,\n",
      "  author       = {Liu, X. and Salokas, K. and Tamene, F. and Jiu, Y. and Weldatsadik, R. G. and Ohman, T. and Varjosalo, M.},\n",
      "  title        = {An AP-MS- and BioID-compatible MAC-tag enables comprehensive mapping of protein interactions and subcellular localizations.},\n",
      "  journal      = {Nat Commun},\n",
      "  year         = {2018},\n",
      "  volume       = {9},\n",
      "  number       = {1},\n",
      "  pages        = {1188},\n",
      "  url          = {http://www.ncbi.nlm.nih.gov/pubmed/29568061},\n",
      "}\n",
      "\n",
      "@article{PMID:29518281,\n",
      "  author       = {Gauthier, J. and Meijer, I. A. and Lessel, D. and Mencacci, N. E. and Krainc, D. and Hempel, M. and Tsiakas, K. and Prokisch, H. and Rossignol, E. and Helm, M. H. and Rodan, L. H. and Karamchandani, J. and Carecchio, M. and Lubbe, S. J. and Telegrafi, A. and Henderson, L. B. and Lorenzo, K. and Wallace, S. E. and Glass, I. A. and Hamdan, F. F. and Michaud, J. L. and Rouleau, G. A. and Campeau, P. M.},\n",
      "  title        = {Recessive mutations in VPS13D cause childhood onset movement disorders.},\n",
      "  journal      = {Ann Neurol},\n",
      "  year         = {2018},\n",
      "  volume       = {83},\n",
      "  number       = {6},\n",
      "  pages        = {1089--1095},\n",
      "  url          = {http://www.ncbi.nlm.nih.gov/pubmed/29518281},\n",
      "}\n",
      "\n",
      "@article{PMID:29507755,\n",
      "  author       = {Yue, Y. and Liu, J. and Cui, X. and Cao, J. and Luo, G. and Zhang, Z. and Cheng, T. and Gao, M. and Shu, X. and Ma, H. and Wang, F. and Wang, X. and Shen, B. and Wang, Y. and Feng, X. and He, C. and Liu, J.},\n",
      "  title        = {VIRMA mediates preferential m(6)A mRNA methylation in 3'UTR and near stop codon and associates with alternative polyadenylation.},\n",
      "  journal      = {Cell Discov},\n",
      "  year         = {2018},\n",
      "  volume       = {4},\n",
      "  pages        = {10},\n",
      "  url          = {http://www.ncbi.nlm.nih.gov/pubmed/29507755},\n",
      "}\n",
      "\n",
      "@article{PMID:29307555,\n",
      "  author       = {Anding, A. L. and Wang, C. and Chang, T. K. and Sliter, D. A. and Powers, C. M. and Hofmann, K. and Youle, R. J. and Baehrecke, E. H.},\n",
      "  title        = {Vps13D Encodes a Ubiquitin-Binding Protein that Is Required for the Regulation of Mitochondrial Size and Clearance.},\n",
      "  journal      = {Curr Biol},\n",
      "  year         = {2018},\n",
      "  volume       = {28},\n",
      "  number       = {2},\n",
      "  pages        = {287--295.e6},\n",
      "  url          = {http://www.ncbi.nlm.nih.gov/pubmed/29307555},\n",
      "}\n",
      "\n",
      "@article{PMID:26344197,\n",
      "  author       = {Wan, C. and Borgeson, B. and Phanse, S. and Tu, F. and Drew, K. and Clark, G. and Xiong, X. and Kagan, O. and Kwan, J. and Bezginov, A. and Chessman, K. and Pal, S. and Cromar, G. and Papoulas, O. and Ni, Z. and Boutz, D. R. and Stoilova, S. and Havugimana, P. C. and Guo, X. and Malty, R. H. and Sarov, M. and Greenblatt, J. and Babu, M. and Derry, W. B. and Tillier, E. R. and Wallingford, J. B. and Parkinson, J. and Marcotte, E. M. and Emili, A.},\n",
      "  title        = {Panorama of ancient metazoan macromolecular complexes.},\n",
      "  journal      = {Nature},\n",
      "  year         = {2015},\n",
      "  volume       = {525},\n",
      "  number       = {7569},\n",
      "  pages        = {339--344},\n",
      "  url          = {http://www.ncbi.nlm.nih.gov/pubmed/26344197},\n",
      "}\n",
      "\n",
      "@article{PMID:25896417,\n",
      "  author       = {Nakada, T. A. and Boyd, J. H. and Russell, J. A. and Aguirre-Hernandez, R. and Wilkinson, M. D. and Thair, S. A. and Nakada, E. and McConechy, M. K. and Fjell, C. D. and Walley, K. R.},\n",
      "  title        = {VPS13D Gene Variant Is Associated with Altered IL-6 Production and Mortality in Septic Shock.},\n",
      "  journal      = {J Innate Immun},\n",
      "  year         = {2015},\n",
      "  volume       = {7},\n",
      "  number       = {5},\n",
      "  pages        = {545--553},\n",
      "  url          = {http://www.ncbi.nlm.nih.gov/pubmed/25896417},\n",
      "}\n",
      "\n",
      "@article{PMID:21873635,\n",
      "  author       = {Gaudet, P. and Livstone, M. S. and Lewis, S. E. and Thomas, P. D.},\n",
      "  title        = {Phylogenetic-based propagation of functional annotations within the Gene Ontology consortium.},\n",
      "  journal      = {Brief Bioinform},\n",
      "  year         = {2011},\n",
      "  volume       = {12},\n",
      "  number       = {5},\n",
      "  pages        = {449--462},\n",
      "  url          = {http://www.ncbi.nlm.nih.gov/pubmed/21873635},\n",
      "}\n",
      "\n",
      "@article{PMID:21182205,\n",
      "  author       = {Tarallo, R. and Bamundo, A. and Nassa, G. and Nola, E. and Paris, O. and Ambrosino, C. and Facchiano, A. and Baumann, M. and Nyman, T. A. and Weisz, A.},\n",
      "  title        = {Identification of proteins associated with ligand-activated estrogen receptor alpha in human breast cancer cell nuclei by tandem affinity purification and nano LC-MS/MS.},\n",
      "  journal      = {Proteomics},\n",
      "  year         = {2011},\n",
      "  volume       = {11},\n",
      "  number       = {1},\n",
      "  pages        = {172--179},\n",
      "  url          = {http://www.ncbi.nlm.nih.gov/pubmed/21182205},\n",
      "}\n",
      "\n",
      "@article{PMID:20610895,\n",
      "  author       = {Koriyama, H. and Nakagami, H. and Katsuya, T. and Sugimoto, K. and Yamashita, H. and Takami, Y. and Maeda, S. and Kubo, M. and Takahashi, A. and Nakamura, Y. and Ogihara, T. and Rakugi, H. and Kaneda, Y. and Morishita, R.},\n",
      "  title        = {Identification of evidence suggestive of an association with peripheral arterial disease at the OSBPL10 locus by genome-wide investigation in the Japanese population.},\n",
      "  journal      = {J Atheroscler Thromb},\n",
      "  year         = {2010},\n",
      "  volume       = {17},\n",
      "  number       = {10},\n",
      "  pages        = {1054--1062},\n",
      "  url          = {http://www.ncbi.nlm.nih.gov/pubmed/20610895},\n",
      "}\n",
      "\n",
      "@article{PMID:20379614,\n",
      "  author       = {Rose, J. E. and Behm, F. M. and Drgon, T. and Johnson, C. and Uhl, G. R.},\n",
      "  title        = {Personalized smoking cessation: interactions between nicotine dose, dependence and quit-success genotype score.},\n",
      "  journal      = {Mol Med},\n",
      "  year         = {2010},\n",
      "  volume       = {16},\n",
      "  number       = {7-8},\n",
      "  pages        = {247--253},\n",
      "  url          = {http://www.ncbi.nlm.nih.gov/pubmed/20379614},\n",
      "}\n",
      "\n",
      "@article{PMID:20301317,\n",
      "  author       = {Bird, T. D.},\n",
      "  title        = {Hereditary Ataxia Overview},\n",
      "  journal      = {},\n",
      "  year         = {1993},\n",
      "  volume       = {},\n",
      "  url          = {http://www.ncbi.nlm.nih.gov/pubmed/20301317},\n",
      "}\n",
      "\n",
      "@article{PMID:19056867,\n",
      "  author       = {Gonzales, P. A. and Pisitkun, T. and Hoffert, J. D. and Tchapyjnikov, D. and Star, R. A. and Kleta, R. and Wang, N. S. and Knepper, M. A.},\n",
      "  title        = {Large-scale proteomics and phosphoproteomics of urinary exosomes.},\n",
      "  journal      = {J Am Soc Nephrol},\n",
      "  year         = {2009},\n",
      "  volume       = {20},\n",
      "  number       = {2},\n",
      "  pages        = {363--379},\n",
      "  url          = {http://www.ncbi.nlm.nih.gov/pubmed/19056867},\n",
      "}\n",
      "\n",
      "@article{PMID:16710414,\n",
      "  author       = {Gregory, S. G. and Barlow, K. F. and McLay, K. E. and Kaul, R. and Swarbreck, D. and Dunham, A. and Scott, C. E. and Howe, K. L. and Woodfine, K. and Spencer, C. C. and Jones, M. C. and Gillson, C. and Searle, S. and Zhou, Y. and Kokocinski, F. and McDonald, L. and Evans, R. and Phillips, K. and Atkinson, A. and Cooper, R. and Jones, C. and Hall, R. E. and Andrews, T. D. and Lloyd, C. and Ainscough, R. and Almeida, J. P. and Ambrose, K. D. and Anderson, F. and Andrew, R. W. and Ashwell, R. I. and Aubin, K. and Babbage, A. K. and Bagguley, C. L. and Bailey, J. and Beasley, H. and Bethel, G. and Bird, C. P. and Bray-Allen, S. and Brown, J. Y. and Brown, A. J. and Buckley, D. and Burton, J. and Bye, J. and Carder, C. and Chapman, J. C. and Clark, S. Y. and Clarke, G. and Clee, C. and Cobley, V. and Collier, R. E. and Corby, N. and Coville, G. J. and Davies, J. and Deadman, R. and Dunn, M. and Earthrowl, M. and Ellington, A. G. and Errington, H. and Frankish, A. and Frankland, J. and French, L. and Garner, P. and Garnett, J. and Gay, L. and Ghori, M. R. and Gibson, R. and Gilby, L. M. and Gillett, W. and Glithero, R. J. and Grafham, D. V. and Griffiths, C. and Griffiths-Jones, S. and Grocock, R. and Hammond, S. and Harrison, E. S. and Hart, E. and Haugen, E. and Heath, P. D. and Holmes, S. and Holt, K. and Howden, P. J. and Hunt, A. R. and Hunt, S. E. and Hunter, G. and Isherwood, J. and James, R. and Johnson, C. and Johnson, D. and Joy, A. and Kay, M. and Kershaw, J. K. and Kibukawa, M. and Kimberley, A. M. and King, A. and Knights, A. J. and Lad, H. and Laird, G. and Lawlor, S. and Leongamornlert, D. A. and Lloyd, D. M. and Loveland, J. and Lovell, J. and Lush, M. J. and Lyne, R. and Martin, S. and Mashreghi-Mohammadi, M. and Matthews, L. and Matthews, N. S. and McLaren, S. and Milne, S. and Mistry, S. and Moore, M. J. and Nickerson, T. and O'Dell, C. N. and Oliver, K. and Palmeiri, A. and Palmer, S. A. and Parker, A. and Patel, D. and Pearce, A. V. and Peck, A. I. and Pelan, S. and Phelps, K. and Phillimore, B. J. and Plumb, R. and Rajan, J. and Raymond, C. and Rouse, G. and Saenphimmachak, C. and Sehra, H. K. and Sheridan, E. and Shownkeen, R. and Sims, S. and Skuce, C. D. and Smith, M. and Steward, C. and Subramanian, S. and Sycamore, N. and Tracey, A. and Tromans, A. and Van Helmond, Z. and Wall, M. and Wallis, J. M. and White, S. and Whitehead, S. L. and Wilkinson, J. E. and Willey, D. L. and Williams, H. and Wilming, L. and Wray, P. W. and Wu, Z. and Coulson, A. and Vaudin, M. and Sulston, J. E. and Durbin, R. and Hubbard, T. and Wooster, R. and Dunham, I. and Carter, N. P. and McVean, G. and Ross, M. T. and Harrow, J. and Olson, M. V. and Beck, S. and Rogers, J. and Bentley, D. R. and Banerjee, R. and Bryant, S. P. and Burford, D. C. and Burrill, W. D. and Clegg, S. M. and Dhami, P. and Dovey, O. and Faulkner, L. M. and Gribble, S. M. and Langford, C. F. and Pandian, R. D. and Porter, K. M. and Prigmore, E.},\n",
      "  title        = {The DNA sequence and biological annotation of human chromosome 1.},\n",
      "  journal      = {Nature},\n",
      "  year         = {2006},\n",
      "  volume       = {441},\n",
      "  number       = {7091},\n",
      "  pages        = {315--321},\n",
      "  url          = {http://www.ncbi.nlm.nih.gov/pubmed/16710414},\n",
      "}\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "@article{PMID:15498460,\n",
      "  author       = {Velayos-Baeza, A. and Vettori, A. and Copley, R. R. and Dobson-Stone, C. and Monaco, A. P.},\n",
      "  title        = {Analysis of the human VPS13 gene family.},\n",
      "  journal      = {Genomics},\n",
      "  year         = {2004},\n",
      "  volume       = {84},\n",
      "  number       = {3},\n",
      "  pages        = {536--549},\n",
      "  url          = {http://www.ncbi.nlm.nih.gov/pubmed/15498460},\n",
      "}\n",
      "\n",
      "@article{PMID:15489334,\n",
      "  author       = {Gerhard, D. S. and Wagner, L. and Feingold, E. A. and Shenmen, C. M. and Grouse, L. H. and Schuler, G. and Klein, S. L. and Old, S. and Rasooly, R. and Good, P. and Guyer, M. and Peck, A. M. and Derge, J. G. and Lipman, D. and Collins, F. S. and Jang, W. and Sherry, S. and Feolo, M. and Misquitta, L. and Lee, E. and Rotmistrovsky, K. and Greenhut, S. F. and Schaefer, C. F. and Buetow, K. and Bonner, T. I. and Haussler, D. and Kent, J. and Kiekhaus, M. and Furey, T. and Brent, M. and Prange, C. and Schreiber, K. and Shapiro, N. and Bhat, N. K. and Hopkins, R. F. and Hsie, F. and Driscoll, T. and Soares, M. B. and Casavant, T. L. and Scheetz, T. E. and Brown-stein, M. J. and Usdin, T. B. and Toshiyuki, S. and Carninci, P. and Piao, Y. and Dudekula, D. B. and Ko, M. S. and Kawakami, K. and Suzuki, Y. and Sugano, S. and Gruber, C. E. and Smith, M. R. and Simmons, B. and Moore, T. and Waterman, R. and Johnson, S. L. and Ruan, Y. and Wei, C. L. and Mathavan, S. and Gunaratne, P. H. and Wu, J. and Garcia, A. M. and Hulyk, S. W. and Fuh, E. and Yuan, Y. and Sneed, A. and Kowis, C. and Hodgson, A. and Muzny, D. M. and McPherson, J. and Gibbs, R. A. and Fahey, J. and Helton, E. and Ketteman, M. and Madan, A. and Rodrigues, S. and Sanchez, A. and Whiting, M. and Madari, A. and Young, A. C. and Wetherby, K. D. and Granite, S. J. and Kwong, P. N. and Brinkley, C. P. and Pearson, R. L. and Bouffard, G. G. and Blakesly, R. W. and Green, E. D. and Dickson, M. C. and Rodriguez, A. C. and Grimwood, J. and Schmutz, J. and Myers, R. M. and Butterfield, Y. S. and Griffith, M. and Griffith, O. L. and Krzywinski, M. I. and Liao, N. and Morin, R. and Palmquist, D. and Petrescu, A. S. and Skalska, U. and Smailus, D. E. and Stott, J. M. and Schnerch, A. and Schein, J. E. and Jones, S. J. and Holt, R. A. and Baross, A. and Marra, M. A. and Clifton, S. and Makowski, K. A. and Bosak, S. and Malek, J.},\n",
      "  title        = {The status, quality, and expansion of the NIH full-length cDNA project: the Mammalian Gene Collection (MGC).},\n",
      "  journal      = {Genome Res},\n",
      "  year         = {2004},\n",
      "  volume       = {14},\n",
      "  number       = {10B},\n",
      "  pages        = {2121--2127},\n",
      "  url          = {http://www.ncbi.nlm.nih.gov/pubmed/15489334},\n",
      "}\n",
      "\n",
      "@article{PMID:14702039,\n",
      "  author       = {Ota, T. and Suzuki, Y. and Nishikawa, T. and Otsuki, T. and Sugiyama, T. and Irie, R. and Wakamatsu, A. and Hayashi, K. and Sato, H. and Nagai, K. and Kimura, K. and Makita, H. and Sekine, M. and Obayashi, M. and Nishi, T. and Shibahara, T. and Tanaka, T. and Ishii, S. and Yamamoto, J. and Saito, K. and Kawai, Y. and Isono, Y. and Nakamura, Y. and Nagahari, K. and Murakami, K. and Yasuda, T. and Iwayanagi, T. and Wagatsuma, M. and Shiratori, A. and Sudo, H. and Hosoiri, T. and Kaku, Y. and Kodaira, H. and Kondo, H. and Sugawara, M. and Takahashi, M. and Kanda, K. and Yokoi, T. and Furuya, T. and Kikkawa, E. and Omura, Y. and Abe, K. and Kamihara, K. and Katsuta, N. and Sato, K. and Tanikawa, M. and Yamazaki, M. and Ninomiya, K. and Ishibashi, T. and Yamashita, H. and Murakawa, K. and Fujimori, K. and Tanai, H. and Kimata, M. and Watanabe, M. and Hiraoka, S. and Chiba, Y. and Ishida, S. and Ono, Y. and Takiguchi, S. and Watanabe, S. and Yosida, M. and Hotuta, T. and Kusano, J. and Kanehori, K. and Takahashi-Fujii, A. and Hara, H. and Tanase, T. O. and Nomura, Y. and Togiya, S. and Komai, F. and Hara, R. and Takeuchi, K. and Arita, M. and Imose, N. and Musashino, K. and Yuuki, H. and Oshima, A. and Sasaki, N. and Aotsuka, S. and Yoshikawa, Y. and Matsunawa, H. and Ichihara, T. and Shiohata, N. and Sano, S. and Moriya, S. and Momiyama, H. and Satoh, N. and Takami, S. and Terashima, Y. and Suzuki, O. and Nakagawa, S. and Senoh, A. and Mizoguchi, H. and Goto, Y. and Shimizu, F. and Wakebe, H. and Hishigaki, H. and Watanabe, T. and Sugiyama, A. and Takemoto, M. and Kawakami, B. and Yamazaki, M. and Watanabe, K. and Kumagai, A. and Itakura, S. and Fukuzumi, Y. and Fujimori, Y. and Komiyama, M. and Tashiro, H. and Tanigami, A. and Fujiwara, T. and Ono, T. and Yamada, K. and Fujii, Y. and Ozaki, K. and Hirao, M. and Ohmori, Y. and Kawabata, A. and Hikiji, T. and Kobatake, N. and Inagaki, H. and Ikema, Y. and Okamoto, S. and Okitani, R. and Kawakami, T. and Noguchi, S. and Itoh, T. and Shigeta, K. and Senba, T. and Matsumura, K. and Nakajima, Y. and Mizuno, T. and Morinaga, M. and Sasaki, M. and Togashi, T. and Oyama, M. and Hata, H. and Watanabe, M. and Komatsu, T. and Mizushima-Sugano, J. and Satoh, T. and Shirai, Y. and Takahashi, Y. and Nakagawa, K. and Okumura, K. and Nagase, T. and Nomura, N. and Kikuchi, H. and Masuho, Y. and Yamashita, R. and Nakai, K. and Yada, T. and Nakamura, Y. and Ohara, O. and Isogai, T. and Sugano, S.},\n",
      "  title        = {Complete sequencing and characterization of 21,243 full-length human cDNAs.},\n",
      "  journal      = {Nat Genet},\n",
      "  year         = {2004},\n",
      "  volume       = {36},\n",
      "  number       = {1},\n",
      "  pages        = {40--45},\n",
      "  url          = {http://www.ncbi.nlm.nih.gov/pubmed/14702039},\n",
      "}\n",
      "\n",
      "@article{PMID:14681893,\n",
      "  author       = {Swartz, B. E. and Li, S. and Bespalova, I. and Burmeister, M. and Dulaney, E. and Robinson, F. R. and Leigh, R. J.},\n",
      "  title        = {Pathogenesis of clinical signs in recessive ataxia with saccadic intrusions.},\n",
      "  journal      = {Ann Neurol},\n",
      "  year         = {2003},\n",
      "  volume       = {54},\n",
      "  number       = {6},\n",
      "  pages        = {824--828},\n",
      "  url          = {http://www.ncbi.nlm.nih.gov/pubmed/14681893},\n",
      "}\n",
      "\n",
      "@article{PMID:12477932,\n",
      "  author       = {Strausberg, R. L. and Feingold, E. A. and Grouse, L. H. and Derge, J. G. and Klausner, R. D. and Collins, F. S. and Wagner, L. and Shenmen, C. M. and Schuler, G. D. and Altschul, S. F. and Zeeberg, B. and Buetow, K. H. and Schaefer, C. F. and Bhat, N. K. and Hopkins, R. F. and Jordan, H. and Moore, T. and Max, S. I. and Wang, J. and Hsieh, F. and Diatchenko, L. and Marusina, K. and Farmer, A. A. and Rubin, G. M. and Hong, L. and Stapleton, M. and Soares, M. B. and Bonaldo, M. F. and Casavant, T. L. and Scheetz, T. E. and Brownstein, M. J. and Usdin, T. B. and Toshiyuki, S. and Carninci, P. and Prange, C. and Raha, S. S. and Loquellano, N. A. and Peters, G. J. and Abramson, R. D. and Mullahy, S. J. and Bosak, S. A. and McEwan, P. J. and McKernan, K. J. and Malek, J. A. and Gunaratne, P. H. and Richards, S. and Worley, K. C. and Hale, S. and Garcia, A. M. and Gay, L. J. and Hulyk, S. W. and Villalon, D. K. and Muzny, D. M. and Sodergren, E. J. and Lu, X. and Gibbs, R. A. and Fahey, J. and Helton, E. and Ketteman, M. and Madan, A. and Rodrigues, S. and Sanchez, A. and Whiting, M. and Madan, A. and Young, A. C. and Shevchenko, Y. and Bouffard, G. G. and Blakesley, R. W. and Touchman, J. W. and Green, E. D. and Dickson, M. C. and Rodriguez, A. C. and Grimwood, J. and Schmutz, J. and Myers, R. M. and Butterfield, Y. S. and Krzywinski, M. I. and Skalska, U. and Smailus, D. E. and Schnerch, A. and Schein, J. E. and Jones, S. J. and Marra, M. A.},\n",
      "  title        = {Generation and initial analysis of more than 15,000 full-length human and mouse cDNA sequences.},\n",
      "  journal      = {Proc Natl Acad Sci U S A},\n",
      "  year         = {2002},\n",
      "  volume       = {99},\n",
      "  number       = {26},\n",
      "  pages        = {16899--16903},\n",
      "  url          = {http://www.ncbi.nlm.nih.gov/pubmed/12477932},\n",
      "}\n",
      "\n",
      "@article{PMID:12421765,\n",
      "  author       = {Nakayama, M. and Kikuno, R. and Ohara, O.},\n",
      "  title        = {Protein-protein interactions between large proteins: two-hybrid screening using a functionally classified library composed of long cDNAs.},\n",
      "  journal      = {Genome Res},\n",
      "  year         = {2002},\n",
      "  volume       = {12},\n",
      "  number       = {11},\n",
      "  pages        = {1773--1784},\n",
      "  url          = {http://www.ncbi.nlm.nih.gov/pubmed/12421765},\n",
      "}\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "@article{PMID:11960835,\n",
      "  author       = {Swartz, B. E. and Burmeister, M. and Somers, J. T. and Rottach, K. G. and Bespalova, I. N. and Leigh, R. J.},\n",
      "  title        = {A form of inherited cerebellar ataxia with saccadic intrusions, increased saccadic speed, sensory neuropathy, and myoclonus.},\n",
      "  journal      = {Ann N Y Acad Sci},\n",
      "  year         = {2002},\n",
      "  volume       = {956},\n",
      "  pages        = {441--444},\n",
      "  url          = {http://www.ncbi.nlm.nih.gov/pubmed/11960835},\n",
      "}\n",
      "\n",
      "@article{PMID:9455484,\n",
      "  author       = {Seki, N. and Ohira, M. and Nagase, T. and Ishikawa, K. and Miyajima, N. and Nakajima, D. and Nomura, N. and Ohara, O.},\n",
      "  title        = {Characterization of cDNA clones in size-fractionated cDNA libraries from human brain.},\n",
      "  journal      = {DNA Res},\n",
      "  year         = {1997},\n",
      "  volume       = {4},\n",
      "  number       = {5},\n",
      "  pages        = {345--349},\n",
      "  url          = {http://www.ncbi.nlm.nih.gov/pubmed/9455484},\n",
      "}\n",
      "\n",
      "@article{PMID:9110174,\n",
      "  author       = {Yu, W. and Andersson, B. and Worley, K. C. and Muzny, D. M. and Ding, Y. and Liu, W. and Ricafrente, J. Y. and Wentland, M. A. and Lennon, G. and Gibbs, R. A.},\n",
      "  title        = {Large-scale concatenation cDNA sequencing.},\n",
      "  journal      = {Genome Res},\n",
      "  year         = {1997},\n",
      "  volume       = {7},\n",
      "  number       = {4},\n",
      "  pages        = {353--358},\n",
      "  url          = {http://www.ncbi.nlm.nih.gov/pubmed/9110174},\n",
      "}\n",
      "\n",
      "@article{PMID:8619474,\n",
      "  author       = {Andersson, B. and Wentland, M. A. and Ricafrente, J. Y. and Liu, W. and Gibbs, R. A.},\n",
      "  title        = {A \"double adaptor\" method for improved shotgun library construction.},\n",
      "  journal      = {Anal Biochem},\n",
      "  year         = {1996},\n",
      "  volume       = {236},\n",
      "  number       = {1},\n",
      "  pages        = {107--113},\n",
      "  url          = {http://www.ncbi.nlm.nih.gov/pubmed/8619474},\n",
      "}\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "[\"PMID- 30789691\\nSTAT- Publisher\\nDA  - 20190222\\nCTDT- 20190221\\nPB  - University of Washington, Seattle\\nDP  - 1993\\nTI  - VPS13D Movement Disorder\\nBTI - GeneReviews((R))\\nAB  - CLINICAL CHARACTERISTICS: VPS13D movement disorder is a hyperkinetic movement\\n      disorder (dystonia, chorea, and/or ataxia) of variable age of onset that can be\\n      associated with developmental delay. Onset ranges from birth to adulthood.\\n      Individuals can present in childhood with motor delays and gait instability.\\n      Cognitive impairment ranging from mild intellectual disability to developmental\\n      delay has been reported, and several individuals have normal cognitive function. \\n      Individuals have also presented as young adults with gait difficulties caused by \\n      spastic ataxia or ataxia. In addition to gait ataxia, affected individuals had\\n      limb ataxia, dysarthria, and eye movement abnormalities (macro-saccadic\\n      oscillations, nystagmus, and saccadic pursuit). Additional features reported in\\n      some individuals include peripheral neuropathy and/or seizures. The disorder\\n      progresses to spastic ataxia or generalized dystonia, which can lead to loss of\\n      independent ambulation. DIAGNOSIS: The diagnosis of VPS13D movement disorder is\\n      established in a proband by identification of biallelic pathogenic variants in\\n      VPS13D on molecular genetic testing. MANAGEMENT: Treatment of manifestations:\\n      Standard treatment for seizures; spasticity treatments include baclofen,\\n      tizanidine, benzodiazepines, dantrolene sodium, gabapentin, botulinum toxin\\n      injections, and intrathecal baclofen; treatment options for dystonia include\\n      trihexyphenidyl, botulinum toxin injections, benzodiazepines, baclofen, and\\n      levodopa. A multidisciplinary team including occupational and physical therapists\\n      and a physiatrist is important; supportive developmental therapies should be\\n      provided as needed. Surveillance: Monitor for urinary urgency, contractures, and \\n      seizures; evaluate fall risk and gait stability; monitor developmental progress\\n      and educational needs. GENETIC COUNSELING: VPS13D movement disorder is inherited \\n      in an autosomal recessive manner. At conception, each sib of an affected\\n      individual has a 25% chance of being affected, a 50% chance of being an\\n      asymptomatic carrier, and a 25% chance of being unaffected and not a carrier.\\n      Carrier testing for at-risk relatives and prenatal testing for pregnancies at\\n      increased risk are possible if the VPS13D pathogenic variants in the family are\\n      known.\\nCI  - Copyright (c) 1993-2019, University of Washington, Seattle. GeneReviews is a\\n      registered trademark of the University of Washington, Seattle. All rights\\n      reserved.\\nFED - Adam, Margaret P\\nED  - Adam MP\\nFED - Ardinger, Holly H\\nED  - Ardinger HH\\nFED - Pagon, Roberta A\\nED  - Pagon RA\\nFED - Wallace, Stephanie E\\nED  - Wallace SE\\nFED - Bean, Lora JH\\nED  - Bean LJH\\nFED - Stephens, Karen\\nED  - Stephens K\\nFED - Amemiya, Anne\\nED  - Amemiya A\\nFAU - Meijer, Inge A\\nAU  - Meijer IA\\nLA  - eng\\nGR  - HHSN276201400262U/NLM NIH HHS/United States\\nPT  - Review\\nPT  - Book Chapter\\nPL  - Seattle (WA)\\nEDAT- 2019/02/22 06:01\\nMHDA- 2019/02/22 06:01\\nCDAT- 2019/02/22 06:01\\nAID - NBK537720 [bookaccession]\", \"PMID- 29604224\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20190812\\nLR  - 20190812\\nIS  - 1531-8249 (Electronic)\\nIS  - 0364-5134 (Linking)\\nVI  - 83\\nIP  - 6\\nDP  - 2018 Jun\\nTI  - Mutations in VPS13D lead to a new recessive ataxia with spasticity and\\n      mitochondrial defects.\\nPG  - 1075-1088\\nLID - 10.1002/ana.25220 [doi]\\nAB  - OBJECTIVE: To identify novel causes of recessive ataxias, including\\n      spinocerebellar ataxia with saccadic intrusions, spastic ataxias, and spastic\\n      paraplegia. METHODS: In an international collaboration, we independently\\n      performed exome sequencing in 7 families with recessive ataxia and/or spastic\\n      paraplegia. To evaluate the role of VPS13D mutations, we evaluated a Drosophila\\n      knockout model and investigated mitochondrial function in patient-derived\\n      fibroblast cultures. RESULTS: Exome sequencing identified compound heterozygous\\n      mutations in VPS13D on chromosome 1p36 in all 7 families. This included a large\\n      family with 5 affected siblings with spinocerebellar ataxia with saccadic\\n      intrusions (SCASI), or spinocerebellar ataxia, recessive, type 4 (SCAR4). Linkage\\n      to chromosome 1p36 was found in this family with a logarithm of odds score of\\n      3.1. The phenotypic spectrum in our 12 patients was broad. Although most\\n      presented with ataxia, additional or predominant spasticity was present in 5\\n      patients. Disease onset ranged from infancy to 39 years, and symptoms were slowly\\n      progressive and included loss of independent ambulation in 5. All but 2 patients \\n      carried a loss-of-function (nonsense or splice site) mutation on one and a\\n      missense mutation on the other allele. Knockdown or removal of Vps13D in\\n      Drosophila neurons led to changes in mitochondrial morphology and impairment in\\n      mitochondrial distribution along axons. Patient fibroblasts showed altered\\n      morphology and functionality including reduced energy production. INTERPRETATION:\\n      Our study demonstrates that compound heterozygous mutations in VPS13D cause\\n      movement disorders along the ataxia-spasticity spectrum, making VPS13D the fourth\\n      VPS13 paralog involved in neurological disorders. Ann Neurol 2018.\\nCI  - (c) 2018 American Neurological Association.\\nFAU - Seong, Eunju\\nAU  - Seong E\\nAD  - Molecular and Behavioral Neuroscience Institute, University of Michigan, Ann\\n      Arbor, MI.\\nFAU - Insolera, Ryan\\nAU  - Insolera R\\nAD  - Department of Molecular, Cellular, and Developmental Biology, University of\\n      Michigan, Ann Arbor, MI.\\nFAU - Dulovic, Marija\\nAU  - Dulovic M\\nAD  - Institute of Neurogenetics, University of Lubeck, Lubeck, Germany.\\nFAU - Kamsteeg, Erik-Jan\\nAU  - Kamsteeg EJ\\nAD  - Department of Human Genetics, Radboud University Medical Centre, Nijmegen, the\\n      Netherlands.\\nFAU - Trinh, Joanne\\nAU  - Trinh J\\nAD  - Institute of Neurogenetics, University of Lubeck, Lubeck, Germany.\\nFAU - Bruggemann, Norbert\\nAU  - Bruggemann N\\nAD  - Department of Neurology, University of Lubeck, Lubeck, Germany.\\nFAU - Sandford, Erin\\nAU  - Sandford E\\nAD  - Molecular and Behavioral Neuroscience Institute, University of Michigan, Ann\\n      Arbor, MI.\\nFAU - Li, Sheng\\nAU  - Li S\\nAD  - Molecular and Behavioral Neuroscience Institute, University of Michigan, Ann\\n      Arbor, MI.\\nFAU - Ozel, Ayse Bilge\\nAU  - Ozel AB\\nAD  - Department of Human Genetics, University of Michigan, Ann Arbor, MI.\\nFAU - Li, Jun Z\\nAU  - Li JZ\\nAUID- ORCID: 0000-0001-6727-0812\\nAD  - Department of Human Genetics, University of Michigan, Ann Arbor, MI.\\nAD  - Department of Computational Medicine and Bioinformatics, University of Michigan, \\n      Ann Arbor, MI.\\nFAU - Jewett, Tamison\\nAU  - Jewett T\\nAD  - Department of Pediatrics, Section on Medical Genetics, Wake Forest School of\\n      Medicine, Winston-Salem, NC.\\nFAU - Kievit, Anneke J A\\nAU  - Kievit AJA\\nAD  - Department of Clinical Genetics, Erasmus MC, Rotterdam, the Netherlands.\\nFAU - Munchau, Alexander\\nAU  - Munchau A\\nAD  - Institute of Neurogenetics, University of Lubeck, Lubeck, Germany.\\nFAU - Shakkottai, Vikram\\nAU  - Shakkottai V\\nAD  - Departments of Neurology and Molecular and Integrative Physiology, University of \\n      Michigan, Ann Arbor, MI.\\nFAU - Klein, Christine\\nAU  - Klein C\\nAD  - Institute of Neurogenetics, University of Lubeck, Lubeck, Germany.\\nFAU - Collins, Catherine A\\nAU  - Collins CA\\nAUID- ORCID: 0000-0002-1608-6692\\nAD  - Department of Molecular, Cellular, and Developmental Biology, University of\\n      Michigan, Ann Arbor, MI.\\nFAU - Lohmann, Katja\\nAU  - Lohmann K\\nAD  - Institute of Neurogenetics, University of Lubeck, Lubeck, Germany.\\nFAU - van de Warrenburg, Bart P\\nAU  - van de Warrenburg BP\\nAD  - Department of Neurology, Donders Institute for Brain, Cognition, and Behavior,\\n      Radboud University Medical Centre, Nijmegen, the Netherlands.\\nFAU - Burmeister, Margit\\nAU  - Burmeister M\\nAUID- ORCID: 0000-0002-1914-2434\\nAD  - Molecular and Behavioral Neuroscience Institute, University of Michigan, Ann\\n      Arbor, MI.\\nAD  - Department of Human Genetics, University of Michigan, Ann Arbor, MI.\\nAD  - Department of Computational Medicine and Bioinformatics, University of Michigan, \\n      Ann Arbor, MI.\\nAD  - Department of Psychiatry, University of Michigan, Ann Arbor, MI.\\nLA  - eng\\nGR  - F31 NS056780/NS/NINDS NIH HHS/United States\\nGR  - F32 NS098611/NS/NINDS NIH HHS/United States\\nGR  - R01 NS069844/NS/NINDS NIH HHS/United States\\nGR  - R01 NS078560/NS/NINDS NIH HHS/United States\\nPT  - Journal Article\\nPT  - Research Support, N.I.H., Extramural\\nPT  - Research Support, Non-U.S. Gov't\\nDEP - 20180630\\nPL  - United States\\nTA  - Ann Neurol\\nJT  - Annals of neurology\\nJID - 7707449\\nRN  - 0 (Proteins)\\nRN  - 0 (VPS13D protein, human)\\nRN  - Spastic Ataxia\\nCIN - Ann Neurol. 2018 Jun;83(6):1072-1074. PMID: 29908061\\nMH  - Adult\\nMH  - Cerebellar Ataxia/genetics\\nMH  - Female\\nMH  - Genetic Linkage\\nMH  - Humans\\nMH  - Intellectual Disability/*genetics\\nMH  - Male\\nMH  - Middle Aged\\nMH  - Mitochondria/*genetics\\nMH  - Muscle Spasticity/*genetics\\nMH  - Mutation/*genetics\\nMH  - Optic Atrophy/*genetics\\nMH  - Pedigree\\nMH  - Proteins/*genetics\\nMH  - Spastic Paraplegia, Hereditary/genetics\\nMH  - Spinocerebellar Ataxias/*genetics\\nPMC - PMC6105379\\nMID - NIHMS955732\\nEDAT- 2018/04/01 06:00\\nMHDA- 2019/08/14 06:00\\nCRDT- 2018/04/01 06:00\\nPHST- 2017/11/01 00:00 [received]\\nPHST- 2018/03/11 00:00 [revised]\\nPHST- 2018/03/19 00:00 [accepted]\\nPHST- 2018/04/01 06:00 [pubmed]\\nPHST- 2019/08/14 06:00 [medline]\\nPHST- 2018/04/01 06:00 [entrez]\\nAID - 10.1002/ana.25220 [doi]\\nPST - ppublish\\nSO  - Ann Neurol. 2018 Jun;83(6):1075-1088. doi: 10.1002/ana.25220. Epub 2018 Jun 30.\", \"PMID- 29518281\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20190812\\nLR  - 20191114\\nIS  - 1531-8249 (Electronic)\\nIS  - 0364-5134 (Linking)\\nVI  - 83\\nIP  - 6\\nDP  - 2018 Jun\\nTI  - Recessive mutations in VPS13D cause childhood onset movement disorders.\\nPG  - 1089-1095\\nLID - 10.1002/ana.25204 [doi]\\nAB  - VPS13 protein family members VPS13A through VPS13C have been associated with\\n      various recessive movement disorders. We describe the first disease association\\n      of rare recessive VPS13D variants including frameshift, missense, and partial\\n      duplication mutations with a novel complex, hyperkinetic neurological disorder.\\n      The clinical features include developmental delay, a childhood onset movement\\n      disorder (chorea, dystonia, or tremor), and progressive spastic ataxia or\\n      paraparesis. Characteristic brain magnetic resonance imaging shows basal ganglia \\n      or diffuse white matter T2 hyperintensities as seen in Leigh syndrome and\\n      choreoacanthocytosis. Muscle biopsy in 1 case showed mitochondrial aggregates and\\n      lipidosis, suggesting mitochondrial dysfunction. These findings underline the\\n      importance of the VPS13 complex in neurological diseases and a possible role in\\n      mitochondrial function. Ann Neurol 2018;83:1089-1095.\\nCI  - (c) 2018 American Neurological Association.\\nFAU - Gauthier, Julie\\nAU  - Gauthier J\\nAD  - Molecular Diagnostic Laboratory and Division of Medical Genetics, Department of\\n      Pediatrics, Saint Justine University Hospital Center, Montreal, Canada.\\nFAU - Meijer, Inge A\\nAU  - Meijer IA\\nAD  - Department of Neuroscience, University of Montreal, Montreal, Canada.\\nFAU - Lessel, Davor\\nAU  - Lessel D\\nAUID- ORCID: 0000-0003-4496-244X\\nAD  - Institute of Human Genetics, University Medical Center Hamburg-Eppendorf,\\n      Hamburg, Germany.\\nFAU - Mencacci, Niccolo E\\nAU  - Mencacci NE\\nAD  - Department of Neurology, Northwestern University, Feinberg School of Medicine,\\n      Chicago, IL.\\nFAU - Krainc, Dimitri\\nAU  - Krainc D\\nAD  - Department of Neurology, Northwestern University, Feinberg School of Medicine,\\n      Chicago, IL.\\nFAU - Hempel, Maja\\nAU  - Hempel M\\nAD  - Institute of Human Genetics, University Medical Center Hamburg-Eppendorf,\\n      Hamburg, Germany.\\nFAU - Tsiakas, Konstantinos\\nAU  - Tsiakas K\\nAD  - Department of Pediatrics, University Medical Center Hamburg-Eppendorf, Hamburg,\\n      Germany.\\nFAU - Prokisch, Holger\\nAU  - Prokisch H\\nAD  - Institute of Human Genetics, Helmholtz Center Munich, Neuherberg, Germany.\\nAD  - Institute of Human Genetics, Technical University Munich, Munich, Germany.\\nFAU - Rossignol, Elsa\\nAU  - Rossignol E\\nAD  - Department of Neuroscience, University of Montreal, Montreal, Canada.\\nFAU - Helm, Margaret H\\nAU  - Helm MH\\nAD  - Maine Medical Partners Pediatric Specialty Care, Portland, ME.\\nFAU - Rodan, Lance H\\nAU  - Rodan LH\\nAD  - Department of Neurology, Boston Children's Hospital, Boston, MA.\\nFAU - Karamchandani, Jason\\nAU  - Karamchandani J\\nAD  - Department of Pathology, McGill University, Montreal Neurological Institute,\\n      Montreal, Canada.\\nFAU - Carecchio, Miryam\\nAU  - Carecchio M\\nAD  - Molecular Neurogenetics Unit, Institute for Research and Health Care (IRCCS)\\n      Foundation Carlo Besta Neurological Institute, Milan, Italy.\\nFAU - Lubbe, Steven J\\nAU  - Lubbe SJ\\nAD  - Department of Neurology, Northwestern University, Feinberg School of Medicine,\\n      Chicago, IL.\\nFAU - Telegrafi, Aida\\nAU  - Telegrafi A\\nAD  - GeneDx, Gaithersburg, MD.\\nFAU - Henderson, Lindsay B\\nAU  - Henderson LB\\nAD  - GeneDx, Gaithersburg, MD.\\nFAU - Lorenzo, Kerry\\nAU  - Lorenzo K\\nAD  - Kadlec Clinic Genetic Counseling, Richland, WA.\\nFAU - Wallace, Stephanie E\\nAU  - Wallace SE\\nAD  - Division of Genetic Medicine, Department of Pediatrics, Seattle Children's\\n      Hospital and University of Washington, Seattle, WA.\\nFAU - Glass, Ian A\\nAU  - Glass IA\\nAD  - Division of Genetic Medicine, Department of Pediatrics, Seattle Children's\\n      Hospital and University of Washington, Seattle, WA.\\nFAU - Hamdan, Fadi F\\nAU  - Hamdan FF\\nAD  - Molecular Diagnostic Laboratory and Division of Medical Genetics, Department of\\n      Pediatrics, Saint Justine University Hospital Center, Montreal, Canada.\\nFAU - Michaud, Jacques L\\nAU  - Michaud JL\\nAD  - Department of Pediatrics, Saint Justine University Hospital Center and University\\n      of Montreal, Montreal, Canada.\\nFAU - Rouleau, Guy A\\nAU  - Rouleau GA\\nAD  - Montreal Neurological Institute, Department of Neurology and Neurosurgery, McGill\\n      University, Montreal, Canada.\\nFAU - Campeau, Philippe M\\nAU  - Campeau PM\\nAUID- ORCID: 0000-0001-9713-7107\\nAD  - Department of Pediatrics, Saint Justine University Hospital Center and University\\n      of Montreal, Montreal, Canada.\\nLA  - eng\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov't\\nDEP - 20180410\\nPL  - United States\\nTA  - Ann Neurol\\nJT  - Annals of neurology\\nJID - 7707449\\nRN  - 0 (Proteins)\\nRN  - 0 (VPS13D protein, human)\\nRN  - Spastic Ataxia\\nCIN - Ann Neurol. 2018 Jun;83(6):1072-1074. PMID: 29908061\\nMH  - Basal Ganglia/pathology\\nMH  - Brain/pathology\\nMH  - Child\\nMH  - Humans\\nMH  - Intellectual Disability/*genetics\\nMH  - Leigh Disease/pathology\\nMH  - Magnetic Resonance Imaging/methods\\nMH  - Movement Disorders/*genetics\\nMH  - Muscle Spasticity/*genetics/pathology\\nMH  - Mutation/*genetics\\nMH  - Optic Atrophy/*genetics\\nMH  - Pedigree\\nMH  - Proteins/*genetics\\nMH  - Spinocerebellar Ataxias/*genetics\\nEDAT- 2018/03/09 06:00\\nMHDA- 2019/08/14 06:00\\nCRDT- 2018/03/09 06:00\\nPHST- 2017/11/10 00:00 [received]\\nPHST- 2018/03/06 00:00 [revised]\\nPHST- 2018/03/07 00:00 [accepted]\\nPHST- 2018/03/09 06:00 [pubmed]\\nPHST- 2019/08/14 06:00 [medline]\\nPHST- 2018/03/09 06:00 [entrez]\\nAID - 10.1002/ana.25204 [doi]\\nPST - ppublish\\nSO  - Ann Neurol. 2018 Jun;83(6):1089-1095. doi: 10.1002/ana.25204. Epub 2018 Apr 10.\", \"PMID- 29307555\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20190514\\nLR  - 20190517\\nIS  - 1879-0445 (Electronic)\\nIS  - 0960-9822 (Linking)\\nVI  - 28\\nIP  - 2\\nDP  - 2018 Jan 22\\nTI  - Vps13D Encodes a Ubiquitin-Binding Protein that Is Required for the Regulation of\\n      Mitochondrial Size and Clearance.\\nPG  - 287-295.e6\\nLID - S0960-9822(17)31589-0 [pii]\\nLID - 10.1016/j.cub.2017.11.064 [doi]\\nAB  - The clearance of mitochondria by autophagy, mitophagy, is important for cell and \\n      organism health [1], and known to be regulated by ubiquitin. During Drosophila\\n      intestine development, cells undergo a dramatic reduction in cell size and\\n      clearance of mitochondria that depends on autophagy, the E1 ubiquitin-activating \\n      enzyme Uba1, and ubiquitin [2]. Here we screen a collection of putative\\n      ubiquitin-binding domain-encoding genes for cell size reduction and autophagy\\n      phenotypes. We identify the endosomal sorting complex required for transport\\n      (ESCRT) components TSG101 and Vps36, as well as the novel gene Vps13D. Vps13D is \\n      an essential gene that is necessary for autophagy, mitochondrial size, and\\n      mitochondrial clearance in Drosophila. Interestingly, a similar mitochondrial\\n      phenotype is observed in VPS13D mutant human cells. The ubiquitin-associated\\n      (UBA) domain of Vps13D binds K63 ubiquitin chains, and mutants lacking the UBA\\n      domain have defects in mitochondrial size and clearance and exhibit\\n      semi-lethality, highlighting the importance of Vps13D ubiquitin binding in both\\n      mitochondrial health and development. VPS13D mutant cells possess phosphorylated \\n      DRP1 and mitochondrial fission factor (MFF) as well as DRP1 association with\\n      mitochondria, suggesting that VPS13D functions downstream of these known\\n      regulators of mitochondrial fission. In addition, the large Vps13D mitochondrial \\n      and cell size phenotypes are suppressed by decreased mitochondrial fusion gene\\n      function. Thus, these results provide a previously unknown link between\\n      ubiquitin, mitochondrial size regulation, and autophagy.\\nCI  - Copyright (c) 2017 Elsevier Ltd. All rights reserved.\\nFAU - Anding, Allyson L\\nAU  - Anding AL\\nAD  - Department of Molecular, Cell and Cancer Biology, University of Massachusetts\\n      Medical School, Worcester, MA 01605, USA.\\nFAU - Wang, Chunxin\\nAU  - Wang C\\nAD  - Biochemistry Section, Surgical Neurology Branch, National Institute of\\n      Neurological Disorders and Stroke, NIH, Bethesda, MD 20892, USA.\\nFAU - Chang, Tsun-Kai\\nAU  - Chang TK\\nAD  - Department of Molecular, Cell and Cancer Biology, University of Massachusetts\\n      Medical School, Worcester, MA 01605, USA.\\nFAU - Sliter, Danielle A\\nAU  - Sliter DA\\nAD  - Biochemistry Section, Surgical Neurology Branch, National Institute of\\n      Neurological Disorders and Stroke, NIH, Bethesda, MD 20892, USA.\\nFAU - Powers, Christine M\\nAU  - Powers CM\\nAD  - Department of Molecular, Cell and Cancer Biology, University of Massachusetts\\n      Medical School, Worcester, MA 01605, USA.\\nFAU - Hofmann, Kay\\nAU  - Hofmann K\\nAD  - Institute for Genetics, University of Cologne, 50674 Cologne, Germany.\\nFAU - Youle, Richard J\\nAU  - Youle RJ\\nAD  - Biochemistry Section, Surgical Neurology Branch, National Institute of\\n      Neurological Disorders and Stroke, NIH, Bethesda, MD 20892, USA.\\nFAU - Baehrecke, Eric H\\nAU  - Baehrecke EH\\nAD  - Department of Molecular, Cell and Cancer Biology, University of Massachusetts\\n      Medical School, Worcester, MA 01605, USA. Electronic address:\\n      eric.baehrecke@umassmed.edu.\\nLA  - eng\\nGR  - R01 GM111658/GM/NIGMS NIH HHS/United States\\nGR  - S10 RR027897/RR/NCRR NIH HHS/United States\\nPT  - Journal Article\\nPT  - Research Support, N.I.H., Extramural\\nPT  - Research Support, N.I.H., Intramural\\nDEP - 20180104\\nPL  - England\\nTA  - Curr Biol\\nJT  - Current biology : CB\\nJID - 9107782\\nRN  - 0 (Carrier Proteins)\\nRN  - 0 (Drosophila Proteins)\\nRN  - 0 (Intracellular Signaling Peptides and Proteins)\\nRN  - 0 (Ubiquitin)\\nRN  - 0 (Vps13D protein, Drosophila)\\nSB  - IM\\nCIN - Curr Biol. 2018 Jan 22;28(2):R66-R68. PMID: 29374445\\nMH  - Animals\\nMH  - Carrier Proteins/genetics/metabolism\\nMH  - Drosophila Proteins/*genetics/metabolism\\nMH  - Drosophila melanogaster/genetics/*physiology\\nMH  - Intracellular Signaling Peptides and Proteins/*genetics/metabolism\\nMH  - Mitochondria/physiology\\nMH  - Mitochondrial Degradation/*genetics\\nMH  - Mitochondrial Size/*genetics\\nMH  - Ubiquitin/metabolism\\nMH  - Ubiquitination\\nPMC - PMC5787036\\nMID - NIHMS925912\\nOTO - NOTNLM\\nOT  - *Drosophila\\nOT  - *Vps13D\\nOT  - *autophagy\\nOT  - *fission\\nOT  - *mitochondria\\nOT  - *mitophagy\\nOT  - *ubiquitin\\nEDAT- 2018/01/09 06:00\\nMHDA- 2019/05/15 06:00\\nCRDT- 2018/01/09 06:00\\nPHST- 2017/10/30 00:00 [received]\\nPHST- 2017/11/21 00:00 [revised]\\nPHST- 2017/11/28 00:00 [accepted]\\nPHST- 2018/01/09 06:00 [pubmed]\\nPHST- 2019/05/15 06:00 [medline]\\nPHST- 2018/01/09 06:00 [entrez]\\nAID - S0960-9822(17)31589-0 [pii]\\nAID - 10.1016/j.cub.2017.11.064 [doi]\\nPST - ppublish\\nSO  - Curr Biol. 2018 Jan 22;28(2):287-295.e6. doi: 10.1016/j.cub.2017.11.064. Epub\\n      2018 Jan 4.\", \"PMID- 25896417\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20160620\\nLR  - 20150828\\nIS  - 1662-8128 (Electronic)\\nIS  - 1662-811X (Linking)\\nVI  - 7\\nIP  - 5\\nDP  - 2015\\nTI  - VPS13D Gene Variant Is Associated with Altered IL-6 Production and Mortality in\\n      Septic Shock.\\nPG  - 545-53\\nLID - 10.1159/000381265 [doi]\\nAB  - BACKGROUND: Genetic variations contribute to septic shock mortality. To discover \\n      a novel locus, we performed in vitro genome-wide association studies (GWAS) and\\n      further tested the result in a cohort of septic shock patients. METHODS: Two in\\n      vitro GWAS using a quantitative trait locus analysis of stimulated IL-6\\n      production in lymphoblastoid cells from 60 individuals of European ancestry were \\n      performed. VPS13D rs6685273 was genotyped in European ancestry patients (n =\\n      498). The VPS13D gene was silenced in vitro. RESULTS: Two GWAS using\\n      lymphoblastoid cells identified the locus of VPS13D rs6685273 that was\\n      significant in the same direction in both GWAS. The VPS13D rs6685273 C allele was\\n      associated with increased IL-6 production. Patients with septic shock who had the\\n      VPS13D rs6685273 CC genotype had an increased 28-day mortality (p = 0.023) and\\n      more organ failure (p < 0.05) compared to the CT/TT genotypes. VPS13D in vitro\\n      gene silencing in the HeLa cell line increased IL-6 production. Furthermore, the \\n      rs6685273 genotype was associated with differential VPS13D splice variant\\n      expression. CONCLUSIONS: The VPS13D rs6685273 C allele was associated with\\n      increased IL-6 production in vitro. The patients with the VPS13D rs6685273 CC\\n      genotype had increased 28-day mortality and increased organ failure. VPS13D\\n      appears to regulate IL-6 production.\\nCI  - (c) 2015 S. Karger AG, Basel.\\nFAU - Nakada, Taka-aki\\nAU  - Nakada TA\\nAD  - Critical Care Research Laboratories, Centre for Heart Lung Innovation, St. Paul's\\n      Hospital, University of British Columbia, Vancouver, B.C., Canada.\\nFAU - Boyd, John H\\nAU  - Boyd JH\\nFAU - Russell, James A\\nAU  - Russell JA\\nFAU - Aguirre-Hernandez, Rosalia\\nAU  - Aguirre-Hernandez R\\nFAU - Wilkinson, Mark D\\nAU  - Wilkinson MD\\nFAU - Thair, Simone A\\nAU  - Thair SA\\nFAU - Nakada, Emiri\\nAU  - Nakada E\\nFAU - McConechy, Melissa K\\nAU  - McConechy MK\\nFAU - Fjell, Christopher D\\nAU  - Fjell CD\\nFAU - Walley, Keith R\\nAU  - Walley KR\\nLA  - eng\\nGR  - Canadian Institutes of Health Research/Canada\\nPT  - Journal Article\\nPT  - Multicenter Study\\nPT  - Randomized Controlled Trial\\nPT  - Research Support, Non-U.S. Gov't\\nDEP - 20150417\\nPL  - Switzerland\\nTA  - J Innate Immun\\nJT  - Journal of innate immunity\\nJID - 101469471\\nRN  - 0 (Interleukin-6)\\nRN  - 0 (Proteins)\\nRN  - 0 (VPS13D protein, human)\\nSB  - IM\\nMH  - Aged\\nMH  - Cohort Studies\\nMH  - European Continental Ancestry Group/genetics\\nMH  - Female\\nMH  - Gene Frequency\\nMH  - Genetic Predisposition to Disease\\nMH  - Genome-Wide Association Study\\nMH  - Genotype\\nMH  - HeLa Cells\\nMH  - Humans\\nMH  - Interleukin-6/genetics/*metabolism\\nMH  - Lymphocyte Activation/genetics\\nMH  - Lymphocytes/*physiology\\nMH  - Male\\nMH  - Middle Aged\\nMH  - Polymorphism, Single Nucleotide\\nMH  - Proteins/*genetics\\nMH  - Shock, Septic/*genetics/immunology/*mortality\\nMH  - Spain\\nMH  - Survival Analysis\\nEDAT- 2015/04/22 06:00\\nMHDA- 2016/06/21 06:00\\nCRDT- 2015/04/22 06:00\\nPHST- 2014/12/17 00:00 [received]\\nPHST- 2015/02/26 00:00 [accepted]\\nPHST- 2015/04/22 06:00 [entrez]\\nPHST- 2015/04/22 06:00 [pubmed]\\nPHST- 2016/06/21 06:00 [medline]\\nAID - 000381265 [pii]\\nAID - 10.1159/000381265 [doi]\\nPST - ppublish\\nSO  - J Innate Immun. 2015;7(5):545-53. doi: 10.1159/000381265. Epub 2015 Apr 17.\", \"PMID- 20610895\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20110308\\nLR  - 20190608\\nIS  - 1880-3873 (Electronic)\\nIS  - 1340-3478 (Linking)\\nVI  - 17\\nIP  - 10\\nDP  - 2010 Oct 27\\nTI  - Identification of evidence suggestive of an association with peripheral arterial \\n      disease at the OSBPL10 locus by genome-wide investigation in the Japanese\\n      population.\\nPG  - 1054-62\\nAB  - AIM: Peripheral arterial disease (PAD) is a common cause of cardiovascular\\n      morbidity and an independent predictor of cardiovascular mortality. However,\\n      little is known about the genetic basis of PAD. To elucidate this, we performed a\\n      two-staged genome-wide association study in Japanese individuals. METHODS: We\\n      initially tested 222,285 single-nucleotide polymorphisms (SNPs). After the first \\n      screening in a panel of 195 PAD cases and 1,358 controls, 2,696 SNPs (1.2%) were \\n      further genotyped in the second screening using another panel of 699 PAD cases\\n      and 1540 controls. In both screenings, controls were subjects affected with some \\n      diseases other than PAD. RESULTS: When analyzed in the combined panel, the\\n      strongest signal of PAD association was observed at rs1902341 in the intron of\\n      OSBPL10 (p=4.7E-7 for trend test; OR=1.31, 95% CI 1.18-1.46). Also, PAD was\\n      modestly associated at several other loci such as rs2554503 in CSMD1 (p=5.7E-5;\\n      OR=1.32, 95% CI 1.15-1.51) or rs235243 in VSP13D (p=0.04; OR=1.18, 95% CI\\n      1.01-1.37). CONCLUSION: Our genome-wide exploration identified suggestive\\n      evidence of PAD association at the OSBPL10 locus. Because the association has not\\n      reached a genome-wide significant level, further replication study is warranted\\n      for verification in the Japanese population.\\nFAU - Koriyama, Hiroshi\\nAU  - Koriyama H\\nAD  - Department of Geriatric Medicine and Nephrology, Osaka University Graduate School\\n      of Medicine, 2-2 Yamada-oka, Suita, Japan.\\nFAU - Nakagami, Hironori\\nAU  - Nakagami H\\nFAU - Katsuya, Tomohiro\\nAU  - Katsuya T\\nFAU - Sugimoto, Ken\\nAU  - Sugimoto K\\nFAU - Yamashita, Hidetoshi\\nAU  - Yamashita H\\nFAU - Takami, Yoichi\\nAU  - Takami Y\\nFAU - Maeda, Shiro\\nAU  - Maeda S\\nFAU - Kubo, Michiaki\\nAU  - Kubo M\\nFAU - Takahashi, Atsushi\\nAU  - Takahashi A\\nFAU - Nakamura, Yusuke\\nAU  - Nakamura Y\\nFAU - Ogihara, Toshio\\nAU  - Ogihara T\\nFAU - Rakugi, Hiromi\\nAU  - Rakugi H\\nFAU - Kaneda, Yasufumi\\nAU  - Kaneda Y\\nFAU - Morishita, Ryuichi\\nAU  - Morishita R\\nLA  - eng\\nPT  - Comparative Study\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov't\\nDEP - 20100702\\nPL  - Japan\\nTA  - J Atheroscler Thromb\\nJT  - Journal of atherosclerosis and thrombosis\\nJID - 9506298\\nRN  - 0 (Receptors, Steroid)\\nRN  - 0 (oxysterol binding protein)\\nSB  - IM\\nMH  - Aged\\nMH  - Asian Continental Ancestry Group/*genetics\\nMH  - Case-Control Studies\\nMH  - Female\\nMH  - Genetic Predisposition to Disease\\nMH  - Genome, Human\\nMH  - *Genome-Wide Association Study\\nMH  - Genotype\\nMH  - Humans\\nMH  - Japan/epidemiology\\nMH  - Linkage Disequilibrium\\nMH  - Male\\nMH  - Middle Aged\\nMH  - Peripheral Vascular Diseases/epidemiology/*genetics\\nMH  - Polymorphism, Single Nucleotide/*genetics\\nMH  - Receptors, Steroid/*genetics\\nEDAT- 2010/07/09 06:00\\nMHDA- 2011/03/09 06:00\\nCRDT- 2010/07/09 06:00\\nPHST- 2010/07/09 06:00 [entrez]\\nPHST- 2010/07/09 06:00 [pubmed]\\nPHST- 2011/03/09 06:00 [medline]\\nAID - JST.JSTAGE/jat/4291 [pii]\\nAID - 10.5551/jat.4291 [doi]\\nPST - ppublish\\nSO  - J Atheroscler Thromb. 2010 Oct 27;17(10):1054-62. doi: 10.5551/jat.4291. Epub\\n      2010 Jul 2.\", \"PMID- 20301317\\nSTAT- Publisher\\nDA  - 20100320\\nDRDT- 20190725\\nCTDT- 19981028\\nPB  - University of Washington, Seattle\\nDP  - 1993\\nTI  - Hereditary Ataxia Overview\\nBTI - GeneReviews((R))\\nAB  - CLINICAL CHARACTERISTICS: The hereditary ataxias are a group of genetic disorders\\n      characterized by slowly progressive incoordination of gait and often associated\\n      with poor coordination of hands, speech, and eye movements. Frequently, atrophy\\n      of the cerebellum occurs. In this GeneReview the hereditary ataxias are\\n      categorized by mode of inheritance and gene (or chromosome locus) in which\\n      pathogenic variants occur. DIAGNOSIS/TESTING: Inherited (genetic) forms of ataxia\\n      must be distinguished from the many acquired (non-genetic) causes of ataxia. The \\n      genetic forms of ataxia are diagnosed by family history, physical examination,\\n      neuroimaging, and molecular genetic testing. GENETIC COUNSELING: The hereditary\\n      ataxias can be inherited in an autosomal dominant, autosomal recessive, or\\n      X-linked manner. Genetic counseling and risk assessment depend on determination\\n      of the specific cause of an inherited ataxia in an individual. MANAGEMENT:\\n      Treatment of manifestations: Canes, walkers, and wheelchairs for gait ataxia; use\\n      of special devices to assist with handwriting, buttoning, and use of eating\\n      utensils; speech therapy and/or computer-based devices for those with dysarthria \\n      and severe speech deficits. Prevention of primary manifestations: With the\\n      exception of vitamin E therapy for ataxia with vitamin E deficiency (AVED), no\\n      specific treatments exist for hereditary ataxia.\\nCI  - Copyright (c) 1993-2019, University of Washington, Seattle. GeneReviews is a\\n      registered trademark of the University of Washington, Seattle. All rights\\n      reserved.\\nFED - Adam, Margaret P\\nED  - Adam MP\\nFED - Ardinger, Holly H\\nED  - Ardinger HH\\nFED - Pagon, Roberta A\\nED  - Pagon RA\\nFED - Wallace, Stephanie E\\nED  - Wallace SE\\nFED - Bean, Lora JH\\nED  - Bean LJH\\nFED - Stephens, Karen\\nED  - Stephens K\\nFED - Amemiya, Anne\\nED  - Amemiya A\\nFAU - Bird, Thomas D\\nAU  - Bird TD\\nLA  - eng\\nGR  - HHSN276201400262U/NLM NIH HHS/United States\\nPT  - Review\\nPT  - Book Chapter\\nPL  - Seattle (WA)\\nOTO - NLM\\nOT  - Aconitate hydratase, mitochondrial\\nOT  - AFG3-like protein 2\\nOT  - Alpha-tocopherol transfer protein\\nOT  - Aprataxin\\nOT  - Ataxin-1\\nOT  - Ataxin-10\\nOT  - Ataxin-2\\nOT  - Ataxin-3\\nOT  - Ataxin-7\\nOT  - Ataxin-8\\nOT  - Atrophin-1\\nOT  - Atypical kinase COQ8A, mitochondrial\\nOT  - Bifunctional polynucleotide phosphatase/kinase\\nOT  - Calpain-1 catalytic subunit\\nOT  - Ceroid-lipofuscinosis neuronal protein 5\\nOT  - CWF19-like protein 1\\nOT  - Decaprenyl-diphosphate synthase subunit 1\\nOT  - Decaprenyl-diphosphate synthase subunit 2\\nOT  - DNA-directed RNA polymerase III subunit RPC1\\nOT  - DNA-directed RNA polymerase III subunit RPC2\\nOT  - E3 ubiquitin-protein ligase CHIP\\nOT  - E3 ubiquitin-protein ligase RNF216\\nOT  - Elongation factor 2\\nOT  - Elongation factor Ts, mitochondrial\\nOT  - Elongation of very long chain fatty acids protein 4\\nOT  - Elongation of very long chain fatty acids protein 5\\nOT  - Excitatory amino acid transporter 1\\nOT  - Fibroblast growth factor 12\\nOT  - Fibroblast growth factor 14\\nOT  - Frataxin, mitochondrial\\nOT  - Ganglioside-induced differentiation-associated protein 2\\nOT  - Glutamate receptor ionotropic, delta-2\\nOT  - Inositol 1,4,5-trisphosphate receptor type 1\\nOT  - Laminin subunit alpha-1\\nOT  - Metabotropic glutamate receptor 1\\nOT  - Microtubule cross-linking factor 1\\nOT  - Neprilysin\\nOT  - Neuropathy target esterase\\nOT  - Nucleolar protein 56\\nOT  - Nucleotide exchange factor SIL1\\nOT  - Oligophrenin-1\\nOT  - Para-hydroxybenzoate--polyprenyltransferase, mitochondrial\\nOT  - Paraplegin\\nOT  - Peroxisomal targeting signal 2 receptor\\nOT  - Phospholipase D3\\nOT  - Phytanoyl-CoA dioxygenase, peroxisomal\\nOT  - Plasma membrane calcium-transporting ATPase 3\\nOT  - Potassium voltage-gated channel subfamily A member 1\\nOT  - Potassium voltage-gated channel subfamily C member 3\\nOT  - Potassium voltage-gated channel subfamily D member 3\\nOT  - Probable helicase senataxin\\nOT  - Proenkephalin-B\\nOT  - Protein BEAN1\\nOT  - Protein Daple\\nOT  - Protein kinase C gamma type\\nOT  - Protein-glutamine gamma-glutamyltransferase 6\\nOT  - Protocadherin Fat 2\\nOT  - Protocadherin-12\\nOT  - Pumilio homolog 1\\nOT  - Replication factor C subunit 1\\nOT  - Sacsin\\nOT  - Sequestosome-1\\nOT  - Serine/threonine-protein phosphatase 2A 55 kDa regulatory subunit B beta isoform\\nOT  - Serine-protein kinase ATM\\nOT  - Short transient receptor potential channel 3\\nOT  - Sodium/hydrogen exchanger 1\\nOT  - Sodium/hydrogen exchanger 6\\nOT  - Sodium/potassium-transporting ATPase subunit alpha-3\\nOT  - Solute carrier family 25 member 46\\nOT  - Solute carrier family 52, riboflavin transporter, member 2\\nOT  - Sorting nexin-14\\nOT  - Sorting nexin-4\\nOT  - Spectrin beta chain, non-erythrocytic 2\\nOT  - Sterol 26-hydroxylase, mitochondrial\\nOT  - TATA-box-binding protein\\nOT  - Tau-tubulin kinase 2\\nOT  - Transmembrane protein 240\\nOT  - Twinkle protein, mitochondrial\\nOT  - Tyrosyl-DNA phosphodiesterase 2\\nOT  - Ubiquinone biosynthesis protein COQ9, mitochondrial\\nOT  - Vacuolar protein sorting-associated protein 13D\\nOT  - Vesicle-associated membrane protein 1\\nOT  - Voltage-dependent L-type calcium channel subunit beta-4\\nOT  - Voltage-dependent P/Q-type calcium channel subunit alpha-1A\\nOT  - von Willebrand factor A domain-containing protein 3B\\nOT  - WD repeat-containing protein 73\\nOT  - Wolframin\\nOT  - ACO2\\nOT  - AFG3L2\\nOT  - APTX\\nOT  - ATM\\nOT  - ATN1\\nOT  - ATP1A3\\nOT  - ATP2B3\\nOT  - ATXN1\\nOT  - ATXN10\\nOT  - ATXN2\\nOT  - ATXN3\\nOT  - ATXN7\\nOT  - ATXN8\\nOT  - ATXN8OS\\nOT  - BEAN1\\nOT  - CACNA1A\\nOT  - CACNB4\\nOT  - CAPN1\\nOT  - CCDC88C\\nOT  - CLN5\\nOT  - COQ2\\nOT  - COQ8A\\nOT  - COQ9\\nOT  - CWF19L1\\nOT  - CYP27A1\\nOT  - EEF2\\nOT  - ELOVL4\\nOT  - ELOVL5\\nOT  - FAT2\\nOT  - FGF12\\nOT  - FGF14\\nOT  - FXN\\nOT  - GDAP2\\nOT  - GRID2\\nOT  - GRM1\\nOT  - ITPR1\\nOT  - KCNA1\\nOT  - KCNC3\\nOT  - KCND3\\nOT  - LAMA1\\nOT  - MME\\nOT  - MTCL1\\nOT  - NOP56\\nOT  - OPHN1\\nOT  - PCDH12\\nOT  - PDSS1\\nOT  - PDSS2\\nOT  - PDYN\\nOT  - PEX7\\nOT  - PHYH\\nOT  - PLD3\\nOT  - PNKP\\nOT  - PNPLA6\\nOT  - POLR3A\\nOT  - POLR3B\\nOT  - PPP2R2B\\nOT  - PRKCG\\nOT  - PUM1\\nOT  - RFC1\\nOT  - RNF216\\nOT  - SACS\\nOT  - SETX\\nOT  - SIL1\\nOT  - SLC1A3\\nOT  - SLC25A46\\nOT  - SLC52A2\\nOT  - SLC9A1\\nOT  - SLC9A6\\nOT  - SNX14\\nOT  - SNX4\\nOT  - SPG7\\nOT  - SPTBN2\\nOT  - SQSTM1\\nOT  - STUB1\\nOT  - TBP\\nOT  - TDP2\\nOT  - TGM6\\nOT  - TMEM240\\nOT  - TRPC3\\nOT  - TSFM\\nOT  - TTBK2\\nOT  - TTPA\\nOT  - TWNK\\nOT  - VAMP1\\nOT  - VPS13D\\nOT  - VWA3B\\nOT  - WDR73\\nOT  - WFS1\\nOT  - Hereditary Ataxia\\nOT  - Overview\\nEDAT- 2019/07/25 06:00\\nMHDA- 2019/07/25 06:00\\nCDAT- 2010/03/20 06:00\\nAID - NBK1138 [bookaccession]\", \"PMID- 19056867\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20090226\\nLR  - 20191003\\nIS  - 1533-3450 (Electronic)\\nIS  - 1046-6673 (Linking)\\nVI  - 20\\nIP  - 2\\nDP  - 2009 Feb\\nTI  - Large-scale proteomics and phosphoproteomics of urinary exosomes.\\nPG  - 363-79\\nLID - 10.1681/ASN.2008040406 [doi]\\nAB  - Normal human urine contains large numbers of exosomes, which are 40- to 100-nm\\n      vesicles that originate as the internal vesicles in multivesicular bodies from\\n      every renal epithelial cell type facing the urinary space. Here, we used LC-MS/MS\\n      to profile the proteome of human urinary exosomes. Overall, the analysis\\n      identified 1132 proteins unambiguously, including 177 that are represented on the\\n      Online Mendelian Inheritance in Man database of disease-related genes, suggesting\\n      that exosome analysis is a potential approach to discover urinary biomarkers. We \\n      extended the proteomic analysis to phosphoproteomic profiling using neutral loss \\n      scanning, and this yielded multiple novel phosphorylation sites, including\\n      serine-811 in the thiazide-sensitive Na-Cl co-transporter, NCC. To demonstrate\\n      the potential use of exosome analysis to identify a genetic renal disease, we\\n      carried out immunoblotting of exosomes from urine samples of patients with a\\n      clinical diagnosis of Bartter syndrome type I, showing an absence of the\\n      sodium-potassium-chloride co-transporter 2, NKCC2. The proteomic data are\\n      publicly accessible at http://dir.nhlbi.nih.gov/papers/lkem/exosome/.\\nFAU - Gonzales, Patricia A\\nAU  - Gonzales PA\\nAD  - Laboratory of Kidney and Electrolyte Metabolism, National Heart, Lung, and Blood \\n      Institute, National Institutes of Health, Bethesda, MD 20892, USA.\\nFAU - Pisitkun, Trairak\\nAU  - Pisitkun T\\nFAU - Hoffert, Jason D\\nAU  - Hoffert JD\\nFAU - Tchapyjnikov, Dmitry\\nAU  - Tchapyjnikov D\\nFAU - Star, Robert A\\nAU  - Star RA\\nFAU - Kleta, Robert\\nAU  - Kleta R\\nFAU - Wang, Nam Sun\\nAU  - Wang NS\\nFAU - Knepper, Mark A\\nAU  - Knepper MA\\nLA  - eng\\nGR  - Z01 HL001285-21/Intramural NIH HHS/United States\\nGR  - Z99 HL999999/Intramural NIH HHS/United States\\nPT  - Journal Article\\nDEP - 20081203\\nPL  - United States\\nTA  - J Am Soc Nephrol\\nJT  - Journal of the American Society of Nephrology : JASN\\nJID - 9013836\\nRN  - 0 (Phosphoproteins)\\nRN  - 0 (Proteome)\\nRN  - 0 (Thiazides)\\nRN  - EC 3.6.1.- (Adenosine Triphosphatases)\\nSB  - IM\\nMH  - Adenosine Triphosphatases/chemistry\\nMH  - Adult\\nMH  - Bartter Syndrome/urine\\nMH  - Chromatography, Liquid/methods\\nMH  - Exosomes/metabolism\\nMH  - Female\\nMH  - Humans\\nMH  - Male\\nMH  - Mass Spectrometry/methods\\nMH  - Phosphoproteins/chemistry\\nMH  - Phosphorylation\\nMH  - Proteome\\nMH  - Proteomics/*methods\\nMH  - Thiazides/chemistry\\nMH  - *Urine\\nPMC - PMC2637050\\nEDAT- 2008/12/06 09:00\\nMHDA- 2009/02/27 09:00\\nCRDT- 2008/12/06 09:00\\nPHST- 2008/12/06 09:00 [pubmed]\\nPHST- 2009/02/27 09:00 [medline]\\nPHST- 2008/12/06 09:00 [entrez]\\nAID - ASN.2008040406 [pii]\\nAID - 10.1681/ASN.2008040406 [doi]\\nPST - ppublish\\nSO  - J Am Soc Nephrol. 2009 Feb;20(2):363-79. doi: 10.1681/ASN.2008040406. Epub 2008\\n      Dec 3.\", \"PMID- 15498460\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20050124\\nLR  - 20070516\\nIS  - 0888-7543 (Print)\\nIS  - 0888-7543 (Linking)\\nVI  - 84\\nIP  - 3\\nDP  - 2004 Sep\\nTI  - Analysis of the human VPS13 gene family.\\nPG  - 536-49\\nAB  - The gene mutated in chorea-acanthocytosis (CHAC; approved gene symbol VPS13A)\\n      encodes chorein, a protein similar to yeast Vps13p. We detected several similar\\n      putative human proteins by BLAST analysis of chorein. We characterized the\\n      structure of three new genes encoding these CHAC-similar proteins, located on\\n      chromosomes 1p36, 8q22, and 15q21. The most similar gene in yeast to all four\\n      human genes is Vps13, and therefore the human genes were named VPS13A (CHAC,\\n      9q21), VPS13B (8q22), VPS13C (15q21), and VPS13D (1p36). VPS13B has recently been\\n      reported as COH1, altered in Cohen syndrome. For each gene, we describe several\\n      alternative splicing variants; at least two transcripts per gene are major forms.\\n      The expression pattern of these genes is ubiquitous, with some tissue-specific\\n      differences between several transcript variants. Protein sequence comparisons\\n      suggest that intramolecular duplications have played an important role in the\\n      evolution of this gene family.\\nFAU - Velayos-Baeza, Antonio\\nAU  - Velayos-Baeza A\\nAD  - Wellcome Trust Centre for Human Genetics, University of Oxford, Headington, OX3\\n      7BN Oxford, UK.\\nFAU - Vettori, Andrea\\nAU  - Vettori A\\nFAU - Copley, Richard R\\nAU  - Copley RR\\nFAU - Dobson-Stone, Carol\\nAU  - Dobson-Stone C\\nFAU - Monaco, A P\\nAU  - Monaco AP\\nLA  - eng\\nSI  - GENBANK/AJ608769\\nSI  - GENBANK/AJ608770\\nSI  - GENBANK/AJ608771\\nSI  - GENBANK/AJ608772\\nSI  - GENBANK/AJ608773\\nSI  - GENBANK/AJ608774\\nSI  - GENBANK/AJ608775\\nSI  - GENBANK/AJ626859\\nSI  - GENBANK/AJ626860\\nSI  - GENBANK/AJ626861\\nPT  - Comparative Study\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov't\\nPL  - United States\\nTA  - Genomics\\nJT  - Genomics\\nJID - 8800135\\nRN  - 0 (Membrane Proteins)\\nRN  - 0 (Proteins)\\nRN  - 0 (VPS13A protein, human)\\nRN  - 0 (VPS13B protein, human)\\nRN  - 0 (VPS13C protein, human)\\nRN  - 0 (VPS13D protein, human)\\nRN  - 0 (Vesicular Transport Proteins)\\nSB  - IM\\nMH  - Alternative Splicing/*genetics\\nMH  - Amino Acid Sequence\\nMH  - Base Sequence\\nMH  - Blotting, Northern\\nMH  - Evolution, Molecular\\nMH  - Gene Components\\nMH  - Gene Duplication\\nMH  - Gene Expression Profiling\\nMH  - Humans\\nMH  - Membrane Proteins/*genetics\\nMH  - Molecular Sequence Data\\nMH  - Multigene Family/*genetics\\nMH  - Phylogeny\\nMH  - Proteins/*genetics\\nMH  - Reverse Transcriptase Polymerase Chain Reaction\\nMH  - Sequence Alignment\\nMH  - Sequence Analysis, DNA\\nMH  - Sequence Homology\\nMH  - Vesicular Transport Proteins\\nEDAT- 2004/10/23 09:00\\nMHDA- 2005/01/26 09:00\\nCRDT- 2004/10/23 09:00\\nPHST- 2004/02/13 00:00 [received]\\nPHST- 2004/04/19 00:00 [accepted]\\nPHST- 2004/10/23 09:00 [pubmed]\\nPHST- 2005/01/26 09:00 [medline]\\nPHST- 2004/10/23 09:00 [entrez]\\nAID - S0888754304001156 [pii]\\nAID - 10.1016/j.ygeno.2004.04.012 [doi]\\nPST - ppublish\\nSO  - Genomics. 2004 Sep;84(3):536-49. doi: 10.1016/j.ygeno.2004.04.012.\", \"PMID- 15070428\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20040604\\nLR  - 20181223\\nIS  - 1471-2229 (Electronic)\\nIS  - 1471-2229 (Linking)\\nVI  - 4\\nDP  - 2004 Mar 12\\nTI  - Comparison of ESTs from juvenile and adult phases of the giant unicellular green \\n      alga Acetabularia acetabulum.\\nPG  - 3\\nAB  - BACKGROUND: Acetabularia acetabulum is a giant unicellular green alga whose size \\n      and complex life cycle make it an attractive model for understanding\\n      morphogenesis and subcellular compartmentalization. The life cycle of this marine\\n      unicell is composed of several developmental phases. Juvenile and adult phases\\n      are temporally sequential but physiologically and morphologically distinct. To\\n      identify genes specific to juvenile and adult phases, we created two subtracted\\n      cDNA libraries, one adult-specific and one juvenile-specific, and analyzed 941\\n      randomly chosen ESTs from them. RESULTS: Clustering analysis suggests virtually\\n      no overlap between the two libraries. Preliminary expression data also suggests\\n      that we were successful at isolating transcripts differentially expressed between\\n      the two developmental phases and that many transcripts are specific to one phase \\n      or the other. Comparison of our EST sequences against publicly available sequence\\n      databases indicates that ESTs from the adult and the juvenile libraries partition\\n      into different functional classes. Three conserved sequence elements were common \\n      to several of the ESTs and were also found within the genomic sequence of the\\n      carbonic anhydrase1 gene from A. acetabulum. To date, these conserved elements\\n      are specific to A. acetabulum. CONCLUSIONS: Our data provide strong evidence that\\n      adult and juvenile phases in A. acetabulum vary significantly in gene expression.\\n      We discuss their possible roles in cell growth and morphogenesis as well as in\\n      phase change. We also discuss the potential role of the conserved elements found \\n      within the EST sequences in post-transcriptional regulation, particularly mRNA\\n      localization and/or stability.\\nFAU - Henry, Isabelle M\\nAU  - Henry IM\\nAD  - Department of Biology, University of Washington, Box 355325, 1521 NE Pacific\\n      Street, Seattle, WA 98195-5325, USA. henryi@u.washington.edu\\nFAU - Wilkinson, Mark D\\nAU  - Wilkinson MD\\nFAU - Hernandez, J Marcela\\nAU  - Hernandez JM\\nFAU - Schwarz-Sommer, Zsuzsanna\\nAU  - Schwarz-Sommer Z\\nFAU - Grotewold, Erich\\nAU  - Grotewold E\\nFAU - Mandoli, Dina F\\nAU  - Mandoli DF\\nLA  - eng\\nPT  - Comparative Study\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov't\\nPT  - Research Support, U.S. Gov't, Non-P.H.S.\\nDEP - 20040312\\nPL  - England\\nTA  - BMC Plant Biol\\nJT  - BMC plant biology\\nJID - 100967807\\nRN  - 0 (DNA, Complementary)\\nRN  - 0 (RNA, Messenger)\\nRN  - EC 4.2.1.1 (Carbonic Anhydrases)\\nSB  - IM\\nMH  - Acetabularia/*genetics/growth & development\\nMH  - Base Sequence\\nMH  - Blotting, Northern\\nMH  - Carbonic Anhydrases/genetics\\nMH  - Cluster Analysis\\nMH  - DNA, Complementary/chemistry/genetics\\nMH  - *Expressed Sequence Tags\\nMH  - *Gene Expression Regulation, Developmental\\nMH  - Gene Library\\nMH  - Molecular Sequence Data\\nMH  - Nucleic Acid Hybridization/methods\\nMH  - RNA, Messenger/genetics/isolation & purification/metabolism\\nMH  - Sequence Alignment\\nMH  - Sequence Analysis, DNA\\nMH  - Sequence Homology, Nucleic Acid\\nPMC - PMC385229\\nEDAT- 2004/04/09 05:00\\nMHDA- 2004/06/05 05:00\\nCRDT- 2004/04/09 05:00\\nPHST- 2003/11/19 00:00 [received]\\nPHST- 2004/03/12 00:00 [accepted]\\nPHST- 2004/04/09 05:00 [pubmed]\\nPHST- 2004/06/05 05:00 [medline]\\nPHST- 2004/04/09 05:00 [entrez]\\nAID - 10.1186/1471-2229-4-3 [doi]\\nAID - 1471-2229-4-3 [pii]\\nPST - epublish\\nSO  - BMC Plant Biol. 2004 Mar 12;4:3. doi: 10.1186/1471-2229-4-3.\", \"PMID- 12421765\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20030114\\nLR  - 20181113\\nIS  - 1088-9051 (Print)\\nIS  - 1088-9051 (Linking)\\nVI  - 12\\nIP  - 11\\nDP  - 2002 Nov\\nTI  - Protein-protein interactions between large proteins: two-hybrid screening using a\\n      functionally classified library composed of long cDNAs.\\nPG  - 1773-84\\nAB  - Large proteins have multiple domains that are potentially capable of binding many\\n      kinds of partners. It is conceivable, therefore, that such proteins could\\n      function as an intricate framework of assembly protein complexes. To\\n      comprehensively study protein-protein interactions between large KIAA proteins,\\n      we have constructed a library composed of 1087 KIAA cDNA clones based on prior\\n      functional classifications done in silico. We were guided by two principles that \\n      raise the success rate for detecting interactions per tested combination: we\\n      avoided testing low-probability combinations, and reduced the number of potential\\n      false negatives that arise from the fact that large proteins cannot reliably be\\n      expressed in yeast. The latter was addressed by constructing a cDNA library\\n      comprised of random fragments encoding large proteins. Cytoplasmic domains of\\n      KIAA transmembrane proteins (>1000 amino acids) were used as bait for yeast\\n      two-hybrid screening. Our analyses reveal that several KIAA proteins bearing a\\n      transmembrane region have the capability of binding to other KIAA proteins\\n      containing domains (e.g., PDZ, SH3, rhoGEF, and spectrin) known to be localized\\n      to highly specialized submembranous sites, indicating that they participate in\\n      cellular junction formation, receptor or channel clustering, and intracellular\\n      signaling events. Our representative library should be a very useful resource for\\n      detecting previously unidentified interactions because it complements\\n      conventional expression libraries, which seldom contain large cDNAs.\\nFAU - Nakayama, Manabu\\nAU  - Nakayama M\\nAD  - Department of Human Gene Research, Kazusa DNA Research Institute, Kisarazu, Chiba\\n      292-0818, Japan. nmanabu@kazusa.or.jp\\nFAU - Kikuno, Reiko\\nAU  - Kikuno R\\nFAU - Ohara, Osamu\\nAU  - Ohara O\\nLA  - eng\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov't\\nPL  - United States\\nTA  - Genome Res\\nJT  - Genome research\\nJID - 9518021\\nRN  - 0 (DNA, Complementary)\\nRN  - 0 (Peptides)\\nRN  - 0 (Proteins)\\nSB  - IM\\nMH  - Brain Chemistry/genetics\\nMH  - Cloning, Molecular\\nMH  - Computational Biology/instrumentation/methods\\nMH  - Cytoplasm/chemistry\\nMH  - DNA, Complementary/chemistry/*genetics\\nMH  - *Gene Library\\nMH  - Humans\\nMH  - Leukemia, Myeloid/genetics/pathology\\nMH  - Peptide Mapping/methods\\nMH  - Peptides/genetics/physiology\\nMH  - Protein Interaction Mapping/*methods\\nMH  - Protein Structure, Tertiary/physiology\\nMH  - Proteins/chemistry/genetics/physiology\\nMH  - Software\\nMH  - Tumor Cells, Cultured\\nMH  - *Two-Hybrid System Techniques\\nMH  - src Homology Domains/genetics/physiology\\nPMC - PMC187542\\nEDAT- 2002/11/08 04:00\\nMHDA- 2003/01/15 04:00\\nCRDT- 2002/11/08 04:00\\nPHST- 2002/11/08 04:00 [pubmed]\\nPHST- 2003/01/15 04:00 [medline]\\nPHST- 2002/11/08 04:00 [entrez]\\nAID - 10.1101/gr.406902 [doi]\\nPST - ppublish\\nSO  - Genome Res. 2002 Nov;12(11):1773-84. doi: 10.1101/gr.406902.\", \"PMID- 30789691\\nSTAT- Publisher\\nDA  - 20190222\\nCTDT- 20190221\\nPB  - University of Washington, Seattle\\nDP  - 1993\\nTI  - VPS13D Movement Disorder\\nBTI - GeneReviews((R))\\nAB  - CLINICAL CHARACTERISTICS: VPS13D movement disorder is a hyperkinetic movement\\n      disorder (dystonia, chorea, and/or ataxia) of variable age of onset that can be\\n      associated with developmental delay. Onset ranges from birth to adulthood.\\n      Individuals can present in childhood with motor delays and gait instability.\\n      Cognitive impairment ranging from mild intellectual disability to developmental\\n      delay has been reported, and several individuals have normal cognitive function. \\n      Individuals have also presented as young adults with gait difficulties caused by \\n      spastic ataxia or ataxia. In addition to gait ataxia, affected individuals had\\n      limb ataxia, dysarthria, and eye movement abnormalities (macro-saccadic\\n      oscillations, nystagmus, and saccadic pursuit). Additional features reported in\\n      some individuals include peripheral neuropathy and/or seizures. The disorder\\n      progresses to spastic ataxia or generalized dystonia, which can lead to loss of\\n      independent ambulation. DIAGNOSIS: The diagnosis of VPS13D movement disorder is\\n      established in a proband by identification of biallelic pathogenic variants in\\n      VPS13D on molecular genetic testing. MANAGEMENT: Treatment of manifestations:\\n      Standard treatment for seizures; spasticity treatments include baclofen,\\n      tizanidine, benzodiazepines, dantrolene sodium, gabapentin, botulinum toxin\\n      injections, and intrathecal baclofen; treatment options for dystonia include\\n      trihexyphenidyl, botulinum toxin injections, benzodiazepines, baclofen, and\\n      levodopa. A multidisciplinary team including occupational and physical therapists\\n      and a physiatrist is important; supportive developmental therapies should be\\n      provided as needed. Surveillance: Monitor for urinary urgency, contractures, and \\n      seizures; evaluate fall risk and gait stability; monitor developmental progress\\n      and educational needs. GENETIC COUNSELING: VPS13D movement disorder is inherited \\n      in an autosomal recessive manner. At conception, each sib of an affected\\n      individual has a 25% chance of being affected, a 50% chance of being an\\n      asymptomatic carrier, and a 25% chance of being unaffected and not a carrier.\\n      Carrier testing for at-risk relatives and prenatal testing for pregnancies at\\n      increased risk are possible if the VPS13D pathogenic variants in the family are\\n      known.\\nCI  - Copyright (c) 1993-2019, University of Washington, Seattle. GeneReviews is a\\n      registered trademark of the University of Washington, Seattle. All rights\\n      reserved.\\nFED - Adam, Margaret P\\nED  - Adam MP\\nFED - Ardinger, Holly H\\nED  - Ardinger HH\\nFED - Pagon, Roberta A\\nED  - Pagon RA\\nFED - Wallace, Stephanie E\\nED  - Wallace SE\\nFED - Bean, Lora JH\\nED  - Bean LJH\\nFED - Stephens, Karen\\nED  - Stephens K\\nFED - Amemiya, Anne\\nED  - Amemiya A\\nFAU - Meijer, Inge A\\nAU  - Meijer IA\\nLA  - eng\\nGR  - HHSN276201400262U/NLM NIH HHS/United States\\nPT  - Review\\nPT  - Book Chapter\\nPL  - Seattle (WA)\\nEDAT- 2019/02/22 06:01\\nMHDA- 2019/02/22 06:01\\nCDAT- 2019/02/22 06:01\\nAID - NBK537720 [bookaccession]\", \"PMID- 30258100\\nOWN - NLM\\nSTAT- In-Process\\nLR  - 20191115\\nIS  - 2045-2322 (Electronic)\\nIS  - 2045-2322 (Linking)\\nVI  - 8\\nIP  - 1\\nDP  - 2018 Sep 26\\nTI  - USP45 and Spindly are part of the same complex implicated in cell migration.\\nPG  - 14375\\nLID - 10.1038/s41598-018-32685-8 [doi]\\nAB  - Ubiquitylation is a protein modification implicated in several cellular\\n      processes. This process is reversible by the action of deubiquinating enzymes\\n      (DUBs). USP45 is a ubiquitin specific protease about which little is known, aside\\n      from roles in DNA damage repair and differentiation of the vertebrate retina.\\n      Here, by using mass spectrometry we have identified Spindly as a new target of\\n      USP45. Our data show that Spindly and USP45 are part of the same complex and that\\n      their interaction specifically depends on the catalytic activity of USP45. In\\n      addition, we describe the type of ubiquitin chains associated with the complex\\n      that can be cleaved by USP45, with a preferential activity on K48 ubiquitin chain\\n      type and potentially K6. Here, we also show that Spindly is mono-ubiquitylated\\n      and this can be specifically removed by USP45 in its active form but not by the\\n      catalytic inactive form. Lastly, we identified a new role for USP45 in cell\\n      migration, similar to that which was recently described for Spindly.\\nFAU - Conte, Claudia\\nAU  - Conte C\\nAD  - MRC Protein Phosphorylation and Ubiquitylation Unit, College of Life Sciences,\\n      University of Dundee, Dundee, UK.\\nAD  - Cell and Developmental Biology, Center of Genomic Regulation (CRG), Barcelona,\\n      Spain.\\nFAU - Griffis, Eric R\\nAU  - Griffis ER\\nAD  - MRC Protein Phosphorylation and Ubiquitylation Unit, College of Life Sciences,\\n      University of Dundee, Dundee, UK.\\nAD  - Nikon Instruments Incorporated, Melville, NY, 11747, USA.\\nFAU - Hickson, Ian\\nAU  - Hickson I\\nAD  - Janssen Research & Development, LLC, Spring House, PA, USA.\\nAD  - Northern Institute for Cancer Research, Newcastle University, Medical School,\\n      Newcastle upon Tyne, NE2 4HH, UK.\\nFAU - Perez-Oliva, Ana B\\nAU  - Perez-Oliva AB\\nAUID- ORCID: http://orcid.org/0000-0002-2596-0871\\nAD  - MRC Protein Phosphorylation and Ubiquitylation Unit, College of Life Sciences,\\n      University of Dundee, Dundee, UK. anabpo@um.es.\\nLA  - eng\\nGR  - MC_UU_12016/2/Medical Research Council (MRC)/International\\nGR  - 097945/B/11/Z/WT_/Wellcome Trust/United Kingdom\\nGR  - 097945/B/11/Z/WT_/Wellcome Trust/United Kingdom\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov't\\nDEP - 20180926\\nPL  - England\\nTA  - Sci Rep\\nJT  - Scientific reports\\nJID - 101563288\\nSB  - IM\\nPMC - PMC6158257\\nEDAT- 2018/09/28 06:00\\nMHDA- 2018/09/28 06:00\\nCRDT- 2018/09/28 06:00\\nPHST- 2018/05/21 00:00 [received]\\nPHST- 2018/09/11 00:00 [accepted]\\nPHST- 2018/09/28 06:00 [entrez]\\nPHST- 2018/09/28 06:00 [pubmed]\\nPHST- 2018/09/28 06:00 [medline]\\nAID - 10.1038/s41598-018-32685-8 [doi]\\nAID - 10.1038/s41598-018-32685-8 [pii]\\nPST - epublish\\nSO  - Sci Rep. 2018 Sep 26;8(1):14375. doi: 10.1038/s41598-018-32685-8.\", \"PMID- 29604224\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20190812\\nLR  - 20190812\\nIS  - 1531-8249 (Electronic)\\nIS  - 0364-5134 (Linking)\\nVI  - 83\\nIP  - 6\\nDP  - 2018 Jun\\nTI  - Mutations in VPS13D lead to a new recessive ataxia with spasticity and\\n      mitochondrial defects.\\nPG  - 1075-1088\\nLID - 10.1002/ana.25220 [doi]\\nAB  - OBJECTIVE: To identify novel causes of recessive ataxias, including\\n      spinocerebellar ataxia with saccadic intrusions, spastic ataxias, and spastic\\n      paraplegia. METHODS: In an international collaboration, we independently\\n      performed exome sequencing in 7 families with recessive ataxia and/or spastic\\n      paraplegia. To evaluate the role of VPS13D mutations, we evaluated a Drosophila\\n      knockout model and investigated mitochondrial function in patient-derived\\n      fibroblast cultures. RESULTS: Exome sequencing identified compound heterozygous\\n      mutations in VPS13D on chromosome 1p36 in all 7 families. This included a large\\n      family with 5 affected siblings with spinocerebellar ataxia with saccadic\\n      intrusions (SCASI), or spinocerebellar ataxia, recessive, type 4 (SCAR4). Linkage\\n      to chromosome 1p36 was found in this family with a logarithm of odds score of\\n      3.1. The phenotypic spectrum in our 12 patients was broad. Although most\\n      presented with ataxia, additional or predominant spasticity was present in 5\\n      patients. Disease onset ranged from infancy to 39 years, and symptoms were slowly\\n      progressive and included loss of independent ambulation in 5. All but 2 patients \\n      carried a loss-of-function (nonsense or splice site) mutation on one and a\\n      missense mutation on the other allele. Knockdown or removal of Vps13D in\\n      Drosophila neurons led to changes in mitochondrial morphology and impairment in\\n      mitochondrial distribution along axons. Patient fibroblasts showed altered\\n      morphology and functionality including reduced energy production. INTERPRETATION:\\n      Our study demonstrates that compound heterozygous mutations in VPS13D cause\\n      movement disorders along the ataxia-spasticity spectrum, making VPS13D the fourth\\n      VPS13 paralog involved in neurological disorders. Ann Neurol 2018.\\nCI  - (c) 2018 American Neurological Association.\\nFAU - Seong, Eunju\\nAU  - Seong E\\nAD  - Molecular and Behavioral Neuroscience Institute, University of Michigan, Ann\\n      Arbor, MI.\\nFAU - Insolera, Ryan\\nAU  - Insolera R\\nAD  - Department of Molecular, Cellular, and Developmental Biology, University of\\n      Michigan, Ann Arbor, MI.\\nFAU - Dulovic, Marija\\nAU  - Dulovic M\\nAD  - Institute of Neurogenetics, University of Lubeck, Lubeck, Germany.\\nFAU - Kamsteeg, Erik-Jan\\nAU  - Kamsteeg EJ\\nAD  - Department of Human Genetics, Radboud University Medical Centre, Nijmegen, the\\n      Netherlands.\\nFAU - Trinh, Joanne\\nAU  - Trinh J\\nAD  - Institute of Neurogenetics, University of Lubeck, Lubeck, Germany.\\nFAU - Bruggemann, Norbert\\nAU  - Bruggemann N\\nAD  - Department of Neurology, University of Lubeck, Lubeck, Germany.\\nFAU - Sandford, Erin\\nAU  - Sandford E\\nAD  - Molecular and Behavioral Neuroscience Institute, University of Michigan, Ann\\n      Arbor, MI.\\nFAU - Li, Sheng\\nAU  - Li S\\nAD  - Molecular and Behavioral Neuroscience Institute, University of Michigan, Ann\\n      Arbor, MI.\\nFAU - Ozel, Ayse Bilge\\nAU  - Ozel AB\\nAD  - Department of Human Genetics, University of Michigan, Ann Arbor, MI.\\nFAU - Li, Jun Z\\nAU  - Li JZ\\nAUID- ORCID: 0000-0001-6727-0812\\nAD  - Department of Human Genetics, University of Michigan, Ann Arbor, MI.\\nAD  - Department of Computational Medicine and Bioinformatics, University of Michigan, \\n      Ann Arbor, MI.\\nFAU - Jewett, Tamison\\nAU  - Jewett T\\nAD  - Department of Pediatrics, Section on Medical Genetics, Wake Forest School of\\n      Medicine, Winston-Salem, NC.\\nFAU - Kievit, Anneke J A\\nAU  - Kievit AJA\\nAD  - Department of Clinical Genetics, Erasmus MC, Rotterdam, the Netherlands.\\nFAU - Munchau, Alexander\\nAU  - Munchau A\\nAD  - Institute of Neurogenetics, University of Lubeck, Lubeck, Germany.\\nFAU - Shakkottai, Vikram\\nAU  - Shakkottai V\\nAD  - Departments of Neurology and Molecular and Integrative Physiology, University of \\n      Michigan, Ann Arbor, MI.\\nFAU - Klein, Christine\\nAU  - Klein C\\nAD  - Institute of Neurogenetics, University of Lubeck, Lubeck, Germany.\\nFAU - Collins, Catherine A\\nAU  - Collins CA\\nAUID- ORCID: 0000-0002-1608-6692\\nAD  - Department of Molecular, Cellular, and Developmental Biology, University of\\n      Michigan, Ann Arbor, MI.\\nFAU - Lohmann, Katja\\nAU  - Lohmann K\\nAD  - Institute of Neurogenetics, University of Lubeck, Lubeck, Germany.\\nFAU - van de Warrenburg, Bart P\\nAU  - van de Warrenburg BP\\nAD  - Department of Neurology, Donders Institute for Brain, Cognition, and Behavior,\\n      Radboud University Medical Centre, Nijmegen, the Netherlands.\\nFAU - Burmeister, Margit\\nAU  - Burmeister M\\nAUID- ORCID: 0000-0002-1914-2434\\nAD  - Molecular and Behavioral Neuroscience Institute, University of Michigan, Ann\\n      Arbor, MI.\\nAD  - Department of Human Genetics, University of Michigan, Ann Arbor, MI.\\nAD  - Department of Computational Medicine and Bioinformatics, University of Michigan, \\n      Ann Arbor, MI.\\nAD  - Department of Psychiatry, University of Michigan, Ann Arbor, MI.\\nLA  - eng\\nGR  - F31 NS056780/NS/NINDS NIH HHS/United States\\nGR  - F32 NS098611/NS/NINDS NIH HHS/United States\\nGR  - R01 NS069844/NS/NINDS NIH HHS/United States\\nGR  - R01 NS078560/NS/NINDS NIH HHS/United States\\nPT  - Journal Article\\nPT  - Research Support, N.I.H., Extramural\\nPT  - Research Support, Non-U.S. Gov't\\nDEP - 20180630\\nPL  - United States\\nTA  - Ann Neurol\\nJT  - Annals of neurology\\nJID - 7707449\\nRN  - 0 (Proteins)\\nRN  - 0 (VPS13D protein, human)\\nRN  - Spastic Ataxia\\nCIN - Ann Neurol. 2018 Jun;83(6):1072-1074. PMID: 29908061\\nMH  - Adult\\nMH  - Cerebellar Ataxia/genetics\\nMH  - Female\\nMH  - Genetic Linkage\\nMH  - Humans\\nMH  - Intellectual Disability/*genetics\\nMH  - Male\\nMH  - Middle Aged\\nMH  - Mitochondria/*genetics\\nMH  - Muscle Spasticity/*genetics\\nMH  - Mutation/*genetics\\nMH  - Optic Atrophy/*genetics\\nMH  - Pedigree\\nMH  - Proteins/*genetics\\nMH  - Spastic Paraplegia, Hereditary/genetics\\nMH  - Spinocerebellar Ataxias/*genetics\\nPMC - PMC6105379\\nMID - NIHMS955732\\nEDAT- 2018/04/01 06:00\\nMHDA- 2019/08/14 06:00\\nCRDT- 2018/04/01 06:00\\nPHST- 2017/11/01 00:00 [received]\\nPHST- 2018/03/11 00:00 [revised]\\nPHST- 2018/03/19 00:00 [accepted]\\nPHST- 2018/04/01 06:00 [pubmed]\\nPHST- 2019/08/14 06:00 [medline]\\nPHST- 2018/04/01 06:00 [entrez]\\nAID - 10.1002/ana.25220 [doi]\\nPST - ppublish\\nSO  - Ann Neurol. 2018 Jun;83(6):1075-1088. doi: 10.1002/ana.25220. Epub 2018 Jun 30.\", \"PMID- 29568061\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20181211\\nLR  - 20190322\\nIS  - 2041-1723 (Electronic)\\nIS  - 2041-1723 (Linking)\\nVI  - 9\\nIP  - 1\\nDP  - 2018 Mar 22\\nTI  - An AP-MS- and BioID-compatible MAC-tag enables comprehensive mapping of protein\\n      interactions and subcellular localizations.\\nPG  - 1188\\nLID - 10.1038/s41467-018-03523-2 [doi]\\nAB  - Protein-protein interactions govern almost all cellular functions. These complex \\n      networks of stable and transient associations can be mapped by affinity\\n      purification mass spectrometry (AP-MS) and complementary proximity-based labeling\\n      methods such as BioID. To exploit the advantages of both strategies, we here\\n      design and optimize an integrated approach combining AP-MS and BioID in a single \\n      construct, which we term MAC-tag. We systematically apply the MAC-tag approach to\\n      18 subcellular and 3 sub-organelle localization markers, generating a molecular\\n      context database, which can be used to define a protein's molecular location. In \\n      addition, we show that combining the AP-MS and BioID results makes it possible to\\n      obtain interaction distances within a protein complex. Taken together, our\\n      integrated strategy enables the comprehensive mapping of the physical and\\n      functional interactions of proteins, defining their molecular context and\\n      improving our understanding of the cellular interactome.\\nFAU - Liu, Xiaonan\\nAU  - Liu X\\nAUID- ORCID: http://orcid.org/0000-0002-9600-0536\\nAD  - Institute of Biotechnology, University of Helsinki, Helsinki, 00014, Finland.\\nAD  - Helsinki Institute of Life Science, University of Helsinki, Helsinki, 00014,\\n      Finland.\\nFAU - Salokas, Kari\\nAU  - Salokas K\\nAUID- ORCID: http://orcid.org/0000-0002-4471-6698\\nAD  - Institute of Biotechnology, University of Helsinki, Helsinki, 00014, Finland.\\nAD  - Helsinki Institute of Life Science, University of Helsinki, Helsinki, 00014,\\n      Finland.\\nFAU - Tamene, Fitsum\\nAU  - Tamene F\\nAD  - Institute of Biotechnology, University of Helsinki, Helsinki, 00014, Finland.\\nAD  - Helsinki Institute of Life Science, University of Helsinki, Helsinki, 00014,\\n      Finland.\\nAD  - Proteomics Unit, University of Helsinki, Helsinki, 00014, Finland.\\nFAU - Jiu, Yaming\\nAU  - Jiu Y\\nAUID- ORCID: http://orcid.org/0000-0002-8601-8820\\nAD  - Institute of Biotechnology, University of Helsinki, Helsinki, 00014, Finland.\\nAD  - Helsinki Institute of Life Science, University of Helsinki, Helsinki, 00014,\\n      Finland.\\nFAU - Weldatsadik, Rigbe G\\nAU  - Weldatsadik RG\\nAD  - Institute of Biotechnology, University of Helsinki, Helsinki, 00014, Finland.\\nAD  - Helsinki Institute of Life Science, University of Helsinki, Helsinki, 00014,\\n      Finland.\\nAD  - Proteomics Unit, University of Helsinki, Helsinki, 00014, Finland.\\nFAU - Ohman, Tiina\\nAU  - Ohman T\\nAD  - Institute of Biotechnology, University of Helsinki, Helsinki, 00014, Finland.\\nAD  - Helsinki Institute of Life Science, University of Helsinki, Helsinki, 00014,\\n      Finland.\\nAD  - Proteomics Unit, University of Helsinki, Helsinki, 00014, Finland.\\nFAU - Varjosalo, Markku\\nAU  - Varjosalo M\\nAUID- ORCID: http://orcid.org/0000-0002-1340-9732\\nAD  - Institute of Biotechnology, University of Helsinki, Helsinki, 00014, Finland.\\n      markku.varjosalo@helsinki.fi.\\nAD  - Helsinki Institute of Life Science, University of Helsinki, Helsinki, 00014,\\n      Finland. markku.varjosalo@helsinki.fi.\\nAD  - Proteomics Unit, University of Helsinki, Helsinki, 00014, Finland.\\n      markku.varjosalo@helsinki.fi.\\nLA  - eng\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov't\\nDEP - 20180322\\nPL  - England\\nTA  - Nat Commun\\nJT  - Nature communications\\nJID - 101528555\\nRN  - 0 (Proteins)\\nSB  - IM\\nMH  - Biotinylation\\nMH  - Cell Line\\nMH  - Chromatography, Affinity\\nMH  - High-Throughput Screening Assays/instrumentation/*methods\\nMH  - Humans\\nMH  - Mass Spectrometry\\nMH  - Protein Binding\\nMH  - Protein Interaction Mapping/instrumentation/*methods\\nMH  - Protein Transport\\nMH  - Proteins/*chemistry/isolation & purification/metabolism\\nPMC - PMC5864832\\nEDAT- 2018/03/24 06:00\\nMHDA- 2018/12/12 06:00\\nCRDT- 2018/03/24 06:00\\nPHST- 2018/02/01 00:00 [received]\\nPHST- 2018/02/21 00:00 [accepted]\\nPHST- 2018/03/24 06:00 [entrez]\\nPHST- 2018/03/24 06:00 [pubmed]\\nPHST- 2018/12/12 06:00 [medline]\\nAID - 10.1038/s41467-018-03523-2 [doi]\\nAID - 10.1038/s41467-018-03523-2 [pii]\\nPST - epublish\\nSO  - Nat Commun. 2018 Mar 22;9(1):1188. doi: 10.1038/s41467-018-03523-2.\", \"PMID- 29518281\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20190812\\nLR  - 20191114\\nIS  - 1531-8249 (Electronic)\\nIS  - 0364-5134 (Linking)\\nVI  - 83\\nIP  - 6\\nDP  - 2018 Jun\\nTI  - Recessive mutations in VPS13D cause childhood onset movement disorders.\\nPG  - 1089-1095\\nLID - 10.1002/ana.25204 [doi]\\nAB  - VPS13 protein family members VPS13A through VPS13C have been associated with\\n      various recessive movement disorders. We describe the first disease association\\n      of rare recessive VPS13D variants including frameshift, missense, and partial\\n      duplication mutations with a novel complex, hyperkinetic neurological disorder.\\n      The clinical features include developmental delay, a childhood onset movement\\n      disorder (chorea, dystonia, or tremor), and progressive spastic ataxia or\\n      paraparesis. Characteristic brain magnetic resonance imaging shows basal ganglia \\n      or diffuse white matter T2 hyperintensities as seen in Leigh syndrome and\\n      choreoacanthocytosis. Muscle biopsy in 1 case showed mitochondrial aggregates and\\n      lipidosis, suggesting mitochondrial dysfunction. These findings underline the\\n      importance of the VPS13 complex in neurological diseases and a possible role in\\n      mitochondrial function. Ann Neurol 2018;83:1089-1095.\\nCI  - (c) 2018 American Neurological Association.\\nFAU - Gauthier, Julie\\nAU  - Gauthier J\\nAD  - Molecular Diagnostic Laboratory and Division of Medical Genetics, Department of\\n      Pediatrics, Saint Justine University Hospital Center, Montreal, Canada.\\nFAU - Meijer, Inge A\\nAU  - Meijer IA\\nAD  - Department of Neuroscience, University of Montreal, Montreal, Canada.\\nFAU - Lessel, Davor\\nAU  - Lessel D\\nAUID- ORCID: 0000-0003-4496-244X\\nAD  - Institute of Human Genetics, University Medical Center Hamburg-Eppendorf,\\n      Hamburg, Germany.\\nFAU - Mencacci, Niccolo E\\nAU  - Mencacci NE\\nAD  - Department of Neurology, Northwestern University, Feinberg School of Medicine,\\n      Chicago, IL.\\nFAU - Krainc, Dimitri\\nAU  - Krainc D\\nAD  - Department of Neurology, Northwestern University, Feinberg School of Medicine,\\n      Chicago, IL.\\nFAU - Hempel, Maja\\nAU  - Hempel M\\nAD  - Institute of Human Genetics, University Medical Center Hamburg-Eppendorf,\\n      Hamburg, Germany.\\nFAU - Tsiakas, Konstantinos\\nAU  - Tsiakas K\\nAD  - Department of Pediatrics, University Medical Center Hamburg-Eppendorf, Hamburg,\\n      Germany.\\nFAU - Prokisch, Holger\\nAU  - Prokisch H\\nAD  - Institute of Human Genetics, Helmholtz Center Munich, Neuherberg, Germany.\\nAD  - Institute of Human Genetics, Technical University Munich, Munich, Germany.\\nFAU - Rossignol, Elsa\\nAU  - Rossignol E\\nAD  - Department of Neuroscience, University of Montreal, Montreal, Canada.\\nFAU - Helm, Margaret H\\nAU  - Helm MH\\nAD  - Maine Medical Partners Pediatric Specialty Care, Portland, ME.\\nFAU - Rodan, Lance H\\nAU  - Rodan LH\\nAD  - Department of Neurology, Boston Children's Hospital, Boston, MA.\\nFAU - Karamchandani, Jason\\nAU  - Karamchandani J\\nAD  - Department of Pathology, McGill University, Montreal Neurological Institute,\\n      Montreal, Canada.\\nFAU - Carecchio, Miryam\\nAU  - Carecchio M\\nAD  - Molecular Neurogenetics Unit, Institute for Research and Health Care (IRCCS)\\n      Foundation Carlo Besta Neurological Institute, Milan, Italy.\\nFAU - Lubbe, Steven J\\nAU  - Lubbe SJ\\nAD  - Department of Neurology, Northwestern University, Feinberg School of Medicine,\\n      Chicago, IL.\\nFAU - Telegrafi, Aida\\nAU  - Telegrafi A\\nAD  - GeneDx, Gaithersburg, MD.\\nFAU - Henderson, Lindsay B\\nAU  - Henderson LB\\nAD  - GeneDx, Gaithersburg, MD.\\nFAU - Lorenzo, Kerry\\nAU  - Lorenzo K\\nAD  - Kadlec Clinic Genetic Counseling, Richland, WA.\\nFAU - Wallace, Stephanie E\\nAU  - Wallace SE\\nAD  - Division of Genetic Medicine, Department of Pediatrics, Seattle Children's\\n      Hospital and University of Washington, Seattle, WA.\\nFAU - Glass, Ian A\\nAU  - Glass IA\\nAD  - Division of Genetic Medicine, Department of Pediatrics, Seattle Children's\\n      Hospital and University of Washington, Seattle, WA.\\nFAU - Hamdan, Fadi F\\nAU  - Hamdan FF\\nAD  - Molecular Diagnostic Laboratory and Division of Medical Genetics, Department of\\n      Pediatrics, Saint Justine University Hospital Center, Montreal, Canada.\\nFAU - Michaud, Jacques L\\nAU  - Michaud JL\\nAD  - Department of Pediatrics, Saint Justine University Hospital Center and University\\n      of Montreal, Montreal, Canada.\\nFAU - Rouleau, Guy A\\nAU  - Rouleau GA\\nAD  - Montreal Neurological Institute, Department of Neurology and Neurosurgery, McGill\\n      University, Montreal, Canada.\\nFAU - Campeau, Philippe M\\nAU  - Campeau PM\\nAUID- ORCID: 0000-0001-9713-7107\\nAD  - Department of Pediatrics, Saint Justine University Hospital Center and University\\n      of Montreal, Montreal, Canada.\\nLA  - eng\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov't\\nDEP - 20180410\\nPL  - United States\\nTA  - Ann Neurol\\nJT  - Annals of neurology\\nJID - 7707449\\nRN  - 0 (Proteins)\\nRN  - 0 (VPS13D protein, human)\\nRN  - Spastic Ataxia\\nCIN - Ann Neurol. 2018 Jun;83(6):1072-1074. PMID: 29908061\\nMH  - Basal Ganglia/pathology\\nMH  - Brain/pathology\\nMH  - Child\\nMH  - Humans\\nMH  - Intellectual Disability/*genetics\\nMH  - Leigh Disease/pathology\\nMH  - Magnetic Resonance Imaging/methods\\nMH  - Movement Disorders/*genetics\\nMH  - Muscle Spasticity/*genetics/pathology\\nMH  - Mutation/*genetics\\nMH  - Optic Atrophy/*genetics\\nMH  - Pedigree\\nMH  - Proteins/*genetics\\nMH  - Spinocerebellar Ataxias/*genetics\\nEDAT- 2018/03/09 06:00\\nMHDA- 2019/08/14 06:00\\nCRDT- 2018/03/09 06:00\\nPHST- 2017/11/10 00:00 [received]\\nPHST- 2018/03/06 00:00 [revised]\\nPHST- 2018/03/07 00:00 [accepted]\\nPHST- 2018/03/09 06:00 [pubmed]\\nPHST- 2019/08/14 06:00 [medline]\\nPHST- 2018/03/09 06:00 [entrez]\\nAID - 10.1002/ana.25204 [doi]\\nPST - ppublish\\nSO  - Ann Neurol. 2018 Jun;83(6):1089-1095. doi: 10.1002/ana.25204. Epub 2018 Apr 10.\", \"PMID- 29507755\\nOWN - NLM\\nSTAT- PubMed-not-MEDLINE\\nLR  - 20181114\\nIS  - 2056-5968 (Print)\\nIS  - 2056-5968 (Linking)\\nVI  - 4\\nDP  - 2018\\nTI  - VIRMA mediates preferential m(6)A mRNA methylation in 3'UTR and near stop codon\\n      and associates with alternative polyadenylation.\\nPG  - 10\\nLID - 10.1038/s41421-018-0019-0 [doi]\\nAB  - N(6)-methyladenosine (m(6)A) is enriched in 3'untranslated region (3'UTR) and\\n      near stop codon of mature polyadenylated mRNAs in mammalian systems and has\\n      regulatory roles in eukaryotic mRNA transcriptome switch. Significantly, the\\n      mechanism for this modification preference remains unknown, however. Herein we\\n      report a characterization of the full m(6)A methyltransferase complex in HeLa\\n      cells identifying METTL3/METTL14/WTAP/VIRMA/HAKAI/ZC3H13 as the key components,\\n      and we show that VIRMA mediates preferential mRNA methylation in 3'UTR and near\\n      stop codon. Biochemical studies reveal that VIRMA recruits the catalytic core\\n      components METTL3/METTL14/WTAP to guide region-selective methylations. Around 60%\\n      of VIRMA mRNA immunoprecipitation targets manifest strong m(6)A enrichment in\\n      3'UTR. Depletions of VIRMA and METTL3 induce 3'UTR lengthening of several hundred\\n      mRNAs with over 50% targets in common. VIRMA associates with polyadenylation\\n      cleavage factors CPSF5 and CPSF6 in an RNA-dependent manner. Depletion of CPSF5\\n      leads to significant shortening of 3'UTR of over 2800 mRNAs, 84% of which are\\n      modified with m(6)A and have increased m(6)A peak density in 3'UTR and near stop \\n      codon after CPSF5 knockdown. Together, our studies provide insights into m(6)A\\n      deposition specificity in 3'UTR and its correlation with alternative\\n      polyadenylation.\\nFAU - Yue, Yanan\\nAU  - Yue Y\\nAD  - 1MOE Key Laboratory of Macromolecular Synthesis and Functionalization, Department\\n      of Polymer Science and Engineering, Zhejiang University, Hangzhou, Zhejiang\\n      310027 China.0000 0004 1759 700Xgrid.13402.34\\nFAU - Liu, Jun\\nAU  - Liu J\\nAD  - 2Department of Chemistry, Department of Biochemistry and Molecular Biology,\\n      Institute for Biophysical Dynamics, Howard Hughes Medical Institute, The\\n      University of Chicago, Chicago, IL 60637 USA.0000 0004 1936 7822grid.170205.1\\nFAU - Cui, Xiaolong\\nAU  - Cui X\\nAD  - 2Department of Chemistry, Department of Biochemistry and Molecular Biology,\\n      Institute for Biophysical Dynamics, Howard Hughes Medical Institute, The\\n      University of Chicago, Chicago, IL 60637 USA.0000 0004 1936 7822grid.170205.1\\nFAU - Cao, Jie\\nAU  - Cao J\\nAD  - 1MOE Key Laboratory of Macromolecular Synthesis and Functionalization, Department\\n      of Polymer Science and Engineering, Zhejiang University, Hangzhou, Zhejiang\\n      310027 China.0000 0004 1759 700Xgrid.13402.34\\nFAU - Luo, Guanzheng\\nAU  - Luo G\\nAUID- ORCID: 0000-0002-6797-9319\\nAD  - 2Department of Chemistry, Department of Biochemistry and Molecular Biology,\\n      Institute for Biophysical Dynamics, Howard Hughes Medical Institute, The\\n      University of Chicago, Chicago, IL 60637 USA.0000 0004 1936 7822grid.170205.1\\nFAU - Zhang, Zezhou\\nAU  - Zhang Z\\nAD  - 1MOE Key Laboratory of Macromolecular Synthesis and Functionalization, Department\\n      of Polymer Science and Engineering, Zhejiang University, Hangzhou, Zhejiang\\n      310027 China.0000 0004 1759 700Xgrid.13402.34\\nFAU - Cheng, Tao\\nAU  - Cheng T\\nAD  - 1MOE Key Laboratory of Macromolecular Synthesis and Functionalization, Department\\n      of Polymer Science and Engineering, Zhejiang University, Hangzhou, Zhejiang\\n      310027 China.0000 0004 1759 700Xgrid.13402.34\\nFAU - Gao, Minsong\\nAU  - Gao M\\nAD  - 1MOE Key Laboratory of Macromolecular Synthesis and Functionalization, Department\\n      of Polymer Science and Engineering, Zhejiang University, Hangzhou, Zhejiang\\n      310027 China.0000 0004 1759 700Xgrid.13402.34\\nFAU - Shu, Xiao\\nAU  - Shu X\\nAD  - 1MOE Key Laboratory of Macromolecular Synthesis and Functionalization, Department\\n      of Polymer Science and Engineering, Zhejiang University, Hangzhou, Zhejiang\\n      310027 China.0000 0004 1759 700Xgrid.13402.34\\nFAU - Ma, Honghui\\nAU  - Ma H\\nAD  - 2Department of Chemistry, Department of Biochemistry and Molecular Biology,\\n      Institute for Biophysical Dynamics, Howard Hughes Medical Institute, The\\n      University of Chicago, Chicago, IL 60637 USA.0000 0004 1936 7822grid.170205.1\\nFAU - Wang, Fengqin\\nAU  - Wang F\\nAD  - 3College of Animal Sciences, Key Laboratory of Molecular Nutrition, Ministry of\\n      Education, Zhejiang University, Hangzhou, Zhejiang 310058 China.0000 0004 1759\\n      700Xgrid.13402.34\\nFAU - Wang, Xinxia\\nAU  - Wang X\\nAD  - 3College of Animal Sciences, Key Laboratory of Molecular Nutrition, Ministry of\\n      Education, Zhejiang University, Hangzhou, Zhejiang 310058 China.0000 0004 1759\\n      700Xgrid.13402.34\\nFAU - Shen, Bin\\nAU  - Shen B\\nAD  - 4State Key Laboratory of Reproductive Medicine, Department of Histology and\\n      Embryology, Nanjing Medical University, Nanjing, Jiangsu 210029 China.0000 0000\\n      9255 8984grid.89957.3a\\nFAU - Wang, Yizhen\\nAU  - Wang Y\\nAD  - 3College of Animal Sciences, Key Laboratory of Molecular Nutrition, Ministry of\\n      Education, Zhejiang University, Hangzhou, Zhejiang 310058 China.0000 0004 1759\\n      700Xgrid.13402.34\\nFAU - Feng, Xinhua\\nAU  - Feng X\\nAD  - 5Life Sciences Institute, Zhejiang University, Hangzhou, Zhejiang 310058\\n      China.0000 0004 1759 700Xgrid.13402.34\\nFAU - He, Chuan\\nAU  - He C\\nAUID- ORCID: 0000-0003-4319-7424\\nAD  - 2Department of Chemistry, Department of Biochemistry and Molecular Biology,\\n      Institute for Biophysical Dynamics, Howard Hughes Medical Institute, The\\n      University of Chicago, Chicago, IL 60637 USA.0000 0004 1936 7822grid.170205.1\\nFAU - Liu, Jianzhao\\nAU  - Liu J\\nAUID- ORCID: 0000-0001-9465-6075\\nAD  - 1MOE Key Laboratory of Macromolecular Synthesis and Functionalization, Department\\n      of Polymer Science and Engineering, Zhejiang University, Hangzhou, Zhejiang\\n      310027 China.0000 0004 1759 700Xgrid.13402.34\\nAD  - 5Life Sciences Institute, Zhejiang University, Hangzhou, Zhejiang 310058\\n      China.0000 0004 1759 700Xgrid.13402.34\\nLA  - eng\\nGR  - R01 HG008688/HG/NHGRI NIH HHS/United States\\nPT  - Journal Article\\nDEP - 20180227\\nPL  - England\\nTA  - Cell Discov\\nJT  - Cell discovery\\nJID - 101661034\\nPMC - PMC5826926\\nCOIS- C.H. is a scientific founder of the Accent Therapeutics and a member of the\\n      Scientific Advisory Committee. The remaining authors declare that they have no\\n      conflict of interest.\\nEDAT- 2018/03/07 06:00\\nMHDA- 2018/03/07 06:01\\nCRDT- 2018/03/07 06:00\\nPHST- 2017/11/24 00:00 [received]\\nPHST- 2018/02/02 00:00 [revised]\\nPHST- 2018/02/04 00:00 [accepted]\\nPHST- 2018/03/07 06:00 [entrez]\\nPHST- 2018/03/07 06:00 [pubmed]\\nPHST- 2018/03/07 06:01 [medline]\\nAID - 10.1038/s41421-018-0019-0 [doi]\\nAID - 19 [pii]\\nPST - epublish\\nSO  - Cell Discov. 2018 Feb 27;4:10. doi: 10.1038/s41421-018-0019-0. eCollection 2018.\", \"PMID- 29307555\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20190514\\nLR  - 20190517\\nIS  - 1879-0445 (Electronic)\\nIS  - 0960-9822 (Linking)\\nVI  - 28\\nIP  - 2\\nDP  - 2018 Jan 22\\nTI  - Vps13D Encodes a Ubiquitin-Binding Protein that Is Required for the Regulation of\\n      Mitochondrial Size and Clearance.\\nPG  - 287-295.e6\\nLID - S0960-9822(17)31589-0 [pii]\\nLID - 10.1016/j.cub.2017.11.064 [doi]\\nAB  - The clearance of mitochondria by autophagy, mitophagy, is important for cell and \\n      organism health [1], and known to be regulated by ubiquitin. During Drosophila\\n      intestine development, cells undergo a dramatic reduction in cell size and\\n      clearance of mitochondria that depends on autophagy, the E1 ubiquitin-activating \\n      enzyme Uba1, and ubiquitin [2]. Here we screen a collection of putative\\n      ubiquitin-binding domain-encoding genes for cell size reduction and autophagy\\n      phenotypes. We identify the endosomal sorting complex required for transport\\n      (ESCRT) components TSG101 and Vps36, as well as the novel gene Vps13D. Vps13D is \\n      an essential gene that is necessary for autophagy, mitochondrial size, and\\n      mitochondrial clearance in Drosophila. Interestingly, a similar mitochondrial\\n      phenotype is observed in VPS13D mutant human cells. The ubiquitin-associated\\n      (UBA) domain of Vps13D binds K63 ubiquitin chains, and mutants lacking the UBA\\n      domain have defects in mitochondrial size and clearance and exhibit\\n      semi-lethality, highlighting the importance of Vps13D ubiquitin binding in both\\n      mitochondrial health and development. VPS13D mutant cells possess phosphorylated \\n      DRP1 and mitochondrial fission factor (MFF) as well as DRP1 association with\\n      mitochondria, suggesting that VPS13D functions downstream of these known\\n      regulators of mitochondrial fission. In addition, the large Vps13D mitochondrial \\n      and cell size phenotypes are suppressed by decreased mitochondrial fusion gene\\n      function. Thus, these results provide a previously unknown link between\\n      ubiquitin, mitochondrial size regulation, and autophagy.\\nCI  - Copyright (c) 2017 Elsevier Ltd. All rights reserved.\\nFAU - Anding, Allyson L\\nAU  - Anding AL\\nAD  - Department of Molecular, Cell and Cancer Biology, University of Massachusetts\\n      Medical School, Worcester, MA 01605, USA.\\nFAU - Wang, Chunxin\\nAU  - Wang C\\nAD  - Biochemistry Section, Surgical Neurology Branch, National Institute of\\n      Neurological Disorders and Stroke, NIH, Bethesda, MD 20892, USA.\\nFAU - Chang, Tsun-Kai\\nAU  - Chang TK\\nAD  - Department of Molecular, Cell and Cancer Biology, University of Massachusetts\\n      Medical School, Worcester, MA 01605, USA.\\nFAU - Sliter, Danielle A\\nAU  - Sliter DA\\nAD  - Biochemistry Section, Surgical Neurology Branch, National Institute of\\n      Neurological Disorders and Stroke, NIH, Bethesda, MD 20892, USA.\\nFAU - Powers, Christine M\\nAU  - Powers CM\\nAD  - Department of Molecular, Cell and Cancer Biology, University of Massachusetts\\n      Medical School, Worcester, MA 01605, USA.\\nFAU - Hofmann, Kay\\nAU  - Hofmann K\\nAD  - Institute for Genetics, University of Cologne, 50674 Cologne, Germany.\\nFAU - Youle, Richard J\\nAU  - Youle RJ\\nAD  - Biochemistry Section, Surgical Neurology Branch, National Institute of\\n      Neurological Disorders and Stroke, NIH, Bethesda, MD 20892, USA.\\nFAU - Baehrecke, Eric H\\nAU  - Baehrecke EH\\nAD  - Department of Molecular, Cell and Cancer Biology, University of Massachusetts\\n      Medical School, Worcester, MA 01605, USA. Electronic address:\\n      eric.baehrecke@umassmed.edu.\\nLA  - eng\\nGR  - R01 GM111658/GM/NIGMS NIH HHS/United States\\nGR  - S10 RR027897/RR/NCRR NIH HHS/United States\\nPT  - Journal Article\\nPT  - Research Support, N.I.H., Extramural\\nPT  - Research Support, N.I.H., Intramural\\nDEP - 20180104\\nPL  - England\\nTA  - Curr Biol\\nJT  - Current biology : CB\\nJID - 9107782\\nRN  - 0 (Carrier Proteins)\\nRN  - 0 (Drosophila Proteins)\\nRN  - 0 (Intracellular Signaling Peptides and Proteins)\\nRN  - 0 (Ubiquitin)\\nRN  - 0 (Vps13D protein, Drosophila)\\nSB  - IM\\nCIN - Curr Biol. 2018 Jan 22;28(2):R66-R68. PMID: 29374445\\nMH  - Animals\\nMH  - Carrier Proteins/genetics/metabolism\\nMH  - Drosophila Proteins/*genetics/metabolism\\nMH  - Drosophila melanogaster/genetics/*physiology\\nMH  - Intracellular Signaling Peptides and Proteins/*genetics/metabolism\\nMH  - Mitochondria/physiology\\nMH  - Mitochondrial Degradation/*genetics\\nMH  - Mitochondrial Size/*genetics\\nMH  - Ubiquitin/metabolism\\nMH  - Ubiquitination\\nPMC - PMC5787036\\nMID - NIHMS925912\\nOTO - NOTNLM\\nOT  - *Drosophila\\nOT  - *Vps13D\\nOT  - *autophagy\\nOT  - *fission\\nOT  - *mitochondria\\nOT  - *mitophagy\\nOT  - *ubiquitin\\nEDAT- 2018/01/09 06:00\\nMHDA- 2019/05/15 06:00\\nCRDT- 2018/01/09 06:00\\nPHST- 2017/10/30 00:00 [received]\\nPHST- 2017/11/21 00:00 [revised]\\nPHST- 2017/11/28 00:00 [accepted]\\nPHST- 2018/01/09 06:00 [pubmed]\\nPHST- 2019/05/15 06:00 [medline]\\nPHST- 2018/01/09 06:00 [entrez]\\nAID - S0960-9822(17)31589-0 [pii]\\nAID - 10.1016/j.cub.2017.11.064 [doi]\\nPST - ppublish\\nSO  - Curr Biol. 2018 Jan 22;28(2):287-295.e6. doi: 10.1016/j.cub.2017.11.064. Epub\\n      2018 Jan 4.\", \"PMID- 26344197\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20151014\\nLR  - 20181113\\nIS  - 1476-4687 (Electronic)\\nIS  - 0028-0836 (Linking)\\nVI  - 525\\nIP  - 7569\\nDP  - 2015 Sep 17\\nTI  - Panorama of ancient metazoan macromolecular complexes.\\nPG  - 339-44\\nLID - 10.1038/nature14877 [doi]\\nAB  - Macromolecular complexes are essential to conserved biological processes, but\\n      their prevalence across animals is unclear. By combining extensive biochemical\\n      fractionation with quantitative mass spectrometry, here we directly examined the \\n      composition of soluble multiprotein complexes among diverse metazoan models.\\n      Using an integrative approach, we generated a draft conservation map consisting\\n      of more than one million putative high-confidence co-complex interactions for\\n      species with fully sequenced genomes that encompasses functional modules present \\n      broadly across all extant animals. Clustering reveals a spectrum of conservation,\\n      ranging from ancient eukaryotic assemblies that have probably served cellular\\n      housekeeping roles for at least one billion years, ancestral complexes that have \\n      accrued contemporary components, and rarer metazoan innovations linked to\\n      multicellularity. We validated these projections by independent co-fractionation \\n      experiments in evolutionarily distant species, affinity purification and\\n      functional analyses. The comprehensiveness, centrality and modularity of these\\n      reconstructed interactomes reflect their fundamental mechanistic importance and\\n      adaptive value to animal cell systems.\\nFAU - Wan, Cuihong\\nAU  - Wan C\\nAD  - Donnelly Centre for Cellular and Biomolecular Research, University of Toronto,\\n      Toronto, Ontario M5S 3E1, Canada.\\nAD  - Center for Systems and Synthetic Biology, Institute for Cellular and Molecular\\n      Biology, University of Texas at Austin, Austin, Texas 78712, USA.\\nFAU - Borgeson, Blake\\nAU  - Borgeson B\\nAD  - Center for Systems and Synthetic Biology, Institute for Cellular and Molecular\\n      Biology, University of Texas at Austin, Austin, Texas 78712, USA.\\nFAU - Phanse, Sadhna\\nAU  - Phanse S\\nAD  - Donnelly Centre for Cellular and Biomolecular Research, University of Toronto,\\n      Toronto, Ontario M5S 3E1, Canada.\\nFAU - Tu, Fan\\nAU  - Tu F\\nAD  - Center for Systems and Synthetic Biology, Institute for Cellular and Molecular\\n      Biology, University of Texas at Austin, Austin, Texas 78712, USA.\\nFAU - Drew, Kevin\\nAU  - Drew K\\nAD  - Center for Systems and Synthetic Biology, Institute for Cellular and Molecular\\n      Biology, University of Texas at Austin, Austin, Texas 78712, USA.\\nFAU - Clark, Greg\\nAU  - Clark G\\nAD  - Department of Medical Biophysics, Toronto, Ontario M5G 1L7, Canada.\\nFAU - Xiong, Xuejian\\nAU  - Xiong X\\nAD  - Department of Molecular Genetics, University of Toronto, Toronto, Ontario M5S\\n      1A8, Canada.\\nAD  - Hospital for Sick Children, Toronto, Ontario M5G 1X8, Canada.\\nFAU - Kagan, Olga\\nAU  - Kagan O\\nAD  - Donnelly Centre for Cellular and Biomolecular Research, University of Toronto,\\n      Toronto, Ontario M5S 3E1, Canada.\\nFAU - Kwan, Julian\\nAU  - Kwan J\\nAD  - Donnelly Centre for Cellular and Biomolecular Research, University of Toronto,\\n      Toronto, Ontario M5S 3E1, Canada.\\nAD  - Department of Molecular Genetics, University of Toronto, Toronto, Ontario M5S\\n      1A8, Canada.\\nFAU - Bezginov, Alexandr\\nAU  - Bezginov A\\nAD  - Department of Medical Biophysics, Toronto, Ontario M5G 1L7, Canada.\\nFAU - Chessman, Kyle\\nAU  - Chessman K\\nAD  - Department of Molecular Genetics, University of Toronto, Toronto, Ontario M5S\\n      1A8, Canada.\\nAD  - Hospital for Sick Children, Toronto, Ontario M5G 1X8, Canada.\\nFAU - Pal, Swati\\nAU  - Pal S\\nAD  - Hospital for Sick Children, Toronto, Ontario M5G 1X8, Canada.\\nFAU - Cromar, Graham\\nAU  - Cromar G\\nAD  - Department of Molecular Genetics, University of Toronto, Toronto, Ontario M5S\\n      1A8, Canada.\\nAD  - Hospital for Sick Children, Toronto, Ontario M5G 1X8, Canada.\\nFAU - Papoulas, Ophelia\\nAU  - Papoulas O\\nAD  - Center for Systems and Synthetic Biology, Institute for Cellular and Molecular\\n      Biology, University of Texas at Austin, Austin, Texas 78712, USA.\\nFAU - Ni, Zuyao\\nAU  - Ni Z\\nAD  - Donnelly Centre for Cellular and Biomolecular Research, University of Toronto,\\n      Toronto, Ontario M5S 3E1, Canada.\\nFAU - Boutz, Daniel R\\nAU  - Boutz DR\\nAD  - Center for Systems and Synthetic Biology, Institute for Cellular and Molecular\\n      Biology, University of Texas at Austin, Austin, Texas 78712, USA.\\nFAU - Stoilova, Snejana\\nAU  - Stoilova S\\nAD  - Donnelly Centre for Cellular and Biomolecular Research, University of Toronto,\\n      Toronto, Ontario M5S 3E1, Canada.\\nFAU - Havugimana, Pierre C\\nAU  - Havugimana PC\\nAD  - Donnelly Centre for Cellular and Biomolecular Research, University of Toronto,\\n      Toronto, Ontario M5S 3E1, Canada.\\nFAU - Guo, Xinghua\\nAU  - Guo X\\nAD  - Donnelly Centre for Cellular and Biomolecular Research, University of Toronto,\\n      Toronto, Ontario M5S 3E1, Canada.\\nFAU - Malty, Ramy H\\nAU  - Malty RH\\nAD  - Department of Biochemistry, University of Regina, Regina, Saskatchewan S4S 0A2,\\n      Canada.\\nFAU - Sarov, Mihail\\nAU  - Sarov M\\nAD  - Max Planck Institute of Molecular Cell Biology and Genetics, 01307 Dresden,\\n      Germany.\\nFAU - Greenblatt, Jack\\nAU  - Greenblatt J\\nAD  - Donnelly Centre for Cellular and Biomolecular Research, University of Toronto,\\n      Toronto, Ontario M5S 3E1, Canada.\\nAD  - Department of Molecular Genetics, University of Toronto, Toronto, Ontario M5S\\n      1A8, Canada.\\nFAU - Babu, Mohan\\nAU  - Babu M\\nAD  - Department of Biochemistry, University of Regina, Regina, Saskatchewan S4S 0A2,\\n      Canada.\\nFAU - Derry, W Brent\\nAU  - Derry WB\\nAD  - Department of Molecular Genetics, University of Toronto, Toronto, Ontario M5S\\n      1A8, Canada.\\nAD  - Hospital for Sick Children, Toronto, Ontario M5G 1X8, Canada.\\nFAU - Tillier, Elisabeth R\\nAU  - Tillier ER\\nAD  - Department of Medical Biophysics, Toronto, Ontario M5G 1L7, Canada.\\nFAU - Wallingford, John B\\nAU  - Wallingford JB\\nAD  - Center for Systems and Synthetic Biology, Institute for Cellular and Molecular\\n      Biology, University of Texas at Austin, Austin, Texas 78712, USA.\\nAD  - Department of Molecular Biosciences, University of Texas at Austin, Austin, Texas\\n      78712, USA.\\nFAU - Parkinson, John\\nAU  - Parkinson J\\nAD  - Department of Molecular Genetics, University of Toronto, Toronto, Ontario M5S\\n      1A8, Canada.\\nAD  - Hospital for Sick Children, Toronto, Ontario M5G 1X8, Canada.\\nFAU - Marcotte, Edward M\\nAU  - Marcotte EM\\nAD  - Center for Systems and Synthetic Biology, Institute for Cellular and Molecular\\n      Biology, University of Texas at Austin, Austin, Texas 78712, USA.\\nAD  - Department of Molecular Biosciences, University of Texas at Austin, Austin, Texas\\n      78712, USA.\\nFAU - Emili, Andrew\\nAU  - Emili A\\nAD  - Donnelly Centre for Cellular and Biomolecular Research, University of Toronto,\\n      Toronto, Ontario M5S 3E1, Canada.\\nAD  - Department of Molecular Genetics, University of Toronto, Toronto, Ontario M5S\\n      1A8, Canada.\\nLA  - eng\\nGR  - F32GM112495/GM/NIGMS NIH HHS/United States\\nGR  - R01 GM076536/GM/NIGMS NIH HHS/United States\\nGR  - DP1 GM106408/GM/NIGMS NIH HHS/United States\\nGR  - R01 GM067779/GM/NIGMS NIH HHS/United States\\nGR  - Canadian Institutes of Health Research/Canada\\nGR  - F32 GM112495/GM/NIGMS NIH HHS/United States\\nPT  - Journal Article\\nPT  - Research Support, N.I.H., Extramural\\nPT  - Research Support, Non-U.S. Gov't\\nPT  - Research Support, U.S. Gov't, Non-P.H.S.\\nDEP - 20150907\\nPL  - England\\nTA  - Nature\\nJT  - Nature\\nJID - 0410462\\nRN  - 0 (Multiprotein Complexes)\\nSB  - IM\\nCIN - Trends Biochem Sci. 2016 Jan;41(1):4-6. PMID: 26682499\\nCIN - Nat Methods. 2015 Nov;12(11):1011. PMID: 26824107\\nMH  - Animals\\nMH  - Datasets as Topic\\nMH  - *Evolution, Molecular\\nMH  - Humans\\nMH  - Multiprotein Complexes/*chemistry/*metabolism\\nMH  - Protein Interaction Mapping\\nMH  - *Protein Interaction Maps\\nMH  - Reproducibility of Results\\nMH  - Systems Biology\\nMH  - Tandem Mass Spectrometry\\nPMC - PMC5036527\\nMID - NIHMS704400\\nEDAT- 2015/09/08 06:00\\nMHDA- 2015/10/16 06:00\\nCRDT- 2015/09/08 06:00\\nPHST- 2014/12/15 00:00 [received]\\nPHST- 2015/06/30 00:00 [accepted]\\nPHST- 2015/09/08 06:00 [entrez]\\nPHST- 2015/09/08 06:00 [pubmed]\\nPHST- 2015/10/16 06:00 [medline]\\nAID - nature14877 [pii]\\nAID - 10.1038/nature14877 [doi]\\nPST - ppublish\\nSO  - Nature. 2015 Sep 17;525(7569):339-44. doi: 10.1038/nature14877. Epub 2015 Sep 7.\", \"PMID- 25896417\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20160620\\nLR  - 20150828\\nIS  - 1662-8128 (Electronic)\\nIS  - 1662-811X (Linking)\\nVI  - 7\\nIP  - 5\\nDP  - 2015\\nTI  - VPS13D Gene Variant Is Associated with Altered IL-6 Production and Mortality in\\n      Septic Shock.\\nPG  - 545-53\\nLID - 10.1159/000381265 [doi]\\nAB  - BACKGROUND: Genetic variations contribute to septic shock mortality. To discover \\n      a novel locus, we performed in vitro genome-wide association studies (GWAS) and\\n      further tested the result in a cohort of septic shock patients. METHODS: Two in\\n      vitro GWAS using a quantitative trait locus analysis of stimulated IL-6\\n      production in lymphoblastoid cells from 60 individuals of European ancestry were \\n      performed. VPS13D rs6685273 was genotyped in European ancestry patients (n =\\n      498). The VPS13D gene was silenced in vitro. RESULTS: Two GWAS using\\n      lymphoblastoid cells identified the locus of VPS13D rs6685273 that was\\n      significant in the same direction in both GWAS. The VPS13D rs6685273 C allele was\\n      associated with increased IL-6 production. Patients with septic shock who had the\\n      VPS13D rs6685273 CC genotype had an increased 28-day mortality (p = 0.023) and\\n      more organ failure (p < 0.05) compared to the CT/TT genotypes. VPS13D in vitro\\n      gene silencing in the HeLa cell line increased IL-6 production. Furthermore, the \\n      rs6685273 genotype was associated with differential VPS13D splice variant\\n      expression. CONCLUSIONS: The VPS13D rs6685273 C allele was associated with\\n      increased IL-6 production in vitro. The patients with the VPS13D rs6685273 CC\\n      genotype had increased 28-day mortality and increased organ failure. VPS13D\\n      appears to regulate IL-6 production.\\nCI  - (c) 2015 S. Karger AG, Basel.\\nFAU - Nakada, Taka-aki\\nAU  - Nakada TA\\nAD  - Critical Care Research Laboratories, Centre for Heart Lung Innovation, St. Paul's\\n      Hospital, University of British Columbia, Vancouver, B.C., Canada.\\nFAU - Boyd, John H\\nAU  - Boyd JH\\nFAU - Russell, James A\\nAU  - Russell JA\\nFAU - Aguirre-Hernandez, Rosalia\\nAU  - Aguirre-Hernandez R\\nFAU - Wilkinson, Mark D\\nAU  - Wilkinson MD\\nFAU - Thair, Simone A\\nAU  - Thair SA\\nFAU - Nakada, Emiri\\nAU  - Nakada E\\nFAU - McConechy, Melissa K\\nAU  - McConechy MK\\nFAU - Fjell, Christopher D\\nAU  - Fjell CD\\nFAU - Walley, Keith R\\nAU  - Walley KR\\nLA  - eng\\nGR  - Canadian Institutes of Health Research/Canada\\nPT  - Journal Article\\nPT  - Multicenter Study\\nPT  - Randomized Controlled Trial\\nPT  - Research Support, Non-U.S. Gov't\\nDEP - 20150417\\nPL  - Switzerland\\nTA  - J Innate Immun\\nJT  - Journal of innate immunity\\nJID - 101469471\\nRN  - 0 (Interleukin-6)\\nRN  - 0 (Proteins)\\nRN  - 0 (VPS13D protein, human)\\nSB  - IM\\nMH  - Aged\\nMH  - Cohort Studies\\nMH  - European Continental Ancestry Group/genetics\\nMH  - Female\\nMH  - Gene Frequency\\nMH  - Genetic Predisposition to Disease\\nMH  - Genome-Wide Association Study\\nMH  - Genotype\\nMH  - HeLa Cells\\nMH  - Humans\\nMH  - Interleukin-6/genetics/*metabolism\\nMH  - Lymphocyte Activation/genetics\\nMH  - Lymphocytes/*physiology\\nMH  - Male\\nMH  - Middle Aged\\nMH  - Polymorphism, Single Nucleotide\\nMH  - Proteins/*genetics\\nMH  - Shock, Septic/*genetics/immunology/*mortality\\nMH  - Spain\\nMH  - Survival Analysis\\nEDAT- 2015/04/22 06:00\\nMHDA- 2016/06/21 06:00\\nCRDT- 2015/04/22 06:00\\nPHST- 2014/12/17 00:00 [received]\\nPHST- 2015/02/26 00:00 [accepted]\\nPHST- 2015/04/22 06:00 [entrez]\\nPHST- 2015/04/22 06:00 [pubmed]\\nPHST- 2016/06/21 06:00 [medline]\\nAID - 000381265 [pii]\\nAID - 10.1159/000381265 [doi]\\nPST - ppublish\\nSO  - J Innate Immun. 2015;7(5):545-53. doi: 10.1159/000381265. Epub 2015 Apr 17.\", \"PMID- 21873635\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20120403\\nLR  - 20181113\\nIS  - 1477-4054 (Electronic)\\nIS  - 1467-5463 (Linking)\\nVI  - 12\\nIP  - 5\\nDP  - 2011 Sep\\nTI  - Phylogenetic-based propagation of functional annotations within the Gene Ontology\\n      consortium.\\nPG  - 449-62\\nLID - 10.1093/bib/bbr042 [doi]\\nAB  - The goal of the Gene Ontology (GO) project is to provide a uniform way to\\n      describe the functions of gene products from organisms across all kingdoms of\\n      life and thereby enable analysis of genomic data. Protein annotations are either \\n      based on experiments or predicted from protein sequences. Since most sequences\\n      have not been experimentally characterized, most available annotations need to be\\n      based on predictions. To make as accurate inferences as possible, the GO\\n      Consortium's Reference Genome Project is using an explicit evolutionary framework\\n      to infer annotations of proteins from a broad set of genomes from experimental\\n      annotations in a semi-automated manner. Most components in the pipeline, such as \\n      selection of sequences, building multiple sequence alignments and phylogenetic\\n      trees, retrieving experimental annotations and depositing inferred annotations,\\n      are fully automated. However, the most crucial step in our pipeline relies on\\n      software-assisted curation by an expert biologist. This curation tool,\\n      Phylogenetic Annotation and INference Tool (PAINT) helps curators to infer\\n      annotations among members of a protein family. PAINT allows curators to make\\n      precise assertions as to when functions were gained and lost during evolution and\\n      record the evidence (e.g. experimentally supported GO annotations and\\n      phylogenetic information including orthology) for those assertions. In this\\n      article, we describe how we use PAINT to infer protein function in a phylogenetic\\n      context with emphasis on its strengths, limitations and guidelines. We also\\n      discuss specific examples showing how PAINT annotations compare with those\\n      generated by other highly used homology-based methods.\\nFAU - Gaudet, Pascale\\nAU  - Gaudet P\\nAD  - Swiss Institute for Bioinformatics, CMU, 1 Rue Michel Servet, 1211 Geneva 4,\\n      Switzerland. pascale.gaudet@isb-sib.ch\\nFAU - Livstone, Michael S\\nAU  - Livstone MS\\nFAU - Lewis, Suzanna E\\nAU  - Lewis SE\\nFAU - Thomas, Paul D\\nAU  - Thomas PD\\nLA  - eng\\nGR  - P41-HG002273/HG/NHGRI NIH HHS/United States\\nGR  - R01-GM081084/GM/NIGMS NIH HHS/United States\\nPT  - Journal Article\\nPT  - Research Support, N.I.H., Extramural\\nDEP - 20110827\\nPL  - England\\nTA  - Brief Bioinform\\nJT  - Briefings in bioinformatics\\nJID - 100912837\\nRN  - 0 (Proteins)\\nSB  - IM\\nMH  - Databases, Genetic\\nMH  - Genome\\nMH  - Genomics/*methods\\nMH  - Molecular Sequence Annotation/*methods\\nMH  - *Phylogeny\\nMH  - Proteins/*chemistry/genetics\\nPMC - PMC3178059\\nEDAT- 2011/08/30 06:00\\nMHDA- 2012/04/04 06:00\\nCRDT- 2011/08/30 06:00\\nPHST- 2011/08/30 06:00 [entrez]\\nPHST- 2011/08/30 06:00 [pubmed]\\nPHST- 2012/04/04 06:00 [medline]\\nAID - bbr042 [pii]\\nAID - 10.1093/bib/bbr042 [doi]\\nPST - ppublish\\nSO  - Brief Bioinform. 2011 Sep;12(5):449-62. doi: 10.1093/bib/bbr042. Epub 2011 Aug\\n      27.\", \"PMID- 21182205\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20110407\\nLR  - 20101224\\nIS  - 1615-9861 (Electronic)\\nIS  - 1615-9853 (Linking)\\nVI  - 11\\nIP  - 1\\nDP  - 2011 Jan\\nTI  - Identification of proteins associated with ligand-activated estrogen receptor\\n      alpha in human breast cancer cell nuclei by tandem affinity purification and nano\\n      LC-MS/MS.\\nPG  - 172-9\\nLID - 10.1002/pmic.201000217 [doi]\\nAB  - Estrogen receptor alpha (ER-alpha) is a key mediator of estrogen actions in\\n      breast cancer (BC) cells. Understanding the effects of ligand-activated ER-alpha \\n      in target cells requires identification of the molecular partners acting in\\n      concert with this nuclear receptor to transduce the hormonal signal. We applied\\n      tandem affinity purification (TAP), glycerol gradient centrifugation and MS\\n      analysis to isolate and identify proteins interacting with ligand-activated\\n      ER-alpha in MCF-7 cell nuclei. This led to the identification of 264\\n      ER-associated proteins, whose functions highlight the hinge role of ER-alpha in\\n      the coordination of multiple hormone-regulated nuclear processes in BC cells.\\nCI  - Copyright (c) 2011 WILEY-VCH Verlag GmbH & Co. KGaA, Weinheim.\\nFAU - Tarallo, Roberta\\nAU  - Tarallo R\\nAD  - Department of General Pathology, Second University of Naples, Napoli, Italy.\\nFAU - Bamundo, Angela\\nAU  - Bamundo A\\nFAU - Nassa, Giovanni\\nAU  - Nassa G\\nFAU - Nola, Ernesto\\nAU  - Nola E\\nFAU - Paris, Ornella\\nAU  - Paris O\\nFAU - Ambrosino, Concetta\\nAU  - Ambrosino C\\nFAU - Facchiano, Angelo\\nAU  - Facchiano A\\nFAU - Baumann, Marc\\nAU  - Baumann M\\nFAU - Nyman, Tuula A\\nAU  - Nyman TA\\nFAU - Weisz, Alessandro\\nAU  - Weisz A\\nLA  - eng\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov't\\nDEP - 20101201\\nPL  - Germany\\nTA  - Proteomics\\nJT  - Proteomics\\nJID - 101092707\\nRN  - 0 (Estrogen Receptor alpha)\\nSB  - IM\\nMH  - Breast Neoplasms/*metabolism\\nMH  - Cell Line, Tumor\\nMH  - Chromatography, Affinity\\nMH  - Chromatography, Liquid/*methods\\nMH  - Estrogen Receptor alpha/*metabolism\\nMH  - Female\\nMH  - Humans\\nMH  - Tandem Mass Spectrometry/*methods\\nEDAT- 2010/12/25 06:00\\nMHDA- 2011/04/08 06:00\\nCRDT- 2010/12/25 06:00\\nPHST- 2010/04/02 00:00 [received]\\nPHST- 2010/08/03 00:00 [revised]\\nPHST- 2010/10/04 00:00 [accepted]\\nPHST- 2010/12/25 06:00 [entrez]\\nPHST- 2010/12/25 06:00 [pubmed]\\nPHST- 2011/04/08 06:00 [medline]\\nAID - 10.1002/pmic.201000217 [doi]\\nPST - ppublish\\nSO  - Proteomics. 2011 Jan;11(1):172-9. doi: 10.1002/pmic.201000217. Epub 2010 Dec 1.\", \"PMID- 20610895\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20110308\\nLR  - 20190608\\nIS  - 1880-3873 (Electronic)\\nIS  - 1340-3478 (Linking)\\nVI  - 17\\nIP  - 10\\nDP  - 2010 Oct 27\\nTI  - Identification of evidence suggestive of an association with peripheral arterial \\n      disease at the OSBPL10 locus by genome-wide investigation in the Japanese\\n      population.\\nPG  - 1054-62\\nAB  - AIM: Peripheral arterial disease (PAD) is a common cause of cardiovascular\\n      morbidity and an independent predictor of cardiovascular mortality. However,\\n      little is known about the genetic basis of PAD. To elucidate this, we performed a\\n      two-staged genome-wide association study in Japanese individuals. METHODS: We\\n      initially tested 222,285 single-nucleotide polymorphisms (SNPs). After the first \\n      screening in a panel of 195 PAD cases and 1,358 controls, 2,696 SNPs (1.2%) were \\n      further genotyped in the second screening using another panel of 699 PAD cases\\n      and 1540 controls. In both screenings, controls were subjects affected with some \\n      diseases other than PAD. RESULTS: When analyzed in the combined panel, the\\n      strongest signal of PAD association was observed at rs1902341 in the intron of\\n      OSBPL10 (p=4.7E-7 for trend test; OR=1.31, 95% CI 1.18-1.46). Also, PAD was\\n      modestly associated at several other loci such as rs2554503 in CSMD1 (p=5.7E-5;\\n      OR=1.32, 95% CI 1.15-1.51) or rs235243 in VSP13D (p=0.04; OR=1.18, 95% CI\\n      1.01-1.37). CONCLUSION: Our genome-wide exploration identified suggestive\\n      evidence of PAD association at the OSBPL10 locus. Because the association has not\\n      reached a genome-wide significant level, further replication study is warranted\\n      for verification in the Japanese population.\\nFAU - Koriyama, Hiroshi\\nAU  - Koriyama H\\nAD  - Department of Geriatric Medicine and Nephrology, Osaka University Graduate School\\n      of Medicine, 2-2 Yamada-oka, Suita, Japan.\\nFAU - Nakagami, Hironori\\nAU  - Nakagami H\\nFAU - Katsuya, Tomohiro\\nAU  - Katsuya T\\nFAU - Sugimoto, Ken\\nAU  - Sugimoto K\\nFAU - Yamashita, Hidetoshi\\nAU  - Yamashita H\\nFAU - Takami, Yoichi\\nAU  - Takami Y\\nFAU - Maeda, Shiro\\nAU  - Maeda S\\nFAU - Kubo, Michiaki\\nAU  - Kubo M\\nFAU - Takahashi, Atsushi\\nAU  - Takahashi A\\nFAU - Nakamura, Yusuke\\nAU  - Nakamura Y\\nFAU - Ogihara, Toshio\\nAU  - Ogihara T\\nFAU - Rakugi, Hiromi\\nAU  - Rakugi H\\nFAU - Kaneda, Yasufumi\\nAU  - Kaneda Y\\nFAU - Morishita, Ryuichi\\nAU  - Morishita R\\nLA  - eng\\nPT  - Comparative Study\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov't\\nDEP - 20100702\\nPL  - Japan\\nTA  - J Atheroscler Thromb\\nJT  - Journal of atherosclerosis and thrombosis\\nJID - 9506298\\nRN  - 0 (Receptors, Steroid)\\nRN  - 0 (oxysterol binding protein)\\nSB  - IM\\nMH  - Aged\\nMH  - Asian Continental Ancestry Group/*genetics\\nMH  - Case-Control Studies\\nMH  - Female\\nMH  - Genetic Predisposition to Disease\\nMH  - Genome, Human\\nMH  - *Genome-Wide Association Study\\nMH  - Genotype\\nMH  - Humans\\nMH  - Japan/epidemiology\\nMH  - Linkage Disequilibrium\\nMH  - Male\\nMH  - Middle Aged\\nMH  - Peripheral Vascular Diseases/epidemiology/*genetics\\nMH  - Polymorphism, Single Nucleotide/*genetics\\nMH  - Receptors, Steroid/*genetics\\nEDAT- 2010/07/09 06:00\\nMHDA- 2011/03/09 06:00\\nCRDT- 2010/07/09 06:00\\nPHST- 2010/07/09 06:00 [entrez]\\nPHST- 2010/07/09 06:00 [pubmed]\\nPHST- 2011/03/09 06:00 [medline]\\nAID - JST.JSTAGE/jat/4291 [pii]\\nAID - 10.5551/jat.4291 [doi]\\nPST - ppublish\\nSO  - J Atheroscler Thromb. 2010 Oct 27;17(10):1054-62. doi: 10.5551/jat.4291. Epub\\n      2010 Jul 2.\", \"PMID- 20379614\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20101108\\nLR  - 20181113\\nIS  - 1528-3658 (Electronic)\\nIS  - 1076-1551 (Linking)\\nVI  - 16\\nIP  - 7-8\\nDP  - 2010 Jul-Aug\\nTI  - Personalized smoking cessation: interactions between nicotine dose, dependence\\n      and quit-success genotype score.\\nPG  - 247-53\\nLID - 10.2119/molmed.2009.00159 [doi]\\nAB  - Improving and targeting nicotine replacement therapy (NRT) are cost-effective\\n      strategies for reducing adverse health consequences for smokers. Treatment\\n      studies document the efficacy of precessation NRT and support important roles for\\n      level of nicotine dependence and precessation smoking reduction in successful\\n      quitting. However, prior work has not identified the optimal precessation dose or\\n      means for personalizing NRT. Genome-wide association has identified groups of\\n      genomic markers associated with successful quitting, allowing us to develop a\\n      v1.0 \\\"quit-success\\\" genotype score. We now report influences of v1.0 quit-success\\n      genotype score, level of dependence and precessation smoking reduction in a\\n      smoking cessation trial that examined effects of 21 versus 42 mg/24 h\\n      precessation NRT. Four hundred seventy-nine smokers were randomized to 21 or 42\\n      mg NRT, initiated 2 wks prior to target quit dates. We monitored self-reported\\n      abstinence and end-expired air carbon monoxide (CO). Genotyping used Affymetrix\\n      arrays (Santa Clara, CA, USA). The primary outcome was 10-wk continuous smoking\\n      abstinence. NRT dose, level of nicotine dependence and genotype scores displayed \\n      significant interactive effects on successful quitting. Successful abstinence\\n      also was predicted by CO reductions during precessation NRT. These results\\n      document ways in which smoking cessation strategies can be personalized based on \\n      levels of nicotine dependence, genotype scores and CO monitoring. These\\n      assessments, taken together, can help match most smokers with optimal NRT doses\\n      and help rapidly identify some who may be better treated using other methods.\\nFAU - Rose, Jed E\\nAU  - Rose JE\\nAD  - Department of Psychiatry and Center for Nicotine and Smoking Cessation Research, \\n      Duke University, Durham, North Carolina 27705, USA. rose0003@mc.duke.edu\\nFAU - Behm, Frederique M\\nAU  - Behm FM\\nFAU - Drgon, Tomas\\nAU  - Drgon T\\nFAU - Johnson, Catherine\\nAU  - Johnson C\\nFAU - Uhl, George R\\nAU  - Uhl GR\\nLA  - eng\\nSI  - ClinicalTrials.gov/NCT00734617\\nGR  - P50 DA027840/DA/NIDA NIH HHS/United States\\nGR  - Intramural NIH HHS/United States\\nPT  - Journal Article\\nPT  - Randomized Controlled Trial\\nPT  - Research Support, N.I.H., Intramural\\nPT  - Research Support, Non-U.S. Gov't\\nDEP - 20100317\\nPL  - England\\nTA  - Mol Med\\nJT  - Molecular medicine (Cambridge, Mass.)\\nJID - 9501023\\nRN  - 0 (Delayed-Action Preparations)\\nRN  - 6M3C89ZY6R (Nicotine)\\nRN  - 7U1EE4V452 (Carbon Monoxide)\\nSB  - IM\\nEIN - Mol Med. 2012;18(1):729\\nMH  - Administration, Cutaneous\\nMH  - Adult\\nMH  - Carbon Monoxide/analysis\\nMH  - Delayed-Action Preparations\\nMH  - Double-Blind Method\\nMH  - Female\\nMH  - Genome-Wide Association Study\\nMH  - Genotype\\nMH  - Humans\\nMH  - Logistic Models\\nMH  - Male\\nMH  - Middle Aged\\nMH  - Nicotine/*administration & dosage\\nMH  - Polymorphism, Single Nucleotide\\nMH  - Precision Medicine/*methods\\nMH  - Smoking Cessation/*methods\\nMH  - Tobacco Use Disorder/*drug therapy/*genetics\\nPMC - PMC2896464\\nEDAT- 2010/04/10 06:00\\nMHDA- 2010/11/09 06:00\\nCRDT- 2010/04/10 06:00\\nPHST- 2009/10/31 00:00 [received]\\nPHST- 2010/03/16 00:00 [accepted]\\nPHST- 2010/04/10 06:00 [entrez]\\nPHST- 2010/04/10 06:00 [pubmed]\\nPHST- 2010/11/09 06:00 [medline]\\nAID - molmed.2009.00159 [pii]\\nAID - 10.2119/molmed.2009.00159 [doi]\\nPST - ppublish\\nSO  - Mol Med. 2010 Jul-Aug;16(7-8):247-53. doi: 10.2119/molmed.2009.00159. Epub 2010\\n      Mar 17.\", \"PMID- 20301317\\nSTAT- Publisher\\nDA  - 20100320\\nDRDT- 20190725\\nCTDT- 19981028\\nPB  - University of Washington, Seattle\\nDP  - 1993\\nTI  - Hereditary Ataxia Overview\\nBTI - GeneReviews((R))\\nAB  - CLINICAL CHARACTERISTICS: The hereditary ataxias are a group of genetic disorders\\n      characterized by slowly progressive incoordination of gait and often associated\\n      with poor coordination of hands, speech, and eye movements. Frequently, atrophy\\n      of the cerebellum occurs. In this GeneReview the hereditary ataxias are\\n      categorized by mode of inheritance and gene (or chromosome locus) in which\\n      pathogenic variants occur. DIAGNOSIS/TESTING: Inherited (genetic) forms of ataxia\\n      must be distinguished from the many acquired (non-genetic) causes of ataxia. The \\n      genetic forms of ataxia are diagnosed by family history, physical examination,\\n      neuroimaging, and molecular genetic testing. GENETIC COUNSELING: The hereditary\\n      ataxias can be inherited in an autosomal dominant, autosomal recessive, or\\n      X-linked manner. Genetic counseling and risk assessment depend on determination\\n      of the specific cause of an inherited ataxia in an individual. MANAGEMENT:\\n      Treatment of manifestations: Canes, walkers, and wheelchairs for gait ataxia; use\\n      of special devices to assist with handwriting, buttoning, and use of eating\\n      utensils; speech therapy and/or computer-based devices for those with dysarthria \\n      and severe speech deficits. Prevention of primary manifestations: With the\\n      exception of vitamin E therapy for ataxia with vitamin E deficiency (AVED), no\\n      specific treatments exist for hereditary ataxia.\\nCI  - Copyright (c) 1993-2019, University of Washington, Seattle. GeneReviews is a\\n      registered trademark of the University of Washington, Seattle. All rights\\n      reserved.\\nFED - Adam, Margaret P\\nED  - Adam MP\\nFED - Ardinger, Holly H\\nED  - Ardinger HH\\nFED - Pagon, Roberta A\\nED  - Pagon RA\\nFED - Wallace, Stephanie E\\nED  - Wallace SE\\nFED - Bean, Lora JH\\nED  - Bean LJH\\nFED - Stephens, Karen\\nED  - Stephens K\\nFED - Amemiya, Anne\\nED  - Amemiya A\\nFAU - Bird, Thomas D\\nAU  - Bird TD\\nLA  - eng\\nGR  - HHSN276201400262U/NLM NIH HHS/United States\\nPT  - Review\\nPT  - Book Chapter\\nPL  - Seattle (WA)\\nOTO - NLM\\nOT  - Aconitate hydratase, mitochondrial\\nOT  - AFG3-like protein 2\\nOT  - Alpha-tocopherol transfer protein\\nOT  - Aprataxin\\nOT  - Ataxin-1\\nOT  - Ataxin-10\\nOT  - Ataxin-2\\nOT  - Ataxin-3\\nOT  - Ataxin-7\\nOT  - Ataxin-8\\nOT  - Atrophin-1\\nOT  - Atypical kinase COQ8A, mitochondrial\\nOT  - Bifunctional polynucleotide phosphatase/kinase\\nOT  - Calpain-1 catalytic subunit\\nOT  - Ceroid-lipofuscinosis neuronal protein 5\\nOT  - CWF19-like protein 1\\nOT  - Decaprenyl-diphosphate synthase subunit 1\\nOT  - Decaprenyl-diphosphate synthase subunit 2\\nOT  - DNA-directed RNA polymerase III subunit RPC1\\nOT  - DNA-directed RNA polymerase III subunit RPC2\\nOT  - E3 ubiquitin-protein ligase CHIP\\nOT  - E3 ubiquitin-protein ligase RNF216\\nOT  - Elongation factor 2\\nOT  - Elongation factor Ts, mitochondrial\\nOT  - Elongation of very long chain fatty acids protein 4\\nOT  - Elongation of very long chain fatty acids protein 5\\nOT  - Excitatory amino acid transporter 1\\nOT  - Fibroblast growth factor 12\\nOT  - Fibroblast growth factor 14\\nOT  - Frataxin, mitochondrial\\nOT  - Ganglioside-induced differentiation-associated protein 2\\nOT  - Glutamate receptor ionotropic, delta-2\\nOT  - Inositol 1,4,5-trisphosphate receptor type 1\\nOT  - Laminin subunit alpha-1\\nOT  - Metabotropic glutamate receptor 1\\nOT  - Microtubule cross-linking factor 1\\nOT  - Neprilysin\\nOT  - Neuropathy target esterase\\nOT  - Nucleolar protein 56\\nOT  - Nucleotide exchange factor SIL1\\nOT  - Oligophrenin-1\\nOT  - Para-hydroxybenzoate--polyprenyltransferase, mitochondrial\\nOT  - Paraplegin\\nOT  - Peroxisomal targeting signal 2 receptor\\nOT  - Phospholipase D3\\nOT  - Phytanoyl-CoA dioxygenase, peroxisomal\\nOT  - Plasma membrane calcium-transporting ATPase 3\\nOT  - Potassium voltage-gated channel subfamily A member 1\\nOT  - Potassium voltage-gated channel subfamily C member 3\\nOT  - Potassium voltage-gated channel subfamily D member 3\\nOT  - Probable helicase senataxin\\nOT  - Proenkephalin-B\\nOT  - Protein BEAN1\\nOT  - Protein Daple\\nOT  - Protein kinase C gamma type\\nOT  - Protein-glutamine gamma-glutamyltransferase 6\\nOT  - Protocadherin Fat 2\\nOT  - Protocadherin-12\\nOT  - Pumilio homolog 1\\nOT  - Replication factor C subunit 1\\nOT  - Sacsin\\nOT  - Sequestosome-1\\nOT  - Serine/threonine-protein phosphatase 2A 55 kDa regulatory subunit B beta isoform\\nOT  - Serine-protein kinase ATM\\nOT  - Short transient receptor potential channel 3\\nOT  - Sodium/hydrogen exchanger 1\\nOT  - Sodium/hydrogen exchanger 6\\nOT  - Sodium/potassium-transporting ATPase subunit alpha-3\\nOT  - Solute carrier family 25 member 46\\nOT  - Solute carrier family 52, riboflavin transporter, member 2\\nOT  - Sorting nexin-14\\nOT  - Sorting nexin-4\\nOT  - Spectrin beta chain, non-erythrocytic 2\\nOT  - Sterol 26-hydroxylase, mitochondrial\\nOT  - TATA-box-binding protein\\nOT  - Tau-tubulin kinase 2\\nOT  - Transmembrane protein 240\\nOT  - Twinkle protein, mitochondrial\\nOT  - Tyrosyl-DNA phosphodiesterase 2\\nOT  - Ubiquinone biosynthesis protein COQ9, mitochondrial\\nOT  - Vacuolar protein sorting-associated protein 13D\\nOT  - Vesicle-associated membrane protein 1\\nOT  - Voltage-dependent L-type calcium channel subunit beta-4\\nOT  - Voltage-dependent P/Q-type calcium channel subunit alpha-1A\\nOT  - von Willebrand factor A domain-containing protein 3B\\nOT  - WD repeat-containing protein 73\\nOT  - Wolframin\\nOT  - ACO2\\nOT  - AFG3L2\\nOT  - APTX\\nOT  - ATM\\nOT  - ATN1\\nOT  - ATP1A3\\nOT  - ATP2B3\\nOT  - ATXN1\\nOT  - ATXN10\\nOT  - ATXN2\\nOT  - ATXN3\\nOT  - ATXN7\\nOT  - ATXN8\\nOT  - ATXN8OS\\nOT  - BEAN1\\nOT  - CACNA1A\\nOT  - CACNB4\\nOT  - CAPN1\\nOT  - CCDC88C\\nOT  - CLN5\\nOT  - COQ2\\nOT  - COQ8A\\nOT  - COQ9\\nOT  - CWF19L1\\nOT  - CYP27A1\\nOT  - EEF2\\nOT  - ELOVL4\\nOT  - ELOVL5\\nOT  - FAT2\\nOT  - FGF12\\nOT  - FGF14\\nOT  - FXN\\nOT  - GDAP2\\nOT  - GRID2\\nOT  - GRM1\\nOT  - ITPR1\\nOT  - KCNA1\\nOT  - KCNC3\\nOT  - KCND3\\nOT  - LAMA1\\nOT  - MME\\nOT  - MTCL1\\nOT  - NOP56\\nOT  - OPHN1\\nOT  - PCDH12\\nOT  - PDSS1\\nOT  - PDSS2\\nOT  - PDYN\\nOT  - PEX7\\nOT  - PHYH\\nOT  - PLD3\\nOT  - PNKP\\nOT  - PNPLA6\\nOT  - POLR3A\\nOT  - POLR3B\\nOT  - PPP2R2B\\nOT  - PRKCG\\nOT  - PUM1\\nOT  - RFC1\\nOT  - RNF216\\nOT  - SACS\\nOT  - SETX\\nOT  - SIL1\\nOT  - SLC1A3\\nOT  - SLC25A46\\nOT  - SLC52A2\\nOT  - SLC9A1\\nOT  - SLC9A6\\nOT  - SNX14\\nOT  - SNX4\\nOT  - SPG7\\nOT  - SPTBN2\\nOT  - SQSTM1\\nOT  - STUB1\\nOT  - TBP\\nOT  - TDP2\\nOT  - TGM6\\nOT  - TMEM240\\nOT  - TRPC3\\nOT  - TSFM\\nOT  - TTBK2\\nOT  - TTPA\\nOT  - TWNK\\nOT  - VAMP1\\nOT  - VPS13D\\nOT  - VWA3B\\nOT  - WDR73\\nOT  - WFS1\\nOT  - Hereditary Ataxia\\nOT  - Overview\\nEDAT- 2019/07/25 06:00\\nMHDA- 2019/07/25 06:00\\nCDAT- 2010/03/20 06:00\\nAID - NBK1138 [bookaccession]\", \"PMID- 19056867\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20090226\\nLR  - 20191003\\nIS  - 1533-3450 (Electronic)\\nIS  - 1046-6673 (Linking)\\nVI  - 20\\nIP  - 2\\nDP  - 2009 Feb\\nTI  - Large-scale proteomics and phosphoproteomics of urinary exosomes.\\nPG  - 363-79\\nLID - 10.1681/ASN.2008040406 [doi]\\nAB  - Normal human urine contains large numbers of exosomes, which are 40- to 100-nm\\n      vesicles that originate as the internal vesicles in multivesicular bodies from\\n      every renal epithelial cell type facing the urinary space. Here, we used LC-MS/MS\\n      to profile the proteome of human urinary exosomes. Overall, the analysis\\n      identified 1132 proteins unambiguously, including 177 that are represented on the\\n      Online Mendelian Inheritance in Man database of disease-related genes, suggesting\\n      that exosome analysis is a potential approach to discover urinary biomarkers. We \\n      extended the proteomic analysis to phosphoproteomic profiling using neutral loss \\n      scanning, and this yielded multiple novel phosphorylation sites, including\\n      serine-811 in the thiazide-sensitive Na-Cl co-transporter, NCC. To demonstrate\\n      the potential use of exosome analysis to identify a genetic renal disease, we\\n      carried out immunoblotting of exosomes from urine samples of patients with a\\n      clinical diagnosis of Bartter syndrome type I, showing an absence of the\\n      sodium-potassium-chloride co-transporter 2, NKCC2. The proteomic data are\\n      publicly accessible at http://dir.nhlbi.nih.gov/papers/lkem/exosome/.\\nFAU - Gonzales, Patricia A\\nAU  - Gonzales PA\\nAD  - Laboratory of Kidney and Electrolyte Metabolism, National Heart, Lung, and Blood \\n      Institute, National Institutes of Health, Bethesda, MD 20892, USA.\\nFAU - Pisitkun, Trairak\\nAU  - Pisitkun T\\nFAU - Hoffert, Jason D\\nAU  - Hoffert JD\\nFAU - Tchapyjnikov, Dmitry\\nAU  - Tchapyjnikov D\\nFAU - Star, Robert A\\nAU  - Star RA\\nFAU - Kleta, Robert\\nAU  - Kleta R\\nFAU - Wang, Nam Sun\\nAU  - Wang NS\\nFAU - Knepper, Mark A\\nAU  - Knepper MA\\nLA  - eng\\nGR  - Z01 HL001285-21/Intramural NIH HHS/United States\\nGR  - Z99 HL999999/Intramural NIH HHS/United States\\nPT  - Journal Article\\nDEP - 20081203\\nPL  - United States\\nTA  - J Am Soc Nephrol\\nJT  - Journal of the American Society of Nephrology : JASN\\nJID - 9013836\\nRN  - 0 (Phosphoproteins)\\nRN  - 0 (Proteome)\\nRN  - 0 (Thiazides)\\nRN  - EC 3.6.1.- (Adenosine Triphosphatases)\\nSB  - IM\\nMH  - Adenosine Triphosphatases/chemistry\\nMH  - Adult\\nMH  - Bartter Syndrome/urine\\nMH  - Chromatography, Liquid/methods\\nMH  - Exosomes/metabolism\\nMH  - Female\\nMH  - Humans\\nMH  - Male\\nMH  - Mass Spectrometry/methods\\nMH  - Phosphoproteins/chemistry\\nMH  - Phosphorylation\\nMH  - Proteome\\nMH  - Proteomics/*methods\\nMH  - Thiazides/chemistry\\nMH  - *Urine\\nPMC - PMC2637050\\nEDAT- 2008/12/06 09:00\\nMHDA- 2009/02/27 09:00\\nCRDT- 2008/12/06 09:00\\nPHST- 2008/12/06 09:00 [pubmed]\\nPHST- 2009/02/27 09:00 [medline]\\nPHST- 2008/12/06 09:00 [entrez]\\nAID - ASN.2008040406 [pii]\\nAID - 10.1681/ASN.2008040406 [doi]\\nPST - ppublish\\nSO  - J Am Soc Nephrol. 2009 Feb;20(2):363-79. doi: 10.1681/ASN.2008040406. Epub 2008\\n      Dec 3.\", \"PMID- 16710414\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20060614\\nLR  - 20170922\\nIS  - 1476-4687 (Electronic)\\nIS  - 0028-0836 (Linking)\\nVI  - 441\\nIP  - 7091\\nDP  - 2006 May 18\\nTI  - The DNA sequence and biological annotation of human chromosome 1.\\nPG  - 315-21\\nAB  - The reference sequence for each human chromosome provides the framework for\\n      understanding genome function, variation and evolution. Here we report the\\n      finished sequence and biological annotation of human chromosome 1. Chromosome 1\\n      is gene-dense, with 3,141 genes and 991 pseudogenes, and many coding sequences\\n      overlap. Rearrangements and mutations of chromosome 1 are prevalent in cancer and\\n      many other diseases. Patterns of sequence variation reveal signals of recent\\n      selection in specific genes that may contribute to human fitness, and also in\\n      regions where no function is evident. Fine-scale recombination occurs in hotspots\\n      of varying intensity along the sequence, and is enriched near genes. These and\\n      other studies of human biology and disease encoded within chromosome 1 are made\\n      possible with the highly accurate annotated sequence, as part of the completed\\n      set of chromosome sequences that comprise the reference human genome.\\nFAU - Gregory, S G\\nAU  - Gregory SG\\nAD  - The Wellcome Trust Sanger Institute, The Wellcome Trust Genome Campus, Hinxton,\\n      Cambridgeshire CB10 1SA, UK. sgregory@chg.duhs.duke.edu\\nFAU - Barlow, K F\\nAU  - Barlow KF\\nFAU - McLay, K E\\nAU  - McLay KE\\nFAU - Kaul, R\\nAU  - Kaul R\\nFAU - Swarbreck, D\\nAU  - Swarbreck D\\nFAU - Dunham, A\\nAU  - Dunham A\\nFAU - Scott, C E\\nAU  - Scott CE\\nFAU - Howe, K L\\nAU  - Howe KL\\nFAU - Woodfine, K\\nAU  - Woodfine K\\nFAU - Spencer, C C A\\nAU  - Spencer CC\\nFAU - Jones, M C\\nAU  - Jones MC\\nFAU - Gillson, C\\nAU  - Gillson C\\nFAU - Searle, S\\nAU  - Searle S\\nFAU - Zhou, Y\\nAU  - Zhou Y\\nFAU - Kokocinski, F\\nAU  - Kokocinski F\\nFAU - McDonald, L\\nAU  - McDonald L\\nFAU - Evans, R\\nAU  - Evans R\\nFAU - Phillips, K\\nAU  - Phillips K\\nFAU - Atkinson, A\\nAU  - Atkinson A\\nFAU - Cooper, R\\nAU  - Cooper R\\nFAU - Jones, C\\nAU  - Jones C\\nFAU - Hall, R E\\nAU  - Hall RE\\nFAU - Andrews, T D\\nAU  - Andrews TD\\nFAU - Lloyd, C\\nAU  - Lloyd C\\nFAU - Ainscough, R\\nAU  - Ainscough R\\nFAU - Almeida, J P\\nAU  - Almeida JP\\nFAU - Ambrose, K D\\nAU  - Ambrose KD\\nFAU - Anderson, F\\nAU  - Anderson F\\nFAU - Andrew, R W\\nAU  - Andrew RW\\nFAU - Ashwell, R I S\\nAU  - Ashwell RI\\nFAU - Aubin, K\\nAU  - Aubin K\\nFAU - Babbage, A K\\nAU  - Babbage AK\\nFAU - Bagguley, C L\\nAU  - Bagguley CL\\nFAU - Bailey, J\\nAU  - Bailey J\\nFAU - Beasley, H\\nAU  - Beasley H\\nFAU - Bethel, G\\nAU  - Bethel G\\nFAU - Bird, C P\\nAU  - Bird CP\\nFAU - Bray-Allen, S\\nAU  - Bray-Allen S\\nFAU - Brown, J Y\\nAU  - Brown JY\\nFAU - Brown, A J\\nAU  - Brown AJ\\nFAU - Buckley, D\\nAU  - Buckley D\\nFAU - Burton, J\\nAU  - Burton J\\nFAU - Bye, J\\nAU  - Bye J\\nFAU - Carder, C\\nAU  - Carder C\\nFAU - Chapman, J C\\nAU  - Chapman JC\\nFAU - Clark, S Y\\nAU  - Clark SY\\nFAU - Clarke, G\\nAU  - Clarke G\\nFAU - Clee, C\\nAU  - Clee C\\nFAU - Cobley, V\\nAU  - Cobley V\\nFAU - Collier, R E\\nAU  - Collier RE\\nFAU - Corby, N\\nAU  - Corby N\\nFAU - Coville, G J\\nAU  - Coville GJ\\nFAU - Davies, J\\nAU  - Davies J\\nFAU - Deadman, R\\nAU  - Deadman R\\nFAU - Dunn, M\\nAU  - Dunn M\\nFAU - Earthrowl, M\\nAU  - Earthrowl M\\nFAU - Ellington, A G\\nAU  - Ellington AG\\nFAU - Errington, H\\nAU  - Errington H\\nFAU - Frankish, A\\nAU  - Frankish A\\nFAU - Frankland, J\\nAU  - Frankland J\\nFAU - French, L\\nAU  - French L\\nFAU - Garner, P\\nAU  - Garner P\\nFAU - Garnett, J\\nAU  - Garnett J\\nFAU - Gay, L\\nAU  - Gay L\\nFAU - Ghori, M R J\\nAU  - Ghori MR\\nFAU - Gibson, R\\nAU  - Gibson R\\nFAU - Gilby, L M\\nAU  - Gilby LM\\nFAU - Gillett, W\\nAU  - Gillett W\\nFAU - Glithero, R J\\nAU  - Glithero RJ\\nFAU - Grafham, D V\\nAU  - Grafham DV\\nFAU - Griffiths, C\\nAU  - Griffiths C\\nFAU - Griffiths-Jones, S\\nAU  - Griffiths-Jones S\\nFAU - Grocock, R\\nAU  - Grocock R\\nFAU - Hammond, S\\nAU  - Hammond S\\nFAU - Harrison, E S I\\nAU  - Harrison ES\\nFAU - Hart, E\\nAU  - Hart E\\nFAU - Haugen, E\\nAU  - Haugen E\\nFAU - Heath, P D\\nAU  - Heath PD\\nFAU - Holmes, S\\nAU  - Holmes S\\nFAU - Holt, K\\nAU  - Holt K\\nFAU - Howden, P J\\nAU  - Howden PJ\\nFAU - Hunt, A R\\nAU  - Hunt AR\\nFAU - Hunt, S E\\nAU  - Hunt SE\\nFAU - Hunter, G\\nAU  - Hunter G\\nFAU - Isherwood, J\\nAU  - Isherwood J\\nFAU - James, R\\nAU  - James R\\nFAU - Johnson, C\\nAU  - Johnson C\\nFAU - Johnson, D\\nAU  - Johnson D\\nFAU - Joy, A\\nAU  - Joy A\\nFAU - Kay, M\\nAU  - Kay M\\nFAU - Kershaw, J K\\nAU  - Kershaw JK\\nFAU - Kibukawa, M\\nAU  - Kibukawa M\\nFAU - Kimberley, A M\\nAU  - Kimberley AM\\nFAU - King, A\\nAU  - King A\\nFAU - Knights, A J\\nAU  - Knights AJ\\nFAU - Lad, H\\nAU  - Lad H\\nFAU - Laird, G\\nAU  - Laird G\\nFAU - Lawlor, S\\nAU  - Lawlor S\\nFAU - Leongamornlert, D A\\nAU  - Leongamornlert DA\\nFAU - Lloyd, D M\\nAU  - Lloyd DM\\nFAU - Loveland, J\\nAU  - Loveland J\\nFAU - Lovell, J\\nAU  - Lovell J\\nFAU - Lush, M J\\nAU  - Lush MJ\\nFAU - Lyne, R\\nAU  - Lyne R\\nFAU - Martin, S\\nAU  - Martin S\\nFAU - Mashreghi-Mohammadi, M\\nAU  - Mashreghi-Mohammadi M\\nFAU - Matthews, L\\nAU  - Matthews L\\nFAU - Matthews, N S W\\nAU  - Matthews NS\\nFAU - McLaren, S\\nAU  - McLaren S\\nFAU - Milne, S\\nAU  - Milne S\\nFAU - Mistry, S\\nAU  - Mistry S\\nFAU - Moore, M J F\\nAU  - Moore MJ\\nFAU - Nickerson, T\\nAU  - Nickerson T\\nFAU - O'Dell, C N\\nAU  - O'Dell CN\\nFAU - Oliver, K\\nAU  - Oliver K\\nFAU - Palmeiri, A\\nAU  - Palmeiri A\\nFAU - Palmer, S A\\nAU  - Palmer SA\\nFAU - Parker, A\\nAU  - Parker A\\nFAU - Patel, D\\nAU  - Patel D\\nFAU - Pearce, A V\\nAU  - Pearce AV\\nFAU - Peck, A I\\nAU  - Peck AI\\nFAU - Pelan, S\\nAU  - Pelan S\\nFAU - Phelps, K\\nAU  - Phelps K\\nFAU - Phillimore, B J\\nAU  - Phillimore BJ\\nFAU - Plumb, R\\nAU  - Plumb R\\nFAU - Rajan, J\\nAU  - Rajan J\\nFAU - Raymond, C\\nAU  - Raymond C\\nFAU - Rouse, G\\nAU  - Rouse G\\nFAU - Saenphimmachak, C\\nAU  - Saenphimmachak C\\nFAU - Sehra, H K\\nAU  - Sehra HK\\nFAU - Sheridan, E\\nAU  - Sheridan E\\nFAU - Shownkeen, R\\nAU  - Shownkeen R\\nFAU - Sims, S\\nAU  - Sims S\\nFAU - Skuce, C D\\nAU  - Skuce CD\\nFAU - Smith, M\\nAU  - Smith M\\nFAU - Steward, C\\nAU  - Steward C\\nFAU - Subramanian, S\\nAU  - Subramanian S\\nFAU - Sycamore, N\\nAU  - Sycamore N\\nFAU - Tracey, A\\nAU  - Tracey A\\nFAU - Tromans, A\\nAU  - Tromans A\\nFAU - Van Helmond, Z\\nAU  - Van Helmond Z\\nFAU - Wall, M\\nAU  - Wall M\\nFAU - Wallis, J M\\nAU  - Wallis JM\\nFAU - White, S\\nAU  - White S\\nFAU - Whitehead, S L\\nAU  - Whitehead SL\\nFAU - Wilkinson, J E\\nAU  - Wilkinson JE\\nFAU - Willey, D L\\nAU  - Willey DL\\nFAU - Williams, H\\nAU  - Williams H\\nFAU - Wilming, L\\nAU  - Wilming L\\nFAU - Wray, P W\\nAU  - Wray PW\\nFAU - Wu, Z\\nAU  - Wu Z\\nFAU - Coulson, A\\nAU  - Coulson A\\nFAU - Vaudin, M\\nAU  - Vaudin M\\nFAU - Sulston, J E\\nAU  - Sulston JE\\nFAU - Durbin, R\\nAU  - Durbin R\\nFAU - Hubbard, T\\nAU  - Hubbard T\\nFAU - Wooster, R\\nAU  - Wooster R\\nFAU - Dunham, I\\nAU  - Dunham I\\nFAU - Carter, N P\\nAU  - Carter NP\\nFAU - McVean, G\\nAU  - McVean G\\nFAU - Ross, M T\\nAU  - Ross MT\\nFAU - Harrow, J\\nAU  - Harrow J\\nFAU - Olson, M V\\nAU  - Olson MV\\nFAU - Beck, S\\nAU  - Beck S\\nFAU - Rogers, J\\nAU  - Rogers J\\nFAU - Bentley, D R\\nAU  - Bentley DR\\nFAU - Banerjee, R\\nAU  - Banerjee R\\nFAU - Bryant, S P\\nAU  - Bryant SP\\nFAU - Burford, D C\\nAU  - Burford DC\\nFAU - Burrill, W D H\\nAU  - Burrill WD\\nFAU - Clegg, S M\\nAU  - Clegg SM\\nFAU - Dhami, P\\nAU  - Dhami P\\nFAU - Dovey, O\\nAU  - Dovey O\\nFAU - Faulkner, L M\\nAU  - Faulkner LM\\nFAU - Gribble, S M\\nAU  - Gribble SM\\nFAU - Langford, C F\\nAU  - Langford CF\\nFAU - Pandian, R D\\nAU  - Pandian RD\\nFAU - Porter, K M\\nAU  - Porter KM\\nFAU - Prigmore, E\\nAU  - Prigmore E\\nLA  - eng\\nSI  - RefSeq/NC_000001\\nGR  - G0000107/Medical Research Council/United Kingdom\\nGR  - Wellcome Trust/United Kingdom\\nPT  - Journal Article\\nPT  - Research Support, N.I.H., Extramural\\nPT  - Research Support, Non-U.S. Gov't\\nPL  - England\\nTA  - Nature\\nJT  - Nature\\nJID - 0410462\\nSB  - IM\\nEIN - Nature. 2006 Oct 26;443(7114):1013. Banerjee, R [added]; Bryant, SP [added];\\n      Burford, DC [added]; Burrill, WDH [added]; Clegg, SM [added]; Dhami, P [added];\\n      Dovey, O [added]; Faulkner, LM [added]; Gribble, SM [added]; Langford, CF\\n      [added]; Pandian, RD [added]; Porter, KM [added]; Prigmore, E\\nMH  - Base Sequence\\nMH  - Chromosomes, Human, Pair 1/*genetics\\nMH  - DNA Replication Timing\\nMH  - Disease\\nMH  - Gene Duplication\\nMH  - Genes/genetics\\nMH  - Genetic Variation/genetics\\nMH  - Genomics\\nMH  - Humans\\nMH  - Molecular Sequence Data\\nMH  - Open Reading Frames/genetics\\nMH  - Pseudogenes/genetics\\nMH  - Recombination, Genetic/genetics\\nMH  - Selection, Genetic\\nMH  - Sequence Analysis, DNA\\nEDAT- 2006/05/20 09:00\\nMHDA- 2006/06/15 09:00\\nCRDT- 2006/05/20 09:00\\nPHST- 2005/12/24 00:00 [received]\\nPHST- 2006/03/13 00:00 [accepted]\\nPHST- 2006/05/20 09:00 [pubmed]\\nPHST- 2006/06/15 09:00 [medline]\\nPHST- 2006/05/20 09:00 [entrez]\\nAID - nature04727 [pii]\\nAID - 10.1038/nature04727 [doi]\\nPST - ppublish\\nSO  - Nature. 2006 May 18;441(7091):315-21. doi: 10.1038/nature04727.\", \"PMID- 15498460\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20050124\\nLR  - 20070516\\nIS  - 0888-7543 (Print)\\nIS  - 0888-7543 (Linking)\\nVI  - 84\\nIP  - 3\\nDP  - 2004 Sep\\nTI  - Analysis of the human VPS13 gene family.\\nPG  - 536-49\\nAB  - The gene mutated in chorea-acanthocytosis (CHAC; approved gene symbol VPS13A)\\n      encodes chorein, a protein similar to yeast Vps13p. We detected several similar\\n      putative human proteins by BLAST analysis of chorein. We characterized the\\n      structure of three new genes encoding these CHAC-similar proteins, located on\\n      chromosomes 1p36, 8q22, and 15q21. The most similar gene in yeast to all four\\n      human genes is Vps13, and therefore the human genes were named VPS13A (CHAC,\\n      9q21), VPS13B (8q22), VPS13C (15q21), and VPS13D (1p36). VPS13B has recently been\\n      reported as COH1, altered in Cohen syndrome. For each gene, we describe several\\n      alternative splicing variants; at least two transcripts per gene are major forms.\\n      The expression pattern of these genes is ubiquitous, with some tissue-specific\\n      differences between several transcript variants. Protein sequence comparisons\\n      suggest that intramolecular duplications have played an important role in the\\n      evolution of this gene family.\\nFAU - Velayos-Baeza, Antonio\\nAU  - Velayos-Baeza A\\nAD  - Wellcome Trust Centre for Human Genetics, University of Oxford, Headington, OX3\\n      7BN Oxford, UK.\\nFAU - Vettori, Andrea\\nAU  - Vettori A\\nFAU - Copley, Richard R\\nAU  - Copley RR\\nFAU - Dobson-Stone, Carol\\nAU  - Dobson-Stone C\\nFAU - Monaco, A P\\nAU  - Monaco AP\\nLA  - eng\\nSI  - GENBANK/AJ608769\\nSI  - GENBANK/AJ608770\\nSI  - GENBANK/AJ608771\\nSI  - GENBANK/AJ608772\\nSI  - GENBANK/AJ608773\\nSI  - GENBANK/AJ608774\\nSI  - GENBANK/AJ608775\\nSI  - GENBANK/AJ626859\\nSI  - GENBANK/AJ626860\\nSI  - GENBANK/AJ626861\\nPT  - Comparative Study\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov't\\nPL  - United States\\nTA  - Genomics\\nJT  - Genomics\\nJID - 8800135\\nRN  - 0 (Membrane Proteins)\\nRN  - 0 (Proteins)\\nRN  - 0 (VPS13A protein, human)\\nRN  - 0 (VPS13B protein, human)\\nRN  - 0 (VPS13C protein, human)\\nRN  - 0 (VPS13D protein, human)\\nRN  - 0 (Vesicular Transport Proteins)\\nSB  - IM\\nMH  - Alternative Splicing/*genetics\\nMH  - Amino Acid Sequence\\nMH  - Base Sequence\\nMH  - Blotting, Northern\\nMH  - Evolution, Molecular\\nMH  - Gene Components\\nMH  - Gene Duplication\\nMH  - Gene Expression Profiling\\nMH  - Humans\\nMH  - Membrane Proteins/*genetics\\nMH  - Molecular Sequence Data\\nMH  - Multigene Family/*genetics\\nMH  - Phylogeny\\nMH  - Proteins/*genetics\\nMH  - Reverse Transcriptase Polymerase Chain Reaction\\nMH  - Sequence Alignment\\nMH  - Sequence Analysis, DNA\\nMH  - Sequence Homology\\nMH  - Vesicular Transport Proteins\\nEDAT- 2004/10/23 09:00\\nMHDA- 2005/01/26 09:00\\nCRDT- 2004/10/23 09:00\\nPHST- 2004/02/13 00:00 [received]\\nPHST- 2004/04/19 00:00 [accepted]\\nPHST- 2004/10/23 09:00 [pubmed]\\nPHST- 2005/01/26 09:00 [medline]\\nPHST- 2004/10/23 09:00 [entrez]\\nAID - S0888754304001156 [pii]\\nAID - 10.1016/j.ygeno.2004.04.012 [doi]\\nPST - ppublish\\nSO  - Genomics. 2004 Sep;84(3):536-49. doi: 10.1016/j.ygeno.2004.04.012.\", \"PMID- 15489334\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20041116\\nLR  - 20181113\\nIS  - 1088-9051 (Print)\\nIS  - 1088-9051 (Linking)\\nVI  - 14\\nIP  - 10B\\nDP  - 2004 Oct\\nTI  - The status, quality, and expansion of the NIH full-length cDNA project: the\\n      Mammalian Gene Collection (MGC).\\nPG  - 2121-7\\nAB  - The National Institutes of Health's Mammalian Gene Collection (MGC) project was\\n      designed to generate and sequence a publicly accessible cDNA resource containing \\n      a complete open reading frame (ORF) for every human and mouse gene. The project\\n      initially used a random strategy to select clones from a large number of cDNA\\n      libraries from diverse tissues. Candidate clones were chosen based on 5'-EST\\n      sequences, and then fully sequenced to high accuracy and analyzed by algorithms\\n      developed for this project. Currently, more than 11,000 human and 10,000 mouse\\n      genes are represented in MGC by at least one clone with a full ORF. The random\\n      selection approach is now reaching a saturation point, and a transition to\\n      protocols targeted at the missing transcripts is now required to complete the\\n      mouse and human collections. Comparison of the sequence of the MGC clones to\\n      reference genome sequences reveals that most cDNA clones are of very high\\n      sequence quality, although it is likely that some cDNAs may carry missense\\n      variants as a consequence of experimental artifact, such as PCR, cloning, or\\n      reverse transcriptase errors. Recently, a rat cDNA component was added to the\\n      project, and ongoing frog (Xenopus) and zebrafish (Danio) cDNA projects were\\n      expanded to take advantage of the high-throughput MGC pipeline.\\nFAU - Gerhard, Daniela S\\nAU  - Gerhard DS\\nFAU - Wagner, Lukas\\nAU  - Wagner L\\nFAU - Feingold, Elise A\\nAU  - Feingold EA\\nFAU - Shenmen, Carolyn M\\nAU  - Shenmen CM\\nFAU - Grouse, Lynette H\\nAU  - Grouse LH\\nFAU - Schuler, Greg\\nAU  - Schuler G\\nFAU - Klein, Steven L\\nAU  - Klein SL\\nFAU - Old, Susan\\nAU  - Old S\\nFAU - Rasooly, Rebekah\\nAU  - Rasooly R\\nFAU - Good, Peter\\nAU  - Good P\\nFAU - Guyer, Mark\\nAU  - Guyer M\\nFAU - Peck, Allison M\\nAU  - Peck AM\\nFAU - Derge, Jeffery G\\nAU  - Derge JG\\nFAU - Lipman, David\\nAU  - Lipman D\\nFAU - Collins, Francis S\\nAU  - Collins FS\\nFAU - Jang, Wonhee\\nAU  - Jang W\\nFAU - Sherry, Steven\\nAU  - Sherry S\\nFAU - Feolo, Mike\\nAU  - Feolo M\\nFAU - Misquitta, Leonie\\nAU  - Misquitta L\\nFAU - Lee, Eduardo\\nAU  - Lee E\\nFAU - Rotmistrovsky, Kirill\\nAU  - Rotmistrovsky K\\nFAU - Greenhut, Susan F\\nAU  - Greenhut SF\\nFAU - Schaefer, Carl F\\nAU  - Schaefer CF\\nFAU - Buetow, Kenneth\\nAU  - Buetow K\\nFAU - Bonner, Tom I\\nAU  - Bonner TI\\nFAU - Haussler, David\\nAU  - Haussler D\\nFAU - Kent, Jim\\nAU  - Kent J\\nFAU - Kiekhaus, Mark\\nAU  - Kiekhaus M\\nFAU - Furey, Terry\\nAU  - Furey T\\nFAU - Brent, Michael\\nAU  - Brent M\\nFAU - Prange, Christa\\nAU  - Prange C\\nFAU - Schreiber, Kirsten\\nAU  - Schreiber K\\nFAU - Shapiro, Nicole\\nAU  - Shapiro N\\nFAU - Bhat, Narayan K\\nAU  - Bhat NK\\nFAU - Hopkins, Ralph F\\nAU  - Hopkins RF\\nFAU - Hsie, Florence\\nAU  - Hsie F\\nFAU - Driscoll, Tom\\nAU  - Driscoll T\\nFAU - Soares, M Bento\\nAU  - Soares MB\\nFAU - Casavant, Tom L\\nAU  - Casavant TL\\nFAU - Scheetz, Todd E\\nAU  - Scheetz TE\\nFAU - Brown-stein, Michael J\\nAU  - Brown-stein MJ\\nFAU - Usdin, Ted B\\nAU  - Usdin TB\\nFAU - Toshiyuki, Shiraki\\nAU  - Toshiyuki S\\nFAU - Carninci, Piero\\nAU  - Carninci P\\nFAU - Piao, Yulan\\nAU  - Piao Y\\nFAU - Dudekula, Dawood B\\nAU  - Dudekula DB\\nFAU - Ko, Minoru S H\\nAU  - Ko MS\\nFAU - Kawakami, Koichi\\nAU  - Kawakami K\\nFAU - Suzuki, Yutaka\\nAU  - Suzuki Y\\nFAU - Sugano, Sumio\\nAU  - Sugano S\\nFAU - Gruber, C E\\nAU  - Gruber CE\\nFAU - Smith, M R\\nAU  - Smith MR\\nFAU - Simmons, Blake\\nAU  - Simmons B\\nFAU - Moore, Troy\\nAU  - Moore T\\nFAU - Waterman, Richard\\nAU  - Waterman R\\nFAU - Johnson, Stephen L\\nAU  - Johnson SL\\nFAU - Ruan, Yijun\\nAU  - Ruan Y\\nFAU - Wei, Chia Lin\\nAU  - Wei CL\\nFAU - Mathavan, S\\nAU  - Mathavan S\\nFAU - Gunaratne, Preethi H\\nAU  - Gunaratne PH\\nFAU - Wu, Jiaqian\\nAU  - Wu J\\nFAU - Garcia, Angela M\\nAU  - Garcia AM\\nFAU - Hulyk, Stephen W\\nAU  - Hulyk SW\\nFAU - Fuh, Edwin\\nAU  - Fuh E\\nFAU - Yuan, Ye\\nAU  - Yuan Y\\nFAU - Sneed, Anna\\nAU  - Sneed A\\nFAU - Kowis, Carla\\nAU  - Kowis C\\nFAU - Hodgson, Anne\\nAU  - Hodgson A\\nFAU - Muzny, Donna M\\nAU  - Muzny DM\\nFAU - McPherson, John\\nAU  - McPherson J\\nFAU - Gibbs, Richard A\\nAU  - Gibbs RA\\nFAU - Fahey, Jessica\\nAU  - Fahey J\\nFAU - Helton, Erin\\nAU  - Helton E\\nFAU - Ketteman, Mark\\nAU  - Ketteman M\\nFAU - Madan, Anuradha\\nAU  - Madan A\\nFAU - Rodrigues, Stephanie\\nAU  - Rodrigues S\\nFAU - Sanchez, Amy\\nAU  - Sanchez A\\nFAU - Whiting, Michelle\\nAU  - Whiting M\\nFAU - Madari, Anup\\nAU  - Madari A\\nFAU - Young, Alice C\\nAU  - Young AC\\nFAU - Wetherby, Keith D\\nAU  - Wetherby KD\\nFAU - Granite, Steven J\\nAU  - Granite SJ\\nFAU - Kwong, Peggy N\\nAU  - Kwong PN\\nFAU - Brinkley, Charles P\\nAU  - Brinkley CP\\nFAU - Pearson, Russell L\\nAU  - Pearson RL\\nFAU - Bouffard, Gerard G\\nAU  - Bouffard GG\\nFAU - Blakesly, Robert W\\nAU  - Blakesly RW\\nFAU - Green, Eric D\\nAU  - Green ED\\nFAU - Dickson, Mark C\\nAU  - Dickson MC\\nFAU - Rodriguez, Alex C\\nAU  - Rodriguez AC\\nFAU - Grimwood, Jane\\nAU  - Grimwood J\\nFAU - Schmutz, Jeremy\\nAU  - Schmutz J\\nFAU - Myers, Richard M\\nAU  - Myers RM\\nFAU - Butterfield, Yaron S N\\nAU  - Butterfield YS\\nFAU - Griffith, Malachi\\nAU  - Griffith M\\nFAU - Griffith, Obi L\\nAU  - Griffith OL\\nFAU - Krzywinski, Martin I\\nAU  - Krzywinski MI\\nFAU - Liao, Nancy\\nAU  - Liao N\\nFAU - Morin, Ryan\\nAU  - Morin R\\nFAU - Palmquist, Diana\\nAU  - Palmquist D\\nFAU - Petrescu, Anca S\\nAU  - Petrescu AS\\nFAU - Skalska, Ursula\\nAU  - Skalska U\\nFAU - Smailus, Duane E\\nAU  - Smailus DE\\nFAU - Stott, Jeff M\\nAU  - Stott JM\\nFAU - Schnerch, Angelique\\nAU  - Schnerch A\\nFAU - Schein, Jacqueline E\\nAU  - Schein JE\\nFAU - Jones, Steven J M\\nAU  - Jones SJ\\nFAU - Holt, Robert A\\nAU  - Holt RA\\nFAU - Baross, Agnes\\nAU  - Baross A\\nFAU - Marra, Marco A\\nAU  - Marra MA\\nFAU - Clifton, Sandra\\nAU  - Clifton S\\nFAU - Makowski, Kathryn A\\nAU  - Makowski KA\\nFAU - Bosak, Stephanie\\nAU  - Bosak S\\nFAU - Malek, Joel\\nAU  - Malek J\\nCN  - MGC Project Team\\nLA  - eng\\nGR  - N01-C0-12400/PHS HHS/United States\\nPT  - Comparative Study\\nPT  - Journal Article\\nPT  - Research Support, U.S. Gov't, P.H.S.\\nPL  - United States\\nTA  - Genome Res\\nJT  - Genome research\\nJID - 9518021\\nRN  - 0 (DNA Primers)\\nRN  - 0 (DNA, Complementary)\\nSB  - IM\\nEIN - Genome Res. 2006 Jun;16(6):804. Morrin, Ryan [corrected to Morin, Ryan]\\nMH  - Animals\\nMH  - Cloning, Molecular/*methods\\nMH  - Computational Biology\\nMH  - DNA Primers\\nMH  - *DNA, Complementary/genetics/metabolism\\nMH  - *Gene Library\\nMH  - Humans\\nMH  - Mice\\nMH  - National Institutes of Health (U.S.)\\nMH  - Open Reading Frames/*physiology\\nMH  - Rats\\nMH  - United States\\nMH  - Xenopus laevis/genetics\\nMH  - Zebrafish/genetics\\nPMC - PMC528928\\nEDAT- 2004/10/19 09:00\\nMHDA- 2004/11/17 09:00\\nCRDT- 2004/10/19 09:00\\nPHST- 2004/10/19 09:00 [pubmed]\\nPHST- 2004/11/17 09:00 [medline]\\nPHST- 2004/10/19 09:00 [entrez]\\nAID - 14/10b/2121 [pii]\\nAID - 10.1101/gr.2596504 [doi]\\nPST - ppublish\\nSO  - Genome Res. 2004 Oct;14(10B):2121-7. doi: 10.1101/gr.2596504.\", \"PMID- 14702039\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20040202\\nLR  - 20061115\\nIS  - 1061-4036 (Print)\\nIS  - 1061-4036 (Linking)\\nVI  - 36\\nIP  - 1\\nDP  - 2004 Jan\\nTI  - Complete sequencing and characterization of 21,243 full-length human cDNAs.\\nPG  - 40-5\\nAB  - As a base for human transcriptome and functional genomics, we created the\\n      \\\"full-length long Japan\\\" (FLJ) collection of sequenced human cDNAs. We determined\\n      the entire sequence of 21,243 selected clones and found that 14,490 cDNAs (10,897\\n      clusters) were unique to the FLJ collection. About half of them (5,416) seemed to\\n      be protein-coding. Of those, 1,999 clusters had not been predicted by\\n      computational methods. The distribution of GC content of nonpredicted cDNAs had a\\n      peak at approximately 58% compared with a peak at approximately 42%for predicted \\n      cDNAs. Thus, there seems to be a slight bias against GC-rich transcripts in\\n      current gene prediction procedures. The rest of the cDNAs unique to the FLJ\\n      collection (5,481) contained no obvious open reading frames (ORFs) and thus are\\n      candidate noncoding RNAs. About one-fourth of them (1,378) showed a clear pattern\\n      of splicing. The distribution of GC content of noncoding cDNAs was narrow and had\\n      a peak at approximately 42%, relatively low compared with that of protein-coding \\n      cDNAs.\\nFAU - Ota, Toshio\\nAU  - Ota T\\nAD  - Helix Research Institute, 1532-3 Yana, Kisarazu, Chiba 292-0812, Japan.\\nFAU - Suzuki, Yutaka\\nAU  - Suzuki Y\\nFAU - Nishikawa, Tetsuo\\nAU  - Nishikawa T\\nFAU - Otsuki, Tetsuji\\nAU  - Otsuki T\\nFAU - Sugiyama, Tomoyasu\\nAU  - Sugiyama T\\nFAU - Irie, Ryotaro\\nAU  - Irie R\\nFAU - Wakamatsu, Ai\\nAU  - Wakamatsu A\\nFAU - Hayashi, Koji\\nAU  - Hayashi K\\nFAU - Sato, Hiroyuki\\nAU  - Sato H\\nFAU - Nagai, Keiichi\\nAU  - Nagai K\\nFAU - Kimura, Kouichi\\nAU  - Kimura K\\nFAU - Makita, Hiroshi\\nAU  - Makita H\\nFAU - Sekine, Mitsuo\\nAU  - Sekine M\\nFAU - Obayashi, Masaya\\nAU  - Obayashi M\\nFAU - Nishi, Tatsunari\\nAU  - Nishi T\\nFAU - Shibahara, Toshikazu\\nAU  - Shibahara T\\nFAU - Tanaka, Toshihiro\\nAU  - Tanaka T\\nFAU - Ishii, Shizuko\\nAU  - Ishii S\\nFAU - Yamamoto, Jun-ichi\\nAU  - Yamamoto J\\nFAU - Saito, Kaoru\\nAU  - Saito K\\nFAU - Kawai, Yuri\\nAU  - Kawai Y\\nFAU - Isono, Yuko\\nAU  - Isono Y\\nFAU - Nakamura, Yoshitaka\\nAU  - Nakamura Y\\nFAU - Nagahari, Kenji\\nAU  - Nagahari K\\nFAU - Murakami, Katsuhiko\\nAU  - Murakami K\\nFAU - Yasuda, Tomohiro\\nAU  - Yasuda T\\nFAU - Iwayanagi, Takao\\nAU  - Iwayanagi T\\nFAU - Wagatsuma, Masako\\nAU  - Wagatsuma M\\nFAU - Shiratori, Akiko\\nAU  - Shiratori A\\nFAU - Sudo, Hiroaki\\nAU  - Sudo H\\nFAU - Hosoiri, Takehiko\\nAU  - Hosoiri T\\nFAU - Kaku, Yoshiko\\nAU  - Kaku Y\\nFAU - Kodaira, Hiroyo\\nAU  - Kodaira H\\nFAU - Kondo, Hiroshi\\nAU  - Kondo H\\nFAU - Sugawara, Masanori\\nAU  - Sugawara M\\nFAU - Takahashi, Makiko\\nAU  - Takahashi M\\nFAU - Kanda, Katsuhiro\\nAU  - Kanda K\\nFAU - Yokoi, Takahide\\nAU  - Yokoi T\\nFAU - Furuya, Takako\\nAU  - Furuya T\\nFAU - Kikkawa, Emiko\\nAU  - Kikkawa E\\nFAU - Omura, Yuhi\\nAU  - Omura Y\\nFAU - Abe, Kumi\\nAU  - Abe K\\nFAU - Kamihara, Kumiko\\nAU  - Kamihara K\\nFAU - Katsuta, Naoko\\nAU  - Katsuta N\\nFAU - Sato, Kazuomi\\nAU  - Sato K\\nFAU - Tanikawa, Machiko\\nAU  - Tanikawa M\\nFAU - Yamazaki, Makoto\\nAU  - Yamazaki M\\nFAU - Ninomiya, Ken\\nAU  - Ninomiya K\\nFAU - Ishibashi, Tadashi\\nAU  - Ishibashi T\\nFAU - Yamashita, Hiromichi\\nAU  - Yamashita H\\nFAU - Murakawa, Katsuji\\nAU  - Murakawa K\\nFAU - Fujimori, Kiyoshi\\nAU  - Fujimori K\\nFAU - Tanai, Hiroyuki\\nAU  - Tanai H\\nFAU - Kimata, Manabu\\nAU  - Kimata M\\nFAU - Watanabe, Motoji\\nAU  - Watanabe M\\nFAU - Hiraoka, Susumu\\nAU  - Hiraoka S\\nFAU - Chiba, Yoshiyuki\\nAU  - Chiba Y\\nFAU - Ishida, Shinichi\\nAU  - Ishida S\\nFAU - Ono, Yukio\\nAU  - Ono Y\\nFAU - Takiguchi, Sumiyo\\nAU  - Takiguchi S\\nFAU - Watanabe, Susumu\\nAU  - Watanabe S\\nFAU - Yosida, Makoto\\nAU  - Yosida M\\nFAU - Hotuta, Tomoko\\nAU  - Hotuta T\\nFAU - Kusano, Junko\\nAU  - Kusano J\\nFAU - Kanehori, Keiichi\\nAU  - Kanehori K\\nFAU - Takahashi-Fujii, Asako\\nAU  - Takahashi-Fujii A\\nFAU - Hara, Hiroto\\nAU  - Hara H\\nFAU - Tanase, Tomo-o\\nAU  - Tanase TO\\nFAU - Nomura, Yoshiko\\nAU  - Nomura Y\\nFAU - Togiya, Sakae\\nAU  - Togiya S\\nFAU - Komai, Fukuyo\\nAU  - Komai F\\nFAU - Hara, Reiko\\nAU  - Hara R\\nFAU - Takeuchi, Kazuha\\nAU  - Takeuchi K\\nFAU - Arita, Miho\\nAU  - Arita M\\nFAU - Imose, Nobuyuki\\nAU  - Imose N\\nFAU - Musashino, Kaoru\\nAU  - Musashino K\\nFAU - Yuuki, Hisatsugu\\nAU  - Yuuki H\\nFAU - Oshima, Atsushi\\nAU  - Oshima A\\nFAU - Sasaki, Naokazu\\nAU  - Sasaki N\\nFAU - Aotsuka, Satoshi\\nAU  - Aotsuka S\\nFAU - Yoshikawa, Yoko\\nAU  - Yoshikawa Y\\nFAU - Matsunawa, Hiroshi\\nAU  - Matsunawa H\\nFAU - Ichihara, Tatsuo\\nAU  - Ichihara T\\nFAU - Shiohata, Namiko\\nAU  - Shiohata N\\nFAU - Sano, Sanae\\nAU  - Sano S\\nFAU - Moriya, Shogo\\nAU  - Moriya S\\nFAU - Momiyama, Hiroko\\nAU  - Momiyama H\\nFAU - Satoh, Noriko\\nAU  - Satoh N\\nFAU - Takami, Sachiko\\nAU  - Takami S\\nFAU - Terashima, Yuko\\nAU  - Terashima Y\\nFAU - Suzuki, Osamu\\nAU  - Suzuki O\\nFAU - Nakagawa, Satoshi\\nAU  - Nakagawa S\\nFAU - Senoh, Akihiro\\nAU  - Senoh A\\nFAU - Mizoguchi, Hiroshi\\nAU  - Mizoguchi H\\nFAU - Goto, Yoshihiro\\nAU  - Goto Y\\nFAU - Shimizu, Fumio\\nAU  - Shimizu F\\nFAU - Wakebe, Hirokazu\\nAU  - Wakebe H\\nFAU - Hishigaki, Haretsugu\\nAU  - Hishigaki H\\nFAU - Watanabe, Takeshi\\nAU  - Watanabe T\\nFAU - Sugiyama, Akio\\nAU  - Sugiyama A\\nFAU - Takemoto, Makoto\\nAU  - Takemoto M\\nFAU - Kawakami, Bunsei\\nAU  - Kawakami B\\nFAU - Yamazaki, Masaaki\\nAU  - Yamazaki M\\nFAU - Watanabe, Koji\\nAU  - Watanabe K\\nFAU - Kumagai, Ayako\\nAU  - Kumagai A\\nFAU - Itakura, Shoko\\nAU  - Itakura S\\nFAU - Fukuzumi, Yasuhito\\nAU  - Fukuzumi Y\\nFAU - Fujimori, Yoshifumi\\nAU  - Fujimori Y\\nFAU - Komiyama, Megumi\\nAU  - Komiyama M\\nFAU - Tashiro, Hiroyuki\\nAU  - Tashiro H\\nFAU - Tanigami, Akira\\nAU  - Tanigami A\\nFAU - Fujiwara, Tsutomu\\nAU  - Fujiwara T\\nFAU - Ono, Toshihide\\nAU  - Ono T\\nFAU - Yamada, Katsue\\nAU  - Yamada K\\nFAU - Fujii, Yuka\\nAU  - Fujii Y\\nFAU - Ozaki, Kouichi\\nAU  - Ozaki K\\nFAU - Hirao, Maasa\\nAU  - Hirao M\\nFAU - Ohmori, Yoshihiro\\nAU  - Ohmori Y\\nFAU - Kawabata, Ayako\\nAU  - Kawabata A\\nFAU - Hikiji, Takeshi\\nAU  - Hikiji T\\nFAU - Kobatake, Naoko\\nAU  - Kobatake N\\nFAU - Inagaki, Hiromi\\nAU  - Inagaki H\\nFAU - Ikema, Yasuko\\nAU  - Ikema Y\\nFAU - Okamoto, Sachiko\\nAU  - Okamoto S\\nFAU - Okitani, Rie\\nAU  - Okitani R\\nFAU - Kawakami, Takuma\\nAU  - Kawakami T\\nFAU - Noguchi, Saori\\nAU  - Noguchi S\\nFAU - Itoh, Tomoko\\nAU  - Itoh T\\nFAU - Shigeta, Keiko\\nAU  - Shigeta K\\nFAU - Senba, Tadashi\\nAU  - Senba T\\nFAU - Matsumura, Kyoka\\nAU  - Matsumura K\\nFAU - Nakajima, Yoshie\\nAU  - Nakajima Y\\nFAU - Mizuno, Takae\\nAU  - Mizuno T\\nFAU - Morinaga, Misato\\nAU  - Morinaga M\\nFAU - Sasaki, Masahide\\nAU  - Sasaki M\\nFAU - Togashi, Takushi\\nAU  - Togashi T\\nFAU - Oyama, Masaaki\\nAU  - Oyama M\\nFAU - Hata, Hiroko\\nAU  - Hata H\\nFAU - Watanabe, Manabu\\nAU  - Watanabe M\\nFAU - Komatsu, Takami\\nAU  - Komatsu T\\nFAU - Mizushima-Sugano, Junko\\nAU  - Mizushima-Sugano J\\nFAU - Satoh, Tadashi\\nAU  - Satoh T\\nFAU - Shirai, Yuko\\nAU  - Shirai Y\\nFAU - Takahashi, Yukiko\\nAU  - Takahashi Y\\nFAU - Nakagawa, Kiyomi\\nAU  - Nakagawa K\\nFAU - Okumura, Koji\\nAU  - Okumura K\\nFAU - Nagase, Takahiro\\nAU  - Nagase T\\nFAU - Nomura, Nobuo\\nAU  - Nomura N\\nFAU - Kikuchi, Hisashi\\nAU  - Kikuchi H\\nFAU - Masuho, Yasuhiko\\nAU  - Masuho Y\\nFAU - Yamashita, Riu\\nAU  - Yamashita R\\nFAU - Nakai, Kenta\\nAU  - Nakai K\\nFAU - Yada, Tetsushi\\nAU  - Yada T\\nFAU - Nakamura, Yusuke\\nAU  - Nakamura Y\\nFAU - Ohara, Osamu\\nAU  - Ohara O\\nFAU - Isogai, Takao\\nAU  - Isogai T\\nFAU - Sugano, Sumio\\nAU  - Sugano S\\nLA  - eng\\nPT  - Comparative Study\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov't\\nDEP - 20031221\\nPL  - United States\\nTA  - Nat Genet\\nJT  - Nature genetics\\nJID - 9216904\\nRN  - 0 (DNA, Complementary)\\nRN  - 0 (RNA, Messenger)\\nSB  - IM\\nMH  - Chromosomes, Human, 21-22 and Y\\nMH  - Chromosomes, Human, Pair 20\\nMH  - Computational Biology\\nMH  - *DNA, Complementary\\nMH  - Humans\\nMH  - Open Reading Frames\\nMH  - RNA, Messenger\\nMH  - *Sequence Analysis, DNA\\nEDAT- 2004/01/01 05:00\\nMHDA- 2004/02/03 05:00\\nCRDT- 2004/01/01 05:00\\nPHST- 2003/10/07 00:00 [received]\\nPHST- 2003/12/01 00:00 [accepted]\\nPHST- 2004/01/01 05:00 [pubmed]\\nPHST- 2004/02/03 05:00 [medline]\\nPHST- 2004/01/01 05:00 [entrez]\\nAID - 10.1038/ng1285 [doi]\\nAID - ng1285 [pii]\\nPST - ppublish\\nSO  - Nat Genet. 2004 Jan;36(1):40-5. doi: 10.1038/ng1285. Epub 2003 Dec 21.\", \"PMID- 12477932\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20030121\\nLR  - 20190208\\nIS  - 0027-8424 (Print)\\nIS  - 0027-8424 (Linking)\\nVI  - 99\\nIP  - 26\\nDP  - 2002 Dec 24\\nTI  - Generation and initial analysis of more than 15,000 full-length human and mouse\\n      cDNA sequences.\\nPG  - 16899-903\\nAB  - The National Institutes of Health Mammalian Gene Collection (MGC) Program is a\\n      multiinstitutional effort to identify and sequence a cDNA clone containing a\\n      complete ORF for each human and mouse gene. ESTs were generated from libraries\\n      enriched for full-length cDNAs and analyzed to identify candidate full-ORF\\n      clones, which then were sequenced to high accuracy. The MGC has currently\\n      sequenced and verified the full ORF for a nonredundant set of >9,000 human and\\n      >6,000 mouse genes. Candidate full-ORF clones for an additional 7,800 human and\\n      3,500 mouse genes also have been identified. All MGC sequences and clones are\\n      available without restriction through public databases and clone distribution\\n      networks (see http:mgc.nci.nih.gov).\\nFAU - Strausberg, Robert L\\nAU  - Strausberg RL\\nAD  - National Cancer Institute, Bethessda, MD 20892-2580, USA. rls@nih.gov\\nFAU - Feingold, Elise A\\nAU  - Feingold EA\\nFAU - Grouse, Lynette H\\nAU  - Grouse LH\\nFAU - Derge, Jeffery G\\nAU  - Derge JG\\nFAU - Klausner, Richard D\\nAU  - Klausner RD\\nFAU - Collins, Francis S\\nAU  - Collins FS\\nFAU - Wagner, Lukas\\nAU  - Wagner L\\nFAU - Shenmen, Carolyn M\\nAU  - Shenmen CM\\nFAU - Schuler, Gregory D\\nAU  - Schuler GD\\nFAU - Altschul, Stephen F\\nAU  - Altschul SF\\nFAU - Zeeberg, Barry\\nAU  - Zeeberg B\\nFAU - Buetow, Kenneth H\\nAU  - Buetow KH\\nFAU - Schaefer, Carl F\\nAU  - Schaefer CF\\nFAU - Bhat, Narayan K\\nAU  - Bhat NK\\nFAU - Hopkins, Ralph F\\nAU  - Hopkins RF\\nFAU - Jordan, Heather\\nAU  - Jordan H\\nFAU - Moore, Troy\\nAU  - Moore T\\nFAU - Max, Steve I\\nAU  - Max SI\\nFAU - Wang, Jun\\nAU  - Wang J\\nFAU - Hsieh, Florence\\nAU  - Hsieh F\\nFAU - Diatchenko, Luda\\nAU  - Diatchenko L\\nFAU - Marusina, Kate\\nAU  - Marusina K\\nFAU - Farmer, Andrew A\\nAU  - Farmer AA\\nFAU - Rubin, Gerald M\\nAU  - Rubin GM\\nFAU - Hong, Ling\\nAU  - Hong L\\nFAU - Stapleton, Mark\\nAU  - Stapleton M\\nFAU - Soares, M Bento\\nAU  - Soares MB\\nFAU - Bonaldo, Maria F\\nAU  - Bonaldo MF\\nFAU - Casavant, Tom L\\nAU  - Casavant TL\\nFAU - Scheetz, Todd E\\nAU  - Scheetz TE\\nFAU - Brownstein, Michael J\\nAU  - Brownstein MJ\\nFAU - Usdin, Ted B\\nAU  - Usdin TB\\nFAU - Toshiyuki, Shiraki\\nAU  - Toshiyuki S\\nFAU - Carninci, Piero\\nAU  - Carninci P\\nFAU - Prange, Christa\\nAU  - Prange C\\nFAU - Raha, Sam S\\nAU  - Raha SS\\nFAU - Loquellano, Naomi A\\nAU  - Loquellano NA\\nFAU - Peters, Garrick J\\nAU  - Peters GJ\\nFAU - Abramson, Rick D\\nAU  - Abramson RD\\nFAU - Mullahy, Sara J\\nAU  - Mullahy SJ\\nFAU - Bosak, Stephanie A\\nAU  - Bosak SA\\nFAU - McEwan, Paul J\\nAU  - McEwan PJ\\nFAU - McKernan, Kevin J\\nAU  - McKernan KJ\\nFAU - Malek, Joel A\\nAU  - Malek JA\\nFAU - Gunaratne, Preethi H\\nAU  - Gunaratne PH\\nFAU - Richards, Stephen\\nAU  - Richards S\\nFAU - Worley, Kim C\\nAU  - Worley KC\\nFAU - Hale, Sarah\\nAU  - Hale S\\nFAU - Garcia, Angela M\\nAU  - Garcia AM\\nFAU - Gay, Laura J\\nAU  - Gay LJ\\nFAU - Hulyk, Stephen W\\nAU  - Hulyk SW\\nFAU - Villalon, Debbie K\\nAU  - Villalon DK\\nFAU - Muzny, Donna M\\nAU  - Muzny DM\\nFAU - Sodergren, Erica J\\nAU  - Sodergren EJ\\nFAU - Lu, Xiuhua\\nAU  - Lu X\\nFAU - Gibbs, Richard A\\nAU  - Gibbs RA\\nFAU - Fahey, Jessica\\nAU  - Fahey J\\nFAU - Helton, Erin\\nAU  - Helton E\\nFAU - Ketteman, Mark\\nAU  - Ketteman M\\nFAU - Madan, Anuradha\\nAU  - Madan A\\nFAU - Rodrigues, Stephanie\\nAU  - Rodrigues S\\nFAU - Sanchez, Amy\\nAU  - Sanchez A\\nFAU - Whiting, Michelle\\nAU  - Whiting M\\nFAU - Madan, Anup\\nAU  - Madan A\\nFAU - Young, Alice C\\nAU  - Young AC\\nFAU - Shevchenko, Yuriy\\nAU  - Shevchenko Y\\nFAU - Bouffard, Gerard G\\nAU  - Bouffard GG\\nFAU - Blakesley, Robert W\\nAU  - Blakesley RW\\nFAU - Touchman, Jeffrey W\\nAU  - Touchman JW\\nFAU - Green, Eric D\\nAU  - Green ED\\nFAU - Dickson, Mark C\\nAU  - Dickson MC\\nFAU - Rodriguez, Alex C\\nAU  - Rodriguez AC\\nFAU - Grimwood, Jane\\nAU  - Grimwood J\\nFAU - Schmutz, Jeremy\\nAU  - Schmutz J\\nFAU - Myers, Richard M\\nAU  - Myers RM\\nFAU - Butterfield, Yaron S N\\nAU  - Butterfield YS\\nFAU - Krzywinski, Martin I\\nAU  - Krzywinski MI\\nFAU - Skalska, Ursula\\nAU  - Skalska U\\nFAU - Smailus, Duane E\\nAU  - Smailus DE\\nFAU - Schnerch, Angelique\\nAU  - Schnerch A\\nFAU - Schein, Jacqueline E\\nAU  - Schein JE\\nFAU - Jones, Steven J M\\nAU  - Jones SJ\\nFAU - Marra, Marco A\\nAU  - Marra MA\\nCN  - Mammalian Gene Collection Program Team\\nLA  - eng\\nPT  - Journal Article\\nPT  - Research Support, U.S. Gov't, P.H.S.\\nDEP - 20021211\\nPL  - United States\\nTA  - Proc Natl Acad Sci U S A\\nJT  - Proceedings of the National Academy of Sciences of the United States of America\\nJID - 7505876\\nRN  - 0 (DNA, Complementary)\\nSB  - IM\\nMH  - Algorithms\\nMH  - Animals\\nMH  - DNA, Complementary/analysis/*chemistry\\nMH  - Gene Library\\nMH  - Humans\\nMH  - Mice\\nMH  - Open Reading Frames\\nMH  - *Sequence Analysis, DNA\\nPMC - PMC139241\\nEDAT- 2002/12/13 04:00\\nMHDA- 2003/01/22 04:00\\nCRDT- 2002/12/13 04:00\\nPHST- 2002/12/13 04:00 [pubmed]\\nPHST- 2003/01/22 04:00 [medline]\\nPHST- 2002/12/13 04:00 [entrez]\\nAID - 10.1073/pnas.242603899 [doi]\\nAID - 242603899 [pii]\\nPST - ppublish\\nSO  - Proc Natl Acad Sci U S A. 2002 Dec 24;99(26):16899-903. doi:\\n      10.1073/pnas.242603899. Epub 2002 Dec 11.\", \"PMID- 12421765\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20030114\\nLR  - 20181113\\nIS  - 1088-9051 (Print)\\nIS  - 1088-9051 (Linking)\\nVI  - 12\\nIP  - 11\\nDP  - 2002 Nov\\nTI  - Protein-protein interactions between large proteins: two-hybrid screening using a\\n      functionally classified library composed of long cDNAs.\\nPG  - 1773-84\\nAB  - Large proteins have multiple domains that are potentially capable of binding many\\n      kinds of partners. It is conceivable, therefore, that such proteins could\\n      function as an intricate framework of assembly protein complexes. To\\n      comprehensively study protein-protein interactions between large KIAA proteins,\\n      we have constructed a library composed of 1087 KIAA cDNA clones based on prior\\n      functional classifications done in silico. We were guided by two principles that \\n      raise the success rate for detecting interactions per tested combination: we\\n      avoided testing low-probability combinations, and reduced the number of potential\\n      false negatives that arise from the fact that large proteins cannot reliably be\\n      expressed in yeast. The latter was addressed by constructing a cDNA library\\n      comprised of random fragments encoding large proteins. Cytoplasmic domains of\\n      KIAA transmembrane proteins (>1000 amino acids) were used as bait for yeast\\n      two-hybrid screening. Our analyses reveal that several KIAA proteins bearing a\\n      transmembrane region have the capability of binding to other KIAA proteins\\n      containing domains (e.g., PDZ, SH3, rhoGEF, and spectrin) known to be localized\\n      to highly specialized submembranous sites, indicating that they participate in\\n      cellular junction formation, receptor or channel clustering, and intracellular\\n      signaling events. Our representative library should be a very useful resource for\\n      detecting previously unidentified interactions because it complements\\n      conventional expression libraries, which seldom contain large cDNAs.\\nFAU - Nakayama, Manabu\\nAU  - Nakayama M\\nAD  - Department of Human Gene Research, Kazusa DNA Research Institute, Kisarazu, Chiba\\n      292-0818, Japan. nmanabu@kazusa.or.jp\\nFAU - Kikuno, Reiko\\nAU  - Kikuno R\\nFAU - Ohara, Osamu\\nAU  - Ohara O\\nLA  - eng\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov't\\nPL  - United States\\nTA  - Genome Res\\nJT  - Genome research\\nJID - 9518021\\nRN  - 0 (DNA, Complementary)\\nRN  - 0 (Peptides)\\nRN  - 0 (Proteins)\\nSB  - IM\\nMH  - Brain Chemistry/genetics\\nMH  - Cloning, Molecular\\nMH  - Computational Biology/instrumentation/methods\\nMH  - Cytoplasm/chemistry\\nMH  - DNA, Complementary/chemistry/*genetics\\nMH  - *Gene Library\\nMH  - Humans\\nMH  - Leukemia, Myeloid/genetics/pathology\\nMH  - Peptide Mapping/methods\\nMH  - Peptides/genetics/physiology\\nMH  - Protein Interaction Mapping/*methods\\nMH  - Protein Structure, Tertiary/physiology\\nMH  - Proteins/chemistry/genetics/physiology\\nMH  - Software\\nMH  - Tumor Cells, Cultured\\nMH  - *Two-Hybrid System Techniques\\nMH  - src Homology Domains/genetics/physiology\\nPMC - PMC187542\\nEDAT- 2002/11/08 04:00\\nMHDA- 2003/01/15 04:00\\nCRDT- 2002/11/08 04:00\\nPHST- 2002/11/08 04:00 [pubmed]\\nPHST- 2003/01/15 04:00 [medline]\\nPHST- 2002/11/08 04:00 [entrez]\\nAID - 10.1101/gr.406902 [doi]\\nPST - ppublish\\nSO  - Genome Res. 2002 Nov;12(11):1773-84. doi: 10.1101/gr.406902.\", \"PMID- 9455484\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 19980227\\nLR  - 20190512\\nIS  - 1340-2838 (Print)\\nIS  - 1340-2838 (Linking)\\nVI  - 4\\nIP  - 5\\nDP  - 1997 Oct 31\\nTI  - Characterization of cDNA clones in size-fractionated cDNA libraries from human\\n      brain.\\nPG  - 345-9\\nAB  - To evaluate the size-fractionated cDNA libraries of human brain previously\\n      constructed (O. O'hara et al. DNA Research, 4, 53-59, 1997), the occurrence of\\n      chimeric clones and the content of clones with coding potentiality were analyzed \\n      using the randomly sampled clones with insert sizes of 5 to 7 kb. When the\\n      chromosomal location of 30 clones was determined by the radiation-hybrid mapping \\n      method, the map positions assigned from the 3'- and 5'-end sequences separately\\n      were coincident for 29 clones, suggesting that the occurrence of chimeric clones \\n      is at most 1/30. Using 91 clones mapped to chromosome 1, the content of clones\\n      that have the potentiality coding for proteins larger than 100 amino acid\\n      residues was estimated to be approximately 50% (46 out of 91 clones) on the basis\\n      of nucleotide sequence analysis and coding potentiality assay in vitro. No\\n      significant open reading frames were detected in the remaining clones. Although\\n      the clones coding for short peptides may not have been included in the above\\n      estimation, the libraries constructed from the whole brain mRNA fraction appear\\n      to contain a considerable amount of clones corresponding to the 5'-truncated\\n      transcripts in an unprocessed form and/or those with long 3'-untranslated\\n      regions.\\nFAU - Seki, N\\nAU  - Seki N\\nAD  - Kazusa DNA Research Institute, Chiba, Japan. nseki@kazusa.or.jp\\nFAU - Ohira, M\\nAU  - Ohira M\\nFAU - Nagase, T\\nAU  - Nagase T\\nFAU - Ishikawa, K\\nAU  - Ishikawa K\\nFAU - Miyajima, N\\nAU  - Miyajima N\\nFAU - Nakajima, D\\nAU  - Nakajima D\\nFAU - Nomura, N\\nAU  - Nomura N\\nFAU - Ohara, O\\nAU  - Ohara O\\nLA  - eng\\nSI  - GENBANK/AB007913\\nSI  - GENBANK/AB007914\\nSI  - GENBANK/AB007915\\nSI  - GENBANK/AB007916\\nSI  - GENBANK/AB007917\\nSI  - GENBANK/AB007918\\nSI  - GENBANK/AB007919\\nSI  - GENBANK/AB007920\\nSI  - GENBANK/AB007921\\nSI  - GENBANK/AB007922\\nSI  - GENBANK/AB007923\\nSI  - GENBANK/AB007924\\nSI  - GENBANK/AB007925\\nSI  - GENBANK/AB007926\\nSI  - GENBANK/AB007927\\nSI  - GENBANK/AB007928\\nSI  - GENBANK/AB007929\\nSI  - GENBANK/AB007930\\nSI  - GENBANK/AB007931\\nSI  - GENBANK/AB007932\\nSI  - GENBANK/AB007933\\nSI  - GENBANK/AB007934\\nSI  - GENBANK/AB007935\\nSI  - GENBANK/AB007936\\nSI  - GENBANK/AB007937\\nSI  - GENBANK/AB007938\\nSI  - GENBANK/AB007939\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov't\\nPL  - England\\nTA  - DNA Res\\nJT  - DNA research : an international journal for rapid publication of reports on genes\\n      and genomes\\nJID - 9423827\\nRN  - 0 (DNA, Complementary)\\nRN  - 0 (DNAJC6 protein, human)\\nRN  - 0 (HSP40 Heat-Shock Proteins)\\nRN  - 0 (RNA, Messenger)\\nSB  - IM\\nMH  - Animals\\nMH  - Brain Chemistry/*genetics\\nMH  - Chimera\\nMH  - *Chromosome Mapping\\nMH  - *Chromosomes, Human, Pair 1\\nMH  - Cloning, Molecular\\nMH  - DNA, Complementary/*chemistry/genetics/*isolation & purification\\nMH  - Electrophoresis, Polyacrylamide Gel\\nMH  - Gene Library\\nMH  - HSP40 Heat-Shock Proteins\\nMH  - Humans\\nMH  - Open Reading Frames\\nMH  - Polymerase Chain Reaction\\nMH  - RNA, Messenger/genetics\\nMH  - Repetitive Sequences, Nucleic Acid\\nMH  - Sequence Homology, Nucleic Acid\\nMH  - Sequence Tagged Sites\\nMH  - Transcription, Genetic\\nEDAT- 1998/02/10 00:00\\nMHDA- 1998/02/10 00:01\\nCRDT- 1998/02/10 00:00\\nPHST- 1998/02/10 00:00 [pubmed]\\nPHST- 1998/02/10 00:01 [medline]\\nPHST- 1998/02/10 00:00 [entrez]\\nAID - 10.1093/dnares/4.5.345 [doi]\\nPST - ppublish\\nSO  - DNA Res. 1997 Oct 31;4(5):345-9. doi: 10.1093/dnares/4.5.345.\", \"PMID- 9110174\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 19970617\\nLR  - 20191108\\nIS  - 1088-9051 (Print)\\nIS  - 1088-9051 (Linking)\\nVI  - 7\\nIP  - 4\\nDP  - 1997 Apr\\nTI  - Large-scale concatenation cDNA sequencing.\\nPG  - 353-8\\nAB  - A total of 100 kb of DNA derived from 69 individual human brain cDNA clones of\\n      0.7-2.0 kb were sequenced by concatenated cDNA sequencing (CCS), whereby multiple\\n      individual DNA fragments are sequenced simultaneously in a single shotgun\\n      library. The method yielded accurate sequences and a similar efficiency compared \\n      with other shotgun libraries constructed from single DNA fragments (> 20 kb).\\n      Computer analyses were carried out on 65 cDNA clone sequences and their\\n      corresponding end sequences to examine both nucleic acid and amino acid sequence \\n      similarities in the databases. Thirty-seven clones revealed no DNA database\\n      matches, 12 clones generated exact matches (> or = 98% identity), and 16 clones\\n      generated nonexact matches (57%-97% identity) to either known human or other\\n      species genes. Of those 28 matched clones, 8 had corresponding end sequences that\\n      failed to identify similarities. In a protein similarity search, 27 clone\\n      sequences displayed significant matches, whereas only 20 of the end sequences had\\n      matches to known protein sequences. Our data indicate that full-length cDNA\\n      insert sequences provide significantly more nucleic acid and protein sequence\\n      similarity matches than expressed sequence tags (ESTs) for database searching.\\nFAU - Yu, W\\nAU  - Yu W\\nFAU - Andersson, B\\nAU  - Andersson B\\nFAU - Worley, K C\\nAU  - Worley KC\\nFAU - Muzny, D M\\nAU  - Muzny DM\\nFAU - Ding, Y\\nAU  - Ding Y\\nFAU - Liu, W\\nAU  - Liu W\\nFAU - Ricafrente, J Y\\nAU  - Ricafrente JY\\nFAU - Wentland, M A\\nAU  - Wentland MA\\nFAU - Lennon, G\\nAU  - Lennon G\\nFAU - Gibbs, R A\\nAU  - Gibbs RA\\nLA  - eng\\nSI  - GENBANK/AF007128\\nSI  - GENBANK/AF007129\\nSI  - GENBANK/AF007130\\nSI  - GENBANK/AF007131\\nSI  - GENBANK/AF007132\\nSI  - GENBANK/AF007133\\nSI  - GENBANK/AF007134\\nSI  - GENBANK/AF007135\\nSI  - GENBANK/AF007136\\nSI  - GENBANK/AF007137\\nSI  - GENBANK/AF007138\\nSI  - GENBANK/AF007139\\nSI  - GENBANK/AF007140\\nSI  - GENBANK/AF007141\\nSI  - GENBANK/AF007142\\nSI  - GENBANK/AF007143\\nSI  - GENBANK/AF007144\\nSI  - GENBANK/AF007145\\nSI  - GENBANK/AF007146\\nSI  - GENBANK/AF007147\\nSI  - GENBANK/AF007148\\nSI  - GENBANK/AF007149\\nSI  - GENBANK/AF007150\\nSI  - GENBANK/AF007151\\nSI  - GENBANK/AF007152\\nSI  - GENBANK/AF007153\\nSI  - GENBANK/AF007154\\nGR  - P30 HG00210-05/HG/NHGRI NIH HHS/United States\\nGR  - R01 HG00823/HG/NHGRI NIH HHS/United States\\nGR  - 1F32 HG00169-01/HG/NHGRI NIH HHS/United States\\nPT  - Letter\\nPT  - Research Support, U.S. Gov't, Non-P.H.S.\\nPT  - Research Support, U.S. Gov't, P.H.S.\\nPL  - United States\\nTA  - Genome Res\\nJT  - Genome research\\nJID - 9518021\\nRN  - 0 (DNA Transposable Elements)\\nRN  - 0 (DNA, Complementary)\\nRN  - 0 (Proteins)\\nSB  - IM\\nMH  - DNA Transposable Elements\\nMH  - DNA, Complementary/chemistry/*genetics\\nMH  - Databases, Factual\\nMH  - Gene Library\\nMH  - Humans\\nMH  - Molecular Sequence Data\\nMH  - Proteins/chemistry/*genetics\\nMH  - Sequence Alignment/*methods\\nMH  - Sequence Analysis, DNA/*methods\\nMH  - Sequence Homology, Amino Acid\\nMH  - Sequence Homology, Nucleic Acid\\nMH  - Software\\nPMC - PMC139146\\nEDAT- 1997/04/01 00:00\\nMHDA- 1997/04/01 00:01\\nCRDT- 1997/04/01 00:00\\nPHST- 1997/04/01 00:00 [pubmed]\\nPHST- 1997/04/01 00:01 [medline]\\nPHST- 1997/04/01 00:00 [entrez]\\nAID - 10.1101/gr.7.4.353 [doi]\\nPST - ppublish\\nSO  - Genome Res. 1997 Apr;7(4):353-8. doi: 10.1101/gr.7.4.353.\", \"PMID- 8619474\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 19960613\\nLR  - 20071114\\nIS  - 0003-2697 (Print)\\nIS  - 0003-2697 (Linking)\\nVI  - 236\\nIP  - 1\\nDP  - 1996 Apr 5\\nTI  - A \\\"double adaptor\\\" method for improved shotgun library construction.\\nPG  - 107-13\\nAB  - The efficiency of shotgun DNA sequencing depends to a great extent on the quality\\n      of the random-subclone libraries used. We here describe a novel \\\"double adaptor\\\" \\n      strategy for efficient construction of high-quality shotgun libraries. In this\\n      method, randomly sheared and end-repaired fragments are ligated to\\n      oligonucleotide adaptors creating 12-base overhangs. Nonphosphorylated\\n      oligonucleotides are used, which prevents formation of adaptor dimers and ensures\\n      efficient ligation of insert to adaptor. The vector is prepared from a modified\\n      M13 vector, by KpnI/PstI digestion followed by ligation to oligonucleotides with \\n      ends complementary to the overhangs created in the digest. These adaptors create \\n      5'-overhangs complementary to those on the inserts. Following annealing of insert\\n      to vector, the DNA is directly used for transformation without a ligation step.\\n      This protocol is robust and shows three- to fivefold higher yield of clones\\n      compared to previous protocols. No chimeric clones can be detected and the\\n      background of clones without an insert is <1%. The procedure is rapid and shows\\n      potential for automation.\\nFAU - Andersson, B\\nAU  - Andersson B\\nAD  - Department of Molecular and Human Genetics, Baylor College of Medicine, One\\n      Baylor Plaza, Houston, Texas, 77030, USA.\\nFAU - Wentland, M A\\nAU  - Wentland MA\\nFAU - Ricafrente, J Y\\nAU  - Ricafrente JY\\nFAU - Liu, W\\nAU  - Liu W\\nFAU - Gibbs, R A\\nAU  - Gibbs RA\\nLA  - eng\\nSI  - GENBANK/AF007128\\nSI  - GENBANK/AF007129\\nSI  - GENBANK/AF007130\\nSI  - GENBANK/AF007131\\nSI  - GENBANK/AF007132\\nSI  - GENBANK/AF007133\\nSI  - GENBANK/AF007134\\nSI  - GENBANK/AF007135\\nSI  - GENBANK/AF007136\\nSI  - GENBANK/AF007137\\nSI  - GENBANK/AF007138\\nSI  - GENBANK/AF007139\\nSI  - GENBANK/AF007140\\nSI  - GENBANK/AF007141\\nSI  - GENBANK/AF007142\\nSI  - GENBANK/AF007143\\nSI  - GENBANK/AF007144\\nSI  - GENBANK/AF007145\\nSI  - GENBANK/AF007146\\nSI  - GENBANK/AF007147\\nSI  - GENBANK/AF007148\\nSI  - GENBANK/AF007149\\nSI  - GENBANK/AF007150\\nSI  - GENBANK/AF007151\\nSI  - GENBANK/AF007152\\nSI  - GENBANK/AF007153\\nSI  - GENBANK/AF007154\\nGR  - R01 HG00823/HG/NHGRI NIH HHS/United States\\nPT  - Journal Article\\nPT  - Research Support, U.S. Gov't, P.H.S.\\nPL  - United States\\nTA  - Anal Biochem\\nJT  - Analytical biochemistry\\nJID - 0370535\\nRN  - 0 (Oligonucleotides)\\nSB  - IM\\nMH  - Base Sequence\\nMH  - Cloning, Molecular/*methods\\nMH  - Genetic Vectors\\nMH  - *Genomic Library\\nMH  - Humans\\nMH  - Molecular Sequence Data\\nMH  - Oligonucleotides/chemistry\\nMH  - Polymerase Chain Reaction\\nMH  - X Chromosome\\nEDAT- 1996/04/05 00:00\\nMHDA- 1996/04/05 00:01\\nCRDT- 1996/04/05 00:00\\nPHST- 1996/04/05 00:00 [pubmed]\\nPHST- 1996/04/05 00:01 [medline]\\nPHST- 1996/04/05 00:00 [entrez]\\nAID - S0003-2697(96)90138-4 [pii]\\nAID - 10.1006/abio.1996.0138 [doi]\\nPST - ppublish\\nSO  - Anal Biochem. 1996 Apr 5;236(1):107-13. doi: 10.1006/abio.1996.0138.\", \"PMID- 30789691\\nSTAT- Publisher\\nDA  - 20190222\\nCTDT- 20190221\\nPB  - University of Washington, Seattle\\nDP  - 1993\\nTI  - VPS13D Movement Disorder\\nBTI - GeneReviews((R))\\nAB  - CLINICAL CHARACTERISTICS: VPS13D movement disorder is a hyperkinetic movement\\n      disorder (dystonia, chorea, and/or ataxia) of variable age of onset that can be\\n      associated with developmental delay. Onset ranges from birth to adulthood.\\n      Individuals can present in childhood with motor delays and gait instability.\\n      Cognitive impairment ranging from mild intellectual disability to developmental\\n      delay has been reported, and several individuals have normal cognitive function. \\n      Individuals have also presented as young adults with gait difficulties caused by \\n      spastic ataxia or ataxia. In addition to gait ataxia, affected individuals had\\n      limb ataxia, dysarthria, and eye movement abnormalities (macro-saccadic\\n      oscillations, nystagmus, and saccadic pursuit). Additional features reported in\\n      some individuals include peripheral neuropathy and/or seizures. The disorder\\n      progresses to spastic ataxia or generalized dystonia, which can lead to loss of\\n      independent ambulation. DIAGNOSIS: The diagnosis of VPS13D movement disorder is\\n      established in a proband by identification of biallelic pathogenic variants in\\n      VPS13D on molecular genetic testing. MANAGEMENT: Treatment of manifestations:\\n      Standard treatment for seizures; spasticity treatments include baclofen,\\n      tizanidine, benzodiazepines, dantrolene sodium, gabapentin, botulinum toxin\\n      injections, and intrathecal baclofen; treatment options for dystonia include\\n      trihexyphenidyl, botulinum toxin injections, benzodiazepines, baclofen, and\\n      levodopa. A multidisciplinary team including occupational and physical therapists\\n      and a physiatrist is important; supportive developmental therapies should be\\n      provided as needed. Surveillance: Monitor for urinary urgency, contractures, and \\n      seizures; evaluate fall risk and gait stability; monitor developmental progress\\n      and educational needs. GENETIC COUNSELING: VPS13D movement disorder is inherited \\n      in an autosomal recessive manner. At conception, each sib of an affected\\n      individual has a 25% chance of being affected, a 50% chance of being an\\n      asymptomatic carrier, and a 25% chance of being unaffected and not a carrier.\\n      Carrier testing for at-risk relatives and prenatal testing for pregnancies at\\n      increased risk are possible if the VPS13D pathogenic variants in the family are\\n      known.\\nCI  - Copyright (c) 1993-2019, University of Washington, Seattle. GeneReviews is a\\n      registered trademark of the University of Washington, Seattle. All rights\\n      reserved.\\nFED - Adam, Margaret P\\nED  - Adam MP\\nFED - Ardinger, Holly H\\nED  - Ardinger HH\\nFED - Pagon, Roberta A\\nED  - Pagon RA\\nFED - Wallace, Stephanie E\\nED  - Wallace SE\\nFED - Bean, Lora JH\\nED  - Bean LJH\\nFED - Stephens, Karen\\nED  - Stephens K\\nFED - Amemiya, Anne\\nED  - Amemiya A\\nFAU - Meijer, Inge A\\nAU  - Meijer IA\\nLA  - eng\\nGR  - HHSN276201400262U/NLM NIH HHS/United States\\nPT  - Review\\nPT  - Book Chapter\\nPL  - Seattle (WA)\\nEDAT- 2019/02/22 06:01\\nMHDA- 2019/02/22 06:01\\nCDAT- 2019/02/22 06:01\\nAID - NBK537720 [bookaccession]\", \"PMID- 30258100\\nOWN - NLM\\nSTAT- In-Process\\nLR  - 20191115\\nIS  - 2045-2322 (Electronic)\\nIS  - 2045-2322 (Linking)\\nVI  - 8\\nIP  - 1\\nDP  - 2018 Sep 26\\nTI  - USP45 and Spindly are part of the same complex implicated in cell migration.\\nPG  - 14375\\nLID - 10.1038/s41598-018-32685-8 [doi]\\nAB  - Ubiquitylation is a protein modification implicated in several cellular\\n      processes. This process is reversible by the action of deubiquinating enzymes\\n      (DUBs). USP45 is a ubiquitin specific protease about which little is known, aside\\n      from roles in DNA damage repair and differentiation of the vertebrate retina.\\n      Here, by using mass spectrometry we have identified Spindly as a new target of\\n      USP45. Our data show that Spindly and USP45 are part of the same complex and that\\n      their interaction specifically depends on the catalytic activity of USP45. In\\n      addition, we describe the type of ubiquitin chains associated with the complex\\n      that can be cleaved by USP45, with a preferential activity on K48 ubiquitin chain\\n      type and potentially K6. Here, we also show that Spindly is mono-ubiquitylated\\n      and this can be specifically removed by USP45 in its active form but not by the\\n      catalytic inactive form. Lastly, we identified a new role for USP45 in cell\\n      migration, similar to that which was recently described for Spindly.\\nFAU - Conte, Claudia\\nAU  - Conte C\\nAD  - MRC Protein Phosphorylation and Ubiquitylation Unit, College of Life Sciences,\\n      University of Dundee, Dundee, UK.\\nAD  - Cell and Developmental Biology, Center of Genomic Regulation (CRG), Barcelona,\\n      Spain.\\nFAU - Griffis, Eric R\\nAU  - Griffis ER\\nAD  - MRC Protein Phosphorylation and Ubiquitylation Unit, College of Life Sciences,\\n      University of Dundee, Dundee, UK.\\nAD  - Nikon Instruments Incorporated, Melville, NY, 11747, USA.\\nFAU - Hickson, Ian\\nAU  - Hickson I\\nAD  - Janssen Research & Development, LLC, Spring House, PA, USA.\\nAD  - Northern Institute for Cancer Research, Newcastle University, Medical School,\\n      Newcastle upon Tyne, NE2 4HH, UK.\\nFAU - Perez-Oliva, Ana B\\nAU  - Perez-Oliva AB\\nAUID- ORCID: http://orcid.org/0000-0002-2596-0871\\nAD  - MRC Protein Phosphorylation and Ubiquitylation Unit, College of Life Sciences,\\n      University of Dundee, Dundee, UK. anabpo@um.es.\\nLA  - eng\\nGR  - MC_UU_12016/2/Medical Research Council (MRC)/International\\nGR  - 097945/B/11/Z/WT_/Wellcome Trust/United Kingdom\\nGR  - 097945/B/11/Z/WT_/Wellcome Trust/United Kingdom\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov't\\nDEP - 20180926\\nPL  - England\\nTA  - Sci Rep\\nJT  - Scientific reports\\nJID - 101563288\\nSB  - IM\\nPMC - PMC6158257\\nEDAT- 2018/09/28 06:00\\nMHDA- 2018/09/28 06:00\\nCRDT- 2018/09/28 06:00\\nPHST- 2018/05/21 00:00 [received]\\nPHST- 2018/09/11 00:00 [accepted]\\nPHST- 2018/09/28 06:00 [entrez]\\nPHST- 2018/09/28 06:00 [pubmed]\\nPHST- 2018/09/28 06:00 [medline]\\nAID - 10.1038/s41598-018-32685-8 [doi]\\nAID - 10.1038/s41598-018-32685-8 [pii]\\nPST - epublish\\nSO  - Sci Rep. 2018 Sep 26;8(1):14375. doi: 10.1038/s41598-018-32685-8.\", \"PMID- 29604224\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20190812\\nLR  - 20190812\\nIS  - 1531-8249 (Electronic)\\nIS  - 0364-5134 (Linking)\\nVI  - 83\\nIP  - 6\\nDP  - 2018 Jun\\nTI  - Mutations in VPS13D lead to a new recessive ataxia with spasticity and\\n      mitochondrial defects.\\nPG  - 1075-1088\\nLID - 10.1002/ana.25220 [doi]\\nAB  - OBJECTIVE: To identify novel causes of recessive ataxias, including\\n      spinocerebellar ataxia with saccadic intrusions, spastic ataxias, and spastic\\n      paraplegia. METHODS: In an international collaboration, we independently\\n      performed exome sequencing in 7 families with recessive ataxia and/or spastic\\n      paraplegia. To evaluate the role of VPS13D mutations, we evaluated a Drosophila\\n      knockout model and investigated mitochondrial function in patient-derived\\n      fibroblast cultures. RESULTS: Exome sequencing identified compound heterozygous\\n      mutations in VPS13D on chromosome 1p36 in all 7 families. This included a large\\n      family with 5 affected siblings with spinocerebellar ataxia with saccadic\\n      intrusions (SCASI), or spinocerebellar ataxia, recessive, type 4 (SCAR4). Linkage\\n      to chromosome 1p36 was found in this family with a logarithm of odds score of\\n      3.1. The phenotypic spectrum in our 12 patients was broad. Although most\\n      presented with ataxia, additional or predominant spasticity was present in 5\\n      patients. Disease onset ranged from infancy to 39 years, and symptoms were slowly\\n      progressive and included loss of independent ambulation in 5. All but 2 patients \\n      carried a loss-of-function (nonsense or splice site) mutation on one and a\\n      missense mutation on the other allele. Knockdown or removal of Vps13D in\\n      Drosophila neurons led to changes in mitochondrial morphology and impairment in\\n      mitochondrial distribution along axons. Patient fibroblasts showed altered\\n      morphology and functionality including reduced energy production. INTERPRETATION:\\n      Our study demonstrates that compound heterozygous mutations in VPS13D cause\\n      movement disorders along the ataxia-spasticity spectrum, making VPS13D the fourth\\n      VPS13 paralog involved in neurological disorders. Ann Neurol 2018.\\nCI  - (c) 2018 American Neurological Association.\\nFAU - Seong, Eunju\\nAU  - Seong E\\nAD  - Molecular and Behavioral Neuroscience Institute, University of Michigan, Ann\\n      Arbor, MI.\\nFAU - Insolera, Ryan\\nAU  - Insolera R\\nAD  - Department of Molecular, Cellular, and Developmental Biology, University of\\n      Michigan, Ann Arbor, MI.\\nFAU - Dulovic, Marija\\nAU  - Dulovic M\\nAD  - Institute of Neurogenetics, University of Lubeck, Lubeck, Germany.\\nFAU - Kamsteeg, Erik-Jan\\nAU  - Kamsteeg EJ\\nAD  - Department of Human Genetics, Radboud University Medical Centre, Nijmegen, the\\n      Netherlands.\\nFAU - Trinh, Joanne\\nAU  - Trinh J\\nAD  - Institute of Neurogenetics, University of Lubeck, Lubeck, Germany.\\nFAU - Bruggemann, Norbert\\nAU  - Bruggemann N\\nAD  - Department of Neurology, University of Lubeck, Lubeck, Germany.\\nFAU - Sandford, Erin\\nAU  - Sandford E\\nAD  - Molecular and Behavioral Neuroscience Institute, University of Michigan, Ann\\n      Arbor, MI.\\nFAU - Li, Sheng\\nAU  - Li S\\nAD  - Molecular and Behavioral Neuroscience Institute, University of Michigan, Ann\\n      Arbor, MI.\\nFAU - Ozel, Ayse Bilge\\nAU  - Ozel AB\\nAD  - Department of Human Genetics, University of Michigan, Ann Arbor, MI.\\nFAU - Li, Jun Z\\nAU  - Li JZ\\nAUID- ORCID: 0000-0001-6727-0812\\nAD  - Department of Human Genetics, University of Michigan, Ann Arbor, MI.\\nAD  - Department of Computational Medicine and Bioinformatics, University of Michigan, \\n      Ann Arbor, MI.\\nFAU - Jewett, Tamison\\nAU  - Jewett T\\nAD  - Department of Pediatrics, Section on Medical Genetics, Wake Forest School of\\n      Medicine, Winston-Salem, NC.\\nFAU - Kievit, Anneke J A\\nAU  - Kievit AJA\\nAD  - Department of Clinical Genetics, Erasmus MC, Rotterdam, the Netherlands.\\nFAU - Munchau, Alexander\\nAU  - Munchau A\\nAD  - Institute of Neurogenetics, University of Lubeck, Lubeck, Germany.\\nFAU - Shakkottai, Vikram\\nAU  - Shakkottai V\\nAD  - Departments of Neurology and Molecular and Integrative Physiology, University of \\n      Michigan, Ann Arbor, MI.\\nFAU - Klein, Christine\\nAU  - Klein C\\nAD  - Institute of Neurogenetics, University of Lubeck, Lubeck, Germany.\\nFAU - Collins, Catherine A\\nAU  - Collins CA\\nAUID- ORCID: 0000-0002-1608-6692\\nAD  - Department of Molecular, Cellular, and Developmental Biology, University of\\n      Michigan, Ann Arbor, MI.\\nFAU - Lohmann, Katja\\nAU  - Lohmann K\\nAD  - Institute of Neurogenetics, University of Lubeck, Lubeck, Germany.\\nFAU - van de Warrenburg, Bart P\\nAU  - van de Warrenburg BP\\nAD  - Department of Neurology, Donders Institute for Brain, Cognition, and Behavior,\\n      Radboud University Medical Centre, Nijmegen, the Netherlands.\\nFAU - Burmeister, Margit\\nAU  - Burmeister M\\nAUID- ORCID: 0000-0002-1914-2434\\nAD  - Molecular and Behavioral Neuroscience Institute, University of Michigan, Ann\\n      Arbor, MI.\\nAD  - Department of Human Genetics, University of Michigan, Ann Arbor, MI.\\nAD  - Department of Computational Medicine and Bioinformatics, University of Michigan, \\n      Ann Arbor, MI.\\nAD  - Department of Psychiatry, University of Michigan, Ann Arbor, MI.\\nLA  - eng\\nGR  - F31 NS056780/NS/NINDS NIH HHS/United States\\nGR  - F32 NS098611/NS/NINDS NIH HHS/United States\\nGR  - R01 NS069844/NS/NINDS NIH HHS/United States\\nGR  - R01 NS078560/NS/NINDS NIH HHS/United States\\nPT  - Journal Article\\nPT  - Research Support, N.I.H., Extramural\\nPT  - Research Support, Non-U.S. Gov't\\nDEP - 20180630\\nPL  - United States\\nTA  - Ann Neurol\\nJT  - Annals of neurology\\nJID - 7707449\\nRN  - 0 (Proteins)\\nRN  - 0 (VPS13D protein, human)\\nRN  - Spastic Ataxia\\nCIN - Ann Neurol. 2018 Jun;83(6):1072-1074. PMID: 29908061\\nMH  - Adult\\nMH  - Cerebellar Ataxia/genetics\\nMH  - Female\\nMH  - Genetic Linkage\\nMH  - Humans\\nMH  - Intellectual Disability/*genetics\\nMH  - Male\\nMH  - Middle Aged\\nMH  - Mitochondria/*genetics\\nMH  - Muscle Spasticity/*genetics\\nMH  - Mutation/*genetics\\nMH  - Optic Atrophy/*genetics\\nMH  - Pedigree\\nMH  - Proteins/*genetics\\nMH  - Spastic Paraplegia, Hereditary/genetics\\nMH  - Spinocerebellar Ataxias/*genetics\\nPMC - PMC6105379\\nMID - NIHMS955732\\nEDAT- 2018/04/01 06:00\\nMHDA- 2019/08/14 06:00\\nCRDT- 2018/04/01 06:00\\nPHST- 2017/11/01 00:00 [received]\\nPHST- 2018/03/11 00:00 [revised]\\nPHST- 2018/03/19 00:00 [accepted]\\nPHST- 2018/04/01 06:00 [pubmed]\\nPHST- 2019/08/14 06:00 [medline]\\nPHST- 2018/04/01 06:00 [entrez]\\nAID - 10.1002/ana.25220 [doi]\\nPST - ppublish\\nSO  - Ann Neurol. 2018 Jun;83(6):1075-1088. doi: 10.1002/ana.25220. Epub 2018 Jun 30.\", \"PMID- 29568061\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20181211\\nLR  - 20190322\\nIS  - 2041-1723 (Electronic)\\nIS  - 2041-1723 (Linking)\\nVI  - 9\\nIP  - 1\\nDP  - 2018 Mar 22\\nTI  - An AP-MS- and BioID-compatible MAC-tag enables comprehensive mapping of protein\\n      interactions and subcellular localizations.\\nPG  - 1188\\nLID - 10.1038/s41467-018-03523-2 [doi]\\nAB  - Protein-protein interactions govern almost all cellular functions. These complex \\n      networks of stable and transient associations can be mapped by affinity\\n      purification mass spectrometry (AP-MS) and complementary proximity-based labeling\\n      methods such as BioID. To exploit the advantages of both strategies, we here\\n      design and optimize an integrated approach combining AP-MS and BioID in a single \\n      construct, which we term MAC-tag. We systematically apply the MAC-tag approach to\\n      18 subcellular and 3 sub-organelle localization markers, generating a molecular\\n      context database, which can be used to define a protein's molecular location. In \\n      addition, we show that combining the AP-MS and BioID results makes it possible to\\n      obtain interaction distances within a protein complex. Taken together, our\\n      integrated strategy enables the comprehensive mapping of the physical and\\n      functional interactions of proteins, defining their molecular context and\\n      improving our understanding of the cellular interactome.\\nFAU - Liu, Xiaonan\\nAU  - Liu X\\nAUID- ORCID: http://orcid.org/0000-0002-9600-0536\\nAD  - Institute of Biotechnology, University of Helsinki, Helsinki, 00014, Finland.\\nAD  - Helsinki Institute of Life Science, University of Helsinki, Helsinki, 00014,\\n      Finland.\\nFAU - Salokas, Kari\\nAU  - Salokas K\\nAUID- ORCID: http://orcid.org/0000-0002-4471-6698\\nAD  - Institute of Biotechnology, University of Helsinki, Helsinki, 00014, Finland.\\nAD  - Helsinki Institute of Life Science, University of Helsinki, Helsinki, 00014,\\n      Finland.\\nFAU - Tamene, Fitsum\\nAU  - Tamene F\\nAD  - Institute of Biotechnology, University of Helsinki, Helsinki, 00014, Finland.\\nAD  - Helsinki Institute of Life Science, University of Helsinki, Helsinki, 00014,\\n      Finland.\\nAD  - Proteomics Unit, University of Helsinki, Helsinki, 00014, Finland.\\nFAU - Jiu, Yaming\\nAU  - Jiu Y\\nAUID- ORCID: http://orcid.org/0000-0002-8601-8820\\nAD  - Institute of Biotechnology, University of Helsinki, Helsinki, 00014, Finland.\\nAD  - Helsinki Institute of Life Science, University of Helsinki, Helsinki, 00014,\\n      Finland.\\nFAU - Weldatsadik, Rigbe G\\nAU  - Weldatsadik RG\\nAD  - Institute of Biotechnology, University of Helsinki, Helsinki, 00014, Finland.\\nAD  - Helsinki Institute of Life Science, University of Helsinki, Helsinki, 00014,\\n      Finland.\\nAD  - Proteomics Unit, University of Helsinki, Helsinki, 00014, Finland.\\nFAU - Ohman, Tiina\\nAU  - Ohman T\\nAD  - Institute of Biotechnology, University of Helsinki, Helsinki, 00014, Finland.\\nAD  - Helsinki Institute of Life Science, University of Helsinki, Helsinki, 00014,\\n      Finland.\\nAD  - Proteomics Unit, University of Helsinki, Helsinki, 00014, Finland.\\nFAU - Varjosalo, Markku\\nAU  - Varjosalo M\\nAUID- ORCID: http://orcid.org/0000-0002-1340-9732\\nAD  - Institute of Biotechnology, University of Helsinki, Helsinki, 00014, Finland.\\n      markku.varjosalo@helsinki.fi.\\nAD  - Helsinki Institute of Life Science, University of Helsinki, Helsinki, 00014,\\n      Finland. markku.varjosalo@helsinki.fi.\\nAD  - Proteomics Unit, University of Helsinki, Helsinki, 00014, Finland.\\n      markku.varjosalo@helsinki.fi.\\nLA  - eng\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov't\\nDEP - 20180322\\nPL  - England\\nTA  - Nat Commun\\nJT  - Nature communications\\nJID - 101528555\\nRN  - 0 (Proteins)\\nSB  - IM\\nMH  - Biotinylation\\nMH  - Cell Line\\nMH  - Chromatography, Affinity\\nMH  - High-Throughput Screening Assays/instrumentation/*methods\\nMH  - Humans\\nMH  - Mass Spectrometry\\nMH  - Protein Binding\\nMH  - Protein Interaction Mapping/instrumentation/*methods\\nMH  - Protein Transport\\nMH  - Proteins/*chemistry/isolation & purification/metabolism\\nPMC - PMC5864832\\nEDAT- 2018/03/24 06:00\\nMHDA- 2018/12/12 06:00\\nCRDT- 2018/03/24 06:00\\nPHST- 2018/02/01 00:00 [received]\\nPHST- 2018/02/21 00:00 [accepted]\\nPHST- 2018/03/24 06:00 [entrez]\\nPHST- 2018/03/24 06:00 [pubmed]\\nPHST- 2018/12/12 06:00 [medline]\\nAID - 10.1038/s41467-018-03523-2 [doi]\\nAID - 10.1038/s41467-018-03523-2 [pii]\\nPST - epublish\\nSO  - Nat Commun. 2018 Mar 22;9(1):1188. doi: 10.1038/s41467-018-03523-2.\", \"PMID- 29518281\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20190812\\nLR  - 20191114\\nIS  - 1531-8249 (Electronic)\\nIS  - 0364-5134 (Linking)\\nVI  - 83\\nIP  - 6\\nDP  - 2018 Jun\\nTI  - Recessive mutations in VPS13D cause childhood onset movement disorders.\\nPG  - 1089-1095\\nLID - 10.1002/ana.25204 [doi]\\nAB  - VPS13 protein family members VPS13A through VPS13C have been associated with\\n      various recessive movement disorders. We describe the first disease association\\n      of rare recessive VPS13D variants including frameshift, missense, and partial\\n      duplication mutations with a novel complex, hyperkinetic neurological disorder.\\n      The clinical features include developmental delay, a childhood onset movement\\n      disorder (chorea, dystonia, or tremor), and progressive spastic ataxia or\\n      paraparesis. Characteristic brain magnetic resonance imaging shows basal ganglia \\n      or diffuse white matter T2 hyperintensities as seen in Leigh syndrome and\\n      choreoacanthocytosis. Muscle biopsy in 1 case showed mitochondrial aggregates and\\n      lipidosis, suggesting mitochondrial dysfunction. These findings underline the\\n      importance of the VPS13 complex in neurological diseases and a possible role in\\n      mitochondrial function. Ann Neurol 2018;83:1089-1095.\\nCI  - (c) 2018 American Neurological Association.\\nFAU - Gauthier, Julie\\nAU  - Gauthier J\\nAD  - Molecular Diagnostic Laboratory and Division of Medical Genetics, Department of\\n      Pediatrics, Saint Justine University Hospital Center, Montreal, Canada.\\nFAU - Meijer, Inge A\\nAU  - Meijer IA\\nAD  - Department of Neuroscience, University of Montreal, Montreal, Canada.\\nFAU - Lessel, Davor\\nAU  - Lessel D\\nAUID- ORCID: 0000-0003-4496-244X\\nAD  - Institute of Human Genetics, University Medical Center Hamburg-Eppendorf,\\n      Hamburg, Germany.\\nFAU - Mencacci, Niccolo E\\nAU  - Mencacci NE\\nAD  - Department of Neurology, Northwestern University, Feinberg School of Medicine,\\n      Chicago, IL.\\nFAU - Krainc, Dimitri\\nAU  - Krainc D\\nAD  - Department of Neurology, Northwestern University, Feinberg School of Medicine,\\n      Chicago, IL.\\nFAU - Hempel, Maja\\nAU  - Hempel M\\nAD  - Institute of Human Genetics, University Medical Center Hamburg-Eppendorf,\\n      Hamburg, Germany.\\nFAU - Tsiakas, Konstantinos\\nAU  - Tsiakas K\\nAD  - Department of Pediatrics, University Medical Center Hamburg-Eppendorf, Hamburg,\\n      Germany.\\nFAU - Prokisch, Holger\\nAU  - Prokisch H\\nAD  - Institute of Human Genetics, Helmholtz Center Munich, Neuherberg, Germany.\\nAD  - Institute of Human Genetics, Technical University Munich, Munich, Germany.\\nFAU - Rossignol, Elsa\\nAU  - Rossignol E\\nAD  - Department of Neuroscience, University of Montreal, Montreal, Canada.\\nFAU - Helm, Margaret H\\nAU  - Helm MH\\nAD  - Maine Medical Partners Pediatric Specialty Care, Portland, ME.\\nFAU - Rodan, Lance H\\nAU  - Rodan LH\\nAD  - Department of Neurology, Boston Children's Hospital, Boston, MA.\\nFAU - Karamchandani, Jason\\nAU  - Karamchandani J\\nAD  - Department of Pathology, McGill University, Montreal Neurological Institute,\\n      Montreal, Canada.\\nFAU - Carecchio, Miryam\\nAU  - Carecchio M\\nAD  - Molecular Neurogenetics Unit, Institute for Research and Health Care (IRCCS)\\n      Foundation Carlo Besta Neurological Institute, Milan, Italy.\\nFAU - Lubbe, Steven J\\nAU  - Lubbe SJ\\nAD  - Department of Neurology, Northwestern University, Feinberg School of Medicine,\\n      Chicago, IL.\\nFAU - Telegrafi, Aida\\nAU  - Telegrafi A\\nAD  - GeneDx, Gaithersburg, MD.\\nFAU - Henderson, Lindsay B\\nAU  - Henderson LB\\nAD  - GeneDx, Gaithersburg, MD.\\nFAU - Lorenzo, Kerry\\nAU  - Lorenzo K\\nAD  - Kadlec Clinic Genetic Counseling, Richland, WA.\\nFAU - Wallace, Stephanie E\\nAU  - Wallace SE\\nAD  - Division of Genetic Medicine, Department of Pediatrics, Seattle Children's\\n      Hospital and University of Washington, Seattle, WA.\\nFAU - Glass, Ian A\\nAU  - Glass IA\\nAD  - Division of Genetic Medicine, Department of Pediatrics, Seattle Children's\\n      Hospital and University of Washington, Seattle, WA.\\nFAU - Hamdan, Fadi F\\nAU  - Hamdan FF\\nAD  - Molecular Diagnostic Laboratory and Division of Medical Genetics, Department of\\n      Pediatrics, Saint Justine University Hospital Center, Montreal, Canada.\\nFAU - Michaud, Jacques L\\nAU  - Michaud JL\\nAD  - Department of Pediatrics, Saint Justine University Hospital Center and University\\n      of Montreal, Montreal, Canada.\\nFAU - Rouleau, Guy A\\nAU  - Rouleau GA\\nAD  - Montreal Neurological Institute, Department of Neurology and Neurosurgery, McGill\\n      University, Montreal, Canada.\\nFAU - Campeau, Philippe M\\nAU  - Campeau PM\\nAUID- ORCID: 0000-0001-9713-7107\\nAD  - Department of Pediatrics, Saint Justine University Hospital Center and University\\n      of Montreal, Montreal, Canada.\\nLA  - eng\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov't\\nDEP - 20180410\\nPL  - United States\\nTA  - Ann Neurol\\nJT  - Annals of neurology\\nJID - 7707449\\nRN  - 0 (Proteins)\\nRN  - 0 (VPS13D protein, human)\\nRN  - Spastic Ataxia\\nCIN - Ann Neurol. 2018 Jun;83(6):1072-1074. PMID: 29908061\\nMH  - Basal Ganglia/pathology\\nMH  - Brain/pathology\\nMH  - Child\\nMH  - Humans\\nMH  - Intellectual Disability/*genetics\\nMH  - Leigh Disease/pathology\\nMH  - Magnetic Resonance Imaging/methods\\nMH  - Movement Disorders/*genetics\\nMH  - Muscle Spasticity/*genetics/pathology\\nMH  - Mutation/*genetics\\nMH  - Optic Atrophy/*genetics\\nMH  - Pedigree\\nMH  - Proteins/*genetics\\nMH  - Spinocerebellar Ataxias/*genetics\\nEDAT- 2018/03/09 06:00\\nMHDA- 2019/08/14 06:00\\nCRDT- 2018/03/09 06:00\\nPHST- 2017/11/10 00:00 [received]\\nPHST- 2018/03/06 00:00 [revised]\\nPHST- 2018/03/07 00:00 [accepted]\\nPHST- 2018/03/09 06:00 [pubmed]\\nPHST- 2019/08/14 06:00 [medline]\\nPHST- 2018/03/09 06:00 [entrez]\\nAID - 10.1002/ana.25204 [doi]\\nPST - ppublish\\nSO  - Ann Neurol. 2018 Jun;83(6):1089-1095. doi: 10.1002/ana.25204. Epub 2018 Apr 10.\", \"PMID- 29507755\\nOWN - NLM\\nSTAT- PubMed-not-MEDLINE\\nLR  - 20181114\\nIS  - 2056-5968 (Print)\\nIS  - 2056-5968 (Linking)\\nVI  - 4\\nDP  - 2018\\nTI  - VIRMA mediates preferential m(6)A mRNA methylation in 3'UTR and near stop codon\\n      and associates with alternative polyadenylation.\\nPG  - 10\\nLID - 10.1038/s41421-018-0019-0 [doi]\\nAB  - N(6)-methyladenosine (m(6)A) is enriched in 3'untranslated region (3'UTR) and\\n      near stop codon of mature polyadenylated mRNAs in mammalian systems and has\\n      regulatory roles in eukaryotic mRNA transcriptome switch. Significantly, the\\n      mechanism for this modification preference remains unknown, however. Herein we\\n      report a characterization of the full m(6)A methyltransferase complex in HeLa\\n      cells identifying METTL3/METTL14/WTAP/VIRMA/HAKAI/ZC3H13 as the key components,\\n      and we show that VIRMA mediates preferential mRNA methylation in 3'UTR and near\\n      stop codon. Biochemical studies reveal that VIRMA recruits the catalytic core\\n      components METTL3/METTL14/WTAP to guide region-selective methylations. Around 60%\\n      of VIRMA mRNA immunoprecipitation targets manifest strong m(6)A enrichment in\\n      3'UTR. Depletions of VIRMA and METTL3 induce 3'UTR lengthening of several hundred\\n      mRNAs with over 50% targets in common. VIRMA associates with polyadenylation\\n      cleavage factors CPSF5 and CPSF6 in an RNA-dependent manner. Depletion of CPSF5\\n      leads to significant shortening of 3'UTR of over 2800 mRNAs, 84% of which are\\n      modified with m(6)A and have increased m(6)A peak density in 3'UTR and near stop \\n      codon after CPSF5 knockdown. Together, our studies provide insights into m(6)A\\n      deposition specificity in 3'UTR and its correlation with alternative\\n      polyadenylation.\\nFAU - Yue, Yanan\\nAU  - Yue Y\\nAD  - 1MOE Key Laboratory of Macromolecular Synthesis and Functionalization, Department\\n      of Polymer Science and Engineering, Zhejiang University, Hangzhou, Zhejiang\\n      310027 China.0000 0004 1759 700Xgrid.13402.34\\nFAU - Liu, Jun\\nAU  - Liu J\\nAD  - 2Department of Chemistry, Department of Biochemistry and Molecular Biology,\\n      Institute for Biophysical Dynamics, Howard Hughes Medical Institute, The\\n      University of Chicago, Chicago, IL 60637 USA.0000 0004 1936 7822grid.170205.1\\nFAU - Cui, Xiaolong\\nAU  - Cui X\\nAD  - 2Department of Chemistry, Department of Biochemistry and Molecular Biology,\\n      Institute for Biophysical Dynamics, Howard Hughes Medical Institute, The\\n      University of Chicago, Chicago, IL 60637 USA.0000 0004 1936 7822grid.170205.1\\nFAU - Cao, Jie\\nAU  - Cao J\\nAD  - 1MOE Key Laboratory of Macromolecular Synthesis and Functionalization, Department\\n      of Polymer Science and Engineering, Zhejiang University, Hangzhou, Zhejiang\\n      310027 China.0000 0004 1759 700Xgrid.13402.34\\nFAU - Luo, Guanzheng\\nAU  - Luo G\\nAUID- ORCID: 0000-0002-6797-9319\\nAD  - 2Department of Chemistry, Department of Biochemistry and Molecular Biology,\\n      Institute for Biophysical Dynamics, Howard Hughes Medical Institute, The\\n      University of Chicago, Chicago, IL 60637 USA.0000 0004 1936 7822grid.170205.1\\nFAU - Zhang, Zezhou\\nAU  - Zhang Z\\nAD  - 1MOE Key Laboratory of Macromolecular Synthesis and Functionalization, Department\\n      of Polymer Science and Engineering, Zhejiang University, Hangzhou, Zhejiang\\n      310027 China.0000 0004 1759 700Xgrid.13402.34\\nFAU - Cheng, Tao\\nAU  - Cheng T\\nAD  - 1MOE Key Laboratory of Macromolecular Synthesis and Functionalization, Department\\n      of Polymer Science and Engineering, Zhejiang University, Hangzhou, Zhejiang\\n      310027 China.0000 0004 1759 700Xgrid.13402.34\\nFAU - Gao, Minsong\\nAU  - Gao M\\nAD  - 1MOE Key Laboratory of Macromolecular Synthesis and Functionalization, Department\\n      of Polymer Science and Engineering, Zhejiang University, Hangzhou, Zhejiang\\n      310027 China.0000 0004 1759 700Xgrid.13402.34\\nFAU - Shu, Xiao\\nAU  - Shu X\\nAD  - 1MOE Key Laboratory of Macromolecular Synthesis and Functionalization, Department\\n      of Polymer Science and Engineering, Zhejiang University, Hangzhou, Zhejiang\\n      310027 China.0000 0004 1759 700Xgrid.13402.34\\nFAU - Ma, Honghui\\nAU  - Ma H\\nAD  - 2Department of Chemistry, Department of Biochemistry and Molecular Biology,\\n      Institute for Biophysical Dynamics, Howard Hughes Medical Institute, The\\n      University of Chicago, Chicago, IL 60637 USA.0000 0004 1936 7822grid.170205.1\\nFAU - Wang, Fengqin\\nAU  - Wang F\\nAD  - 3College of Animal Sciences, Key Laboratory of Molecular Nutrition, Ministry of\\n      Education, Zhejiang University, Hangzhou, Zhejiang 310058 China.0000 0004 1759\\n      700Xgrid.13402.34\\nFAU - Wang, Xinxia\\nAU  - Wang X\\nAD  - 3College of Animal Sciences, Key Laboratory of Molecular Nutrition, Ministry of\\n      Education, Zhejiang University, Hangzhou, Zhejiang 310058 China.0000 0004 1759\\n      700Xgrid.13402.34\\nFAU - Shen, Bin\\nAU  - Shen B\\nAD  - 4State Key Laboratory of Reproductive Medicine, Department of Histology and\\n      Embryology, Nanjing Medical University, Nanjing, Jiangsu 210029 China.0000 0000\\n      9255 8984grid.89957.3a\\nFAU - Wang, Yizhen\\nAU  - Wang Y\\nAD  - 3College of Animal Sciences, Key Laboratory of Molecular Nutrition, Ministry of\\n      Education, Zhejiang University, Hangzhou, Zhejiang 310058 China.0000 0004 1759\\n      700Xgrid.13402.34\\nFAU - Feng, Xinhua\\nAU  - Feng X\\nAD  - 5Life Sciences Institute, Zhejiang University, Hangzhou, Zhejiang 310058\\n      China.0000 0004 1759 700Xgrid.13402.34\\nFAU - He, Chuan\\nAU  - He C\\nAUID- ORCID: 0000-0003-4319-7424\\nAD  - 2Department of Chemistry, Department of Biochemistry and Molecular Biology,\\n      Institute for Biophysical Dynamics, Howard Hughes Medical Institute, The\\n      University of Chicago, Chicago, IL 60637 USA.0000 0004 1936 7822grid.170205.1\\nFAU - Liu, Jianzhao\\nAU  - Liu J\\nAUID- ORCID: 0000-0001-9465-6075\\nAD  - 1MOE Key Laboratory of Macromolecular Synthesis and Functionalization, Department\\n      of Polymer Science and Engineering, Zhejiang University, Hangzhou, Zhejiang\\n      310027 China.0000 0004 1759 700Xgrid.13402.34\\nAD  - 5Life Sciences Institute, Zhejiang University, Hangzhou, Zhejiang 310058\\n      China.0000 0004 1759 700Xgrid.13402.34\\nLA  - eng\\nGR  - R01 HG008688/HG/NHGRI NIH HHS/United States\\nPT  - Journal Article\\nDEP - 20180227\\nPL  - England\\nTA  - Cell Discov\\nJT  - Cell discovery\\nJID - 101661034\\nPMC - PMC5826926\\nCOIS- C.H. is a scientific founder of the Accent Therapeutics and a member of the\\n      Scientific Advisory Committee. The remaining authors declare that they have no\\n      conflict of interest.\\nEDAT- 2018/03/07 06:00\\nMHDA- 2018/03/07 06:01\\nCRDT- 2018/03/07 06:00\\nPHST- 2017/11/24 00:00 [received]\\nPHST- 2018/02/02 00:00 [revised]\\nPHST- 2018/02/04 00:00 [accepted]\\nPHST- 2018/03/07 06:00 [entrez]\\nPHST- 2018/03/07 06:00 [pubmed]\\nPHST- 2018/03/07 06:01 [medline]\\nAID - 10.1038/s41421-018-0019-0 [doi]\\nAID - 19 [pii]\\nPST - epublish\\nSO  - Cell Discov. 2018 Feb 27;4:10. doi: 10.1038/s41421-018-0019-0. eCollection 2018.\", \"PMID- 29307555\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20190514\\nLR  - 20190517\\nIS  - 1879-0445 (Electronic)\\nIS  - 0960-9822 (Linking)\\nVI  - 28\\nIP  - 2\\nDP  - 2018 Jan 22\\nTI  - Vps13D Encodes a Ubiquitin-Binding Protein that Is Required for the Regulation of\\n      Mitochondrial Size and Clearance.\\nPG  - 287-295.e6\\nLID - S0960-9822(17)31589-0 [pii]\\nLID - 10.1016/j.cub.2017.11.064 [doi]\\nAB  - The clearance of mitochondria by autophagy, mitophagy, is important for cell and \\n      organism health [1], and known to be regulated by ubiquitin. During Drosophila\\n      intestine development, cells undergo a dramatic reduction in cell size and\\n      clearance of mitochondria that depends on autophagy, the E1 ubiquitin-activating \\n      enzyme Uba1, and ubiquitin [2]. Here we screen a collection of putative\\n      ubiquitin-binding domain-encoding genes for cell size reduction and autophagy\\n      phenotypes. We identify the endosomal sorting complex required for transport\\n      (ESCRT) components TSG101 and Vps36, as well as the novel gene Vps13D. Vps13D is \\n      an essential gene that is necessary for autophagy, mitochondrial size, and\\n      mitochondrial clearance in Drosophila. Interestingly, a similar mitochondrial\\n      phenotype is observed in VPS13D mutant human cells. The ubiquitin-associated\\n      (UBA) domain of Vps13D binds K63 ubiquitin chains, and mutants lacking the UBA\\n      domain have defects in mitochondrial size and clearance and exhibit\\n      semi-lethality, highlighting the importance of Vps13D ubiquitin binding in both\\n      mitochondrial health and development. VPS13D mutant cells possess phosphorylated \\n      DRP1 and mitochondrial fission factor (MFF) as well as DRP1 association with\\n      mitochondria, suggesting that VPS13D functions downstream of these known\\n      regulators of mitochondrial fission. In addition, the large Vps13D mitochondrial \\n      and cell size phenotypes are suppressed by decreased mitochondrial fusion gene\\n      function. Thus, these results provide a previously unknown link between\\n      ubiquitin, mitochondrial size regulation, and autophagy.\\nCI  - Copyright (c) 2017 Elsevier Ltd. All rights reserved.\\nFAU - Anding, Allyson L\\nAU  - Anding AL\\nAD  - Department of Molecular, Cell and Cancer Biology, University of Massachusetts\\n      Medical School, Worcester, MA 01605, USA.\\nFAU - Wang, Chunxin\\nAU  - Wang C\\nAD  - Biochemistry Section, Surgical Neurology Branch, National Institute of\\n      Neurological Disorders and Stroke, NIH, Bethesda, MD 20892, USA.\\nFAU - Chang, Tsun-Kai\\nAU  - Chang TK\\nAD  - Department of Molecular, Cell and Cancer Biology, University of Massachusetts\\n      Medical School, Worcester, MA 01605, USA.\\nFAU - Sliter, Danielle A\\nAU  - Sliter DA\\nAD  - Biochemistry Section, Surgical Neurology Branch, National Institute of\\n      Neurological Disorders and Stroke, NIH, Bethesda, MD 20892, USA.\\nFAU - Powers, Christine M\\nAU  - Powers CM\\nAD  - Department of Molecular, Cell and Cancer Biology, University of Massachusetts\\n      Medical School, Worcester, MA 01605, USA.\\nFAU - Hofmann, Kay\\nAU  - Hofmann K\\nAD  - Institute for Genetics, University of Cologne, 50674 Cologne, Germany.\\nFAU - Youle, Richard J\\nAU  - Youle RJ\\nAD  - Biochemistry Section, Surgical Neurology Branch, National Institute of\\n      Neurological Disorders and Stroke, NIH, Bethesda, MD 20892, USA.\\nFAU - Baehrecke, Eric H\\nAU  - Baehrecke EH\\nAD  - Department of Molecular, Cell and Cancer Biology, University of Massachusetts\\n      Medical School, Worcester, MA 01605, USA. Electronic address:\\n      eric.baehrecke@umassmed.edu.\\nLA  - eng\\nGR  - R01 GM111658/GM/NIGMS NIH HHS/United States\\nGR  - S10 RR027897/RR/NCRR NIH HHS/United States\\nPT  - Journal Article\\nPT  - Research Support, N.I.H., Extramural\\nPT  - Research Support, N.I.H., Intramural\\nDEP - 20180104\\nPL  - England\\nTA  - Curr Biol\\nJT  - Current biology : CB\\nJID - 9107782\\nRN  - 0 (Carrier Proteins)\\nRN  - 0 (Drosophila Proteins)\\nRN  - 0 (Intracellular Signaling Peptides and Proteins)\\nRN  - 0 (Ubiquitin)\\nRN  - 0 (Vps13D protein, Drosophila)\\nSB  - IM\\nCIN - Curr Biol. 2018 Jan 22;28(2):R66-R68. PMID: 29374445\\nMH  - Animals\\nMH  - Carrier Proteins/genetics/metabolism\\nMH  - Drosophila Proteins/*genetics/metabolism\\nMH  - Drosophila melanogaster/genetics/*physiology\\nMH  - Intracellular Signaling Peptides and Proteins/*genetics/metabolism\\nMH  - Mitochondria/physiology\\nMH  - Mitochondrial Degradation/*genetics\\nMH  - Mitochondrial Size/*genetics\\nMH  - Ubiquitin/metabolism\\nMH  - Ubiquitination\\nPMC - PMC5787036\\nMID - NIHMS925912\\nOTO - NOTNLM\\nOT  - *Drosophila\\nOT  - *Vps13D\\nOT  - *autophagy\\nOT  - *fission\\nOT  - *mitochondria\\nOT  - *mitophagy\\nOT  - *ubiquitin\\nEDAT- 2018/01/09 06:00\\nMHDA- 2019/05/15 06:00\\nCRDT- 2018/01/09 06:00\\nPHST- 2017/10/30 00:00 [received]\\nPHST- 2017/11/21 00:00 [revised]\\nPHST- 2017/11/28 00:00 [accepted]\\nPHST- 2018/01/09 06:00 [pubmed]\\nPHST- 2019/05/15 06:00 [medline]\\nPHST- 2018/01/09 06:00 [entrez]\\nAID - S0960-9822(17)31589-0 [pii]\\nAID - 10.1016/j.cub.2017.11.064 [doi]\\nPST - ppublish\\nSO  - Curr Biol. 2018 Jan 22;28(2):287-295.e6. doi: 10.1016/j.cub.2017.11.064. Epub\\n      2018 Jan 4.\", \"PMID- 26344197\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20151014\\nLR  - 20181113\\nIS  - 1476-4687 (Electronic)\\nIS  - 0028-0836 (Linking)\\nVI  - 525\\nIP  - 7569\\nDP  - 2015 Sep 17\\nTI  - Panorama of ancient metazoan macromolecular complexes.\\nPG  - 339-44\\nLID - 10.1038/nature14877 [doi]\\nAB  - Macromolecular complexes are essential to conserved biological processes, but\\n      their prevalence across animals is unclear. By combining extensive biochemical\\n      fractionation with quantitative mass spectrometry, here we directly examined the \\n      composition of soluble multiprotein complexes among diverse metazoan models.\\n      Using an integrative approach, we generated a draft conservation map consisting\\n      of more than one million putative high-confidence co-complex interactions for\\n      species with fully sequenced genomes that encompasses functional modules present \\n      broadly across all extant animals. Clustering reveals a spectrum of conservation,\\n      ranging from ancient eukaryotic assemblies that have probably served cellular\\n      housekeeping roles for at least one billion years, ancestral complexes that have \\n      accrued contemporary components, and rarer metazoan innovations linked to\\n      multicellularity. We validated these projections by independent co-fractionation \\n      experiments in evolutionarily distant species, affinity purification and\\n      functional analyses. The comprehensiveness, centrality and modularity of these\\n      reconstructed interactomes reflect their fundamental mechanistic importance and\\n      adaptive value to animal cell systems.\\nFAU - Wan, Cuihong\\nAU  - Wan C\\nAD  - Donnelly Centre for Cellular and Biomolecular Research, University of Toronto,\\n      Toronto, Ontario M5S 3E1, Canada.\\nAD  - Center for Systems and Synthetic Biology, Institute for Cellular and Molecular\\n      Biology, University of Texas at Austin, Austin, Texas 78712, USA.\\nFAU - Borgeson, Blake\\nAU  - Borgeson B\\nAD  - Center for Systems and Synthetic Biology, Institute for Cellular and Molecular\\n      Biology, University of Texas at Austin, Austin, Texas 78712, USA.\\nFAU - Phanse, Sadhna\\nAU  - Phanse S\\nAD  - Donnelly Centre for Cellular and Biomolecular Research, University of Toronto,\\n      Toronto, Ontario M5S 3E1, Canada.\\nFAU - Tu, Fan\\nAU  - Tu F\\nAD  - Center for Systems and Synthetic Biology, Institute for Cellular and Molecular\\n      Biology, University of Texas at Austin, Austin, Texas 78712, USA.\\nFAU - Drew, Kevin\\nAU  - Drew K\\nAD  - Center for Systems and Synthetic Biology, Institute for Cellular and Molecular\\n      Biology, University of Texas at Austin, Austin, Texas 78712, USA.\\nFAU - Clark, Greg\\nAU  - Clark G\\nAD  - Department of Medical Biophysics, Toronto, Ontario M5G 1L7, Canada.\\nFAU - Xiong, Xuejian\\nAU  - Xiong X\\nAD  - Department of Molecular Genetics, University of Toronto, Toronto, Ontario M5S\\n      1A8, Canada.\\nAD  - Hospital for Sick Children, Toronto, Ontario M5G 1X8, Canada.\\nFAU - Kagan, Olga\\nAU  - Kagan O\\nAD  - Donnelly Centre for Cellular and Biomolecular Research, University of Toronto,\\n      Toronto, Ontario M5S 3E1, Canada.\\nFAU - Kwan, Julian\\nAU  - Kwan J\\nAD  - Donnelly Centre for Cellular and Biomolecular Research, University of Toronto,\\n      Toronto, Ontario M5S 3E1, Canada.\\nAD  - Department of Molecular Genetics, University of Toronto, Toronto, Ontario M5S\\n      1A8, Canada.\\nFAU - Bezginov, Alexandr\\nAU  - Bezginov A\\nAD  - Department of Medical Biophysics, Toronto, Ontario M5G 1L7, Canada.\\nFAU - Chessman, Kyle\\nAU  - Chessman K\\nAD  - Department of Molecular Genetics, University of Toronto, Toronto, Ontario M5S\\n      1A8, Canada.\\nAD  - Hospital for Sick Children, Toronto, Ontario M5G 1X8, Canada.\\nFAU - Pal, Swati\\nAU  - Pal S\\nAD  - Hospital for Sick Children, Toronto, Ontario M5G 1X8, Canada.\\nFAU - Cromar, Graham\\nAU  - Cromar G\\nAD  - Department of Molecular Genetics, University of Toronto, Toronto, Ontario M5S\\n      1A8, Canada.\\nAD  - Hospital for Sick Children, Toronto, Ontario M5G 1X8, Canada.\\nFAU - Papoulas, Ophelia\\nAU  - Papoulas O\\nAD  - Center for Systems and Synthetic Biology, Institute for Cellular and Molecular\\n      Biology, University of Texas at Austin, Austin, Texas 78712, USA.\\nFAU - Ni, Zuyao\\nAU  - Ni Z\\nAD  - Donnelly Centre for Cellular and Biomolecular Research, University of Toronto,\\n      Toronto, Ontario M5S 3E1, Canada.\\nFAU - Boutz, Daniel R\\nAU  - Boutz DR\\nAD  - Center for Systems and Synthetic Biology, Institute for Cellular and Molecular\\n      Biology, University of Texas at Austin, Austin, Texas 78712, USA.\\nFAU - Stoilova, Snejana\\nAU  - Stoilova S\\nAD  - Donnelly Centre for Cellular and Biomolecular Research, University of Toronto,\\n      Toronto, Ontario M5S 3E1, Canada.\\nFAU - Havugimana, Pierre C\\nAU  - Havugimana PC\\nAD  - Donnelly Centre for Cellular and Biomolecular Research, University of Toronto,\\n      Toronto, Ontario M5S 3E1, Canada.\\nFAU - Guo, Xinghua\\nAU  - Guo X\\nAD  - Donnelly Centre for Cellular and Biomolecular Research, University of Toronto,\\n      Toronto, Ontario M5S 3E1, Canada.\\nFAU - Malty, Ramy H\\nAU  - Malty RH\\nAD  - Department of Biochemistry, University of Regina, Regina, Saskatchewan S4S 0A2,\\n      Canada.\\nFAU - Sarov, Mihail\\nAU  - Sarov M\\nAD  - Max Planck Institute of Molecular Cell Biology and Genetics, 01307 Dresden,\\n      Germany.\\nFAU - Greenblatt, Jack\\nAU  - Greenblatt J\\nAD  - Donnelly Centre for Cellular and Biomolecular Research, University of Toronto,\\n      Toronto, Ontario M5S 3E1, Canada.\\nAD  - Department of Molecular Genetics, University of Toronto, Toronto, Ontario M5S\\n      1A8, Canada.\\nFAU - Babu, Mohan\\nAU  - Babu M\\nAD  - Department of Biochemistry, University of Regina, Regina, Saskatchewan S4S 0A2,\\n      Canada.\\nFAU - Derry, W Brent\\nAU  - Derry WB\\nAD  - Department of Molecular Genetics, University of Toronto, Toronto, Ontario M5S\\n      1A8, Canada.\\nAD  - Hospital for Sick Children, Toronto, Ontario M5G 1X8, Canada.\\nFAU - Tillier, Elisabeth R\\nAU  - Tillier ER\\nAD  - Department of Medical Biophysics, Toronto, Ontario M5G 1L7, Canada.\\nFAU - Wallingford, John B\\nAU  - Wallingford JB\\nAD  - Center for Systems and Synthetic Biology, Institute for Cellular and Molecular\\n      Biology, University of Texas at Austin, Austin, Texas 78712, USA.\\nAD  - Department of Molecular Biosciences, University of Texas at Austin, Austin, Texas\\n      78712, USA.\\nFAU - Parkinson, John\\nAU  - Parkinson J\\nAD  - Department of Molecular Genetics, University of Toronto, Toronto, Ontario M5S\\n      1A8, Canada.\\nAD  - Hospital for Sick Children, Toronto, Ontario M5G 1X8, Canada.\\nFAU - Marcotte, Edward M\\nAU  - Marcotte EM\\nAD  - Center for Systems and Synthetic Biology, Institute for Cellular and Molecular\\n      Biology, University of Texas at Austin, Austin, Texas 78712, USA.\\nAD  - Department of Molecular Biosciences, University of Texas at Austin, Austin, Texas\\n      78712, USA.\\nFAU - Emili, Andrew\\nAU  - Emili A\\nAD  - Donnelly Centre for Cellular and Biomolecular Research, University of Toronto,\\n      Toronto, Ontario M5S 3E1, Canada.\\nAD  - Department of Molecular Genetics, University of Toronto, Toronto, Ontario M5S\\n      1A8, Canada.\\nLA  - eng\\nGR  - F32GM112495/GM/NIGMS NIH HHS/United States\\nGR  - R01 GM076536/GM/NIGMS NIH HHS/United States\\nGR  - DP1 GM106408/GM/NIGMS NIH HHS/United States\\nGR  - R01 GM067779/GM/NIGMS NIH HHS/United States\\nGR  - Canadian Institutes of Health Research/Canada\\nGR  - F32 GM112495/GM/NIGMS NIH HHS/United States\\nPT  - Journal Article\\nPT  - Research Support, N.I.H., Extramural\\nPT  - Research Support, Non-U.S. Gov't\\nPT  - Research Support, U.S. Gov't, Non-P.H.S.\\nDEP - 20150907\\nPL  - England\\nTA  - Nature\\nJT  - Nature\\nJID - 0410462\\nRN  - 0 (Multiprotein Complexes)\\nSB  - IM\\nCIN - Trends Biochem Sci. 2016 Jan;41(1):4-6. PMID: 26682499\\nCIN - Nat Methods. 2015 Nov;12(11):1011. PMID: 26824107\\nMH  - Animals\\nMH  - Datasets as Topic\\nMH  - *Evolution, Molecular\\nMH  - Humans\\nMH  - Multiprotein Complexes/*chemistry/*metabolism\\nMH  - Protein Interaction Mapping\\nMH  - *Protein Interaction Maps\\nMH  - Reproducibility of Results\\nMH  - Systems Biology\\nMH  - Tandem Mass Spectrometry\\nPMC - PMC5036527\\nMID - NIHMS704400\\nEDAT- 2015/09/08 06:00\\nMHDA- 2015/10/16 06:00\\nCRDT- 2015/09/08 06:00\\nPHST- 2014/12/15 00:00 [received]\\nPHST- 2015/06/30 00:00 [accepted]\\nPHST- 2015/09/08 06:00 [entrez]\\nPHST- 2015/09/08 06:00 [pubmed]\\nPHST- 2015/10/16 06:00 [medline]\\nAID - nature14877 [pii]\\nAID - 10.1038/nature14877 [doi]\\nPST - ppublish\\nSO  - Nature. 2015 Sep 17;525(7569):339-44. doi: 10.1038/nature14877. Epub 2015 Sep 7.\", \"PMID- 25896417\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20160620\\nLR  - 20150828\\nIS  - 1662-8128 (Electronic)\\nIS  - 1662-811X (Linking)\\nVI  - 7\\nIP  - 5\\nDP  - 2015\\nTI  - VPS13D Gene Variant Is Associated with Altered IL-6 Production and Mortality in\\n      Septic Shock.\\nPG  - 545-53\\nLID - 10.1159/000381265 [doi]\\nAB  - BACKGROUND: Genetic variations contribute to septic shock mortality. To discover \\n      a novel locus, we performed in vitro genome-wide association studies (GWAS) and\\n      further tested the result in a cohort of septic shock patients. METHODS: Two in\\n      vitro GWAS using a quantitative trait locus analysis of stimulated IL-6\\n      production in lymphoblastoid cells from 60 individuals of European ancestry were \\n      performed. VPS13D rs6685273 was genotyped in European ancestry patients (n =\\n      498). The VPS13D gene was silenced in vitro. RESULTS: Two GWAS using\\n      lymphoblastoid cells identified the locus of VPS13D rs6685273 that was\\n      significant in the same direction in both GWAS. The VPS13D rs6685273 C allele was\\n      associated with increased IL-6 production. Patients with septic shock who had the\\n      VPS13D rs6685273 CC genotype had an increased 28-day mortality (p = 0.023) and\\n      more organ failure (p < 0.05) compared to the CT/TT genotypes. VPS13D in vitro\\n      gene silencing in the HeLa cell line increased IL-6 production. Furthermore, the \\n      rs6685273 genotype was associated with differential VPS13D splice variant\\n      expression. CONCLUSIONS: The VPS13D rs6685273 C allele was associated with\\n      increased IL-6 production in vitro. The patients with the VPS13D rs6685273 CC\\n      genotype had increased 28-day mortality and increased organ failure. VPS13D\\n      appears to regulate IL-6 production.\\nCI  - (c) 2015 S. Karger AG, Basel.\\nFAU - Nakada, Taka-aki\\nAU  - Nakada TA\\nAD  - Critical Care Research Laboratories, Centre for Heart Lung Innovation, St. Paul's\\n      Hospital, University of British Columbia, Vancouver, B.C., Canada.\\nFAU - Boyd, John H\\nAU  - Boyd JH\\nFAU - Russell, James A\\nAU  - Russell JA\\nFAU - Aguirre-Hernandez, Rosalia\\nAU  - Aguirre-Hernandez R\\nFAU - Wilkinson, Mark D\\nAU  - Wilkinson MD\\nFAU - Thair, Simone A\\nAU  - Thair SA\\nFAU - Nakada, Emiri\\nAU  - Nakada E\\nFAU - McConechy, Melissa K\\nAU  - McConechy MK\\nFAU - Fjell, Christopher D\\nAU  - Fjell CD\\nFAU - Walley, Keith R\\nAU  - Walley KR\\nLA  - eng\\nGR  - Canadian Institutes of Health Research/Canada\\nPT  - Journal Article\\nPT  - Multicenter Study\\nPT  - Randomized Controlled Trial\\nPT  - Research Support, Non-U.S. Gov't\\nDEP - 20150417\\nPL  - Switzerland\\nTA  - J Innate Immun\\nJT  - Journal of innate immunity\\nJID - 101469471\\nRN  - 0 (Interleukin-6)\\nRN  - 0 (Proteins)\\nRN  - 0 (VPS13D protein, human)\\nSB  - IM\\nMH  - Aged\\nMH  - Cohort Studies\\nMH  - European Continental Ancestry Group/genetics\\nMH  - Female\\nMH  - Gene Frequency\\nMH  - Genetic Predisposition to Disease\\nMH  - Genome-Wide Association Study\\nMH  - Genotype\\nMH  - HeLa Cells\\nMH  - Humans\\nMH  - Interleukin-6/genetics/*metabolism\\nMH  - Lymphocyte Activation/genetics\\nMH  - Lymphocytes/*physiology\\nMH  - Male\\nMH  - Middle Aged\\nMH  - Polymorphism, Single Nucleotide\\nMH  - Proteins/*genetics\\nMH  - Shock, Septic/*genetics/immunology/*mortality\\nMH  - Spain\\nMH  - Survival Analysis\\nEDAT- 2015/04/22 06:00\\nMHDA- 2016/06/21 06:00\\nCRDT- 2015/04/22 06:00\\nPHST- 2014/12/17 00:00 [received]\\nPHST- 2015/02/26 00:00 [accepted]\\nPHST- 2015/04/22 06:00 [entrez]\\nPHST- 2015/04/22 06:00 [pubmed]\\nPHST- 2016/06/21 06:00 [medline]\\nAID - 000381265 [pii]\\nAID - 10.1159/000381265 [doi]\\nPST - ppublish\\nSO  - J Innate Immun. 2015;7(5):545-53. doi: 10.1159/000381265. Epub 2015 Apr 17.\", \"PMID- 21873635\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20120403\\nLR  - 20181113\\nIS  - 1477-4054 (Electronic)\\nIS  - 1467-5463 (Linking)\\nVI  - 12\\nIP  - 5\\nDP  - 2011 Sep\\nTI  - Phylogenetic-based propagation of functional annotations within the Gene Ontology\\n      consortium.\\nPG  - 449-62\\nLID - 10.1093/bib/bbr042 [doi]\\nAB  - The goal of the Gene Ontology (GO) project is to provide a uniform way to\\n      describe the functions of gene products from organisms across all kingdoms of\\n      life and thereby enable analysis of genomic data. Protein annotations are either \\n      based on experiments or predicted from protein sequences. Since most sequences\\n      have not been experimentally characterized, most available annotations need to be\\n      based on predictions. To make as accurate inferences as possible, the GO\\n      Consortium's Reference Genome Project is using an explicit evolutionary framework\\n      to infer annotations of proteins from a broad set of genomes from experimental\\n      annotations in a semi-automated manner. Most components in the pipeline, such as \\n      selection of sequences, building multiple sequence alignments and phylogenetic\\n      trees, retrieving experimental annotations and depositing inferred annotations,\\n      are fully automated. However, the most crucial step in our pipeline relies on\\n      software-assisted curation by an expert biologist. This curation tool,\\n      Phylogenetic Annotation and INference Tool (PAINT) helps curators to infer\\n      annotations among members of a protein family. PAINT allows curators to make\\n      precise assertions as to when functions were gained and lost during evolution and\\n      record the evidence (e.g. experimentally supported GO annotations and\\n      phylogenetic information including orthology) for those assertions. In this\\n      article, we describe how we use PAINT to infer protein function in a phylogenetic\\n      context with emphasis on its strengths, limitations and guidelines. We also\\n      discuss specific examples showing how PAINT annotations compare with those\\n      generated by other highly used homology-based methods.\\nFAU - Gaudet, Pascale\\nAU  - Gaudet P\\nAD  - Swiss Institute for Bioinformatics, CMU, 1 Rue Michel Servet, 1211 Geneva 4,\\n      Switzerland. pascale.gaudet@isb-sib.ch\\nFAU - Livstone, Michael S\\nAU  - Livstone MS\\nFAU - Lewis, Suzanna E\\nAU  - Lewis SE\\nFAU - Thomas, Paul D\\nAU  - Thomas PD\\nLA  - eng\\nGR  - P41-HG002273/HG/NHGRI NIH HHS/United States\\nGR  - R01-GM081084/GM/NIGMS NIH HHS/United States\\nPT  - Journal Article\\nPT  - Research Support, N.I.H., Extramural\\nDEP - 20110827\\nPL  - England\\nTA  - Brief Bioinform\\nJT  - Briefings in bioinformatics\\nJID - 100912837\\nRN  - 0 (Proteins)\\nSB  - IM\\nMH  - Databases, Genetic\\nMH  - Genome\\nMH  - Genomics/*methods\\nMH  - Molecular Sequence Annotation/*methods\\nMH  - *Phylogeny\\nMH  - Proteins/*chemistry/genetics\\nPMC - PMC3178059\\nEDAT- 2011/08/30 06:00\\nMHDA- 2012/04/04 06:00\\nCRDT- 2011/08/30 06:00\\nPHST- 2011/08/30 06:00 [entrez]\\nPHST- 2011/08/30 06:00 [pubmed]\\nPHST- 2012/04/04 06:00 [medline]\\nAID - bbr042 [pii]\\nAID - 10.1093/bib/bbr042 [doi]\\nPST - ppublish\\nSO  - Brief Bioinform. 2011 Sep;12(5):449-62. doi: 10.1093/bib/bbr042. Epub 2011 Aug\\n      27.\", \"PMID- 21182205\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20110407\\nLR  - 20101224\\nIS  - 1615-9861 (Electronic)\\nIS  - 1615-9853 (Linking)\\nVI  - 11\\nIP  - 1\\nDP  - 2011 Jan\\nTI  - Identification of proteins associated with ligand-activated estrogen receptor\\n      alpha in human breast cancer cell nuclei by tandem affinity purification and nano\\n      LC-MS/MS.\\nPG  - 172-9\\nLID - 10.1002/pmic.201000217 [doi]\\nAB  - Estrogen receptor alpha (ER-alpha) is a key mediator of estrogen actions in\\n      breast cancer (BC) cells. Understanding the effects of ligand-activated ER-alpha \\n      in target cells requires identification of the molecular partners acting in\\n      concert with this nuclear receptor to transduce the hormonal signal. We applied\\n      tandem affinity purification (TAP), glycerol gradient centrifugation and MS\\n      analysis to isolate and identify proteins interacting with ligand-activated\\n      ER-alpha in MCF-7 cell nuclei. This led to the identification of 264\\n      ER-associated proteins, whose functions highlight the hinge role of ER-alpha in\\n      the coordination of multiple hormone-regulated nuclear processes in BC cells.\\nCI  - Copyright (c) 2011 WILEY-VCH Verlag GmbH & Co. KGaA, Weinheim.\\nFAU - Tarallo, Roberta\\nAU  - Tarallo R\\nAD  - Department of General Pathology, Second University of Naples, Napoli, Italy.\\nFAU - Bamundo, Angela\\nAU  - Bamundo A\\nFAU - Nassa, Giovanni\\nAU  - Nassa G\\nFAU - Nola, Ernesto\\nAU  - Nola E\\nFAU - Paris, Ornella\\nAU  - Paris O\\nFAU - Ambrosino, Concetta\\nAU  - Ambrosino C\\nFAU - Facchiano, Angelo\\nAU  - Facchiano A\\nFAU - Baumann, Marc\\nAU  - Baumann M\\nFAU - Nyman, Tuula A\\nAU  - Nyman TA\\nFAU - Weisz, Alessandro\\nAU  - Weisz A\\nLA  - eng\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov't\\nDEP - 20101201\\nPL  - Germany\\nTA  - Proteomics\\nJT  - Proteomics\\nJID - 101092707\\nRN  - 0 (Estrogen Receptor alpha)\\nSB  - IM\\nMH  - Breast Neoplasms/*metabolism\\nMH  - Cell Line, Tumor\\nMH  - Chromatography, Affinity\\nMH  - Chromatography, Liquid/*methods\\nMH  - Estrogen Receptor alpha/*metabolism\\nMH  - Female\\nMH  - Humans\\nMH  - Tandem Mass Spectrometry/*methods\\nEDAT- 2010/12/25 06:00\\nMHDA- 2011/04/08 06:00\\nCRDT- 2010/12/25 06:00\\nPHST- 2010/04/02 00:00 [received]\\nPHST- 2010/08/03 00:00 [revised]\\nPHST- 2010/10/04 00:00 [accepted]\\nPHST- 2010/12/25 06:00 [entrez]\\nPHST- 2010/12/25 06:00 [pubmed]\\nPHST- 2011/04/08 06:00 [medline]\\nAID - 10.1002/pmic.201000217 [doi]\\nPST - ppublish\\nSO  - Proteomics. 2011 Jan;11(1):172-9. doi: 10.1002/pmic.201000217. Epub 2010 Dec 1.\", \"PMID- 20610895\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20110308\\nLR  - 20190608\\nIS  - 1880-3873 (Electronic)\\nIS  - 1340-3478 (Linking)\\nVI  - 17\\nIP  - 10\\nDP  - 2010 Oct 27\\nTI  - Identification of evidence suggestive of an association with peripheral arterial \\n      disease at the OSBPL10 locus by genome-wide investigation in the Japanese\\n      population.\\nPG  - 1054-62\\nAB  - AIM: Peripheral arterial disease (PAD) is a common cause of cardiovascular\\n      morbidity and an independent predictor of cardiovascular mortality. However,\\n      little is known about the genetic basis of PAD. To elucidate this, we performed a\\n      two-staged genome-wide association study in Japanese individuals. METHODS: We\\n      initially tested 222,285 single-nucleotide polymorphisms (SNPs). After the first \\n      screening in a panel of 195 PAD cases and 1,358 controls, 2,696 SNPs (1.2%) were \\n      further genotyped in the second screening using another panel of 699 PAD cases\\n      and 1540 controls. In both screenings, controls were subjects affected with some \\n      diseases other than PAD. RESULTS: When analyzed in the combined panel, the\\n      strongest signal of PAD association was observed at rs1902341 in the intron of\\n      OSBPL10 (p=4.7E-7 for trend test; OR=1.31, 95% CI 1.18-1.46). Also, PAD was\\n      modestly associated at several other loci such as rs2554503 in CSMD1 (p=5.7E-5;\\n      OR=1.32, 95% CI 1.15-1.51) or rs235243 in VSP13D (p=0.04; OR=1.18, 95% CI\\n      1.01-1.37). CONCLUSION: Our genome-wide exploration identified suggestive\\n      evidence of PAD association at the OSBPL10 locus. Because the association has not\\n      reached a genome-wide significant level, further replication study is warranted\\n      for verification in the Japanese population.\\nFAU - Koriyama, Hiroshi\\nAU  - Koriyama H\\nAD  - Department of Geriatric Medicine and Nephrology, Osaka University Graduate School\\n      of Medicine, 2-2 Yamada-oka, Suita, Japan.\\nFAU - Nakagami, Hironori\\nAU  - Nakagami H\\nFAU - Katsuya, Tomohiro\\nAU  - Katsuya T\\nFAU - Sugimoto, Ken\\nAU  - Sugimoto K\\nFAU - Yamashita, Hidetoshi\\nAU  - Yamashita H\\nFAU - Takami, Yoichi\\nAU  - Takami Y\\nFAU - Maeda, Shiro\\nAU  - Maeda S\\nFAU - Kubo, Michiaki\\nAU  - Kubo M\\nFAU - Takahashi, Atsushi\\nAU  - Takahashi A\\nFAU - Nakamura, Yusuke\\nAU  - Nakamura Y\\nFAU - Ogihara, Toshio\\nAU  - Ogihara T\\nFAU - Rakugi, Hiromi\\nAU  - Rakugi H\\nFAU - Kaneda, Yasufumi\\nAU  - Kaneda Y\\nFAU - Morishita, Ryuichi\\nAU  - Morishita R\\nLA  - eng\\nPT  - Comparative Study\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov't\\nDEP - 20100702\\nPL  - Japan\\nTA  - J Atheroscler Thromb\\nJT  - Journal of atherosclerosis and thrombosis\\nJID - 9506298\\nRN  - 0 (Receptors, Steroid)\\nRN  - 0 (oxysterol binding protein)\\nSB  - IM\\nMH  - Aged\\nMH  - Asian Continental Ancestry Group/*genetics\\nMH  - Case-Control Studies\\nMH  - Female\\nMH  - Genetic Predisposition to Disease\\nMH  - Genome, Human\\nMH  - *Genome-Wide Association Study\\nMH  - Genotype\\nMH  - Humans\\nMH  - Japan/epidemiology\\nMH  - Linkage Disequilibrium\\nMH  - Male\\nMH  - Middle Aged\\nMH  - Peripheral Vascular Diseases/epidemiology/*genetics\\nMH  - Polymorphism, Single Nucleotide/*genetics\\nMH  - Receptors, Steroid/*genetics\\nEDAT- 2010/07/09 06:00\\nMHDA- 2011/03/09 06:00\\nCRDT- 2010/07/09 06:00\\nPHST- 2010/07/09 06:00 [entrez]\\nPHST- 2010/07/09 06:00 [pubmed]\\nPHST- 2011/03/09 06:00 [medline]\\nAID - JST.JSTAGE/jat/4291 [pii]\\nAID - 10.5551/jat.4291 [doi]\\nPST - ppublish\\nSO  - J Atheroscler Thromb. 2010 Oct 27;17(10):1054-62. doi: 10.5551/jat.4291. Epub\\n      2010 Jul 2.\", \"PMID- 20379614\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20101108\\nLR  - 20181113\\nIS  - 1528-3658 (Electronic)\\nIS  - 1076-1551 (Linking)\\nVI  - 16\\nIP  - 7-8\\nDP  - 2010 Jul-Aug\\nTI  - Personalized smoking cessation: interactions between nicotine dose, dependence\\n      and quit-success genotype score.\\nPG  - 247-53\\nLID - 10.2119/molmed.2009.00159 [doi]\\nAB  - Improving and targeting nicotine replacement therapy (NRT) are cost-effective\\n      strategies for reducing adverse health consequences for smokers. Treatment\\n      studies document the efficacy of precessation NRT and support important roles for\\n      level of nicotine dependence and precessation smoking reduction in successful\\n      quitting. However, prior work has not identified the optimal precessation dose or\\n      means for personalizing NRT. Genome-wide association has identified groups of\\n      genomic markers associated with successful quitting, allowing us to develop a\\n      v1.0 \\\"quit-success\\\" genotype score. We now report influences of v1.0 quit-success\\n      genotype score, level of dependence and precessation smoking reduction in a\\n      smoking cessation trial that examined effects of 21 versus 42 mg/24 h\\n      precessation NRT. Four hundred seventy-nine smokers were randomized to 21 or 42\\n      mg NRT, initiated 2 wks prior to target quit dates. We monitored self-reported\\n      abstinence and end-expired air carbon monoxide (CO). Genotyping used Affymetrix\\n      arrays (Santa Clara, CA, USA). The primary outcome was 10-wk continuous smoking\\n      abstinence. NRT dose, level of nicotine dependence and genotype scores displayed \\n      significant interactive effects on successful quitting. Successful abstinence\\n      also was predicted by CO reductions during precessation NRT. These results\\n      document ways in which smoking cessation strategies can be personalized based on \\n      levels of nicotine dependence, genotype scores and CO monitoring. These\\n      assessments, taken together, can help match most smokers with optimal NRT doses\\n      and help rapidly identify some who may be better treated using other methods.\\nFAU - Rose, Jed E\\nAU  - Rose JE\\nAD  - Department of Psychiatry and Center for Nicotine and Smoking Cessation Research, \\n      Duke University, Durham, North Carolina 27705, USA. rose0003@mc.duke.edu\\nFAU - Behm, Frederique M\\nAU  - Behm FM\\nFAU - Drgon, Tomas\\nAU  - Drgon T\\nFAU - Johnson, Catherine\\nAU  - Johnson C\\nFAU - Uhl, George R\\nAU  - Uhl GR\\nLA  - eng\\nSI  - ClinicalTrials.gov/NCT00734617\\nGR  - P50 DA027840/DA/NIDA NIH HHS/United States\\nGR  - Intramural NIH HHS/United States\\nPT  - Journal Article\\nPT  - Randomized Controlled Trial\\nPT  - Research Support, N.I.H., Intramural\\nPT  - Research Support, Non-U.S. Gov't\\nDEP - 20100317\\nPL  - England\\nTA  - Mol Med\\nJT  - Molecular medicine (Cambridge, Mass.)\\nJID - 9501023\\nRN  - 0 (Delayed-Action Preparations)\\nRN  - 6M3C89ZY6R (Nicotine)\\nRN  - 7U1EE4V452 (Carbon Monoxide)\\nSB  - IM\\nEIN - Mol Med. 2012;18(1):729\\nMH  - Administration, Cutaneous\\nMH  - Adult\\nMH  - Carbon Monoxide/analysis\\nMH  - Delayed-Action Preparations\\nMH  - Double-Blind Method\\nMH  - Female\\nMH  - Genome-Wide Association Study\\nMH  - Genotype\\nMH  - Humans\\nMH  - Logistic Models\\nMH  - Male\\nMH  - Middle Aged\\nMH  - Nicotine/*administration & dosage\\nMH  - Polymorphism, Single Nucleotide\\nMH  - Precision Medicine/*methods\\nMH  - Smoking Cessation/*methods\\nMH  - Tobacco Use Disorder/*drug therapy/*genetics\\nPMC - PMC2896464\\nEDAT- 2010/04/10 06:00\\nMHDA- 2010/11/09 06:00\\nCRDT- 2010/04/10 06:00\\nPHST- 2009/10/31 00:00 [received]\\nPHST- 2010/03/16 00:00 [accepted]\\nPHST- 2010/04/10 06:00 [entrez]\\nPHST- 2010/04/10 06:00 [pubmed]\\nPHST- 2010/11/09 06:00 [medline]\\nAID - molmed.2009.00159 [pii]\\nAID - 10.2119/molmed.2009.00159 [doi]\\nPST - ppublish\\nSO  - Mol Med. 2010 Jul-Aug;16(7-8):247-53. doi: 10.2119/molmed.2009.00159. Epub 2010\\n      Mar 17.\", \"PMID- 20301317\\nSTAT- Publisher\\nDA  - 20100320\\nDRDT- 20190725\\nCTDT- 19981028\\nPB  - University of Washington, Seattle\\nDP  - 1993\\nTI  - Hereditary Ataxia Overview\\nBTI - GeneReviews((R))\\nAB  - CLINICAL CHARACTERISTICS: The hereditary ataxias are a group of genetic disorders\\n      characterized by slowly progressive incoordination of gait and often associated\\n      with poor coordination of hands, speech, and eye movements. Frequently, atrophy\\n      of the cerebellum occurs. In this GeneReview the hereditary ataxias are\\n      categorized by mode of inheritance and gene (or chromosome locus) in which\\n      pathogenic variants occur. DIAGNOSIS/TESTING: Inherited (genetic) forms of ataxia\\n      must be distinguished from the many acquired (non-genetic) causes of ataxia. The \\n      genetic forms of ataxia are diagnosed by family history, physical examination,\\n      neuroimaging, and molecular genetic testing. GENETIC COUNSELING: The hereditary\\n      ataxias can be inherited in an autosomal dominant, autosomal recessive, or\\n      X-linked manner. Genetic counseling and risk assessment depend on determination\\n      of the specific cause of an inherited ataxia in an individual. MANAGEMENT:\\n      Treatment of manifestations: Canes, walkers, and wheelchairs for gait ataxia; use\\n      of special devices to assist with handwriting, buttoning, and use of eating\\n      utensils; speech therapy and/or computer-based devices for those with dysarthria \\n      and severe speech deficits. Prevention of primary manifestations: With the\\n      exception of vitamin E therapy for ataxia with vitamin E deficiency (AVED), no\\n      specific treatments exist for hereditary ataxia.\\nCI  - Copyright (c) 1993-2019, University of Washington, Seattle. GeneReviews is a\\n      registered trademark of the University of Washington, Seattle. All rights\\n      reserved.\\nFED - Adam, Margaret P\\nED  - Adam MP\\nFED - Ardinger, Holly H\\nED  - Ardinger HH\\nFED - Pagon, Roberta A\\nED  - Pagon RA\\nFED - Wallace, Stephanie E\\nED  - Wallace SE\\nFED - Bean, Lora JH\\nED  - Bean LJH\\nFED - Stephens, Karen\\nED  - Stephens K\\nFED - Amemiya, Anne\\nED  - Amemiya A\\nFAU - Bird, Thomas D\\nAU  - Bird TD\\nLA  - eng\\nGR  - HHSN276201400262U/NLM NIH HHS/United States\\nPT  - Review\\nPT  - Book Chapter\\nPL  - Seattle (WA)\\nOTO - NLM\\nOT  - Aconitate hydratase, mitochondrial\\nOT  - AFG3-like protein 2\\nOT  - Alpha-tocopherol transfer protein\\nOT  - Aprataxin\\nOT  - Ataxin-1\\nOT  - Ataxin-10\\nOT  - Ataxin-2\\nOT  - Ataxin-3\\nOT  - Ataxin-7\\nOT  - Ataxin-8\\nOT  - Atrophin-1\\nOT  - Atypical kinase COQ8A, mitochondrial\\nOT  - Bifunctional polynucleotide phosphatase/kinase\\nOT  - Calpain-1 catalytic subunit\\nOT  - Ceroid-lipofuscinosis neuronal protein 5\\nOT  - CWF19-like protein 1\\nOT  - Decaprenyl-diphosphate synthase subunit 1\\nOT  - Decaprenyl-diphosphate synthase subunit 2\\nOT  - DNA-directed RNA polymerase III subunit RPC1\\nOT  - DNA-directed RNA polymerase III subunit RPC2\\nOT  - E3 ubiquitin-protein ligase CHIP\\nOT  - E3 ubiquitin-protein ligase RNF216\\nOT  - Elongation factor 2\\nOT  - Elongation factor Ts, mitochondrial\\nOT  - Elongation of very long chain fatty acids protein 4\\nOT  - Elongation of very long chain fatty acids protein 5\\nOT  - Excitatory amino acid transporter 1\\nOT  - Fibroblast growth factor 12\\nOT  - Fibroblast growth factor 14\\nOT  - Frataxin, mitochondrial\\nOT  - Ganglioside-induced differentiation-associated protein 2\\nOT  - Glutamate receptor ionotropic, delta-2\\nOT  - Inositol 1,4,5-trisphosphate receptor type 1\\nOT  - Laminin subunit alpha-1\\nOT  - Metabotropic glutamate receptor 1\\nOT  - Microtubule cross-linking factor 1\\nOT  - Neprilysin\\nOT  - Neuropathy target esterase\\nOT  - Nucleolar protein 56\\nOT  - Nucleotide exchange factor SIL1\\nOT  - Oligophrenin-1\\nOT  - Para-hydroxybenzoate--polyprenyltransferase, mitochondrial\\nOT  - Paraplegin\\nOT  - Peroxisomal targeting signal 2 receptor\\nOT  - Phospholipase D3\\nOT  - Phytanoyl-CoA dioxygenase, peroxisomal\\nOT  - Plasma membrane calcium-transporting ATPase 3\\nOT  - Potassium voltage-gated channel subfamily A member 1\\nOT  - Potassium voltage-gated channel subfamily C member 3\\nOT  - Potassium voltage-gated channel subfamily D member 3\\nOT  - Probable helicase senataxin\\nOT  - Proenkephalin-B\\nOT  - Protein BEAN1\\nOT  - Protein Daple\\nOT  - Protein kinase C gamma type\\nOT  - Protein-glutamine gamma-glutamyltransferase 6\\nOT  - Protocadherin Fat 2\\nOT  - Protocadherin-12\\nOT  - Pumilio homolog 1\\nOT  - Replication factor C subunit 1\\nOT  - Sacsin\\nOT  - Sequestosome-1\\nOT  - Serine/threonine-protein phosphatase 2A 55 kDa regulatory subunit B beta isoform\\nOT  - Serine-protein kinase ATM\\nOT  - Short transient receptor potential channel 3\\nOT  - Sodium/hydrogen exchanger 1\\nOT  - Sodium/hydrogen exchanger 6\\nOT  - Sodium/potassium-transporting ATPase subunit alpha-3\\nOT  - Solute carrier family 25 member 46\\nOT  - Solute carrier family 52, riboflavin transporter, member 2\\nOT  - Sorting nexin-14\\nOT  - Sorting nexin-4\\nOT  - Spectrin beta chain, non-erythrocytic 2\\nOT  - Sterol 26-hydroxylase, mitochondrial\\nOT  - TATA-box-binding protein\\nOT  - Tau-tubulin kinase 2\\nOT  - Transmembrane protein 240\\nOT  - Twinkle protein, mitochondrial\\nOT  - Tyrosyl-DNA phosphodiesterase 2\\nOT  - Ubiquinone biosynthesis protein COQ9, mitochondrial\\nOT  - Vacuolar protein sorting-associated protein 13D\\nOT  - Vesicle-associated membrane protein 1\\nOT  - Voltage-dependent L-type calcium channel subunit beta-4\\nOT  - Voltage-dependent P/Q-type calcium channel subunit alpha-1A\\nOT  - von Willebrand factor A domain-containing protein 3B\\nOT  - WD repeat-containing protein 73\\nOT  - Wolframin\\nOT  - ACO2\\nOT  - AFG3L2\\nOT  - APTX\\nOT  - ATM\\nOT  - ATN1\\nOT  - ATP1A3\\nOT  - ATP2B3\\nOT  - ATXN1\\nOT  - ATXN10\\nOT  - ATXN2\\nOT  - ATXN3\\nOT  - ATXN7\\nOT  - ATXN8\\nOT  - ATXN8OS\\nOT  - BEAN1\\nOT  - CACNA1A\\nOT  - CACNB4\\nOT  - CAPN1\\nOT  - CCDC88C\\nOT  - CLN5\\nOT  - COQ2\\nOT  - COQ8A\\nOT  - COQ9\\nOT  - CWF19L1\\nOT  - CYP27A1\\nOT  - EEF2\\nOT  - ELOVL4\\nOT  - ELOVL5\\nOT  - FAT2\\nOT  - FGF12\\nOT  - FGF14\\nOT  - FXN\\nOT  - GDAP2\\nOT  - GRID2\\nOT  - GRM1\\nOT  - ITPR1\\nOT  - KCNA1\\nOT  - KCNC3\\nOT  - KCND3\\nOT  - LAMA1\\nOT  - MME\\nOT  - MTCL1\\nOT  - NOP56\\nOT  - OPHN1\\nOT  - PCDH12\\nOT  - PDSS1\\nOT  - PDSS2\\nOT  - PDYN\\nOT  - PEX7\\nOT  - PHYH\\nOT  - PLD3\\nOT  - PNKP\\nOT  - PNPLA6\\nOT  - POLR3A\\nOT  - POLR3B\\nOT  - PPP2R2B\\nOT  - PRKCG\\nOT  - PUM1\\nOT  - RFC1\\nOT  - RNF216\\nOT  - SACS\\nOT  - SETX\\nOT  - SIL1\\nOT  - SLC1A3\\nOT  - SLC25A46\\nOT  - SLC52A2\\nOT  - SLC9A1\\nOT  - SLC9A6\\nOT  - SNX14\\nOT  - SNX4\\nOT  - SPG7\\nOT  - SPTBN2\\nOT  - SQSTM1\\nOT  - STUB1\\nOT  - TBP\\nOT  - TDP2\\nOT  - TGM6\\nOT  - TMEM240\\nOT  - TRPC3\\nOT  - TSFM\\nOT  - TTBK2\\nOT  - TTPA\\nOT  - TWNK\\nOT  - VAMP1\\nOT  - VPS13D\\nOT  - VWA3B\\nOT  - WDR73\\nOT  - WFS1\\nOT  - Hereditary Ataxia\\nOT  - Overview\\nEDAT- 2019/07/25 06:00\\nMHDA- 2019/07/25 06:00\\nCDAT- 2010/03/20 06:00\\nAID - NBK1138 [bookaccession]\", \"PMID- 19056867\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20090226\\nLR  - 20191003\\nIS  - 1533-3450 (Electronic)\\nIS  - 1046-6673 (Linking)\\nVI  - 20\\nIP  - 2\\nDP  - 2009 Feb\\nTI  - Large-scale proteomics and phosphoproteomics of urinary exosomes.\\nPG  - 363-79\\nLID - 10.1681/ASN.2008040406 [doi]\\nAB  - Normal human urine contains large numbers of exosomes, which are 40- to 100-nm\\n      vesicles that originate as the internal vesicles in multivesicular bodies from\\n      every renal epithelial cell type facing the urinary space. Here, we used LC-MS/MS\\n      to profile the proteome of human urinary exosomes. Overall, the analysis\\n      identified 1132 proteins unambiguously, including 177 that are represented on the\\n      Online Mendelian Inheritance in Man database of disease-related genes, suggesting\\n      that exosome analysis is a potential approach to discover urinary biomarkers. We \\n      extended the proteomic analysis to phosphoproteomic profiling using neutral loss \\n      scanning, and this yielded multiple novel phosphorylation sites, including\\n      serine-811 in the thiazide-sensitive Na-Cl co-transporter, NCC. To demonstrate\\n      the potential use of exosome analysis to identify a genetic renal disease, we\\n      carried out immunoblotting of exosomes from urine samples of patients with a\\n      clinical diagnosis of Bartter syndrome type I, showing an absence of the\\n      sodium-potassium-chloride co-transporter 2, NKCC2. The proteomic data are\\n      publicly accessible at http://dir.nhlbi.nih.gov/papers/lkem/exosome/.\\nFAU - Gonzales, Patricia A\\nAU  - Gonzales PA\\nAD  - Laboratory of Kidney and Electrolyte Metabolism, National Heart, Lung, and Blood \\n      Institute, National Institutes of Health, Bethesda, MD 20892, USA.\\nFAU - Pisitkun, Trairak\\nAU  - Pisitkun T\\nFAU - Hoffert, Jason D\\nAU  - Hoffert JD\\nFAU - Tchapyjnikov, Dmitry\\nAU  - Tchapyjnikov D\\nFAU - Star, Robert A\\nAU  - Star RA\\nFAU - Kleta, Robert\\nAU  - Kleta R\\nFAU - Wang, Nam Sun\\nAU  - Wang NS\\nFAU - Knepper, Mark A\\nAU  - Knepper MA\\nLA  - eng\\nGR  - Z01 HL001285-21/Intramural NIH HHS/United States\\nGR  - Z99 HL999999/Intramural NIH HHS/United States\\nPT  - Journal Article\\nDEP - 20081203\\nPL  - United States\\nTA  - J Am Soc Nephrol\\nJT  - Journal of the American Society of Nephrology : JASN\\nJID - 9013836\\nRN  - 0 (Phosphoproteins)\\nRN  - 0 (Proteome)\\nRN  - 0 (Thiazides)\\nRN  - EC 3.6.1.- (Adenosine Triphosphatases)\\nSB  - IM\\nMH  - Adenosine Triphosphatases/chemistry\\nMH  - Adult\\nMH  - Bartter Syndrome/urine\\nMH  - Chromatography, Liquid/methods\\nMH  - Exosomes/metabolism\\nMH  - Female\\nMH  - Humans\\nMH  - Male\\nMH  - Mass Spectrometry/methods\\nMH  - Phosphoproteins/chemistry\\nMH  - Phosphorylation\\nMH  - Proteome\\nMH  - Proteomics/*methods\\nMH  - Thiazides/chemistry\\nMH  - *Urine\\nPMC - PMC2637050\\nEDAT- 2008/12/06 09:00\\nMHDA- 2009/02/27 09:00\\nCRDT- 2008/12/06 09:00\\nPHST- 2008/12/06 09:00 [pubmed]\\nPHST- 2009/02/27 09:00 [medline]\\nPHST- 2008/12/06 09:00 [entrez]\\nAID - ASN.2008040406 [pii]\\nAID - 10.1681/ASN.2008040406 [doi]\\nPST - ppublish\\nSO  - J Am Soc Nephrol. 2009 Feb;20(2):363-79. doi: 10.1681/ASN.2008040406. Epub 2008\\n      Dec 3.\", \"PMID- 16710414\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20060614\\nLR  - 20170922\\nIS  - 1476-4687 (Electronic)\\nIS  - 0028-0836 (Linking)\\nVI  - 441\\nIP  - 7091\\nDP  - 2006 May 18\\nTI  - The DNA sequence and biological annotation of human chromosome 1.\\nPG  - 315-21\\nAB  - The reference sequence for each human chromosome provides the framework for\\n      understanding genome function, variation and evolution. Here we report the\\n      finished sequence and biological annotation of human chromosome 1. Chromosome 1\\n      is gene-dense, with 3,141 genes and 991 pseudogenes, and many coding sequences\\n      overlap. Rearrangements and mutations of chromosome 1 are prevalent in cancer and\\n      many other diseases. Patterns of sequence variation reveal signals of recent\\n      selection in specific genes that may contribute to human fitness, and also in\\n      regions where no function is evident. Fine-scale recombination occurs in hotspots\\n      of varying intensity along the sequence, and is enriched near genes. These and\\n      other studies of human biology and disease encoded within chromosome 1 are made\\n      possible with the highly accurate annotated sequence, as part of the completed\\n      set of chromosome sequences that comprise the reference human genome.\\nFAU - Gregory, S G\\nAU  - Gregory SG\\nAD  - The Wellcome Trust Sanger Institute, The Wellcome Trust Genome Campus, Hinxton,\\n      Cambridgeshire CB10 1SA, UK. sgregory@chg.duhs.duke.edu\\nFAU - Barlow, K F\\nAU  - Barlow KF\\nFAU - McLay, K E\\nAU  - McLay KE\\nFAU - Kaul, R\\nAU  - Kaul R\\nFAU - Swarbreck, D\\nAU  - Swarbreck D\\nFAU - Dunham, A\\nAU  - Dunham A\\nFAU - Scott, C E\\nAU  - Scott CE\\nFAU - Howe, K L\\nAU  - Howe KL\\nFAU - Woodfine, K\\nAU  - Woodfine K\\nFAU - Spencer, C C A\\nAU  - Spencer CC\\nFAU - Jones, M C\\nAU  - Jones MC\\nFAU - Gillson, C\\nAU  - Gillson C\\nFAU - Searle, S\\nAU  - Searle S\\nFAU - Zhou, Y\\nAU  - Zhou Y\\nFAU - Kokocinski, F\\nAU  - Kokocinski F\\nFAU - McDonald, L\\nAU  - McDonald L\\nFAU - Evans, R\\nAU  - Evans R\\nFAU - Phillips, K\\nAU  - Phillips K\\nFAU - Atkinson, A\\nAU  - Atkinson A\\nFAU - Cooper, R\\nAU  - Cooper R\\nFAU - Jones, C\\nAU  - Jones C\\nFAU - Hall, R E\\nAU  - Hall RE\\nFAU - Andrews, T D\\nAU  - Andrews TD\\nFAU - Lloyd, C\\nAU  - Lloyd C\\nFAU - Ainscough, R\\nAU  - Ainscough R\\nFAU - Almeida, J P\\nAU  - Almeida JP\\nFAU - Ambrose, K D\\nAU  - Ambrose KD\\nFAU - Anderson, F\\nAU  - Anderson F\\nFAU - Andrew, R W\\nAU  - Andrew RW\\nFAU - Ashwell, R I S\\nAU  - Ashwell RI\\nFAU - Aubin, K\\nAU  - Aubin K\\nFAU - Babbage, A K\\nAU  - Babbage AK\\nFAU - Bagguley, C L\\nAU  - Bagguley CL\\nFAU - Bailey, J\\nAU  - Bailey J\\nFAU - Beasley, H\\nAU  - Beasley H\\nFAU - Bethel, G\\nAU  - Bethel G\\nFAU - Bird, C P\\nAU  - Bird CP\\nFAU - Bray-Allen, S\\nAU  - Bray-Allen S\\nFAU - Brown, J Y\\nAU  - Brown JY\\nFAU - Brown, A J\\nAU  - Brown AJ\\nFAU - Buckley, D\\nAU  - Buckley D\\nFAU - Burton, J\\nAU  - Burton J\\nFAU - Bye, J\\nAU  - Bye J\\nFAU - Carder, C\\nAU  - Carder C\\nFAU - Chapman, J C\\nAU  - Chapman JC\\nFAU - Clark, S Y\\nAU  - Clark SY\\nFAU - Clarke, G\\nAU  - Clarke G\\nFAU - Clee, C\\nAU  - Clee C\\nFAU - Cobley, V\\nAU  - Cobley V\\nFAU - Collier, R E\\nAU  - Collier RE\\nFAU - Corby, N\\nAU  - Corby N\\nFAU - Coville, G J\\nAU  - Coville GJ\\nFAU - Davies, J\\nAU  - Davies J\\nFAU - Deadman, R\\nAU  - Deadman R\\nFAU - Dunn, M\\nAU  - Dunn M\\nFAU - Earthrowl, M\\nAU  - Earthrowl M\\nFAU - Ellington, A G\\nAU  - Ellington AG\\nFAU - Errington, H\\nAU  - Errington H\\nFAU - Frankish, A\\nAU  - Frankish A\\nFAU - Frankland, J\\nAU  - Frankland J\\nFAU - French, L\\nAU  - French L\\nFAU - Garner, P\\nAU  - Garner P\\nFAU - Garnett, J\\nAU  - Garnett J\\nFAU - Gay, L\\nAU  - Gay L\\nFAU - Ghori, M R J\\nAU  - Ghori MR\\nFAU - Gibson, R\\nAU  - Gibson R\\nFAU - Gilby, L M\\nAU  - Gilby LM\\nFAU - Gillett, W\\nAU  - Gillett W\\nFAU - Glithero, R J\\nAU  - Glithero RJ\\nFAU - Grafham, D V\\nAU  - Grafham DV\\nFAU - Griffiths, C\\nAU  - Griffiths C\\nFAU - Griffiths-Jones, S\\nAU  - Griffiths-Jones S\\nFAU - Grocock, R\\nAU  - Grocock R\\nFAU - Hammond, S\\nAU  - Hammond S\\nFAU - Harrison, E S I\\nAU  - Harrison ES\\nFAU - Hart, E\\nAU  - Hart E\\nFAU - Haugen, E\\nAU  - Haugen E\\nFAU - Heath, P D\\nAU  - Heath PD\\nFAU - Holmes, S\\nAU  - Holmes S\\nFAU - Holt, K\\nAU  - Holt K\\nFAU - Howden, P J\\nAU  - Howden PJ\\nFAU - Hunt, A R\\nAU  - Hunt AR\\nFAU - Hunt, S E\\nAU  - Hunt SE\\nFAU - Hunter, G\\nAU  - Hunter G\\nFAU - Isherwood, J\\nAU  - Isherwood J\\nFAU - James, R\\nAU  - James R\\nFAU - Johnson, C\\nAU  - Johnson C\\nFAU - Johnson, D\\nAU  - Johnson D\\nFAU - Joy, A\\nAU  - Joy A\\nFAU - Kay, M\\nAU  - Kay M\\nFAU - Kershaw, J K\\nAU  - Kershaw JK\\nFAU - Kibukawa, M\\nAU  - Kibukawa M\\nFAU - Kimberley, A M\\nAU  - Kimberley AM\\nFAU - King, A\\nAU  - King A\\nFAU - Knights, A J\\nAU  - Knights AJ\\nFAU - Lad, H\\nAU  - Lad H\\nFAU - Laird, G\\nAU  - Laird G\\nFAU - Lawlor, S\\nAU  - Lawlor S\\nFAU - Leongamornlert, D A\\nAU  - Leongamornlert DA\\nFAU - Lloyd, D M\\nAU  - Lloyd DM\\nFAU - Loveland, J\\nAU  - Loveland J\\nFAU - Lovell, J\\nAU  - Lovell J\\nFAU - Lush, M J\\nAU  - Lush MJ\\nFAU - Lyne, R\\nAU  - Lyne R\\nFAU - Martin, S\\nAU  - Martin S\\nFAU - Mashreghi-Mohammadi, M\\nAU  - Mashreghi-Mohammadi M\\nFAU - Matthews, L\\nAU  - Matthews L\\nFAU - Matthews, N S W\\nAU  - Matthews NS\\nFAU - McLaren, S\\nAU  - McLaren S\\nFAU - Milne, S\\nAU  - Milne S\\nFAU - Mistry, S\\nAU  - Mistry S\\nFAU - Moore, M J F\\nAU  - Moore MJ\\nFAU - Nickerson, T\\nAU  - Nickerson T\\nFAU - O'Dell, C N\\nAU  - O'Dell CN\\nFAU - Oliver, K\\nAU  - Oliver K\\nFAU - Palmeiri, A\\nAU  - Palmeiri A\\nFAU - Palmer, S A\\nAU  - Palmer SA\\nFAU - Parker, A\\nAU  - Parker A\\nFAU - Patel, D\\nAU  - Patel D\\nFAU - Pearce, A V\\nAU  - Pearce AV\\nFAU - Peck, A I\\nAU  - Peck AI\\nFAU - Pelan, S\\nAU  - Pelan S\\nFAU - Phelps, K\\nAU  - Phelps K\\nFAU - Phillimore, B J\\nAU  - Phillimore BJ\\nFAU - Plumb, R\\nAU  - Plumb R\\nFAU - Rajan, J\\nAU  - Rajan J\\nFAU - Raymond, C\\nAU  - Raymond C\\nFAU - Rouse, G\\nAU  - Rouse G\\nFAU - Saenphimmachak, C\\nAU  - Saenphimmachak C\\nFAU - Sehra, H K\\nAU  - Sehra HK\\nFAU - Sheridan, E\\nAU  - Sheridan E\\nFAU - Shownkeen, R\\nAU  - Shownkeen R\\nFAU - Sims, S\\nAU  - Sims S\\nFAU - Skuce, C D\\nAU  - Skuce CD\\nFAU - Smith, M\\nAU  - Smith M\\nFAU - Steward, C\\nAU  - Steward C\\nFAU - Subramanian, S\\nAU  - Subramanian S\\nFAU - Sycamore, N\\nAU  - Sycamore N\\nFAU - Tracey, A\\nAU  - Tracey A\\nFAU - Tromans, A\\nAU  - Tromans A\\nFAU - Van Helmond, Z\\nAU  - Van Helmond Z\\nFAU - Wall, M\\nAU  - Wall M\\nFAU - Wallis, J M\\nAU  - Wallis JM\\nFAU - White, S\\nAU  - White S\\nFAU - Whitehead, S L\\nAU  - Whitehead SL\\nFAU - Wilkinson, J E\\nAU  - Wilkinson JE\\nFAU - Willey, D L\\nAU  - Willey DL\\nFAU - Williams, H\\nAU  - Williams H\\nFAU - Wilming, L\\nAU  - Wilming L\\nFAU - Wray, P W\\nAU  - Wray PW\\nFAU - Wu, Z\\nAU  - Wu Z\\nFAU - Coulson, A\\nAU  - Coulson A\\nFAU - Vaudin, M\\nAU  - Vaudin M\\nFAU - Sulston, J E\\nAU  - Sulston JE\\nFAU - Durbin, R\\nAU  - Durbin R\\nFAU - Hubbard, T\\nAU  - Hubbard T\\nFAU - Wooster, R\\nAU  - Wooster R\\nFAU - Dunham, I\\nAU  - Dunham I\\nFAU - Carter, N P\\nAU  - Carter NP\\nFAU - McVean, G\\nAU  - McVean G\\nFAU - Ross, M T\\nAU  - Ross MT\\nFAU - Harrow, J\\nAU  - Harrow J\\nFAU - Olson, M V\\nAU  - Olson MV\\nFAU - Beck, S\\nAU  - Beck S\\nFAU - Rogers, J\\nAU  - Rogers J\\nFAU - Bentley, D R\\nAU  - Bentley DR\\nFAU - Banerjee, R\\nAU  - Banerjee R\\nFAU - Bryant, S P\\nAU  - Bryant SP\\nFAU - Burford, D C\\nAU  - Burford DC\\nFAU - Burrill, W D H\\nAU  - Burrill WD\\nFAU - Clegg, S M\\nAU  - Clegg SM\\nFAU - Dhami, P\\nAU  - Dhami P\\nFAU - Dovey, O\\nAU  - Dovey O\\nFAU - Faulkner, L M\\nAU  - Faulkner LM\\nFAU - Gribble, S M\\nAU  - Gribble SM\\nFAU - Langford, C F\\nAU  - Langford CF\\nFAU - Pandian, R D\\nAU  - Pandian RD\\nFAU - Porter, K M\\nAU  - Porter KM\\nFAU - Prigmore, E\\nAU  - Prigmore E\\nLA  - eng\\nSI  - RefSeq/NC_000001\\nGR  - G0000107/Medical Research Council/United Kingdom\\nGR  - Wellcome Trust/United Kingdom\\nPT  - Journal Article\\nPT  - Research Support, N.I.H., Extramural\\nPT  - Research Support, Non-U.S. Gov't\\nPL  - England\\nTA  - Nature\\nJT  - Nature\\nJID - 0410462\\nSB  - IM\\nEIN - Nature. 2006 Oct 26;443(7114):1013. Banerjee, R [added]; Bryant, SP [added];\\n      Burford, DC [added]; Burrill, WDH [added]; Clegg, SM [added]; Dhami, P [added];\\n      Dovey, O [added]; Faulkner, LM [added]; Gribble, SM [added]; Langford, CF\\n      [added]; Pandian, RD [added]; Porter, KM [added]; Prigmore, E\\nMH  - Base Sequence\\nMH  - Chromosomes, Human, Pair 1/*genetics\\nMH  - DNA Replication Timing\\nMH  - Disease\\nMH  - Gene Duplication\\nMH  - Genes/genetics\\nMH  - Genetic Variation/genetics\\nMH  - Genomics\\nMH  - Humans\\nMH  - Molecular Sequence Data\\nMH  - Open Reading Frames/genetics\\nMH  - Pseudogenes/genetics\\nMH  - Recombination, Genetic/genetics\\nMH  - Selection, Genetic\\nMH  - Sequence Analysis, DNA\\nEDAT- 2006/05/20 09:00\\nMHDA- 2006/06/15 09:00\\nCRDT- 2006/05/20 09:00\\nPHST- 2005/12/24 00:00 [received]\\nPHST- 2006/03/13 00:00 [accepted]\\nPHST- 2006/05/20 09:00 [pubmed]\\nPHST- 2006/06/15 09:00 [medline]\\nPHST- 2006/05/20 09:00 [entrez]\\nAID - nature04727 [pii]\\nAID - 10.1038/nature04727 [doi]\\nPST - ppublish\\nSO  - Nature. 2006 May 18;441(7091):315-21. doi: 10.1038/nature04727.\", \"PMID- 15498460\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20050124\\nLR  - 20070516\\nIS  - 0888-7543 (Print)\\nIS  - 0888-7543 (Linking)\\nVI  - 84\\nIP  - 3\\nDP  - 2004 Sep\\nTI  - Analysis of the human VPS13 gene family.\\nPG  - 536-49\\nAB  - The gene mutated in chorea-acanthocytosis (CHAC; approved gene symbol VPS13A)\\n      encodes chorein, a protein similar to yeast Vps13p. We detected several similar\\n      putative human proteins by BLAST analysis of chorein. We characterized the\\n      structure of three new genes encoding these CHAC-similar proteins, located on\\n      chromosomes 1p36, 8q22, and 15q21. The most similar gene in yeast to all four\\n      human genes is Vps13, and therefore the human genes were named VPS13A (CHAC,\\n      9q21), VPS13B (8q22), VPS13C (15q21), and VPS13D (1p36). VPS13B has recently been\\n      reported as COH1, altered in Cohen syndrome. For each gene, we describe several\\n      alternative splicing variants; at least two transcripts per gene are major forms.\\n      The expression pattern of these genes is ubiquitous, with some tissue-specific\\n      differences between several transcript variants. Protein sequence comparisons\\n      suggest that intramolecular duplications have played an important role in the\\n      evolution of this gene family.\\nFAU - Velayos-Baeza, Antonio\\nAU  - Velayos-Baeza A\\nAD  - Wellcome Trust Centre for Human Genetics, University of Oxford, Headington, OX3\\n      7BN Oxford, UK.\\nFAU - Vettori, Andrea\\nAU  - Vettori A\\nFAU - Copley, Richard R\\nAU  - Copley RR\\nFAU - Dobson-Stone, Carol\\nAU  - Dobson-Stone C\\nFAU - Monaco, A P\\nAU  - Monaco AP\\nLA  - eng\\nSI  - GENBANK/AJ608769\\nSI  - GENBANK/AJ608770\\nSI  - GENBANK/AJ608771\\nSI  - GENBANK/AJ608772\\nSI  - GENBANK/AJ608773\\nSI  - GENBANK/AJ608774\\nSI  - GENBANK/AJ608775\\nSI  - GENBANK/AJ626859\\nSI  - GENBANK/AJ626860\\nSI  - GENBANK/AJ626861\\nPT  - Comparative Study\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov't\\nPL  - United States\\nTA  - Genomics\\nJT  - Genomics\\nJID - 8800135\\nRN  - 0 (Membrane Proteins)\\nRN  - 0 (Proteins)\\nRN  - 0 (VPS13A protein, human)\\nRN  - 0 (VPS13B protein, human)\\nRN  - 0 (VPS13C protein, human)\\nRN  - 0 (VPS13D protein, human)\\nRN  - 0 (Vesicular Transport Proteins)\\nSB  - IM\\nMH  - Alternative Splicing/*genetics\\nMH  - Amino Acid Sequence\\nMH  - Base Sequence\\nMH  - Blotting, Northern\\nMH  - Evolution, Molecular\\nMH  - Gene Components\\nMH  - Gene Duplication\\nMH  - Gene Expression Profiling\\nMH  - Humans\\nMH  - Membrane Proteins/*genetics\\nMH  - Molecular Sequence Data\\nMH  - Multigene Family/*genetics\\nMH  - Phylogeny\\nMH  - Proteins/*genetics\\nMH  - Reverse Transcriptase Polymerase Chain Reaction\\nMH  - Sequence Alignment\\nMH  - Sequence Analysis, DNA\\nMH  - Sequence Homology\\nMH  - Vesicular Transport Proteins\\nEDAT- 2004/10/23 09:00\\nMHDA- 2005/01/26 09:00\\nCRDT- 2004/10/23 09:00\\nPHST- 2004/02/13 00:00 [received]\\nPHST- 2004/04/19 00:00 [accepted]\\nPHST- 2004/10/23 09:00 [pubmed]\\nPHST- 2005/01/26 09:00 [medline]\\nPHST- 2004/10/23 09:00 [entrez]\\nAID - S0888754304001156 [pii]\\nAID - 10.1016/j.ygeno.2004.04.012 [doi]\\nPST - ppublish\\nSO  - Genomics. 2004 Sep;84(3):536-49. doi: 10.1016/j.ygeno.2004.04.012.\", \"PMID- 15489334\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20041116\\nLR  - 20181113\\nIS  - 1088-9051 (Print)\\nIS  - 1088-9051 (Linking)\\nVI  - 14\\nIP  - 10B\\nDP  - 2004 Oct\\nTI  - The status, quality, and expansion of the NIH full-length cDNA project: the\\n      Mammalian Gene Collection (MGC).\\nPG  - 2121-7\\nAB  - The National Institutes of Health's Mammalian Gene Collection (MGC) project was\\n      designed to generate and sequence a publicly accessible cDNA resource containing \\n      a complete open reading frame (ORF) for every human and mouse gene. The project\\n      initially used a random strategy to select clones from a large number of cDNA\\n      libraries from diverse tissues. Candidate clones were chosen based on 5'-EST\\n      sequences, and then fully sequenced to high accuracy and analyzed by algorithms\\n      developed for this project. Currently, more than 11,000 human and 10,000 mouse\\n      genes are represented in MGC by at least one clone with a full ORF. The random\\n      selection approach is now reaching a saturation point, and a transition to\\n      protocols targeted at the missing transcripts is now required to complete the\\n      mouse and human collections. Comparison of the sequence of the MGC clones to\\n      reference genome sequences reveals that most cDNA clones are of very high\\n      sequence quality, although it is likely that some cDNAs may carry missense\\n      variants as a consequence of experimental artifact, such as PCR, cloning, or\\n      reverse transcriptase errors. Recently, a rat cDNA component was added to the\\n      project, and ongoing frog (Xenopus) and zebrafish (Danio) cDNA projects were\\n      expanded to take advantage of the high-throughput MGC pipeline.\\nFAU - Gerhard, Daniela S\\nAU  - Gerhard DS\\nFAU - Wagner, Lukas\\nAU  - Wagner L\\nFAU - Feingold, Elise A\\nAU  - Feingold EA\\nFAU - Shenmen, Carolyn M\\nAU  - Shenmen CM\\nFAU - Grouse, Lynette H\\nAU  - Grouse LH\\nFAU - Schuler, Greg\\nAU  - Schuler G\\nFAU - Klein, Steven L\\nAU  - Klein SL\\nFAU - Old, Susan\\nAU  - Old S\\nFAU - Rasooly, Rebekah\\nAU  - Rasooly R\\nFAU - Good, Peter\\nAU  - Good P\\nFAU - Guyer, Mark\\nAU  - Guyer M\\nFAU - Peck, Allison M\\nAU  - Peck AM\\nFAU - Derge, Jeffery G\\nAU  - Derge JG\\nFAU - Lipman, David\\nAU  - Lipman D\\nFAU - Collins, Francis S\\nAU  - Collins FS\\nFAU - Jang, Wonhee\\nAU  - Jang W\\nFAU - Sherry, Steven\\nAU  - Sherry S\\nFAU - Feolo, Mike\\nAU  - Feolo M\\nFAU - Misquitta, Leonie\\nAU  - Misquitta L\\nFAU - Lee, Eduardo\\nAU  - Lee E\\nFAU - Rotmistrovsky, Kirill\\nAU  - Rotmistrovsky K\\nFAU - Greenhut, Susan F\\nAU  - Greenhut SF\\nFAU - Schaefer, Carl F\\nAU  - Schaefer CF\\nFAU - Buetow, Kenneth\\nAU  - Buetow K\\nFAU - Bonner, Tom I\\nAU  - Bonner TI\\nFAU - Haussler, David\\nAU  - Haussler D\\nFAU - Kent, Jim\\nAU  - Kent J\\nFAU - Kiekhaus, Mark\\nAU  - Kiekhaus M\\nFAU - Furey, Terry\\nAU  - Furey T\\nFAU - Brent, Michael\\nAU  - Brent M\\nFAU - Prange, Christa\\nAU  - Prange C\\nFAU - Schreiber, Kirsten\\nAU  - Schreiber K\\nFAU - Shapiro, Nicole\\nAU  - Shapiro N\\nFAU - Bhat, Narayan K\\nAU  - Bhat NK\\nFAU - Hopkins, Ralph F\\nAU  - Hopkins RF\\nFAU - Hsie, Florence\\nAU  - Hsie F\\nFAU - Driscoll, Tom\\nAU  - Driscoll T\\nFAU - Soares, M Bento\\nAU  - Soares MB\\nFAU - Casavant, Tom L\\nAU  - Casavant TL\\nFAU - Scheetz, Todd E\\nAU  - Scheetz TE\\nFAU - Brown-stein, Michael J\\nAU  - Brown-stein MJ\\nFAU - Usdin, Ted B\\nAU  - Usdin TB\\nFAU - Toshiyuki, Shiraki\\nAU  - Toshiyuki S\\nFAU - Carninci, Piero\\nAU  - Carninci P\\nFAU - Piao, Yulan\\nAU  - Piao Y\\nFAU - Dudekula, Dawood B\\nAU  - Dudekula DB\\nFAU - Ko, Minoru S H\\nAU  - Ko MS\\nFAU - Kawakami, Koichi\\nAU  - Kawakami K\\nFAU - Suzuki, Yutaka\\nAU  - Suzuki Y\\nFAU - Sugano, Sumio\\nAU  - Sugano S\\nFAU - Gruber, C E\\nAU  - Gruber CE\\nFAU - Smith, M R\\nAU  - Smith MR\\nFAU - Simmons, Blake\\nAU  - Simmons B\\nFAU - Moore, Troy\\nAU  - Moore T\\nFAU - Waterman, Richard\\nAU  - Waterman R\\nFAU - Johnson, Stephen L\\nAU  - Johnson SL\\nFAU - Ruan, Yijun\\nAU  - Ruan Y\\nFAU - Wei, Chia Lin\\nAU  - Wei CL\\nFAU - Mathavan, S\\nAU  - Mathavan S\\nFAU - Gunaratne, Preethi H\\nAU  - Gunaratne PH\\nFAU - Wu, Jiaqian\\nAU  - Wu J\\nFAU - Garcia, Angela M\\nAU  - Garcia AM\\nFAU - Hulyk, Stephen W\\nAU  - Hulyk SW\\nFAU - Fuh, Edwin\\nAU  - Fuh E\\nFAU - Yuan, Ye\\nAU  - Yuan Y\\nFAU - Sneed, Anna\\nAU  - Sneed A\\nFAU - Kowis, Carla\\nAU  - Kowis C\\nFAU - Hodgson, Anne\\nAU  - Hodgson A\\nFAU - Muzny, Donna M\\nAU  - Muzny DM\\nFAU - McPherson, John\\nAU  - McPherson J\\nFAU - Gibbs, Richard A\\nAU  - Gibbs RA\\nFAU - Fahey, Jessica\\nAU  - Fahey J\\nFAU - Helton, Erin\\nAU  - Helton E\\nFAU - Ketteman, Mark\\nAU  - Ketteman M\\nFAU - Madan, Anuradha\\nAU  - Madan A\\nFAU - Rodrigues, Stephanie\\nAU  - Rodrigues S\\nFAU - Sanchez, Amy\\nAU  - Sanchez A\\nFAU - Whiting, Michelle\\nAU  - Whiting M\\nFAU - Madari, Anup\\nAU  - Madari A\\nFAU - Young, Alice C\\nAU  - Young AC\\nFAU - Wetherby, Keith D\\nAU  - Wetherby KD\\nFAU - Granite, Steven J\\nAU  - Granite SJ\\nFAU - Kwong, Peggy N\\nAU  - Kwong PN\\nFAU - Brinkley, Charles P\\nAU  - Brinkley CP\\nFAU - Pearson, Russell L\\nAU  - Pearson RL\\nFAU - Bouffard, Gerard G\\nAU  - Bouffard GG\\nFAU - Blakesly, Robert W\\nAU  - Blakesly RW\\nFAU - Green, Eric D\\nAU  - Green ED\\nFAU - Dickson, Mark C\\nAU  - Dickson MC\\nFAU - Rodriguez, Alex C\\nAU  - Rodriguez AC\\nFAU - Grimwood, Jane\\nAU  - Grimwood J\\nFAU - Schmutz, Jeremy\\nAU  - Schmutz J\\nFAU - Myers, Richard M\\nAU  - Myers RM\\nFAU - Butterfield, Yaron S N\\nAU  - Butterfield YS\\nFAU - Griffith, Malachi\\nAU  - Griffith M\\nFAU - Griffith, Obi L\\nAU  - Griffith OL\\nFAU - Krzywinski, Martin I\\nAU  - Krzywinski MI\\nFAU - Liao, Nancy\\nAU  - Liao N\\nFAU - Morin, Ryan\\nAU  - Morin R\\nFAU - Palmquist, Diana\\nAU  - Palmquist D\\nFAU - Petrescu, Anca S\\nAU  - Petrescu AS\\nFAU - Skalska, Ursula\\nAU  - Skalska U\\nFAU - Smailus, Duane E\\nAU  - Smailus DE\\nFAU - Stott, Jeff M\\nAU  - Stott JM\\nFAU - Schnerch, Angelique\\nAU  - Schnerch A\\nFAU - Schein, Jacqueline E\\nAU  - Schein JE\\nFAU - Jones, Steven J M\\nAU  - Jones SJ\\nFAU - Holt, Robert A\\nAU  - Holt RA\\nFAU - Baross, Agnes\\nAU  - Baross A\\nFAU - Marra, Marco A\\nAU  - Marra MA\\nFAU - Clifton, Sandra\\nAU  - Clifton S\\nFAU - Makowski, Kathryn A\\nAU  - Makowski KA\\nFAU - Bosak, Stephanie\\nAU  - Bosak S\\nFAU - Malek, Joel\\nAU  - Malek J\\nCN  - MGC Project Team\\nLA  - eng\\nGR  - N01-C0-12400/PHS HHS/United States\\nPT  - Comparative Study\\nPT  - Journal Article\\nPT  - Research Support, U.S. Gov't, P.H.S.\\nPL  - United States\\nTA  - Genome Res\\nJT  - Genome research\\nJID - 9518021\\nRN  - 0 (DNA Primers)\\nRN  - 0 (DNA, Complementary)\\nSB  - IM\\nEIN - Genome Res. 2006 Jun;16(6):804. Morrin, Ryan [corrected to Morin, Ryan]\\nMH  - Animals\\nMH  - Cloning, Molecular/*methods\\nMH  - Computational Biology\\nMH  - DNA Primers\\nMH  - *DNA, Complementary/genetics/metabolism\\nMH  - *Gene Library\\nMH  - Humans\\nMH  - Mice\\nMH  - National Institutes of Health (U.S.)\\nMH  - Open Reading Frames/*physiology\\nMH  - Rats\\nMH  - United States\\nMH  - Xenopus laevis/genetics\\nMH  - Zebrafish/genetics\\nPMC - PMC528928\\nEDAT- 2004/10/19 09:00\\nMHDA- 2004/11/17 09:00\\nCRDT- 2004/10/19 09:00\\nPHST- 2004/10/19 09:00 [pubmed]\\nPHST- 2004/11/17 09:00 [medline]\\nPHST- 2004/10/19 09:00 [entrez]\\nAID - 14/10b/2121 [pii]\\nAID - 10.1101/gr.2596504 [doi]\\nPST - ppublish\\nSO  - Genome Res. 2004 Oct;14(10B):2121-7. doi: 10.1101/gr.2596504.\", \"PMID- 14702039\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20040202\\nLR  - 20061115\\nIS  - 1061-4036 (Print)\\nIS  - 1061-4036 (Linking)\\nVI  - 36\\nIP  - 1\\nDP  - 2004 Jan\\nTI  - Complete sequencing and characterization of 21,243 full-length human cDNAs.\\nPG  - 40-5\\nAB  - As a base for human transcriptome and functional genomics, we created the\\n      \\\"full-length long Japan\\\" (FLJ) collection of sequenced human cDNAs. We determined\\n      the entire sequence of 21,243 selected clones and found that 14,490 cDNAs (10,897\\n      clusters) were unique to the FLJ collection. About half of them (5,416) seemed to\\n      be protein-coding. Of those, 1,999 clusters had not been predicted by\\n      computational methods. The distribution of GC content of nonpredicted cDNAs had a\\n      peak at approximately 58% compared with a peak at approximately 42%for predicted \\n      cDNAs. Thus, there seems to be a slight bias against GC-rich transcripts in\\n      current gene prediction procedures. The rest of the cDNAs unique to the FLJ\\n      collection (5,481) contained no obvious open reading frames (ORFs) and thus are\\n      candidate noncoding RNAs. About one-fourth of them (1,378) showed a clear pattern\\n      of splicing. The distribution of GC content of noncoding cDNAs was narrow and had\\n      a peak at approximately 42%, relatively low compared with that of protein-coding \\n      cDNAs.\\nFAU - Ota, Toshio\\nAU  - Ota T\\nAD  - Helix Research Institute, 1532-3 Yana, Kisarazu, Chiba 292-0812, Japan.\\nFAU - Suzuki, Yutaka\\nAU  - Suzuki Y\\nFAU - Nishikawa, Tetsuo\\nAU  - Nishikawa T\\nFAU - Otsuki, Tetsuji\\nAU  - Otsuki T\\nFAU - Sugiyama, Tomoyasu\\nAU  - Sugiyama T\\nFAU - Irie, Ryotaro\\nAU  - Irie R\\nFAU - Wakamatsu, Ai\\nAU  - Wakamatsu A\\nFAU - Hayashi, Koji\\nAU  - Hayashi K\\nFAU - Sato, Hiroyuki\\nAU  - Sato H\\nFAU - Nagai, Keiichi\\nAU  - Nagai K\\nFAU - Kimura, Kouichi\\nAU  - Kimura K\\nFAU - Makita, Hiroshi\\nAU  - Makita H\\nFAU - Sekine, Mitsuo\\nAU  - Sekine M\\nFAU - Obayashi, Masaya\\nAU  - Obayashi M\\nFAU - Nishi, Tatsunari\\nAU  - Nishi T\\nFAU - Shibahara, Toshikazu\\nAU  - Shibahara T\\nFAU - Tanaka, Toshihiro\\nAU  - Tanaka T\\nFAU - Ishii, Shizuko\\nAU  - Ishii S\\nFAU - Yamamoto, Jun-ichi\\nAU  - Yamamoto J\\nFAU - Saito, Kaoru\\nAU  - Saito K\\nFAU - Kawai, Yuri\\nAU  - Kawai Y\\nFAU - Isono, Yuko\\nAU  - Isono Y\\nFAU - Nakamura, Yoshitaka\\nAU  - Nakamura Y\\nFAU - Nagahari, Kenji\\nAU  - Nagahari K\\nFAU - Murakami, Katsuhiko\\nAU  - Murakami K\\nFAU - Yasuda, Tomohiro\\nAU  - Yasuda T\\nFAU - Iwayanagi, Takao\\nAU  - Iwayanagi T\\nFAU - Wagatsuma, Masako\\nAU  - Wagatsuma M\\nFAU - Shiratori, Akiko\\nAU  - Shiratori A\\nFAU - Sudo, Hiroaki\\nAU  - Sudo H\\nFAU - Hosoiri, Takehiko\\nAU  - Hosoiri T\\nFAU - Kaku, Yoshiko\\nAU  - Kaku Y\\nFAU - Kodaira, Hiroyo\\nAU  - Kodaira H\\nFAU - Kondo, Hiroshi\\nAU  - Kondo H\\nFAU - Sugawara, Masanori\\nAU  - Sugawara M\\nFAU - Takahashi, Makiko\\nAU  - Takahashi M\\nFAU - Kanda, Katsuhiro\\nAU  - Kanda K\\nFAU - Yokoi, Takahide\\nAU  - Yokoi T\\nFAU - Furuya, Takako\\nAU  - Furuya T\\nFAU - Kikkawa, Emiko\\nAU  - Kikkawa E\\nFAU - Omura, Yuhi\\nAU  - Omura Y\\nFAU - Abe, Kumi\\nAU  - Abe K\\nFAU - Kamihara, Kumiko\\nAU  - Kamihara K\\nFAU - Katsuta, Naoko\\nAU  - Katsuta N\\nFAU - Sato, Kazuomi\\nAU  - Sato K\\nFAU - Tanikawa, Machiko\\nAU  - Tanikawa M\\nFAU - Yamazaki, Makoto\\nAU  - Yamazaki M\\nFAU - Ninomiya, Ken\\nAU  - Ninomiya K\\nFAU - Ishibashi, Tadashi\\nAU  - Ishibashi T\\nFAU - Yamashita, Hiromichi\\nAU  - Yamashita H\\nFAU - Murakawa, Katsuji\\nAU  - Murakawa K\\nFAU - Fujimori, Kiyoshi\\nAU  - Fujimori K\\nFAU - Tanai, Hiroyuki\\nAU  - Tanai H\\nFAU - Kimata, Manabu\\nAU  - Kimata M\\nFAU - Watanabe, Motoji\\nAU  - Watanabe M\\nFAU - Hiraoka, Susumu\\nAU  - Hiraoka S\\nFAU - Chiba, Yoshiyuki\\nAU  - Chiba Y\\nFAU - Ishida, Shinichi\\nAU  - Ishida S\\nFAU - Ono, Yukio\\nAU  - Ono Y\\nFAU - Takiguchi, Sumiyo\\nAU  - Takiguchi S\\nFAU - Watanabe, Susumu\\nAU  - Watanabe S\\nFAU - Yosida, Makoto\\nAU  - Yosida M\\nFAU - Hotuta, Tomoko\\nAU  - Hotuta T\\nFAU - Kusano, Junko\\nAU  - Kusano J\\nFAU - Kanehori, Keiichi\\nAU  - Kanehori K\\nFAU - Takahashi-Fujii, Asako\\nAU  - Takahashi-Fujii A\\nFAU - Hara, Hiroto\\nAU  - Hara H\\nFAU - Tanase, Tomo-o\\nAU  - Tanase TO\\nFAU - Nomura, Yoshiko\\nAU  - Nomura Y\\nFAU - Togiya, Sakae\\nAU  - Togiya S\\nFAU - Komai, Fukuyo\\nAU  - Komai F\\nFAU - Hara, Reiko\\nAU  - Hara R\\nFAU - Takeuchi, Kazuha\\nAU  - Takeuchi K\\nFAU - Arita, Miho\\nAU  - Arita M\\nFAU - Imose, Nobuyuki\\nAU  - Imose N\\nFAU - Musashino, Kaoru\\nAU  - Musashino K\\nFAU - Yuuki, Hisatsugu\\nAU  - Yuuki H\\nFAU - Oshima, Atsushi\\nAU  - Oshima A\\nFAU - Sasaki, Naokazu\\nAU  - Sasaki N\\nFAU - Aotsuka, Satoshi\\nAU  - Aotsuka S\\nFAU - Yoshikawa, Yoko\\nAU  - Yoshikawa Y\\nFAU - Matsunawa, Hiroshi\\nAU  - Matsunawa H\\nFAU - Ichihara, Tatsuo\\nAU  - Ichihara T\\nFAU - Shiohata, Namiko\\nAU  - Shiohata N\\nFAU - Sano, Sanae\\nAU  - Sano S\\nFAU - Moriya, Shogo\\nAU  - Moriya S\\nFAU - Momiyama, Hiroko\\nAU  - Momiyama H\\nFAU - Satoh, Noriko\\nAU  - Satoh N\\nFAU - Takami, Sachiko\\nAU  - Takami S\\nFAU - Terashima, Yuko\\nAU  - Terashima Y\\nFAU - Suzuki, Osamu\\nAU  - Suzuki O\\nFAU - Nakagawa, Satoshi\\nAU  - Nakagawa S\\nFAU - Senoh, Akihiro\\nAU  - Senoh A\\nFAU - Mizoguchi, Hiroshi\\nAU  - Mizoguchi H\\nFAU - Goto, Yoshihiro\\nAU  - Goto Y\\nFAU - Shimizu, Fumio\\nAU  - Shimizu F\\nFAU - Wakebe, Hirokazu\\nAU  - Wakebe H\\nFAU - Hishigaki, Haretsugu\\nAU  - Hishigaki H\\nFAU - Watanabe, Takeshi\\nAU  - Watanabe T\\nFAU - Sugiyama, Akio\\nAU  - Sugiyama A\\nFAU - Takemoto, Makoto\\nAU  - Takemoto M\\nFAU - Kawakami, Bunsei\\nAU  - Kawakami B\\nFAU - Yamazaki, Masaaki\\nAU  - Yamazaki M\\nFAU - Watanabe, Koji\\nAU  - Watanabe K\\nFAU - Kumagai, Ayako\\nAU  - Kumagai A\\nFAU - Itakura, Shoko\\nAU  - Itakura S\\nFAU - Fukuzumi, Yasuhito\\nAU  - Fukuzumi Y\\nFAU - Fujimori, Yoshifumi\\nAU  - Fujimori Y\\nFAU - Komiyama, Megumi\\nAU  - Komiyama M\\nFAU - Tashiro, Hiroyuki\\nAU  - Tashiro H\\nFAU - Tanigami, Akira\\nAU  - Tanigami A\\nFAU - Fujiwara, Tsutomu\\nAU  - Fujiwara T\\nFAU - Ono, Toshihide\\nAU  - Ono T\\nFAU - Yamada, Katsue\\nAU  - Yamada K\\nFAU - Fujii, Yuka\\nAU  - Fujii Y\\nFAU - Ozaki, Kouichi\\nAU  - Ozaki K\\nFAU - Hirao, Maasa\\nAU  - Hirao M\\nFAU - Ohmori, Yoshihiro\\nAU  - Ohmori Y\\nFAU - Kawabata, Ayako\\nAU  - Kawabata A\\nFAU - Hikiji, Takeshi\\nAU  - Hikiji T\\nFAU - Kobatake, Naoko\\nAU  - Kobatake N\\nFAU - Inagaki, Hiromi\\nAU  - Inagaki H\\nFAU - Ikema, Yasuko\\nAU  - Ikema Y\\nFAU - Okamoto, Sachiko\\nAU  - Okamoto S\\nFAU - Okitani, Rie\\nAU  - Okitani R\\nFAU - Kawakami, Takuma\\nAU  - Kawakami T\\nFAU - Noguchi, Saori\\nAU  - Noguchi S\\nFAU - Itoh, Tomoko\\nAU  - Itoh T\\nFAU - Shigeta, Keiko\\nAU  - Shigeta K\\nFAU - Senba, Tadashi\\nAU  - Senba T\\nFAU - Matsumura, Kyoka\\nAU  - Matsumura K\\nFAU - Nakajima, Yoshie\\nAU  - Nakajima Y\\nFAU - Mizuno, Takae\\nAU  - Mizuno T\\nFAU - Morinaga, Misato\\nAU  - Morinaga M\\nFAU - Sasaki, Masahide\\nAU  - Sasaki M\\nFAU - Togashi, Takushi\\nAU  - Togashi T\\nFAU - Oyama, Masaaki\\nAU  - Oyama M\\nFAU - Hata, Hiroko\\nAU  - Hata H\\nFAU - Watanabe, Manabu\\nAU  - Watanabe M\\nFAU - Komatsu, Takami\\nAU  - Komatsu T\\nFAU - Mizushima-Sugano, Junko\\nAU  - Mizushima-Sugano J\\nFAU - Satoh, Tadashi\\nAU  - Satoh T\\nFAU - Shirai, Yuko\\nAU  - Shirai Y\\nFAU - Takahashi, Yukiko\\nAU  - Takahashi Y\\nFAU - Nakagawa, Kiyomi\\nAU  - Nakagawa K\\nFAU - Okumura, Koji\\nAU  - Okumura K\\nFAU - Nagase, Takahiro\\nAU  - Nagase T\\nFAU - Nomura, Nobuo\\nAU  - Nomura N\\nFAU - Kikuchi, Hisashi\\nAU  - Kikuchi H\\nFAU - Masuho, Yasuhiko\\nAU  - Masuho Y\\nFAU - Yamashita, Riu\\nAU  - Yamashita R\\nFAU - Nakai, Kenta\\nAU  - Nakai K\\nFAU - Yada, Tetsushi\\nAU  - Yada T\\nFAU - Nakamura, Yusuke\\nAU  - Nakamura Y\\nFAU - Ohara, Osamu\\nAU  - Ohara O\\nFAU - Isogai, Takao\\nAU  - Isogai T\\nFAU - Sugano, Sumio\\nAU  - Sugano S\\nLA  - eng\\nPT  - Comparative Study\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov't\\nDEP - 20031221\\nPL  - United States\\nTA  - Nat Genet\\nJT  - Nature genetics\\nJID - 9216904\\nRN  - 0 (DNA, Complementary)\\nRN  - 0 (RNA, Messenger)\\nSB  - IM\\nMH  - Chromosomes, Human, 21-22 and Y\\nMH  - Chromosomes, Human, Pair 20\\nMH  - Computational Biology\\nMH  - *DNA, Complementary\\nMH  - Humans\\nMH  - Open Reading Frames\\nMH  - RNA, Messenger\\nMH  - *Sequence Analysis, DNA\\nEDAT- 2004/01/01 05:00\\nMHDA- 2004/02/03 05:00\\nCRDT- 2004/01/01 05:00\\nPHST- 2003/10/07 00:00 [received]\\nPHST- 2003/12/01 00:00 [accepted]\\nPHST- 2004/01/01 05:00 [pubmed]\\nPHST- 2004/02/03 05:00 [medline]\\nPHST- 2004/01/01 05:00 [entrez]\\nAID - 10.1038/ng1285 [doi]\\nAID - ng1285 [pii]\\nPST - ppublish\\nSO  - Nat Genet. 2004 Jan;36(1):40-5. doi: 10.1038/ng1285. Epub 2003 Dec 21.\", \"PMID- 14681893\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20040114\\nLR  - 20071114\\nIS  - 0364-5134 (Print)\\nIS  - 0364-5134 (Linking)\\nVI  - 54\\nIP  - 6\\nDP  - 2003 Dec\\nTI  - Pathogenesis of clinical signs in recessive ataxia with saccadic intrusions.\\nPG  - 824-8\\nAB  - We describe a family of Slovenian descent with progressive ataxia, corticospinal \\n      signs, axonal sensorimotor neuropathy, and disruption of visual fixation by\\n      saccadic intrusions. Chromosome mapping indicated a mutation on 1p36, and this\\n      recessive disorder has been designated spinocerebellar ataxia with saccadic\\n      intrusions. Affected patients showed overshooting horizontal saccades,\\n      macrosaccadic oscillations, and increased velocity of larger saccades; other eye \\n      movements were normal. Slowed conduction in axons that are selectively vulnerable\\n      to the molecular defect could explain both the sensorimotor neuropathy and the\\n      saccadic disorder, which would be caused by delayed feedback control because of\\n      slow conduction in cerebellar parallel fibers.\\nFAU - Swartz, Barbara E\\nAU  - Swartz BE\\nAD  - Department of Neurology, Veterans Affairs Medical Center, and University\\n      Hospitals, Case Western Reserve University, 11100 Euclid Avenue, Cleveland, OH\\n      44106-5040, USA.\\nFAU - Li, Sheng\\nAU  - Li S\\nFAU - Bespalova, Irina\\nAU  - Bespalova I\\nFAU - Burmeister, Margit\\nAU  - Burmeister M\\nFAU - Dulaney, Eugene\\nAU  - Dulaney E\\nFAU - Robinson, Farrel R\\nAU  - Robinson FR\\nFAU - Leigh, R John\\nAU  - Leigh RJ\\nLA  - eng\\nGR  - EY 06717/EY/NEI NIH HHS/United States\\nGR  - EY 10578/EY/NEI NIH HHS/United States\\nGR  - RR 00166/RR/NCRR NIH HHS/United States\\nPT  - Comparative Study\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov't\\nPT  - Research Support, U.S. Gov't, Non-P.H.S.\\nPT  - Research Support, U.S. Gov't, P.H.S.\\nPL  - United States\\nTA  - Ann Neurol\\nJT  - Annals of neurology\\nJID - 7707449\\nSB  - IM\\nMH  - Adult\\nMH  - Chromosomes, Human, Pair 1/genetics\\nMH  - Confidence Intervals\\nMH  - Female\\nMH  - *Genes, Recessive\\nMH  - Humans\\nMH  - Male\\nMH  - Middle Aged\\nMH  - Ocular Motility Disorders/complications/*genetics/pathology\\nMH  - Spinocerebellar Degenerations/complications/*genetics/pathology\\nEDAT- 2003/12/19 05:00\\nMHDA- 2004/01/15 05:00\\nCRDT- 2003/12/19 05:00\\nPHST- 2003/12/19 05:00 [pubmed]\\nPHST- 2004/01/15 05:00 [medline]\\nPHST- 2003/12/19 05:00 [entrez]\\nAID - 10.1002/ana.10758 [doi]\\nPST - ppublish\\nSO  - Ann Neurol. 2003 Dec;54(6):824-8. doi: 10.1002/ana.10758.\", \"PMID- 12477932\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20030121\\nLR  - 20190208\\nIS  - 0027-8424 (Print)\\nIS  - 0027-8424 (Linking)\\nVI  - 99\\nIP  - 26\\nDP  - 2002 Dec 24\\nTI  - Generation and initial analysis of more than 15,000 full-length human and mouse\\n      cDNA sequences.\\nPG  - 16899-903\\nAB  - The National Institutes of Health Mammalian Gene Collection (MGC) Program is a\\n      multiinstitutional effort to identify and sequence a cDNA clone containing a\\n      complete ORF for each human and mouse gene. ESTs were generated from libraries\\n      enriched for full-length cDNAs and analyzed to identify candidate full-ORF\\n      clones, which then were sequenced to high accuracy. The MGC has currently\\n      sequenced and verified the full ORF for a nonredundant set of >9,000 human and\\n      >6,000 mouse genes. Candidate full-ORF clones for an additional 7,800 human and\\n      3,500 mouse genes also have been identified. All MGC sequences and clones are\\n      available without restriction through public databases and clone distribution\\n      networks (see http:mgc.nci.nih.gov).\\nFAU - Strausberg, Robert L\\nAU  - Strausberg RL\\nAD  - National Cancer Institute, Bethessda, MD 20892-2580, USA. rls@nih.gov\\nFAU - Feingold, Elise A\\nAU  - Feingold EA\\nFAU - Grouse, Lynette H\\nAU  - Grouse LH\\nFAU - Derge, Jeffery G\\nAU  - Derge JG\\nFAU - Klausner, Richard D\\nAU  - Klausner RD\\nFAU - Collins, Francis S\\nAU  - Collins FS\\nFAU - Wagner, Lukas\\nAU  - Wagner L\\nFAU - Shenmen, Carolyn M\\nAU  - Shenmen CM\\nFAU - Schuler, Gregory D\\nAU  - Schuler GD\\nFAU - Altschul, Stephen F\\nAU  - Altschul SF\\nFAU - Zeeberg, Barry\\nAU  - Zeeberg B\\nFAU - Buetow, Kenneth H\\nAU  - Buetow KH\\nFAU - Schaefer, Carl F\\nAU  - Schaefer CF\\nFAU - Bhat, Narayan K\\nAU  - Bhat NK\\nFAU - Hopkins, Ralph F\\nAU  - Hopkins RF\\nFAU - Jordan, Heather\\nAU  - Jordan H\\nFAU - Moore, Troy\\nAU  - Moore T\\nFAU - Max, Steve I\\nAU  - Max SI\\nFAU - Wang, Jun\\nAU  - Wang J\\nFAU - Hsieh, Florence\\nAU  - Hsieh F\\nFAU - Diatchenko, Luda\\nAU  - Diatchenko L\\nFAU - Marusina, Kate\\nAU  - Marusina K\\nFAU - Farmer, Andrew A\\nAU  - Farmer AA\\nFAU - Rubin, Gerald M\\nAU  - Rubin GM\\nFAU - Hong, Ling\\nAU  - Hong L\\nFAU - Stapleton, Mark\\nAU  - Stapleton M\\nFAU - Soares, M Bento\\nAU  - Soares MB\\nFAU - Bonaldo, Maria F\\nAU  - Bonaldo MF\\nFAU - Casavant, Tom L\\nAU  - Casavant TL\\nFAU - Scheetz, Todd E\\nAU  - Scheetz TE\\nFAU - Brownstein, Michael J\\nAU  - Brownstein MJ\\nFAU - Usdin, Ted B\\nAU  - Usdin TB\\nFAU - Toshiyuki, Shiraki\\nAU  - Toshiyuki S\\nFAU - Carninci, Piero\\nAU  - Carninci P\\nFAU - Prange, Christa\\nAU  - Prange C\\nFAU - Raha, Sam S\\nAU  - Raha SS\\nFAU - Loquellano, Naomi A\\nAU  - Loquellano NA\\nFAU - Peters, Garrick J\\nAU  - Peters GJ\\nFAU - Abramson, Rick D\\nAU  - Abramson RD\\nFAU - Mullahy, Sara J\\nAU  - Mullahy SJ\\nFAU - Bosak, Stephanie A\\nAU  - Bosak SA\\nFAU - McEwan, Paul J\\nAU  - McEwan PJ\\nFAU - McKernan, Kevin J\\nAU  - McKernan KJ\\nFAU - Malek, Joel A\\nAU  - Malek JA\\nFAU - Gunaratne, Preethi H\\nAU  - Gunaratne PH\\nFAU - Richards, Stephen\\nAU  - Richards S\\nFAU - Worley, Kim C\\nAU  - Worley KC\\nFAU - Hale, Sarah\\nAU  - Hale S\\nFAU - Garcia, Angela M\\nAU  - Garcia AM\\nFAU - Gay, Laura J\\nAU  - Gay LJ\\nFAU - Hulyk, Stephen W\\nAU  - Hulyk SW\\nFAU - Villalon, Debbie K\\nAU  - Villalon DK\\nFAU - Muzny, Donna M\\nAU  - Muzny DM\\nFAU - Sodergren, Erica J\\nAU  - Sodergren EJ\\nFAU - Lu, Xiuhua\\nAU  - Lu X\\nFAU - Gibbs, Richard A\\nAU  - Gibbs RA\\nFAU - Fahey, Jessica\\nAU  - Fahey J\\nFAU - Helton, Erin\\nAU  - Helton E\\nFAU - Ketteman, Mark\\nAU  - Ketteman M\\nFAU - Madan, Anuradha\\nAU  - Madan A\\nFAU - Rodrigues, Stephanie\\nAU  - Rodrigues S\\nFAU - Sanchez, Amy\\nAU  - Sanchez A\\nFAU - Whiting, Michelle\\nAU  - Whiting M\\nFAU - Madan, Anup\\nAU  - Madan A\\nFAU - Young, Alice C\\nAU  - Young AC\\nFAU - Shevchenko, Yuriy\\nAU  - Shevchenko Y\\nFAU - Bouffard, Gerard G\\nAU  - Bouffard GG\\nFAU - Blakesley, Robert W\\nAU  - Blakesley RW\\nFAU - Touchman, Jeffrey W\\nAU  - Touchman JW\\nFAU - Green, Eric D\\nAU  - Green ED\\nFAU - Dickson, Mark C\\nAU  - Dickson MC\\nFAU - Rodriguez, Alex C\\nAU  - Rodriguez AC\\nFAU - Grimwood, Jane\\nAU  - Grimwood J\\nFAU - Schmutz, Jeremy\\nAU  - Schmutz J\\nFAU - Myers, Richard M\\nAU  - Myers RM\\nFAU - Butterfield, Yaron S N\\nAU  - Butterfield YS\\nFAU - Krzywinski, Martin I\\nAU  - Krzywinski MI\\nFAU - Skalska, Ursula\\nAU  - Skalska U\\nFAU - Smailus, Duane E\\nAU  - Smailus DE\\nFAU - Schnerch, Angelique\\nAU  - Schnerch A\\nFAU - Schein, Jacqueline E\\nAU  - Schein JE\\nFAU - Jones, Steven J M\\nAU  - Jones SJ\\nFAU - Marra, Marco A\\nAU  - Marra MA\\nCN  - Mammalian Gene Collection Program Team\\nLA  - eng\\nPT  - Journal Article\\nPT  - Research Support, U.S. Gov't, P.H.S.\\nDEP - 20021211\\nPL  - United States\\nTA  - Proc Natl Acad Sci U S A\\nJT  - Proceedings of the National Academy of Sciences of the United States of America\\nJID - 7505876\\nRN  - 0 (DNA, Complementary)\\nSB  - IM\\nMH  - Algorithms\\nMH  - Animals\\nMH  - DNA, Complementary/analysis/*chemistry\\nMH  - Gene Library\\nMH  - Humans\\nMH  - Mice\\nMH  - Open Reading Frames\\nMH  - *Sequence Analysis, DNA\\nPMC - PMC139241\\nEDAT- 2002/12/13 04:00\\nMHDA- 2003/01/22 04:00\\nCRDT- 2002/12/13 04:00\\nPHST- 2002/12/13 04:00 [pubmed]\\nPHST- 2003/01/22 04:00 [medline]\\nPHST- 2002/12/13 04:00 [entrez]\\nAID - 10.1073/pnas.242603899 [doi]\\nAID - 242603899 [pii]\\nPST - ppublish\\nSO  - Proc Natl Acad Sci U S A. 2002 Dec 24;99(26):16899-903. doi:\\n      10.1073/pnas.242603899. Epub 2002 Dec 11.\", \"PMID- 12421765\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20030114\\nLR  - 20181113\\nIS  - 1088-9051 (Print)\\nIS  - 1088-9051 (Linking)\\nVI  - 12\\nIP  - 11\\nDP  - 2002 Nov\\nTI  - Protein-protein interactions between large proteins: two-hybrid screening using a\\n      functionally classified library composed of long cDNAs.\\nPG  - 1773-84\\nAB  - Large proteins have multiple domains that are potentially capable of binding many\\n      kinds of partners. It is conceivable, therefore, that such proteins could\\n      function as an intricate framework of assembly protein complexes. To\\n      comprehensively study protein-protein interactions between large KIAA proteins,\\n      we have constructed a library composed of 1087 KIAA cDNA clones based on prior\\n      functional classifications done in silico. We were guided by two principles that \\n      raise the success rate for detecting interactions per tested combination: we\\n      avoided testing low-probability combinations, and reduced the number of potential\\n      false negatives that arise from the fact that large proteins cannot reliably be\\n      expressed in yeast. The latter was addressed by constructing a cDNA library\\n      comprised of random fragments encoding large proteins. Cytoplasmic domains of\\n      KIAA transmembrane proteins (>1000 amino acids) were used as bait for yeast\\n      two-hybrid screening. Our analyses reveal that several KIAA proteins bearing a\\n      transmembrane region have the capability of binding to other KIAA proteins\\n      containing domains (e.g., PDZ, SH3, rhoGEF, and spectrin) known to be localized\\n      to highly specialized submembranous sites, indicating that they participate in\\n      cellular junction formation, receptor or channel clustering, and intracellular\\n      signaling events. Our representative library should be a very useful resource for\\n      detecting previously unidentified interactions because it complements\\n      conventional expression libraries, which seldom contain large cDNAs.\\nFAU - Nakayama, Manabu\\nAU  - Nakayama M\\nAD  - Department of Human Gene Research, Kazusa DNA Research Institute, Kisarazu, Chiba\\n      292-0818, Japan. nmanabu@kazusa.or.jp\\nFAU - Kikuno, Reiko\\nAU  - Kikuno R\\nFAU - Ohara, Osamu\\nAU  - Ohara O\\nLA  - eng\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov't\\nPL  - United States\\nTA  - Genome Res\\nJT  - Genome research\\nJID - 9518021\\nRN  - 0 (DNA, Complementary)\\nRN  - 0 (Peptides)\\nRN  - 0 (Proteins)\\nSB  - IM\\nMH  - Brain Chemistry/genetics\\nMH  - Cloning, Molecular\\nMH  - Computational Biology/instrumentation/methods\\nMH  - Cytoplasm/chemistry\\nMH  - DNA, Complementary/chemistry/*genetics\\nMH  - *Gene Library\\nMH  - Humans\\nMH  - Leukemia, Myeloid/genetics/pathology\\nMH  - Peptide Mapping/methods\\nMH  - Peptides/genetics/physiology\\nMH  - Protein Interaction Mapping/*methods\\nMH  - Protein Structure, Tertiary/physiology\\nMH  - Proteins/chemistry/genetics/physiology\\nMH  - Software\\nMH  - Tumor Cells, Cultured\\nMH  - *Two-Hybrid System Techniques\\nMH  - src Homology Domains/genetics/physiology\\nPMC - PMC187542\\nEDAT- 2002/11/08 04:00\\nMHDA- 2003/01/15 04:00\\nCRDT- 2002/11/08 04:00\\nPHST- 2002/11/08 04:00 [pubmed]\\nPHST- 2003/01/15 04:00 [medline]\\nPHST- 2002/11/08 04:00 [entrez]\\nAID - 10.1101/gr.406902 [doi]\\nPST - ppublish\\nSO  - Genome Res. 2002 Nov;12(11):1773-84. doi: 10.1101/gr.406902.\", \"PMID- 11960835\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20020606\\nLR  - 20190616\\nIS  - 0077-8923 (Print)\\nIS  - 0077-8923 (Linking)\\nVI  - 956\\nDP  - 2002 Apr\\nTI  - A form of inherited cerebellar ataxia with saccadic intrusions, increased\\n      saccadic speed, sensory neuropathy, and myoclonus.\\nPG  - 441-4\\nFAU - Swartz, Barbara E\\nAU  - Swartz BE\\nAD  - Veterans' Affairs Medical Center and Case Western University, Cleveland, Ohio\\n      44106, USA. Barbara.Swartz@uhhs.com\\nFAU - Burmeister, Margit\\nAU  - Burmeister M\\nFAU - Somers, Jeffrey T\\nAU  - Somers JT\\nFAU - Rottach, Klaus G\\nAU  - Rottach KG\\nFAU - Bespalova, Irina N\\nAU  - Bespalova IN\\nFAU - Leigh, R John\\nAU  - Leigh RJ\\nLA  - eng\\nGR  - EY06717/EY/NEI NIH HHS/United States\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov't\\nPT  - Research Support, U.S. Gov't, Non-P.H.S.\\nPT  - Research Support, U.S. Gov't, P.H.S.\\nPL  - United States\\nTA  - Ann N Y Acad Sci\\nJT  - Annals of the New York Academy of Sciences\\nJID - 7506858\\nSB  - IM\\nMH  - Brain/physiopathology\\nMH  - Cerebellar Ataxia/*genetics\\nMH  - Electroencephalography\\nMH  - Electromyography\\nMH  - Humans\\nMH  - Magnetic Resonance Imaging\\nMH  - Motor Neurons/physiology\\nMH  - Myoclonus/*genetics\\nMH  - Nuclear Family\\nMH  - Reaction Time\\nMH  - Saccades/*genetics\\nEDAT- 2002/04/19 10:00\\nMHDA- 2002/06/12 10:01\\nCRDT- 2002/04/19 10:00\\nPHST- 2002/04/19 10:00 [pubmed]\\nPHST- 2002/06/12 10:01 [medline]\\nPHST- 2002/04/19 10:00 [entrez]\\nAID - 10.1111/j.1749-6632.2002.tb02850.x [doi]\\nPST - ppublish\\nSO  - Ann N Y Acad Sci. 2002 Apr;956:441-4. doi: 10.1111/j.1749-6632.2002.tb02850.x.\", \"PMID- 9455484\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 19980227\\nLR  - 20190512\\nIS  - 1340-2838 (Print)\\nIS  - 1340-2838 (Linking)\\nVI  - 4\\nIP  - 5\\nDP  - 1997 Oct 31\\nTI  - Characterization of cDNA clones in size-fractionated cDNA libraries from human\\n      brain.\\nPG  - 345-9\\nAB  - To evaluate the size-fractionated cDNA libraries of human brain previously\\n      constructed (O. O'hara et al. DNA Research, 4, 53-59, 1997), the occurrence of\\n      chimeric clones and the content of clones with coding potentiality were analyzed \\n      using the randomly sampled clones with insert sizes of 5 to 7 kb. When the\\n      chromosomal location of 30 clones was determined by the radiation-hybrid mapping \\n      method, the map positions assigned from the 3'- and 5'-end sequences separately\\n      were coincident for 29 clones, suggesting that the occurrence of chimeric clones \\n      is at most 1/30. Using 91 clones mapped to chromosome 1, the content of clones\\n      that have the potentiality coding for proteins larger than 100 amino acid\\n      residues was estimated to be approximately 50% (46 out of 91 clones) on the basis\\n      of nucleotide sequence analysis and coding potentiality assay in vitro. No\\n      significant open reading frames were detected in the remaining clones. Although\\n      the clones coding for short peptides may not have been included in the above\\n      estimation, the libraries constructed from the whole brain mRNA fraction appear\\n      to contain a considerable amount of clones corresponding to the 5'-truncated\\n      transcripts in an unprocessed form and/or those with long 3'-untranslated\\n      regions.\\nFAU - Seki, N\\nAU  - Seki N\\nAD  - Kazusa DNA Research Institute, Chiba, Japan. nseki@kazusa.or.jp\\nFAU - Ohira, M\\nAU  - Ohira M\\nFAU - Nagase, T\\nAU  - Nagase T\\nFAU - Ishikawa, K\\nAU  - Ishikawa K\\nFAU - Miyajima, N\\nAU  - Miyajima N\\nFAU - Nakajima, D\\nAU  - Nakajima D\\nFAU - Nomura, N\\nAU  - Nomura N\\nFAU - Ohara, O\\nAU  - Ohara O\\nLA  - eng\\nSI  - GENBANK/AB007913\\nSI  - GENBANK/AB007914\\nSI  - GENBANK/AB007915\\nSI  - GENBANK/AB007916\\nSI  - GENBANK/AB007917\\nSI  - GENBANK/AB007918\\nSI  - GENBANK/AB007919\\nSI  - GENBANK/AB007920\\nSI  - GENBANK/AB007921\\nSI  - GENBANK/AB007922\\nSI  - GENBANK/AB007923\\nSI  - GENBANK/AB007924\\nSI  - GENBANK/AB007925\\nSI  - GENBANK/AB007926\\nSI  - GENBANK/AB007927\\nSI  - GENBANK/AB007928\\nSI  - GENBANK/AB007929\\nSI  - GENBANK/AB007930\\nSI  - GENBANK/AB007931\\nSI  - GENBANK/AB007932\\nSI  - GENBANK/AB007933\\nSI  - GENBANK/AB007934\\nSI  - GENBANK/AB007935\\nSI  - GENBANK/AB007936\\nSI  - GENBANK/AB007937\\nSI  - GENBANK/AB007938\\nSI  - GENBANK/AB007939\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov't\\nPL  - England\\nTA  - DNA Res\\nJT  - DNA research : an international journal for rapid publication of reports on genes\\n      and genomes\\nJID - 9423827\\nRN  - 0 (DNA, Complementary)\\nRN  - 0 (DNAJC6 protein, human)\\nRN  - 0 (HSP40 Heat-Shock Proteins)\\nRN  - 0 (RNA, Messenger)\\nSB  - IM\\nMH  - Animals\\nMH  - Brain Chemistry/*genetics\\nMH  - Chimera\\nMH  - *Chromosome Mapping\\nMH  - *Chromosomes, Human, Pair 1\\nMH  - Cloning, Molecular\\nMH  - DNA, Complementary/*chemistry/genetics/*isolation & purification\\nMH  - Electrophoresis, Polyacrylamide Gel\\nMH  - Gene Library\\nMH  - HSP40 Heat-Shock Proteins\\nMH  - Humans\\nMH  - Open Reading Frames\\nMH  - Polymerase Chain Reaction\\nMH  - RNA, Messenger/genetics\\nMH  - Repetitive Sequences, Nucleic Acid\\nMH  - Sequence Homology, Nucleic Acid\\nMH  - Sequence Tagged Sites\\nMH  - Transcription, Genetic\\nEDAT- 1998/02/10 00:00\\nMHDA- 1998/02/10 00:01\\nCRDT- 1998/02/10 00:00\\nPHST- 1998/02/10 00:00 [pubmed]\\nPHST- 1998/02/10 00:01 [medline]\\nPHST- 1998/02/10 00:00 [entrez]\\nAID - 10.1093/dnares/4.5.345 [doi]\\nPST - ppublish\\nSO  - DNA Res. 1997 Oct 31;4(5):345-9. doi: 10.1093/dnares/4.5.345.\", \"PMID- 9110174\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 19970617\\nLR  - 20191108\\nIS  - 1088-9051 (Print)\\nIS  - 1088-9051 (Linking)\\nVI  - 7\\nIP  - 4\\nDP  - 1997 Apr\\nTI  - Large-scale concatenation cDNA sequencing.\\nPG  - 353-8\\nAB  - A total of 100 kb of DNA derived from 69 individual human brain cDNA clones of\\n      0.7-2.0 kb were sequenced by concatenated cDNA sequencing (CCS), whereby multiple\\n      individual DNA fragments are sequenced simultaneously in a single shotgun\\n      library. The method yielded accurate sequences and a similar efficiency compared \\n      with other shotgun libraries constructed from single DNA fragments (> 20 kb).\\n      Computer analyses were carried out on 65 cDNA clone sequences and their\\n      corresponding end sequences to examine both nucleic acid and amino acid sequence \\n      similarities in the databases. Thirty-seven clones revealed no DNA database\\n      matches, 12 clones generated exact matches (> or = 98% identity), and 16 clones\\n      generated nonexact matches (57%-97% identity) to either known human or other\\n      species genes. Of those 28 matched clones, 8 had corresponding end sequences that\\n      failed to identify similarities. In a protein similarity search, 27 clone\\n      sequences displayed significant matches, whereas only 20 of the end sequences had\\n      matches to known protein sequences. Our data indicate that full-length cDNA\\n      insert sequences provide significantly more nucleic acid and protein sequence\\n      similarity matches than expressed sequence tags (ESTs) for database searching.\\nFAU - Yu, W\\nAU  - Yu W\\nFAU - Andersson, B\\nAU  - Andersson B\\nFAU - Worley, K C\\nAU  - Worley KC\\nFAU - Muzny, D M\\nAU  - Muzny DM\\nFAU - Ding, Y\\nAU  - Ding Y\\nFAU - Liu, W\\nAU  - Liu W\\nFAU - Ricafrente, J Y\\nAU  - Ricafrente JY\\nFAU - Wentland, M A\\nAU  - Wentland MA\\nFAU - Lennon, G\\nAU  - Lennon G\\nFAU - Gibbs, R A\\nAU  - Gibbs RA\\nLA  - eng\\nSI  - GENBANK/AF007128\\nSI  - GENBANK/AF007129\\nSI  - GENBANK/AF007130\\nSI  - GENBANK/AF007131\\nSI  - GENBANK/AF007132\\nSI  - GENBANK/AF007133\\nSI  - GENBANK/AF007134\\nSI  - GENBANK/AF007135\\nSI  - GENBANK/AF007136\\nSI  - GENBANK/AF007137\\nSI  - GENBANK/AF007138\\nSI  - GENBANK/AF007139\\nSI  - GENBANK/AF007140\\nSI  - GENBANK/AF007141\\nSI  - GENBANK/AF007142\\nSI  - GENBANK/AF007143\\nSI  - GENBANK/AF007144\\nSI  - GENBANK/AF007145\\nSI  - GENBANK/AF007146\\nSI  - GENBANK/AF007147\\nSI  - GENBANK/AF007148\\nSI  - GENBANK/AF007149\\nSI  - GENBANK/AF007150\\nSI  - GENBANK/AF007151\\nSI  - GENBANK/AF007152\\nSI  - GENBANK/AF007153\\nSI  - GENBANK/AF007154\\nGR  - P30 HG00210-05/HG/NHGRI NIH HHS/United States\\nGR  - R01 HG00823/HG/NHGRI NIH HHS/United States\\nGR  - 1F32 HG00169-01/HG/NHGRI NIH HHS/United States\\nPT  - Letter\\nPT  - Research Support, U.S. Gov't, Non-P.H.S.\\nPT  - Research Support, U.S. Gov't, P.H.S.\\nPL  - United States\\nTA  - Genome Res\\nJT  - Genome research\\nJID - 9518021\\nRN  - 0 (DNA Transposable Elements)\\nRN  - 0 (DNA, Complementary)\\nRN  - 0 (Proteins)\\nSB  - IM\\nMH  - DNA Transposable Elements\\nMH  - DNA, Complementary/chemistry/*genetics\\nMH  - Databases, Factual\\nMH  - Gene Library\\nMH  - Humans\\nMH  - Molecular Sequence Data\\nMH  - Proteins/chemistry/*genetics\\nMH  - Sequence Alignment/*methods\\nMH  - Sequence Analysis, DNA/*methods\\nMH  - Sequence Homology, Amino Acid\\nMH  - Sequence Homology, Nucleic Acid\\nMH  - Software\\nPMC - PMC139146\\nEDAT- 1997/04/01 00:00\\nMHDA- 1997/04/01 00:01\\nCRDT- 1997/04/01 00:00\\nPHST- 1997/04/01 00:00 [pubmed]\\nPHST- 1997/04/01 00:01 [medline]\\nPHST- 1997/04/01 00:00 [entrez]\\nAID - 10.1101/gr.7.4.353 [doi]\\nPST - ppublish\\nSO  - Genome Res. 1997 Apr;7(4):353-8. doi: 10.1101/gr.7.4.353.\", \"PMID- 8619474\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 19960613\\nLR  - 20071114\\nIS  - 0003-2697 (Print)\\nIS  - 0003-2697 (Linking)\\nVI  - 236\\nIP  - 1\\nDP  - 1996 Apr 5\\nTI  - A \\\"double adaptor\\\" method for improved shotgun library construction.\\nPG  - 107-13\\nAB  - The efficiency of shotgun DNA sequencing depends to a great extent on the quality\\n      of the random-subclone libraries used. We here describe a novel \\\"double adaptor\\\" \\n      strategy for efficient construction of high-quality shotgun libraries. In this\\n      method, randomly sheared and end-repaired fragments are ligated to\\n      oligonucleotide adaptors creating 12-base overhangs. Nonphosphorylated\\n      oligonucleotides are used, which prevents formation of adaptor dimers and ensures\\n      efficient ligation of insert to adaptor. The vector is prepared from a modified\\n      M13 vector, by KpnI/PstI digestion followed by ligation to oligonucleotides with \\n      ends complementary to the overhangs created in the digest. These adaptors create \\n      5'-overhangs complementary to those on the inserts. Following annealing of insert\\n      to vector, the DNA is directly used for transformation without a ligation step.\\n      This protocol is robust and shows three- to fivefold higher yield of clones\\n      compared to previous protocols. No chimeric clones can be detected and the\\n      background of clones without an insert is <1%. The procedure is rapid and shows\\n      potential for automation.\\nFAU - Andersson, B\\nAU  - Andersson B\\nAD  - Department of Molecular and Human Genetics, Baylor College of Medicine, One\\n      Baylor Plaza, Houston, Texas, 77030, USA.\\nFAU - Wentland, M A\\nAU  - Wentland MA\\nFAU - Ricafrente, J Y\\nAU  - Ricafrente JY\\nFAU - Liu, W\\nAU  - Liu W\\nFAU - Gibbs, R A\\nAU  - Gibbs RA\\nLA  - eng\\nSI  - GENBANK/AF007128\\nSI  - GENBANK/AF007129\\nSI  - GENBANK/AF007130\\nSI  - GENBANK/AF007131\\nSI  - GENBANK/AF007132\\nSI  - GENBANK/AF007133\\nSI  - GENBANK/AF007134\\nSI  - GENBANK/AF007135\\nSI  - GENBANK/AF007136\\nSI  - GENBANK/AF007137\\nSI  - GENBANK/AF007138\\nSI  - GENBANK/AF007139\\nSI  - GENBANK/AF007140\\nSI  - GENBANK/AF007141\\nSI  - GENBANK/AF007142\\nSI  - GENBANK/AF007143\\nSI  - GENBANK/AF007144\\nSI  - GENBANK/AF007145\\nSI  - GENBANK/AF007146\\nSI  - GENBANK/AF007147\\nSI  - GENBANK/AF007148\\nSI  - GENBANK/AF007149\\nSI  - GENBANK/AF007150\\nSI  - GENBANK/AF007151\\nSI  - GENBANK/AF007152\\nSI  - GENBANK/AF007153\\nSI  - GENBANK/AF007154\\nGR  - R01 HG00823/HG/NHGRI NIH HHS/United States\\nPT  - Journal Article\\nPT  - Research Support, U.S. Gov't, P.H.S.\\nPL  - United States\\nTA  - Anal Biochem\\nJT  - Analytical biochemistry\\nJID - 0370535\\nRN  - 0 (Oligonucleotides)\\nSB  - IM\\nMH  - Base Sequence\\nMH  - Cloning, Molecular/*methods\\nMH  - Genetic Vectors\\nMH  - *Genomic Library\\nMH  - Humans\\nMH  - Molecular Sequence Data\\nMH  - Oligonucleotides/chemistry\\nMH  - Polymerase Chain Reaction\\nMH  - X Chromosome\\nEDAT- 1996/04/05 00:00\\nMHDA- 1996/04/05 00:01\\nCRDT- 1996/04/05 00:00\\nPHST- 1996/04/05 00:00 [pubmed]\\nPHST- 1996/04/05 00:01 [medline]\\nPHST- 1996/04/05 00:00 [entrez]\\nAID - S0003-2697(96)90138-4 [pii]\\nAID - 10.1006/abio.1996.0138 [doi]\\nPST - ppublish\\nSO  - Anal Biochem. 1996 Apr 5;236(1):107-13. doi: 10.1006/abio.1996.0138.\"]"
      ]
     },
     "execution_count": 10,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "require 'net/http'\n",
    "require 'bio'\n",
    "require 'nokogiri'\n",
    "\n",
    "\n",
    "ncbi = Bio::NCBI::REST.new\n",
    "max = 5\n",
    "ids = ncbi.esearch('\"Wilkinson MD\"[AUTH]', {\"db\"=>\"nucleotide\", \"rettype\"=>\"gb\", \"retmax\" => max})\n",
    "idlist = ids.join(\",\")\n",
    "\n",
    "elink = URI(\"https://eutils.ncbi.nlm.nih.gov:443/entrez/eutils/elink.fcgi\")\n",
    "request = Net::HTTP::Post.new(elink)\n",
    "https = Net::HTTP.new(elink.host,elink.port)\n",
    "https.use_ssl = true  # tell the HTTP connection to use ecryption (SSL = Secure Socket Layer)\n",
    "\n",
    "request.set_form_data(\n",
    "  'email' => 'markw@illuminae.com',\n",
    "  'dbfrom' => 'nuccore', \n",
    "  'db' => 'pubmed',\n",
    "  'retmode' => 'xml',\n",
    "  'id' => idlist)\n",
    "\n",
    "output = https.request(request)   # make the request (we really should check if it is successful)\n",
    "linkset = output.body\n",
    "\n",
    "doc = Nokogiri::XML(linkset)\n",
    "pubmedids = []  # force it to be a string so that we can use '<<' operator\n",
    "doc.xpath('//LinkSet/LinkSetDb/Link').each do |node|    # the \"path\" (\"xpath\")to the data, starts with '//'\n",
    "   pubmedids << node.at_xpath('Id').content.to_s\n",
    "end\n",
    "\n",
    "Bio::PubMed.efetch(pubmedids).each do |pubmedrecord|\n",
    "  # pubmedrecord is a string of the record, in pubmed format\n",
    "  # we can use that to create a new Bio::MEDLINE object\n",
    "  medline = Bio::MEDLINE.new(pubmedrecord)\n",
    "  reference = medline.reference  # reference is a Bio::Reference object\n",
    "  puts reference.bibtex   # the .bibtex method creates the bibtex string of the Bio::Reference\n",
    "end"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "...and you can write that to a file and import it into EndNote, or Mendeley, or PaperPile, or whatever reference manager you prefer!\n",
    "\n",
    "\n",
    "DONE!\n"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Ruby 2.6.3",
   "language": "ruby",
   "name": "ruby"
  },
  "language_info": {
   "file_extension": ".rb",
   "mimetype": "application/x-ruby",
   "name": "ruby",
   "version": "2.6.3"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
